<!doctype HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">    
    <title>Seed Viewer - Organism</title>

    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/TabView.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/Hover.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/Table.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/web_app_default.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/commonviewer.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/seedviewer.css" >
       

    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/PopupTooltip.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/TabView.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Ajax.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Hover.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Tree.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Table.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/fade.js" ></script>
    
    <meta name="HTML_PATH" content="./Html" />
    
    <META HTTP-EQUIV="PRAGMA" CONTENT="NO-CACHE">
    
    <META HTTP-EQUIV="CACHE-CONTROL" CONTENT="NO-CACHE">
    
    <meta name="robots" content="noindex,nofollow" />
    
    

  </head>

  <body>
<div id="header">
  <span id="logo">
  <img height="95px" src="./Html/seed-logo-green.png" alt="The SEED" />
  </span>
  <p>
  <span id="title">
  <img src="./Html/seed-theseed-viewer.png" alt="The SEED Viewer" />
  </span>
  <span id="version">SEED Viewer version 2.0</span>
  </p>
  <p>Welcome to the SEED Viewer - a read-only browser of the curated SEED data. <br/>
  For more information about The SEED please visit <a href="http://www.theseed.org">theSEED.org.</a>

  </p>




</div>

<div id="topbar">
  <table style="width: 100%; border-spacing: 0px;">
    <tr>
      <td style='width: 100%; padding: 0px;'>
	
	  <div id='menu'>
	<ul id='nav'>
		<li><div><a href="?page=Home" >&raquo;Navigate</a></div>
		<ul>
			<li><a href="?page=Home" >Startpage</a></li>
			<li><a href="?page=OrganismSelect" >Organisms</a></li>
			<li><a href="?page=SubsystemSelect" >Browse Subsystems</a></li>
			<li><a href="https://rast.nmpdr.org/SubsysEditor.cgi" >Curate Subsystems</a></li>
			<li><a href="?page=Scenarios" >Scenarios</a></li>
			<li><a href="?page=FigFamViewer" >FigFams</a></li>
			<li><a href="?page=BlastRun" >BLAST Search</a></li>
			<li><a href="http://www.theseed.org/daily" >Visit the Daily SEED</a></li>
			<li><a href="?page=SubsystemPrimer" >Request a Subsystem</a></li>
		</ul>
		</li>
		<li><div><a  >&raquo;Organism</a></div>
		<ul>
			<li><a href="?page=Organism&organism=991.25" >General Information</a></li>
			<li><a href="?page=BrowseGenome&tabular=1&organism=991.25" >Feature Table</a></li>
			<li><a href="?page=BrowseGenome&organism=991.25" >Genome Browser</a></li>
			<li><a href="?page=Scenarios&organism=991.25" >Scenarios</a></li>
			<li><a href="?page=SubsystemSelect&organism=991.25" >Subsystems</a></li>
			<li><a href="?page=Export&organism=991.25" >Export</a></li>
		</ul>
		</li>
		<li><div><a  >&raquo;Comparative Tools</a></div>
		<ul>
			<li><a href="?page=CompareMetabolicReconstruction&organism=991.25" >Function based Comparison</a></li>
			<li><a href="?page=MultiGenomeCompare&organism=991.25" >Sequence based Comparison</a></li>
			<li><a href="?page=Kegg&organism=991.25" >Kegg Metabolic Analysis</a></li>
			<li><a href="?page=BlastRun&organism=991.25" >BLAST search</a></li>
		</ul>
		</li>
		<li><div><a href="http://www.theseed.org" target="help">&raquo;Help</a></div>
		<ul>
			<li><a href="http://www.theseed.org/wiki/index.php/Home_of_the_SEED" >What is the SEED</a></li>
			<li><a href="http://www.theseed.org/wiki/index.php/SEED_Viewer_Tutorial" >HowTo use the SEED Viewer</a></li>
			<li><a href="http://www.theseed.org/wiki/index.php/RAST_Tutorial" >Submitting Data to SEED</a></li>
			<li><a href="mailto:rast@mcs.anl.gov" >Contact</a></li>
			<li><a href="?page=Register" >Register</a></li>
			<li><a href="?page=RequestNewPassword" >I forgot my Password</a></li>
		</ul>
		</li>
	</ul>
</div>

	
      </td>
      <td style="border-right: 2px solid #5DA668; padding: 0px;">

	<div id='search'><form style='margin: 0px; padding: 0px;'><input type='submit' value='find' style='width:40px;cursor: pointer;'><input type='text' name='pattern'><input type='hidden' name='page' value='SearchResult'><input type='hidden' name='action' value='check_search'><input type='submit' style='display:none;'></form></div>

      </td>
      <td style="text-align: right; padding: 0px;">
	
	  <div id='user'><a href='seedviewer.cgi?page=AccountManagement'><img class='imglink' src='https://rast.nmpdr.org/Html/wac_people.png' title='Account Management' /></a>
<a href='seedviewer.cgi?page=Organism&organism=991.25&logout=1'><img class='imglink' src='https://rast.nmpdr.org/Html/wac_logout.png' title='Logout' /></a>
Ekaterina Pushmina</div>
	
      </td>
    </tr>
  </table>
</div>


 

  
  
  <div id="content">
    <script>function initialize_all () {

}</script><style>.hideme { display: none; }</style><h2>Organism Overview for Flavobacterium hydatis ATCC 29551 (991.25)</h2><table><tr><td width=550px><div><table><tr><th style='width: 170px;'>Genome</th><td>Flavobacterium hydatis ATCC 29551 (Taxonomy ID: <a href='http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=991&lvl=3&lin=f&keep=1&srchmode=1&unlock' target=outbound>991</a>)</td></tr><tr><th>Domain</th><td>Bacteria</td></tr><tr><th>Taxonomy</th><td>Bacteria; Bacteroidetes; Flavobacteriia; Flavobacteriales; Flavobacteriaceae; Flavobacterium; Flavobacterium hydatis ; Flavobacterium hydatis ATCC 29551</td></tr><tr><th>Neighbors</th><td><a href='seedviewer.cgi?page=ClosestNeighbors&organism=991.25'>View closest neighbors</a></td></tr><tr><th>Size</th><td>5,902,573</td></tr><tr><th>GC Content</th><td>32.7</td></tr><tr><th>N50</th><td>184642</td></tr><tr><th>L50</th><td>9</td></tr><tr><th>Number of Contigs (with PEGs)</th><td>72</td></tr><tr><th>Number of Subsystems</th><td>267</td></tr><tr><th>Number of Coding Sequences</th><td>5157</td></tr><tr><th>Number of RNAs</th><td>71</td></tr></table></div></td><td>For each genome we offer a wide set of information to browse, compare and download.<br><br><input type='hidden' id='OrganismInfoContent' value='0' /><table style='height: 125px; width: 325px;  border-spacing: 0px;border-collapse: expression("separate", cellSpacing = "0px");' id='tab_view_table_0'><tr><td class='tab_view_title_selected' name='0_tabs' id='0_tab_0' onclick='tab_view_select("0", "0");'>Browse</td><td class='tab_view_title' name='0_tabs' id='0_tab_1' onclick='tab_view_select("0", "1");'>Compare</td><td class='tab_view_title' name='0_tabs' id='0_tab_2' onclick='tab_view_select("0", "2");'>Download</td><td class='tab_view_title' name='0_tabs' id='0_tab_3' onclick='tab_view_select("0", "3");'>Annotate</td><td class='tab_view_title_filler'>&nbsp;</td></tr><tr><td colspan='5' class='tab_view_content_td'><div style='height: 100%; width: 100%;' class='tab_view_content_selected' name='0_contents' id='0_content_0'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">Browse through the features of <a href="seedviewer.cgi?page=BrowseGenome&organism=991.25">Flavobacterium hydatis ATCC 29551</a> both graphically and through a table. Both allow quick navigation and filtering for features of your interest. Each feature is linked to its own detail page.<br><br>Click <a href="seedviewer.cgi?page=BrowseGenome&organism=991.25">here</a> to get to the Genome Browser</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_1'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">Compare the metabolic reconstruction of this organism to that of another organism.<br><br>Available comparisons are <a href="seedviewer.cgi?page=CompareMetabolicReconstruction&organism=991.25">function based</a>, <a href="?page=MultiGenomeCompare&organism=991.25">sequence based</a> or via <a href="seedviewer.cgi?page=Kegg&organism=991.25">KEGG</a>. You can also <a href="seedviewer.cgi?page=BlastRun&organism=991.25">BLAST</a> against this organism.</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_2'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">You can export all information about this organism (e.g. annotations, scenarios, subsystems) into a variety of formats (e.g. EMBL, Excel) for further analysis on your own system.<br><br>Click <a href="seedviewer.cgi?page=Export&organism=991.25">here</a> to get to the Export page.</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_3'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">The SeedViewer has several options to edit the annotations of your private genome. To find out how to use these capabilities and enable them for your genomes, click <a href="?page=HowToAnnotate">here</a>.</div></div></td></tr></table>
<img src='https://rast.nmpdr.org/Html/clear.gif'  onload='initialize_tab_view("0"); tab_view_select("0", "0");'>
</td></tr></table><h2>Subsystem Information</h2><input type='hidden' id='SubsystemTabView' value='1' /><table style='height: 200px; width: 95%px;  border-spacing: 0px;border-collapse: expression("separate", cellSpacing = "0px");' id='tab_view_table_1'><tr><td class='tab_view_title_selected' name='1_tabs' id='1_tab_0' onclick='tab_view_select("1", "0");'>Subsystem Statistics</td><td class='tab_view_title' name='1_tabs' id='1_tab_1' onclick='tab_view_select("1", "1");'>Features in Subsystems</td><td class='tab_view_title_filler'>&nbsp;</td></tr><tr><td colspan='3' class='tab_view_content_td'><div style='height: 100%; width: 100%;' class='tab_view_content_selected' name='1_contents' id='1_content_0'><table style='padding-top: 25px;'><tr><td><b>Subsystem Coverage</b></td><td align=center><b>Subsystem Category Distribution</b></td><td align=center><b>Subsystem Feature Counts</b></td></tr><tr><td style='padding-right: 10px;'><img src="data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAAGQCAMAAAAOUBefAAAAolBMVEX///8AAADIyMjrBSg8PL4y/zLghQ8AAJv/3QAAm5vIZMgbhTSHQUEAWv9fZGRQ0pbh+qCqHpH/132MpdKgD/otm7mqzXgePL5zr6DChQ8AAH3h3QD/AAAA/wAAAP///wAA////AP//fwD/AH8A/39/AP9//wAAf///f39//39/f/8AAH9/AAAAfwB/fwB/AH8Af3//f////39///9QdJpdpmhNMpuCAAABFElEQVR4nO3cQQqDMBAF0PFO9v5Xa4yp1RYUndhN3xciEnjMTFy5MOIow7kcejE8zgQIBAKBQCAQeCNY9suSBCMaVa1oz9fBClUloj6svQtgq6iBS8Hplhdw6yVnGF9eAnxV2Wbaq+V6Jjkwll7bK5ivcDdAIBAIBAKBfcHun0zHMwECgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCDwR2DZL0sOjIpM92mZrhxYS4pWW13X3nXwXVgkwW3Ln16i5XHDdgTbMHuB8zR7zHDeG5fjToD7AQKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFA4H1g79/7i4iIiPxxnuAGEUzjfka/AAAAAElFTkSuQmCC" usemap="#imap_barchart_0"><map name='imap_barchart_0'>
<area shape='rect' coords='33,67,81,370' onmouseover='hover(event, "bar_0", "2");'onclick='window.top.location="?page=BrowseGenome&organism=991.25&tabular=1&noss=1"' style='cursor: pointer;'>
<area shape='rect' coords='33,0,81,67' onmouseover='hover(event, "bar_1", "2");'onclick='tab_view_select("1", 1);' style='cursor: pointer;'>
</map><input type='hidden' id='hover_redundancies_2' value='bar_0~#bar_1'><span style='display: none;' id='tooltip_2_bar_0'><table><tr><th>not in Subsystem</th></tr><tr><td>total (4222)</td></tr><tr><td>non-hypothetical (1739)</td></tr><tr><td>hypothetical (2483)</td></tr></table></span>
<span style='display: none;' id='tooltip_2_bar_1'><table><tr><th>in Subsystem</th></tr><tr><td>total (935)</td></tr><tr><td>non-hypothetical (900)</td></tr><tr><td>hypothetical (35)</td></tr></table></span>
</td><td style='padding-right: 10px;'><img style="border: none;" src="data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAZAAAAGQCAMAAAC3Ycb+AAABDlBMVEX///8AAADIyMjrBSg8PL4y/zL/AAAAgAAAAICAgACAAIDAwMCAgICZmf+ZM2b//8xmAGb/gIAAZszMzP//AP///wAA//+AAAAAgIAAAP8AzMzM///M/8z//5mZzP//mczMmf//zJkzZv8zzMyZzAD/zAD/mQD/ZgBmZpmWlpYAM2YzmWYAMwAzMwCZMwAzM5kzMzPghQ8AAJv/3QAAm5vIZMgbhTSHQUEAWv9fZGRQ0pbh+qCqHpH/132MpdKgD/otm7mqzXgePL5zr6DChQ8AAH3h3QAA/wD/fwD/AH8A/39/AP9//wAAf///f39//39/f/8AAH9/AAAAfwB/fwB/AH8Af3//f////39////pA/ySAAAQE0lEQVR4nO3d2VIbWRAE0H4a8MCEIcJjx1gt/P9/OQKxtLrvUrcqa2spv0DKE3mLF2CaUubPJn+t4/0RryFbhgbIjUY3LYwOyA0Fmp4EDeTmgglRgwxyM5GErjECcjMZz5AEA+TmMhAOBhPkhtINV4MPcjOpR6AhArmZFCLDEIPcUFaRe8hBbiSfAXBAQG4ir4FogEBuJigNHMhVmwA5kCDXSgLlwIJcIQlYAw5yXSZ4DQ2Q6zFR8VABuQoSHQ4tkL2LaGnogezbRNFDEWS3JJocuiD7FNH10AXZIYkyhzrIrkTUMSxAdoRi4mECsg8RGw8bkD2IGHkYgaQnseKwA8ktYudhB5KYxJDDFCSriKmHKUhKElsOa5B8ItYe1iDJSMw5HEAyiby8XANIGpKXl2sBySHy8uIi4gKSgOTlI1cCEl3k5epAQpO8LHM1IHfetVfz8uIo4udxF1XkZZ2rALm7iyqy8bgikIAkWw5jEV+PcCLHo7eIs0cwkeOxLLJ3kLu7mCLHo7+IP0gckWMdxE7E3yOMyPHYENkzyNojiMjxGEIkgkcEkuNFHEVieLiLHI9RRKKAOIusQUoi+wSpefiCbDz8JhLFw1Vk6+EnEsbDj6TE4fdoBfJwEql4eImEAnERqYL4iITy8ACpe5REdgXS93AQmechEH2RWB7WJPNrYomEA7EUmed4IuE8LEVmjshOQOgediDzHFAknsfdvblHJJGAHvc2IvNME7l2kPt7G5F5jikS0sNAZO0RRiSmh7rI1mNIJDcIg0NbpOQRRCSqh65IGSSESCCQezOQikdDxO6MhPVQFKl6NEh2A8L3UBNpefiLBPZQEml7uIsEASl66Ij0QMgiGUGEHhogXQ9nkdAeCiIEj6qIyaMVwKPOoSBCAqGKJANBeKBBaB6eIu4gbQ+wCNXDUSS6B1aEDuImEt4DCTLgQRVJAwLzAIoMedREtCfiCkLywIkMgviIJPBAgYx6EEVSgEA9QCLjHi4ibh50DpAIB8RBJIUHAoTlQRPZA8iYB0CE6VERUZxIDg+5CBvEWiSJhxSE72EtksRDKCLxIInEBdHykInIQExF7EGYHhIQoYepSBqP+29+HhSRmCCKHKf4eZRFVCaSyIMrAgExEzEFkXH4gvRF4oEoeXz7iqOHlUgqD1+QIglcJLrHkuPbg6+HiUhsj0uOU5xBCiJZQcQeDw8sEKyHgYiNB2YeDBG0h75IWI8ix7AIHqQjEgRE1+PhgQui4KEtYgCC5BgUUQHRFYno0eIYAtHxKIgEAzH1GBHRAtEUCefR4xgAUfPQFAnm0ecYEFEE0RPRBRF4VDnIIJoebRFXEHMOsoguiJZIHA8yBxFE2WMrgnm0oniQjseQiDpIS8QLRMOjz/HwGANERUQLRJPjkQJi4KEiEsBjmIMkYgLSEPEAQXtQNQKB4EW8PUbm8bhIEI+NiPjR0gDR5wgEghbBe+i8Vo/rRPFoiNiCIOcxrBEKBCvi5iHj6ICYemBFwnuUOToixiBVEXcQO45YIDNuIh4eVI6GRhvE3GNNIhDx9JBwNEU8QCoiJiAiDxRHA8TFAyVi7IHjaIg4gWBEUCDAedA06iBeHhgRSw8sR1XED2T2AGF7wDkCggBEzD1wHBURTw+ACAIEwzGsERKkKKIIwvMgvFYcjpAgUhFLDzRHEcTbQyqi76HHURTx5pgvRHRBOB5dDoFGVBCRiAyEOg8ljgKIt8U5AhFVj948pBwFEW+K9/BFFD0MOMKCFETQIFwPTY3AIGwRPkgEjo2IN8MizEdLx8OMIzDIVgQJwvGw4IgMwhNR8GjNA6wRG4QlAvew5YgNshFRAgnEERyEIaLhYcgRHWRcBOlR59DSWIt411/I6BnBedRfK02O8CBrETHImIc5R3yQQZFBkHAcFyDe1VcyJALxqHHoa1yKeDdfzYWICITu4cWRAmRARO5RmYcVxwLEu/VWyI8WHSQox0LEu/RmqBNBeLhqZAGhiog8QnBkASGKCDyCcKQBoYnIPCJw5AFZishAaPPw0cgEMhMmwvOIxPEJ4t02JX0RtkcYjk8R77JJ6T5aXY/4HKlAuiI9kD6Ht8ZjLpCeiNTD2+KcRB4LETpI1SMmRzKQuTmREY+oHNlAmiJjHiE1HtOBtEQaIGk48oHM9TNC9Lh8rbz73yQdSF2E7BGYIyNIVaTiUeXwrr6chCAzDaTgEZ8jJ0hFpAhy6RGeIylIWaTjkYLjDcS7XU7OIg2QtcfXa+XdeC85QUoibY8kHGlBCiIbkIwceUHmzRmpeaTiSAwyrydS9vjg8O6ZnLwgryRLkQuP1Ty8W6bnn8wglyIFjwwc/6ySGuRVpAQSmWMNsC+QpcjGI4hGT2BfIPPXo3Xp4cgxKLAzkPl4AfLu8fZa5QDYHcinyNJDlwMrsDuQD5EzyJkjk8D+QN5F3j0gHJYCOwSZj2eQ13nwOBwB1tkFyPw6kVcPMod37fXsA+RV5K/7bykFVtkJyPznz31OgE12AjL/8S4SlZ2A/P77+en79+/ebQKyC5DD7/9+Pj8/vZJkV9nDDTlx/Pf3z39fRT5M8qrkBzn8PnmcQE4TeRP5JMnJkh7k3eMMUiDJppIc5MTx6vHr5HEGOYusSRKppAZ55zgP5B3kXaRkkkIlMcjh0+PXG8i/K5EiSXiVtCBvHEuPT5AvkRpJZJWkIGeOs8cGZCHSIAnKkhLksJjHm8cbyM/nkkjPJJpKQpDDxTwWID8qIl2SSCrpQD44th6LN2stQiEJopINZOtRBmGS+KvkAtlwnAdSBFmLDJh4qmQC+eRYenwNZA2yFRkh8WLJA3KoenyC/OiKjJLYq2QBWXAsPDogJREGialKjt/6rHC8e3yAbN6ssgjTxEYlxa9FLzkuPC4HUgKpiHBJ9FUSgBx6Hk2QmoiARFUlPMglx9pjDbI5Ig0RGYmWSnCQFsd2IGWQBonYBM8SG6TnsQEpvVlNEQQJVCUyyIqj5kEAaYmASFAqcUHWHGuPEkjxqndFcCQAlaggB5ZH7Yj0RLAmIpWYIH2OYZCeCJiErRLxL1tvOVoelyDVN6svgifhsMQDOVA9CgNpgvRFdEiGVML97fcSR8GjMpATSP3NoojomRBVgoFQORYDGQHxJiGoxPqPLhyPS5DWVSeL6JI0VUL9y6MiR9GjOpDOESGLqJPUVAKBlDm6HsMgRBETky1LGJDDwDw6IJ03iy5iRHKhEgSkwsHwoIDQRexIPlRCgNQ4Wh4ikAERU5JTAoAcRj0uBlIA6R6RMRFbk8lbZJzjciBMkKgkkzNInaPu0R4I8c0aFbEicQZheyBABkVsSCZPkQYH2UMCMipiYDI5gvA4+gMhHxGOiDaJH8gB4yEEYYjokkxOIk2OvkcXhPpmsUQ0SXxA2hxNj/VAxCAsETWTyQPkwJ8HaSAjV50vokPiANLh6HhQBjJ2RGKRmIP0OEgeFJCRibBF8CTGIAcbj3EQtgjaxBaky9Hz0AMRiEBJJkMROcfWAwciEQGS2IH0OageSiAiERTJZAXSPx4ED/JAPEQwJkYgFA6yhx6IVARAsgZREQFxlAYCBhGLSEk2HgogB7AHaSBcEG8SfRAaB9dDAUQuIjHZgmBFiBwkj2EQRxEuScEDCgKcR9lDBQQiwiPRBYHO48ODOBARCEaEQ1ICQYlQOYgegwORHBGYyLiJHshBycMIBCYyRlL0QIDAOcYHInuzgCIjJGUQsQidY9jDDgQoQiapeAhBDnYeqiBQEppJDUQiMsJB9/ABgYoQSKoefBAlDo7HGSSSSJekDsIV0fJggSAmAhZpkzQ8eCBDHAgPAxC0SMsEDDLGwfIYGwgGBC9SJWmBDIsc9ObBHQgIREGkTNL0GAQZ5GB6sEBiipRI2iAjIqMcKA8KCGIiKiJbExTImgPtwR4IDsSEpONBFRnngHl0QVBHRE/kCQ2izxEFREvkiexBANlwBPOAXXVVkXeTPkhXhMEh8OCCwCaiKPJE8uiAcDjGPRoDMQfRFHmSgnBeKwZHayD2IJoiFJC6CIvD3gMOoidC8qiB8Dg4HhiQDCI0kKLIlsPHgwKC/TFLkYToUQCx5BAPROHNUhKhgqxFuBwKHn4gCiJkjwnDIfYQgoQXoYNMjh7tgXiCoEUGPCY/DoSHFghYZARkknIAPKQg8CMCFhnymCoc6h6dgRBBdK46WGQMZBJxIDwEA9F7s4Aigx7T5MGBGogiCIpk2GMLYusRFwQjMg4yMTk0PcZAVI4ISIThMfE4JB6wgSiDAEQ4IJPxPIAD+QCJKyICMZrH0kMKontE5CIsj2mYQ+YBHIg+iFCEBzK5eYBA9I6IUITpcRIZ4RB69AcSC0RAwvaYJjsO7EDUr7pEROAxAKLvEQ+EKSIBIYtIPXKCsEREHlQRCw8GiPIRYYkIPUggYg74QGyuOktECkIQsfHggYQTEXv0RcAe2UDGRAAePRCAh8JADEGGRBAgTREEh8ZAzK76GAnEoyVi58EFCSUC8qiDQDz2AEIUQYHURAJ7mB4RqgjMoyyC4dgLCEEE6FECsfWID9IXQYJsRVQ8gAOxPiJ9EajHRgTloTYQB5C2CNhj0uGgDiQHSFMEDTK5eiQBaZDAPRYiOA/FgThc9ZaIgsfk6CEACSKiATKhOfYJUhRR8XgTyePhBlIQUfI4iSh67AdkI6LmMU2KIA2PbCDPZh5QEPpARCD+IpogQBFtD1eQZysPoIg6iOebtSBR9oCJDHjkBHk28kCJ6A/kC8RRxMADI2IwEN8jchYx8UCI/LoKkGcjD4DIiEdeEDMPMYjJQK4JRCpiMhD3q27oISQZGogcxGkith4SESsPVxBrDonINYA4eLBBxjxuINoiaw8TEGsRFw8miZ3HJ4j5RLw8OCKGA/F6s/w4OCKDHglBXD2GQUwHco0ggyKjHjAQOxFnj0ESJxDDiXhrvEbRIxuIN8V72B7KAzEH8Yb4TNCBLEEsRLwZFlEaiBTE9qp7I1wk5EBsQbwJVgk5EMsj4g2wDd4jE4h3+6XEBlEV8a6+knAeViDexVcz4rEjEO/aW0EOBABickS8O28n1EAsQLwL7yfSQPTfLO+2KQENJAOId9XUQDwSgHj3TE+UgXyBKBwR75LHEmMgmiDeDY+m5WEHovZmedfLSYCBqIF4d8tLgIHogHgXyw9/IHAQ4BHxblUS9kA0QDAi3pVK4+uBB/HuU55dgXiXCQnHIyiId5WgOA4Ee9W9i0TGaSC4iXgXiE9uEO/2NDLgEQ3EuzqtOAwEAuLdm17sB7IA4V5179KU4woyLuJdl0VMPYRvlndXNkkD4l2UYRKAeFdkntAg3uX4xMSDc9W9i/FLSBDvUpwTDMS7jhBR9Rg5It5FxIkdSEvEu4VIsQFpTsS7gnhxBPH+6mGj4dEF8f7SwWMM4v11c0QT5McNgx08yPtEvL9Y4qBFTiDeXyl9oCDeX2ZHEYJ4f/x9hgni/bGvITeG6NlN//8DdW5IXFpOaEoAAAAASUVORK5CYII=" usemap="#imap_piechart_0"><map name='imap_piechart_0'><area shape='poly' coords='199,199,199,0,216,0,233,3,250,6,267,12,283,18,298,26,313,35,322,43' id='pie_0_slice_0'  onmouseover='hover(event, "pie_0_slice_0", "0");'>
<area shape='poly' coords='199,199,322,43,335,54,347,66' id='pie_0_slice_1'  onmouseover='hover(event, "pie_0_slice_1", "0");'>
<area shape='poly' coords='199,199,347,66,358,80,368,94,369,96' id='pie_0_slice_2'  onmouseover='hover(event, "pie_0_slice_2", "0");'>
<area shape='poly' coords='199,199,369,96,374,104' id='pie_0_slice_3'  onmouseover='hover(event, "pie_0_slice_3", "0");'>
<area shape='poly' coords='199,199,374,104,381,119' id='pie_0_slice_5'  onmouseover='hover(event, "pie_0_slice_5", "0");'>
<area shape='poly' coords='199,199,381,119,387,135,392,152,395,169,397,186,397,186' id='pie_0_slice_6'  onmouseover='hover(event, "pie_0_slice_6", "0");'>
<area shape='poly' coords='199,199,397,186,397,204,396,221,394,238,393,239' id='pie_0_slice_7'  onmouseover='hover(event, "pie_0_slice_7", "0");'>
<area shape='poly' coords='199,199,393,239,389,256,383,272,379,282' id='pie_0_slice_9'  onmouseover='hover(event, "pie_0_slice_9", "0");'>
<area shape='poly' coords='199,199,379,282,371,297,362,312,352,326,345,333' id='pie_0_slice_10'  onmouseover='hover(event, "pie_0_slice_10", "0");'>
<area shape='poly' coords='199,199,345,333,333,345,320,356,305,366,290,375,275,382,258,388,241,393,239,393' id='pie_0_slice_11'  onmouseover='hover(event, "pie_0_slice_11", "0");'>
<area shape='poly' coords='199,199,239,393,236,394' id='pie_0_slice_14'  onmouseover='hover(event, "pie_0_slice_14", "0");'>
<area shape='poly' coords='199,199,236,394,225,396' id='pie_0_slice_15'  onmouseover='hover(event, "pie_0_slice_15", "0");'>
<area shape='poly' coords='199,199,225,396,208,397,190,397,173,396,156,393,143,390' id='pie_0_slice_16'  onmouseover='hover(event, "pie_0_slice_16", "0");'>
<area shape='poly' coords='199,199,143,390,127,384,111,377,105,374' id='pie_0_slice_17'  onmouseover='hover(event, "pie_0_slice_17", "0");'>
<area shape='poly' coords='199,199,105,374,92,367' id='pie_0_slice_18'  onmouseover='hover(event, "pie_0_slice_18", "0");'>
<area shape='poly' coords='199,199,92,367,78,357,65,346,52,334,42,322' id='pie_0_slice_20'  onmouseover='hover(event, "pie_0_slice_20", "0");'>
<area shape='poly' coords='199,199,42,322,32,308,28,301' id='pie_0_slice_21'  onmouseover='hover(event, "pie_0_slice_21", "0");'>
<area shape='poly' coords='199,199,28,301,23,291' id='pie_0_slice_22'  onmouseover='hover(event, "pie_0_slice_22", "0");'>
<area shape='poly' coords='199,199,23,291,15,276,9,259,4,243,1,226,0,208,0,191,1,174,4,157,8,140,14,123,21,108,30,93,40,78,51,65,57,59' id='pie_0_slice_23'  onmouseover='hover(event, "pie_0_slice_23", "0");'>
<area shape='poly' coords='199,199,57,59,69,47,72,45' id='pie_0_slice_24'  onmouseover='hover(event, "pie_0_slice_24", "0");'>
<area shape='poly' coords='199,199,72,45,85,35' id='pie_0_slice_25'  onmouseover='hover(event, "pie_0_slice_25", "0");'>
<area shape='poly' coords='199,199,85,35,100,26,115,18,131,11,148,6,165,2,182,0,196,0' id='pie_0_slice_26'  onmouseover='hover(event, "pie_0_slice_26", "0");'></map><input type='hidden' id='hover_redundancies_0' value='pie_0_slice_5~#pie_0_slice_2~#pie_0_slice_24~#pie_0_slice_10~#pie_0_slice_20~#pie_0_slice_16~#pie_0_slice_6~#pie_0_slice_15~#pie_0_slice_1~#pie_0_slice_7~#pie_0_slice_9~#pie_0_slice_17~#pie_0_slice_0~#pie_0_slice_11~#pie_0_slice_22~#pie_0_slice_21~#pie_0_slice_18~#pie_0_slice_23~#pie_0_slice_3~#pie_0_slice_25~#pie_0_slice_26~#pie_0_slice_14'><span style='display: none;' id='tooltip_0_pie_0_slice_0'><table><tr><th>Cofactors, Vitamins, Prosthetic Groups, Pigments</th></tr><tr><td>136 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_1'><table><tr><th>Cell Wall and Capsule</th></tr><tr><td>35 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_2'><table><tr><th>Virulence, Disease and Defense</th></tr><tr><td>38 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_3'><table><tr><th>Potassium metabolism</th></tr><tr><td>9 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_5'><table><tr><th>Miscellaneous</th></tr><tr><td>17 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_6'><table><tr><th>Phages, Prophages, Transposable elements, Plasmids</th></tr><tr><td>71 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_7'><table><tr><th>Membrane Transport</th></tr><tr><td>54 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_9'><table><tr><th>RNA Metabolism</th></tr><tr><td>46 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_10'><table><tr><th>Nucleosides and Nucleotides</th></tr><tr><td>63 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_11'><table><tr><th>Protein Metabolism</th></tr><tr><td>126 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_14'><table><tr><th>Regulation and Cell signaling</th></tr><tr><td>4 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_15'><table><tr><th>Secondary Metabolism</th></tr><tr><td>11 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_16'><table><tr><th>DNA Metabolism</th></tr><tr><td>84 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_17'><table><tr><th>Fatty Acids, Lipids, and Isoprenoids</th></tr><tr><td>42 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_18'><table><tr><th>Nitrogen Metabolism</th></tr><tr><td>15 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_20'><table><tr><th>Respiration</th></tr><tr><td>69 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_21'><table><tr><th>Stress Response</th></tr><tr><td>26 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_22'><table><tr><th>Metabolism of Aromatic Compounds</th></tr><tr><td>11 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_23'><table><tr><th>Amino Acids and Derivatives</th></tr><tr><td>256 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_24'><table><tr><th>Sulfur Metabolism</th></tr><tr><td>21 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_25'><table><tr><th>Phosphorus Metabolism</th></tr><tr><td>17 / 1274</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_26'><table><tr><th>Carbohydrates</th></tr><tr><td>120 / 1274</td></tr></table></span>
</td><td>
<div id='tree_0'>
<div name='tree_level_0' id='tree_node_0_0'><img id='tree_img_0_0' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "0");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/AAAZ4gk3AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cofactors, Vitamins, Prosthetic Groups, Pigments (136)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_0'>
<img id='tree_img_0_1' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "1");'>&nbsp;Biotin (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_1'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=991.25'>Biotin biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=991.25'>Biotin biosynthesis Experimental</a> (4)<br></div>
<img id='tree_img_0_4' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "4");'>&nbsp;Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_4'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=991.25'>Thiamin biosynthesis</a> (8)<br></div>
<span style='cursor: pointer;'>&nbsp;Quinone cofactors (0)<br>
<img id='tree_img_0_7' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "7");'>&nbsp;Tetrapyrroles (20)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_7'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=991.25'>Cobalamin synthesis</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25'>Heme and Siroheme Biosynthesis</a> (15)<br></div>
<img id='tree_img_0_10' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "10");'>&nbsp;Riboflavin, FMN, FAD (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_10'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=991.25'>riboflavin to FAD</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=991.25'>Riboflavin, FMN and FAD metabolism</a> (9)<br></div>
<span style='cursor: pointer;'>&nbsp;Fe-S clusters (0)<br>
<span style='cursor: pointer;'>&nbsp;Mycofactocin (0)<br>
<img id='tree_img_0_15' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "15");'>&nbsp;Pyridoxine (11)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_15'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=991.25'>Pyridoxin (Vitamin B6) Biosynthesis</a> (11)<br></div>
<img id='tree_img_0_17' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "17");'>&nbsp;NAD and NADP (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_17'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25'>NAD and NADP cofactor biosynthesis global</a> (12)<br></div>
<span style='cursor: pointer;'>&nbsp;Coenzyme B (0)<br>
<img id='tree_img_0_20' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "20");'>&nbsp;Folate and pterines (45)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_20'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25'>Folate biosynthesis cluster</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25'>Folate Biosynthesis</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25'>5-FCL-like protein</a> (17)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=991.25'>Pterin synthesis related cluster</a> (7)<br></div>
<span style='cursor: pointer;'>&nbsp;Lipoic acid (0)<br>
<span style='cursor: pointer;'>&nbsp;Coenzyme F420 (0)<br>
<span style='cursor: pointer;'>&nbsp;Coenzyme M (0)<br>
<img id='tree_img_0_28' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "28");'>&nbsp;Coenzyme A (16)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_28'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=991.25'>Coenzyme A Biosynthesis cluster</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25'>Coenzyme A Biosynthesis</a> (11)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_31'><img id='tree_img_0_31' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "31");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAgACc+aWRAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cell Wall and Capsule (35)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_31'>
<img id='tree_img_0_32' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "32");'>&nbsp;Capsular and extracellular polysacchrides (24)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_32'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=991.25'>dTDP-rhamnose synthesis</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Streptococcal_Hyaluronic_Acid_Capsule&organism=991.25'>Streptococcal Hyaluronic Acid Capsule</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=991.25'>CMP-N-acetylneuraminate Biosynthesis</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polysaccharide_deacetylases&organism=991.25'>Polysaccharide deacetylases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=991.25'>Rhamnose containing glycans</a> (11)<br></div>
<img id='tree_img_0_38' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "38");'>&nbsp;Gram-Negative cell wall components (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_38'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lipoprotein_sorting_system&organism=991.25'>Lipoprotein sorting system</a> (1)<br></div>
<img id='tree_img_0_40' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "40");'>&nbsp;Cell Wall and Capsule - no subcategory (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_40'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=991.25'>Murein Hydrolases</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=991.25'>Recycling of Peptidoglycan Amino Acids</a> (3)<br></div>
<img id='tree_img_0_43' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "43");'>&nbsp;Gram-Positive cell wall components (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_43'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Teichoic_and_lipoteichoic_acids_biosynthesis&organism=991.25'>Teichoic and lipoteichoic acids biosynthesis</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Teichuronic_acid_biosynthesis&organism=991.25'>Teichuronic acid biosynthesis</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Cell wall of Mycobacteria (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_47'><img id='tree_img_0_47' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "47");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAAIBKwr76AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Virulence, Disease and Defense (38)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_47'>
<span style='cursor: pointer;'>&nbsp;Adhesion (0)<br>
<span style='cursor: pointer;'>&nbsp;Toxins and superantigens (0)<br>
<span style='cursor: pointer;'>&nbsp;Bacteriocins, ribosomally synthesized antibacterial peptides (0)<br>
<img id='tree_img_0_51' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "51");'>&nbsp;Resistance to antibiotics and toxic compounds (22)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_51'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=991.25'>Copper homeostasis</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Bile_hydrolysis&organism=991.25'>Bile hydrolysis</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=991.25'>Cobalt-zinc-cadmium resistance</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=991.25'>Resistance to fluoroquinolones</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis:_copper_tolerance&organism=991.25'>Copper homeostasis: copper tolerance</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Beta-lactamase&organism=991.25'>Beta-lactamase</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Virulence, Disease and Defense - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Detection (0)<br>
<img id='tree_img_0_60' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "60");'>&nbsp;Invasion and intracellular resistance (16)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_60'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=991.25'>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=991.25'>Mycobacterium virulence operon involved in DNA transcription</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=991.25'>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Listeria_surface_proteins:_Internalin-like_proteins&organism=991.25'>Listeria surface proteins: Internalin-like proteins</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=991.25'>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a> (3)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_66'><img id='tree_img_0_66' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "66");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAgAB9zL4RAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Potassium metabolism (9)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_66'>
<img id='tree_img_0_67' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "67");'>&nbsp;Potassium metabolism - no subcategory (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_67'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25'>Potassium homeostasis</a> (9)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_69'><img id='tree_img_0_69' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "69");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAICr96V6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Photosynthesis (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_69'>
<span style='cursor: pointer;'>&nbsp;Light-harvesting complexes (0)<br>
<span style='cursor: pointer;'>&nbsp;Photosynthesis - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Electron transport and photophosphorylation (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_73'><img id='tree_img_0_73' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "73");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXAwMBmS75kAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Miscellaneous (17)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_73'>
<span style='cursor: pointer;'>&nbsp;Conversion of Succinyl-CoA to Propionyl-CoA (0)<br>
<img id='tree_img_0_75' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "75");'>&nbsp;Plant-Prokaryote DOE project (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_75'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=991.25'>Iron-sulfur cluster assembly</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Single-Rhodanese-domain_proteins&organism=991.25'>Single-Rhodanese-domain proteins</a> (1)<br></div>
<img id='tree_img_0_78' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "78");'>&nbsp;Miscellaneous - no subcategory (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_78'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phosphoglycerate_mutase_protein_family&organism=991.25'>Phosphoglycerate mutase protein family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DedA_family_of_inner_membrane_proteins&organism=991.25'>DedA family of inner membrane proteins</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Muconate_lactonizing_enzyme_family&organism=991.25'>Muconate lactonizing enzyme family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=991.25'>Broadly distributed proteins not in subsystems</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_83'><img id='tree_img_0_83' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "83");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAgICQdD0xAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Phages, Prophages, Transposable elements, Plasmids (71)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_83'>
<span style='cursor: pointer;'>&nbsp;Phage family-specific subsystems (0)<br>
<img id='tree_img_0_85' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "85");'>&nbsp;Transposable elements (69)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_85'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25'>Conjugative transposon, Bacteroidales</a> (69)<br></div>
<img id='tree_img_0_87' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "87");'>&nbsp;Phages, Prophages (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_87'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phage_tail_fiber_proteins&organism=991.25'>Phage tail fiber proteins</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phage_lysis_modules&organism=991.25'>Phage lysis modules</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Phages, Prophages, Transposable elements, Plasmids - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Pathogenicity islands (0)<br>
<span style='cursor: pointer;'>&nbsp;Gene Transfer Agent (GTA) (0)<br>
<span style='cursor: pointer;'>&nbsp;Plasmid related functions (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_94'><img id='tree_img_0_94' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "94");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZmf/YOWB7AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Membrane Transport (54)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_94'>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type II (0)<br>
<span style='cursor: pointer;'>&nbsp;ABC transporters (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VII (Chaperone/Usher pathway, CU) (0)<br>
<img id='tree_img_0_98' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "98");'>&nbsp;Protein translocation across cytoplasmic membrane (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_98'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=991.25'>Twin-arginine translocation system</a> (4)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type V (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type I (0)<br>
<img id='tree_img_0_102' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "102");'>&nbsp;Cation transporters (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_102'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Magnesium_transport&organism=991.25'>Magnesium transport</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_Transport_System&organism=991.25'>Copper Transport System</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_transport_and_blue_copper_proteins&organism=991.25'>Copper transport and blue copper proteins</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type III (0)<br>
<img id='tree_img_0_107' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "107");'>&nbsp;Uni- Sym- and Antiporters (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_107'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters&organism=991.25'>NhaA, NhaD and Sodium-dependent phosphate transporters</a> (1)<br></div>
<img id='tree_img_0_109' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "109");'>&nbsp;Membrane Transport - no subcategory (39)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_109'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=991.25'>Ton and Tol transport systems</a> (39)<br></div>
<span style='cursor: pointer;'>&nbsp;TRAP transporters (0)<br>
<span style='cursor: pointer;'>&nbsp;Sugar Phosphotransferase Systems, PTS (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VI (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP) (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein and nucleoprotein secretion system, Type IV (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_116'><img id='tree_img_0_116' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "116");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZM2YFAh/4AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Iron acquisition and metabolism (2)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_116'>
<img id='tree_img_0_117' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "117");'>&nbsp;Siderophores (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_117'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Siderophore_Aerobactin&organism=991.25'>Siderophore Aerobactin</a> (1)<br></div>
<img id='tree_img_0_119' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "119");'>&nbsp;Iron acquisition and metabolism - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_119'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Encapsulating_protein_for_DyP-type_peroxidase_and_ferritin-like_protein_oligomers&organism=991.25'>Encapsulating protein for DyP-type peroxidase and ferritin-like protein oligomers</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Iron transpot (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_122'><img id='tree_img_0_122' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "122");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//8wYFHreAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;RNA Metabolism (46)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_122'>
<img id='tree_img_0_123' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "123");'>&nbsp;RNA processing and modification (22)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_123'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribonucleases_in_Bacillus&organism=991.25'>Ribonucleases in Bacillus</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=991.25'>mnm5U34 biosynthesis bacteria</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=991.25'>Ribonuclease H</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_3'-terminal_phosphate_cyclase&organism=991.25'>RNA 3'-terminal phosphate cyclase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=991.25'>RNA processing and degradation, bacterial</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=991.25'>tRNA processing</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=991.25'>RNA methylation</a> (5)<br></div>
<img id='tree_img_0_131' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "131");'>&nbsp;Transcription (22)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_131'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=991.25'>Transcription initiation, bacterial sigma factors</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=991.25'>RNA polymerase bacterial</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25'>Transcription factors bacterial</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=991.25'>Rrf2 family transcriptional regulators</a> (5)<br></div>
<img id='tree_img_0_136' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "136");'>&nbsp;RNA Metabolism - no subcategory (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_136'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Group_II_intron-associated_genes&organism=991.25'>Group II intron-associated genes</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_138'><img id='tree_img_0_138' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "138");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEVmAGZP8U4lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Nucleosides and Nucleotides (63)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_138'>
<img id='tree_img_0_139' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "139");'>&nbsp;Pyrimidines (18)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_139'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25'>pyrimidine conversions</a> (18)<br></div>
<img id='tree_img_0_141' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "141");'>&nbsp;Purines (35)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_141'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25'>De Novo Purine Biosynthesis</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=991.25'>GMP synthase</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25'>Purine conversions</a> (21)<br></div>
<img id='tree_img_0_145' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "145");'>&nbsp;Nucleosides and Nucleotides - no subcategory (6)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_145'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=991.25'>Ribonucleotide reduction</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=dNTP_triphosphohydrolase_protein_family&organism=991.25'>dNTP triphosphohydrolase protein family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=991.25'>Adenosyl nucleosidases</a> (2)<br></div>
<img id='tree_img_0_149' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "149");'>&nbsp;Detoxification (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_149'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nucleoside_triphosphate_pyrophosphohydrolase_MazG&organism=991.25'>Nucleoside triphosphate pyrophosphohydrolase MazG</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=991.25'>Nudix proteins (nucleoside triphosphate hydrolases)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=991.25'>Housecleaning nucleoside triphosphate pyrophosphatases</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_153'><img id='tree_img_0_153' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "153");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/gIDP2RJcAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Protein Metabolism (126)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_153'>
<img id='tree_img_0_154' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "154");'>&nbsp;Protein folding (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_154'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=991.25'>Protein chaperones</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_disulfide_interchange&organism=991.25'>Periplasmic disulfide interchange</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Peptidyl-prolyl_cis-trans_isomerase&organism=991.25'>Peptidyl-prolyl cis-trans isomerase</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Selenoproteins (0)<br>
<img id='tree_img_0_159' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "159");'>&nbsp;Protein biosynthesis (73)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_159'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Val&organism=991.25'>tRNA aminoacylation, Val</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Met&organism=991.25'>tRNA aminoacylation, Met</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=rRNAs&organism=991.25'>rRNAs</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ile&organism=991.25'>tRNA aminoacylation, Ile</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Arg&organism=991.25'>tRNA aminoacylation, Arg</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=991.25'>Translation initiation factors bacterial</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25'>Ribosome SSU bacterial</a> (23)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Gly&organism=991.25'>tRNA aminoacylation, Gly</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ala&organism=991.25'>tRNA aminoacylation, Ala</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Trp&organism=991.25'>tRNA aminoacylation, Trp</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=991.25'>Programmed frameshift</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Cys&organism=991.25'>tRNA aminoacylation, Cys</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25'>Translation termination factors bacterial</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_His&organism=991.25'>tRNA aminoacylation, His</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=991.25'>tRNA aminoacylation, Asp and Asn</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=991.25'>Translation elongation factors bacterial</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Lys&organism=991.25'>tRNA aminoacylation, Lys</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Thr&organism=991.25'>tRNA aminoacylation, Thr</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Pro&organism=991.25'>tRNA aminoacylation, Pro</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factor_G_family&organism=991.25'>Translation elongation factor G family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=991.25'>tRNA aminoacylation, Glu and Gln</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ser&organism=991.25'>tRNA aminoacylation, Ser</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Tyr&organism=991.25'>tRNA aminoacylation, Tyr</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Leu&organism=991.25'>tRNA aminoacylation, Leu</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=991.25'>tRNA aminoacylation, Phe</a> (2)<br></div>
<img id='tree_img_0_185' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "185");'>&nbsp;Protein processing and modification (27)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_185'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein-L-isoaspartate_O-methyltransferase&organism=991.25'>Protein-L-isoaspartate O-methyltransferase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=991.25'>Signal peptidase</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=991.25'>Modification of eukaryotic initiation factor 5A</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25'>G3E family of P-loop GTPases (metallocenter biosynthesis)</a> (15)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=991.25'>N-linked Glycosylation in Bacteria</a> (6)<br></div>
<img id='tree_img_0_191' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "191");'>&nbsp;Protein degradation (17)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_191'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=991.25'>Aminopeptidases (EC 3.4.11.-)</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=991.25'>Protein degradation</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=991.25'>Metallocarboxypeptidases (EC 3.4.17.-)</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dipeptidases_(EC_3.4.13.-)&organism=991.25'>Dipeptidases (EC 3.4.13.-)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Serine_endopeptidase_(EC_3.4.21.-)&organism=991.25'>Serine endopeptidase (EC 3.4.21.-)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=991.25'>Proteolysis in bacteria, ATP-dependent</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_198'><img id='tree_img_0_198' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "198");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAZswGD39gAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cell Division and Cell Cycle (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_198'>
<span style='cursor: pointer;'>&nbsp;Checkpoint control (0)<br>
<span style='cursor: pointer;'>&nbsp;Heterocyst formation (0)<br>
<span style='cursor: pointer;'>&nbsp;Cell Division and Cell Cycle - no subcategory (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_202'><img id='tree_img_0_202' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "202");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXMzP91giUxAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Motility and Chemotaxis (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_202'>
<span style='cursor: pointer;'>&nbsp;Magnetotaxis (0)<br>
<span style='cursor: pointer;'>&nbsp;Motility and Chemotaxis - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Flagellar motility in Prokaryota (0)<br>
<span style='cursor: pointer;'>&nbsp;Social motility and nonflagellar swimming in bacteria (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_207'><img id='tree_img_0_207' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "207");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/AP804Oa6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Regulation and Cell signaling (4)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_207'>
<img id='tree_img_0_208' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "208");'>&nbsp;Regulation and Cell signaling - no subcategory (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_208'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Salmonella_enterica_Typhimurium&organism=991.25'>LysR-family proteins in Salmonella enterica Typhimurium</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Escherichia_coli&organism=991.25'>LysR-family proteins in Escherichia coli</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=991.25'>Stringent Response, (p)ppGpp metabolism</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Signal transduction in Eukaryotes (0)<br>
<span style='cursor: pointer;'>&nbsp;Quorum sensing and biofilm formation (0)<br>
<span style='cursor: pointer;'>&nbsp;Proteolytic pathway (0)<br>
<span style='cursor: pointer;'>&nbsp;Regulation of virulence (0)<br>
<span style='cursor: pointer;'>&nbsp;Programmed Cell Death and Toxin-antitoxin Systems (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_217'><img id='tree_img_0_217' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "217");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//wCKxvRFAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Secondary Metabolism (11)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_217'>
<img id='tree_img_0_218' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "218");'>&nbsp;Secondary Metabolism - no subcategory (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_218'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lanthionine_Synthetases&organism=991.25'>Lanthionine Synthetases</a> (5)<br></div>
<span style='cursor: pointer;'>&nbsp;Lipid-derived mediators (0)<br>
<span style='cursor: pointer;'>&nbsp;Plant Octadecanoids (0)<br>
<span style='cursor: pointer;'>&nbsp;Bacterial cytostatics, differentiation factors and antibiotics (0)<br>
<span style='cursor: pointer;'>&nbsp;Biosynthesis of phenylpropanoids (0)<br>
<span style='cursor: pointer;'>&nbsp;Hydrocarbons (0)<br>
<span style='cursor: pointer;'>&nbsp;Aromatic amino acids and derivatives (0)<br>
<span style='cursor: pointer;'>&nbsp;UV-absorbing secondary metabolites (0)<br>
<img id='tree_img_0_227' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "227");'>&nbsp;Plant Alkaloids (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_227'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Alkaloid_biosynthesis_from_L-lysine&organism=991.25'>Alkaloid biosynthesis from L-lysine</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Biologically active compounds in metazoan cell defence and differentiation (0)<br>
<img id='tree_img_0_230' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "230");'>&nbsp;Plant Hormones (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_230'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=991.25'>Auxin biosynthesis</a> (5)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_232'><img id='tree_img_0_232' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "232");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUA//8ZXC8lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;DNA Metabolism (84)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_232'>
<img id='tree_img_0_233' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "233");'>&nbsp;DNA repair (52)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_233'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25'>DNA repair, bacterial</a> (25)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=991.25'>DNA repair, bacterial RecFOR pathway</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=991.25'>DNA repair, bacterial MutL-MutS system</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=991.25'>DNA repair, UvrABC system</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_photolyase&organism=991.25'>DNA repair, bacterial photolyase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RecA_and_RecX&organism=991.25'>RecA and RecX</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=991.25'>DNA repair system including RecA, MutS and a hypothetical protein</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;CRISPs (0)<br>
<img id='tree_img_0_242' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "242");'>&nbsp;DNA Metabolism - no subcategory (19)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_242'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=991.25'>Type I Restriction-Modification</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=991.25'>Restriction-Modification System</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=YcfH&organism=991.25'>YcfH</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=991.25'>DNA structural proteins, bacterial</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_phosphorothioation&organism=991.25'>DNA phosphorothioation</a> (5)<br></div>
<img id='tree_img_0_248' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "248");'>&nbsp;DNA replication (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_248'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=991.25'>DNA topoisomerases, Type I, ATP-independent</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_replication_strays&organism=991.25'>DNA replication strays</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=991.25'>DNA topoisomerases, Type II, ATP-dependent</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;DNA recombination (0)<br>
<img id='tree_img_0_253' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "253");'>&nbsp;DNA uptake, competence (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_253'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=991.25'>DNA processing cluster</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_255'><img id='tree_img_0_255' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "255");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAICr96V6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Fatty Acids, Lipids, and Isoprenoids (42)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_255'>
<span style='cursor: pointer;'>&nbsp;Phospholipids (0)<br>
<span style='cursor: pointer;'>&nbsp;Triacylglycerols (0)<br>
<img id='tree_img_0_258' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "258");'>&nbsp;Fatty acids (21)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_258'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25'>Fatty Acid Biosynthesis FASII</a> (17)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=991.25'>Fatty acid metabolism cluster</a> (4)<br></div>
<span style='cursor: pointer;'>&nbsp;Fatty Acids, Lipids, and Isoprenoids - no subcategory (0)<br>
<img id='tree_img_0_262' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "262");'>&nbsp;Isoprenoids (21)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_262'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Myxoxanthophyll_biosynthesis_in_Cyanobacteria&organism=991.25'>Myxoxanthophyll biosynthesis in Cyanobacteria</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=991.25'>Mevalonate Branch of Isoprenoid Biosynthesis</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=991.25'>Isoprenoinds for Quinones</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=991.25'>Isoprenoid Biosynthesis</a> (9)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_267'><img id='tree_img_0_267' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "267");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAABGTyZaAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Nitrogen Metabolism (15)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_267'>
<img id='tree_img_0_268' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "268");'>&nbsp;Nitrogen Metabolism - no subcategory (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_268'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nitrosative_stress&organism=991.25'>Nitrosative stress</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nitrate_and_nitrite_ammonification&organism=991.25'>Nitrate and nitrite ammonification</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=991.25'>Ammonia assimilation</a> (7)<br></div>
<img id='tree_img_0_272' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "272");'>&nbsp;Denitrification (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_272'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=991.25'>Denitrifying reductase gene clusters</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=991.25'>Denitrification</a> (3)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_275'><img id='tree_img_0_275' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "275");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAgIBxQSaxAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Dormancy and Sporulation (1)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_275'>
<span style='cursor: pointer;'>&nbsp;Spore DNA protection (0)<br>
<img id='tree_img_0_277' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "277");'>&nbsp;Dormancy and Sporulation - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_277'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Sporulation-associated_proteins_with_broader_functions&organism=991.25'>Sporulation-associated proteins with broader functions</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_279'><img id='tree_img_0_279' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "279");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAAP+KeNJXAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Respiration (69)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_279'>
<span style='cursor: pointer;'>&nbsp;Biotin (10)<br>
<img id='tree_img_0_281' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "281");'>&nbsp;ATP synthases (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_281'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25'>F0F1-type ATP synthase</a> (10)<br></div>
<img id='tree_img_0_283' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "283");'>&nbsp;Electron accepting reactions (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_283'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_d_ubiquinol_oxidases&organism=991.25'>Terminal cytochrome d ubiquinol oxidases</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=991.25'>Anaerobic respiratory reductases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_oxidases&organism=991.25'>Terminal cytochrome oxidases</a> (2)<br></div>
<img id='tree_img_0_287' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "287");'>&nbsp;Electron donating reactions (39)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_287'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25'>Respiratory Complex I</a> (15)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=991.25'>Respiratory dehydrogenases 1</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=991.25'>NiFe hydrogenase maturation</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25'>NADH ubiquinone oxidoreductase</a> (14)<br></div>
<span style='cursor: pointer;'>&nbsp;Reverse electron transport (0)<br>
<img id='tree_img_0_293' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "293");'>&nbsp;Respiration - no subcategory (13)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_293'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Quinone_oxidoreductase_family&organism=991.25'>Quinone oxidoreductase family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cytochrome_c_oxidases&organism=991.25'>Biogenesis of cytochrome c oxidases</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=991.25'>Biogenesis of cbb3-type cytochrome c oxidases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=991.25'>Biogenesis of c-type cytochromes</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=991.25'>Soluble cytochromes and functionally related electron carriers</a> (5)<br></div>
<span style='cursor: pointer;'>&nbsp;Sodium Ion-Coupled Energetics (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_300'><img id='tree_img_0_300' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "300");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAzMxS5ysDAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Stress Response (26)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_300'>
<img id='tree_img_0_301' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "301");'>&nbsp;Osmotic stress (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_301'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=991.25'>Osmoregulation</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;Dessication stress (0)<br>
<span style='cursor: pointer;'>&nbsp;Acid stress (0)<br>
<img id='tree_img_0_305' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "305");'>&nbsp;Oxidative stress (15)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_305'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=991.25'>Oxidative stress</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Non-redox_reactions&organism=991.25'>Glutathione: Non-redox reactions</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Redox_cycle&organism=991.25'>Glutathione: Redox cycle</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Cold shock (0)<br>
<span style='cursor: pointer;'>&nbsp;Heat shock (0)<br>
<img id='tree_img_0_311' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "311");'>&nbsp;Detoxification (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_311'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Uptake_of_selenate_and_selenite&organism=991.25'>Uptake of selenate and selenite</a> (1)<br></div>
<img id='tree_img_0_313' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "313");'>&nbsp;Stress Response - no subcategory (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_313'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dimethylarginine_metabolism&organism=991.25'>Dimethylarginine metabolism</a> (2)<br></div>
<img id='tree_img_0_315' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "315");'>&nbsp;Periplasmic Stress (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_315'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=991.25'>Periplasmic Stress Response</a> (5)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_317'><img id='tree_img_0_317' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "317");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXM//+B6UABAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Metabolism of Aromatic Compounds (11)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_317'>
<img id='tree_img_0_318' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "318");'>&nbsp;Peripheral pathways for catabolism of aromatic compounds (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_318'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Quinate_degradation&organism=991.25'>Quinate degradation</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Anaerobic degradation of aromatic compounds (0)<br>
<img id='tree_img_0_321' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "321");'>&nbsp;Metabolism of central aromatic intermediates (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_321'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=991.25'>Catechol branch of beta-ketoadipate pathway</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=991.25'>Salicylate and gentisate catabolism</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=991.25'>Homogentisate pathway of aromatic compound degradation</a> (4)<br></div>
<img id='tree_img_0_325' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "325");'>&nbsp;Metabolism of Aromatic Compounds - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_325'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Gentisate_degradation&organism=991.25'>Gentisate degradation</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_327'><img id='tree_img_0_327' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "327");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXM/8w+OSEXAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Amino Acids and Derivatives (256)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_327'>
<img id='tree_img_0_328' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "328");'>&nbsp;Glutamine, glutamate, aspartate, asparagine; ammonia assimilation (24)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_328'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25'>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a> (20)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=991.25'>Glutamate dehydrogenases</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=991.25'>Glutamine synthetases</a> (2)<br></div>
<img id='tree_img_0_332' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "332");'>&nbsp;Histidine Metabolism (11)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_332'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=991.25'>Histidine Degradation</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=991.25'>Histidine Biosynthesis</a> (7)<br></div>
<img id='tree_img_0_335' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "335");'>&nbsp;Arginine; urea cycle, polyamines (27)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_335'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=991.25'>Urease subunits</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=991.25'>Polyamine Metabolism</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=991.25'>Arginine and Ornithine Degradation</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25'>Urea decomposition</a> (9)<br></div>
<img id='tree_img_0_340' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "340");'>&nbsp;Lysine, threonine, methionine, and cysteine (74)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_340'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25'>Methionine Biosynthesis</a> (26)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=991.25'>Threonine and Homoserine Biosynthesis</a> (14)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=991.25'>Threonine degradation</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=991.25'>Lysine Biosynthesis DAP Pathway, GJO scratch</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=991.25'>Cysteine Biosynthesis</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=991.25'>Lysine degradation</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=991.25'>Lysine Biosynthesis DAP Pathway</a> (10)<br></div>
<span style='cursor: pointer;'>&nbsp;Amino Acids and Derivatives - no subcategory (0)<br>
<img id='tree_img_0_349' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "349");'>&nbsp;Branched-chain amino acids (28)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_349'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=991.25'>Isoleucine degradation</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25'>Branched-Chain Amino Acid Biosynthesis</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=991.25'>Valine degradation</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=991.25'>Leucine Biosynthesis</a> (6)<br></div>
<span style='cursor: pointer;'>&nbsp;Polyamines (0)<br>
<img id='tree_img_0_355' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "355");'>&nbsp;Aromatic amino acids and derivatives (39)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_355'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=991.25'>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25'>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=991.25'>Phenylalanine and Tyrosine Branches from Chorismate</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25'>Tryptophan synthesis</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=991.25'>Aromatic amino acid degradation</a> (5)<br></div>
<img id='tree_img_0_361' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "361");'>&nbsp;Proline and 4-hydroxyproline (11)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_361'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=991.25'>Proline Synthesis</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=991.25'>Proline, 4-hydroxyproline uptake and utilization</a> (7)<br></div>
<img id='tree_img_0_364' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "364");'>&nbsp;Alanine, serine, and glycine (42)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_364'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=991.25'>Glycine Biosynthesis</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=991.25'>Alanine biosynthesis</a> (8)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=991.25'>Serine Biosynthesis</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=991.25'>Glycine cleavage system</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25'>Glycine and Serine Utilization</a> (18)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_370'><img id='tree_img_0_370' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "370");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//5kDFd+lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Sulfur Metabolism (21)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_370'>
<span style='cursor: pointer;'>&nbsp;Inorganic sulfur assimilation (0)<br>
<img id='tree_img_0_372' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "372");'>&nbsp;Sulfur Metabolism - no subcategory (21)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_372'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=991.25'>Thioredoxin-disulfide reductase</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=991.25'>Galactosylceramide and Sulfatide metabolism</a> (11)<br></div>
<span style='cursor: pointer;'>&nbsp;DsrE (0)<br>
<span style='cursor: pointer;'>&nbsp;Organic sulfur assimilation (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_377'><img id='tree_img_0_377' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "377");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZzP8f9clqAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Phosphorus Metabolism (17)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_377'>
<img id='tree_img_0_378' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "378");'>&nbsp;Phosphorus Metabolism - no subcategory (17)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_378'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=991.25'>Phosphate metabolism</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=991.25'>Polyphosphate</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_381'><img id='tree_img_0_381' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "381");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/mcwrs7b/AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Carbohydrates (120)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_381'>
<img id='tree_img_0_382' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "382");'>&nbsp;Central carbohydrate metabolism (52)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_382'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=991.25'>Methylglyoxal Metabolism</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=991.25'>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=991.25'>Pyruvate metabolism I: anaplerotic reactions, PEP</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=991.25'>Pyruvate Alanine Serine Interconversions</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=991.25'>Pentose phosphate pathway</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=991.25'>Glyoxylate bypass</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=991.25'>Dehydrogenase complexes</a> (8)<br></div>
<img id='tree_img_0_390' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "390");'>&nbsp;Aminosugars (13)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_390'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=991.25'>Chitin and N-acetylglucosamine utilization</a> (13)<br></div>
<img id='tree_img_0_392' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "392");'>&nbsp;Di- and oligosaccharides (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_392'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=991.25'>Lactose and Galactose Uptake and Utilization</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lactose_utilization&organism=991.25'>Lactose utilization</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Glycoside hydrolases (0)<br>
<img id='tree_img_0_396' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "396");'>&nbsp;One-carbon Metabolism (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_396'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=991.25'>One-carbon metabolism by tetrahydropterines</a> (5)<br></div>
<span style='cursor: pointer;'>&nbsp;Organic acids (0)<br>
<img id='tree_img_0_399' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "399");'>&nbsp;Fermentation (19)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_399'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=991.25'>Butanol Biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Acetolactate_synthase_subunits&organism=991.25'>Acetolactate synthase subunits</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=991.25'>Fermentations: Lactate</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=991.25'>Acetyl-CoA fermentation to Butyrate</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Acetoin,_butanediol_metabolism&organism=991.25'>Acetoin, butanediol metabolism</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;CO2 fixation (0)<br>
<img id='tree_img_0_406' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "406");'>&nbsp;Sugar alcohols (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_406'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=991.25'>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a> (9)<br></div>
<span style='cursor: pointer;'>&nbsp;Carbohydrates - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Polysaccharides (0)<br>
<img id='tree_img_0_410' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "410");'>&nbsp;Monosaccharides (13)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_410'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=991.25'>Mannose Metabolism</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=D-ribose_utilization&organism=991.25'>D-ribose utilization</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=991.25'>Deoxyribose and Deoxynucleoside Catabolism</a> (5)<br></div></div></div></div>
</td></tr></table></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='1_contents' id='1_content_1'><table class='table_table'><tr><td><input type='button' class='button' value='export to file' onclick='export_table("0", 0, 1, 0);'>&nbsp;&nbsp;<input type='button' class='button' value='clear all filters' onclick='table_reset_filters("0");'></td></tr><tr><td style='width: 100%; text-align: center; vertical-align: middle;'><table style='width: 100%;'>
<tr><td align=center><span class='table_perpage'>display&nbsp;<input type='text' id='table_perpage_0' name='table_perpage_0' size='3' value='15' onkeypress='return check_submit_filter(event, "0");'>&nbsp;items per page</span></td></tr></table></td></tr>
<tr><td style='width: 100%; text-align: center;'><table style='width: 100%;'><tr><td align='left' width='20%'><a href='javascript: table_first("0");' name='table_first_0'>&laquo;first</a>&nbsp;&nbsp;<a href='javascript: table_prev("0");' name='table_prev_0'>&laquo;prev</a></td><td align='center' width='60%'>displaying <span id='table_start_top_0'>1</span> - <span id='table_stop_top_0'>950</span> of <span id='table_total_top_0'>950</span></td><td align='right' width='20%'><a href='javascript: table_next("0");' name='table_next_0'>next&raquo;</a>&nbsp;&nbsp;<a href='javascript: table_last("0");' name='table_last_0'>last&raquo;</a></td></tr></table></td></tr><tr><td><input type='hidden' id='SubsystemTable' value='0' />
<input type='hidden' id='table_data_0' value='Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=991.25@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)&subsystem_name=Biotin_biosynthesis@1>Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1254@1>fig|991.25.peg.1254</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2011@1>fig|991.25.peg.2011</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=991.25@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin synthase (EC 2.8.1.6)&subsystem_name=Biotin_biosynthesis@1>Biotin synthase (EC 2.8.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.61@1>fig|991.25.peg.61</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=991.25@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-ketoacyl-CoA thiolase (EC 2.3.1.16)&subsystem_name=Biotin_biosynthesis@1>3-ketoacyl-CoA thiolase (EC 2.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3221@1>fig|991.25.peg.3221</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=991.25@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)&subsystem_name=Biotin_biosynthesis@1>Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.55@1>fig|991.25.peg.55</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=991.25@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=8-amino-7-oxononanoate synthase (EC 2.3.1.47)&subsystem_name=Biotin_biosynthesis@1>8-amino-7-oxononanoate synthase (EC 2.3.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.49@1>fig|991.25.peg.49</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=991.25@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin synthase (EC 2.8.1.6)&subsystem_name=Biotin_biosynthesis_Experimental@1>Biotin synthase (EC 2.8.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.61@1>fig|991.25.peg.61</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=991.25@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Competence protein F homolog, phosphoribosyltransferase domain&subsystem_name=Biotin_biosynthesis_Experimental@1>Competence protein F homolog, phosphoribosyltransferase domain</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.65@1>fig|991.25.peg.65</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=991.25@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)&subsystem_name=Biotin_biosynthesis_Experimental@1>Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.55@1>fig|991.25.peg.55</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=991.25@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=8-amino-7-oxononanoate synthase (EC 2.3.1.47)&subsystem_name=Biotin_biosynthesis_Experimental@1>8-amino-7-oxononanoate synthase (EC 2.3.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.49@1>fig|991.25.peg.49</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=991.25@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-regulated outer membrane receptor Omr1&subsystem_name=Thiamin_biosynthesis@1>Thiamin-regulated outer membrane receptor Omr1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.846@1>fig|991.25.peg.846</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2948@1>fig|991.25.peg.2948</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=991.25@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin pyrophosphokinase (EC 2.7.6.2)&subsystem_name=Thiamin_biosynthesis@1>Thiamin pyrophosphokinase (EC 2.7.6.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.360@1>fig|991.25.peg.360</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=991.25@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamine-monophosphate kinase (EC 2.7.4.16)&subsystem_name=Thiamin_biosynthesis@1>Thiamine-monophosphate kinase (EC 2.7.4.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.71@1>fig|991.25.peg.71</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=991.25@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)&subsystem_name=Thiamin_biosynthesis@1>Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2934@1>fig|991.25.peg.2934</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2936@1>fig|991.25.peg.2936</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=991.25@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfur carrier protein ThiS&subsystem_name=Thiamin_biosynthesis@1>Sulfur carrier protein ThiS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2932@1>fig|991.25.peg.2932</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=991.25@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Thiamin_biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1262@1>fig|991.25.peg.1262</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=991.25@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)&subsystem_name=Cobalamin_synthesis@1>Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2428@1>fig|991.25.peg.2428</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=991.25@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62)&subsystem_name=Cobalamin_synthesis@1>Adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2431@1>fig|991.25.peg.2431</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=991.25@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobalamin synthase (EC 2.7.8.26)&subsystem_name=Cobalamin_synthesis@1>Cobalamin synthase (EC 2.7.8.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2429@1>fig|991.25.peg.2429</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=991.25@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylcobinamide kinase (EC 2.7.1.156)&subsystem_name=Cobalamin_synthesis@1>Adenosylcobinamide kinase (EC 2.7.1.156)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2431@1>fig|991.25.peg.2431</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=991.25@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)&subsystem_name=Cobalamin_synthesis@1>Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2430@1>fig|991.25.peg.2430</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA synthetase (EC 6.1.1.17)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Glutamyl-tRNA synthetase (EC 6.1.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5030@1>fig|991.25.peg.5030</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.765@1>fig|991.25.peg.765</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen-III synthase (EC 4.2.1.75)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen-III synthase (EC 4.2.1.75)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.261@1>fig|991.25.peg.261</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Precorrin-2 oxidase (EC 1.3.1.76)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Precorrin-2 oxidase (EC 1.3.1.76)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1234@1>fig|991.25.peg.1234</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Porphobilinogen synthase (EC 4.2.1.24)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Porphobilinogen synthase (EC 4.2.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.769@1>fig|991.25.peg.769</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1759@1>fig|991.25.peg.1759</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA reductase (EC 1.2.1.70)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Glutamyl-tRNA reductase (EC 1.2.1.70)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.758@1>fig|991.25.peg.758</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4617@1>fig|991.25.peg.4617</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Porphobilinogen deaminase (EC 2.5.1.61)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Porphobilinogen deaminase (EC 2.5.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.759@1>fig|991.25.peg.759</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen-III methyltransferase (EC 2.1.1.107)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen-III methyltransferase (EC 2.1.1.107)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1233@1>fig|991.25.peg.1233</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2121@1>fig|991.25.peg.2121</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protoporphyrinogen IX oxidase, novel form, HemJ (EC 1.3.-.-)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Protoporphyrinogen IX oxidase, novel form, HemJ (EC 1.3.-.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.755@1>fig|991.25.peg.755</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.756@1>fig|991.25.peg.756</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4626@1>fig|991.25.peg.4626</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=991.25@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen III decarboxylase (EC 4.1.1.37)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen III decarboxylase (EC 4.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.761@1>fig|991.25.peg.761</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=991.25@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FMN adenylyltransferase (EC 2.7.7.2)&subsystem_name=riboflavin_to_FAD@1>FMN adenylyltransferase (EC 2.7.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.63@1>fig|991.25.peg.63</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=991.25@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)&subsystem_name=riboflavin_to_FAD@1>Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.184@1>fig|991.25.peg.184</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=991.25@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin kinase (EC 2.7.1.26)&subsystem_name=riboflavin_to_FAD@1>Riboflavin kinase (EC 2.7.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.63@1>fig|991.25.peg.63</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=991.25@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)&subsystem_name=riboflavin_to_FAD@1>3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.533@1>fig|991.25.peg.533</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2060@1>fig|991.25.peg.2060</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=991.25@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FMN adenylyltransferase (EC 2.7.7.2)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>FMN adenylyltransferase (EC 2.7.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.63@1>fig|991.25.peg.63</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=991.25@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.184@1>fig|991.25.peg.184</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=991.25@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminohydroxyphosphoribosylaminopyrimidine deaminase (EC 3.5.4.26)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Diaminohydroxyphosphoribosylaminopyrimidine deaminase (EC 3.5.4.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4466@1>fig|991.25.peg.4466</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=991.25@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1880@1>fig|991.25.peg.1880</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=991.25@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-amino-6-(5-phosphoribosylamino)uracil reductase (EC 1.1.1.193)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>5-amino-6-(5-phosphoribosylamino)uracil reductase (EC 1.1.1.193)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4466@1>fig|991.25.peg.4466</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=991.25@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin kinase (EC 2.7.1.26)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Riboflavin kinase (EC 2.7.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.63@1>fig|991.25.peg.63</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=991.25@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase II (EC 3.5.4.25)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>GTP cyclohydrolase II (EC 3.5.4.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.533@1>fig|991.25.peg.533</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=991.25@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.533@1>fig|991.25.peg.533</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2060@1>fig|991.25.peg.2060</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=991.25@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.15@1>fig|991.25.peg.15</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1096@1>fig|991.25.peg.1096</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4196@1>fig|991.25.peg.4196</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=991.25@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.185@1>fig|991.25.peg.185</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=991.25@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyridoxamine 5@1-phosphate oxidase (EC 1.4.3.5)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Pyridoxamine 5@1-phosphate oxidase (EC 1.4.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4655@1>fig|991.25.peg.4655</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=991.25@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyridoxine 5@1-phosphate synthase (EC 2.6.99.2)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Pyridoxine 5@1-phosphate synthase (EC 2.6.99.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3704@1>fig|991.25.peg.3704</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=991.25@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4197@1>fig|991.25.peg.4197</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=991.25@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1864@1>fig|991.25.peg.1864</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2172@1>fig|991.25.peg.2172</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4445@1>fig|991.25.peg.4445</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=991.25@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1262@1>fig|991.25.peg.1262</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate synthetase (EC 2.5.1.72)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Quinolinate synthetase (EC 2.5.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3233@1>fig|991.25.peg.3233</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1659@1>fig|991.25.peg.1659</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD kinase (EC 2.7.1.23)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>NAD kinase (EC 2.7.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3706@1>fig|991.25.peg.3706</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinamide phosphoribosyltransferase (EC 2.4.2.12)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Nicotinamide phosphoribosyltransferase (EC 2.4.2.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.95@1>fig|991.25.peg.95</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxyanthranilate 3,4-dioxygenase (EC 1.13.11.6)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>3-hydroxyanthranilate 3,4-dioxygenase (EC 1.13.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.888@1>fig|991.25.peg.888</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD synthetase (EC 6.3.1.5)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>NAD synthetase (EC 6.3.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3596@1>fig|991.25.peg.3596</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan 2,3-dioxygenase (EC 1.13.11.11)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Tryptophan 2,3-dioxygenase (EC 1.13.11.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.109@1>fig|991.25.peg.109</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-aspartate oxidase (EC 1.4.3.16)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>L-aspartate oxidase (EC 1.4.3.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3232@1>fig|991.25.peg.3232</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosyl nicotinamide transporter, PnuC-like&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Ribosyl nicotinamide transporter, PnuC-like</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2951@1>fig|991.25.peg.2951</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kynurenine 3-monooxygenase (EC 1.14.13.9)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Kynurenine 3-monooxygenase (EC 1.14.13.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.319@1>fig|991.25.peg.319</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.96@1>fig|991.25.peg.96</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=991.25@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kynureninase (EC 3.7.1.3)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Kynureninase (EC 3.7.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.314@1>fig|991.25.peg.314</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Folate_biosynthesis_cluster@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4031@1>fig|991.25.peg.4031</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydropteroate synthase (EC 2.5.1.15)&subsystem_name=Folate_biosynthesis_cluster@1>Dihydropteroate synthase (EC 2.5.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1682@1>fig|991.25.peg.1682</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)&subsystem_name=Folate_biosynthesis_cluster@1>tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.697@1>fig|991.25.peg.697</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)&subsystem_name=Folate_biosynthesis_cluster@1>Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1355@1>fig|991.25.peg.1355</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Folate_biosynthesis_cluster@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.36@1>fig|991.25.peg.36</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Folate_biosynthesis_cluster@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2556@1>fig|991.25.peg.2556</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Folate_biosynthesis_cluster@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.37@1>fig|991.25.peg.37</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)&subsystem_name=Folate_biosynthesis_cluster@1>2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3676@1>fig|991.25.peg.3676</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=991.25@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroneopterin aldolase (EC 4.1.2.25)&subsystem_name=Folate_biosynthesis_cluster@1>Dihydroneopterin aldolase (EC 4.1.2.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3797@1>fig|991.25.peg.3797</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Folate_Biosynthesis@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.710@1>fig|991.25.peg.710</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Folylpolyglutamate synthase (EC 6.3.2.17)&subsystem_name=Folate_Biosynthesis@1>Folylpolyglutamate synthase (EC 6.3.2.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1303@1>fig|991.25.peg.1303</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydropteroate synthase (EC 2.5.1.15)&subsystem_name=Folate_Biosynthesis@1>Dihydropteroate synthase (EC 2.5.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1682@1>fig|991.25.peg.1682</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=Folate_Biosynthesis@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.118@1>fig|991.25.peg.118</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate reductase (EC 1.5.1.3)&subsystem_name=Folate_Biosynthesis@1>Dihydrofolate reductase (EC 1.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1193@1>fig|991.25.peg.1193</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate synthase (EC 6.3.2.12)&subsystem_name=Folate_Biosynthesis@1>Dihydrofolate synthase (EC 6.3.2.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1303@1>fig|991.25.peg.1303</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Folate_Biosynthesis@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2556@1>fig|991.25.peg.2556</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Folate_Biosynthesis@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.696@1>fig|991.25.peg.696</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidylate synthase (EC 2.1.1.45)&subsystem_name=Folate_Biosynthesis@1>Thymidylate synthase (EC 2.1.1.45)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1195@1>fig|991.25.peg.1195</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)&subsystem_name=Folate_Biosynthesis@1>2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3676@1>fig|991.25.peg.3676</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroneopterin aldolase (EC 4.1.2.25)&subsystem_name=Folate_Biosynthesis@1>Dihydroneopterin aldolase (EC 4.1.2.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3797@1>fig|991.25.peg.3797</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=991.25@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Folate_Biosynthesis@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.431@1>fig|991.25.peg.431</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alcohol dehydrogenase (EC 1.1.1.1)&subsystem_name=5-FCL-like_protein@1>Alcohol dehydrogenase (EC 1.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2355@1>fig|991.25.peg.2355</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4503@1>fig|991.25.peg.4503</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)&subsystem_name=5-FCL-like_protein@1>Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.367@1>fig|991.25.peg.367</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formyltetrahydrofolate deformylase (EC 3.5.1.10)&subsystem_name=5-FCL-like_protein@1>Formyltetrahydrofolate deformylase (EC 3.5.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4990@1>fig|991.25.peg.4990</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=5-FCL-like_protein@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.118@1>fig|991.25.peg.118</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate reductase (EC 1.5.1.3)&subsystem_name=5-FCL-like_protein@1>Dihydrofolate reductase (EC 1.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1193@1>fig|991.25.peg.1193</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamine-monophosphate kinase (EC 2.7.4.16)&subsystem_name=5-FCL-like_protein@1>Thiamine-monophosphate kinase (EC 2.7.4.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.71@1>fig|991.25.peg.71</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=5-FCL-like_protein@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4558@1>fig|991.25.peg.4558</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)&subsystem_name=5-FCL-like_protein@1>Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1538@1>fig|991.25.peg.1538</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)&subsystem_name=5-FCL-like_protein@1>Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3902@1>fig|991.25.peg.3902</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)&subsystem_name=5-FCL-like_protein@1>Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1183@1>fig|991.25.peg.1183</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)&subsystem_name=5-FCL-like_protein@1>Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2934@1>fig|991.25.peg.2934</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2936@1>fig|991.25.peg.2936</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=5-FCL-like_protein@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1241@1>fig|991.25.peg.1241</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=5-FCL-like_protein@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2993@1>fig|991.25.peg.2993</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=991.25@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=5-FCL-like_protein@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.693@1>fig|991.25.peg.693</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2087@1>fig|991.25.peg.2087</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=991.25@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkaline phosphatase (EC 3.1.3.1)&subsystem_name=Pterin_synthesis_related_cluster@1>Alkaline phosphatase (EC 3.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.349@1>fig|991.25.peg.349</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1689@1>fig|991.25.peg.1689</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3641@1>fig|991.25.peg.3641</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=991.25@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Pterin_synthesis_related_cluster@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2556@1>fig|991.25.peg.2556</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=991.25@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)&subsystem_name=Pterin_synthesis_related_cluster@1>Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4802@1>fig|991.25.peg.4802</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=991.25@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Pterin_synthesis_related_cluster@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.427@1>fig|991.25.peg.427</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=991.25@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6-pyruvoyl tetrahydrobiopterin synthase (EC 4.2.3.12)&subsystem_name=Pterin_synthesis_related_cluster@1>6-pyruvoyl tetrahydrobiopterin synthase (EC 4.2.3.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1505@1>fig|991.25.peg.1505</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=991.25@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4031@1>fig|991.25.peg.4031</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=991.25@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2024@1>fig|991.25.peg.2024</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=991.25@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.37@1>fig|991.25.peg.37</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=991.25@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1156@1>fig|991.25.peg.1156</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=991.25@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.36@1>fig|991.25.peg.36</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4031@1>fig|991.25.peg.4031</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)&subsystem_name=Coenzyme_A_Biosynthesis@1>3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2024@1>fig|991.25.peg.2024</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dephospho-CoA kinase (EC 2.7.1.24)&subsystem_name=Coenzyme_A_Biosynthesis@1>Dephospho-CoA kinase (EC 2.7.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1630@1>fig|991.25.peg.1630</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)&subsystem_name=Coenzyme_A_Biosynthesis@1>Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1156@1>fig|991.25.peg.1156</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Coenzyme_A_Biosynthesis@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.36@1>fig|991.25.peg.36</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-dehydropantoate 2-reductase (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis@1>2-dehydropantoate 2-reductase (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3536@1>fig|991.25.peg.3536</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3771@1>fig|991.25.peg.3771</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantetheine adenylyltransferase (EC 2.7.7.3)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantetheine adenylyltransferase (EC 2.7.7.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4732@1>fig|991.25.peg.4732</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantothenoylcysteine synthetase (EC 6.3.2.5)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantothenoylcysteine synthetase (EC 6.3.2.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1635@1>fig|991.25.peg.1635</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantothenoylcysteine decarboxylase (EC 4.1.1.36)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantothenoylcysteine decarboxylase (EC 4.1.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1635@1>fig|991.25.peg.1635</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=991.25@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Coenzyme_A_Biosynthesis@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.37@1>fig|991.25.peg.37</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=991.25@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1452@1>fig|991.25.peg.1452</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=991.25@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1425@1>fig|991.25.peg.1425</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=991.25@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1451@1>fig|991.25.peg.1451</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=991.25@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)&subsystem_name=dTDP-rhamnose_synthesis@1>Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1453@1>fig|991.25.peg.1453</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3846@1>fig|991.25.peg.3846</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4212@1>fig|991.25.peg.4212</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=991.25@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-Rha:A-D-GlcNAc-diphosphoryl polyprenol, A-3-L-rhamnosyl transferase WbbL&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-Rha:A-D-GlcNAc-diphosphoryl polyprenol, A-3-L-rhamnosyl transferase WbbL</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2640@1>fig|991.25.peg.2640</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Streptococcal_Hyaluronic_Acid_Capsule&organism=991.25@1>Streptococcal Hyaluronic Acid Capsule</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hyaluronan synthase (EC 2.4.1.212)&subsystem_name=Streptococcal_Hyaluronic_Acid_Capsule@1>Hyaluronan synthase (EC 2.4.1.212)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3164@1>fig|991.25.peg.3164</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=991.25@1>CMP-N-acetylneuraminate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylneuraminate synthase (EC 2.5.1.56)&subsystem_name=CMP-N-acetylneuraminate_Biosynthesis@1>N-acetylneuraminate synthase (EC 2.5.1.56)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1468@1>fig|991.25.peg.1468</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=991.25@1>CMP-N-acetylneuraminate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)&subsystem_name=CMP-N-acetylneuraminate_Biosynthesis@1>UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4858@1>fig|991.25.peg.4858</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polysaccharide_deacetylases&organism=991.25@1>Polysaccharide deacetylases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidoglycan N-acetylglucosamine deacetylase (EC 3.5.1.-)&subsystem_name=Polysaccharide_deacetylases@1>Peptidoglycan N-acetylglucosamine deacetylase (EC 3.5.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3866@1>fig|991.25.peg.3866</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polysaccharide_deacetylases&organism=991.25@1>Polysaccharide deacetylases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polysaccharide deacetylase&subsystem_name=Polysaccharide_deacetylases@1>Polysaccharide deacetylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1779@1>fig|991.25.peg.1779</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2910@1>fig|991.25.peg.2910</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=991.25@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=Rhamnose_containing_glycans@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1420@1>fig|991.25.peg.1420</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1445@1>fig|991.25.peg.1445</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3667@1>fig|991.25.peg.3667</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=991.25@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1452@1>fig|991.25.peg.1452</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=991.25@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-L-Rha alpha-1,3-L-rhamnosyltransferase (EC 2.4.1.-)&subsystem_name=Rhamnose_containing_glycans@1>Alpha-L-Rha alpha-1,3-L-rhamnosyltransferase (EC 2.4.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1440@1>fig|991.25.peg.1440</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=991.25@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=putative glycosyltransferase - possibly involved in cell wall localization and side chain formation of rhamnose-glucose polysaccharide&subsystem_name=Rhamnose_containing_glycans@1>putative glycosyltransferase - possibly involved in cell wall localization and side chain formation of rhamnose-glucose polysaccharide</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1786@1>fig|991.25.peg.1786</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=991.25@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1425@1>fig|991.25.peg.1425</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=991.25@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1451@1>fig|991.25.peg.1451</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=991.25@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)&subsystem_name=Rhamnose_containing_glycans@1>Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1453@1>fig|991.25.peg.1453</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3846@1>fig|991.25.peg.3846</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4212@1>fig|991.25.peg.4212</a>@~Cell Wall and Capsule@^Gram-Negative cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lipoprotein_sorting_system&organism=991.25@1>Lipoprotein sorting system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane lipoprotein carrier protein LolA&subsystem_name=Lipoprotein_sorting_system@1>Outer membrane lipoprotein carrier protein LolA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.531@1>fig|991.25.peg.531</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=991.25@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)&subsystem_name=Murein_Hydrolases@1>N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.395@1>fig|991.25.peg.395</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2178@1>fig|991.25.peg.2178</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=991.25@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)&subsystem_name=Murein_Hydrolases@1>D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.512@1>fig|991.25.peg.512</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1286@1>fig|991.25.peg.1286</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=991.25@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Murein_Hydrolases@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.885@1>fig|991.25.peg.885</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=991.25@1>Recycling of Peptidoglycan Amino Acids</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)&subsystem_name=Recycling_of_Peptidoglycan_Amino_Acids@1>N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.395@1>fig|991.25.peg.395</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2178@1>fig|991.25.peg.2178</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=991.25@1>Recycling of Peptidoglycan Amino Acids</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Recycling_of_Peptidoglycan_Amino_Acids@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.885@1>fig|991.25.peg.885</a>@~Cell Wall and Capsule@^Gram-Positive cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Teichoic_and_lipoteichoic_acids_biosynthesis&organism=991.25@1>Teichoic and lipoteichoic acids biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=CDP-glycerol:poly(glycerophosphate) glycerophosphotransferase (EC 2.7.8.12)&subsystem_name=Teichoic_and_lipoteichoic_acids_biosynthesis@1>CDP-glycerol:poly(glycerophosphate) glycerophosphotransferase (EC 2.7.8.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2577@1>fig|991.25.peg.2577</a>@~Cell Wall and Capsule@^Gram-Positive cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Teichuronic_acid_biosynthesis&organism=991.25@1>Teichuronic acid biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative N-acetylgalactosaminyl-diphosphoundecaprenol glucuronosyltransferase&subsystem_name=Teichuronic_acid_biosynthesis@1>Putative N-acetylgalactosaminyl-diphosphoundecaprenol glucuronosyltransferase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3813@1>fig|991.25.peg.3813</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=991.25@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multicopper oxidase&subsystem_name=Copper_homeostasis@1>Multicopper oxidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.476@1>fig|991.25.peg.476</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=991.25@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-translocating P-type ATPase (EC 3.6.3.4)&subsystem_name=Copper_homeostasis@1>Copper-translocating P-type ATPase (EC 3.6.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.475@1>fig|991.25.peg.475</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1666@1>fig|991.25.peg.1666</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1767@1>fig|991.25.peg.1767</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3391@1>fig|991.25.peg.3391</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3400@1>fig|991.25.peg.3400</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=991.25@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multidrug resistance transporter, Bcr/CflA family&subsystem_name=Copper_homeostasis@1>Multidrug resistance transporter, Bcr/CflA family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3350@1>fig|991.25.peg.3350</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Bile_hydrolysis&organism=991.25@1>Bile hydrolysis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Choloylglycine hydrolase (EC 3.5.1.24)&subsystem_name=Bile_hydrolysis@1>Choloylglycine hydrolase (EC 3.5.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3388@1>fig|991.25.peg.3388</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=991.25@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-binding heavy metal response regulator&subsystem_name=Cobalt-zinc-cadmium_resistance@1>DNA-binding heavy metal response regulator</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3327@1>fig|991.25.peg.3327</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=991.25@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobalt-zinc-cadmium resistance protein&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Cobalt-zinc-cadmium resistance protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4767@1>fig|991.25.peg.4767</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=991.25@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Probable Co/Zn/Cd efflux system membrane fusion protein&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Probable Co/Zn/Cd efflux system membrane fusion protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2654@1>fig|991.25.peg.2654</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2764@1>fig|991.25.peg.2764</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2773@1>fig|991.25.peg.2773</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3306@1>fig|991.25.peg.3306</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4911@1>fig|991.25.peg.4911</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=991.25@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional regulator, MerR family&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Transcriptional regulator, MerR family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2458@1>fig|991.25.peg.2458</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4352@1>fig|991.25.peg.4352</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=991.25@1>Resistance to fluoroquinolones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit B (EC 5.99.1.3)&subsystem_name=Resistance_to_fluoroquinolones@1>DNA gyrase subunit B (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2563@1>fig|991.25.peg.2563</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=991.25@1>Resistance to fluoroquinolones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit A (EC 5.99.1.3)&subsystem_name=Resistance_to_fluoroquinolones@1>DNA gyrase subunit A (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1715@1>fig|991.25.peg.1715</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis:_copper_tolerance&organism=991.25@1>Copper homeostasis: copper tolerance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytoplasmic copper homeostasis protein CutC&subsystem_name=Copper_homeostasis:_copper_tolerance@1>Cytoplasmic copper homeostasis protein CutC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1528@1>fig|991.25.peg.1528</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Beta-lactamase&organism=991.25@1>Beta-lactamase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-lactamase (EC 3.5.2.6)&subsystem_name=Beta-lactamase@1>Beta-lactamase (EC 3.5.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.373@1>fig|991.25.peg.373</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Beta-lactamase&organism=991.25@1>Beta-lactamase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Metal-dependent hydrolases of the beta-lactamase superfamily I&subsystem_name=Beta-lactamase@1>Metal-dependent hydrolases of the beta-lactamase superfamily I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4173@1>fig|991.25.peg.4173</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=991.25@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S7p (S5e)&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>SSU ribosomal protein S7p (S5e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1379@1>fig|991.25.peg.1379</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=991.25@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1380@1>fig|991.25.peg.1380</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=991.25@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Tu&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>Translation elongation factor Tu</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4418@1>fig|991.25.peg.4418</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=991.25@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S12p (S23e)&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>SSU ribosomal protein S12p (S23e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1378@1>fig|991.25.peg.1378</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=991.25@1>Mycobacterium virulence operon involved in DNA transcription</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)&subsystem_name=Mycobacterium_virulence_operon_involved_in_DNA_transcription@1>DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4427@1>fig|991.25.peg.4427</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=991.25@1>Mycobacterium virulence operon involved in DNA transcription</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)&subsystem_name=Mycobacterium_virulence_operon_involved_in_DNA_transcription@1>DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4426@1>fig|991.25.peg.4426</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=991.25@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate synthetase (EC 2.5.1.72)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>Quinolinate synthetase (EC 2.5.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3233@1>fig|991.25.peg.3233</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=991.25@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1659@1>fig|991.25.peg.1659</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=991.25@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-aspartate oxidase (EC 1.4.3.16)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>L-aspartate oxidase (EC 1.4.3.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3232@1>fig|991.25.peg.3232</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Listeria_surface_proteins:_Internalin-like_proteins&organism=991.25@1>Listeria surface proteins: Internalin-like proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=internalin, putative&subsystem_name=Listeria_surface_proteins:_Internalin-like_proteins@1>internalin, putative</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3374@1>fig|991.25.peg.3374</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4301@1>fig|991.25.peg.4301</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4805@1>fig|991.25.peg.4805</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Listeria_surface_proteins:_Internalin-like_proteins&organism=991.25@1>Listeria surface proteins: Internalin-like proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Internalin-like protein (LPXTG motif) Lmo0331 homolog&subsystem_name=Listeria_surface_proteins:_Internalin-like_proteins@1>Internalin-like protein (LPXTG motif) Lmo0331 homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1185@1>fig|991.25.peg.1185</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=991.25@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L35p&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>LSU ribosomal protein L35p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4832@1>fig|991.25.peg.4832</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=991.25@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 3&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>Translation initiation factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4833@1>fig|991.25.peg.4833</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=991.25@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L20p&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>LSU ribosomal protein L20p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4831@1>fig|991.25.peg.4831</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium-transporting ATPase A chain (EC 3.6.3.12) (TC 3.A.3.7.1)&subsystem_name=Potassium_homeostasis@1>Potassium-transporting ATPase A chain (EC 3.6.3.12) (TC 3.A.3.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2028@1>fig|991.25.peg.2028</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium-transporting ATPase C chain (EC 3.6.3.12) (TC 3.A.3.7.1)&subsystem_name=Potassium_homeostasis@1>Potassium-transporting ATPase C chain (EC 3.6.3.12) (TC 3.A.3.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2030@1>fig|991.25.peg.2030</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium voltage-gated channel subfamily KQT&subsystem_name=Potassium_homeostasis@1>Potassium voltage-gated channel subfamily KQT</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1625@1>fig|991.25.peg.1625</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=POTASSIUM/PROTON ANTIPORTER ROSB&subsystem_name=Potassium_homeostasis@1>POTASSIUM/PROTON ANTIPORTER ROSB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3113@1>fig|991.25.peg.3113</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium efflux system KefA protein&subsystem_name=Potassium_homeostasis@1>Potassium efflux system KefA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2289@1>fig|991.25.peg.2289</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium-transporting ATPase B chain (EC 3.6.3.12) (TC 3.A.3.7.1)&subsystem_name=Potassium_homeostasis@1>Potassium-transporting ATPase B chain (EC 3.6.3.12) (TC 3.A.3.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2029@1>fig|991.25.peg.2029</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Large-conductance mechanosensitive channel&subsystem_name=Potassium_homeostasis@1>Large-conductance mechanosensitive channel</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1012@1>fig|991.25.peg.1012</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)&subsystem_name=Potassium_homeostasis@1>FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3098@1>fig|991.25.peg.3098</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=991.25@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kup system potassium uptake protein&subsystem_name=Potassium_homeostasis@1>Kup system potassium uptake protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3067@1>fig|991.25.peg.3067</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=991.25@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=PaaD-like protein (DUF59) involved in Fe-S cluster assembly&subsystem_name=Iron-sulfur_cluster_assembly@1>PaaD-like protein (DUF59) involved in Fe-S cluster assembly</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.852@1>fig|991.25.peg.852</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=991.25@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=probable iron binding protein from the HesB_IscA_SufA family&subsystem_name=Iron-sulfur_cluster_assembly@1>probable iron binding protein from the HesB_IscA_SufA family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.862@1>fig|991.25.peg.862</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=991.25@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Iron-sulfur_cluster_assembly@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1041@1>fig|991.25.peg.1041</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=991.25@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=Iron-sulfur_cluster_assembly@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.619@1>fig|991.25.peg.619</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1795@1>fig|991.25.peg.1795</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2495@1>fig|991.25.peg.2495</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3276@1>fig|991.25.peg.3276</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4068@1>fig|991.25.peg.4068</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=991.25@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly protein SufB&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly protein SufB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.861@1>fig|991.25.peg.861</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=991.25@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly protein SufD&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly protein SufD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.857@1>fig|991.25.peg.857</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=991.25@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly ATPase protein SufC&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly ATPase protein SufC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.858@1>fig|991.25.peg.858</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Single-Rhodanese-domain_proteins&organism=991.25@1>Single-Rhodanese-domain proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rhodanese domain protein, Enterobacterial subgroup, YceA homolog&subsystem_name=Single-Rhodanese-domain_proteins@1>Rhodanese domain protein, Enterobacterial subgroup, YceA homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.789@1>fig|991.25.peg.789</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphoglycerate_mutase_protein_family&organism=991.25@1>Phosphoglycerate mutase protein family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)&subsystem_name=Phosphoglycerate_mutase_protein_family@1>Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2428@1>fig|991.25.peg.2428</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DedA_family_of_inner_membrane_proteins&organism=991.25@1>DedA family of inner membrane proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DedA protein&subsystem_name=DedA_family_of_inner_membrane_proteins@1>DedA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3503@1>fig|991.25.peg.3503</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Muconate_lactonizing_enzyme_family&organism=991.25@1>Muconate lactonizing enzyme family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylbenzoate synthase (EC 4.2.1.113)&subsystem_name=Muconate_lactonizing_enzyme_family@1>O-succinylbenzoate synthase (EC 4.2.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4060@1>fig|991.25.peg.4060</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=991.25@1>Broadly distributed proteins not in subsystems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=YpfJ protein, zinc metalloprotease superfamily&subsystem_name=Broadly_distributed_proteins_not_in_subsystems@1>YpfJ protein, zinc metalloprotease superfamily</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2019@1>fig|991.25.peg.2019</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=991.25@1>Broadly distributed proteins not in subsystems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UPF0225 protein YchJ&subsystem_name=Broadly_distributed_proteins_not_in_subsystems@1>UPF0225 protein YchJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4288@1>fig|991.25.peg.4288</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraO&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraO</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1916@1>fig|991.25.peg.1916</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraB&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1928@1>fig|991.25.peg.1928</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraN&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.541@1>fig|991.25.peg.541</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1917@1>fig|991.25.peg.1917</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2734@1>fig|991.25.peg.2734</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3979@1>fig|991.25.peg.3979</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4276@1>fig|991.25.peg.4276</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraG&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.549@1>fig|991.25.peg.549</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1923@1>fig|991.25.peg.1923</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2726@1>fig|991.25.peg.2726</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3971@1>fig|991.25.peg.3971</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4269@1>fig|991.25.peg.4269</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraA&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1929@1>fig|991.25.peg.1929</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraM&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraM</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.542@1>fig|991.25.peg.542</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1918@1>fig|991.25.peg.1918</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2733@1>fig|991.25.peg.2733</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3978@1>fig|991.25.peg.3978</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4275@1>fig|991.25.peg.4275</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraJ&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.545@1>fig|991.25.peg.545</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1921@1>fig|991.25.peg.1921</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2730@1>fig|991.25.peg.2730</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3975@1>fig|991.25.peg.3975</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4273@1>fig|991.25.peg.4273</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraK&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraK</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.544@1>fig|991.25.peg.544</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1920@1>fig|991.25.peg.1920</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2731@1>fig|991.25.peg.2731</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3976@1>fig|991.25.peg.3976</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4274@1>fig|991.25.peg.4274</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraI&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraI</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.546@1>fig|991.25.peg.546</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.547@1>fig|991.25.peg.547</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.548@1>fig|991.25.peg.548</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1922@1>fig|991.25.peg.1922</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2727@1>fig|991.25.peg.2727</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2728@1>fig|991.25.peg.2728</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2729@1>fig|991.25.peg.2729</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3972@1>fig|991.25.peg.3972</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3973@1>fig|991.25.peg.3973</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3974@1>fig|991.25.peg.3974</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4270@1>fig|991.25.peg.4270</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4271@1>fig|991.25.peg.4271</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4272@1>fig|991.25.peg.4272</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraE&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.553@1>fig|991.25.peg.553</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1925@1>fig|991.25.peg.1925</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2724@1>fig|991.25.peg.2724</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3969@1>fig|991.25.peg.3969</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4267@1>fig|991.25.peg.4267</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative conjugative transposon mobilization protein BF0132&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Putative conjugative transposon mobilization protein BF0132</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.590@1>fig|991.25.peg.590</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1931@1>fig|991.25.peg.1931</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2511@1>fig|991.25.peg.2511</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2512@1>fig|991.25.peg.2512</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2692@1>fig|991.25.peg.2692</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3951@1>fig|991.25.peg.3951</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4690@1>fig|991.25.peg.4690</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=hypothetical protein clusted with conjugative transposons, BF0131&subsystem_name=Conjugative_transposon,_Bacteroidales@1>hypothetical protein clusted with conjugative transposons, BF0131</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.589@1>fig|991.25.peg.589</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1930@1>fig|991.25.peg.1930</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2693@1>fig|991.25.peg.2693</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3950@1>fig|991.25.peg.3950</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraQ&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraQ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1915@1>fig|991.25.peg.1915</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative mobilization protein BF0133&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Putative mobilization protein BF0133</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.591@1>fig|991.25.peg.591</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1932@1>fig|991.25.peg.1932</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2488@1>fig|991.25.peg.2488</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2691@1>fig|991.25.peg.2691</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4689@1>fig|991.25.peg.4689</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraD&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1927@1>fig|991.25.peg.1927</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=991.25@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraF&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.550@1>fig|991.25.peg.550</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1924@1>fig|991.25.peg.1924</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2725@1>fig|991.25.peg.2725</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3970@1>fig|991.25.peg.3970</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4268@1>fig|991.25.peg.4268</a>@~Phages, Prophages, Transposable elements, Plasmids@^Phages, Prophages@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phage_tail_fiber_proteins&organism=991.25@1>Phage tail fiber proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phage tail fiber protein&subsystem_name=Phage_tail_fiber_proteins@1>Phage tail fiber protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1584@1>fig|991.25.peg.1584</a>@~Phages, Prophages, Transposable elements, Plasmids@^Phages, Prophages@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phage_lysis_modules&organism=991.25@1>Phage lysis modules</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phage endolysin&subsystem_name=Phage_lysis_modules@1>Phage endolysin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.666@1>fig|991.25.peg.666</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=991.25@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Twin-arginine translocation protein TatC&subsystem_name=Twin-arginine_translocation_system@1>Twin-arginine translocation protein TatC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1491@1>fig|991.25.peg.1491</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=991.25@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Twin-arginine translocation protein TatA&subsystem_name=Twin-arginine_translocation_system@1>Twin-arginine translocation protein TatA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3611@1>fig|991.25.peg.3611</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3807@1>fig|991.25.peg.3807</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=991.25@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribonuclease TatD&subsystem_name=Twin-arginine_translocation_system@1>Deoxyribonuclease TatD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4581@1>fig|991.25.peg.4581</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Magnesium_transport&organism=991.25@1>Magnesium transport</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Magnesium and cobalt transport protein CorA&subsystem_name=Magnesium_transport@1>Magnesium and cobalt transport protein CorA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.889@1>fig|991.25.peg.889</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.890@1>fig|991.25.peg.890</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4555@1>fig|991.25.peg.4555</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Magnesium_transport&organism=991.25@1>Magnesium transport</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mg(2+) transport ATPase, P-type (EC 3.6.3.2)&subsystem_name=Magnesium_transport@1>Mg(2+) transport ATPase, P-type (EC 3.6.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2353@1>fig|991.25.peg.2353</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_Transport_System&organism=991.25@1>Copper Transport System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-translocating P-type ATPase (EC 3.6.3.4)&subsystem_name=Copper_Transport_System@1>Copper-translocating P-type ATPase (EC 3.6.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.475@1>fig|991.25.peg.475</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1666@1>fig|991.25.peg.1666</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1767@1>fig|991.25.peg.1767</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3391@1>fig|991.25.peg.3391</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3400@1>fig|991.25.peg.3400</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_transport_and_blue_copper_proteins&organism=991.25@1>Copper transport and blue copper proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Azurin&subsystem_name=Copper_transport_and_blue_copper_proteins@1>Azurin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4149@1>fig|991.25.peg.4149</a>@~Membrane Transport@^Uni- Sym- and Antiporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters&organism=991.25@1>NhaA, NhaD and Sodium-dependent phosphate transporters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Na+/H+ antiporter NhaA type&subsystem_name=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters@1>Na+/H+ antiporter NhaA type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.14@1>fig|991.25.peg.14</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=991.25@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric siderophore transport system, biopolymer transport protein ExbB&subsystem_name=Ton_and_Tol_transport_systems@1>Ferric siderophore transport system, biopolymer transport protein ExbB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1304@1>fig|991.25.peg.1304</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=991.25@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=TonB-dependent receptor&subsystem_name=Ton_and_Tol_transport_systems@1>TonB-dependent receptor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.43@1>fig|991.25.peg.43</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.242@1>fig|991.25.peg.242</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.364@1>fig|991.25.peg.364</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.366@1>fig|991.25.peg.366</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.509@1>fig|991.25.peg.509</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.799@1>fig|991.25.peg.799</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1103@1>fig|991.25.peg.1103</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1109@1>fig|991.25.peg.1109</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1211@1>fig|991.25.peg.1211</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1245@1>fig|991.25.peg.1245</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1532@1>fig|991.25.peg.1532</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2372@1>fig|991.25.peg.2372</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2758@1>fig|991.25.peg.2758</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3154@1>fig|991.25.peg.3154</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3197@1>fig|991.25.peg.3197</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3423@1>fig|991.25.peg.3423</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3440@1>fig|991.25.peg.3440</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3456@1>fig|991.25.peg.3456</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3519@1>fig|991.25.peg.3519</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4008@1>fig|991.25.peg.4008</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4201@1>fig|991.25.peg.4201</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4347@1>fig|991.25.peg.4347</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=991.25@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane lipoprotein omp16 precursor&subsystem_name=Ton_and_Tol_transport_systems@1>Outer membrane lipoprotein omp16 precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1567@1>fig|991.25.peg.1567</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1581@1>fig|991.25.peg.1581</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1644@1>fig|991.25.peg.1644</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2550@1>fig|991.25.peg.2550</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4303@1>fig|991.25.peg.4303</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4961@1>fig|991.25.peg.4961</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=991.25@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biopolymer transport protein ExbD/TolR&subsystem_name=Ton_and_Tol_transport_systems@1>Biopolymer transport protein ExbD/TolR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.322@1>fig|991.25.peg.322</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.323@1>fig|991.25.peg.323</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1305@1>fig|991.25.peg.1305</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1692@1>fig|991.25.peg.1692</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1693@1>fig|991.25.peg.1693</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=991.25@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxybenzoyl-CoA thioesterase family active site&subsystem_name=Ton_and_Tol_transport_systems@1>4-hydroxybenzoyl-CoA thioesterase family active site</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3216@1>fig|991.25.peg.3216</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4463@1>fig|991.25.peg.4463</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=991.25@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Ton_and_Tol_transport_systems@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1618@1>fig|991.25.peg.1618</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=991.25@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aerobactin siderophore receptor IutA&subsystem_name=Ton_and_Tol_transport_systems@1>Aerobactin siderophore receptor IutA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3053@1>fig|991.25.peg.3053</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=991.25@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=MotA/TolQ/ExbB proton channel family protein&subsystem_name=Ton_and_Tol_transport_systems@1>MotA/TolQ/ExbB proton channel family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.321@1>fig|991.25.peg.321</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_Aerobactin&organism=991.25@1>Siderophore Aerobactin</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aerobactin siderophore receptor IutA&subsystem_name=Siderophore_Aerobactin@1>Aerobactin siderophore receptor IutA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3053@1>fig|991.25.peg.3053</a>@~Iron acquisition and metabolism@^Iron acquisition and metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Encapsulating_protein_for_DyP-type_peroxidase_and_ferritin-like_protein_oligomers&organism=991.25@1>Encapsulating protein for DyP-type peroxidase and ferritin-like protein oligomers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted dye-decolorizing peroxidase (DyP), encapsulated subgroup&subsystem_name=Encapsulating_protein_for_DyP-type_peroxidase_and_ferritin-like_protein_oligomers@1>Predicted dye-decolorizing peroxidase (DyP), encapsulated subgroup</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2662@1>fig|991.25.peg.2662</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleases_in_Bacillus&organism=991.25@1>Ribonucleases in Bacillus</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease HII (EC 3.1.26.4)&subsystem_name=Ribonucleases_in_Bacillus@1>Ribonuclease HII (EC 3.1.26.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4402@1>fig|991.25.peg.4402</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleases_in_Bacillus&organism=991.25@1>Ribonucleases in Bacillus</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Metallo-beta-lactamase family protein, RNA-specific&subsystem_name=Ribonucleases_in_Bacillus@1>Metallo-beta-lactamase family protein, RNA-specific</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3507@1>fig|991.25.peg.3507</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=991.25@1>mnm5U34 biosynthesis bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)&subsystem_name=mnm5U34_biosynthesis_bacteria@1>tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3079@1>fig|991.25.peg.3079</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=991.25@1>mnm5U34 biosynthesis bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=mnm5U34_biosynthesis_bacteria@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.619@1>fig|991.25.peg.619</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1795@1>fig|991.25.peg.1795</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2495@1>fig|991.25.peg.2495</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3276@1>fig|991.25.peg.3276</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4068@1>fig|991.25.peg.4068</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=991.25@1>Ribonuclease H</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease HI (EC 3.1.26.4)&subsystem_name=Ribonuclease_H@1>Ribonuclease HI (EC 3.1.26.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3903@1>fig|991.25.peg.3903</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=991.25@1>Ribonuclease H</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease HII (EC 3.1.26.4)&subsystem_name=Ribonuclease_H@1>Ribonuclease HII (EC 3.1.26.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4402@1>fig|991.25.peg.4402</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_3@1-terminal_phosphate_cyclase&organism=991.25@1>RNA 3@1-terminal phosphate cyclase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA-2@1,3@1-PO4:RNA-5@1-OH ligase&subsystem_name=RNA_3@1-terminal_phosphate_cyclase@1>RNA-2@1,3@1-PO4:RNA-5@1-OH ligase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2620@1>fig|991.25.peg.2620</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=991.25@1>RNA processing and degradation, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3@1-to-5@1 exoribonuclease RNase R&subsystem_name=RNA_processing_and_degradation,_bacterial@1>3@1-to-5@1 exoribonuclease RNase R</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3800@1>fig|991.25.peg.3800</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=991.25@1>RNA processing and degradation, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease III (EC 3.1.26.3)&subsystem_name=RNA_processing_and_degradation,_bacterial@1>Ribonuclease III (EC 3.1.26.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3899@1>fig|991.25.peg.3899</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=991.25@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease Z (EC 3.1.26.11)&subsystem_name=tRNA_processing@1>Ribonuclease Z (EC 3.1.26.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4654@1>fig|991.25.peg.4654</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=991.25@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease P protein component (EC 3.1.26.5)&subsystem_name=tRNA_processing@1>Ribonuclease P protein component (EC 3.1.26.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.747@1>fig|991.25.peg.747</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=991.25@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA dimethylallyltransferase (EC 2.5.1.75)&subsystem_name=tRNA_processing@1>tRNA dimethylallyltransferase (EC 2.5.1.75)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1626@1>fig|991.25.peg.1626</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=991.25@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)&subsystem_name=tRNA_processing@1>tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.697@1>fig|991.25.peg.697</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=991.25@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (guanosine(18)-2@1-O)-methyltransferase (EC 2.1.1.34)&subsystem_name=RNA_methylation@1>tRNA (guanosine(18)-2@1-O)-methyltransferase (EC 2.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3107@1>fig|991.25.peg.3107</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=991.25@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)&subsystem_name=RNA_methylation@1>tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3079@1>fig|991.25.peg.3079</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=991.25@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (cytidine(34)-2@1-O)-methyltransferase (EC 2.1.1.207)&subsystem_name=RNA_methylation@1>tRNA (cytidine(34)-2@1-O)-methyltransferase (EC 2.1.1.207)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2438@1>fig|991.25.peg.2438</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=991.25@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)&subsystem_name=RNA_methylation@1>SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.11@1>fig|991.25.peg.11</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=991.25@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)&subsystem_name=RNA_methylation@1>16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3776@1>fig|991.25.peg.3776</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=991.25@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma-54 factor RpoN&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma-54 factor RpoN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.227@1>fig|991.25.peg.227</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=991.25@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma factor RpoD&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma factor RpoD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3523@1>fig|991.25.peg.3523</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=991.25@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma-70 factor&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma-70 factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4045@1>fig|991.25.peg.4045</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=991.25@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1407@1>fig|991.25.peg.1407</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=991.25@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4427@1>fig|991.25.peg.4427</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=991.25@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1636@1>fig|991.25.peg.1636</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=991.25@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4426@1>fig|991.25.peg.4426</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination protein NusB&subsystem_name=Transcription_factors_bacterial@1>Transcription termination protein NusB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.392@1>fig|991.25.peg.392</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription accessory protein (S1 RNA-binding domain)&subsystem_name=Transcription_factors_bacterial@1>Transcription accessory protein (S1 RNA-binding domain)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3806@1>fig|991.25.peg.3806</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription elongation factor GreA&subsystem_name=Transcription_factors_bacterial@1>Transcription elongation factor GreA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2946@1>fig|991.25.peg.2946</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination factor Rho&subsystem_name=Transcription_factors_bacterial@1>Transcription termination factor Rho</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2096@1>fig|991.25.peg.2096</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription antitermination protein NusG&subsystem_name=Transcription_factors_bacterial@1>Transcription antitermination protein NusG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4420@1>fig|991.25.peg.4420</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription-repair coupling factor&subsystem_name=Transcription_factors_bacterial@1>Transcription-repair coupling factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.875@1>fig|991.25.peg.875</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Regulator of nucleoside diphosphate kinase&subsystem_name=Transcription_factors_bacterial@1>Regulator of nucleoside diphosphate kinase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.384@1>fig|991.25.peg.384</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3404@1>fig|991.25.peg.3404</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator RfaH&subsystem_name=Transcription_factors_bacterial@1>Transcriptional activator RfaH</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1422@1>fig|991.25.peg.1422</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=991.25@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination protein NusA&subsystem_name=Transcription_factors_bacterial@1>Transcription termination protein NusA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2904@1>fig|991.25.peg.2904</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=991.25@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted transcriptional regulator of sulfate adenylyltransferase, Rrf2 family&subsystem_name=Rrf2_family_transcriptional_regulators@1>Predicted transcriptional regulator of sulfate adenylyltransferase, Rrf2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4348@1>fig|991.25.peg.4348</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=991.25@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted transcriptional regulator of sulfate transport, Rrf2 family&subsystem_name=Rrf2_family_transcriptional_regulators@1>Predicted transcriptional regulator of sulfate transport, Rrf2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1226@1>fig|991.25.peg.1226</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=991.25@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rrf2 family transcriptional regulator&subsystem_name=Rrf2_family_transcriptional_regulators@1>Rrf2 family transcriptional regulator</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3458@1>fig|991.25.peg.3458</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4148@1>fig|991.25.peg.4148</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=991.25@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rrf2 family transcriptional regulator, group III&subsystem_name=Rrf2_family_transcriptional_regulators@1>Rrf2 family transcriptional regulator, group III</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3117@1>fig|991.25.peg.3117</a>@~RNA Metabolism@^RNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Group_II_intron-associated_genes&organism=991.25@1>Group II intron-associated genes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Retron-type RNA-directed DNA polymerase (EC 2.7.7.49)&subsystem_name=Group_II_intron-associated_genes@1>Retron-type RNA-directed DNA polymerase (EC 2.7.7.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2499@1>fig|991.25.peg.2499</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5105@1>fig|991.25.peg.5105</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside diphosphate kinase (EC 2.7.4.6)&subsystem_name=pyrimidine_conversions@1>Nucleoside diphosphate kinase (EC 2.7.4.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2625@1>fig|991.25.peg.2625</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thioredoxin reductase (EC 1.8.1.9)&subsystem_name=pyrimidine_conversions@1>Thioredoxin reductase (EC 1.8.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.800@1>fig|991.25.peg.800</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1235@1>fig|991.25.peg.1235</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1971@1>fig|991.25.peg.1971</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4064@1>fig|991.25.peg.4064</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4065@1>fig|991.25.peg.4065</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytidylate kinase (EC 2.7.4.25)&subsystem_name=pyrimidine_conversions@1>Cytidylate kinase (EC 2.7.4.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1699@1>fig|991.25.peg.1699</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidine kinase (EC 2.7.1.21)&subsystem_name=pyrimidine_conversions@1>Thymidine kinase (EC 2.7.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.225@1>fig|991.25.peg.225</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=pyrimidine_conversions@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1862@1>fig|991.25.peg.1862</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2855@1>fig|991.25.peg.2855</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-nucleotidase (EC 3.1.3.5)&subsystem_name=pyrimidine_conversions@1>5@1-nucleotidase (EC 3.1.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1640@1>fig|991.25.peg.1640</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1641@1>fig|991.25.peg.1641</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytidine deaminase (EC 3.5.4.5)&subsystem_name=pyrimidine_conversions@1>Cytidine deaminase (EC 3.5.4.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1578@1>fig|991.25.peg.1578</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2991@1>fig|991.25.peg.2991</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidylate synthase (EC 2.1.1.45)&subsystem_name=pyrimidine_conversions@1>Thymidylate synthase (EC 2.1.1.45)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1195@1>fig|991.25.peg.1195</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uridine kinase (EC 2.7.1.48)&subsystem_name=pyrimidine_conversions@1>Uridine kinase (EC 2.7.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4456@1>fig|991.25.peg.4456</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uracil phosphoribosyltransferase (EC 2.4.2.9)&subsystem_name=pyrimidine_conversions@1>Uracil phosphoribosyltransferase (EC 2.4.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3819@1>fig|991.25.peg.3819</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=991.25@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=CTP synthase (EC 6.3.4.2)&subsystem_name=pyrimidine_conversions@1>CTP synthase (EC 6.3.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4112@1>fig|991.25.peg.4112</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylamine--glycine ligase (EC 6.3.4.13)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylamine--glycine ligase (EC 6.3.4.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3816@1>fig|991.25.peg.3816</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=IMP cyclohydrolase (EC 3.5.4.10)&subsystem_name=De_Novo_Purine_Biosynthesis@1>IMP cyclohydrolase (EC 3.5.4.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1183@1>fig|991.25.peg.1183</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Amidophosphoribosyltransferase (EC 2.4.2.14)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Amidophosphoribosyltransferase (EC 2.4.2.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3908@1>fig|991.25.peg.3908</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.87@1>fig|991.25.peg.87</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3913@1>fig|991.25.peg.3913</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazole-succinocarboxamide synthase (EC 6.3.2.6)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylaminoimidazole-succinocarboxamide synthase (EC 6.3.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4944@1>fig|991.25.peg.4944</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3902@1>fig|991.25.peg.3902</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate lyase (EC 4.3.2.2)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Adenylosuccinate lyase (EC 4.3.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3112@1>fig|991.25.peg.3112</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformylglycinamidine cyclo-ligase (EC 6.3.3.1)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylformylglycinamidine cyclo-ligase (EC 6.3.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4246@1>fig|991.25.peg.4246</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1183@1>fig|991.25.peg.1183</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2013@1>fig|991.25.peg.2013</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=991.25@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformylglycinamidine synthase, glutamine amidotransferase subunit (EC 6.3.5.3)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylformylglycinamidine synthase, glutamine amidotransferase subunit (EC 6.3.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2013@1>fig|991.25.peg.2013</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=991.25@1>GMP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)&subsystem_name=GMP_synthase@1>GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4226@1>fig|991.25.peg.4226</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=991.25@1>GMP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)&subsystem_name=GMP_synthase@1>GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4226@1>fig|991.25.peg.4226</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate synthetase (EC 6.3.4.4)&subsystem_name=Purine_conversions@1>Adenylosuccinate synthetase (EC 6.3.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2585@1>fig|991.25.peg.2585</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dNTP triphosphohydrolase, broad substrate specificity, subgroup 3&subsystem_name=Purine_conversions@1>dNTP triphosphohydrolase, broad substrate specificity, subgroup 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1260@1>fig|991.25.peg.1260</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)&subsystem_name=Purine_conversions@1>GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4226@1>fig|991.25.peg.4226</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inosine-5@1-monophosphate dehydrogenase (EC 1.1.1.205)&subsystem_name=Purine_conversions@1>Inosine-5@1-monophosphate dehydrogenase (EC 1.1.1.205)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2599@1>fig|991.25.peg.2599</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)&subsystem_name=Purine_conversions@1>Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1355@1>fig|991.25.peg.1355</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate lyase (EC 4.3.2.2)&subsystem_name=Purine_conversions@1>Adenylosuccinate lyase (EC 4.3.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3112@1>fig|991.25.peg.3112</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Purine_conversions@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3755@1>fig|991.25.peg.3755</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenine phosphoribosyltransferase (EC 2.4.2.7)&subsystem_name=Purine_conversions@1>Adenine phosphoribosyltransferase (EC 2.4.2.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4499@1>fig|991.25.peg.4499</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyadenosine kinase (EC 2.7.1.76)&subsystem_name=Purine_conversions@1>Deoxyadenosine kinase (EC 2.7.1.76)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1204@1>fig|991.25.peg.1204</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3676@1>fig|991.25.peg.3676</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylate kinase (EC 2.7.4.3)&subsystem_name=Purine_conversions@1>Adenylate kinase (EC 2.7.4.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1356@1>fig|991.25.peg.1356</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside diphosphate kinase (EC 2.7.4.6)&subsystem_name=Purine_conversions@1>Nucleoside diphosphate kinase (EC 2.7.4.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2625@1>fig|991.25.peg.2625</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyguanosine kinase (EC 2.7.1.113)&subsystem_name=Purine_conversions@1>Deoxyguanosine kinase (EC 2.7.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1204@1>fig|991.25.peg.1204</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3676@1>fig|991.25.peg.3676</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Purine_conversions@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1862@1>fig|991.25.peg.1862</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2855@1>fig|991.25.peg.2855</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-nucleotidase (EC 3.1.3.5)&subsystem_name=Purine_conversions@1>5@1-nucleotidase (EC 3.1.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1640@1>fig|991.25.peg.1640</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1641@1>fig|991.25.peg.1641</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)&subsystem_name=Purine_conversions@1>GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4226@1>fig|991.25.peg.4226</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosine deaminase (EC 3.5.4.4)&subsystem_name=Purine_conversions@1>Adenosine deaminase (EC 3.5.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1671@1>fig|991.25.peg.1671</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=991.25@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Guanylate kinase (EC 2.7.4.8)&subsystem_name=Purine_conversions@1>Guanylate kinase (EC 2.7.4.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.98@1>fig|991.25.peg.98</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=991.25@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class II (coenzyme B12-dependent) (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class II (coenzyme B12-dependent) (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4328@1>fig|991.25.peg.4328</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=991.25@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.294@1>fig|991.25.peg.294</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=991.25@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.295@1>fig|991.25.peg.295</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dNTP_triphosphohydrolase_protein_family&organism=991.25@1>dNTP triphosphohydrolase protein family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dNTP triphosphohydrolase, broad substrate specificity, subgroup 3&subsystem_name=dNTP_triphosphohydrolase_protein_family@1>dNTP triphosphohydrolase, broad substrate specificity, subgroup 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1260@1>fig|991.25.peg.1260</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=991.25@1>Adenosyl nucleosidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Adenosyl_nucleosidases@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1862@1>fig|991.25.peg.1862</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2855@1>fig|991.25.peg.2855</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nucleoside_triphosphate_pyrophosphohydrolase_MazG&organism=991.25@1>Nucleoside triphosphate pyrophosphohydrolase MazG</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)&subsystem_name=Nucleoside_triphosphate_pyrophosphohydrolase_MazG@1>Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1769@1>fig|991.25.peg.1769</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=991.25@1>Nudix proteins (nucleoside triphosphate hydrolases)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)&subsystem_name=Nudix_proteins_(nucleoside_triphosphate_hydrolases)@1>Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3844@1>fig|991.25.peg.3844</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=991.25@1>Housecleaning nucleoside triphosphate pyrophosphatases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)&subsystem_name=Housecleaning_nucleoside_triphosphate_pyrophosphatases@1>Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3844@1>fig|991.25.peg.3844</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=991.25@1>Housecleaning nucleoside triphosphate pyrophosphatases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cof protein, HD superfamily hydrolase&subsystem_name=Housecleaning_nucleoside_triphosphate_pyrophosphatases@1>Cof protein, HD superfamily hydrolase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4946@1>fig|991.25.peg.4946</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=991.25@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Heat shock protein GrpE&subsystem_name=Protein_chaperones@1>Heat shock protein GrpE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4472@1>fig|991.25.peg.4472</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=991.25@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein DnaK&subsystem_name=Protein_chaperones@1>Chaperone protein DnaK</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4707@1>fig|991.25.peg.4707</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=991.25@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein DnaJ&subsystem_name=Protein_chaperones@1>Chaperone protein DnaJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4473@1>fig|991.25.peg.4473</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=991.25@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DnaJ-class molecular chaperone CbpA&subsystem_name=Protein_chaperones@1>DnaJ-class molecular chaperone CbpA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4231@1>fig|991.25.peg.4231</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=991.25@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein HtpG&subsystem_name=Protein_chaperones@1>Chaperone protein HtpG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1522@1>fig|991.25.peg.1522</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_disulfide_interchange&organism=991.25@1>Periplasmic disulfide interchange</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)&subsystem_name=Periplasmic_disulfide_interchange@1>Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.698@1>fig|991.25.peg.698</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2767@1>fig|991.25.peg.2767</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Peptidyl-prolyl_cis-trans_isomerase&organism=991.25@1>Peptidyl-prolyl cis-trans isomerase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)&subsystem_name=Peptidyl-prolyl_cis-trans_isomerase@1>FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3098@1>fig|991.25.peg.3098</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Peptidyl-prolyl_cis-trans_isomerase&organism=991.25@1>Peptidyl-prolyl cis-trans isomerase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FKBP-type peptidyl-prolyl cis-trans isomerase FklB (EC 5.2.1.8)&subsystem_name=Peptidyl-prolyl_cis-trans_isomerase@1>FKBP-type peptidyl-prolyl cis-trans isomerase FklB (EC 5.2.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2589@1>fig|991.25.peg.2589</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Val&organism=991.25@1>tRNA aminoacylation, Val</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Valyl-tRNA synthetase (EC 6.1.1.9)&subsystem_name=tRNA_aminoacylation,_Val@1>Valyl-tRNA synthetase (EC 6.1.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2548@1>fig|991.25.peg.2548</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Met&organism=991.25@1>tRNA aminoacylation, Met</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionyl-tRNA synthetase (EC 6.1.1.10)&subsystem_name=tRNA_aminoacylation,_Met@1>Methionyl-tRNA synthetase (EC 6.1.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3923@1>fig|991.25.peg.3923</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=rRNAs&organism=991.25@1>rRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5S rRNA&subsystem_name=rRNAs@1>5S rRNA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.rna.35@1>fig|991.25.rna.35</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=rRNAs&organism=991.25@1>rRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU rRNA&subsystem_name=rRNAs@1>SSU rRNA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.rna.39@1>fig|991.25.rna.39</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=rRNAs&organism=991.25@1>rRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU rRNA&subsystem_name=rRNAs@1>LSU rRNA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.rna.36@1>fig|991.25.rna.36</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ile&organism=991.25@1>tRNA aminoacylation, Ile</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isoleucyl-tRNA synthetase (EC 6.1.1.5)&subsystem_name=tRNA_aminoacylation,_Ile@1>Isoleucyl-tRNA synthetase (EC 6.1.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.138@1>fig|991.25.peg.138</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Arg&organism=991.25@1>tRNA aminoacylation, Arg</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Arginyl-tRNA synthetase (EC 6.1.1.19)&subsystem_name=tRNA_aminoacylation,_Arg@1>Arginyl-tRNA synthetase (EC 6.1.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1530@1>fig|991.25.peg.1530</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=991.25@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor SUI1-related protein&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor SUI1-related protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2856@1>fig|991.25.peg.2856</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=991.25@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 1&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1403@1>fig|991.25.peg.1403</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=991.25@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionyl-tRNA formyltransferase (EC 2.1.2.9)&subsystem_name=Translation_initiation_factors_bacterial@1>Methionyl-tRNA formyltransferase (EC 2.1.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.707@1>fig|991.25.peg.707</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=991.25@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 3&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4833@1>fig|991.25.peg.4833</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=991.25@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 2&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2903@1>fig|991.25.peg.2903</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=991.25@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosome-binding factor A&subsystem_name=Translation_initiation_factors_bacterial@1>Ribosome-binding factor A</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3851@1>fig|991.25.peg.3851</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S11p (S14e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S11p (S14e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1405@1>fig|991.25.peg.1405</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S1p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S1p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1696@1>fig|991.25.peg.1696</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S6p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S6p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1653@1>fig|991.25.peg.1653</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S15p (S13e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S15p (S13e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3513@1>fig|991.25.peg.3513</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S14p (S29e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S14p (S29e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1395@1>fig|991.25.peg.1395</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S5p (S2e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S5p (S2e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1399@1>fig|991.25.peg.1399</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S18p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S18p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1652@1>fig|991.25.peg.1652</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S17p (S11e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S17p (S11e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1391@1>fig|991.25.peg.1391</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S20p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S20p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.403@1>fig|991.25.peg.403</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S12p (S23e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S12p (S23e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1378@1>fig|991.25.peg.1378</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S9p (S16e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S9p (S16e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2924@1>fig|991.25.peg.2924</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S8p (S15Ae)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S8p (S15Ae)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1396@1>fig|991.25.peg.1396</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S4p (S9e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S4p (S9e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1406@1>fig|991.25.peg.1406</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S14p (S29e), zinc-independent&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S14p (S29e), zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1395@1>fig|991.25.peg.1395</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S19p (S15e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S19p (S15e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1386@1>fig|991.25.peg.1386</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S10p (S20e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S10p (S20e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1381@1>fig|991.25.peg.1381</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S16p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S16p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1069@1>fig|991.25.peg.1069</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S3p (S3e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S3p (S3e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1388@1>fig|991.25.peg.1388</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S18p, zinc-independent&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S18p, zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1652@1>fig|991.25.peg.1652</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S7p (S5e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S7p (S5e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1379@1>fig|991.25.peg.1379</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S2p (SAe)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S2p (SAe)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2925@1>fig|991.25.peg.2925</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S21p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S21p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4415@1>fig|991.25.peg.4415</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=991.25@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S13p (S18e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S13p (S18e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1404@1>fig|991.25.peg.1404</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Gly&organism=991.25@1>tRNA aminoacylation, Gly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycyl-tRNA synthetase (EC 6.1.1.14)&subsystem_name=tRNA_aminoacylation,_Gly@1>Glycyl-tRNA synthetase (EC 6.1.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.66@1>fig|991.25.peg.66</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ala&organism=991.25@1>tRNA aminoacylation, Ala</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanyl-tRNA synthetase (EC 6.1.1.7)&subsystem_name=tRNA_aminoacylation,_Ala@1>Alanyl-tRNA synthetase (EC 6.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4354@1>fig|991.25.peg.4354</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4640@1>fig|991.25.peg.4640</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Trp&organism=991.25@1>tRNA aminoacylation, Trp</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophanyl-tRNA synthetase (EC 6.1.1.2)&subsystem_name=tRNA_aminoacylation,_Trp@1>Tryptophanyl-tRNA synthetase (EC 6.1.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5043@1>fig|991.25.peg.5043</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=991.25@1>Programmed frameshift</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=programmed frameshift-containing&subsystem_name=Programmed_frameshift@1>programmed frameshift-containing</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1751@1>fig|991.25.peg.1751</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=991.25@1>Programmed frameshift</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 2&subsystem_name=Programmed_frameshift@1>Peptide chain release factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1751@1>fig|991.25.peg.1751</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Cys&organism=991.25@1>tRNA aminoacylation, Cys</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteinyl-tRNA synthetase (EC 6.1.1.16)&subsystem_name=tRNA_aminoacylation,_Cys@1>Cysteinyl-tRNA synthetase (EC 6.1.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2558@1>fig|991.25.peg.2558</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosome recycling factor&subsystem_name=Translation_termination_factors_bacterial@1>Ribosome recycling factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.252@1>fig|991.25.peg.252</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor homolog&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2618@1>fig|991.25.peg.2618</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidyl-tRNA hydrolase (EC 3.1.1.29)&subsystem_name=Translation_termination_factors_bacterial@1>Peptidyl-tRNA hydrolase (EC 3.1.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3910@1>fig|991.25.peg.3910</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide deformylase (EC 3.5.1.88)&subsystem_name=Translation_termination_factors_bacterial@1>Peptide deformylase (EC 3.5.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1771@1>fig|991.25.peg.1771</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 3&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4430@1>fig|991.25.peg.4430</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tmRNA-binding protein SmpB&subsystem_name=Translation_termination_factors_bacterial@1>tmRNA-binding protein SmpB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1615@1>fig|991.25.peg.1615</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 1&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5005@1>fig|991.25.peg.5005</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine aminopeptidase (EC 3.4.11.18)&subsystem_name=Translation_termination_factors_bacterial@1>Methionine aminopeptidase (EC 3.4.11.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3627@1>fig|991.25.peg.3627</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=991.25@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 2&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1751@1>fig|991.25.peg.1751</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_His&organism=991.25@1>tRNA aminoacylation, His</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidyl-tRNA synthetase (EC 6.1.1.21)&subsystem_name=tRNA_aminoacylation,_His@1>Histidyl-tRNA synthetase (EC 6.1.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4471@1>fig|991.25.peg.4471</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=991.25@1>tRNA aminoacylation, Asp and Asn</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartyl-tRNA synthetase (EC 6.1.1.12)&subsystem_name=tRNA_aminoacylation,_Asp_and_Asn@1>Aspartyl-tRNA synthetase (EC 6.1.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.918@1>fig|991.25.peg.918</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=991.25@1>tRNA aminoacylation, Asp and Asn</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Asparaginyl-tRNA synthetase (EC 6.1.1.22)&subsystem_name=tRNA_aminoacylation,_Asp_and_Asn@1>Asparaginyl-tRNA synthetase (EC 6.1.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.228@1>fig|991.25.peg.228</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=991.25@1>tRNA aminoacylation, Asp and Asn</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartyl-tRNA(Asn) amidotransferase subunit A (EC 6.3.5.6)&subsystem_name=tRNA_aminoacylation,_Asp_and_Asn@1>Aspartyl-tRNA(Asn) amidotransferase subunit A (EC 6.3.5.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3564@1>fig|991.25.peg.3564</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=991.25@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1380@1>fig|991.25.peg.1380</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=991.25@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor LepA&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor LepA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.76@1>fig|991.25.peg.76</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=991.25@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Tu&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor Tu</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4418@1>fig|991.25.peg.4418</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=991.25@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Ts&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor Ts</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2926@1>fig|991.25.peg.2926</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=991.25@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor P&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor P</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3313@1>fig|991.25.peg.3313</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Lys&organism=991.25@1>tRNA aminoacylation, Lys</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)&subsystem_name=tRNA_aminoacylation,_Lys@1>Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4389@1>fig|991.25.peg.4389</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Thr&organism=991.25@1>tRNA aminoacylation, Thr</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonyl-tRNA synthetase (EC 6.1.1.3)&subsystem_name=tRNA_aminoacylation,_Thr@1>Threonyl-tRNA synthetase (EC 6.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4834@1>fig|991.25.peg.4834</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Pro&organism=991.25@1>tRNA aminoacylation, Pro</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prolyl-tRNA synthetase (EC 6.1.1.15), archaeal/eukaryal type&subsystem_name=tRNA_aminoacylation,_Pro@1>Prolyl-tRNA synthetase (EC 6.1.1.15), archaeal/eukaryal type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.401@1>fig|991.25.peg.401</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factor_G_family&organism=991.25@1>Translation elongation factor G family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Translation_elongation_factor_G_family@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1380@1>fig|991.25.peg.1380</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=991.25@1>tRNA aminoacylation, Glu and Gln</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA(Gln) amidotransferase subunit A (EC 6.3.5.7)&subsystem_name=tRNA_aminoacylation,_Glu_and_Gln@1>Glutamyl-tRNA(Gln) amidotransferase subunit A (EC 6.3.5.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3564@1>fig|991.25.peg.3564</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=991.25@1>tRNA aminoacylation, Glu and Gln</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA synthetase (EC 6.1.1.17)&subsystem_name=tRNA_aminoacylation,_Glu_and_Gln@1>Glutamyl-tRNA synthetase (EC 6.1.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5030@1>fig|991.25.peg.5030</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=991.25@1>tRNA aminoacylation, Glu and Gln</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutaminyl-tRNA synthetase (EC 6.1.1.18)&subsystem_name=tRNA_aminoacylation,_Glu_and_Gln@1>Glutaminyl-tRNA synthetase (EC 6.1.1.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3796@1>fig|991.25.peg.3796</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ser&organism=991.25@1>tRNA aminoacylation, Ser</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Seryl-tRNA synthetase (EC 6.1.1.11)&subsystem_name=tRNA_aminoacylation,_Ser@1>Seryl-tRNA synthetase (EC 6.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.8@1>fig|991.25.peg.8</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Tyr&organism=991.25@1>tRNA aminoacylation, Tyr</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tyrosyl-tRNA synthetase (EC 6.1.1.1)&subsystem_name=tRNA_aminoacylation,_Tyr@1>Tyrosyl-tRNA synthetase (EC 6.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.268@1>fig|991.25.peg.268</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Leu&organism=991.25@1>tRNA aminoacylation, Leu</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Leucyl-tRNA synthetase (EC 6.1.1.4)&subsystem_name=tRNA_aminoacylation,_Leu@1>Leucyl-tRNA synthetase (EC 6.1.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.239@1>fig|991.25.peg.239</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=991.25@1>tRNA aminoacylation, Phe</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)&subsystem_name=tRNA_aminoacylation,_Phe@1>Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.80@1>fig|991.25.peg.80</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=991.25@1>tRNA aminoacylation, Phe</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)&subsystem_name=tRNA_aminoacylation,_Phe@1>Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4501@1>fig|991.25.peg.4501</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein-L-isoaspartate_O-methyltransferase&organism=991.25@1>Protein-L-isoaspartate O-methyltransferase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Protein-L-isoaspartate_O-methyltransferase@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1618@1>fig|991.25.peg.1618</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=991.25@1>Signal peptidase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Signal peptidase I (EC 3.4.21.89)&subsystem_name=Signal_peptidase@1>Signal peptidase I (EC 3.4.21.89)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1886@1>fig|991.25.peg.1886</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1887@1>fig|991.25.peg.1887</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=991.25@1>Signal peptidase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lipoprotein signal peptidase (EC 3.4.23.36)&subsystem_name=Signal_peptidase@1>Lipoprotein signal peptidase (EC 3.4.23.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.143@1>fig|991.25.peg.143</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=991.25@1>Modification of eukaryotic initiation factor 5A</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyhypusine synthase (EC 2.5.1.46)&subsystem_name=Modification_of_eukaryotic_initiation_factor_5A@1>Deoxyhypusine synthase (EC 2.5.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4087@1>fig|991.25.peg.4087</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=991.25@1>Modification of eukaryotic initiation factor 5A</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor P&subsystem_name=Modification_of_eukaryotic_initiation_factor_5A@1>Translation elongation factor P</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3313@1>fig|991.25.peg.3313</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase nickel incorporation-associated protein HypB&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase nickel incorporation-associated protein HypB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2596@1>fig|991.25.peg.2596</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreF&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease accessory protein UreF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3187@1>fig|991.25.peg.3187</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypD&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase metallocenter assembly protein HypD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2588@1>fig|991.25.peg.2588</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypE&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase metallocenter assembly protein HypE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2598@1>fig|991.25.peg.2598</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease gamma subunit (EC 3.5.1.5)&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease gamma subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3183@1>fig|991.25.peg.3183</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreE&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease accessory protein UreE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3186@1>fig|991.25.peg.3186</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease alpha subunit (EC 3.5.1.5)&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease alpha subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3185@1>fig|991.25.peg.3185</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypF&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase metallocenter assembly protein HypF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2590@1>fig|991.25.peg.2590</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreD&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease accessory protein UreD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3189@1>fig|991.25.peg.3189</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase nickel incorporation protein HypA&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase nickel incorporation protein HypA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2595@1>fig|991.25.peg.2595</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypC&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase metallocenter assembly protein HypC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2597@1>fig|991.25.peg.2597</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease beta subunit (EC 3.5.1.5)&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease beta subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3184@1>fig|991.25.peg.3184</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=putative periplasmic protein kinase ArgK and related GTPases of G3E family&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>putative periplasmic protein kinase ArgK and related GTPases of G3E family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3119@1>fig|991.25.peg.3119</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylmalonyl-CoA mutase (EC 5.4.99.2)&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Methylmalonyl-CoA mutase (EC 5.4.99.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4980@1>fig|991.25.peg.4980</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=991.25@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreG&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease accessory protein UreG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3188@1>fig|991.25.peg.3188</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=991.25@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1420@1>fig|991.25.peg.1420</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1445@1>fig|991.25.peg.1445</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3667@1>fig|991.25.peg.3667</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=991.25@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-1,4-N-acetylgalactosamine transferase PglH (EC 2.4.1.-)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>Alpha-1,4-N-acetylgalactosamine transferase PglH (EC 2.4.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1482@1>fig|991.25.peg.1482</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=991.25@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lipid carrier : UDP-N-acetylgalactosaminyltransferase (EC 2.4.1.-)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>Lipid carrier : UDP-N-acetylgalactosaminyltransferase (EC 2.4.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1441@1>fig|991.25.peg.1441</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=991.25@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-keto-6-deoxy-N-Acetyl-D-hexosaminyl-(Lipid carrier) aminotransferase&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>4-keto-6-deoxy-N-Acetyl-D-hexosaminyl-(Lipid carrier) aminotransferase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1443@1>fig|991.25.peg.1443</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=991.25@1>Aminopeptidases (EC 3.4.11.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xaa-Pro aminopeptidase (EC 3.4.11.9)&subsystem_name=Aminopeptidases_(EC_3.4.11.-)@1>Xaa-Pro aminopeptidase (EC 3.4.11.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.152@1>fig|991.25.peg.152</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1501@1>fig|991.25.peg.1501</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2295@1>fig|991.25.peg.2295</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=991.25@1>Aminopeptidases (EC 3.4.11.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Membrane alanine aminopeptidase N (EC 3.4.11.2)&subsystem_name=Aminopeptidases_(EC_3.4.11.-)@1>Membrane alanine aminopeptidase N (EC 3.4.11.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1081@1>fig|991.25.peg.1081</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=991.25@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)&subsystem_name=Protein_degradation@1>Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2941@1>fig|991.25.peg.2941</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=991.25@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)&subsystem_name=Protein_degradation@1>Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1349@1>fig|991.25.peg.1349</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2927@1>fig|991.25.peg.2927</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=991.25@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminopeptidase C (EC 3.4.22.40)&subsystem_name=Protein_degradation@1>Aminopeptidase C (EC 3.4.22.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.838@1>fig|991.25.peg.838</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=991.25@1>Metallocarboxypeptidases (EC 3.4.17.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)&subsystem_name=Metallocarboxypeptidases_(EC_3.4.17.-)@1>D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.512@1>fig|991.25.peg.512</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1286@1>fig|991.25.peg.1286</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=991.25@1>Metallocarboxypeptidases (EC 3.4.17.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Metallocarboxypeptidases_(EC_3.4.17.-)@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.885@1>fig|991.25.peg.885</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dipeptidases_(EC_3.4.13.-)&organism=991.25@1>Dipeptidases (EC 3.4.13.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)&subsystem_name=Dipeptidases_(EC_3.4.13.-)@1>Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3617@1>fig|991.25.peg.3617</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_endopeptidase_(EC_3.4.21.-)&organism=991.25@1>Serine endopeptidase (EC 3.4.21.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prolyl endopeptidase (EC 3.4.21.26)&subsystem_name=Serine_endopeptidase_(EC_3.4.21.-)@1>Prolyl endopeptidase (EC 3.4.21.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1019@1>fig|991.25.peg.1019</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=991.25@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RadA&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>DNA repair protein RadA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.34@1>fig|991.25.peg.34</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=991.25@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent Clp protease ATP-binding subunit ClpX&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent Clp protease ATP-binding subunit ClpX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3696@1>fig|991.25.peg.3696</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=991.25@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent Clp protease, ATP-binding subunit ClpC&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent Clp protease, ATP-binding subunit ClpC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1716@1>fig|991.25.peg.1716</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=991.25@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent protease La (EC 3.4.21.53) Type I&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent protease La (EC 3.4.21.53) Type I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1701@1>fig|991.25.peg.1701</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Salmonella_enterica_Typhimurium&organism=991.25@1>LysR-family proteins in Salmonella enterica Typhimurium</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator MetR&subsystem_name=LysR-family_proteins_in_Salmonella_enterica_Typhimurium@1>Transcriptional activator MetR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.814@1>fig|991.25.peg.814</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Escherichia_coli&organism=991.25@1>LysR-family proteins in Escherichia coli</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator MetR&subsystem_name=LysR-family_proteins_in_Escherichia_coli@1>Transcriptional activator MetR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.814@1>fig|991.25.peg.814</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=991.25@1>Stringent Response, (p)ppGpp metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II&subsystem_name=Stringent_Response,_(p)ppGpp_metabolism@1>GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2583@1>fig|991.25.peg.2583</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=991.25@1>Stringent Response, (p)ppGpp metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Guanosine-3@1,5@1-bis(diphosphate) 3@1-pyrophosphohydrolase (EC 3.1.7.2)&subsystem_name=Stringent_Response,_(p)ppGpp_metabolism@1>Guanosine-3@1,5@1-bis(diphosphate) 3@1-pyrophosphohydrolase (EC 3.1.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2583@1>fig|991.25.peg.2583</a>@~Secondary Metabolism@^Secondary Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lanthionine_Synthetases&organism=991.25@1>Lanthionine Synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Lanthionine_Synthetases@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1618@1>fig|991.25.peg.1618</a>@~Secondary Metabolism@^Secondary Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lanthionine_Synthetases&organism=991.25@1>Lanthionine Synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lanthionine biosynthesis protein LanB&subsystem_name=Lanthionine_Synthetases@1>Lanthionine biosynthesis protein LanB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.510@1>fig|991.25.peg.510</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.934@1>fig|991.25.peg.934</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2523@1>fig|991.25.peg.2523</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2524@1>fig|991.25.peg.2524</a>@~Secondary Metabolism@^Plant Alkaloids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alkaloid_biosynthesis_from_L-lysine&organism=991.25@1>Alkaloid biosynthesis from L-lysine</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyhypusine synthase (EC 2.5.1.46)&subsystem_name=Alkaloid_biosynthesis_from_L-lysine@1>Deoxyhypusine synthase (EC 2.5.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4087@1>fig|991.25.peg.4087</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=991.25@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Auxin_biosynthesis@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.423@1>fig|991.25.peg.423</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=991.25@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)&subsystem_name=Auxin_biosynthesis@1>Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1244@1>fig|991.25.peg.1244</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=991.25@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Auxin_biosynthesis@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.429@1>fig|991.25.peg.429</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=991.25@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Auxin_biosynthesis@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.426@1>fig|991.25.peg.426</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=991.25@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Auxin_biosynthesis@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.427@1>fig|991.25.peg.427</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease VII large subunit (EC 3.1.11.6)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease VII large subunit (EC 3.1.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2478@1>fig|991.25.peg.2478</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RadA&subsystem_name=DNA_repair,_bacterial@1>DNA repair protein RadA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.34@1>fig|991.25.peg.34</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ADA regulatory protein&subsystem_name=DNA_repair,_bacterial@1>ADA regulatory protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2968@1>fig|991.25.peg.2968</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RecN&subsystem_name=DNA_repair,_bacterial@1>DNA repair protein RecN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1633@1>fig|991.25.peg.1633</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination protein RmuC&subsystem_name=DNA_repair,_bacterial@1>DNA recombination protein RmuC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2446@1>fig|991.25.peg.2446</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=A/G-specific adenine glycosylase (EC 3.2.2.-)&subsystem_name=DNA_repair,_bacterial@1>A/G-specific adenine glycosylase (EC 3.2.2.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1273@1>fig|991.25.peg.1273</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA polymerase IV (EC 2.7.7.7)&subsystem_name=DNA_repair,_bacterial@1>DNA polymerase IV (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.595@1>fig|991.25.peg.595</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.900@1>fig|991.25.peg.900</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1960@1>fig|991.25.peg.1960</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2485@1>fig|991.25.peg.2485</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2688@1>fig|991.25.peg.2688</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Endonuclease V (EC 3.1.21.7)&subsystem_name=DNA_repair,_bacterial@1>Endonuclease V (EC 3.1.21.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2274@1>fig|991.25.peg.2274</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2491@1>fig|991.25.peg.2491</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2492@1>fig|991.25.peg.2492</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Single-stranded DNA-binding protein&subsystem_name=DNA_repair,_bacterial@1>Single-stranded DNA-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.607@1>fig|991.25.peg.607</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.821@1>fig|991.25.peg.821</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1274@1>fig|991.25.peg.1274</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1904@1>fig|991.25.peg.1904</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2685@1>fig|991.25.peg.2685</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4687@1>fig|991.25.peg.4687</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)&subsystem_name=DNA_repair,_bacterial@1>Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.771@1>fig|991.25.peg.771</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2968@1>fig|991.25.peg.2968</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair,_bacterial@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5046@1>fig|991.25.peg.5046</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease III (EC 3.1.11.2)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease III (EC 3.1.11.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.22@1>fig|991.25.peg.22</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=991.25@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease VII small subunit (EC 3.1.11.6)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease VII small subunit (EC 3.1.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2479@1>fig|991.25.peg.2479</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=991.25@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA helicase RecQ&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>ATP-dependent DNA helicase RecQ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1493@1>fig|991.25.peg.1493</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4085@1>fig|991.25.peg.4085</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=991.25@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Single-stranded DNA-binding protein&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>Single-stranded DNA-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.607@1>fig|991.25.peg.607</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.821@1>fig|991.25.peg.821</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1274@1>fig|991.25.peg.1274</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1904@1>fig|991.25.peg.1904</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2685@1>fig|991.25.peg.2685</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4687@1>fig|991.25.peg.4687</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=991.25@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination protein RecR&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>Recombination protein RecR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1417@1>fig|991.25.peg.1417</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=991.25@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5046@1>fig|991.25.peg.5046</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=991.25@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination and repair protein RecO&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>DNA recombination and repair protein RecO</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.135@1>fig|991.25.peg.135</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=991.25@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination and repair protein RecF&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>DNA recombination and repair protein RecF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1878@1>fig|991.25.peg.1878</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=991.25@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA helicase RecS (RecQ family)&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>ATP-dependent DNA helicase RecS (RecQ family)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.706@1>fig|991.25.peg.706</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=991.25@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination inhibitory protein MutS2&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>Recombination inhibitory protein MutS2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3298@1>fig|991.25.peg.3298</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=991.25@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=MutS-related protein, family 1&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>MutS-related protein, family 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4953@1>fig|991.25.peg.4953</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=991.25@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutS&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>DNA mismatch repair protein MutS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3672@1>fig|991.25.peg.3672</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=991.25@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutL&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>DNA mismatch repair protein MutL</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1881@1>fig|991.25.peg.1881</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=991.25@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit A paralog in greater Bacteroides group&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit A paralog in greater Bacteroides group</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4166@1>fig|991.25.peg.4166</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=991.25@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit C&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit C</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.115@1>fig|991.25.peg.115</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=991.25@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit A&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit A</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3787@1>fig|991.25.peg.3787</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=991.25@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit B&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit B</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.357@1>fig|991.25.peg.357</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_photolyase&organism=991.25@1>DNA repair, bacterial photolyase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribodipyrimidine photolyase (EC 4.1.99.3)&subsystem_name=DNA_repair,_bacterial_photolyase@1>Deoxyribodipyrimidine photolyase (EC 4.1.99.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2453@1>fig|991.25.peg.2453</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RecA_and_RecX&organism=991.25@1>RecA and RecX</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=RecA_and_RecX@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5046@1>fig|991.25.peg.5046</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RecA_and_RecX&organism=991.25@1>RecA and RecX</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Regulatory protein RecX&subsystem_name=RecA_and_RecX@1>Regulatory protein RecX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.57@1>fig|991.25.peg.57</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=991.25@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutS&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>DNA mismatch repair protein MutS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3672@1>fig|991.25.peg.3672</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=991.25@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5046@1>fig|991.25.peg.5046</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=991.25@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Regulatory protein RecX&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>Regulatory protein RecX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.57@1>fig|991.25.peg.57</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=991.25@1>Type I Restriction-Modification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)&subsystem_name=Type_I_Restriction-Modification@1>Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3137@1>fig|991.25.peg.3137</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4258@1>fig|991.25.peg.4258</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=991.25@1>Type I Restriction-Modification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)&subsystem_name=Type_I_Restriction-Modification@1>Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3141@1>fig|991.25.peg.3141</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3144@1>fig|991.25.peg.3144</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4256@1>fig|991.25.peg.4256</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=991.25@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)&subsystem_name=Restriction-Modification_System@1>Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3137@1>fig|991.25.peg.3137</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4258@1>fig|991.25.peg.4258</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=991.25@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)&subsystem_name=Restriction-Modification_System@1>Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3141@1>fig|991.25.peg.3141</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3144@1>fig|991.25.peg.3144</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4256@1>fig|991.25.peg.4256</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=YcfH&organism=991.25@1>YcfH</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribonuclease TatD&subsystem_name=YcfH@1>Deoxyribonuclease TatD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4581@1>fig|991.25.peg.4581</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=YcfH&organism=991.25@1>YcfH</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative deoxyribonuclease YjjV&subsystem_name=YcfH@1>Putative deoxyribonuclease YjjV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4012@1>fig|991.25.peg.4012</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=991.25@1>DNA structural proteins, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-binding protein HU-beta&subsystem_name=DNA_structural_proteins,_bacterial@1>DNA-binding protein HU-beta</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.708@1>fig|991.25.peg.708</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=991.25@1>DNA structural proteins, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Integration host factor alpha/beta&subsystem_name=DNA_structural_proteins,_bacterial@1>Integration host factor alpha/beta</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1272@1>fig|991.25.peg.1272</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_phosphorothioation&organism=991.25@1>DNA phosphorothioation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3@1-phosphoadenosine 5@1-phosphosulfate sulfurtransferase DndC&subsystem_name=DNA_phosphorothioation@1>3@1-phosphoadenosine 5@1-phosphosulfate sulfurtransferase DndC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.614@1>fig|991.25.peg.614</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1903@1>fig|991.25.peg.1903</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_phosphorothioation&organism=991.25@1>DNA phosphorothioation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA sulfur modification protein DndE&subsystem_name=DNA_phosphorothioation@1>DNA sulfur modification protein DndE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.616@1>fig|991.25.peg.616</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_phosphorothioation&organism=991.25@1>DNA phosphorothioation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA sulfur modification protein DndD&subsystem_name=DNA_phosphorothioation@1>DNA sulfur modification protein DndD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.615@1>fig|991.25.peg.615</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1902@1>fig|991.25.peg.1902</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=991.25@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase III (EC 5.99.1.2)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>DNA topoisomerase III (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2540@1>fig|991.25.peg.2540</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=991.25@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase I (EC 5.99.1.2)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>DNA topoisomerase I (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3487@1>fig|991.25.peg.3487</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=991.25@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase III, Bacteroidales-type (EC 5.99.1.2)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>DNA topoisomerase III, Bacteroidales-type (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1939@1>fig|991.25.peg.1939</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_replication_strays&organism=991.25@1>DNA replication strays</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Error-prone repair homolog of DNA polymerase III alpha subunit (EC 2.7.7.7)&subsystem_name=DNA_replication_strays@1>Error-prone repair homolog of DNA polymerase III alpha subunit (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.596@1>fig|991.25.peg.596</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1961@1>fig|991.25.peg.1961</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2484@1>fig|991.25.peg.2484</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2687@1>fig|991.25.peg.2687</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=991.25@1>DNA topoisomerases, Type II, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit B (EC 5.99.1.3)&subsystem_name=DNA_topoisomerases,_Type_II,_ATP-dependent@1>DNA gyrase subunit B (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2563@1>fig|991.25.peg.2563</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=991.25@1>DNA topoisomerases, Type II, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit A (EC 5.99.1.3)&subsystem_name=DNA_topoisomerases,_Type_II,_ATP-dependent@1>DNA gyrase subunit A (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1715@1>fig|991.25.peg.1715</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=991.25@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination protein RecR&subsystem_name=DNA_processing_cluster@1>Recombination protein RecR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1417@1>fig|991.25.peg.1417</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=991.25@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase III (EC 5.99.1.2)&subsystem_name=DNA_processing_cluster@1>DNA topoisomerase III (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2540@1>fig|991.25.peg.2540</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=991.25@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA polymerase III subunits gamma and tau (EC 2.7.7.7)&subsystem_name=DNA_processing_cluster@1>DNA polymerase III subunits gamma and tau (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3761@1>fig|991.25.peg.3761</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3763@1>fig|991.25.peg.3763</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin carboxyl carrier protein of acetyl-CoA carboxylase&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Biotin carboxyl carrier protein of acetyl-CoA carboxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.189@1>fig|991.25.peg.189</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.190@1>fig|991.25.peg.190</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1187@1>fig|991.25.peg.1187</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acyl carrier protein&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acyl carrier protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.633@1>fig|991.25.peg.633</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3355@1>fig|991.25.peg.3355</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3869@1>fig|991.25.peg.3869</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3886@1>fig|991.25.peg.3886</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3901@1>fig|991.25.peg.3901</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4802@1>fig|991.25.peg.4802</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3356@1>fig|991.25.peg.3356</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3887@1>fig|991.25.peg.3887</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acyl carrier protein phosphodiesterase (EC 3.1.4.14)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acyl carrier protein phosphodiesterase (EC 3.1.4.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3791@1>fig|991.25.peg.3791</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.631@1>fig|991.25.peg.631</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3662@1>fig|991.25.peg.3662</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=991.25@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3311@1>fig|991.25.peg.3311</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3357@1>fig|991.25.peg.3357</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3862@1>fig|991.25.peg.3862</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=991.25@1>Fatty acid metabolism cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)&subsystem_name=Fatty_acid_metabolism_cluster@1>Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1254@1>fig|991.25.peg.1254</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2011@1>fig|991.25.peg.2011</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=991.25@1>Fatty acid metabolism cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Fatty_acid_metabolism_cluster@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.753@1>fig|991.25.peg.753</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=991.25@1>Fatty acid metabolism cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-ketoacyl-CoA thiolase (EC 2.3.1.16)&subsystem_name=Fatty_acid_metabolism_cluster@1>3-ketoacyl-CoA thiolase (EC 2.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3221@1>fig|991.25.peg.3221</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Myxoxanthophyll_biosynthesis_in_Cyanobacteria&organism=991.25@1>Myxoxanthophyll biosynthesis in Cyanobacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-carotene hydroxylase&subsystem_name=Myxoxanthophyll_biosynthesis_in_Cyanobacteria@1>Beta-carotene hydroxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2455@1>fig|991.25.peg.2455</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=991.25@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diphosphomevalonate decarboxylase (EC 4.1.1.33)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Diphosphomevalonate decarboxylase (EC 4.1.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1134@1>fig|991.25.peg.1134</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=991.25@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomevalonate kinase (EC 2.7.4.2)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Phosphomevalonate kinase (EC 2.7.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1131@1>fig|991.25.peg.1131</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=991.25@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mevalonate kinase (EC 2.7.1.36)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Mevalonate kinase (EC 2.7.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1162@1>fig|991.25.peg.1162</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=991.25@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.162@1>fig|991.25.peg.162</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=991.25@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1163@1>fig|991.25.peg.1163</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1509@1>fig|991.25.peg.1509</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=991.25@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2143@1>fig|991.25.peg.2143</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=991.25@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Undecaprenyl diphosphate synthase (EC 2.5.1.31)&subsystem_name=Isoprenoinds_for_Quinones@1>Undecaprenyl diphosphate synthase (EC 2.5.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3708@1>fig|991.25.peg.3708</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=991.25@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Geranylgeranyl diphosphate synthase (EC 2.5.1.29)&subsystem_name=Isoprenoinds_for_Quinones@1>Geranylgeranyl diphosphate synthase (EC 2.5.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.931@1>fig|991.25.peg.931</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2155@1>fig|991.25.peg.2155</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=991.25@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hexaprenyl diphosphate synthase (farnesyl-diphosphate specific) (EC 2.5.1.83)&subsystem_name=Isoprenoinds_for_Quinones@1>Hexaprenyl diphosphate synthase (farnesyl-diphosphate specific) (EC 2.5.1.83)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3603@1>fig|991.25.peg.3603</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=991.25@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mevalonate kinase (EC 2.7.1.36)&subsystem_name=Isoprenoid_Biosynthesis@1>Mevalonate kinase (EC 2.7.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1162@1>fig|991.25.peg.1162</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=991.25@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Isoprenoid_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2143@1>fig|991.25.peg.2143</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=991.25@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Isoprenoid_Biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1262@1>fig|991.25.peg.1262</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=991.25@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diphosphomevalonate decarboxylase (EC 4.1.1.33)&subsystem_name=Isoprenoid_Biosynthesis@1>Diphosphomevalonate decarboxylase (EC 4.1.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1134@1>fig|991.25.peg.1134</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=991.25@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Undecaprenyl diphosphate synthase (EC 2.5.1.31)&subsystem_name=Isoprenoid_Biosynthesis@1>Undecaprenyl diphosphate synthase (EC 2.5.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3708@1>fig|991.25.peg.3708</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=991.25@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomevalonate kinase (EC 2.7.4.2)&subsystem_name=Isoprenoid_Biosynthesis@1>Phosphomevalonate kinase (EC 2.7.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1131@1>fig|991.25.peg.1131</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=991.25@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)&subsystem_name=Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.162@1>fig|991.25.peg.162</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=991.25@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)&subsystem_name=Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1163@1>fig|991.25.peg.1163</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1509@1>fig|991.25.peg.1509</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nitrosative_stress&organism=991.25@1>Nitrosative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric-oxide reductase (EC 1.7.99.7), quinol-dependent&subsystem_name=Nitrosative_stress@1>Nitric-oxide reductase (EC 1.7.99.7), quinol-dependent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4150@1>fig|991.25.peg.4150</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nitrate_and_nitrite_ammonification&organism=991.25@1>Nitrate and nitrite ammonification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c552 precursor (EC 1.7.2.2)&subsystem_name=Nitrate_and_nitrite_ammonification@1>Cytochrome c552 precursor (EC 1.7.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.347@1>fig|991.25.peg.347</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nitrate_and_nitrite_ammonification&organism=991.25@1>Nitrate and nitrite ammonification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c nitrite reductase, small subunit NrfH&subsystem_name=Nitrate_and_nitrite_ammonification@1>Cytochrome c nitrite reductase, small subunit NrfH</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.348@1>fig|991.25.peg.348</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=991.25@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Ammonia_assimilation@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4253@1>fig|991.25.peg.4253</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=991.25@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] small chain (EC 1.4.1.13)&subsystem_name=Ammonia_assimilation@1>Glutamate synthase [NADPH] small chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4665@1>fig|991.25.peg.4665</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=991.25@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)&subsystem_name=Ammonia_assimilation@1>Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4146@1>fig|991.25.peg.4146</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=991.25@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ammonium transporter&subsystem_name=Ammonia_assimilation@1>Ammonium transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.699@1>fig|991.25.peg.699</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1028@1>fig|991.25.peg.1028</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=991.25@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)&subsystem_name=Ammonia_assimilation@1>Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4254@1>fig|991.25.peg.4254</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=991.25@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] large chain (EC 1.4.1.13)&subsystem_name=Ammonia_assimilation@1>Glutamate synthase [NADPH] large chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4664@1>fig|991.25.peg.4664</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=991.25@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrite reductase accessory protein NirV&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitrite reductase accessory protein NirV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4154@1>fig|991.25.peg.4154</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=991.25@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-containing nitrite reductase (EC 1.7.2.1)&subsystem_name=Denitrifying_reductase_gene_clusters@1>Copper-containing nitrite reductase (EC 1.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4155@1>fig|991.25.peg.4155</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=991.25@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrite reductase accessory protein NirV&subsystem_name=Denitrification@1>Nitrite reductase accessory protein NirV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4154@1>fig|991.25.peg.4154</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=991.25@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric-oxide reductase (EC 1.7.99.7), quinol-dependent&subsystem_name=Denitrification@1>Nitric-oxide reductase (EC 1.7.99.7), quinol-dependent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4150@1>fig|991.25.peg.4150</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=991.25@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-containing nitrite reductase (EC 1.7.2.1)&subsystem_name=Denitrification@1>Copper-containing nitrite reductase (EC 1.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4155@1>fig|991.25.peg.4155</a>@~Dormancy and Sporulation@^Dormancy and Sporulation - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Sporulation-associated_proteins_with_broader_functions&organism=991.25@1>Sporulation-associated proteins with broader functions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidyl-tRNA hydrolase (EC 3.1.1.29)&subsystem_name=Sporulation-associated_proteins_with_broader_functions@1>Peptidyl-tRNA hydrolase (EC 3.1.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3910@1>fig|991.25.peg.3910</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit c (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit c (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3831@1>fig|991.25.peg.3831</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase delta chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase delta chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3833@1>fig|991.25.peg.3833</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase protein I&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase protein I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3828@1>fig|991.25.peg.3828</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3829@1>fig|991.25.peg.3829</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase beta chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase beta chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.45@1>fig|991.25.peg.45</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase gamma chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase gamma chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3835@1>fig|991.25.peg.3835</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit b (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit b (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3832@1>fig|991.25.peg.3832</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit a (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit a (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3830@1>fig|991.25.peg.3830</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase alpha chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase alpha chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3834@1>fig|991.25.peg.3834</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=991.25@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase epsilon chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase epsilon chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.46@1>fig|991.25.peg.46</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_d_ubiquinol_oxidases&organism=991.25@1>Terminal cytochrome d ubiquinol oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_d_ubiquinol_oxidases@1>Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.443@1>fig|991.25.peg.443</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_d_ubiquinol_oxidases&organism=991.25@1>Terminal cytochrome d ubiquinol oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_d_ubiquinol_oxidases@1>Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.444@1>fig|991.25.peg.444</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=991.25@1>Anaerobic respiratory reductases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Coenzyme F420-dependent oxidoreductase&subsystem_name=Anaerobic_respiratory_reductases@1>Coenzyme F420-dependent oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1569@1>fig|991.25.peg.1569</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=991.25@1>Anaerobic respiratory reductases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Flavodoxin reductases (ferredoxin-NADPH reductases) family 1&subsystem_name=Anaerobic_respiratory_reductases@1>Flavodoxin reductases (ferredoxin-NADPH reductases) family 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2940@1>fig|991.25.peg.2940</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3823@1>fig|991.25.peg.3823</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_oxidases&organism=991.25@1>Terminal cytochrome oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_oxidases@1>Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.443@1>fig|991.25.peg.443</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_oxidases&organism=991.25@1>Terminal cytochrome oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_oxidases@1>Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.444@1>fig|991.25.peg.444</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.906@1>fig|991.25.peg.906</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.905@1>fig|991.25.peg.905</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.907@1>fig|991.25.peg.907</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.903@1>fig|991.25.peg.903</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.909@1>fig|991.25.peg.909</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.912@1>fig|991.25.peg.912</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.915@1>fig|991.25.peg.915</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.908@1>fig|991.25.peg.908</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.904@1>fig|991.25.peg.904</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.911@1>fig|991.25.peg.911</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH dehydrogenase subunit 2&subsystem_name=Respiratory_Complex_I@1>NADH dehydrogenase subunit 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1893@1>fig|991.25.peg.1893</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.910@1>fig|991.25.peg.910</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.914@1>fig|991.25.peg.914</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.916@1>fig|991.25.peg.916</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=991.25@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.913@1>fig|991.25.peg.913</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=991.25@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH dehydrogenase (EC 1.6.99.3)&subsystem_name=Respiratory_dehydrogenases_1@1>NADH dehydrogenase (EC 1.6.99.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3349@1>fig|991.25.peg.3349</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3457@1>fig|991.25.peg.3457</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=991.25@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-methanol-choline (GMC) oxidoreductase:NAD binding site&subsystem_name=Respiratory_dehydrogenases_1@1>Glucose-methanol-choline (GMC) oxidoreductase:NAD binding site</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2228@1>fig|991.25.peg.2228</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=991.25@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)&subsystem_name=Respiratory_dehydrogenases_1@1>Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3489@1>fig|991.25.peg.3489</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=991.25@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypF&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase metallocenter assembly protein HypF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2590@1>fig|991.25.peg.2590</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=991.25@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase nickel incorporation protein HypA&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase nickel incorporation protein HypA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2595@1>fig|991.25.peg.2595</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=991.25@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypC&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase metallocenter assembly protein HypC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2597@1>fig|991.25.peg.2597</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=991.25@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase nickel incorporation-associated protein HypB&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase nickel incorporation-associated protein HypB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2596@1>fig|991.25.peg.2596</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=991.25@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypD&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase metallocenter assembly protein HypD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2588@1>fig|991.25.peg.2588</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=991.25@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypE&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase metallocenter assembly protein HypE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2598@1>fig|991.25.peg.2598</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.906@1>fig|991.25.peg.906</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.905@1>fig|991.25.peg.905</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.907@1>fig|991.25.peg.907</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.903@1>fig|991.25.peg.903</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.909@1>fig|991.25.peg.909</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.912@1>fig|991.25.peg.912</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.915@1>fig|991.25.peg.915</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.908@1>fig|991.25.peg.908</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.904@1>fig|991.25.peg.904</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.911@1>fig|991.25.peg.911</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.910@1>fig|991.25.peg.910</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.914@1>fig|991.25.peg.914</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.916@1>fig|991.25.peg.916</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=991.25@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.913@1>fig|991.25.peg.913</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Quinone_oxidoreductase_family&organism=991.25@1>Quinone oxidoreductase family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinone oxidoreductase (EC 1.6.5.5)&subsystem_name=Quinone_oxidoreductase_family@1>Quinone oxidoreductase (EC 1.6.5.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1500@1>fig|991.25.peg.1500</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cytochrome_c_oxidases&organism=991.25@1>Biogenesis of cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Heme A synthase, cytochrome oxidase biogenesis protein Cox15-CtaA&subsystem_name=Biogenesis_of_cytochrome_c_oxidases@1>Heme A synthase, cytochrome oxidase biogenesis protein Cox15-CtaA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.455@1>fig|991.25.peg.455</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=991.25@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoI&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoI</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1767@1>fig|991.25.peg.1767</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=991.25@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoS, involved in heme b insertion&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoS, involved in heme b insertion</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1766@1>fig|991.25.peg.1766</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=991.25@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoG, involved in Cu oxidation&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoG, involved in Cu oxidation</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1762@1>fig|991.25.peg.1762</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=991.25@1>Biogenesis of c-type cytochromes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative cytochrome C-type biogenesis protein&subsystem_name=Biogenesis_of_c-type_cytochromes@1>Putative cytochrome C-type biogenesis protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4037@1>fig|991.25.peg.4037</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=991.25@1>Biogenesis of c-type cytochromes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)&subsystem_name=Biogenesis_of_c-type_cytochromes@1>Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.698@1>fig|991.25.peg.698</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2767@1>fig|991.25.peg.2767</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=991.25@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1041@1>fig|991.25.peg.1041</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=991.25@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c551/c552&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Cytochrome c551/c552</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4723@1>fig|991.25.peg.4723</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=991.25@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Ferredoxin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4199@1>fig|991.25.peg.4199</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=991.25@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c552 precursor (EC 1.7.2.2)&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Cytochrome c552 precursor (EC 1.7.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.347@1>fig|991.25.peg.347</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=991.25@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c552&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Cytochrome c552</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.770@1>fig|991.25.peg.770</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=991.25@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane protein A precursor&subsystem_name=Osmoregulation@1>Outer membrane protein A precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3918@1>fig|991.25.peg.3918</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=991.25@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aquaporin Z&subsystem_name=Osmoregulation@1>Aquaporin Z</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1515@1>fig|991.25.peg.1515</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=991.25@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol uptake facilitator protein&subsystem_name=Osmoregulation@1>Glycerol uptake facilitator protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3491@1>fig|991.25.peg.3491</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=991.25@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric uptake regulation protein FUR&subsystem_name=Oxidative_stress@1>Ferric uptake regulation protein FUR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2584@1>fig|991.25.peg.2584</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=991.25@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Superoxide dismutase [Mn] (EC 1.15.1.1)&subsystem_name=Oxidative_stress@1>Superoxide dismutase [Mn] (EC 1.15.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3909@1>fig|991.25.peg.3909</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4760@1>fig|991.25.peg.4760</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=991.25@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional regulator, Crp/Fnr family&subsystem_name=Oxidative_stress@1>Transcriptional regulator, Crp/Fnr family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.448@1>fig|991.25.peg.448</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1768@1>fig|991.25.peg.1768</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2359@1>fig|991.25.peg.2359</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2845@1>fig|991.25.peg.2845</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2953@1>fig|991.25.peg.2953</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=991.25@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Oxidative_stress@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.525@1>fig|991.25.peg.525</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3114@1>fig|991.25.peg.3114</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=991.25@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Zinc uptake regulation protein ZUR&subsystem_name=Oxidative_stress@1>Zinc uptake regulation protein ZUR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3392@1>fig|991.25.peg.3392</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=991.25@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)&subsystem_name=Oxidative_stress@1>Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3087@1>fig|991.25.peg.3087</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Non-redox_reactions&organism=991.25@1>Glutathione: Non-redox reactions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lactoylglutathione lyase (EC 4.4.1.5)&subsystem_name=Glutathione:_Non-redox_reactions@1>Lactoylglutathione lyase (EC 4.4.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4901@1>fig|991.25.peg.4901</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Non-redox_reactions&organism=991.25@1>Glutathione: Non-redox reactions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxyacylglutathione hydrolase (EC 3.1.2.6)&subsystem_name=Glutathione:_Non-redox_reactions@1>Hydroxyacylglutathione hydrolase (EC 3.1.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.640@1>fig|991.25.peg.640</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Redox_cycle&organism=991.25@1>Glutathione: Redox cycle</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutathione peroxidase (EC 1.11.1.9)&subsystem_name=Glutathione:_Redox_cycle@1>Glutathione peroxidase (EC 1.11.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4047@1>fig|991.25.peg.4047</a>@~Stress Response@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Uptake_of_selenate_and_selenite&organism=991.25@1>Uptake of selenate and selenite</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DedA protein&subsystem_name=Uptake_of_selenate_and_selenite@1>DedA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3503@1>fig|991.25.peg.3503</a>@~Stress Response@^Stress Response - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dimethylarginine_metabolism&organism=991.25@1>Dimethylarginine metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ornithine aminotransferase (EC 2.6.1.13)&subsystem_name=Dimethylarginine_metabolism@1>Ornithine aminotransferase (EC 2.6.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2102@1>fig|991.25.peg.2102</a>@~Stress Response@^Stress Response - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dimethylarginine_metabolism&organism=991.25@1>Dimethylarginine metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NG,NG-dimethylarginine dimethylaminohydrolase 1 (EC 3.5.3.18)&subsystem_name=Dimethylarginine_metabolism@1>NG,NG-dimethylarginine dimethylaminohydrolase 1 (EC 3.5.3.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1412@1>fig|991.25.peg.1412</a>@~Stress Response@^Periplasmic Stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=991.25@1>Periplasmic Stress Response</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane protein H precursor&subsystem_name=Periplasmic_Stress_Response@1>Outer membrane protein H precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2325@1>fig|991.25.peg.2325</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3710@1>fig|991.25.peg.3710</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3711@1>fig|991.25.peg.3711</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5037@1>fig|991.25.peg.5037</a>@~Stress Response@^Periplasmic Stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=991.25@1>Periplasmic Stress Response</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Intramembrane protease RasP/YluC, implicated in cell division based on FtsL cleavage&subsystem_name=Periplasmic_Stress_Response@1>Intramembrane protease RasP/YluC, implicated in cell division based on FtsL cleavage</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.842@1>fig|991.25.peg.842</a>@~Metabolism of Aromatic Compounds@^Peripheral pathways for catabolism of aromatic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Quinate_degradation&organism=991.25@1>Quinate degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate dehydratase II (EC 4.2.1.10)&subsystem_name=Quinate_degradation@1>3-dehydroquinate dehydratase II (EC 4.2.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4769@1>fig|991.25.peg.4769</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=991.25@1>Catechol branch of beta-ketoadipate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)&subsystem_name=Catechol_branch_of_beta-ketoadipate_pathway@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.779@1>fig|991.25.peg.779</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=991.25@1>Catechol branch of beta-ketoadipate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)&subsystem_name=Catechol_branch_of_beta-ketoadipate_pathway@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.778@1>fig|991.25.peg.778</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=991.25@1>Salicylate and gentisate catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetase (EC 3.7.1.2)&subsystem_name=Salicylate_and_gentisate_catabolism@1>Fumarylacetoacetase (EC 3.7.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3014@1>fig|991.25.peg.3014</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4559@1>fig|991.25.peg.4559</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=991.25@1>Salicylate and gentisate catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetate hydrolase family protein&subsystem_name=Salicylate_and_gentisate_catabolism@1>Fumarylacetoacetate hydrolase family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2637@1>fig|991.25.peg.2637</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=991.25@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homogentisate 1,2-dioxygenase (EC 1.13.11.5)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>Homogentisate 1,2-dioxygenase (EC 1.13.11.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.112@1>fig|991.25.peg.112</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=991.25@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetase (EC 3.7.1.2)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>Fumarylacetoacetase (EC 3.7.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3014@1>fig|991.25.peg.3014</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4559@1>fig|991.25.peg.4559</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=991.25@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.111@1>fig|991.25.peg.111</a>@~Metabolism of Aromatic Compounds@^Metabolism of Aromatic Compounds - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Gentisate_degradation&organism=991.25@1>Gentisate degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetate hydrolase family protein&subsystem_name=Gentisate_degradation@1>Fumarylacetoacetate hydrolase family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2637@1>fig|991.25.peg.2637</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-asparaginase I, cytoplasmic (EC 3.5.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>L-asparaginase I, cytoplasmic (EC 3.5.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.978@1>fig|991.25.peg.978</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4146@1>fig|991.25.peg.4146</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-asparaginase (EC 3.5.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>L-asparaginase (EC 3.5.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2283@1>fig|991.25.peg.2283</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate aminotransferase (EC 2.6.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate aminotransferase (EC 2.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.302@1>fig|991.25.peg.302</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2659@1>fig|991.25.peg.2659</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3644@1>fig|991.25.peg.3644</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4458@1>fig|991.25.peg.4458</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4740@1>fig|991.25.peg.4740</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-specific glutamate dehydrogenase (EC 1.4.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>NAD-specific glutamate dehydrogenase (EC 1.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4294@1>fig|991.25.peg.4294</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate racemase (EC 5.1.1.3)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate racemase (EC 5.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3712@1>fig|991.25.peg.3712</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2564@1>fig|991.25.peg.2564</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2565@1>fig|991.25.peg.2565</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4254@1>fig|991.25.peg.4254</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] large chain (EC 1.4.1.13)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate synthase [NADPH] large chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4664@1>fig|991.25.peg.4664</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4253@1>fig|991.25.peg.4253</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] small chain (EC 1.4.1.13)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate synthase [NADPH] small chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4665@1>fig|991.25.peg.4665</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.133@1>fig|991.25.peg.133</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate ammonia-lyase (EC 4.3.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate ammonia-lyase (EC 4.3.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2282@1>fig|991.25.peg.2282</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=991.25@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate racemase (EC 5.1.1.13)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate racemase (EC 5.1.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.377@1>fig|991.25.peg.377</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2044@1>fig|991.25.peg.2044</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=991.25@1>Glutamate dehydrogenases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Glutamate_dehydrogenases@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.133@1>fig|991.25.peg.133</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=991.25@1>Glutamate dehydrogenases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-specific glutamate dehydrogenase (EC 1.4.1.2)&subsystem_name=Glutamate_dehydrogenases@1>NAD-specific glutamate dehydrogenase (EC 1.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4294@1>fig|991.25.peg.4294</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=991.25@1>Glutamine synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Glutamine_synthetases@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4253@1>fig|991.25.peg.4253</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=991.25@1>Glutamine synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)&subsystem_name=Glutamine_synthetases@1>Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4254@1>fig|991.25.peg.4254</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=991.25@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formiminoglutamase (EC 3.5.3.8)&subsystem_name=Histidine_Degradation@1>Formiminoglutamase (EC 3.5.3.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2192@1>fig|991.25.peg.2192</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=991.25@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Imidazolonepropionase (EC 3.5.2.7)&subsystem_name=Histidine_Degradation@1>Imidazolonepropionase (EC 3.5.2.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2194@1>fig|991.25.peg.2194</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=991.25@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urocanate hydratase (EC 4.2.1.49)&subsystem_name=Histidine_Degradation@1>Urocanate hydratase (EC 4.2.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4372@1>fig|991.25.peg.4372</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=991.25@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidine ammonia-lyase (EC 4.3.1.3)&subsystem_name=Histidine_Degradation@1>Histidine ammonia-lyase (EC 4.3.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1723@1>fig|991.25.peg.1723</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=991.25@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)&subsystem_name=Histidine_Biosynthesis@1>Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3334@1>fig|991.25.peg.3334</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=991.25@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidinol-phosphate aminotransferase (EC 2.6.1.9)&subsystem_name=Histidine_Biosynthesis@1>Histidinol-phosphate aminotransferase (EC 2.6.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3331@1>fig|991.25.peg.3331</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=991.25@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosyl-ATP pyrophosphatase (EC 3.6.1.31)&subsystem_name=Histidine_Biosynthesis@1>Phosphoribosyl-ATP pyrophosphatase (EC 3.6.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3337@1>fig|991.25.peg.3337</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=991.25@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Imidazoleglycerol-phosphate dehydratase (EC 4.2.1.19)&subsystem_name=Histidine_Biosynthesis@1>Imidazoleglycerol-phosphate dehydratase (EC 4.2.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3332@1>fig|991.25.peg.3332</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=991.25@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosyl-AMP cyclohydrolase (EC 3.5.4.19)&subsystem_name=Histidine_Biosynthesis@1>Phosphoribosyl-AMP cyclohydrolase (EC 3.5.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3337@1>fig|991.25.peg.3337</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=991.25@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidinol-phosphatase (EC 3.1.3.15)&subsystem_name=Histidine_Biosynthesis@1>Histidinol-phosphatase (EC 3.1.3.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3332@1>fig|991.25.peg.3332</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=991.25@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidinol dehydrogenase (EC 1.1.1.23)&subsystem_name=Histidine_Biosynthesis@1>Histidinol dehydrogenase (EC 1.1.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3330@1>fig|991.25.peg.3330</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=991.25@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreD&subsystem_name=Urease_subunits@1>Urease accessory protein UreD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3189@1>fig|991.25.peg.3189</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=991.25@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreF&subsystem_name=Urease_subunits@1>Urease accessory protein UreF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3187@1>fig|991.25.peg.3187</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=991.25@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease beta subunit (EC 3.5.1.5)&subsystem_name=Urease_subunits@1>Urease beta subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3184@1>fig|991.25.peg.3184</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=991.25@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreG&subsystem_name=Urease_subunits@1>Urease accessory protein UreG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3188@1>fig|991.25.peg.3188</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=991.25@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease gamma subunit (EC 3.5.1.5)&subsystem_name=Urease_subunits@1>Urease gamma subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3183@1>fig|991.25.peg.3183</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=991.25@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreE&subsystem_name=Urease_subunits@1>Urease accessory protein UreE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3186@1>fig|991.25.peg.3186</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=991.25@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease alpha subunit (EC 3.5.1.5)&subsystem_name=Urease_subunits@1>Urease alpha subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3185@1>fig|991.25.peg.3185</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=991.25@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Spermidine synthase (EC 2.5.1.16)&subsystem_name=Polyamine_Metabolism@1>Spermidine synthase (EC 2.5.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.994@1>fig|991.25.peg.994</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=991.25@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Agmatine deiminase (EC 3.5.3.12)&subsystem_name=Polyamine_Metabolism@1>Agmatine deiminase (EC 3.5.3.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1585@1>fig|991.25.peg.1585</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3462@1>fig|991.25.peg.3462</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=991.25@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biosynthetic arginine decarboxylase (EC 4.1.1.19)&subsystem_name=Polyamine_Metabolism@1>Biosynthetic arginine decarboxylase (EC 4.1.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4088@1>fig|991.25.peg.4088</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=991.25@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Agmatine deiminase (EC 3.5.3.12)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Agmatine deiminase (EC 3.5.3.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1585@1>fig|991.25.peg.1585</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3462@1>fig|991.25.peg.3462</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=991.25@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ornithine cyclodeaminase (EC 4.3.1.12)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Ornithine cyclodeaminase (EC 4.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2304@1>fig|991.25.peg.2304</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=991.25@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3633@1>fig|991.25.peg.3633</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=991.25@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ornithine aminotransferase (EC 2.6.1.13)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Ornithine aminotransferase (EC 2.6.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2102@1>fig|991.25.peg.2102</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=991.25@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Arginine_and_Ornithine_Degradation@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.133@1>fig|991.25.peg.133</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=991.25@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biosynthetic arginine decarboxylase (EC 4.1.1.19)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Biosynthetic arginine decarboxylase (EC 4.1.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4088@1>fig|991.25.peg.4088</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nickel transporter UreH&subsystem_name=Urea_decomposition@1>Nickel transporter UreH</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2587@1>fig|991.25.peg.2587</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreF&subsystem_name=Urea_decomposition@1>Urease accessory protein UreF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3187@1>fig|991.25.peg.3187</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease gamma subunit (EC 3.5.1.5)&subsystem_name=Urea_decomposition@1>Urease gamma subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3183@1>fig|991.25.peg.3183</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreE&subsystem_name=Urea_decomposition@1>Urease accessory protein UreE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3186@1>fig|991.25.peg.3186</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Eukaryotic-type low-affinity urea transporter&subsystem_name=Urea_decomposition@1>Eukaryotic-type low-affinity urea transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3191@1>fig|991.25.peg.3191</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease alpha subunit (EC 3.5.1.5)&subsystem_name=Urea_decomposition@1>Urease alpha subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3185@1>fig|991.25.peg.3185</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreD&subsystem_name=Urea_decomposition@1>Urease accessory protein UreD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3189@1>fig|991.25.peg.3189</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease beta subunit (EC 3.5.1.5)&subsystem_name=Urea_decomposition@1>Urease beta subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3184@1>fig|991.25.peg.3184</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=991.25@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreG&subsystem_name=Urea_decomposition@1>Urease accessory protein UreG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3188@1>fig|991.25.peg.3188</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter ATP-binding protein&subsystem_name=Methionine_Biosynthesis@1>Methionine ABC transporter ATP-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4971@1>fig|991.25.peg.4971</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylmethionine synthetase (EC 2.5.1.6)&subsystem_name=Methionine_Biosynthesis@1>S-adenosylmethionine synthetase (EC 2.5.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1210@1>fig|991.25.peg.1210</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylhomoserine sulfhydrylase (EC 2.5.1.48)&subsystem_name=Methionine_Biosynthesis@1>O-succinylhomoserine sulfhydrylase (EC 2.5.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.690@1>fig|991.25.peg.690</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1215@1>fig|991.25.peg.1215</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1224@1>fig|991.25.peg.1224</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Methionine_Biosynthesis@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.131@1>fig|991.25.peg.131</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine acetyltransferase (EC 2.3.1.30)&subsystem_name=Methionine_Biosynthesis@1>Serine acetyltransferase (EC 2.3.1.30)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1433@1>fig|991.25.peg.1433</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1800@1>fig|991.25.peg.1800</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3447@1>fig|991.25.peg.3447</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine kinase (EC 2.7.1.39)&subsystem_name=Methionine_Biosynthesis@1>Homoserine kinase (EC 2.7.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1728@1>fig|991.25.peg.1728</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase (EC 2.1.1.14)&subsystem_name=Methionine_Biosynthesis@1>5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase (EC 2.1.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1257@1>fig|991.25.peg.1257</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine synthase (EC 2.5.1.47)&subsystem_name=Methionine_Biosynthesis@1>Cysteine synthase (EC 2.5.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1801@1>fig|991.25.peg.1801</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-acetylhomoserine sulfhydrylase (EC 2.5.1.49)&subsystem_name=Methionine_Biosynthesis@1>O-acetylhomoserine sulfhydrylase (EC 2.5.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.690@1>fig|991.25.peg.690</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1215@1>fig|991.25.peg.1215</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1224@1>fig|991.25.peg.1224</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=Methionine_Biosynthesis@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1241@1>fig|991.25.peg.1241</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Methionine_Biosynthesis@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1747@1>fig|991.25.peg.1747</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4408@1>fig|991.25.peg.4408</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine dehydrogenase (EC 1.1.1.3)&subsystem_name=Methionine_Biosynthesis@1>Homoserine dehydrogenase (EC 1.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1217@1>fig|991.25.peg.1217</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1220@1>fig|991.25.peg.1220</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1727@1>fig|991.25.peg.1727</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter substrate-binding protein&subsystem_name=Methionine_Biosynthesis@1>Methionine ABC transporter substrate-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4969@1>fig|991.25.peg.4969</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine O-acetyltransferase (EC 2.3.1.31)&subsystem_name=Methionine_Biosynthesis@1>Homoserine O-acetyltransferase (EC 2.3.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1219@1>fig|991.25.peg.1219</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter permease protein&subsystem_name=Methionine_Biosynthesis@1>Methionine ABC transporter permease protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4970@1>fig|991.25.peg.4970</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator MetR&subsystem_name=Methionine_Biosynthesis@1>Transcriptional activator MetR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.814@1>fig|991.25.peg.814</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=991.25@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylhomocysteinase (EC 3.3.1.1)&subsystem_name=Methionine_Biosynthesis@1>Adenosylhomocysteinase (EC 3.3.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2958@1>fig|991.25.peg.2958</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=991.25@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine dehydrogenase (EC 1.1.1.3)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Homoserine dehydrogenase (EC 1.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1217@1>fig|991.25.peg.1217</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1220@1>fig|991.25.peg.1220</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1727@1>fig|991.25.peg.1727</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=991.25@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1011@1>fig|991.25.peg.1011</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=991.25@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate aminotransferase (EC 2.6.1.1)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartate aminotransferase (EC 2.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.302@1>fig|991.25.peg.302</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2659@1>fig|991.25.peg.2659</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3644@1>fig|991.25.peg.3644</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4458@1>fig|991.25.peg.4458</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4740@1>fig|991.25.peg.4740</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=991.25@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonine synthase (EC 4.2.3.1)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Threonine synthase (EC 4.2.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1729@1>fig|991.25.peg.1729</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=991.25@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine kinase (EC 2.7.1.39)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Homoserine kinase (EC 2.7.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1728@1>fig|991.25.peg.1728</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=991.25@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.281@1>fig|991.25.peg.281</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1220@1>fig|991.25.peg.1220</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1727@1>fig|991.25.peg.1727</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=991.25@1>Threonine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-allo-threonine aldolase (EC 2.1.2.1)&subsystem_name=Threonine_degradation@1>L-allo-threonine aldolase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1750@1>fig|991.25.peg.1750</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=991.25@1>Threonine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=low-specificity D-threonine aldolase&subsystem_name=Threonine_degradation@1>low-specificity D-threonine aldolase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.169@1>fig|991.25.peg.169</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=991.25@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate decarboxylase (EC 4.1.1.20)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Diaminopimelate decarboxylase (EC 4.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4666@1>fig|991.25.peg.4666</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=991.25@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1777@1>fig|991.25.peg.1777</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=991.25@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate epimerase (EC 5.1.1.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Diaminopimelate epimerase (EC 5.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4447@1>fig|991.25.peg.4447</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=991.25@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1011@1>fig|991.25.peg.1011</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=991.25@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1888@1>fig|991.25.peg.1888</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2881@1>fig|991.25.peg.2881</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=991.25@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.281@1>fig|991.25.peg.281</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1220@1>fig|991.25.peg.1220</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1727@1>fig|991.25.peg.1727</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=991.25@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1638@1>fig|991.25.peg.1638</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=991.25@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfate adenylyltransferase subunit 2 (EC 2.7.7.4)&subsystem_name=Cysteine_Biosynthesis@1>Sulfate adenylyltransferase subunit 2 (EC 2.7.7.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1230@1>fig|991.25.peg.1230</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=991.25@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfate adenylyltransferase subunit 1 (EC 2.7.7.4)&subsystem_name=Cysteine_Biosynthesis@1>Sulfate adenylyltransferase subunit 1 (EC 2.7.7.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1231@1>fig|991.25.peg.1231</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=991.25@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine synthase (EC 2.5.1.47)&subsystem_name=Cysteine_Biosynthesis@1>Cysteine synthase (EC 2.5.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1801@1>fig|991.25.peg.1801</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=991.25@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfate permease&subsystem_name=Cysteine_Biosynthesis@1>Sulfate permease</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4727@1>fig|991.25.peg.4727</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=991.25@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Cysteine_Biosynthesis@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.131@1>fig|991.25.peg.131</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=991.25@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylylsulfate kinase (EC 2.7.1.25)&subsystem_name=Cysteine_Biosynthesis@1>Adenylylsulfate kinase (EC 2.7.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3109@1>fig|991.25.peg.3109</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=991.25@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine acetyltransferase (EC 2.3.1.30)&subsystem_name=Cysteine_Biosynthesis@1>Serine acetyltransferase (EC 2.3.1.30)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1433@1>fig|991.25.peg.1433</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1800@1>fig|991.25.peg.1800</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3447@1>fig|991.25.peg.3447</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=991.25@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Cysteine_Biosynthesis@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1747@1>fig|991.25.peg.1747</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4408@1>fig|991.25.peg.4408</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=991.25@1>Lysine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Saccharopine dehydrogenase (EC 1.5.1.9)&subsystem_name=Lysine_degradation@1>Saccharopine dehydrogenase (EC 1.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.483@1>fig|991.25.peg.483</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=991.25@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate decarboxylase (EC 4.1.1.20)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Diaminopimelate decarboxylase (EC 4.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4666@1>fig|991.25.peg.4666</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=991.25@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1777@1>fig|991.25.peg.1777</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=991.25@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate epimerase (EC 5.1.1.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Diaminopimelate epimerase (EC 5.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4447@1>fig|991.25.peg.4447</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=991.25@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1011@1>fig|991.25.peg.1011</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=991.25@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1888@1>fig|991.25.peg.1888</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2881@1>fig|991.25.peg.2881</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=991.25@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.281@1>fig|991.25.peg.281</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1220@1>fig|991.25.peg.1220</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1727@1>fig|991.25.peg.1727</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=991.25@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1638@1>fig|991.25.peg.1638</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=991.25@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Isoleucine_degradation@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.753@1>fig|991.25.peg.753</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=991.25@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Isoleucine_degradation@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3078@1>fig|991.25.peg.3078</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=991.25@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Isoleucine_degradation@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2639@1>fig|991.25.peg.2639</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=991.25@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Isoleucine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3784@1>fig|991.25.peg.3784</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=991.25@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Isoleucine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3784@1>fig|991.25.peg.3784</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=991.25@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-ketoacyl-CoA thiolase (EC 2.3.1.16)&subsystem_name=Isoleucine_degradation@1>3-ketoacyl-CoA thiolase (EC 2.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3221@1>fig|991.25.peg.3221</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1075@1>fig|991.25.peg.1075</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-isopropylmalate synthase (EC 2.3.3.13)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>2-isopropylmalate synthase (EC 2.3.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3368@1>fig|991.25.peg.3368</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3078@1>fig|991.25.peg.3078</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydrogenase (EC 1.1.1.85)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>3-isopropylmalate dehydrogenase (EC 1.1.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1072@1>fig|991.25.peg.1072</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3367@1>fig|991.25.peg.3367</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=(R)-citramalate synthase (EC 2.3.1.182)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>(R)-citramalate synthase (EC 2.3.1.182)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1073@1>fig|991.25.peg.1073</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1076@1>fig|991.25.peg.1076</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonine dehydratase biosynthetic (EC 4.3.1.19)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Threonine dehydratase biosynthetic (EC 4.3.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4370@1>fig|991.25.peg.4370</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase large subunit (EC 2.2.1.6)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Acetolactate synthase large subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3365@1>fig|991.25.peg.3365</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroxy-acid dehydratase (EC 4.2.1.9)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Dihydroxy-acid dehydratase (EC 4.2.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3366@1>fig|991.25.peg.3366</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=991.25@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase small subunit (EC 2.2.1.6)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Acetolactate synthase small subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3364@1>fig|991.25.peg.3364</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=991.25@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Valine_degradation@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.753@1>fig|991.25.peg.753</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=991.25@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Valine_degradation@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3078@1>fig|991.25.peg.3078</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=991.25@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Valine_degradation@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2639@1>fig|991.25.peg.2639</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=991.25@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Valine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3784@1>fig|991.25.peg.3784</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=991.25@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Valine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3784@1>fig|991.25.peg.3784</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=991.25@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)&subsystem_name=Leucine_Biosynthesis@1>3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1075@1>fig|991.25.peg.1075</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=991.25@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)&subsystem_name=Leucine_Biosynthesis@1>3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1076@1>fig|991.25.peg.1076</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=991.25@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-isopropylmalate synthase (EC 2.3.3.13)&subsystem_name=Leucine_Biosynthesis@1>2-isopropylmalate synthase (EC 2.3.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3368@1>fig|991.25.peg.3368</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=991.25@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Leucine_Biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3078@1>fig|991.25.peg.3078</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=991.25@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydrogenase (EC 1.1.1.85)&subsystem_name=Leucine_Biosynthesis@1>3-isopropylmalate dehydrogenase (EC 1.1.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1072@1>fig|991.25.peg.1072</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3367@1>fig|991.25.peg.3367</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=991.25@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Shikimate kinase I (EC 2.7.1.71)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Shikimate kinase I (EC 2.7.1.71)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2461@1>fig|991.25.peg.2461</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=991.25@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4523@1>fig|991.25.peg.4523</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=991.25@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate dehydratase II (EC 4.2.1.10)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>3-dehydroquinate dehydratase II (EC 4.2.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4769@1>fig|991.25.peg.4769</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=991.25@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I beta (EC 2.5.1.54)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I beta (EC 2.5.1.54)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.304@1>fig|991.25.peg.304</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=991.25@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate synthase (EC 4.2.3.4)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>3-dehydroquinate synthase (EC 4.2.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1547@1>fig|991.25.peg.1547</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4090@1>fig|991.25.peg.4090</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=991.25@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chorismate synthase (EC 4.2.3.5)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Chorismate synthase (EC 4.2.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2188@1>fig|991.25.peg.2188</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3334@1>fig|991.25.peg.3334</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, amidotransferase component (EC 4.1.3.27)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate synthase, amidotransferase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.431@1>fig|991.25.peg.431</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.710@1>fig|991.25.peg.710</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.423@1>fig|991.25.peg.423</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.429@1>fig|991.25.peg.429</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.426@1>fig|991.25.peg.426</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Indole-3-glycerol phosphate synthase (EC 4.1.1.48)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Indole-3-glycerol phosphate synthase (EC 4.1.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.428@1>fig|991.25.peg.428</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, aminase component (EC 4.1.3.27)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate synthase, aminase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.433@1>fig|991.25.peg.433</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.427@1>fig|991.25.peg.427</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isochorismate synthase (EC 5.4.4.2)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Isochorismate synthase (EC 5.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2017@1>fig|991.25.peg.2017</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.696@1>fig|991.25.peg.696</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isochorismatase (EC 3.3.2.1)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Isochorismatase (EC 3.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4049@1>fig|991.25.peg.4049</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=991.25@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.431@1>fig|991.25.peg.431</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=991.25@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chorismate mutase I (EC 5.4.99.5)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Chorismate mutase I (EC 5.4.99.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.304@1>fig|991.25.peg.304</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=991.25@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prephenate dehydratase (EC 4.2.1.51)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Prephenate dehydratase (EC 4.2.1.51)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.301@1>fig|991.25.peg.301</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=991.25@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prephenate and/or arogenate dehydrogenase (unknown specificity) (EC 1.3.1.12)(EC 1.3.1.43)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Prephenate and/or arogenate dehydrogenase (unknown specificity) (EC 1.3.1.12)(EC 1.3.1.43)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.303@1>fig|991.25.peg.303</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=991.25@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biosynthetic Aromatic amino acid aminotransferase alpha (EC 2.6.1.57)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Biosynthetic Aromatic amino acid aminotransferase alpha (EC 2.6.1.57)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.302@1>fig|991.25.peg.302</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, amidotransferase component (EC 4.1.3.27)&subsystem_name=Tryptophan_synthesis@1>Anthranilate synthase, amidotransferase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.431@1>fig|991.25.peg.431</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Tryptophan_synthesis@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.710@1>fig|991.25.peg.710</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Tryptophan_synthesis@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.423@1>fig|991.25.peg.423</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Tryptophan_synthesis@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.696@1>fig|991.25.peg.696</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Tryptophan_synthesis@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.429@1>fig|991.25.peg.429</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Tryptophan_synthesis@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.426@1>fig|991.25.peg.426</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Indole-3-glycerol phosphate synthase (EC 4.1.1.48)&subsystem_name=Tryptophan_synthesis@1>Indole-3-glycerol phosphate synthase (EC 4.1.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.428@1>fig|991.25.peg.428</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Tryptophan_synthesis@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.431@1>fig|991.25.peg.431</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, aminase component (EC 4.1.3.27)&subsystem_name=Tryptophan_synthesis@1>Anthranilate synthase, aminase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.433@1>fig|991.25.peg.433</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=991.25@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Tryptophan_synthesis@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.427@1>fig|991.25.peg.427</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=991.25@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan 2,3-dioxygenase (EC 1.13.11.11)&subsystem_name=Aromatic_amino_acid_degradation@1>Tryptophan 2,3-dioxygenase (EC 1.13.11.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.109@1>fig|991.25.peg.109</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=991.25@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)&subsystem_name=Aromatic_amino_acid_degradation@1>Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1244@1>fig|991.25.peg.1244</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=991.25@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)&subsystem_name=Aromatic_amino_acid_degradation@1>4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.111@1>fig|991.25.peg.111</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=991.25@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aromatic amino acid transport protein AroP&subsystem_name=Aromatic_amino_acid_degradation@1>Aromatic amino acid transport protein AroP</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3037@1>fig|991.25.peg.3037</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=991.25@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanine-4-hydroxylase (EC 1.14.16.1)&subsystem_name=Aromatic_amino_acid_degradation@1>Phenylalanine-4-hydroxylase (EC 1.14.16.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4368@1>fig|991.25.peg.4368</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=991.25@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyrroline-5-carboxylate reductase (EC 1.5.1.2)&subsystem_name=Proline_Synthesis@1>Pyrroline-5-carboxylate reductase (EC 1.5.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.13@1>fig|991.25.peg.13</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=991.25@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Gamma-glutamyl phosphate reductase (EC 1.2.1.41)&subsystem_name=Proline_Synthesis@1>Gamma-glutamyl phosphate reductase (EC 1.2.1.41)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1602@1>fig|991.25.peg.1602</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=991.25@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Proline_Synthesis@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.133@1>fig|991.25.peg.133</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=991.25@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate 5-kinase (EC 2.7.2.11)&subsystem_name=Proline_Synthesis@1>Glutamate 5-kinase (EC 2.7.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1601@1>fig|991.25.peg.1601</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=991.25@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Proline iminopeptidase (EC 3.4.11.5)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>Proline iminopeptidase (EC 3.4.11.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.151@1>fig|991.25.peg.151</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2010@1>fig|991.25.peg.2010</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4050@1>fig|991.25.peg.4050</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=991.25@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyproline epimerase (EC 5.1.1.8)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>4-hydroxyproline epimerase (EC 5.1.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2293@1>fig|991.25.peg.2293</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=991.25@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-pyrroline-4-hydroxy-2-carboxylate deaminase (EC 3.5.4.22)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>1-pyrroline-4-hydroxy-2-carboxylate deaminase (EC 3.5.4.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2291@1>fig|991.25.peg.2291</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=991.25@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-amino-acid oxidase (EC 1.4.3.3)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>D-amino-acid oxidase (EC 1.4.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3779@1>fig|991.25.peg.3779</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=991.25@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3633@1>fig|991.25.peg.3633</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=991.25@1>Glycine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)&subsystem_name=Glycine_Biosynthesis@1>2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.145@1>fig|991.25.peg.145</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4156@1>fig|991.25.peg.4156</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=991.25@1>Glycine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Glycine_Biosynthesis@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4558@1>fig|991.25.peg.4558</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=991.25@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Alanine_biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3078@1>fig|991.25.peg.3078</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=991.25@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Alanine_biosynthesis@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1041@1>fig|991.25.peg.1041</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=991.25@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=Alanine_biosynthesis@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.619@1>fig|991.25.peg.619</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1795@1>fig|991.25.peg.1795</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2495@1>fig|991.25.peg.2495</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3276@1>fig|991.25.peg.3276</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4068@1>fig|991.25.peg.4068</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=991.25@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine racemase (EC 5.1.1.1)&subsystem_name=Alanine_biosynthesis@1>Alanine racemase (EC 5.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1013@1>fig|991.25.peg.1013</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=991.25@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Serine_Biosynthesis@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.15@1>fig|991.25.peg.15</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1096@1>fig|991.25.peg.1096</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4196@1>fig|991.25.peg.4196</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=991.25@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine phosphatase (EC 3.1.3.3)&subsystem_name=Serine_Biosynthesis@1>Phosphoserine phosphatase (EC 3.1.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3689@1>fig|991.25.peg.3689</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3689@1>fig|991.25.peg.3689</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=991.25@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Serine_Biosynthesis@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4558@1>fig|991.25.peg.4558</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=991.25@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Serine_Biosynthesis@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4197@1>fig|991.25.peg.4197</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=991.25@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine cleavage system H protein&subsystem_name=Glycine_cleavage_system@1>Glycine cleavage system H protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2360@1>fig|991.25.peg.2360</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3745@1>fig|991.25.peg.3745</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=991.25@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)&subsystem_name=Glycine_cleavage_system@1>Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1322@1>fig|991.25.peg.1322</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=991.25@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=Glycine_cleavage_system@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.693@1>fig|991.25.peg.693</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2087@1>fig|991.25.peg.2087</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=991.25@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)&subsystem_name=Glycine_cleavage_system@1>Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1744@1>fig|991.25.peg.1744</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Glycine_and_Serine_Utilization@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.15@1>fig|991.25.peg.15</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1096@1>fig|991.25.peg.1096</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4196@1>fig|991.25.peg.4196</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, alpha subunit (EC 4.3.1.17)&subsystem_name=Glycine_and_Serine_Utilization@1>L-serine dehydratase, alpha subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2034@1>fig|991.25.peg.2034</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine phosphatase (EC 3.1.3.3)&subsystem_name=Glycine_and_Serine_Utilization@1>Phosphoserine phosphatase (EC 3.1.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3689@1>fig|991.25.peg.3689</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, beta subunit (EC 4.3.1.17)&subsystem_name=Glycine_and_Serine_Utilization@1>L-serine dehydratase, beta subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2034@1>fig|991.25.peg.2034</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Glycine_and_Serine_Utilization@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.131@1>fig|991.25.peg.131</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Glycine_and_Serine_Utilization@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4558@1>fig|991.25.peg.4558</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Seryl-tRNA synthetase (EC 6.1.1.11)&subsystem_name=Glycine_and_Serine_Utilization@1>Seryl-tRNA synthetase (EC 6.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.8@1>fig|991.25.peg.8</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Glycine_and_Serine_Utilization@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4197@1>fig|991.25.peg.4197</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine cleavage system H protein&subsystem_name=Glycine_and_Serine_Utilization@1>Glycine cleavage system H protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2360@1>fig|991.25.peg.2360</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3745@1>fig|991.25.peg.3745</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)&subsystem_name=Glycine_and_Serine_Utilization@1>Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1322@1>fig|991.25.peg.1322</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)&subsystem_name=Glycine_and_Serine_Utilization@1>2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.145@1>fig|991.25.peg.145</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4156@1>fig|991.25.peg.4156</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)&subsystem_name=Glycine_and_Serine_Utilization@1>Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1744@1>fig|991.25.peg.1744</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=991.25@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Glycine_and_Serine_Utilization@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1747@1>fig|991.25.peg.1747</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4408@1>fig|991.25.peg.4408</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=991.25@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiol peroxidase, Tpx-type (EC 1.11.1.15)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thiol peroxidase, Tpx-type (EC 1.11.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.528@1>fig|991.25.peg.528</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=991.25@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiol peroxidase, Bcp-type (EC 1.11.1.15)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thiol peroxidase, Bcp-type (EC 1.11.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1506@1>fig|991.25.peg.1506</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4171@1>fig|991.25.peg.4171</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=991.25@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Thioredoxin-disulfide_reductase@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.525@1>fig|991.25.peg.525</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3114@1>fig|991.25.peg.3114</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=991.25@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thioredoxin reductase (EC 1.8.1.9)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thioredoxin reductase (EC 1.8.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.800@1>fig|991.25.peg.800</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1235@1>fig|991.25.peg.1235</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1971@1>fig|991.25.peg.1971</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4064@1>fig|991.25.peg.4064</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4065@1>fig|991.25.peg.4065</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=991.25@1>Galactosylceramide and Sulfatide metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-galactosidase (EC 3.2.1.23)&subsystem_name=Galactosylceramide_and_Sulfatide_metabolism@1>Beta-galactosidase (EC 3.2.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3048@1>fig|991.25.peg.3048</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=991.25@1>Galactosylceramide and Sulfatide metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=probable beta-D-galactosidase&subsystem_name=Galactosylceramide_and_Sulfatide_metabolism@1>probable beta-D-galactosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3240@1>fig|991.25.peg.3240</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=991.25@1>Galactosylceramide and Sulfatide metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Arylsulfatase (EC 3.1.6.1)&subsystem_name=Galactosylceramide_and_Sulfatide_metabolism@1>Arylsulfatase (EC 3.1.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.823@1>fig|991.25.peg.823</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1555@1>fig|991.25.peg.1555</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1571@1>fig|991.25.peg.1571</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2103@1>fig|991.25.peg.2103</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2108@1>fig|991.25.peg.2108</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3461@1>fig|991.25.peg.3461</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4725@1>fig|991.25.peg.4725</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4779@1>fig|991.25.peg.4779</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4829@1>fig|991.25.peg.4829</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=991.25@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkaline phosphatase (EC 3.1.3.1)&subsystem_name=Phosphate_metabolism@1>Alkaline phosphatase (EC 3.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.349@1>fig|991.25.peg.349</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1689@1>fig|991.25.peg.1689</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3641@1>fig|991.25.peg.3641</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=991.25@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inorganic pyrophosphatase (EC 3.6.1.1)&subsystem_name=Phosphate_metabolism@1>Inorganic pyrophosphatase (EC 3.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1202@1>fig|991.25.peg.1202</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=991.25@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyrophosphate-energized proton pump (EC 3.6.1.1)&subsystem_name=Phosphate_metabolism@1>Pyrophosphate-energized proton pump (EC 3.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1203@1>fig|991.25.peg.1203</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=991.25@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Phosphate_metabolism@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3755@1>fig|991.25.peg.3755</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=991.25@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exopolyphosphatase (EC 3.6.1.11)&subsystem_name=Phosphate_metabolism@1>Exopolyphosphatase (EC 3.6.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3051@1>fig|991.25.peg.3051</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3448@1>fig|991.25.peg.3448</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3756@1>fig|991.25.peg.3756</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=991.25@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate starvation-inducible protein PhoH, predicted ATPase&subsystem_name=Phosphate_metabolism@1>Phosphate starvation-inducible protein PhoH, predicted ATPase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4947@1>fig|991.25.peg.4947</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4947@1>fig|991.25.peg.4947</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=991.25@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Probable low-affinity inorganic phosphate transporter&subsystem_name=Phosphate_metabolism@1>Probable low-affinity inorganic phosphate transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1721@1>fig|991.25.peg.1721</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=991.25@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate transport regulator (distant homolog of PhoU)&subsystem_name=Phosphate_metabolism@1>Phosphate transport regulator (distant homolog of PhoU)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1722@1>fig|991.25.peg.1722</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=991.25@1>Polyphosphate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Polyphosphate@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3755@1>fig|991.25.peg.3755</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=991.25@1>Polyphosphate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exopolyphosphatase (EC 3.6.1.11)&subsystem_name=Polyphosphate@1>Exopolyphosphatase (EC 3.6.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3051@1>fig|991.25.peg.3051</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3448@1>fig|991.25.peg.3448</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3756@1>fig|991.25.peg.3756</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=991.25@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylglyoxal synthase (EC 4.2.3.3)&subsystem_name=Methylglyoxal_Metabolism@1>Methylglyoxal synthase (EC 4.2.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2174@1>fig|991.25.peg.2174</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2175@1>fig|991.25.peg.2175</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=991.25@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lactoylglutathione lyase (EC 4.4.1.5)&subsystem_name=Methylglyoxal_Metabolism@1>Lactoylglutathione lyase (EC 4.4.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4901@1>fig|991.25.peg.4901</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=991.25@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase B (EC 1.2.1.22)&subsystem_name=Methylglyoxal_Metabolism@1>Aldehyde dehydrogenase B (EC 1.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.891@1>fig|991.25.peg.891</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=991.25@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase (EC 1.2.1.3)&subsystem_name=Methylglyoxal_Metabolism@1>Aldehyde dehydrogenase (EC 1.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2022@1>fig|991.25.peg.2022</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4502@1>fig|991.25.peg.4502</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2022@1>fig|991.25.peg.2022</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4502@1>fig|991.25.peg.4502</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=991.25@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxyacylglutathione hydrolase (EC 3.1.2.6)&subsystem_name=Methylglyoxal_Metabolism@1>Hydroxyacylglutathione hydrolase (EC 3.1.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.640@1>fig|991.25.peg.640</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=991.25@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase (EC 1.2.1.3)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Aldehyde dehydrogenase (EC 1.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2022@1>fig|991.25.peg.2022</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4502@1>fig|991.25.peg.4502</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=991.25@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1199@1>fig|991.25.peg.1199</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=991.25@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2993@1>fig|991.25.peg.2993</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=991.25@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate acetyltransferase (EC 2.3.1.8)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Phosphate acetyltransferase (EC 2.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4554@1>fig|991.25.peg.4554</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=991.25@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetate kinase (EC 2.7.2.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Acetate kinase (EC 2.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4553@1>fig|991.25.peg.4553</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=991.25@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2992@1>fig|991.25.peg.2992</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=991.25@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate decarboxylase (EC 4.1.1.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate decarboxylase (EC 4.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3019@1>fig|991.25.peg.3019</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=991.25@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-dependent protein deacetylase of SIR2 family&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>NAD-dependent protein deacetylase of SIR2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3054@1>fig|991.25.peg.3054</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3110@1>fig|991.25.peg.3110</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=991.25@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1105@1>fig|991.25.peg.1105</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=991.25@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoenolpyruvate carboxylase (EC 4.1.1.31)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Phosphoenolpyruvate carboxylase (EC 4.1.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4083@1>fig|991.25.peg.4083</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=991.25@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-dependent malic enzyme (EC 1.1.1.40)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>NADP-dependent malic enzyme (EC 1.1.1.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3748@1>fig|991.25.peg.3748</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=991.25@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate kinase (EC 2.7.1.40)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Pyruvate kinase (EC 2.7.1.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3897@1>fig|991.25.peg.3897</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=991.25@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, alpha subunit (EC 4.3.1.17)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>L-serine dehydratase, alpha subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2034@1>fig|991.25.peg.2034</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=991.25@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3078@1>fig|991.25.peg.3078</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=991.25@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, beta subunit (EC 4.3.1.17)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>L-serine dehydratase, beta subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2034@1>fig|991.25.peg.2034</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=991.25@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine dehydrogenase (EC 1.4.1.1)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Alanine dehydrogenase (EC 1.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3301@1>fig|991.25.peg.3301</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4443@1>fig|991.25.peg.4443</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=991.25@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine racemase (EC 5.1.1.1)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Alanine racemase (EC 5.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1013@1>fig|991.25.peg.1013</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=991.25@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transketolase, N-terminal section (EC 2.2.1.1)&subsystem_name=Pentose_phosphate_pathway@1>Transketolase, N-terminal section (EC 2.2.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4797@1>fig|991.25.peg.4797</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=991.25@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)&subsystem_name=Pentose_phosphate_pathway@1>Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.87@1>fig|991.25.peg.87</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3913@1>fig|991.25.peg.3913</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=991.25@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transketolase, C-terminal section (EC 2.2.1.1)&subsystem_name=Pentose_phosphate_pathway@1>Transketolase, C-terminal section (EC 2.2.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4796@1>fig|991.25.peg.4796</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=991.25@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribulose-phosphate 3-epimerase (EC 5.1.3.1)&subsystem_name=Pentose_phosphate_pathway@1>Ribulose-phosphate 3-epimerase (EC 5.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3524@1>fig|991.25.peg.3524</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=991.25@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-6-phosphate 1-dehydrogenase (EC 1.1.1.49)&subsystem_name=Pentose_phosphate_pathway@1>Glucose-6-phosphate 1-dehydrogenase (EC 1.1.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1557@1>fig|991.25.peg.1557</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=991.25@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6-phosphogluconate dehydrogenase, decarboxylating (EC 1.1.1.44)&subsystem_name=Pentose_phosphate_pathway@1>6-phosphogluconate dehydrogenase, decarboxylating (EC 1.1.1.44)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1556@1>fig|991.25.peg.1556</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=991.25@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6-phosphogluconolactonase (EC 3.1.1.31), eukaryotic type&subsystem_name=Pentose_phosphate_pathway@1>6-phosphogluconolactonase (EC 3.1.1.31), eukaryotic type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1558@1>fig|991.25.peg.1558</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=991.25@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transaldolase (EC 2.2.1.2)&subsystem_name=Pentose_phosphate_pathway@1>Transaldolase (EC 2.2.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2198@1>fig|991.25.peg.2198</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=991.25@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malate dehydrogenase (EC 1.1.1.37)&subsystem_name=Glyoxylate_bypass@1>Malate dehydrogenase (EC 1.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2562@1>fig|991.25.peg.2562</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=991.25@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Citrate synthase (si) (EC 2.3.3.1)&subsystem_name=Glyoxylate_bypass@1>Citrate synthase (si) (EC 2.3.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1411@1>fig|991.25.peg.1411</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=991.25@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aconitate hydratase 2 (EC 4.2.1.3)&subsystem_name=Glyoxylate_bypass@1>Aconitate hydratase 2 (EC 4.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4434@1>fig|991.25.peg.4434</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=991.25@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isocitrate lyase (EC 4.1.3.1)&subsystem_name=Glyoxylate_bypass@1>Isocitrate lyase (EC 4.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3655@1>fig|991.25.peg.3655</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=991.25@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malate synthase (EC 2.3.3.9)&subsystem_name=Glyoxylate_bypass@1>Malate synthase (EC 2.3.3.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3656@1>fig|991.25.peg.3656</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=991.25@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aconitate hydratase (EC 4.2.1.3)&subsystem_name=Glyoxylate_bypass@1>Aconitate hydratase (EC 4.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4435@1>fig|991.25.peg.4435</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=991.25@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.963@1>fig|991.25.peg.963</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=991.25@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2639@1>fig|991.25.peg.2639</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=991.25@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Dehydrogenase_complexes@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3784@1>fig|991.25.peg.3784</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=991.25@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Dehydrogenase_complexes@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3784@1>fig|991.25.peg.3784</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=991.25@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)&subsystem_name=Dehydrogenase_complexes@1>Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1199@1>fig|991.25.peg.1199</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=991.25@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2993@1>fig|991.25.peg.2993</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=991.25@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)&subsystem_name=Dehydrogenase_complexes@1>2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.962@1>fig|991.25.peg.962</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=991.25@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)&subsystem_name=Dehydrogenase_complexes@1>Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2992@1>fig|991.25.peg.2992</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=991.25@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylglucosamine related transporter, NagX&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>N-acetylglucosamine related transporter, NagX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2624@1>fig|991.25.peg.2624</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2645@1>fig|991.25.peg.2645</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3555@1>fig|991.25.peg.3555</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=991.25@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucosamine-6-phosphate deaminase (EC 3.5.99.6)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Glucosamine-6-phosphate deaminase (EC 3.5.99.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3546@1>fig|991.25.peg.3546</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4106@1>fig|991.25.peg.4106</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4210@1>fig|991.25.peg.4210</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4771@1>fig|991.25.peg.4771</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5010@1>fig|991.25.peg.5010</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=991.25@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetyl glucosamine transporter, NagP&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>N-acetyl glucosamine transporter, NagP</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4211@1>fig|991.25.peg.4211</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=991.25@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chitinase (EC 3.2.1.14)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Chitinase (EC 3.2.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3090@1>fig|991.25.peg.3090</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.5012@1>fig|991.25.peg.5012</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=991.25@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted N-acetyl-glucosamine kinase 2, ROK family (EC 2.7.1.59)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Predicted N-acetyl-glucosamine kinase 2, ROK family (EC 2.7.1.59)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2785@1>fig|991.25.peg.2785</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=991.25@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylglucosamine-6-phosphate deacetylase (EC 3.5.1.25)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>N-acetylglucosamine-6-phosphate deacetylase (EC 3.5.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3177@1>fig|991.25.peg.3177</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=991.25@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1420@1>fig|991.25.peg.1420</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1445@1>fig|991.25.peg.1445</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3667@1>fig|991.25.peg.3667</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=991.25@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-galactosidase (EC 3.2.1.23)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Beta-galactosidase (EC 3.2.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3048@1>fig|991.25.peg.3048</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=991.25@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Galactose-1-phosphate uridylyltransferase (EC 2.7.7.10)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Galactose-1-phosphate uridylyltransferase (EC 2.7.7.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3275@1>fig|991.25.peg.3275</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=991.25@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Galactokinase (EC 2.7.1.6)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Galactokinase (EC 2.7.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3274@1>fig|991.25.peg.3274</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=991.25@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldose 1-epimerase (EC 5.1.3.3)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Aldose 1-epimerase (EC 5.1.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3893@1>fig|991.25.peg.3893</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_utilization&organism=991.25@1>Lactose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-galactosidase (EC 3.2.1.23)&subsystem_name=Lactose_utilization@1>Beta-galactosidase (EC 3.2.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3048@1>fig|991.25.peg.3048</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_utilization&organism=991.25@1>Lactose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Galactoside O-acetyltransferase (EC 2.3.1.18)&subsystem_name=Lactose_utilization@1>Galactoside O-acetyltransferase (EC 2.3.1.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.466@1>fig|991.25.peg.466</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=991.25@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formyltetrahydrofolate deformylase (EC 3.5.1.10)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Formyltetrahydrofolate deformylase (EC 3.5.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4990@1>fig|991.25.peg.4990</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=991.25@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.118@1>fig|991.25.peg.118</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=991.25@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1538@1>fig|991.25.peg.1538</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=991.25@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1241@1>fig|991.25.peg.1241</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=991.25@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1538@1>fig|991.25.peg.1538</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=991.25@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alcohol dehydrogenase (EC 1.1.1.1)&subsystem_name=Butanol_Biosynthesis@1>Alcohol dehydrogenase (EC 1.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2355@1>fig|991.25.peg.2355</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4503@1>fig|991.25.peg.4503</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=991.25@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-dependent butanol dehydrogenase A (EC 1.1.1.-)&subsystem_name=Butanol_Biosynthesis@1>NADH-dependent butanol dehydrogenase A (EC 1.1.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.83@1>fig|991.25.peg.83</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=991.25@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Butanol_Biosynthesis@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.753@1>fig|991.25.peg.753</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=991.25@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)&subsystem_name=Butanol_Biosynthesis@1>3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1620@1>fig|991.25.peg.1620</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=991.25@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Butanol_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2143@1>fig|991.25.peg.2143</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetolactate_synthase_subunits&organism=991.25@1>Acetolactate synthase subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase large subunit (EC 2.2.1.6)&subsystem_name=Acetolactate_synthase_subunits@1>Acetolactate synthase large subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3365@1>fig|991.25.peg.3365</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetolactate_synthase_subunits&organism=991.25@1>Acetolactate synthase subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase small subunit (EC 2.2.1.6)&subsystem_name=Acetolactate_synthase_subunits@1>Acetolactate synthase small subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3364@1>fig|991.25.peg.3364</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=991.25@1>Fermentations: Lactate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-lactate dehydrogenase (EC 1.1.1.28)&subsystem_name=Fermentations:_Lactate@1>D-lactate dehydrogenase (EC 1.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4387@1>fig|991.25.peg.4387</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=991.25@1>Fermentations: Lactate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate acetyltransferase (EC 2.3.1.8)&subsystem_name=Fermentations:_Lactate@1>Phosphate acetyltransferase (EC 2.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4554@1>fig|991.25.peg.4554</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=991.25@1>Fermentations: Lactate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetate kinase (EC 2.7.2.1)&subsystem_name=Fermentations:_Lactate@1>Acetate kinase (EC 2.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4553@1>fig|991.25.peg.4553</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=991.25@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.753@1>fig|991.25.peg.753</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=991.25@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Electron transfer flavoprotein, beta subunit&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Electron transfer flavoprotein, beta subunit</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1198@1>fig|991.25.peg.1198</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=991.25@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Electron transfer flavoprotein, alpha subunit&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Electron transfer flavoprotein, alpha subunit</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1197@1>fig|991.25.peg.1197</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=991.25@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1620@1>fig|991.25.peg.1620</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=991.25@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2143@1>fig|991.25.peg.2143</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetoin,_butanediol_metabolism&organism=991.25@1>Acetoin, butanediol metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-acetolactate decarboxylase (EC 4.1.1.5)&subsystem_name=Acetoin,_butanediol_metabolism@1>Alpha-acetolactate decarboxylase (EC 4.1.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4845@1>fig|991.25.peg.4845</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetoin,_butanediol_metabolism&organism=991.25@1>Acetoin, butanediol metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase large subunit (EC 2.2.1.6)&subsystem_name=Acetoin,_butanediol_metabolism@1>Acetolactate synthase large subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3365@1>fig|991.25.peg.3365</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetoin,_butanediol_metabolism&organism=991.25@1>Acetoin, butanediol metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase small subunit (EC 2.2.1.6)&subsystem_name=Acetoin,_butanediol_metabolism@1>Acetolactate synthase small subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3364@1>fig|991.25.peg.3364</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=991.25@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1140@1>fig|991.25.peg.1140</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4386@1>fig|991.25.peg.4386</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4546@1>fig|991.25.peg.4546</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=991.25@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3489@1>fig|991.25.peg.3489</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=991.25@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate regulon repressor GlpR&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol-3-phosphate regulon repressor GlpR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3488@1>fig|991.25.peg.3488</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=991.25@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase [NAD(P)+] (EC 1.1.1.94)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol-3-phosphate dehydrogenase [NAD(P)+] (EC 1.1.1.94)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.82@1>fig|991.25.peg.82</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=991.25@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol kinase (EC 2.7.1.30)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol kinase (EC 2.7.1.30)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2878@1>fig|991.25.peg.2878</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3490@1>fig|991.25.peg.3490</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=991.25@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol uptake facilitator protein&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol uptake facilitator protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3491@1>fig|991.25.peg.3491</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=991.25@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-1,2-mannosidase&subsystem_name=Mannose_Metabolism@1>Alpha-1,2-mannosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2782@1>fig|991.25.peg.2782</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2783@1>fig|991.25.peg.2783</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4242@1>fig|991.25.peg.4242</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=991.25@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomannomutase (EC 5.4.2.8)&subsystem_name=Mannose_Metabolism@1>Phosphomannomutase (EC 5.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3792@1>fig|991.25.peg.3792</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=991.25@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mannose-6-phosphate isomerase (EC 5.3.1.8)&subsystem_name=Mannose_Metabolism@1>Mannose-6-phosphate isomerase (EC 5.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1507@1>fig|991.25.peg.1507</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2777@1>fig|991.25.peg.2777</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=991.25@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-mannosidase (EC 3.2.1.25)&subsystem_name=Mannose_Metabolism@1>Beta-mannosidase (EC 3.2.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2781@1>fig|991.25.peg.2781</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=D-ribose_utilization&organism=991.25@1>D-ribose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribokinase (EC 2.7.1.15)&subsystem_name=D-ribose_utilization@1>Ribokinase (EC 2.7.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3907@1>fig|991.25.peg.3907</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=991.25@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.1862@1>fig|991.25.peg.1862</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.2855@1>fig|991.25.peg.2855</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=991.25@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribokinase (EC 2.7.1.15)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Ribokinase (EC 2.7.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3907@1>fig|991.25.peg.3907</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=991.25@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative deoxyribonuclease YjjV&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Putative deoxyribonuclease YjjV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.4012@1>fig|991.25.peg.4012</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=991.25@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribose-phosphate aldolase (EC 4.1.2.4)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Deoxyribose-phosphate aldolase (EC 4.1.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|991.25.peg.3743@1>fig|991.25.peg.3743</a>'>
<input type='hidden' id='table_onclicks_0' value='@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^'>
<input type='hidden' id='table_highlights_0' value='@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^'>
<input type='hidden' id='table_filtereddata_0' value=''>
<input type='hidden' id='table_rows_0' value='950'>
<input type='hidden' id='table_cols_0' value='5'>
<input type='hidden' id='table_start_0' value='0'>
<input type='hidden' id='table_sortdirection_0' value='up'>
<input type='hidden' id='table_column_types_0' value='@~@~@~@~'>
<input type='hidden' id='table_hoverid_0' value='1'>
<span id='table_input_space_0'></span><input type='hidden' id='ajax_url' value='https://rast.nmpdr.org/ajax.cgi'>
<input type='hidden' id='ajax_params' value='page=Organism&app=SeedViewer&url=seedviewer.cgi&cookies=0' />
<div style='display:none;' id='table_0_download'></div><table id='table_0' class='table_table' style=''><tr><td name='0_col_1' id='0_col_1' class='table_first_row' style=''><a href='javascript: table_sort("0", "1");' class='table_first_row' title='click to sort'><span id='colname_0_col_1'>Category</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_1_operator' value='equal' id='table_0_operator_1'><select name='0_col_1' class='filter_item' id='table_0_operand_1' onchange='return check_submit_filter2("0", this.options[this.selectedIndex].text);' style='width: 100%;' title='Select Filter'></select></td><td name='0_col_2' id='0_col_2' class='table_first_row' style=''><a href='javascript: table_sort("0", "2");' class='table_first_row' title='click to sort'><span id='colname_0_col_2'>Subcategory</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_2_operator' value='equal' id='table_0_operator_2'><select name='0_col_2' class='filter_item' id='table_0_operand_2' onchange='return check_submit_filter2("0", this.options[this.selectedIndex].text);' style='width: 100%;' title='Select Filter'></select></td><td name='0_col_3' id='0_col_3' class='table_first_row' style=''><a href='javascript: table_sort("0", "3");' class='table_first_row' title='click to sort'><span id='colname_0_col_3'>Subsystem</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_3_operator' value='like' id='table_0_operator_3'><input type='text' name='0_col_3' class='filter_item' value='' size=5 id='table_0_operand_3' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td><td name='0_col_4' id='0_col_4' class='table_first_row' style=''><a href='javascript: table_sort("0", "4");' class='table_first_row' title='click to sort'><span id='colname_0_col_4'>Role</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_4_operator' value='like' id='table_0_operator_4'><input type='text' name='0_col_4' class='filter_item' value='' size=5 id='table_0_operand_4' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td><td name='0_col_5' id='0_col_5' class='table_first_row' style=''><span id='colname_0_col_5'>Features</span><br><input type=hidden name='0_col_5_operator' value='like' id='table_0_operator_5'><input type='text' name='0_col_5' class='filter_item' value='' size=5 id='table_0_operand_5' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td></table><input type='hidden' id='table_visible_columns_0' value='1@~1@~1@~1@~1'>
<input type='hidden' id='table_combo_columns_0' value='1@~2'>
<input type='hidden' id='table_sc_0' value=''><img src='https://rast.nmpdr.org/Html/clear.gif' onload='initialize_table("0")'><input type='hidden' id='hover_redundancies_1' value=''></td></tr><tr><td style='width: 100%; text-align: center;'><table style='width: 100%;'><tr><td align='left' width='20%'><a href='javascript: table_first("0");' name='table_first_0'>&laquo;first</a>&nbsp;&nbsp;<a href='javascript: table_prev("0");' name='table_prev_0'>&laquo;prev</a></td><td align='center' width='60%'>displaying <span id='table_start_bottom_0'>1</span> - <span id='table_stop_bottom_0'>950</span> of <span id='table_total_bottom_0'>950</span></td><td align='right' width='20%'><a href='javascript: table_next("0");' name='table_next_0'>next&raquo;</a>&nbsp;&nbsp;<a href='javascript: table_last("0");' name='table_last_0'>last&raquo;</a></td></tr></table></td></tr></table></div></td></tr></table>
<img src='https://rast.nmpdr.org/Html/clear.gif'  onload='initialize_tab_view("1"); tab_view_select("1", "0");'>
<br><br><!-- tracking off -->
  </div>
    
</body>


</html>

