<!doctype HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">    
    <title>Seed Viewer - Organism</title>

    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/TabView.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/Hover.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/Table.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/web_app_default.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/commonviewer.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/seedviewer.css" >
       

    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/PopupTooltip.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/TabView.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Ajax.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Hover.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Tree.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Table.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/fade.js" ></script>
    
    <meta name="HTML_PATH" content="./Html" />
    
    <META HTTP-EQUIV="PRAGMA" CONTENT="NO-CACHE">
    
    <META HTTP-EQUIV="CACHE-CONTROL" CONTENT="NO-CACHE">
    
    <meta name="robots" content="noindex,nofollow" />
    
    

  </head>

  <body>
<div id="header">
  <span id="logo">
  <img height="95px" src="./Html/seed-logo-green.png" alt="The SEED" />
  </span>
  <p>
  <span id="title">
  <img src="./Html/seed-theseed-viewer.png" alt="The SEED Viewer" />
  </span>
  <span id="version">SEED Viewer version 2.0</span>
  </p>
  <p>Welcome to the SEED Viewer - a read-only browser of the curated SEED data. <br/>
  For more information about The SEED please visit <a href="http://www.theseed.org">theSEED.org.</a>

  </p>




</div>

<div id="topbar">
  <table style="width: 100%; border-spacing: 0px;">
    <tr>
      <td style='width: 100%; padding: 0px;'>
	
	  <div id='menu'>
	<ul id='nav'>
		<li><div><a href="?page=Home" >&raquo;Navigate</a></div>
		<ul>
			<li><a href="?page=Home" >Startpage</a></li>
			<li><a href="?page=OrganismSelect" >Organisms</a></li>
			<li><a href="?page=SubsystemSelect" >Browse Subsystems</a></li>
			<li><a href="https://rast.nmpdr.org/SubsysEditor.cgi" >Curate Subsystems</a></li>
			<li><a href="?page=Scenarios" >Scenarios</a></li>
			<li><a href="?page=FigFamViewer" >FigFams</a></li>
			<li><a href="?page=BlastRun" >BLAST Search</a></li>
			<li><a href="http://www.theseed.org/daily" >Visit the Daily SEED</a></li>
			<li><a href="?page=SubsystemPrimer" >Request a Subsystem</a></li>
		</ul>
		</li>
		<li><div><a  >&raquo;Organism</a></div>
		<ul>
			<li><a href="?page=Organism&organism=2665645.4" >General Information</a></li>
			<li><a href="?page=BrowseGenome&tabular=1&organism=2665645.4" >Feature Table</a></li>
			<li><a href="?page=BrowseGenome&organism=2665645.4" >Genome Browser</a></li>
			<li><a href="?page=Scenarios&organism=2665645.4" >Scenarios</a></li>
			<li><a href="?page=SubsystemSelect&organism=2665645.4" >Subsystems</a></li>
			<li><a href="?page=Export&organism=2665645.4" >Export</a></li>
		</ul>
		</li>
		<li><div><a  >&raquo;Comparative Tools</a></div>
		<ul>
			<li><a href="?page=CompareMetabolicReconstruction&organism=2665645.4" >Function based Comparison</a></li>
			<li><a href="?page=MultiGenomeCompare&organism=2665645.4" >Sequence based Comparison</a></li>
			<li><a href="?page=Kegg&organism=2665645.4" >Kegg Metabolic Analysis</a></li>
			<li><a href="?page=BlastRun&organism=2665645.4" >BLAST search</a></li>
		</ul>
		</li>
		<li><div><a href="http://www.theseed.org" target="help">&raquo;Help</a></div>
		<ul>
			<li><a href="http://www.theseed.org/wiki/index.php/Home_of_the_SEED" >What is the SEED</a></li>
			<li><a href="http://www.theseed.org/wiki/index.php/SEED_Viewer_Tutorial" >HowTo use the SEED Viewer</a></li>
			<li><a href="http://www.theseed.org/wiki/index.php/RAST_Tutorial" >Submitting Data to SEED</a></li>
			<li><a href="mailto:rast@mcs.anl.gov" >Contact</a></li>
			<li><a href="?page=Register" >Register</a></li>
			<li><a href="?page=RequestNewPassword" >I forgot my Password</a></li>
		</ul>
		</li>
	</ul>
</div>

	
      </td>
      <td style="border-right: 2px solid #5DA668; padding: 0px;">

	<div id='search'><form style='margin: 0px; padding: 0px;'><input type='submit' value='find' style='width:40px;cursor: pointer;'><input type='text' name='pattern'><input type='hidden' name='page' value='SearchResult'><input type='hidden' name='action' value='check_search'><input type='submit' style='display:none;'></form></div>

      </td>
      <td style="text-align: right; padding: 0px;">
	
	  <div id='user'><a href='seedviewer.cgi?page=AccountManagement'><img class='imglink' src='https://rast.nmpdr.org/Html/wac_people.png' title='Account Management' /></a>
<a href='seedviewer.cgi?page=Organism&organism=2665645.4&logout=1'><img class='imglink' src='https://rast.nmpdr.org/Html/wac_logout.png' title='Logout' /></a>
Ekaterina Pushmina</div>
	
      </td>
    </tr>
  </table>
</div>


 

  
  
  <div id="content">
    <script>function initialize_all () {

}</script><style>.hideme { display: none; }</style><h2>Organism Overview for Flavobacterium sp. SLB02 (2665645.4)</h2><table><tr><td width=550px><div><table><tr><th style='width: 170px;'>Genome</th><td>Flavobacterium sp. SLB02 (Taxonomy ID: <a href='http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=2665645&lvl=3&lin=f&keep=1&srchmode=1&unlock' target=outbound>2665645</a>)&nbsp;&nbsp;&nbsp;<a href='' target='outbound'><img style='width: 20px; height:20px;' src="https://rast.nmpdr.org/Html/wikipedia-logo.png" title='Show Wikipedia entry'></a></td></tr><tr><th>Domain</th><td>Bacteria</td></tr><tr><th>Taxonomy</th><td>Bacteria; Bacteroidetes; Flavobacteriia; Flavobacteriales; Flavobacteriaceae; Flavobacterium; unclassified Flavobacterium; Flavobacterium sp. SLB02 ; Flavobacterium sp. SLB02</td></tr><tr><th>Neighbors</th><td><a href='seedviewer.cgi?page=ClosestNeighbors&organism=2665645.4'>View closest neighbors</a></td></tr><tr><th>Size</th><td>6,363,829</td></tr><tr><th>GC Content</th><td>34.3</td></tr><tr><th>N50</th><td>361257</td></tr><tr><th>L50</th><td>6</td></tr><tr><th>Number of Contigs (with PEGs)</th><td>1</td></tr><tr><th>Number of Subsystems</th><td>274</td></tr><tr><th>Number of Coding Sequences</th><td>5368</td></tr><tr><th>Number of RNAs</th><td>57</td></tr></table></div></td><td>For each genome we offer a wide set of information to browse, compare and download.<br><br><input type='hidden' id='OrganismInfoContent' value='0' /><table style='height: 125px; width: 325px;  border-spacing: 0px;border-collapse: expression("separate", cellSpacing = "0px");' id='tab_view_table_0'><tr><td class='tab_view_title_selected' name='0_tabs' id='0_tab_0' onclick='tab_view_select("0", "0");'>Browse</td><td class='tab_view_title' name='0_tabs' id='0_tab_1' onclick='tab_view_select("0", "1");'>Compare</td><td class='tab_view_title' name='0_tabs' id='0_tab_2' onclick='tab_view_select("0", "2");'>Download</td><td class='tab_view_title' name='0_tabs' id='0_tab_3' onclick='tab_view_select("0", "3");'>Annotate</td><td class='tab_view_title_filler'>&nbsp;</td></tr><tr><td colspan='5' class='tab_view_content_td'><div style='height: 100%; width: 100%;' class='tab_view_content_selected' name='0_contents' id='0_content_0'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">Browse through the features of <a href="seedviewer.cgi?page=BrowseGenome&organism=2665645.4">Flavobacterium sp. SLB02</a> both graphically and through a table. Both allow quick navigation and filtering for features of your interest. Each feature is linked to its own detail page.<br><br>Click <a href="seedviewer.cgi?page=BrowseGenome&organism=2665645.4">here</a> to get to the Genome Browser</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_1'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">Compare the metabolic reconstruction of this organism to that of another organism.<br><br>Available comparisons are <a href="seedviewer.cgi?page=CompareMetabolicReconstruction&organism=2665645.4">function based</a>, <a href="?page=MultiGenomeCompare&organism=2665645.4">sequence based</a> or via <a href="seedviewer.cgi?page=Kegg&organism=2665645.4">KEGG</a>. You can also <a href="seedviewer.cgi?page=BlastRun&organism=2665645.4">BLAST</a> against this organism.</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_2'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">You can export all information about this organism (e.g. annotations, scenarios, subsystems) into a variety of formats (e.g. EMBL, Excel) for further analysis on your own system.<br><br>Click <a href="seedviewer.cgi?page=Export&organism=2665645.4">here</a> to get to the Export page.</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_3'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">The SeedViewer has several options to edit the annotations of your private genome. To find out how to use these capabilities and enable them for your genomes, click <a href="?page=HowToAnnotate">here</a>.</div></div></td></tr></table>
<img src='https://rast.nmpdr.org/Html/clear.gif'  onload='initialize_tab_view("0"); tab_view_select("0", "0");'>
</td></tr></table><h2>Subsystem Information</h2><input type='hidden' id='SubsystemTabView' value='1' /><table style='height: 200px; width: 95%px;  border-spacing: 0px;border-collapse: expression("separate", cellSpacing = "0px");' id='tab_view_table_1'><tr><td class='tab_view_title_selected' name='1_tabs' id='1_tab_0' onclick='tab_view_select("1", "0");'>Subsystem Statistics</td><td class='tab_view_title' name='1_tabs' id='1_tab_1' onclick='tab_view_select("1", "1");'>Features in Subsystems</td><td class='tab_view_title_filler'>&nbsp;</td></tr><tr><td colspan='3' class='tab_view_content_td'><div style='height: 100%; width: 100%;' class='tab_view_content_selected' name='1_contents' id='1_content_0'><table style='padding-top: 25px;'><tr><td><b>Subsystem Coverage</b></td><td align=center><b>Subsystem Category Distribution</b></td><td align=center><b>Subsystem Feature Counts</b></td></tr><tr><td style='padding-right: 10px;'><img src="data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAAGQCAMAAAAOUBefAAAAolBMVEX///8AAADIyMjrBSg8PL4y/zLghQ8AAJv/3QAAm5vIZMgbhTSHQUEAWv9fZGRQ0pbh+qCqHpH/132MpdKgD/otm7mqzXgePL5zr6DChQ8AAH3h3QD/AAAA/wAAAP///wAA////AP//fwD/AH8A/39/AP9//wAAf///f39//39/f/8AAH9/AAAAfwB/fwB/AH8Af3//f////39///9QdJpdpmhNMpuCAAABEElEQVR4nO3dQQ6DIBAF0PFO9P5XK8WxVhc1Chvj+yYSJHn5iis3RhxlOpdDL6bXmQCBQCAQCATeGKzr9dQJRiTVrMj5dbBBTYlok1/vApiNEvwWHnXLe+86uDTceQPAfKajwLYnfeB8cX1tBjT8GyAQCAQCgfcCh3/uK2cCBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIDBT1+upD4yG5BCfow9sTKxDbAp2g6VsC3bc8kJtvesNcz/2XgeYUpQYtynzLMa8NvNaidLd8CBAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAh8Mjj6FwkiIiIiD84bw4gPX60x0lsAAAAASUVORK5CYII=" usemap="#imap_barchart_0"><map name='imap_barchart_0'>
<area shape='rect' coords='33,63,81,370' onmouseover='hover(event, "bar_0", "2");'onclick='window.top.location="?page=BrowseGenome&organism=2665645.4&tabular=1&noss=1"' style='cursor: pointer;'>
<area shape='rect' coords='33,0,81,63' onmouseover='hover(event, "bar_1", "2");'onclick='tab_view_select("1", 1);' style='cursor: pointer;'>
</map><input type='hidden' id='hover_redundancies_2' value='bar_1~#bar_0'><span style='display: none;' id='tooltip_2_bar_0'><table><tr><th>not in Subsystem</th></tr><tr><td>total (4440)</td></tr><tr><td>non-hypothetical (1946)</td></tr><tr><td>hypothetical (2494)</td></tr></table></span>
<span style='display: none;' id='tooltip_2_bar_1'><table><tr><th>in Subsystem</th></tr><tr><td>total (928)</td></tr><tr><td>non-hypothetical (888)</td></tr><tr><td>hypothetical (40)</td></tr></table></span>
</td><td style='padding-right: 10px;'><img style="border: none;" src="data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAZAAAAGQCAMAAAC3Ycb+AAABDlBMVEX///8AAADIyMjrBSg8PL4y/zL/AAAAgAAAAICAgACAAIDAwMCAgICZmf+ZM2b//8xmAGb/gIAAZszMzP//AP///wAA//+AAAAAgIAAAP8AzMzM///M/8z//5mZzP//mczMmf//zJkzZv8zzMyZzAD/zAD/mQD/ZgBmZpmWlpYAM2YzmWYAMwAzMwCZMwAzM5kzMzPghQ8AAJv/3QAAm5vIZMgbhTSHQUEAWv9fZGRQ0pbh+qCqHpH/132MpdKgD/otm7mqzXgePL5zr6DChQ8AAH3h3QAA/wD/fwD/AH8A/39/AP9//wAAf///f39//39/f/8AAH9/AAAAfwB/fwB/AH8Af3//f////39////pA/ySAAAPp0lEQVR4nO3d2VJbSRCEYV0N2IaYUYSXsIXw+7/lILRwll6qu6sqsw76nwD1F1nyDXi3C9nfVf8sQ/+In6E1QwHkTmNbCaMCckdRrSYhA7m76CTUEIPcTUaSa7SA3E3aa5LoALm7NNSD0QlyR6nWq9EPcjfJN6AxBHI3STSGMQxyR1k07jEOcie5pcChAnIXOaWioQRyN9HS0AP51CaKHJogn5VElUMX5BOSKGuog3wuE30NC5DPY2LiYQLyKUhsOKxAti5ipWEHsm0TQw9DkM2SWHLYgmxTxNbDFmSDJMYc5iCbEjHH8ADZEIqLhwvINkR8PHxAtiDi5OEEEp7Ei8MPJLaIn4cfSGASRw5XkKgirh6uICFJfDm8QeKJeHt4gwQjcecAgEQSAXgAQMKQIDgwIDFEMB4YkAAkIA4YCLsIzAMGQk2C40CCPKCfPRvSAwfy8MAqAvWAgTw8sIpgPbAghCRgDhjIwwOnCJoDBfLwwCmC1vgLAnl44BRBY5zCg/CIoC3ew3vQiKApzhF4kIigJS4xeDCQoB1ucXjARdAMH7GAgEXQDB+xeGBB0AqTWDygImiEaTQeOBI0wTwiD5AIWmARFQhEBC2wiMoDAYIGWEblARBBv/8qLg9vEvTrJ6IDefzcHn4gUo9HRxH046fi8/ADQb99Mj4PNxH006cj9HASQb98JiaQx0dHkddX9NOno/RwEHl9JRWh8Xic5+DBKULqYSzyeg39/OtYPWxFXnlFOEASHpYgr6+8IrQehiKvr8QivB5mIq+vzCLEHkYiSw8yETxI3sNGZA1CJULtYQGS8KAiofYwEDkeyUW4PfRFjscMCRriGrmHNsjxyC4CBal7KIscj/Qi7B66IsdjiQRt8R7OQ8ahCnI88ovQeyiKHI81EQISFIjcQ0/keKyToD2sQDQ9tECWHpwiATyURNYelCIRPHREUiBJku2BqHtogKQ9+EQAIO0eCiIvLy8xRGJ4jIu85EW4SIJ4jIK8nAsgEsTj8YuGRwSRGB5f3lIBaSD5FCADHiMgLy9xRAJ4fLmk45ElIRFx9BiZx4jI0kM+kvAghvMYEFmDMI/EDWScQxOEVySSR59I0iNHghch9lhx6IJIScKCOHj0gOQ9OEVYPVIcXSIlkDQJVsQDRI2jA6TsQTiSWB5fvil78I2E0KPA8ZY6CJkInUeZoxFE4iEk2QqImse3awYgTCJcHnWOJhCpR5oEI2LqoT+PNpEGEMlIooB4zqMJpMWDZiQ0HlKOBpFGkBSJv4gdiI7HmkMM0uwhGQk/iIpHC4dYpANEMBJ2EEuPDIcQpMsDL0Lg0cohFOkEqZMwgyh4dHAYg2BFTECGPcocIpB+jxSJ39nCenTNQyYyBAIcCaFHnePbkzVIdSSMIKMe/RxvWXvARNRBrDmeztmD1EjoQAw8xBxVEQ2PBImDiK6HG4cTCEAE4dHH8TTPxQNAQuLRylERUQTxFlEEceXwA1mR2IoQePRolEF0PcojoQDp9uiYR4ajKKINUhwJAUivhypHAUTfw1EE69GvURKxACmREIIgOLIgNh5eI8F5jHI8PT37gixJbER8PBo5RBpveYMUSGAgPR76HM+XvD0cRMZB2j2UNCAgCxJ9kVEPb47nWQiQ7EgQILYejRppEHOPJYmyiK1HA0frOHIiHiCWZ8vSQ5MjpYEDmZOoioyANHr0c2Q0UiBOHnZny8xDzNGvkRJxA8mNxA9kyKOXo6yBBZmRqInYeOhwVDXQIOmRYEAUOMY11iK+Hi/pkbiAdHt0cUg18CDqZ0vdQzYPJQ0CEO2RKHuIOPQ0KEB0RSw9mjmaNThApiTDIh0gTBwkIKmRGIL0zMNHgwYkNRIzkI55NHJ0ayxFcB5qIkoeAxxDGkQgSiT6Hq4aVCATkn6RJhBlDgWNOQjaQ0NE16OBQ0djLoLmOLUkUQape5TnYa5BBzIqMujRw6GqMQVBU1wbEhGDtF8rF42pCBri1mIkeiAVjxKHmwYjyMhIVDygHJQg/SLdHnkOXw1SkCWJAkjRo43DUoMWpFNEAlLwEHAYa/CC3EiaROoecg6IxgcI+vlTTUnGQNo9QBofIujHT9Y8kmYPKYebBjfIlUQsUgGRcUA12EEaz1afR5HDWeMqgn73Qi0iLR4SDoAGP0iLiNwjxUEwjhAgU5J2kKIH3TiCgIhF8iBtHECNGCAXkurZavEg1YgCIhuJxGM1Dy6N5yggk5HIQdYeRQ60xKUoIGeSoojQg1njORLIbSQtICIOtMGsQCC3kYhAkh4JDjTAskgeFZGSx4yDV+NUKJB3ktwXSd4jzYF++UzBQAojWYLMPKJwxAO5jqQIMvWYcPBrPEcEyYlkPRYc6BevFBDkQpIFmXh8zCOGxvPzvyFBkiJrjxsHs8a/i2KCnEiWIhOQ+bXi0lgCbAVkTbLwuHDgNWoCmwFZisw8phzkAtsBeSOZisw9rhz0AJsCOY1kBnL2eOcw1NAV2BTIaSQzkPd5nDmiCGwM5EPk4qHC4SmwNZA3kivIw/u16tEAAizbAMhlJO8e34Qa6GfPtwWQl5eTyD8njoACi7YBcjpbj08hAVZtA+Tl5S/6IbXaCsjv/f6/U+j3HG4jIL+//txfSP4LDbON75DDnzPI3CSkyyZA3jzeQL7vkyTBXLYAcvb4+WO/L5BEgdkAyMnjBHKbSJGE3SU+yJ8/byC/FiB1E1aX6CCHd4/fX08gP+YiEhI+mOAgF480SAMJj0tskKvHr3eQn99XIo0kBC6hQU4el4G8g6wn0mkChIkMcvP4AEmKdJMgXOKCHD48LhcrCzJI4uoSFmTicQN5+xLJiSiQ+MBEBTl7XEC+XkHyE9E0MXUJCjLz+DUFKYmokhi5xASZeTSAGJBow0QEOaw9LiACESsSLZdYv/X53tWjE8TYZNAl1K9Fn1t6rEAEIuYk3TDxQK4cSQ/pRNxIml3Cgaw8EiAyEU8SuUs0kJuHAgjApA4TC+SQ85iAtIlgSAouoUBSHkuQxolASVIukUCSHmmQJhE0ybl4IBOP1UCmIM0ToTE5FQck7ZEDaRZhIaH+y9aTDjmP5cXqullEJEFAch7rgfRPhIKE+2+/35p5GILgTWKAFDzyIL0iWBLi/9Hlo7zHdSAzkMGJQElo/8ujSYeCRxFkQARGEgBkzrEciBUIyoQfpOhxG8gcZPxmwUjoQcoeFZBhEX8SdpCaR/JiKYK4k3CDHJYewoG8geiJ+JrsmEXqHjkQzYm4kuyYQaoeAhAtES8SZpC6hyeIE8mOV2TtURhIAkRfxMFkRwtyEHiUQCwm4kBCCyLykIEoi9iS7EhFxj2MbpY1CSlIyqMfRF/EzGTHCZLiSHiUQexuliEJJUhyHhWPNIitiAUJI4jYAw5iQEIIkvZID0QMYiaibcIHIveYDSQF4jERbZIdm0iDRxOIqYgiCRnIoc2jfLEcQdRIdlwgTR71gbiK6JhwgeQ8goBokCxBoCLNHm0gHiKjJCsPJEijx3wgeRDXiYySEIEcWj1EIP4TGTNZg6BE8h75gdCCdJMkPEAg7R6LgRRAICJ9JDQgeY6Ch2QgOJAukhQIQqTDQwoyv1m+Iu0mJCAFjxKI5GJBJ9JMkvRwByl8fcgHIgXxF2khSYM4i/R5BAKRk2Q8fEE6PZYXKw/CICI0yYF4ihQ9GgbCDiIhyXo4gvR6DIDARKokeRAvkUO3R0iQCknBwwnEx4NKpGQCB6l4bBQkT1IC8RAZ9IgLkiEpejiAjHisB1IE4RNJkZRBrEVqHGWP+CD7tQkUpOpRHcgQCIfIvsXDVmTQIzGQOgjfRPYzEiTIqEczCOtE9h8kVQ9DEA2PpovFDLK/mNRBrEQOdQ/1gaxvFpXIXuRhBKLgsT2Q/R4GouTRCrK+WWQiEhALEQ2P1EBEIMQTEXkYgEg8ugbSAUIlIgPRFjmoeGwRROihDKLkkbxYVRBuESmIqoiiR/tAuEHEHpogMg9XEBoROYieiIxD4NF5sZgn0uChBSKch9BDDYREpAVER0TRY4MgTR4qIFIPKcjaQwzCKNIGoiCi7dEJwjqRRo9xEVUPbRC8SLPHIMhB36PvK4T1ZrWDDInIPcwHkgZBi3R4jIBoewyBUN6sHpB+ERMPZRCwiCuIukduIC0gXBPp8ugEOTR4IEGgIn0gXSJWHgMgfDer06NHxMIjO5AmECKRbo92kRYOscfWQAY8WkGaPJoGMnKxciAokRGQJpGmc+U4ELKJDHm0iFh62IBARAY95CBWHneQPhEzj/zFkoMQ3axhD6FIo4fKQMZB/EUUPEQgdh53kA6Rg6FH4WI1g+BFVDyqIuYewwNhmYiSRwWk2YMJxFdEC6QoYuuhc7FIbpaaR0nEwcMUxFFE0SML0s7R5qEGwnCzNEEyIh0eegPpAUGKqHqkRcw9tgSi7JECsfdwAXES0QZZi4TygE9E3WMpcujx4ATxEDHw2Pl73EHEIo4eWiD5f/g6iJh4TER8PMoD6QSBTMTI4ybS50EMYixi5nEWObh6bADE0GPn6VEZSDMITMQSZOftoTcQGIipx27X6cEOYidi7DEXEXN0eKhfrCuI70TMPaYi9h6qIIiJOHh8iJh6bAPExeMqIvfoB8l7DID4iTh5nEU8PIKDuHns7D1MQMo3S1/ED2Rn7lG9WCMgThNx9Hgr4kAqIMoivh5ykYGBRAbx5hCLGHn0gfiJADxkIH0edxAzkVggaiIQDwmJmUcniNNEUB5VkU4PHIiKCI6jJtLrERsE6mEDYudxAzG8WViQkgjhQBwmAvYokHR7hAZBa5wCeIyD2IigKS7dQbg8kiKjHlYg1Zs1IIJmmKToERcEjTBLz0N0sRRA1EXQBIuiDMQMBA2wTnUgdiD1m9Ulgn79VCoe1hfLZiLop8+k54EFaRZBP3y2TwqCfvZSox6yizUEoi6CfvNyKh6RQNAPXm8LIHIR9GtLIvfQnAj6qaXFB5GJoN9Z3qcAQT9yW5YeaiBDIugXbo0YRGMi6OftKTZIWQT9tn1ZeeiBdE4E/bD9RQbJi6BfdSROkKGbhX7S0Rg9RiaCfs/xNgWCfkyV2jyIQNYi6KdUSnkgCiCdE0E/pGYhQb5vFOOc2sWCgKBfzyKtgWiANIqgn86qqCDod7OL5WKJQb5vmuO9YCDo5/KI4WJJRdBv5dPYQBxB0A/lWAAQ9BO51+uhDpISQT8OJihIaSLoh8FFCYJ+FHAoj8zNQj8HRTQg6IfgCQGyvlnoV2BK7mEB8uPOkQoIgv7otEFA0B+aPD+Qn3cNaU4g6I8ZLjMP9AcLnAEI+iOFTxUE/WE21CAI+sffZp0g6B/7M3RnYG8z7/8/gf6RTLULq1wAAAAASUVORK5CYII=" usemap="#imap_piechart_0"><map name='imap_piechart_0'><area shape='poly' coords='199,199,199,0,216,0,233,3,250,6,267,12,283,18,298,26,313,35,322,43' id='pie_0_slice_0'  onmouseover='hover(event, "pie_0_slice_0", "0");'>
<area shape='poly' coords='199,199,322,43,335,54,346,64' id='pie_0_slice_1'  onmouseover='hover(event, "pie_0_slice_1", "0");'>
<area shape='poly' coords='199,199,346,64,357,78,367,92,368,94' id='pie_0_slice_2'  onmouseover='hover(event, "pie_0_slice_2", "0");'>
<area shape='poly' coords='199,199,368,94,374,105' id='pie_0_slice_3'  onmouseover='hover(event, "pie_0_slice_3", "0");'>
<area shape='poly' coords='199,199,374,105,382,121,382,122' id='pie_0_slice_5'  onmouseover='hover(event, "pie_0_slice_5", "0");'>
<area shape='poly' coords='199,199,382,122,387,134' id='pie_0_slice_6'  onmouseover='hover(event, "pie_0_slice_6", "0");'>
<area shape='poly' coords='199,199,387,134,392,151,395,168,397,185,397,186' id='pie_0_slice_7'  onmouseover='hover(event, "pie_0_slice_7", "0");'>
<area shape='poly' coords='199,199,397,186,397,203,396,221,395,229' id='pie_0_slice_9'  onmouseover='hover(event, "pie_0_slice_9", "0");'>
<area shape='poly' coords='199,199,395,229,392,246,387,262,381,279,380,281' id='pie_0_slice_10'  onmouseover='hover(event, "pie_0_slice_10", "0");'>
<area shape='poly' coords='199,199,380,281,372,297,362,311,352,325,340,338,328,350,314,361,299,370,284,378' id='pie_0_slice_11'  onmouseover='hover(event, "pie_0_slice_11", "0");'>
<area shape='poly' coords='199,199,284,378,268,385,257,389' id='pie_0_slice_14'  onmouseover='hover(event, "pie_0_slice_14", "0");'>
<area shape='poly' coords='199,199,257,389,250,391' id='pie_0_slice_15'  onmouseover='hover(event, "pie_0_slice_15", "0");'>
<area shape='poly' coords='199,199,250,391,233,395,216,397,198,397,181,397,174,396' id='pie_0_slice_16'  onmouseover='hover(event, "pie_0_slice_16", "0");'>
<area shape='poly' coords='199,199,174,396,157,393,140,389,137,388' id='pie_0_slice_17'  onmouseover='hover(event, "pie_0_slice_17", "0");'>
<area shape='poly' coords='199,199,137,388,130,385' id='pie_0_slice_18'  onmouseover='hover(event, "pie_0_slice_18", "0");'>
<area shape='poly' coords='199,199,130,385,114,379,99,371,84,361,76,356' id='pie_0_slice_20'  onmouseover='hover(event, "pie_0_slice_20", "0");'>
<area shape='poly' coords='199,199,76,356,63,344,53,334' id='pie_0_slice_21'  onmouseover='hover(event, "pie_0_slice_21", "0");'>
<area shape='poly' coords='199,199,53,334,46,326' id='pie_0_slice_22'  onmouseover='hover(event, "pie_0_slice_22", "0");'>
<area shape='poly' coords='199,199,46,326,35,312,26,297,18,282,11,266,6,249,2,232,0,215,0,198,0,180,3,163,6,146,12,130,18,114,27,98,33,88' id='pie_0_slice_23'  onmouseover='hover(event, "pie_0_slice_23", "0");'>
<area shape='poly' coords='199,199,33,88,41,77' id='pie_0_slice_24'  onmouseover='hover(event, "pie_0_slice_24", "0");'>
<area shape='poly' coords='199,199,41,77,50,66' id='pie_0_slice_25'  onmouseover='hover(event, "pie_0_slice_25", "0");'>
<area shape='poly' coords='199,199,50,66,62,54,75,42,89,32,104,23,120,16,136,9,153,5,170,2,187,0,197,0' id='pie_0_slice_26'  onmouseover='hover(event, "pie_0_slice_26", "0");'></map><input type='hidden' id='hover_redundancies_0' value='pie_0_slice_16~#pie_0_slice_2~#pie_0_slice_5~#pie_0_slice_25~#pie_0_slice_14~#pie_0_slice_6~#pie_0_slice_22~#pie_0_slice_10~#pie_0_slice_11~#pie_0_slice_0~#pie_0_slice_1~#pie_0_slice_3~#pie_0_slice_15~#pie_0_slice_18~#pie_0_slice_24~#pie_0_slice_23~#pie_0_slice_17~#pie_0_slice_26~#pie_0_slice_9~#pie_0_slice_21~#pie_0_slice_20~#pie_0_slice_7'><span style='display: none;' id='tooltip_0_pie_0_slice_0'><table><tr><th>Cofactors, Vitamins, Prosthetic Groups, Pigments</th></tr><tr><td>134 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_1'><table><tr><th>Cell Wall and Capsule</th></tr><tr><td>32 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_2'><table><tr><th>Virulence, Disease and Defense</th></tr><tr><td>37 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_3'><table><tr><th>Potassium metabolism</th></tr><tr><td>13 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_5'><table><tr><th>Miscellaneous</th></tr><tr><td>19 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_6'><table><tr><th>Phages, Prophages, Transposable elements, Plasmids</th></tr><tr><td>13 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_7'><table><tr><th>Membrane Transport</th></tr><tr><td>53 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_9'><table><tr><th>RNA Metabolism</th></tr><tr><td>43 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_10'><table><tr><th>Nucleosides and Nucleotides</th></tr><tr><td>55 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_11'><table><tr><th>Protein Metabolism</th></tr><tr><td>139 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_14'><table><tr><th>Regulation and Cell signaling</th></tr><tr><td>29 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_15'><table><tr><th>Secondary Metabolism</th></tr><tr><td>8 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_16'><table><tr><th>DNA Metabolism</th></tr><tr><td>77 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_17'><table><tr><th>Fatty Acids, Lipids, and Isoprenoids</th></tr><tr><td>38 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_18'><table><tr><th>Nitrogen Metabolism</th></tr><tr><td>7 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_20'><table><tr><th>Respiration</th></tr><tr><td>62 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_21'><table><tr><th>Stress Response</th></tr><tr><td>32 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_22'><table><tr><th>Metabolism of Aromatic Compounds</th></tr><tr><td>11 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_23'><table><tr><th>Amino Acids and Derivatives</th></tr><tr><td>256 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_24'><table><tr><th>Sulfur Metabolism</th></tr><tr><td>14 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_25'><table><tr><th>Phosphorus Metabolism</th></tr><tr><td>14 / 1254</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_26'><table><tr><th>Carbohydrates</th></tr><tr><td>166 / 1254</td></tr></table></span>
</td><td>
<div id='tree_0'>
<div name='tree_level_0' id='tree_node_0_0'><img id='tree_img_0_0' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "0");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/AAAZ4gk3AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cofactors, Vitamins, Prosthetic Groups, Pigments (134)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_0'>
<img id='tree_img_0_1' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "1");'>&nbsp;Biotin (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_1'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=2665645.4'>Biotin biosynthesis</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=2665645.4'>Biotin biosynthesis Experimental</a> (4)<br></div>
<img id='tree_img_0_4' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "4");'>&nbsp;Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_4'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=2665645.4'>Thiamin biosynthesis</a> (7)<br></div>
<span style='cursor: pointer;'>&nbsp;Quinone cofactors (0)<br>
<img id='tree_img_0_7' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "7");'>&nbsp;Tetrapyrroles (18)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_7'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=2665645.4'>Cobalamin synthesis</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4'>Heme and Siroheme Biosynthesis</a> (13)<br></div>
<img id='tree_img_0_10' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "10");'>&nbsp;Riboflavin, FMN, FAD (15)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_10'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=2665645.4'>riboflavin to FAD</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Flavodoxin&organism=2665645.4'>Flavodoxin</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=2665645.4'>Riboflavin, FMN and FAD metabolism</a> (9)<br></div>
<span style='cursor: pointer;'>&nbsp;Fe-S clusters (0)<br>
<span style='cursor: pointer;'>&nbsp;Mycofactocin (0)<br>
<img id='tree_img_0_16' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "16");'>&nbsp;Pyridoxine (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_16'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=2665645.4'>Pyridoxin (Vitamin B6) Biosynthesis</a> (10)<br></div>
<img id='tree_img_0_18' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "18");'>&nbsp;NAD and NADP (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_18'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4'>NAD and NADP cofactor biosynthesis global</a> (12)<br></div>
<span style='cursor: pointer;'>&nbsp;Coenzyme B (0)<br>
<img id='tree_img_0_21' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "21");'>&nbsp;Folate and pterines (48)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_21'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4'>Folate biosynthesis cluster</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4'>Folate Biosynthesis</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4'>5-FCL-like protein</a> (18)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=2665645.4'>Pterin synthesis related cluster</a> (8)<br></div>
<span style='cursor: pointer;'>&nbsp;Lipoic acid (0)<br>
<span style='cursor: pointer;'>&nbsp;Coenzyme F420 (0)<br>
<span style='cursor: pointer;'>&nbsp;Coenzyme M (0)<br>
<img id='tree_img_0_29' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "29");'>&nbsp;Coenzyme A (15)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_29'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=2665645.4'>Coenzyme A Biosynthesis cluster</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4'>Coenzyme A Biosynthesis</a> (10)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_32'><img id='tree_img_0_32' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "32");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAgACc+aWRAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cell Wall and Capsule (32)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_32'>
<img id='tree_img_0_33' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "33");'>&nbsp;Capsular and extracellular polysacchrides (23)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_33'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=2665645.4'>dTDP-rhamnose synthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Streptococcal_Hyaluronic_Acid_Capsule&organism=2665645.4'>Streptococcal Hyaluronic Acid Capsule</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=2665645.4'>CMP-N-acetylneuraminate Biosynthesis</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polysaccharide_deacetylases&organism=2665645.4'>Polysaccharide deacetylases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=2665645.4'>Rhamnose containing glycans</a> (9)<br></div>
<img id='tree_img_0_39' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "39");'>&nbsp;Gram-Negative cell wall components (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_39'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lipoprotein_sorting_system&organism=2665645.4'>Lipoprotein sorting system</a> (1)<br></div>
<img id='tree_img_0_41' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "41");'>&nbsp;Cell Wall and Capsule - no subcategory (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_41'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=2665645.4'>Murein Hydrolases</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=2665645.4'>Recycling of Peptidoglycan Amino Acids</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;Gram-Positive cell wall components (0)<br>
<span style='cursor: pointer;'>&nbsp;Cell wall of Mycobacteria (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_46'><img id='tree_img_0_46' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "46");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAAIBKwr76AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Virulence, Disease and Defense (37)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_46'>
<span style='cursor: pointer;'>&nbsp;Adhesion (0)<br>
<span style='cursor: pointer;'>&nbsp;Toxins and superantigens (0)<br>
<span style='cursor: pointer;'>&nbsp;Bacteriocins, ribosomally synthesized antibacterial peptides (0)<br>
<img id='tree_img_0_50' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "50");'>&nbsp;Resistance to antibiotics and toxic compounds (24)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_50'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=2665645.4'>Copper homeostasis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=2665645.4'>Cobalt-zinc-cadmium resistance</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mercuric_reductase&organism=2665645.4'>Mercuric reductase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Aminoglycoside_adenylyltransferases&organism=2665645.4'>Aminoglycoside adenylyltransferases</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=2665645.4'>Resistance to fluoroquinolones</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis:_copper_tolerance&organism=2665645.4'>Copper homeostasis: copper tolerance</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Beta-lactamase&organism=2665645.4'>Beta-lactamase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=2665645.4'>Multidrug Resistance Efflux Pumps</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_chromium_compounds&organism=2665645.4'>Resistance to chromium compounds</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Virulence, Disease and Defense - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Detection (0)<br>
<img id='tree_img_0_62' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "62");'>&nbsp;Invasion and intracellular resistance (13)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_62'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=2665645.4'>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=2665645.4'>Mycobacterium virulence operon involved in DNA transcription</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=2665645.4'>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Listeria_surface_proteins:_Internalin-like_proteins&organism=2665645.4'>Listeria surface proteins: Internalin-like proteins</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=2665645.4'>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a> (3)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_68'><img id='tree_img_0_68' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "68");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAgAB9zL4RAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Potassium metabolism (13)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_68'>
<img id='tree_img_0_69' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "69");'>&nbsp;Potassium metabolism - no subcategory (13)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_69'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4'>Potassium homeostasis</a> (13)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_71'><img id='tree_img_0_71' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "71");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAICr96V6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Photosynthesis (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_71'>
<span style='cursor: pointer;'>&nbsp;Light-harvesting complexes (0)<br>
<span style='cursor: pointer;'>&nbsp;Photosynthesis - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Electron transport and photophosphorylation (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_75'><img id='tree_img_0_75' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "75");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXAwMBmS75kAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Miscellaneous (19)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_75'>
<span style='cursor: pointer;'>&nbsp;Conversion of Succinyl-CoA to Propionyl-CoA (0)<br>
<img id='tree_img_0_77' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "77");'>&nbsp;Plant-Prokaryote DOE project (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_77'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=2665645.4'>Iron-sulfur cluster assembly</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Single-Rhodanese-domain_proteins&organism=2665645.4'>Single-Rhodanese-domain proteins</a> (1)<br></div>
<img id='tree_img_0_80' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "80");'>&nbsp;Miscellaneous - no subcategory (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_80'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phosphoglycerate_mutase_protein_family&organism=2665645.4'>Phosphoglycerate mutase protein family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DedA_family_of_inner_membrane_proteins&organism=2665645.4'>DedA family of inner membrane proteins</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Muconate_lactonizing_enzyme_family&organism=2665645.4'>Muconate lactonizing enzyme family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Bacillus_subtilis_scratch_-_gjo&organism=2665645.4'>Bacillus subtilis scratch - gjo</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=2665645.4'>Broadly distributed proteins not in subsystems</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_86'><img id='tree_img_0_86' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "86");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAgICQdD0xAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Phages, Prophages, Transposable elements, Plasmids (13)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_86'>
<span style='cursor: pointer;'>&nbsp;Phage family-specific subsystems (0)<br>
<img id='tree_img_0_88' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "88");'>&nbsp;Transposable elements (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_88'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4'>Conjugative transposon, Bacteroidales</a> (12)<br></div>
<img id='tree_img_0_90' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "90");'>&nbsp;Phages, Prophages (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_90'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phage_packaging_machinery&organism=2665645.4'>Phage packaging machinery</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Phages, Prophages, Transposable elements, Plasmids - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Pathogenicity islands (0)<br>
<span style='cursor: pointer;'>&nbsp;Gene Transfer Agent (GTA) (0)<br>
<span style='cursor: pointer;'>&nbsp;Plasmid related functions (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_96'><img id='tree_img_0_96' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "96");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZmf/YOWB7AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Membrane Transport (53)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_96'>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type II (0)<br>
<span style='cursor: pointer;'>&nbsp;ABC transporters (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VII (Chaperone/Usher pathway, CU) (0)<br>
<img id='tree_img_0_100' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "100");'>&nbsp;Protein translocation across cytoplasmic membrane (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_100'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=2665645.4'>Twin-arginine translocation system</a> (4)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type V (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type I (0)<br>
<img id='tree_img_0_104' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "104");'>&nbsp;Cation transporters (6)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_104'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Magnesium_transport&organism=2665645.4'>Magnesium transport</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_Transport_System&organism=2665645.4'>Copper Transport System</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_transport_and_blue_copper_proteins&organism=2665645.4'>Copper transport and blue copper proteins</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type III (0)<br>
<img id='tree_img_0_109' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "109");'>&nbsp;Uni- Sym- and Antiporters (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_109'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters&organism=2665645.4'>NhaA, NhaD and Sodium-dependent phosphate transporters</a> (1)<br></div>
<img id='tree_img_0_111' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "111");'>&nbsp;Membrane Transport - no subcategory (39)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_111'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Choline_Transport&organism=2665645.4'>Choline Transport</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4'>Ton and Tol transport systems</a> (38)<br></div>
<span style='cursor: pointer;'>&nbsp;TRAP transporters (0)<br>
<span style='cursor: pointer;'>&nbsp;Sugar Phosphotransferase Systems, PTS (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VI (0)<br>
<img id='tree_img_0_117' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "117");'>&nbsp;Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP) (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_117'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Curli_production&organism=2665645.4'>Curli production</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein and nucleoprotein secretion system, Type IV (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_120'><img id='tree_img_0_120' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "120");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZM2YFAh/4AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Iron acquisition and metabolism (1)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_120'>
<span style='cursor: pointer;'>&nbsp;Siderophores (0)<br>
<img id='tree_img_0_122' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "122");'>&nbsp;Iron acquisition and metabolism - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_122'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Hemin_transport_system&organism=2665645.4'>Hemin transport system</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Iron transpot (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_125'><img id='tree_img_0_125' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "125");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//8wYFHreAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;RNA Metabolism (43)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_125'>
<img id='tree_img_0_126' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "126");'>&nbsp;RNA processing and modification (21)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_126'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=2665645.4'>mnm5U34 biosynthesis bacteria</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=2665645.4'>Ribonuclease H</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_3'-terminal_phosphate_cyclase&organism=2665645.4'>RNA 3'-terminal phosphate cyclase</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=2665645.4'>RNA processing and degradation, bacterial</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=2665645.4'>tRNA processing</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=2665645.4'>RNA methylation</a> (5)<br></div>
<img id='tree_img_0_133' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "133");'>&nbsp;Transcription (21)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_133'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=2665645.4'>Transcription initiation, bacterial sigma factors</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=2665645.4'>RNA polymerase bacterial</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4'>Transcription factors bacterial</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=2665645.4'>Rrf2 family transcriptional regulators</a> (4)<br></div>
<img id='tree_img_0_138' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "138");'>&nbsp;RNA Metabolism - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_138'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Group_II_intron-associated_genes&organism=2665645.4'>Group II intron-associated genes</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_140'><img id='tree_img_0_140' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "140");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEVmAGZP8U4lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Nucleosides and Nucleotides (55)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_140'>
<img id='tree_img_0_141' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "141");'>&nbsp;Pyrimidines (16)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_141'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4'>pyrimidine conversions</a> (16)<br></div>
<img id='tree_img_0_143' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "143");'>&nbsp;Purines (26)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_143'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Purine_Utilization&organism=2665645.4'>Purine Utilization</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=2665645.4'>GMP synthase</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4'>Purine conversions</a> (22)<br></div>
<img id='tree_img_0_147' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "147");'>&nbsp;Nucleosides and Nucleotides - no subcategory (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_147'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=2665645.4'>Ribonucleotide reduction</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=dNTP_triphosphohydrolase_protein_family&organism=2665645.4'>dNTP triphosphohydrolase protein family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=2665645.4'>Adenosyl nucleosidases</a> (3)<br></div>
<img id='tree_img_0_151' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "151");'>&nbsp;Detoxification (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_151'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nucleoside_triphosphate_pyrophosphohydrolase_MazG&organism=2665645.4'>Nucleoside triphosphate pyrophosphohydrolase MazG</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=2665645.4'>Nudix proteins (nucleoside triphosphate hydrolases)</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=2665645.4'>Housecleaning nucleoside triphosphate pyrophosphatases</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_155'><img id='tree_img_0_155' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "155");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/gIDP2RJcAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Protein Metabolism (139)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_155'>
<img id='tree_img_0_156' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "156");'>&nbsp;Protein folding (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_156'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=2665645.4'>Protein chaperones</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_disulfide_interchange&organism=2665645.4'>Periplasmic disulfide interchange</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Peptidyl-prolyl_cis-trans_isomerase&organism=2665645.4'>Peptidyl-prolyl cis-trans isomerase</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Selenoproteins (0)<br>
<img id='tree_img_0_161' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "161");'>&nbsp;Protein biosynthesis (102)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_161'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Val&organism=2665645.4'>tRNA aminoacylation, Val</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Met&organism=2665645.4'>tRNA aminoacylation, Met</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ile&organism=2665645.4'>tRNA aminoacylation, Ile</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Arg&organism=2665645.4'>tRNA aminoacylation, Arg</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=2665645.4'>Translation initiation factors bacterial</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4'>Ribosome SSU bacterial</a> (23)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Gly&organism=2665645.4'>tRNA aminoacylation, Gly</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ala&organism=2665645.4'>tRNA aminoacylation, Ala</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Trp&organism=2665645.4'>tRNA aminoacylation, Trp</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4'>Ribosome LSU bacterial</a> (34)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=2665645.4'>Programmed frameshift</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Cys&organism=2665645.4'>tRNA aminoacylation, Cys</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4'>Translation termination factors bacterial</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_His&organism=2665645.4'>tRNA aminoacylation, His</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=2665645.4'>tRNA aminoacylation, Asp and Asn</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=2665645.4'>Translation elongation factors bacterial</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Lys&organism=2665645.4'>tRNA aminoacylation, Lys</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Thr&organism=2665645.4'>tRNA aminoacylation, Thr</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Pro&organism=2665645.4'>tRNA aminoacylation, Pro</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factor_G_family&organism=2665645.4'>Translation elongation factor G family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=2665645.4'>tRNA aminoacylation, Glu and Gln</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ser&organism=2665645.4'>tRNA aminoacylation, Ser</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Tyr&organism=2665645.4'>tRNA aminoacylation, Tyr</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Leu&organism=2665645.4'>tRNA aminoacylation, Leu</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=2665645.4'>tRNA aminoacylation, Phe</a> (2)<br></div>
<img id='tree_img_0_187' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "187");'>&nbsp;Protein processing and modification (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_187'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein-L-isoaspartate_O-methyltransferase&organism=2665645.4'>Protein-L-isoaspartate O-methyltransferase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=2665645.4'>Signal peptidase</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=2665645.4'>Modification of eukaryotic initiation factor 5A</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=2665645.4'>N-linked Glycosylation in Bacteria</a> (6)<br></div>
<img id='tree_img_0_192' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "192");'>&nbsp;Protein degradation (16)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_192'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=2665645.4'>Aminopeptidases (EC 3.4.11.-)</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=2665645.4'>Protein degradation</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=2665645.4'>Metallocarboxypeptidases (EC 3.4.17.-)</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dipeptidases_(EC_3.4.13.-)&organism=2665645.4'>Dipeptidases (EC 3.4.13.-)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Serine_endopeptidase_(EC_3.4.21.-)&organism=2665645.4'>Serine endopeptidase (EC 3.4.21.-)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=2665645.4'>Proteolysis in bacteria, ATP-dependent</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_199'><img id='tree_img_0_199' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "199");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAZswGD39gAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cell Division and Cell Cycle (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_199'>
<span style='cursor: pointer;'>&nbsp;Checkpoint control (0)<br>
<span style='cursor: pointer;'>&nbsp;Heterocyst formation (0)<br>
<span style='cursor: pointer;'>&nbsp;Cell Division and Cell Cycle - no subcategory (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_203'><img id='tree_img_0_203' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "203");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXMzP91giUxAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Motility and Chemotaxis (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_203'>
<span style='cursor: pointer;'>&nbsp;Magnetotaxis (0)<br>
<span style='cursor: pointer;'>&nbsp;Motility and Chemotaxis - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Flagellar motility in Prokaryota (0)<br>
<span style='cursor: pointer;'>&nbsp;Social motility and nonflagellar swimming in bacteria (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_208'><img id='tree_img_0_208' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "208");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/AP804Oa6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Regulation and Cell signaling (29)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_208'>
<img id='tree_img_0_209' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "209");'>&nbsp;Regulation and Cell signaling - no subcategory (29)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_209'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=cAMP_signaling_in_bacteria&organism=2665645.4'>cAMP signaling in bacteria</a> (25)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Salmonella_enterica_Typhimurium&organism=2665645.4'>LysR-family proteins in Salmonella enterica Typhimurium</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Escherichia_coli&organism=2665645.4'>LysR-family proteins in Escherichia coli</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=2665645.4'>Stringent Response, (p)ppGpp metabolism</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Signal transduction in Eukaryotes (0)<br>
<span style='cursor: pointer;'>&nbsp;Quorum sensing and biofilm formation (0)<br>
<span style='cursor: pointer;'>&nbsp;Proteolytic pathway (0)<br>
<span style='cursor: pointer;'>&nbsp;Regulation of virulence (0)<br>
<span style='cursor: pointer;'>&nbsp;Programmed Cell Death and Toxin-antitoxin Systems (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_219'><img id='tree_img_0_219' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "219");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//wCKxvRFAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Secondary Metabolism (8)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_219'>
<img id='tree_img_0_220' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "220");'>&nbsp;Secondary Metabolism - no subcategory (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_220'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lanthionine_Synthetases&organism=2665645.4'>Lanthionine Synthetases</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Lipid-derived mediators (0)<br>
<span style='cursor: pointer;'>&nbsp;Plant Octadecanoids (0)<br>
<span style='cursor: pointer;'>&nbsp;Bacterial cytostatics, differentiation factors and antibiotics (0)<br>
<span style='cursor: pointer;'>&nbsp;Biosynthesis of phenylpropanoids (0)<br>
<span style='cursor: pointer;'>&nbsp;Hydrocarbons (0)<br>
<span style='cursor: pointer;'>&nbsp;Aromatic amino acids and derivatives (0)<br>
<span style='cursor: pointer;'>&nbsp;UV-absorbing secondary metabolites (0)<br>
<img id='tree_img_0_229' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "229");'>&nbsp;Plant Alkaloids (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_229'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Alkaloid_biosynthesis_from_L-lysine&organism=2665645.4'>Alkaloid biosynthesis from L-lysine</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Biologically active compounds in metazoan cell defence and differentiation (0)<br>
<img id='tree_img_0_232' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "232");'>&nbsp;Plant Hormones (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_232'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=2665645.4'>Auxin biosynthesis</a> (5)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_234'><img id='tree_img_0_234' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "234");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUA//8ZXC8lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;DNA Metabolism (77)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_234'>
<img id='tree_img_0_235' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "235");'>&nbsp;DNA repair (50)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_235'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=2665645.4'>DNA repair, bacterial MutL-MutS system</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=2665645.4'>DNA repair, UvrABC system</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_photolyase&organism=2665645.4'>DNA repair, bacterial photolyase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=2-phosphoglycolate_salvage&organism=2665645.4'>2-phosphoglycolate salvage</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=2665645.4'>DNA repair system including RecA, MutS and a hypothetical protein</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4'>DNA repair, bacterial</a> (19)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=2665645.4'>DNA repair, bacterial RecFOR pathway</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nonhomologous_End-Joining_in_Bacteria&organism=2665645.4'>Nonhomologous End-Joining in Bacteria</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_Repair_Base_Excision&organism=2665645.4'>DNA Repair Base Excision</a> (8)<br></div>
<span style='cursor: pointer;'>&nbsp;CRISPs (0)<br>
<img id='tree_img_0_246' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "246");'>&nbsp;DNA Metabolism - no subcategory (16)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_246'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=2665645.4'>Type I Restriction-Modification</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=2665645.4'>Restriction-Modification System</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_ligases&organism=2665645.4'>DNA ligases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=YcfH&organism=2665645.4'>YcfH</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=2665645.4'>DNA structural proteins, bacterial</a> (2)<br></div>
<img id='tree_img_0_252' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "252");'>&nbsp;DNA replication (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_252'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=2665645.4'>DNA topoisomerases, Type I, ATP-independent</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_replication_strays&organism=2665645.4'>DNA replication strays</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=2665645.4'>DNA topoisomerases, Type II, ATP-dependent</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;DNA recombination (0)<br>
<img id='tree_img_0_257' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "257");'>&nbsp;DNA uptake, competence (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_257'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=2665645.4'>DNA processing cluster</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_259'><img id='tree_img_0_259' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "259");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAICr96V6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Fatty Acids, Lipids, and Isoprenoids (38)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_259'>
<span style='cursor: pointer;'>&nbsp;Phospholipids (0)<br>
<img id='tree_img_0_261' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "261");'>&nbsp;Triacylglycerols (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_261'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Triacylglycerol_metabolism&organism=2665645.4'>Triacylglycerol metabolism</a> (1)<br></div>
<img id='tree_img_0_263' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "263");'>&nbsp;Fatty acids (19)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_263'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4'>Fatty Acid Biosynthesis FASII</a> (16)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=2665645.4'>Fatty acid metabolism cluster</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;Fatty Acids, Lipids, and Isoprenoids - no subcategory (0)<br>
<img id='tree_img_0_267' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "267");'>&nbsp;Isoprenoids (18)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_267'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Myxoxanthophyll_biosynthesis_in_Cyanobacteria&organism=2665645.4'>Myxoxanthophyll biosynthesis in Cyanobacteria</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=2665645.4'>Mevalonate Branch of Isoprenoid Biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=2665645.4'>Isoprenoinds for Quinones</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=2665645.4'>Isoprenoid Biosynthesis</a> (8)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_272'><img id='tree_img_0_272' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "272");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAABGTyZaAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Nitrogen Metabolism (7)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_272'>
<img id='tree_img_0_273' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "273");'>&nbsp;Nitrogen Metabolism - no subcategory (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_273'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=2665645.4'>Ammonia assimilation</a> (7)<br></div>
<span style='cursor: pointer;'>&nbsp;Denitrification (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_276'><img id='tree_img_0_276' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "276");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAgIBxQSaxAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Dormancy and Sporulation (1)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_276'>
<span style='cursor: pointer;'>&nbsp;Spore DNA protection (0)<br>
<img id='tree_img_0_278' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "278");'>&nbsp;Dormancy and Sporulation - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_278'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Sporulation-associated_proteins_with_broader_functions&organism=2665645.4'>Sporulation-associated proteins with broader functions</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_280'><img id='tree_img_0_280' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "280");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAAP+KeNJXAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Respiration (62)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_280'>
<span style='cursor: pointer;'>&nbsp;Biotin (9)<br>
<img id='tree_img_0_282' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "282");'>&nbsp;ATP synthases (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_282'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4'>F0F1-type ATP synthase</a> (10)<br></div>
<img id='tree_img_0_284' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "284");'>&nbsp;Electron accepting reactions (6)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_284'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_d_ubiquinol_oxidases&organism=2665645.4'>Terminal cytochrome d ubiquinol oxidases</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=2665645.4'>Anaerobic respiratory reductases</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_oxidases&organism=2665645.4'>Terminal cytochrome oxidases</a> (2)<br></div>
<img id='tree_img_0_288' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "288");'>&nbsp;Electron donating reactions (32)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_288'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4'>Respiratory Complex I</a> (14)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=2665645.4'>Respiratory dehydrogenases 1</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4'>NADH ubiquinone oxidoreductase</a> (14)<br></div>
<span style='cursor: pointer;'>&nbsp;Reverse electron transport (0)<br>
<img id='tree_img_0_293' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "293");'>&nbsp;Respiration - no subcategory (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_293'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Quinone_oxidoreductase_family&organism=2665645.4'>Quinone oxidoreductase family</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cytochrome_c_oxidases&organism=2665645.4'>Biogenesis of cytochrome c oxidases</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=2665645.4'>Biogenesis of cbb3-type cytochrome c oxidases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=2665645.4'>Biogenesis of c-type cytochromes</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=2665645.4'>Soluble cytochromes and functionally related electron carriers</a> (5)<br></div>
<span style='cursor: pointer;'>&nbsp;Sodium Ion-Coupled Energetics (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_300'><img id='tree_img_0_300' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "300");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAzMxS5ysDAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Stress Response (32)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_300'>
<img id='tree_img_0_301' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "301");'>&nbsp;Osmotic stress (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_301'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=2665645.4'>Osmoregulation</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;Dessication stress (0)<br>
<span style='cursor: pointer;'>&nbsp;Acid stress (0)<br>
<img id='tree_img_0_305' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "305");'>&nbsp;Oxidative stress (22)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_305'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4'>Oxidative stress</a> (17)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rubrerythrin&organism=2665645.4'>Rubrerythrin</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Redox_cycle&organism=2665645.4'>Glutathione: Redox cycle</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Cold shock (0)<br>
<span style='cursor: pointer;'>&nbsp;Heat shock (0)<br>
<img id='tree_img_0_311' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "311");'>&nbsp;Detoxification (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_311'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Uptake_of_selenate_and_selenite&organism=2665645.4'>Uptake of selenate and selenite</a> (1)<br></div>
<img id='tree_img_0_313' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "313");'>&nbsp;Stress Response - no subcategory (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_313'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dimethylarginine_metabolism&organism=2665645.4'>Dimethylarginine metabolism</a> (2)<br></div>
<img id='tree_img_0_315' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "315");'>&nbsp;Periplasmic Stress (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_315'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=2665645.4'>Periplasmic Stress Response</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_317'><img id='tree_img_0_317' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "317");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXM//+B6UABAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Metabolism of Aromatic Compounds (11)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_317'>
<img id='tree_img_0_318' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "318");'>&nbsp;Peripheral pathways for catabolism of aromatic compounds (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_318'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Quinate_degradation&organism=2665645.4'>Quinate degradation</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Anaerobic degradation of aromatic compounds (0)<br>
<img id='tree_img_0_321' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "321");'>&nbsp;Metabolism of central aromatic intermediates (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_321'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=2665645.4'>Catechol branch of beta-ketoadipate pathway</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=2665645.4'>Salicylate and gentisate catabolism</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=2665645.4'>Homogentisate pathway of aromatic compound degradation</a> (3)<br></div>
<img id='tree_img_0_325' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "325");'>&nbsp;Metabolism of Aromatic Compounds - no subcategory (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_325'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Gentisate_degradation&organism=2665645.4'>Gentisate degradation</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_327'><img id='tree_img_0_327' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "327");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXM/8w+OSEXAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Amino Acids and Derivatives (256)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_327'>
<img id='tree_img_0_328' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "328");'>&nbsp;Glutamine, glutamate, aspartate, asparagine; ammonia assimilation (25)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_328'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4'>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a> (21)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=2665645.4'>Glutamate dehydrogenases</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=2665645.4'>Glutamine synthetases</a> (2)<br></div>
<img id='tree_img_0_332' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "332");'>&nbsp;Histidine Metabolism (11)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_332'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=2665645.4'>Histidine Degradation</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=2665645.4'>Histidine Biosynthesis</a> (7)<br></div>
<img id='tree_img_0_335' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "335");'>&nbsp;Arginine; urea cycle, polyamines (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_335'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=2665645.4'>Polyamine Metabolism</a> (5)<br></div>
<img id='tree_img_0_337' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "337");'>&nbsp;Lysine, threonine, methionine, and cysteine (81)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_337'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4'>Methionine Biosynthesis</a> (27)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=2665645.4'>Threonine degradation</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=2665645.4'>Lysine Biosynthesis DAP Pathway, GJO scratch</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=2665645.4'>Methionine Degradation</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=2665645.4'>Threonine and Homoserine Biosynthesis</a> (14)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=2665645.4'>Cysteine Biosynthesis</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=2665645.4'>Lysine Biosynthesis DAP Pathway</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=2665645.4'>Lysine degradation</a> (2)<br></div>
<img id='tree_img_0_346' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "346");'>&nbsp;Amino Acids and Derivatives - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_346'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Creatine_and_Creatinine_Degradation&organism=2665645.4'>Creatine and Creatinine Degradation</a> (1)<br></div>
<img id='tree_img_0_348' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "348");'>&nbsp;Branched-chain amino acids (46)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_348'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=2665645.4'>Isoleucine degradation</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4'>Leucine Degradation and HMG-CoA Metabolism</a> (15)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4'>Branched-Chain Amino Acid Biosynthesis</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=2665645.4'>Valine degradation</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=2665645.4'>Leucine Biosynthesis</a> (6)<br></div>
<span style='cursor: pointer;'>&nbsp;Polyamines (0)<br>
<img id='tree_img_0_355' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "355");'>&nbsp;Aromatic amino acids and derivatives (37)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_355'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=2665645.4'>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4'>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=2665645.4'>Phenylalanine and Tyrosine Branches from Chorismate</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4'>Tryptophan synthesis</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=2665645.4'>Aromatic amino acid degradation</a> (5)<br></div>
<img id='tree_img_0_361' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "361");'>&nbsp;Proline and 4-hydroxyproline (11)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_361'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=2665645.4'>Proline Synthesis</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=2665645.4'>Proline, 4-hydroxyproline uptake and utilization</a> (7)<br></div>
<img id='tree_img_0_364' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "364");'>&nbsp;Alanine, serine, and glycine (39)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_364'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=2665645.4'>Glycine Biosynthesis</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=2665645.4'>Alanine biosynthesis</a> (8)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=2665645.4'>Serine Biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=2665645.4'>Glycine cleavage system</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4'>Glycine and Serine Utilization</a> (16)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_370'><img id='tree_img_0_370' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "370");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//5kDFd+lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Sulfur Metabolism (14)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_370'>
<span style='cursor: pointer;'>&nbsp;Inorganic sulfur assimilation (0)<br>
<img id='tree_img_0_372' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "372");'>&nbsp;Sulfur Metabolism - no subcategory (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_372'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=2665645.4'>Thioredoxin-disulfide reductase</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=2665645.4'>Galactosylceramide and Sulfatide metabolism</a> (5)<br></div>
<span style='cursor: pointer;'>&nbsp;DsrE (0)<br>
<span style='cursor: pointer;'>&nbsp;Organic sulfur assimilation (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_377'><img id='tree_img_0_377' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "377");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZzP8f9clqAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Phosphorus Metabolism (14)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_377'>
<img id='tree_img_0_378' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "378");'>&nbsp;Phosphorus Metabolism - no subcategory (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_378'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=2665645.4'>Phosphate metabolism</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=2665645.4'>Polyphosphate</a> (3)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_381'><img id='tree_img_0_381' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "381");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/mcwrs7b/AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Carbohydrates (166)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_381'>
<img id='tree_img_0_382' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "382");'>&nbsp;Central carbohydrate metabolism (54)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_382'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=2665645.4'>Methylglyoxal Metabolism</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=2665645.4'>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=2665645.4'>Pyruvate Alanine Serine Interconversions</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=2665645.4'>Glyoxylate bypass</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4'>Dehydrogenase complexes</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=2665645.4'>Pentose phosphate pathway</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=2665645.4'>Pyruvate metabolism I: anaplerotic reactions, PEP</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycolate,_glyoxylate_interconversions&organism=2665645.4'>Glycolate, glyoxylate interconversions</a> (1)<br></div>
<img id='tree_img_0_391' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "391");'>&nbsp;Aminosugars (19)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_391'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=2665645.4'>Chitin and N-acetylglucosamine utilization</a> (19)<br></div>
<img id='tree_img_0_393' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "393");'>&nbsp;Di- and oligosaccharides (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_393'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Sucrose_utilization&organism=2665645.4'>Sucrose utilization</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Trehalose_Uptake_and_Utilization&organism=2665645.4'>Trehalose Uptake and Utilization</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=2665645.4'>Lactose and Galactose Uptake and Utilization</a> (8)<br></div>
<span style='cursor: pointer;'>&nbsp;Glycoside hydrolases (0)<br>
<img id='tree_img_0_398' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "398");'>&nbsp;One-carbon Metabolism (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_398'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=2665645.4'>One-carbon metabolism by tetrahydropterines</a> (5)<br></div>
<img id='tree_img_0_400' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "400");'>&nbsp;Organic acids (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_400'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lactate_utilization&organism=2665645.4'>Lactate utilization</a> (3)<br></div>
<img id='tree_img_0_402' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "402");'>&nbsp;Fermentation (24)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_402'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=2665645.4'>Butanol Biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Mixed_acid&organism=2665645.4'>Fermentations: Mixed acid</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Acetolactate_synthase_subunits&organism=2665645.4'>Acetolactate synthase subunits</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=2665645.4'>Fermentations: Lactate</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=2665645.4'>Acetyl-CoA fermentation to Butyrate</a> (5)<br></div>
<span style='cursor: pointer;'>&nbsp;CO2 fixation (0)<br>
<img id='tree_img_0_409' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "409");'>&nbsp;Sugar alcohols (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_409'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=2665645.4'>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Inositol_catabolism&organism=2665645.4'>Inositol catabolism</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Carbohydrates - no subcategory (0)<br>
<img id='tree_img_0_413' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "413");'>&nbsp;Polysaccharides (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_413'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=2665645.4'>Glycogen metabolism</a> (4)<br></div>
<img id='tree_img_0_415' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "415");'>&nbsp;Monosaccharides (35)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_415'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=2665645.4'>Mannose Metabolism</a> (14)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=D-ribose_utilization&organism=2665645.4'>D-ribose utilization</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Xylose_utilization&organism=2665645.4'>Xylose utilization</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=2665645.4'>Deoxyribose and Deoxynucleoside Catabolism</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=2665645.4'>L-Arabinose utilization</a> (9)<br></div></div></div></div>
</td></tr></table></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='1_contents' id='1_content_1'><table class='table_table'><tr><td><input type='button' class='button' value='export to file' onclick='export_table("0", 0, 1, 0);'>&nbsp;&nbsp;<input type='button' class='button' value='clear all filters' onclick='table_reset_filters("0");'></td></tr><tr><td style='width: 100%; text-align: center; vertical-align: middle;'><table style='width: 100%;'>
<tr><td align=center><span class='table_perpage'>display&nbsp;<input type='text' id='table_perpage_0' name='table_perpage_0' size='3' value='15' onkeypress='return check_submit_filter(event, "0");'>&nbsp;items per page</span></td></tr></table></td></tr>
<tr><td style='width: 100%; text-align: center;'><table style='width: 100%;'><tr><td align='left' width='20%'><a href='javascript: table_first("0");' name='table_first_0'>&laquo;first</a>&nbsp;&nbsp;<a href='javascript: table_prev("0");' name='table_prev_0'>&laquo;prev</a></td><td align='center' width='60%'>displaying <span id='table_start_top_0'>1</span> - <span id='table_stop_top_0'>996</span> of <span id='table_total_top_0'>996</span></td><td align='right' width='20%'><a href='javascript: table_next("0");' name='table_next_0'>next&raquo;</a>&nbsp;&nbsp;<a href='javascript: table_last("0");' name='table_last_0'>last&raquo;</a></td></tr></table></td></tr><tr><td><input type='hidden' id='SubsystemTable' value='0' />
<input type='hidden' id='table_data_0' value='Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=2665645.4@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)&subsystem_name=Biotin_biosynthesis@1>Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.470@1>fig|2665645.4.peg.470</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=2665645.4@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin synthase (EC 2.8.1.6)&subsystem_name=Biotin_biosynthesis@1>Biotin synthase (EC 2.8.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4545@1>fig|2665645.4.peg.4545</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=2665645.4@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-ketoacyl-CoA thiolase (EC 2.3.1.16)&subsystem_name=Biotin_biosynthesis@1>3-ketoacyl-CoA thiolase (EC 2.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2315@1>fig|2665645.4.peg.2315</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=2665645.4@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)&subsystem_name=Biotin_biosynthesis@1>Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4537@1>fig|2665645.4.peg.4537</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=2665645.4@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=8-amino-7-oxononanoate synthase (EC 2.3.1.47)&subsystem_name=Biotin_biosynthesis@1>8-amino-7-oxononanoate synthase (EC 2.3.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4530@1>fig|2665645.4.peg.4530</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=2665645.4@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin synthase (EC 2.8.1.6)&subsystem_name=Biotin_biosynthesis_Experimental@1>Biotin synthase (EC 2.8.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4545@1>fig|2665645.4.peg.4545</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=2665645.4@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Competence protein F homolog, phosphoribosyltransferase domain&subsystem_name=Biotin_biosynthesis_Experimental@1>Competence protein F homolog, phosphoribosyltransferase domain</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4549@1>fig|2665645.4.peg.4549</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=2665645.4@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)&subsystem_name=Biotin_biosynthesis_Experimental@1>Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4537@1>fig|2665645.4.peg.4537</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=2665645.4@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=8-amino-7-oxononanoate synthase (EC 2.3.1.47)&subsystem_name=Biotin_biosynthesis_Experimental@1>8-amino-7-oxononanoate synthase (EC 2.3.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4530@1>fig|2665645.4.peg.4530</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=2665645.4@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-regulated outer membrane receptor Omr1&subsystem_name=Thiamin_biosynthesis@1>Thiamin-regulated outer membrane receptor Omr1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3693@1>fig|2665645.4.peg.3693</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4155@1>fig|2665645.4.peg.4155</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=2665645.4@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin pyrophosphokinase (EC 2.7.6.2)&subsystem_name=Thiamin_biosynthesis@1>Thiamin pyrophosphokinase (EC 2.7.6.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3100@1>fig|2665645.4.peg.3100</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=2665645.4@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamine-monophosphate kinase (EC 2.7.4.16)&subsystem_name=Thiamin_biosynthesis@1>Thiamine-monophosphate kinase (EC 2.7.4.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4556@1>fig|2665645.4.peg.4556</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=2665645.4@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)&subsystem_name=Thiamin_biosynthesis@1>Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3680@1>fig|2665645.4.peg.3680</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=2665645.4@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfur carrier protein ThiS&subsystem_name=Thiamin_biosynthesis@1>Sulfur carrier protein ThiS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3676@1>fig|2665645.4.peg.3676</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=2665645.4@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Thiamin_biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3765@1>fig|2665645.4.peg.3765</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=2665645.4@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)&subsystem_name=Cobalamin_synthesis@1>Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5271@1>fig|2665645.4.peg.5271</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=2665645.4@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62)&subsystem_name=Cobalamin_synthesis@1>Adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5274@1>fig|2665645.4.peg.5274</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=2665645.4@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobalamin synthase (EC 2.7.8.26)&subsystem_name=Cobalamin_synthesis@1>Cobalamin synthase (EC 2.7.8.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5272@1>fig|2665645.4.peg.5272</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=2665645.4@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylcobinamide kinase (EC 2.7.1.156)&subsystem_name=Cobalamin_synthesis@1>Adenosylcobinamide kinase (EC 2.7.1.156)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5274@1>fig|2665645.4.peg.5274</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=2665645.4@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)&subsystem_name=Cobalamin_synthesis@1>Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5273@1>fig|2665645.4.peg.5273</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA synthetase (EC 6.1.1.17)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Glutamyl-tRNA synthetase (EC 6.1.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4331@1>fig|2665645.4.peg.4331</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4389@1>fig|2665645.4.peg.4389</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen-III synthase (EC 4.2.1.75)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen-III synthase (EC 4.2.1.75)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4789@1>fig|2665645.4.peg.4789</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Precorrin-2 oxidase (EC 1.3.1.76)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Precorrin-2 oxidase (EC 1.3.1.76)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3790@1>fig|2665645.4.peg.3790</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Porphobilinogen synthase (EC 4.2.1.24)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Porphobilinogen synthase (EC 4.2.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4384@1>fig|2665645.4.peg.4384</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2690@1>fig|2665645.4.peg.2690</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA reductase (EC 1.2.1.70)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Glutamyl-tRNA reductase (EC 1.2.1.70)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4396@1>fig|2665645.4.peg.4396</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Porphobilinogen deaminase (EC 2.5.1.61)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Porphobilinogen deaminase (EC 2.5.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4395@1>fig|2665645.4.peg.4395</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen-III methyltransferase (EC 2.1.1.107)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen-III methyltransferase (EC 2.1.1.107)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.569@1>fig|2665645.4.peg.569</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3791@1>fig|2665645.4.peg.3791</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protoporphyrinogen IX oxidase, novel form, HemJ (EC 1.3.-.-)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Protoporphyrinogen IX oxidase, novel form, HemJ (EC 1.3.-.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4399@1>fig|2665645.4.peg.4399</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4398@1>fig|2665645.4.peg.4398</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=2665645.4@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen III decarboxylase (EC 4.1.1.37)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen III decarboxylase (EC 4.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4393@1>fig|2665645.4.peg.4393</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=2665645.4@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FMN adenylyltransferase (EC 2.7.7.2)&subsystem_name=riboflavin_to_FAD@1>FMN adenylyltransferase (EC 2.7.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4547@1>fig|2665645.4.peg.4547</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=2665645.4@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)&subsystem_name=riboflavin_to_FAD@1>Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4712@1>fig|2665645.4.peg.4712</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=2665645.4@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin kinase (EC 2.7.1.26)&subsystem_name=riboflavin_to_FAD@1>Riboflavin kinase (EC 2.7.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4547@1>fig|2665645.4.peg.4547</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=2665645.4@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)&subsystem_name=riboflavin_to_FAD@1>3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2253@1>fig|2665645.4.peg.2253</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3231@1>fig|2665645.4.peg.3231</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Flavodoxin&organism=2665645.4@1>Flavodoxin</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Flavodoxin 2&subsystem_name=Flavodoxin@1>Flavodoxin 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3300@1>fig|2665645.4.peg.3300</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=2665645.4@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FMN adenylyltransferase (EC 2.7.7.2)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>FMN adenylyltransferase (EC 2.7.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4547@1>fig|2665645.4.peg.4547</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=2665645.4@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4712@1>fig|2665645.4.peg.4712</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=2665645.4@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminohydroxyphosphoribosylaminopyrimidine deaminase (EC 3.5.4.26)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Diaminohydroxyphosphoribosylaminopyrimidine deaminase (EC 3.5.4.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.6@1>fig|2665645.4.peg.6</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=2665645.4@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5145@1>fig|2665645.4.peg.5145</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=2665645.4@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-amino-6-(5-phosphoribosylamino)uracil reductase (EC 1.1.1.193)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>5-amino-6-(5-phosphoribosylamino)uracil reductase (EC 1.1.1.193)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.6@1>fig|2665645.4.peg.6</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=2665645.4@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin kinase (EC 2.7.1.26)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Riboflavin kinase (EC 2.7.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4547@1>fig|2665645.4.peg.4547</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=2665645.4@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase II (EC 3.5.4.25)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>GTP cyclohydrolase II (EC 3.5.4.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3231@1>fig|2665645.4.peg.3231</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=2665645.4@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2253@1>fig|2665645.4.peg.2253</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3231@1>fig|2665645.4.peg.3231</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=2665645.4@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2773@1>fig|2665645.4.peg.2773</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4497@1>fig|2665645.4.peg.4497</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=2665645.4@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4713@1>fig|2665645.4.peg.4713</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=2665645.4@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyridoxamine 5@1-phosphate oxidase (EC 1.4.3.5)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Pyridoxamine 5@1-phosphate oxidase (EC 1.4.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.706@1>fig|2665645.4.peg.706</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=2665645.4@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyridoxine 5@1-phosphate synthase (EC 2.6.99.2)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Pyridoxine 5@1-phosphate synthase (EC 2.6.99.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3584@1>fig|2665645.4.peg.3584</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=2665645.4@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2772@1>fig|2665645.4.peg.2772</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=2665645.4@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.653@1>fig|2665645.4.peg.653</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5129@1>fig|2665645.4.peg.5129</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5353@1>fig|2665645.4.peg.5353</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=2665645.4@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3765@1>fig|2665645.4.peg.3765</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate synthetase (EC 2.5.1.72)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Quinolinate synthetase (EC 2.5.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.480@1>fig|2665645.4.peg.480</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2522@1>fig|2665645.4.peg.2522</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD kinase (EC 2.7.1.23)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>NAD kinase (EC 2.7.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3586@1>fig|2665645.4.peg.3586</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinamide phosphoribosyltransferase (EC 2.4.2.12)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Nicotinamide phosphoribosyltransferase (EC 2.4.2.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4608@1>fig|2665645.4.peg.4608</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxyanthranilate 3,4-dioxygenase (EC 1.13.11.6)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>3-hydroxyanthranilate 3,4-dioxygenase (EC 1.13.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4113@1>fig|2665645.4.peg.4113</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD synthetase (EC 6.3.1.5)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>NAD synthetase (EC 6.3.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3484@1>fig|2665645.4.peg.3484</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan 2,3-dioxygenase (EC 1.13.11.11)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Tryptophan 2,3-dioxygenase (EC 1.13.11.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4624@1>fig|2665645.4.peg.4624</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-aspartate oxidase (EC 1.4.3.16)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>L-aspartate oxidase (EC 1.4.3.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.481@1>fig|2665645.4.peg.481</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosyl nicotinamide transporter, PnuC-like&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Ribosyl nicotinamide transporter, PnuC-like</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3696@1>fig|2665645.4.peg.3696</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kynurenine 3-monooxygenase (EC 1.14.13.9)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Kynurenine 3-monooxygenase (EC 1.14.13.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4862@1>fig|2665645.4.peg.4862</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4610@1>fig|2665645.4.peg.4610</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=2665645.4@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kynureninase (EC 3.7.1.3)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Kynureninase (EC 3.7.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4850@1>fig|2665645.4.peg.4850</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Folate_biosynthesis_cluster@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1666@1>fig|2665645.4.peg.1666</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydropteroate synthase (EC 2.5.1.15)&subsystem_name=Folate_biosynthesis_cluster@1>Dihydropteroate synthase (EC 2.5.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2532@1>fig|2665645.4.peg.2532</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)&subsystem_name=Folate_biosynthesis_cluster@1>tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3154@1>fig|2665645.4.peg.3154</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)&subsystem_name=Folate_biosynthesis_cluster@1>Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4942@1>fig|2665645.4.peg.4942</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Folate_biosynthesis_cluster@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4515@1>fig|2665645.4.peg.4515</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Folate_biosynthesis_cluster@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3008@1>fig|2665645.4.peg.3008</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Folate_biosynthesis_cluster@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4516@1>fig|2665645.4.peg.4516</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)&subsystem_name=Folate_biosynthesis_cluster@1>2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3562@1>fig|2665645.4.peg.3562</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=2665645.4@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroneopterin aldolase (EC 4.1.2.25)&subsystem_name=Folate_biosynthesis_cluster@1>Dihydroneopterin aldolase (EC 4.1.2.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4323@1>fig|2665645.4.peg.4323</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Folate_Biosynthesis@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5206@1>fig|2665645.4.peg.5206</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Folylpolyglutamate synthase (EC 6.3.2.17)&subsystem_name=Folate_Biosynthesis@1>Folylpolyglutamate synthase (EC 6.3.2.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4901@1>fig|2665645.4.peg.4901</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydropteroate synthase (EC 2.5.1.15)&subsystem_name=Folate_Biosynthesis@1>Dihydropteroate synthase (EC 2.5.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2532@1>fig|2665645.4.peg.2532</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=Folate_Biosynthesis@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4631@1>fig|2665645.4.peg.4631</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate reductase (EC 1.5.1.3)&subsystem_name=Folate_Biosynthesis@1>Dihydrofolate reductase (EC 1.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.699@1>fig|2665645.4.peg.699</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3832@1>fig|2665645.4.peg.3832</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate synthase (EC 6.3.2.12)&subsystem_name=Folate_Biosynthesis@1>Dihydrofolate synthase (EC 6.3.2.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4901@1>fig|2665645.4.peg.4901</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Folate_Biosynthesis@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3008@1>fig|2665645.4.peg.3008</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Folate_Biosynthesis@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3155@1>fig|2665645.4.peg.3155</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidylate synthase (EC 2.1.1.45)&subsystem_name=Folate_Biosynthesis@1>Thymidylate synthase (EC 2.1.1.45)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3829@1>fig|2665645.4.peg.3829</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)&subsystem_name=Folate_Biosynthesis@1>2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3562@1>fig|2665645.4.peg.3562</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroneopterin aldolase (EC 4.1.2.25)&subsystem_name=Folate_Biosynthesis@1>Dihydroneopterin aldolase (EC 4.1.2.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4323@1>fig|2665645.4.peg.4323</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=2665645.4@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Folate_Biosynthesis@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.199@1>fig|2665645.4.peg.199</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alcohol dehydrogenase (EC 1.1.1.1)&subsystem_name=5-FCL-like_protein@1>Alcohol dehydrogenase (EC 1.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2402@1>fig|2665645.4.peg.2402</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)&subsystem_name=5-FCL-like_protein@1>Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1192@1>fig|2665645.4.peg.1192</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3094@1>fig|2665645.4.peg.3094</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formyltetrahydrofolate deformylase (EC 3.5.1.10)&subsystem_name=5-FCL-like_protein@1>Formyltetrahydrofolate deformylase (EC 3.5.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2959@1>fig|2665645.4.peg.2959</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=5-FCL-like_protein@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4631@1>fig|2665645.4.peg.4631</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate reductase (EC 1.5.1.3)&subsystem_name=5-FCL-like_protein@1>Dihydrofolate reductase (EC 1.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.699@1>fig|2665645.4.peg.699</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3832@1>fig|2665645.4.peg.3832</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamine-monophosphate kinase (EC 2.7.4.16)&subsystem_name=5-FCL-like_protein@1>Thiamine-monophosphate kinase (EC 2.7.4.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4556@1>fig|2665645.4.peg.4556</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=5-FCL-like_protein@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2443@1>fig|2665645.4.peg.2443</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)&subsystem_name=5-FCL-like_protein@1>Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5113@1>fig|2665645.4.peg.5113</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)&subsystem_name=5-FCL-like_protein@1>Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4173@1>fig|2665645.4.peg.4173</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)&subsystem_name=5-FCL-like_protein@1>Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3842@1>fig|2665645.4.peg.3842</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)&subsystem_name=5-FCL-like_protein@1>Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3680@1>fig|2665645.4.peg.3680</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=5-FCL-like_protein@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3783@1>fig|2665645.4.peg.3783</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=5-FCL-like_protein@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3735@1>fig|2665645.4.peg.3735</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=2665645.4@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=5-FCL-like_protein@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.535@1>fig|2665645.4.peg.535</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2413@1>fig|2665645.4.peg.2413</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3159@1>fig|2665645.4.peg.3159</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=2665645.4@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkaline phosphatase (EC 3.1.3.1)&subsystem_name=Pterin_synthesis_related_cluster@1>Alkaline phosphatase (EC 3.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.283@1>fig|2665645.4.peg.283</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2446@1>fig|2665645.4.peg.2446</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3528@1>fig|2665645.4.peg.3528</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5350@1>fig|2665645.4.peg.5350</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=2665645.4@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Pterin_synthesis_related_cluster@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3008@1>fig|2665645.4.peg.3008</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=2665645.4@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)&subsystem_name=Pterin_synthesis_related_cluster@1>Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5330@1>fig|2665645.4.peg.5330</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=2665645.4@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Pterin_synthesis_related_cluster@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.195@1>fig|2665645.4.peg.195</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=2665645.4@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6-pyruvoyl tetrahydrobiopterin synthase (EC 4.2.3.12)&subsystem_name=Pterin_synthesis_related_cluster@1>6-pyruvoyl tetrahydrobiopterin synthase (EC 4.2.3.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5090@1>fig|2665645.4.peg.5090</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=2665645.4@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1666@1>fig|2665645.4.peg.1666</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=2665645.4@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.487@1>fig|2665645.4.peg.487</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=2665645.4@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4516@1>fig|2665645.4.peg.4516</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=2665645.4@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3894@1>fig|2665645.4.peg.3894</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=2665645.4@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4515@1>fig|2665645.4.peg.4515</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1666@1>fig|2665645.4.peg.1666</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)&subsystem_name=Coenzyme_A_Biosynthesis@1>3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.487@1>fig|2665645.4.peg.487</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dephospho-CoA kinase (EC 2.7.1.24)&subsystem_name=Coenzyme_A_Biosynthesis@1>Dephospho-CoA kinase (EC 2.7.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2492@1>fig|2665645.4.peg.2492</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)&subsystem_name=Coenzyme_A_Biosynthesis@1>Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3894@1>fig|2665645.4.peg.3894</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Coenzyme_A_Biosynthesis@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4515@1>fig|2665645.4.peg.4515</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-dehydropantoate 2-reductase (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis@1>2-dehydropantoate 2-reductase (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.738@1>fig|2665645.4.peg.738</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantetheine adenylyltransferase (EC 2.7.7.3)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantetheine adenylyltransferase (EC 2.7.7.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2596@1>fig|2665645.4.peg.2596</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantothenoylcysteine synthetase (EC 6.3.2.5)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantothenoylcysteine synthetase (EC 6.3.2.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2497@1>fig|2665645.4.peg.2497</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantothenoylcysteine decarboxylase (EC 4.1.1.36)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantothenoylcysteine decarboxylase (EC 4.1.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2497@1>fig|2665645.4.peg.2497</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=2665645.4@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Coenzyme_A_Biosynthesis@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4516@1>fig|2665645.4.peg.4516</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=2665645.4@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5034@1>fig|2665645.4.peg.5034</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=2665645.4@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5018@1>fig|2665645.4.peg.5018</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=2665645.4@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5033@1>fig|2665645.4.peg.5033</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=2665645.4@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)&subsystem_name=dTDP-rhamnose_synthesis@1>Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2870@1>fig|2665645.4.peg.2870</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4269@1>fig|2665645.4.peg.4269</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5035@1>fig|2665645.4.peg.5035</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Streptococcal_Hyaluronic_Acid_Capsule&organism=2665645.4@1>Streptococcal Hyaluronic Acid Capsule</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hyaluronan synthase (EC 2.4.1.212)&subsystem_name=Streptococcal_Hyaluronic_Acid_Capsule@1>Hyaluronan synthase (EC 2.4.1.212)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.268@1>fig|2665645.4.peg.268</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=2665645.4@1>CMP-N-acetylneuraminate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylneuraminate synthase (EC 2.5.1.56)&subsystem_name=CMP-N-acetylneuraminate_Biosynthesis@1>N-acetylneuraminate synthase (EC 2.5.1.56)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5053@1>fig|2665645.4.peg.5053</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=2665645.4@1>CMP-N-acetylneuraminate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)&subsystem_name=CMP-N-acetylneuraminate_Biosynthesis@1>UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4296@1>fig|2665645.4.peg.4296</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5021@1>fig|2665645.4.peg.5021</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5049@1>fig|2665645.4.peg.5049</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polysaccharide_deacetylases&organism=2665645.4@1>Polysaccharide deacetylases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidoglycan N-acetylglucosamine deacetylase (EC 3.5.1.-)&subsystem_name=Polysaccharide_deacetylases@1>Peptidoglycan N-acetylglucosamine deacetylase (EC 3.5.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4249@1>fig|2665645.4.peg.4249</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polysaccharide_deacetylases&organism=2665645.4@1>Polysaccharide deacetylases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polysaccharide deacetylase&subsystem_name=Polysaccharide_deacetylases@1>Polysaccharide deacetylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.498@1>fig|2665645.4.peg.498</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3657@1>fig|2665645.4.peg.3657</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=2665645.4@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=Rhamnose_containing_glycans@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3551@1>fig|2665645.4.peg.3551</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5012@1>fig|2665645.4.peg.5012</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=2665645.4@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5034@1>fig|2665645.4.peg.5034</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=2665645.4@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5018@1>fig|2665645.4.peg.5018</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=2665645.4@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5033@1>fig|2665645.4.peg.5033</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=2665645.4@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate cytidylyltransferase (EC 2.7.7.39)&subsystem_name=Rhamnose_containing_glycans@1>Glycerol-3-phosphate cytidylyltransferase (EC 2.7.7.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5016@1>fig|2665645.4.peg.5016</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=2665645.4@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)&subsystem_name=Rhamnose_containing_glycans@1>Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2870@1>fig|2665645.4.peg.2870</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4269@1>fig|2665645.4.peg.4269</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5035@1>fig|2665645.4.peg.5035</a>@~Cell Wall and Capsule@^Gram-Negative cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lipoprotein_sorting_system&organism=2665645.4@1>Lipoprotein sorting system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane lipoprotein carrier protein LolA&subsystem_name=Lipoprotein_sorting_system@1>Outer membrane lipoprotein carrier protein LolA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3233@1>fig|2665645.4.peg.3233</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=2665645.4@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)&subsystem_name=Murein_Hydrolases@1>N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.141@1>fig|2665645.4.peg.141</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.659@1>fig|2665645.4.peg.659</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=2665645.4@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)&subsystem_name=Murein_Hydrolases@1>D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.507@1>fig|2665645.4.peg.507</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4857@1>fig|2665645.4.peg.4857</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=2665645.4@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Murein_Hydrolases@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4116@1>fig|2665645.4.peg.4116</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=2665645.4@1>Recycling of Peptidoglycan Amino Acids</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)&subsystem_name=Recycling_of_Peptidoglycan_Amino_Acids@1>N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.141@1>fig|2665645.4.peg.141</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.659@1>fig|2665645.4.peg.659</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=2665645.4@1>Recycling of Peptidoglycan Amino Acids</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Recycling_of_Peptidoglycan_Amino_Acids@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4116@1>fig|2665645.4.peg.4116</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=2665645.4@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multicopper oxidase&subsystem_name=Copper_homeostasis@1>Multicopper oxidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5170@1>fig|2665645.4.peg.5170</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=2665645.4@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-translocating P-type ATPase (EC 3.6.3.4)&subsystem_name=Copper_homeostasis@1>Copper-translocating P-type ATPase (EC 3.6.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2359@1>fig|2665645.4.peg.2359</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2698@1>fig|2665645.4.peg.2698</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5171@1>fig|2665645.4.peg.5171</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=2665645.4@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multidrug resistance transporter, Bcr/CflA family&subsystem_name=Copper_homeostasis@1>Multidrug resistance transporter, Bcr/CflA family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1450@1>fig|2665645.4.peg.1450</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1794@1>fig|2665645.4.peg.1794</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=2665645.4@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-binding heavy metal response regulator&subsystem_name=Cobalt-zinc-cadmium_resistance@1>DNA-binding heavy metal response regulator</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1825@1>fig|2665645.4.peg.1825</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=2665645.4@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobalt-zinc-cadmium resistance protein&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Cobalt-zinc-cadmium resistance protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5217@1>fig|2665645.4.peg.5217</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=2665645.4@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobalt-zinc-cadmium resistance protein CzcA&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Cobalt-zinc-cadmium resistance protein CzcA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2361@1>fig|2665645.4.peg.2361</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=2665645.4@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Probable Co/Zn/Cd efflux system membrane fusion protein&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Probable Co/Zn/Cd efflux system membrane fusion protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1093@1>fig|2665645.4.peg.1093</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=2665645.4@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional regulator, MerR family&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Transcriptional regulator, MerR family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5305@1>fig|2665645.4.peg.5305</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=2665645.4@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cation efflux system protein CusA&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Cation efflux system protein CusA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2361@1>fig|2665645.4.peg.2361</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mercuric_reductase&organism=2665645.4@1>Mercuric reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=PF00070 family, FAD-dependent NAD(P)-disulphide oxidoreductase&subsystem_name=Mercuric_reductase@1>PF00070 family, FAD-dependent NAD(P)-disulphide oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3183@1>fig|2665645.4.peg.3183</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aminoglycoside_adenylyltransferases&organism=2665645.4@1>Aminoglycoside adenylyltransferases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminoglycoside N6@1-acetyltransferase (EC 2.3.1.82)&subsystem_name=Aminoglycoside_adenylyltransferases@1>Aminoglycoside N6@1-acetyltransferase (EC 2.3.1.82)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4642@1>fig|2665645.4.peg.4642</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=2665645.4@1>Resistance to fluoroquinolones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit B (EC 5.99.1.3)&subsystem_name=Resistance_to_fluoroquinolones@1>DNA gyrase subunit B (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3015@1>fig|2665645.4.peg.3015</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=2665645.4@1>Resistance to fluoroquinolones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit A (EC 5.99.1.3)&subsystem_name=Resistance_to_fluoroquinolones@1>DNA gyrase subunit A (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2629@1>fig|2665645.4.peg.2629</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=2665645.4@1>Resistance to fluoroquinolones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Efflux pump Lde&subsystem_name=Resistance_to_fluoroquinolones@1>Efflux pump Lde</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1395@1>fig|2665645.4.peg.1395</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis:_copper_tolerance&organism=2665645.4@1>Copper homeostasis: copper tolerance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytoplasmic copper homeostasis protein CutC&subsystem_name=Copper_homeostasis:_copper_tolerance@1>Cytoplasmic copper homeostasis protein CutC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5103@1>fig|2665645.4.peg.5103</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Beta-lactamase&organism=2665645.4@1>Beta-lactamase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Metal-dependent hydrolases of the beta-lactamase superfamily I&subsystem_name=Beta-lactamase@1>Metal-dependent hydrolases of the beta-lactamase superfamily I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2799@1>fig|2665645.4.peg.2799</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=2665645.4@1>Multidrug Resistance Efflux Pumps</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription regulator of multidrug efflux pump operon, TetR (AcrR) family&subsystem_name=Multidrug_Resistance_Efflux_Pumps@1>Transcription regulator of multidrug efflux pump operon, TetR (AcrR) family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3058@1>fig|2665645.4.peg.3058</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=2665645.4@1>Multidrug Resistance Efflux Pumps</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acriflavin resistance protein&subsystem_name=Multidrug_Resistance_Efflux_Pumps@1>Acriflavin resistance protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.961@1>fig|2665645.4.peg.961</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=2665645.4@1>Multidrug Resistance Efflux Pumps</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multidrug-efflux transporter, major facilitator superfamily (MFS) (TC 2.A.1)&subsystem_name=Multidrug_Resistance_Efflux_Pumps@1>Multidrug-efflux transporter, major facilitator superfamily (MFS) (TC 2.A.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1395@1>fig|2665645.4.peg.1395</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_chromium_compounds&organism=2665645.4@1>Resistance to chromium compounds</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chromate transport protein ChrA&subsystem_name=Resistance_to_chromium_compounds@1>Chromate transport protein ChrA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1077@1>fig|2665645.4.peg.1077</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1078@1>fig|2665645.4.peg.1078</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=2665645.4@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S7p (S5e)&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>SSU ribosomal protein S7p (S5e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4970@1>fig|2665645.4.peg.4970</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=2665645.4@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4971@1>fig|2665645.4.peg.4971</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=2665645.4@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Tu&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>Translation elongation factor Tu</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3287@1>fig|2665645.4.peg.3287</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=2665645.4@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S12p (S23e)&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>SSU ribosomal protein S12p (S23e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4969@1>fig|2665645.4.peg.4969</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=2665645.4@1>Mycobacterium virulence operon involved in DNA transcription</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)&subsystem_name=Mycobacterium_virulence_operon_involved_in_DNA_transcription@1>DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3279@1>fig|2665645.4.peg.3279</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=2665645.4@1>Mycobacterium virulence operon involved in DNA transcription</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)&subsystem_name=Mycobacterium_virulence_operon_involved_in_DNA_transcription@1>DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3280@1>fig|2665645.4.peg.3280</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=2665645.4@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate synthetase (EC 2.5.1.72)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>Quinolinate synthetase (EC 2.5.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.480@1>fig|2665645.4.peg.480</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=2665645.4@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2522@1>fig|2665645.4.peg.2522</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=2665645.4@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-aspartate oxidase (EC 1.4.3.16)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>L-aspartate oxidase (EC 1.4.3.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.481@1>fig|2665645.4.peg.481</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Listeria_surface_proteins:_Internalin-like_proteins&organism=2665645.4@1>Listeria surface proteins: Internalin-like proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=internalin, putative&subsystem_name=Listeria_surface_proteins:_Internalin-like_proteins@1>internalin, putative</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1481@1>fig|2665645.4.peg.1481</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=2665645.4@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L35p&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>LSU ribosomal protein L35p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5345@1>fig|2665645.4.peg.5345</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=2665645.4@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 3&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>Translation initiation factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5346@1>fig|2665645.4.peg.5346</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=2665645.4@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L20p&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>LSU ribosomal protein L20p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5344@1>fig|2665645.4.peg.5344</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium-transporting ATPase A chain (EC 3.6.3.12) (TC 3.A.3.7.1)&subsystem_name=Potassium_homeostasis@1>Potassium-transporting ATPase A chain (EC 3.6.3.12) (TC 3.A.3.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3241@1>fig|2665645.4.peg.3241</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium channel protein&subsystem_name=Potassium_homeostasis@1>Potassium channel protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1507@1>fig|2665645.4.peg.1507</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium-transporting ATPase C chain (EC 3.6.3.12) (TC 3.A.3.7.1)&subsystem_name=Potassium_homeostasis@1>Potassium-transporting ATPase C chain (EC 3.6.3.12) (TC 3.A.3.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3243@1>fig|2665645.4.peg.3243</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Osmosensitive K+ channel histidine kinase KdpD (EC 2.7.3.-)&subsystem_name=Potassium_homeostasis@1>Osmosensitive K+ channel histidine kinase KdpD (EC 2.7.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2037@1>fig|2665645.4.peg.2037</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium voltage-gated channel subfamily KQT&subsystem_name=Potassium_homeostasis@1>Potassium voltage-gated channel subfamily KQT</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2482@1>fig|2665645.4.peg.2482</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=POTASSIUM/PROTON ANTIPORTER ROSB&subsystem_name=Potassium_homeostasis@1>POTASSIUM/PROTON ANTIPORTER ROSB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.51@1>fig|2665645.4.peg.51</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium-transporting ATPase B chain (EC 3.6.3.12) (TC 3.A.3.7.1)&subsystem_name=Potassium_homeostasis@1>Potassium-transporting ATPase B chain (EC 3.6.3.12) (TC 3.A.3.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3242@1>fig|2665645.4.peg.3242</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium efflux system KefA protein&subsystem_name=Potassium_homeostasis@1>Potassium efflux system KefA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.263@1>fig|2665645.4.peg.263</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3165@1>fig|2665645.4.peg.3165</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Large-conductance mechanosensitive channel&subsystem_name=Potassium_homeostasis@1>Large-conductance mechanosensitive channel</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3992@1>fig|2665645.4.peg.3992</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)&subsystem_name=Potassium_homeostasis@1>FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.77@1>fig|2665645.4.peg.77</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5320@1>fig|2665645.4.peg.5320</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=2665645.4@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kup system potassium uptake protein&subsystem_name=Potassium_homeostasis@1>Kup system potassium uptake protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.424@1>fig|2665645.4.peg.424</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=2665645.4@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=PaaD-like protein (DUF59) involved in Fe-S cluster assembly&subsystem_name=Iron-sulfur_cluster_assembly@1>PaaD-like protein (DUF59) involved in Fe-S cluster assembly</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4149@1>fig|2665645.4.peg.4149</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=2665645.4@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=probable iron binding protein from the HesB_IscA_SufA family&subsystem_name=Iron-sulfur_cluster_assembly@1>probable iron binding protein from the HesB_IscA_SufA family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4139@1>fig|2665645.4.peg.4139</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=2665645.4@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Iron-sulfur_cluster_assembly@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3988@1>fig|2665645.4.peg.3988</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=2665645.4@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=Iron-sulfur_cluster_assembly@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1292@1>fig|2665645.4.peg.1292</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1387@1>fig|2665645.4.peg.1387</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1706@1>fig|2665645.4.peg.1706</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2185@1>fig|2665645.4.peg.2185</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2727@1>fig|2665645.4.peg.2727</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=2665645.4@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly protein SufB&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly protein SufB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4140@1>fig|2665645.4.peg.4140</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=2665645.4@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly protein SufD&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly protein SufD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4143@1>fig|2665645.4.peg.4143</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=2665645.4@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly ATPase protein SufC&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly ATPase protein SufC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4142@1>fig|2665645.4.peg.4142</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Single-Rhodanese-domain_proteins&organism=2665645.4@1>Single-Rhodanese-domain proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rhodanese domain protein, Enterobacterial subgroup, YceA homolog&subsystem_name=Single-Rhodanese-domain_proteins@1>Rhodanese domain protein, Enterobacterial subgroup, YceA homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4414@1>fig|2665645.4.peg.4414</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphoglycerate_mutase_protein_family&organism=2665645.4@1>Phosphoglycerate mutase protein family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)&subsystem_name=Phosphoglycerate_mutase_protein_family@1>Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5271@1>fig|2665645.4.peg.5271</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DedA_family_of_inner_membrane_proteins&organism=2665645.4@1>DedA family of inner membrane proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DedA protein&subsystem_name=DedA_family_of_inner_membrane_proteins@1>DedA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3848@1>fig|2665645.4.peg.3848</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Muconate_lactonizing_enzyme_family&organism=2665645.4@1>Muconate lactonizing enzyme family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylbenzoate synthase (EC 4.2.1.113)&subsystem_name=Muconate_lactonizing_enzyme_family@1>O-succinylbenzoate synthase (EC 4.2.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1699@1>fig|2665645.4.peg.1699</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Bacillus_subtilis_scratch_-_gjo&organism=2665645.4@1>Bacillus subtilis scratch - gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ku domain protein&subsystem_name=Bacillus_subtilis_scratch_-_gjo@1>Ku domain protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2222@1>fig|2665645.4.peg.2222</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Bacillus_subtilis_scratch_-_gjo&organism=2665645.4@1>Bacillus subtilis scratch - gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD&subsystem_name=Bacillus_subtilis_scratch_-_gjo@1>ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2221@1>fig|2665645.4.peg.2221</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=2665645.4@1>Broadly distributed proteins not in subsystems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=YpfJ protein, zinc metalloprotease superfamily&subsystem_name=Broadly_distributed_proteins_not_in_subsystems@1>YpfJ protein, zinc metalloprotease superfamily</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.482@1>fig|2665645.4.peg.482</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=2665645.4@1>Broadly distributed proteins not in subsystems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UPF0225 protein YchJ&subsystem_name=Broadly_distributed_proteins_not_in_subsystems@1>UPF0225 protein YchJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.898@1>fig|2665645.4.peg.898</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraN&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1221@1>fig|2665645.4.peg.1221</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraG&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1213@1>fig|2665645.4.peg.1213</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraM&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraM</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1220@1>fig|2665645.4.peg.1220</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraJ&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1217@1>fig|2665645.4.peg.1217</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraK&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraK</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1218@1>fig|2665645.4.peg.1218</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraI&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraI</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1214@1>fig|2665645.4.peg.1214</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1215@1>fig|2665645.4.peg.1215</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1216@1>fig|2665645.4.peg.1216</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraE&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1211@1>fig|2665645.4.peg.1211</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative conjugative transposon mobilization protein BF0132&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Putative conjugative transposon mobilization protein BF0132</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1001@1>fig|2665645.4.peg.1001</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative mobilization protein BF0133&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Putative mobilization protein BF0133</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1004@1>fig|2665645.4.peg.1004</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=2665645.4@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraF&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1212@1>fig|2665645.4.peg.1212</a>@~Phages, Prophages, Transposable elements, Plasmids@^Phages, Prophages@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phage_packaging_machinery&organism=2665645.4@1>Phage packaging machinery</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phage terminase, large subunit&subsystem_name=Phage_packaging_machinery@1>Phage terminase, large subunit</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.387@1>fig|2665645.4.peg.387</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=2665645.4@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Twin-arginine translocation protein TatC&subsystem_name=Twin-arginine_translocation_system@1>Twin-arginine translocation protein TatC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5077@1>fig|2665645.4.peg.5077</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=2665645.4@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Twin-arginine translocation protein TatA&subsystem_name=Twin-arginine_translocation_system@1>Twin-arginine translocation protein TatA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3505@1>fig|2665645.4.peg.3505</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4308@1>fig|2665645.4.peg.4308</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=2665645.4@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribonuclease TatD&subsystem_name=Twin-arginine_translocation_system@1>Deoxyribonuclease TatD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.457@1>fig|2665645.4.peg.457</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Magnesium_transport&organism=2665645.4@1>Magnesium transport</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Magnesium and cobalt transport protein CorA&subsystem_name=Magnesium_transport@1>Magnesium and cobalt transport protein CorA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2569@1>fig|2665645.4.peg.2569</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5117@1>fig|2665645.4.peg.5117</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_Transport_System&organism=2665645.4@1>Copper Transport System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-translocating P-type ATPase (EC 3.6.3.4)&subsystem_name=Copper_Transport_System@1>Copper-translocating P-type ATPase (EC 3.6.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2359@1>fig|2665645.4.peg.2359</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2698@1>fig|2665645.4.peg.2698</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5171@1>fig|2665645.4.peg.5171</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_transport_and_blue_copper_proteins&organism=2665645.4@1>Copper transport and blue copper proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Azurin&subsystem_name=Copper_transport_and_blue_copper_proteins@1>Azurin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2830@1>fig|2665645.4.peg.2830</a>@~Membrane Transport@^Uni- Sym- and Antiporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters&organism=2665645.4@1>NhaA, NhaD and Sodium-dependent phosphate transporters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Na+/H+ antiporter NhaA type&subsystem_name=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters@1>Na+/H+ antiporter NhaA type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4495@1>fig|2665645.4.peg.4495</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Choline_Transport&organism=2665645.4@1>Choline Transport</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sodium-Choline Symporter&subsystem_name=Choline_Transport@1>Sodium-Choline Symporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2345@1>fig|2665645.4.peg.2345</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=TPR domain protein, putative component of TonB system&subsystem_name=Ton_and_Tol_transport_systems@1>TPR domain protein, putative component of TonB system</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2860@1>fig|2665645.4.peg.2860</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric siderophore transport system, periplasmic binding protein TonB&subsystem_name=Ton_and_Tol_transport_systems@1>Ferric siderophore transport system, periplasmic binding protein TonB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3763@1>fig|2665645.4.peg.3763</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-chelator utilization protein&subsystem_name=Ton_and_Tol_transport_systems@1>Iron-chelator utilization protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5191@1>fig|2665645.4.peg.5191</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric siderophore transport system, biopolymer transport protein ExbB&subsystem_name=Ton_and_Tol_transport_systems@1>Ferric siderophore transport system, biopolymer transport protein ExbB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4902@1>fig|2665645.4.peg.4902</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=TonB-dependent receptor&subsystem_name=Ton_and_Tol_transport_systems@1>TonB-dependent receptor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.60@1>fig|2665645.4.peg.60</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.977@1>fig|2665645.4.peg.977</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1065@1>fig|2665645.4.peg.1065</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1372@1>fig|2665645.4.peg.1372</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1393@1>fig|2665645.4.peg.1393</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1520@1>fig|2665645.4.peg.1520</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1636@1>fig|2665645.4.peg.1636</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1638@1>fig|2665645.4.peg.1638</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2147@1>fig|2665645.4.peg.2147</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2197@1>fig|2665645.4.peg.2197</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2257@1>fig|2665645.4.peg.2257</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2768@1>fig|2665645.4.peg.2768</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3343@1>fig|2665645.4.peg.3343</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3779@1>fig|2665645.4.peg.3779</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3811@1>fig|2665645.4.peg.3811</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3940@1>fig|2665645.4.peg.3940</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3945@1>fig|2665645.4.peg.3945</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4070@1>fig|2665645.4.peg.4070</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4409@1>fig|2665645.4.peg.4409</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4431@1>fig|2665645.4.peg.4431</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4521@1>fig|2665645.4.peg.4521</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4888@1>fig|2665645.4.peg.4888</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5107@1>fig|2665645.4.peg.5107</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane lipoprotein omp16 precursor&subsystem_name=Ton_and_Tol_transport_systems@1>Outer membrane lipoprotein omp16 precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2507@1>fig|2665645.4.peg.2507</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2935@1>fig|2665645.4.peg.2935</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3003@1>fig|2665645.4.peg.3003</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biopolymer transport protein ExbD/TolR&subsystem_name=Ton_and_Tol_transport_systems@1>Biopolymer transport protein ExbD/TolR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1630@1>fig|2665645.4.peg.1630</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1631@1>fig|2665645.4.peg.1631</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4865@1>fig|2665645.4.peg.4865</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4866@1>fig|2665645.4.peg.4866</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4903@1>fig|2665645.4.peg.4903</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxybenzoyl-CoA thioesterase family active site&subsystem_name=Ton_and_Tol_transport_systems@1>4-hydroxybenzoyl-CoA thioesterase family active site</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2@1>fig|2665645.4.peg.2</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Ton_and_Tol_transport_systems@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2477@1>fig|2665645.4.peg.2477</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=2665645.4@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=MotA/TolQ/ExbB proton channel family protein&subsystem_name=Ton_and_Tol_transport_systems@1>MotA/TolQ/ExbB proton channel family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4864@1>fig|2665645.4.peg.4864</a>@~Membrane Transport@^Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP)@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Curli_production&organism=2665645.4@1>Curli production</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Minor curlin subunit CsgB, nucleation component of curlin monomers&subsystem_name=Curli_production@1>Minor curlin subunit CsgB, nucleation component of curlin monomers</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2918@1>fig|2665645.4.peg.2918</a>@~Membrane Transport@^Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP)@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Curli_production&organism=2665645.4@1>Curli production</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Curli production assembly/transport component CsgF&subsystem_name=Curli_production@1>Curli production assembly/transport component CsgF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2914@1>fig|2665645.4.peg.2914</a>@~Membrane Transport@^Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP)@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Curli_production&organism=2665645.4@1>Curli production</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Curli production assembly/transport component CsgG&subsystem_name=Curli_production@1>Curli production assembly/transport component CsgG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2913@1>fig|2665645.4.peg.2913</a>@~Iron acquisition and metabolism@^Iron acquisition and metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Hemin_transport_system&organism=2665645.4@1>Hemin transport system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric siderophore transport system, periplasmic binding protein TonB&subsystem_name=Hemin_transport_system@1>Ferric siderophore transport system, periplasmic binding protein TonB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3763@1>fig|2665645.4.peg.3763</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=2665645.4@1>mnm5U34 biosynthesis bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)&subsystem_name=mnm5U34_biosynthesis_bacteria@1>tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.434@1>fig|2665645.4.peg.434</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=2665645.4@1>mnm5U34 biosynthesis bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=mnm5U34_biosynthesis_bacteria@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1292@1>fig|2665645.4.peg.1292</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1387@1>fig|2665645.4.peg.1387</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1706@1>fig|2665645.4.peg.1706</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2185@1>fig|2665645.4.peg.2185</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2727@1>fig|2665645.4.peg.2727</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=2665645.4@1>Ribonuclease H</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease HI (EC 3.1.26.4)&subsystem_name=Ribonuclease_H@1>Ribonuclease HI (EC 3.1.26.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4174@1>fig|2665645.4.peg.4174</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=2665645.4@1>Ribonuclease H</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease HII (EC 3.1.26.4)&subsystem_name=Ribonuclease_H@1>Ribonuclease HII (EC 3.1.26.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3298@1>fig|2665645.4.peg.3298</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_3@1-terminal_phosphate_cyclase&organism=2665645.4@1>RNA 3@1-terminal phosphate cyclase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA-2@1,3@1-PO4:RNA-5@1-OH ligase&subsystem_name=RNA_3@1-terminal_phosphate_cyclase@1>RNA-2@1,3@1-PO4:RNA-5@1-OH ligase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.454@1>fig|2665645.4.peg.454</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2488@1>fig|2665645.4.peg.2488</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=2665645.4@1>RNA processing and degradation, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3@1-to-5@1 exoribonuclease RNase R&subsystem_name=RNA_processing_and_degradation,_bacterial@1>3@1-to-5@1 exoribonuclease RNase R</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4319@1>fig|2665645.4.peg.4319</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=2665645.4@1>RNA processing and degradation, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease III (EC 3.1.26.3)&subsystem_name=RNA_processing_and_degradation,_bacterial@1>Ribonuclease III (EC 3.1.26.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4170@1>fig|2665645.4.peg.4170</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=2665645.4@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease Z (EC 3.1.26.11)&subsystem_name=tRNA_processing@1>Ribonuclease Z (EC 3.1.26.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.705@1>fig|2665645.4.peg.705</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=2665645.4@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease P protein component (EC 3.1.26.5)&subsystem_name=tRNA_processing@1>Ribonuclease P protein component (EC 3.1.26.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4371@1>fig|2665645.4.peg.4371</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=2665645.4@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA dimethylallyltransferase (EC 2.5.1.75)&subsystem_name=tRNA_processing@1>tRNA dimethylallyltransferase (EC 2.5.1.75)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2483@1>fig|2665645.4.peg.2483</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=2665645.4@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)&subsystem_name=tRNA_processing@1>tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3154@1>fig|2665645.4.peg.3154</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=2665645.4@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (guanosine(18)-2@1-O)-methyltransferase (EC 2.1.1.34)&subsystem_name=RNA_methylation@1>tRNA (guanosine(18)-2@1-O)-methyltransferase (EC 2.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.58@1>fig|2665645.4.peg.58</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=2665645.4@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)&subsystem_name=RNA_methylation@1>tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.434@1>fig|2665645.4.peg.434</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=2665645.4@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (cytidine(34)-2@1-O)-methyltransferase (EC 2.1.1.207)&subsystem_name=RNA_methylation@1>tRNA (cytidine(34)-2@1-O)-methyltransferase (EC 2.1.1.207)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5280@1>fig|2665645.4.peg.5280</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=2665645.4@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)&subsystem_name=RNA_methylation@1>SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4490@1>fig|2665645.4.peg.4490</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=2665645.4@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)&subsystem_name=RNA_methylation@1>16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.745@1>fig|2665645.4.peg.745</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=2665645.4@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma-54 factor RpoN&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma-54 factor RpoN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4760@1>fig|2665645.4.peg.4760</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=2665645.4@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma factor RpoD&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma factor RpoD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3865@1>fig|2665645.4.peg.3865</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=2665645.4@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma-70 factor&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma-70 factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1677@1>fig|2665645.4.peg.1677</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=2665645.4@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4999@1>fig|2665645.4.peg.4999</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=2665645.4@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3279@1>fig|2665645.4.peg.3279</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=2665645.4@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2498@1>fig|2665645.4.peg.2498</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=2665645.4@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3280@1>fig|2665645.4.peg.3280</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination protein NusB&subsystem_name=Transcription_factors_bacterial@1>Transcription termination protein NusB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3072@1>fig|2665645.4.peg.3072</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription accessory protein (S1 RNA-binding domain)&subsystem_name=Transcription_factors_bacterial@1>Transcription accessory protein (S1 RNA-binding domain)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4311@1>fig|2665645.4.peg.4311</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription elongation factor GreA&subsystem_name=Transcription_factors_bacterial@1>Transcription elongation factor GreA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3691@1>fig|2665645.4.peg.3691</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination factor Rho&subsystem_name=Transcription_factors_bacterial@1>Transcription termination factor Rho</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.545@1>fig|2665645.4.peg.545</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription antitermination protein NusG&subsystem_name=Transcription_factors_bacterial@1>Transcription antitermination protein NusG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3285@1>fig|2665645.4.peg.3285</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription-repair coupling factor&subsystem_name=Transcription_factors_bacterial@1>Transcription-repair coupling factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4126@1>fig|2665645.4.peg.4126</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Regulator of nucleoside diphosphate kinase&subsystem_name=Transcription_factors_bacterial@1>Regulator of nucleoside diphosphate kinase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2155@1>fig|2665645.4.peg.2155</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3082@1>fig|2665645.4.peg.3082</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator RfaH&subsystem_name=Transcription_factors_bacterial@1>Transcriptional activator RfaH</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5014@1>fig|2665645.4.peg.5014</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=2665645.4@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination protein NusA&subsystem_name=Transcription_factors_bacterial@1>Transcription termination protein NusA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3650@1>fig|2665645.4.peg.3650</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=2665645.4@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted transcriptional regulator of sulfate adenylyltransferase, Rrf2 family&subsystem_name=Rrf2_family_transcriptional_regulators@1>Predicted transcriptional regulator of sulfate adenylyltransferase, Rrf2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3342@1>fig|2665645.4.peg.3342</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=2665645.4@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted transcriptional regulator of sulfate transport, Rrf2 family&subsystem_name=Rrf2_family_transcriptional_regulators@1>Predicted transcriptional regulator of sulfate transport, Rrf2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3799@1>fig|2665645.4.peg.3799</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=2665645.4@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rrf2 family transcriptional regulator&subsystem_name=Rrf2_family_transcriptional_regulators@1>Rrf2 family transcriptional regulator</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2831@1>fig|2665645.4.peg.2831</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=2665645.4@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rrf2 family transcriptional regulator, group III&subsystem_name=Rrf2_family_transcriptional_regulators@1>Rrf2 family transcriptional regulator, group III</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.48@1>fig|2665645.4.peg.48</a>@~RNA Metabolism@^RNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Group_II_intron-associated_genes&organism=2665645.4@1>Group II intron-associated genes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Retron-type RNA-directed DNA polymerase (EC 2.7.7.49)&subsystem_name=Group_II_intron-associated_genes@1>Retron-type RNA-directed DNA polymerase (EC 2.7.7.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4106@1>fig|2665645.4.peg.4106</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside diphosphate kinase (EC 2.7.4.6)&subsystem_name=pyrimidine_conversions@1>Nucleoside diphosphate kinase (EC 2.7.4.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.104@1>fig|2665645.4.peg.104</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytosine deaminase (EC 3.5.4.1)&subsystem_name=pyrimidine_conversions@1>Cytosine deaminase (EC 3.5.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2400@1>fig|2665645.4.peg.2400</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thioredoxin reductase (EC 1.8.1.9)&subsystem_name=pyrimidine_conversions@1>Thioredoxin reductase (EC 1.8.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1703@1>fig|2665645.4.peg.1703</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3789@1>fig|2665645.4.peg.3789</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4407@1>fig|2665645.4.peg.4407</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5161@1>fig|2665645.4.peg.5161</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytidylate kinase (EC 2.7.4.25)&subsystem_name=pyrimidine_conversions@1>Cytidylate kinase (EC 2.7.4.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1622@1>fig|2665645.4.peg.1622</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidine kinase (EC 2.7.1.21)&subsystem_name=pyrimidine_conversions@1>Thymidine kinase (EC 2.7.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4756@1>fig|2665645.4.peg.4756</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=pyrimidine_conversions@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1778@1>fig|2665645.4.peg.1778</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-nucleotidase (EC 3.1.3.5)&subsystem_name=pyrimidine_conversions@1>5@1-nucleotidase (EC 3.1.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2502@1>fig|2665645.4.peg.2502</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2503@1>fig|2665645.4.peg.2503</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytidine deaminase (EC 3.5.4.5)&subsystem_name=pyrimidine_conversions@1>Cytidine deaminase (EC 3.5.4.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3733@1>fig|2665645.4.peg.3733</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidylate synthase (EC 2.1.1.45)&subsystem_name=pyrimidine_conversions@1>Thymidylate synthase (EC 2.1.1.45)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3829@1>fig|2665645.4.peg.3829</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uridine kinase (EC 2.7.1.48)&subsystem_name=pyrimidine_conversions@1>Uridine kinase (EC 2.7.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5362@1>fig|2665645.4.peg.5362</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uracil phosphoribosyltransferase (EC 2.4.2.9)&subsystem_name=pyrimidine_conversions@1>Uracil phosphoribosyltransferase (EC 2.4.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4290@1>fig|2665645.4.peg.4290</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=2665645.4@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=CTP synthase (EC 6.3.4.2)&subsystem_name=pyrimidine_conversions@1>CTP synthase (EC 6.3.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2863@1>fig|2665645.4.peg.2863</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_Utilization&organism=2665645.4@1>Purine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Periplasmic aromatic aldehyde oxidoreductase, FAD binding subunit YagS&subsystem_name=Purine_Utilization@1>Periplasmic aromatic aldehyde oxidoreductase, FAD binding subunit YagS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1295@1>fig|2665645.4.peg.1295</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_Utilization&organism=2665645.4@1>Purine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xanthine and CO dehydrogenases maturation factor, XdhC/CoxF family&subsystem_name=Purine_Utilization@1>Xanthine and CO dehydrogenases maturation factor, XdhC/CoxF family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1290@1>fig|2665645.4.peg.1290</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=2665645.4@1>GMP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)&subsystem_name=GMP_synthase@1>GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2883@1>fig|2665645.4.peg.2883</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=2665645.4@1>GMP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)&subsystem_name=GMP_synthase@1>GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2883@1>fig|2665645.4.peg.2883</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate synthetase (EC 6.3.4.4)&subsystem_name=Purine_conversions@1>Adenylosuccinate synthetase (EC 6.3.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3051@1>fig|2665645.4.peg.3051</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dNTP triphosphohydrolase, broad substrate specificity, subgroup 3&subsystem_name=Purine_conversions@1>dNTP triphosphohydrolase, broad substrate specificity, subgroup 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3767@1>fig|2665645.4.peg.3767</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)&subsystem_name=Purine_conversions@1>GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2883@1>fig|2665645.4.peg.2883</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inosine-5@1-monophosphate dehydrogenase (EC 1.1.1.205)&subsystem_name=Purine_conversions@1>Inosine-5@1-monophosphate dehydrogenase (EC 1.1.1.205)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3061@1>fig|2665645.4.peg.3061</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP reductase (EC 1.7.1.7)&subsystem_name=Purine_conversions@1>GMP reductase (EC 1.7.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2441@1>fig|2665645.4.peg.2441</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)&subsystem_name=Purine_conversions@1>Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4942@1>fig|2665645.4.peg.4942</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate lyase (EC 4.3.2.2)&subsystem_name=Purine_conversions@1>Adenylosuccinate lyase (EC 4.3.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.52@1>fig|2665645.4.peg.52</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Purine_conversions@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.716@1>fig|2665645.4.peg.716</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenine phosphoribosyltransferase (EC 2.4.2.7)&subsystem_name=Purine_conversions@1>Adenine phosphoribosyltransferase (EC 2.4.2.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1895@1>fig|2665645.4.peg.1895</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyadenosine kinase (EC 2.7.1.76)&subsystem_name=Purine_conversions@1>Deoxyadenosine kinase (EC 2.7.1.76)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3562@1>fig|2665645.4.peg.3562</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3820@1>fig|2665645.4.peg.3820</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylate kinase (EC 2.7.4.3)&subsystem_name=Purine_conversions@1>Adenylate kinase (EC 2.7.4.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4943@1>fig|2665645.4.peg.4943</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside diphosphate kinase (EC 2.7.4.6)&subsystem_name=Purine_conversions@1>Nucleoside diphosphate kinase (EC 2.7.4.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.104@1>fig|2665645.4.peg.104</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyguanosine kinase (EC 2.7.1.113)&subsystem_name=Purine_conversions@1>Deoxyguanosine kinase (EC 2.7.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3562@1>fig|2665645.4.peg.3562</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3820@1>fig|2665645.4.peg.3820</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleotide pyrophosphatase (EC 3.6.1.9)&subsystem_name=Purine_conversions@1>Nucleotide pyrophosphatase (EC 3.6.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3431@1>fig|2665645.4.peg.3431</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Purine_conversions@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1778@1>fig|2665645.4.peg.1778</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-nucleotidase (EC 3.1.3.5)&subsystem_name=Purine_conversions@1>5@1-nucleotidase (EC 3.1.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2502@1>fig|2665645.4.peg.2502</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2503@1>fig|2665645.4.peg.2503</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)&subsystem_name=Purine_conversions@1>GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2883@1>fig|2665645.4.peg.2883</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosine deaminase (EC 3.5.4.4)&subsystem_name=Purine_conversions@1>Adenosine deaminase (EC 3.5.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2528@1>fig|2665645.4.peg.2528</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=2665645.4@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Guanylate kinase (EC 2.7.4.8)&subsystem_name=Purine_conversions@1>Guanylate kinase (EC 2.7.4.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4612@1>fig|2665645.4.peg.4612</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=2665645.4@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class II (coenzyme B12-dependent) (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class II (coenzyme B12-dependent) (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1273@1>fig|2665645.4.peg.1273</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=2665645.4@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2261@1>fig|2665645.4.peg.2261</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4827@1>fig|2665645.4.peg.4827</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=2665645.4@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2260@1>fig|2665645.4.peg.2260</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4828@1>fig|2665645.4.peg.4828</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dNTP_triphosphohydrolase_protein_family&organism=2665645.4@1>dNTP triphosphohydrolase protein family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dNTP triphosphohydrolase, broad substrate specificity, subgroup 3&subsystem_name=dNTP_triphosphohydrolase_protein_family@1>dNTP triphosphohydrolase, broad substrate specificity, subgroup 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3767@1>fig|2665645.4.peg.3767</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=2665645.4@1>Adenosyl nucleosidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)&subsystem_name=Adenosyl_nucleosidases@1>S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2141@1>fig|2665645.4.peg.2141</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=2665645.4@1>Adenosyl nucleosidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Adenosyl_nucleosidases@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1778@1>fig|2665645.4.peg.1778</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=2665645.4@1>Adenosyl nucleosidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-methylthioadenosine nucleosidase (EC 3.2.2.16)&subsystem_name=Adenosyl_nucleosidases@1>5@1-methylthioadenosine nucleosidase (EC 3.2.2.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2141@1>fig|2665645.4.peg.2141</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nucleoside_triphosphate_pyrophosphohydrolase_MazG&organism=2665645.4@1>Nucleoside triphosphate pyrophosphohydrolase MazG</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)&subsystem_name=Nucleoside_triphosphate_pyrophosphohydrolase_MazG@1>Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2700@1>fig|2665645.4.peg.2700</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=2665645.4@1>Nudix proteins (nucleoside triphosphate hydrolases)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nudix hydrolase family protein YffH&subsystem_name=Nudix_proteins_(nucleoside_triphosphate_hydrolases)@1>Nudix hydrolase family protein YffH</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3052@1>fig|2665645.4.peg.3052</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=2665645.4@1>Nudix proteins (nucleoside triphosphate hydrolases)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)&subsystem_name=Nudix_proteins_(nucleoside_triphosphate_hydrolases)@1>Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4270@1>fig|2665645.4.peg.4270</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=2665645.4@1>Housecleaning nucleoside triphosphate pyrophosphatases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)&subsystem_name=Housecleaning_nucleoside_triphosphate_pyrophosphatases@1>Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4270@1>fig|2665645.4.peg.4270</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=2665645.4@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Heat shock protein GrpE&subsystem_name=Protein_chaperones@1>Heat shock protein GrpE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.12@1>fig|2665645.4.peg.12</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=2665645.4@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein DnaK&subsystem_name=Protein_chaperones@1>Chaperone protein DnaK</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2573@1>fig|2665645.4.peg.2573</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=2665645.4@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein DnaJ&subsystem_name=Protein_chaperones@1>Chaperone protein DnaJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.13@1>fig|2665645.4.peg.13</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=2665645.4@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DnaJ-class molecular chaperone CbpA&subsystem_name=Protein_chaperones@1>DnaJ-class molecular chaperone CbpA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2890@1>fig|2665645.4.peg.2890</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=2665645.4@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein HtpG&subsystem_name=Protein_chaperones@1>Chaperone protein HtpG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5097@1>fig|2665645.4.peg.5097</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_disulfide_interchange&organism=2665645.4@1>Periplasmic disulfide interchange</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)&subsystem_name=Periplasmic_disulfide_interchange@1>Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1386@1>fig|2665645.4.peg.1386</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3153@1>fig|2665645.4.peg.3153</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Peptidyl-prolyl_cis-trans_isomerase&organism=2665645.4@1>Peptidyl-prolyl cis-trans isomerase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)&subsystem_name=Peptidyl-prolyl_cis-trans_isomerase@1>FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.77@1>fig|2665645.4.peg.77</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5320@1>fig|2665645.4.peg.5320</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Val&organism=2665645.4@1>tRNA aminoacylation, Val</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Valyl-tRNA synthetase (EC 6.1.1.9)&subsystem_name=tRNA_aminoacylation,_Val@1>Valyl-tRNA synthetase (EC 6.1.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3000@1>fig|2665645.4.peg.3000</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Met&organism=2665645.4@1>tRNA aminoacylation, Met</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionyl-tRNA synthetase (EC 6.1.1.10)&subsystem_name=tRNA_aminoacylation,_Met@1>Methionyl-tRNA synthetase (EC 6.1.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4464@1>fig|2665645.4.peg.4464</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ile&organism=2665645.4@1>tRNA aminoacylation, Ile</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isoleucyl-tRNA synthetase (EC 6.1.1.5)&subsystem_name=tRNA_aminoacylation,_Ile@1>Isoleucyl-tRNA synthetase (EC 6.1.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4654@1>fig|2665645.4.peg.4654</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Arg&organism=2665645.4@1>tRNA aminoacylation, Arg</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Arginyl-tRNA synthetase (EC 6.1.1.19)&subsystem_name=tRNA_aminoacylation,_Arg@1>Arginyl-tRNA synthetase (EC 6.1.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5105@1>fig|2665645.4.peg.5105</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=2665645.4@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor SUI1-related protein&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor SUI1-related protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1779@1>fig|2665645.4.peg.1779</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=2665645.4@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 1&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4994@1>fig|2665645.4.peg.4994</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=2665645.4@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionyl-tRNA formyltransferase (EC 2.1.2.9)&subsystem_name=Translation_initiation_factors_bacterial@1>Methionyl-tRNA formyltransferase (EC 2.1.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5209@1>fig|2665645.4.peg.5209</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=2665645.4@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 3&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5346@1>fig|2665645.4.peg.5346</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=2665645.4@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 2&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3649@1>fig|2665645.4.peg.3649</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=2665645.4@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosome-binding factor A&subsystem_name=Translation_initiation_factors_bacterial@1>Ribosome-binding factor A</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4264@1>fig|2665645.4.peg.4264</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S11p (S14e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S11p (S14e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4997@1>fig|2665645.4.peg.4997</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S1p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S1p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1627@1>fig|2665645.4.peg.1627</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S6p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S6p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2516@1>fig|2665645.4.peg.2516</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S15p (S13e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S15p (S13e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3856@1>fig|2665645.4.peg.3856</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S14p (S29e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S14p (S29e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4986@1>fig|2665645.4.peg.4986</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S5p (S2e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S5p (S2e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4990@1>fig|2665645.4.peg.4990</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S18p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S18p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2515@1>fig|2665645.4.peg.2515</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S17p (S11e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S17p (S11e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4982@1>fig|2665645.4.peg.4982</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S20p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S20p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.148@1>fig|2665645.4.peg.148</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S12p (S23e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S12p (S23e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4969@1>fig|2665645.4.peg.4969</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S9p (S16e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S9p (S16e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3669@1>fig|2665645.4.peg.3669</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S8p (S15Ae)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S8p (S15Ae)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4987@1>fig|2665645.4.peg.4987</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S4p (S9e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S4p (S9e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4998@1>fig|2665645.4.peg.4998</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S14p (S29e), zinc-independent&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S14p (S29e), zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4986@1>fig|2665645.4.peg.4986</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S19p (S15e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S19p (S15e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4977@1>fig|2665645.4.peg.4977</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S10p (S20e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S10p (S20e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4972@1>fig|2665645.4.peg.4972</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S16p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S16p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4015@1>fig|2665645.4.peg.4015</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S3p (S3e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S3p (S3e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4979@1>fig|2665645.4.peg.4979</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S18p, zinc-independent&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S18p, zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2515@1>fig|2665645.4.peg.2515</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S7p (S5e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S7p (S5e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4970@1>fig|2665645.4.peg.4970</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S2p (SAe)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S2p (SAe)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3670@1>fig|2665645.4.peg.3670</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S21p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S21p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3290@1>fig|2665645.4.peg.3290</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=2665645.4@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S13p (S18e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S13p (S18e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4996@1>fig|2665645.4.peg.4996</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Gly&organism=2665645.4@1>tRNA aminoacylation, Gly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycyl-tRNA synthetase (EC 6.1.1.14)&subsystem_name=tRNA_aminoacylation,_Gly@1>Glycyl-tRNA synthetase (EC 6.1.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4551@1>fig|2665645.4.peg.4551</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ala&organism=2665645.4@1>tRNA aminoacylation, Ala</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanyl-tRNA synthetase (EC 6.1.1.7)&subsystem_name=tRNA_aminoacylation,_Ala@1>Alanyl-tRNA synthetase (EC 6.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3336@1>fig|2665645.4.peg.3336</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Trp&organism=2665645.4@1>tRNA aminoacylation, Trp</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophanyl-tRNA synthetase (EC 6.1.1.2)&subsystem_name=tRNA_aminoacylation,_Trp@1>Tryptophanyl-tRNA synthetase (EC 6.1.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4349@1>fig|2665645.4.peg.4349</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L18p (L5e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L18p (L5e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4989@1>fig|2665645.4.peg.4989</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L11p (L12e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L11p (L12e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3284@1>fig|2665645.4.peg.3284</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L5p (L11e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L5p (L11e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4985@1>fig|2665645.4.peg.4985</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L23p (L23Ae)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L23p (L23Ae)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4975@1>fig|2665645.4.peg.4975</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L14p (L23e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L14p (L23e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4983@1>fig|2665645.4.peg.4983</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L2p (L8e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L2p (L8e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4976@1>fig|2665645.4.peg.4976</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L31p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L31p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5239@1>fig|2665645.4.peg.5239</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L6p (L9e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L6p (L9e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4988@1>fig|2665645.4.peg.4988</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L21p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L21p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3744@1>fig|2665645.4.peg.3744</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L24p (L26e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L24p (L26e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4984@1>fig|2665645.4.peg.4984</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L29p (L35e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L29p (L35e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4981@1>fig|2665645.4.peg.4981</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L30p (L7e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L30p (L7e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4991@1>fig|2665645.4.peg.4991</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L31p, zinc-independent&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L31p, zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5239@1>fig|2665645.4.peg.5239</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L1p (L10Ae)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L1p (L10Ae)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3283@1>fig|2665645.4.peg.3283</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L16p (L10e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L16p (L10e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4980@1>fig|2665645.4.peg.4980</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L25p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L25p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4185@1>fig|2665645.4.peg.4185</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L3p (L3e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L3p (L3e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4973@1>fig|2665645.4.peg.4973</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L15p (L27Ae)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L15p (L27Ae)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4992@1>fig|2665645.4.peg.4992</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L33p, zinc-independent&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L33p, zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.95@1>fig|2665645.4.peg.95</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L13p (L13Ae)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L13p (L13Ae)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3668@1>fig|2665645.4.peg.3668</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L22p (L17e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L22p (L17e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4978@1>fig|2665645.4.peg.4978</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L34p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L34p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5260@1>fig|2665645.4.peg.5260</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L10p (P0)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L10p (P0)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3282@1>fig|2665645.4.peg.3282</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L9p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L9p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2514@1>fig|2665645.4.peg.2514</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L28p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L28p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.94@1>fig|2665645.4.peg.94</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L19p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L19p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3093@1>fig|2665645.4.peg.3093</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L35p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L35p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5345@1>fig|2665645.4.peg.5345</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L36p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L36p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4995@1>fig|2665645.4.peg.4995</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L20p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L20p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5344@1>fig|2665645.4.peg.5344</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L32p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L32p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4715@1>fig|2665645.4.peg.4715</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L33p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L33p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.95@1>fig|2665645.4.peg.95</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L17p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L17p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5000@1>fig|2665645.4.peg.5000</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L27p&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L27p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3743@1>fig|2665645.4.peg.3743</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_LSU_bacterial&organism=2665645.4@1>Ribosome LSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L4p (L1e)&subsystem_name=Ribosome_LSU_bacterial@1>LSU ribosomal protein L4p (L1e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4974@1>fig|2665645.4.peg.4974</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=2665645.4@1>Programmed frameshift</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=programmed frameshift-containing&subsystem_name=Programmed_frameshift@1>programmed frameshift-containing</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2676@1>fig|2665645.4.peg.2676</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=2665645.4@1>Programmed frameshift</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 2&subsystem_name=Programmed_frameshift@1>Peptide chain release factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2676@1>fig|2665645.4.peg.2676</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Cys&organism=2665645.4@1>tRNA aminoacylation, Cys</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteinyl-tRNA synthetase (EC 6.1.1.16)&subsystem_name=tRNA_aminoacylation,_Cys@1>Cysteinyl-tRNA synthetase (EC 6.1.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3009@1>fig|2665645.4.peg.3009</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosome recycling factor&subsystem_name=Translation_termination_factors_bacterial@1>Ribosome recycling factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4770@1>fig|2665645.4.peg.4770</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor homolog&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2486@1>fig|2665645.4.peg.2486</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidyl-tRNA hydrolase (EC 3.1.1.29)&subsystem_name=Translation_termination_factors_bacterial@1>Peptidyl-tRNA hydrolase (EC 3.1.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4184@1>fig|2665645.4.peg.4184</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide deformylase (EC 3.5.1.88)&subsystem_name=Translation_termination_factors_bacterial@1>Peptide deformylase (EC 3.5.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2702@1>fig|2665645.4.peg.2702</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 3&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3276@1>fig|2665645.4.peg.3276</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tmRNA-binding protein SmpB&subsystem_name=Translation_termination_factors_bacterial@1>tmRNA-binding protein SmpB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2474@1>fig|2665645.4.peg.2474</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 1&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2977@1>fig|2665645.4.peg.2977</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine aminopeptidase (EC 3.4.11.18)&subsystem_name=Translation_termination_factors_bacterial@1>Methionine aminopeptidase (EC 3.4.11.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3517@1>fig|2665645.4.peg.3517</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5197@1>fig|2665645.4.peg.5197</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=2665645.4@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 2&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2676@1>fig|2665645.4.peg.2676</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_His&organism=2665645.4@1>tRNA aminoacylation, His</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidyl-tRNA synthetase (EC 6.1.1.21)&subsystem_name=tRNA_aminoacylation,_His@1>Histidyl-tRNA synthetase (EC 6.1.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.11@1>fig|2665645.4.peg.11</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=2665645.4@1>tRNA aminoacylation, Asp and Asn</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartyl-tRNA synthetase (EC 6.1.1.12)&subsystem_name=tRNA_aminoacylation,_Asp_and_Asn@1>Aspartyl-tRNA synthetase (EC 6.1.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4085@1>fig|2665645.4.peg.4085</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=2665645.4@1>tRNA aminoacylation, Asp and Asn</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Asparaginyl-tRNA synthetase (EC 6.1.1.22)&subsystem_name=tRNA_aminoacylation,_Asp_and_Asn@1>Asparaginyl-tRNA synthetase (EC 6.1.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4761@1>fig|2665645.4.peg.4761</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=2665645.4@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4971@1>fig|2665645.4.peg.4971</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=2665645.4@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor LepA&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor LepA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4562@1>fig|2665645.4.peg.4562</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=2665645.4@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Tu&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor Tu</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3287@1>fig|2665645.4.peg.3287</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=2665645.4@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Ts&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor Ts</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3671@1>fig|2665645.4.peg.3671</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=2665645.4@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor P&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor P</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1848@1>fig|2665645.4.peg.1848</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Lys&organism=2665645.4@1>tRNA aminoacylation, Lys</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)&subsystem_name=tRNA_aminoacylation,_Lys@1>Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3305@1>fig|2665645.4.peg.3305</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Thr&organism=2665645.4@1>tRNA aminoacylation, Thr</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonyl-tRNA synthetase (EC 6.1.1.3)&subsystem_name=tRNA_aminoacylation,_Thr@1>Threonyl-tRNA synthetase (EC 6.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5347@1>fig|2665645.4.peg.5347</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Pro&organism=2665645.4@1>tRNA aminoacylation, Pro</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prolyl-tRNA synthetase (EC 6.1.1.15), archaeal/eukaryal type&subsystem_name=tRNA_aminoacylation,_Pro@1>Prolyl-tRNA synthetase (EC 6.1.1.15), archaeal/eukaryal type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.147@1>fig|2665645.4.peg.147</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factor_G_family&organism=2665645.4@1>Translation elongation factor G family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Translation_elongation_factor_G_family@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4971@1>fig|2665645.4.peg.4971</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=2665645.4@1>tRNA aminoacylation, Glu and Gln</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA synthetase (EC 6.1.1.17)&subsystem_name=tRNA_aminoacylation,_Glu_and_Gln@1>Glutamyl-tRNA synthetase (EC 6.1.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4331@1>fig|2665645.4.peg.4331</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=2665645.4@1>tRNA aminoacylation, Glu and Gln</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutaminyl-tRNA synthetase (EC 6.1.1.18)&subsystem_name=tRNA_aminoacylation,_Glu_and_Gln@1>Glutaminyl-tRNA synthetase (EC 6.1.1.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4324@1>fig|2665645.4.peg.4324</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ser&organism=2665645.4@1>tRNA aminoacylation, Ser</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Seryl-tRNA synthetase (EC 6.1.1.11)&subsystem_name=tRNA_aminoacylation,_Ser@1>Seryl-tRNA synthetase (EC 6.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4485@1>fig|2665645.4.peg.4485</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Tyr&organism=2665645.4@1>tRNA aminoacylation, Tyr</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tyrosyl-tRNA synthetase (EC 6.1.1.1)&subsystem_name=tRNA_aminoacylation,_Tyr@1>Tyrosyl-tRNA synthetase (EC 6.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4795@1>fig|2665645.4.peg.4795</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Leu&organism=2665645.4@1>tRNA aminoacylation, Leu</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Leucyl-tRNA synthetase (EC 6.1.1.4)&subsystem_name=tRNA_aminoacylation,_Leu@1>Leucyl-tRNA synthetase (EC 6.1.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4779@1>fig|2665645.4.peg.4779</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=2665645.4@1>tRNA aminoacylation, Phe</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)&subsystem_name=tRNA_aminoacylation,_Phe@1>Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4584@1>fig|2665645.4.peg.4584</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=2665645.4@1>tRNA aminoacylation, Phe</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)&subsystem_name=tRNA_aminoacylation,_Phe@1>Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1894@1>fig|2665645.4.peg.1894</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein-L-isoaspartate_O-methyltransferase&organism=2665645.4@1>Protein-L-isoaspartate O-methyltransferase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Protein-L-isoaspartate_O-methyltransferase@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2477@1>fig|2665645.4.peg.2477</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=2665645.4@1>Signal peptidase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Signal peptidase I (EC 3.4.21.89)&subsystem_name=Signal_peptidase@1>Signal peptidase I (EC 3.4.21.89)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1752@1>fig|2665645.4.peg.1752</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5151@1>fig|2665645.4.peg.5151</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=2665645.4@1>Signal peptidase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lipoprotein signal peptidase (EC 3.4.23.36)&subsystem_name=Signal_peptidase@1>Lipoprotein signal peptidase (EC 3.4.23.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4659@1>fig|2665645.4.peg.4659</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=2665645.4@1>Modification of eukaryotic initiation factor 5A</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyhypusine synthase (EC 2.5.1.46)&subsystem_name=Modification_of_eukaryotic_initiation_factor_5A@1>Deoxyhypusine synthase (EC 2.5.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1721@1>fig|2665645.4.peg.1721</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=2665645.4@1>Modification of eukaryotic initiation factor 5A</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor P&subsystem_name=Modification_of_eukaryotic_initiation_factor_5A@1>Translation elongation factor P</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1848@1>fig|2665645.4.peg.1848</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=2665645.4@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3551@1>fig|2665645.4.peg.3551</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5012@1>fig|2665645.4.peg.5012</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=2665645.4@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-1,4-N-acetylgalactosamine transferase PglH (EC 2.4.1.-)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>Alpha-1,4-N-acetylgalactosamine transferase PglH (EC 2.4.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5067@1>fig|2665645.4.peg.5067</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=2665645.4@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lipid carrier : UDP-N-acetylgalactosaminyltransferase (EC 2.4.1.-)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>Lipid carrier : UDP-N-acetylgalactosaminyltransferase (EC 2.4.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5030@1>fig|2665645.4.peg.5030</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=2665645.4@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-keto-6-deoxy-N-Acetyl-D-hexosaminyl-(Lipid carrier) aminotransferase&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>4-keto-6-deoxy-N-Acetyl-D-hexosaminyl-(Lipid carrier) aminotransferase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5032@1>fig|2665645.4.peg.5032</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=2665645.4@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-1,3-N-acetylgalactosamine transferase PglA (EC 2.4.1.-)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>Alpha-1,3-N-acetylgalactosamine transferase PglA (EC 2.4.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5029@1>fig|2665645.4.peg.5029</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=2665645.4@1>Aminopeptidases (EC 3.4.11.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xaa-Pro aminopeptidase (EC 3.4.11.9)&subsystem_name=Aminopeptidases_(EC_3.4.11.-)@1>Xaa-Pro aminopeptidase (EC 3.4.11.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4673@1>fig|2665645.4.peg.4673</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5085@1>fig|2665645.4.peg.5085</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=2665645.4@1>Aminopeptidases (EC 3.4.11.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Membrane alanine aminopeptidase N (EC 3.4.11.2)&subsystem_name=Aminopeptidases_(EC_3.4.11.-)@1>Membrane alanine aminopeptidase N (EC 3.4.11.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3960@1>fig|2665645.4.peg.3960</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=2665645.4@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)&subsystem_name=Protein_degradation@1>Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3685@1>fig|2665645.4.peg.3685</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=2665645.4@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)&subsystem_name=Protein_degradation@1>Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3672@1>fig|2665645.4.peg.3672</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4935@1>fig|2665645.4.peg.4935</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=2665645.4@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminopeptidase C (EC 3.4.22.40)&subsystem_name=Protein_degradation@1>Aminopeptidase C (EC 3.4.22.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4162@1>fig|2665645.4.peg.4162</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=2665645.4@1>Metallocarboxypeptidases (EC 3.4.17.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)&subsystem_name=Metallocarboxypeptidases_(EC_3.4.17.-)@1>D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.507@1>fig|2665645.4.peg.507</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4857@1>fig|2665645.4.peg.4857</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=2665645.4@1>Metallocarboxypeptidases (EC 3.4.17.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Metallocarboxypeptidases_(EC_3.4.17.-)@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4116@1>fig|2665645.4.peg.4116</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dipeptidases_(EC_3.4.13.-)&organism=2665645.4@1>Dipeptidases (EC 3.4.13.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)&subsystem_name=Dipeptidases_(EC_3.4.13.-)@1>Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3511@1>fig|2665645.4.peg.3511</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_endopeptidase_(EC_3.4.21.-)&organism=2665645.4@1>Serine endopeptidase (EC 3.4.21.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prolyl endopeptidase (EC 3.4.21.26)&subsystem_name=Serine_endopeptidase_(EC_3.4.21.-)@1>Prolyl endopeptidase (EC 3.4.21.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4000@1>fig|2665645.4.peg.4000</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=2665645.4@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RadA&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>DNA repair protein RadA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4513@1>fig|2665645.4.peg.4513</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=2665645.4@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent Clp protease ATP-binding subunit ClpX&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent Clp protease ATP-binding subunit ClpX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3577@1>fig|2665645.4.peg.3577</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=2665645.4@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent Clp protease, ATP-binding subunit ClpC&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent Clp protease, ATP-binding subunit ClpC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2630@1>fig|2665645.4.peg.2630</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=2665645.4@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent protease La (EC 3.4.21.53) Type I&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent protease La (EC 3.4.21.53) Type I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1618@1>fig|2665645.4.peg.1618</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=cAMP_signaling_in_bacteria&organism=2665645.4@1>cAMP signaling in bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=cAMP-binding proteins - catabolite gene activator and regulatory subunit of cAMP-dependent protein kinases&subsystem_name=cAMP_signaling_in_bacteria@1>cAMP-binding proteins - catabolite gene activator and regulatory subunit of cAMP-dependent protein kinases</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.605@1>fig|2665645.4.peg.605</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.719@1>fig|2665645.4.peg.719</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.937@1>fig|2665645.4.peg.937</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1163@1>fig|2665645.4.peg.1163</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1428@1>fig|2665645.4.peg.1428</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1784@1>fig|2665645.4.peg.1784</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2078@1>fig|2665645.4.peg.2078</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2088@1>fig|2665645.4.peg.2088</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2158@1>fig|2665645.4.peg.2158</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2199@1>fig|2665645.4.peg.2199</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2254@1>fig|2665645.4.peg.2254</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2655@1>fig|2665645.4.peg.2655</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3376@1>fig|2665645.4.peg.3376</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3540@1>fig|2665645.4.peg.3540</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3913@1>fig|2665645.4.peg.3913</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4877@1>fig|2665645.4.peg.4877</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4884@1>fig|2665645.4.peg.4884</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5187@1>fig|2665645.4.peg.5187</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5343@1>fig|2665645.4.peg.5343</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=cAMP_signaling_in_bacteria&organism=2665645.4@1>cAMP signaling in bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted signal-transduction protein containing cAMP-binding and CBS domains&subsystem_name=cAMP_signaling_in_bacteria@1>Predicted signal-transduction protein containing cAMP-binding and CBS domains</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1799@1>fig|2665645.4.peg.1799</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2756@1>fig|2665645.4.peg.2756</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4265@1>fig|2665645.4.peg.4265</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=cAMP_signaling_in_bacteria&organism=2665645.4@1>cAMP signaling in bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylate cyclase (EC 4.6.1.1)&subsystem_name=cAMP_signaling_in_bacteria@1>Adenylate cyclase (EC 4.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2177@1>fig|2665645.4.peg.2177</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=cAMP_signaling_in_bacteria&organism=2665645.4@1>cAMP signaling in bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenine phosphoribosyltransferase (EC 2.4.2.7)&subsystem_name=cAMP_signaling_in_bacteria@1>Adenine phosphoribosyltransferase (EC 2.4.2.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1895@1>fig|2665645.4.peg.1895</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=cAMP_signaling_in_bacteria&organism=2665645.4@1>cAMP signaling in bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3@1,5@1-cyclic-nucleotide phosphodiesterase (EC 3.1.4.17)&subsystem_name=cAMP_signaling_in_bacteria@1>3@1,5@1-cyclic-nucleotide phosphodiesterase (EC 3.1.4.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4109@1>fig|2665645.4.peg.4109</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Salmonella_enterica_Typhimurium&organism=2665645.4@1>LysR-family proteins in Salmonella enterica Typhimurium</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator MetR&subsystem_name=LysR-family_proteins_in_Salmonella_enterica_Typhimurium@1>Transcriptional activator MetR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4446@1>fig|2665645.4.peg.4446</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Escherichia_coli&organism=2665645.4@1>LysR-family proteins in Escherichia coli</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator MetR&subsystem_name=LysR-family_proteins_in_Escherichia_coli@1>Transcriptional activator MetR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4446@1>fig|2665645.4.peg.4446</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=2665645.4@1>Stringent Response, (p)ppGpp metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II&subsystem_name=Stringent_Response,_(p)ppGpp_metabolism@1>GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3049@1>fig|2665645.4.peg.3049</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=2665645.4@1>Stringent Response, (p)ppGpp metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Guanosine-3@1,5@1-bis(diphosphate) 3@1-pyrophosphohydrolase (EC 3.1.7.2)&subsystem_name=Stringent_Response,_(p)ppGpp_metabolism@1>Guanosine-3@1,5@1-bis(diphosphate) 3@1-pyrophosphohydrolase (EC 3.1.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3049@1>fig|2665645.4.peg.3049</a>@~Secondary Metabolism@^Secondary Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lanthionine_Synthetases&organism=2665645.4@1>Lanthionine Synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Lanthionine_Synthetases@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2477@1>fig|2665645.4.peg.2477</a>@~Secondary Metabolism@^Secondary Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lanthionine_Synthetases&organism=2665645.4@1>Lanthionine Synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lanthionine biosynthesis protein LanB&subsystem_name=Lanthionine_Synthetases@1>Lanthionine biosynthesis protein LanB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1902@1>fig|2665645.4.peg.1902</a>@~Secondary Metabolism@^Plant Alkaloids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alkaloid_biosynthesis_from_L-lysine&organism=2665645.4@1>Alkaloid biosynthesis from L-lysine</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyhypusine synthase (EC 2.5.1.46)&subsystem_name=Alkaloid_biosynthesis_from_L-lysine@1>Deoxyhypusine synthase (EC 2.5.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1721@1>fig|2665645.4.peg.1721</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=2665645.4@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Auxin_biosynthesis@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.190@1>fig|2665645.4.peg.190</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=2665645.4@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)&subsystem_name=Auxin_biosynthesis@1>Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3780@1>fig|2665645.4.peg.3780</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=2665645.4@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Auxin_biosynthesis@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.197@1>fig|2665645.4.peg.197</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=2665645.4@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Auxin_biosynthesis@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.194@1>fig|2665645.4.peg.194</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=2665645.4@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Auxin_biosynthesis@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.195@1>fig|2665645.4.peg.195</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=2665645.4@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination inhibitory protein MutS2&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>Recombination inhibitory protein MutS2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1859@1>fig|2665645.4.peg.1859</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=2665645.4@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutS&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>DNA mismatch repair protein MutS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3558@1>fig|2665645.4.peg.3558</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=2665645.4@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutL&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>DNA mismatch repair protein MutL</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5146@1>fig|2665645.4.peg.5146</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=2665645.4@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit A paralog in greater Bacteroides group&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit A paralog in greater Bacteroides group</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2814@1>fig|2665645.4.peg.2814</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=2665645.4@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit C&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit C</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4629@1>fig|2665645.4.peg.4629</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=2665645.4@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit A&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit A</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.758@1>fig|2665645.4.peg.758</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=2665645.4@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit B&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit B</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3104@1>fig|2665645.4.peg.3104</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_photolyase&organism=2665645.4@1>DNA repair, bacterial photolyase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribodipyrimidine photolyase (EC 4.1.99.3)&subsystem_name=DNA_repair,_bacterial_photolyase@1>Deoxyribodipyrimidine photolyase (EC 4.1.99.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5300@1>fig|2665645.4.peg.5300</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=2-phosphoglycolate_salvage&organism=2665645.4@1>2-phosphoglycolate salvage</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoglycolate phosphatase (EC 3.1.3.18)&subsystem_name=2-phosphoglycolate_salvage@1>Phosphoglycolate phosphatase (EC 3.1.3.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3912@1>fig|2665645.4.peg.3912</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=2665645.4@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutS&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>DNA mismatch repair protein MutS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3558@1>fig|2665645.4.peg.3558</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=2665645.4@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4352@1>fig|2665645.4.peg.4352</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease VII large subunit (EC 3.1.11.6)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease VII large subunit (EC 3.1.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1035@1>fig|2665645.4.peg.1035</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RadA&subsystem_name=DNA_repair,_bacterial@1>DNA repair protein RadA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4513@1>fig|2665645.4.peg.4513</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ADA regulatory protein&subsystem_name=DNA_repair,_bacterial@1>ADA regulatory protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3711@1>fig|2665645.4.peg.3711</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RecN&subsystem_name=DNA_repair,_bacterial@1>DNA repair protein RecN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2495@1>fig|2665645.4.peg.2495</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination protein RmuC&subsystem_name=DNA_repair,_bacterial@1>DNA recombination protein RmuC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5290@1>fig|2665645.4.peg.5290</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=A/G-specific adenine glycosylase (EC 3.2.2.-)&subsystem_name=DNA_repair,_bacterial@1>A/G-specific adenine glycosylase (EC 3.2.2.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3753@1>fig|2665645.4.peg.3753</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-cytosine methyltransferase (EC 2.1.1.37)&subsystem_name=DNA_repair,_bacterial@1>DNA-cytosine methyltransferase (EC 2.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1051@1>fig|2665645.4.peg.1051</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1058@1>fig|2665645.4.peg.1058</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA polymerase IV (EC 2.7.7.7)&subsystem_name=DNA_repair,_bacterial@1>DNA polymerase IV (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3250@1>fig|2665645.4.peg.3250</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4435@1>fig|2665645.4.peg.4435</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Very-short-patch mismatch repair endonuclease (G-T specific)&subsystem_name=DNA_repair,_bacterial@1>Very-short-patch mismatch repair endonuclease (G-T specific)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1062@1>fig|2665645.4.peg.1062</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Single-stranded DNA-binding protein&subsystem_name=DNA_repair,_bacterial@1>Single-stranded DNA-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3752@1>fig|2665645.4.peg.3752</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4452@1>fig|2665645.4.peg.4452</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)&subsystem_name=DNA_repair,_bacterial@1>Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3711@1>fig|2665645.4.peg.3711</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4443@1>fig|2665645.4.peg.4443</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair,_bacterial@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4352@1>fig|2665645.4.peg.4352</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease III (EC 3.1.11.2)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease III (EC 3.1.11.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3253@1>fig|2665645.4.peg.3253</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4501@1>fig|2665645.4.peg.4501</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=2665645.4@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease VII small subunit (EC 3.1.11.6)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease VII small subunit (EC 3.1.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1036@1>fig|2665645.4.peg.1036</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=2665645.4@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA helicase RecQ&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>ATP-dependent DNA helicase RecQ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1719@1>fig|2665645.4.peg.1719</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5079@1>fig|2665645.4.peg.5079</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=2665645.4@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Single-stranded DNA-binding protein&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>Single-stranded DNA-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3752@1>fig|2665645.4.peg.3752</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4452@1>fig|2665645.4.peg.4452</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=2665645.4@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination protein RecR&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>Recombination protein RecR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5009@1>fig|2665645.4.peg.5009</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=2665645.4@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4352@1>fig|2665645.4.peg.4352</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=2665645.4@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination and repair protein RecO&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>DNA recombination and repair protein RecO</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4651@1>fig|2665645.4.peg.4651</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=2665645.4@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination and repair protein RecF&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>DNA recombination and repair protein RecF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5143@1>fig|2665645.4.peg.5143</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=2665645.4@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA helicase RecS (RecQ family)&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>ATP-dependent DNA helicase RecS (RecQ family)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5210@1>fig|2665645.4.peg.5210</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nonhomologous_End-Joining_in_Bacteria&organism=2665645.4@1>Nonhomologous End-Joining in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) LigC&subsystem_name=Nonhomologous_End-Joining_in_Bacteria@1>ATP-dependent DNA ligase (EC 6.5.1.1) LigC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1413@1>fig|2665645.4.peg.1413</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nonhomologous_End-Joining_in_Bacteria&organism=2665645.4@1>Nonhomologous End-Joining in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ku domain protein&subsystem_name=Nonhomologous_End-Joining_in_Bacteria@1>Ku domain protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2222@1>fig|2665645.4.peg.2222</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nonhomologous_End-Joining_in_Bacteria&organism=2665645.4@1>Nonhomologous End-Joining in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD&subsystem_name=Nonhomologous_End-Joining_in_Bacteria@1>ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2221@1>fig|2665645.4.peg.2221</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_Repair_Base_Excision&organism=2665645.4@1>DNA Repair Base Excision</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) LigC&subsystem_name=DNA_Repair_Base_Excision@1>ATP-dependent DNA ligase (EC 6.5.1.1) LigC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1413@1>fig|2665645.4.peg.1413</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_Repair_Base_Excision&organism=2665645.4@1>DNA Repair Base Excision</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Endonuclease III (EC 4.2.99.18)&subsystem_name=DNA_Repair_Base_Excision@1>Endonuclease III (EC 4.2.99.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1605@1>fig|2665645.4.peg.1605</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2803@1>fig|2665645.4.peg.2803</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_Repair_Base_Excision&organism=2665645.4@1>DNA Repair Base Excision</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-3-methyladenine glycosylase II (EC 3.2.2.21)&subsystem_name=DNA_Repair_Base_Excision@1>DNA-3-methyladenine glycosylase II (EC 3.2.2.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2163@1>fig|2665645.4.peg.2163</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_Repair_Base_Excision&organism=2665645.4@1>DNA Repair Base Excision</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-3-methyladenine glycosylase (EC 3.2.2.20)&subsystem_name=DNA_Repair_Base_Excision@1>DNA-3-methyladenine glycosylase (EC 3.2.2.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3675@1>fig|2665645.4.peg.3675</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_Repair_Base_Excision&organism=2665645.4@1>DNA Repair Base Excision</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ku domain protein&subsystem_name=DNA_Repair_Base_Excision@1>Ku domain protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2222@1>fig|2665645.4.peg.2222</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_Repair_Base_Excision&organism=2665645.4@1>DNA Repair Base Excision</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA polymerase I (EC 2.7.7.7)&subsystem_name=DNA_Repair_Base_Excision@1>DNA polymerase I (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3664@1>fig|2665645.4.peg.3664</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_Repair_Base_Excision&organism=2665645.4@1>DNA Repair Base Excision</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD&subsystem_name=DNA_Repair_Base_Excision@1>ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2221@1>fig|2665645.4.peg.2221</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=2665645.4@1>Type I Restriction-Modification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)&subsystem_name=Type_I_Restriction-Modification@1>Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.793@1>fig|2665645.4.peg.793</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.814@1>fig|2665645.4.peg.814</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=2665645.4@1>Type I Restriction-Modification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)&subsystem_name=Type_I_Restriction-Modification@1>Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.800@1>fig|2665645.4.peg.800</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.820@1>fig|2665645.4.peg.820</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=2665645.4@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)&subsystem_name=Restriction-Modification_System@1>Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.793@1>fig|2665645.4.peg.793</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.814@1>fig|2665645.4.peg.814</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=2665645.4@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)&subsystem_name=Restriction-Modification_System@1>Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.800@1>fig|2665645.4.peg.800</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.820@1>fig|2665645.4.peg.820</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=2665645.4@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative predicted metal-dependent hydrolase&subsystem_name=Restriction-Modification_System@1>Putative predicted metal-dependent hydrolase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.792@1>fig|2665645.4.peg.792</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_ligases&organism=2665645.4@1>DNA ligases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) LigC&subsystem_name=DNA_ligases@1>ATP-dependent DNA ligase (EC 6.5.1.1) LigC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1413@1>fig|2665645.4.peg.1413</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_ligases&organism=2665645.4@1>DNA ligases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ku domain protein&subsystem_name=DNA_ligases@1>Ku domain protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2222@1>fig|2665645.4.peg.2222</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_ligases&organism=2665645.4@1>DNA ligases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD&subsystem_name=DNA_ligases@1>ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2221@1>fig|2665645.4.peg.2221</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=YcfH&organism=2665645.4@1>YcfH</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribonuclease TatD&subsystem_name=YcfH@1>Deoxyribonuclease TatD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.457@1>fig|2665645.4.peg.457</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=YcfH&organism=2665645.4@1>YcfH</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative deoxyribonuclease YjjV&subsystem_name=YcfH@1>Putative deoxyribonuclease YjjV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1642@1>fig|2665645.4.peg.1642</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=2665645.4@1>DNA structural proteins, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-binding protein HU-beta&subsystem_name=DNA_structural_proteins,_bacterial@1>DNA-binding protein HU-beta</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5208@1>fig|2665645.4.peg.5208</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=2665645.4@1>DNA structural proteins, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Integration host factor alpha/beta&subsystem_name=DNA_structural_proteins,_bacterial@1>Integration host factor alpha/beta</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3754@1>fig|2665645.4.peg.3754</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=2665645.4@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypothetical protein PA2244 (similar to DNA topoisomerase IB, but possibly involved in glycosyl-transfer)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>Hypothetical protein PA2244 (similar to DNA topoisomerase IB, but possibly involved in glycosyl-transfer)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2388@1>fig|2665645.4.peg.2388</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=2665645.4@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase III (EC 5.99.1.2)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>DNA topoisomerase III (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2988@1>fig|2665645.4.peg.2988</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=2665645.4@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase I (EC 5.99.1.2)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>DNA topoisomerase I (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3421@1>fig|2665645.4.peg.3421</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_replication_strays&organism=2665645.4@1>DNA replication strays</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DEDDh 3@1-5@1 exonuclease domain of the epsilon subunit of DNA polymerase III&subsystem_name=DNA_replication_strays@1>DEDDh 3@1-5@1 exonuclease domain of the epsilon subunit of DNA polymerase III</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2757@1>fig|2665645.4.peg.2757</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_replication_strays&organism=2665645.4@1>DNA replication strays</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Error-prone repair homolog of DNA polymerase III alpha subunit (EC 2.7.7.7)&subsystem_name=DNA_replication_strays@1>Error-prone repair homolog of DNA polymerase III alpha subunit (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3251@1>fig|2665645.4.peg.3251</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=2665645.4@1>DNA topoisomerases, Type II, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit B (EC 5.99.1.3)&subsystem_name=DNA_topoisomerases,_Type_II,_ATP-dependent@1>DNA gyrase subunit B (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3015@1>fig|2665645.4.peg.3015</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=2665645.4@1>DNA topoisomerases, Type II, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit A (EC 5.99.1.3)&subsystem_name=DNA_topoisomerases,_Type_II,_ATP-dependent@1>DNA gyrase subunit A (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2629@1>fig|2665645.4.peg.2629</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=2665645.4@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination protein RecR&subsystem_name=DNA_processing_cluster@1>Recombination protein RecR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5009@1>fig|2665645.4.peg.5009</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=2665645.4@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase III (EC 5.99.1.2)&subsystem_name=DNA_processing_cluster@1>DNA topoisomerase III (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2988@1>fig|2665645.4.peg.2988</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=2665645.4@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA polymerase III subunits gamma and tau (EC 2.7.7.7)&subsystem_name=DNA_processing_cluster@1>DNA polymerase III subunits gamma and tau (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.724@1>fig|2665645.4.peg.724</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.725@1>fig|2665645.4.peg.725</a>@~Fatty Acids, Lipids, and Isoprenoids@^Triacylglycerols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Triacylglycerol_metabolism&organism=2665645.4@1>Triacylglycerol metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lysophospholipase (EC 3.1.1.5)&subsystem_name=Triacylglycerol_metabolism@1>Lysophospholipase (EC 3.1.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.943@1>fig|2665645.4.peg.943</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin carboxyl carrier protein of acetyl-CoA carboxylase&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Biotin carboxyl carrier protein of acetyl-CoA carboxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4717@1>fig|2665645.4.peg.4717</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4718@1>fig|2665645.4.peg.4718</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3839@1>fig|2665645.4.peg.3839</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acyl carrier protein&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acyl carrier protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1469@1>fig|2665645.4.peg.1469</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4172@1>fig|2665645.4.peg.4172</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4227@1>fig|2665645.4.peg.4227</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4246@1>fig|2665645.4.peg.4246</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5330@1>fig|2665645.4.peg.5330</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1470@1>fig|2665645.4.peg.1470</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4226@1>fig|2665645.4.peg.4226</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acyl carrier protein phosphodiesterase (EC 3.1.4.14)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acyl carrier protein phosphodiesterase (EC 3.1.4.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.769@1>fig|2665645.4.peg.769</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3549@1>fig|2665645.4.peg.3549</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=2665645.4@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1471@1>fig|2665645.4.peg.1471</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1538@1>fig|2665645.4.peg.1538</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1850@1>fig|2665645.4.peg.1850</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4254@1>fig|2665645.4.peg.4254</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=2665645.4@1>Fatty acid metabolism cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)&subsystem_name=Fatty_acid_metabolism_cluster@1>Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.470@1>fig|2665645.4.peg.470</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=2665645.4@1>Fatty acid metabolism cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Fatty_acid_metabolism_cluster@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4401@1>fig|2665645.4.peg.4401</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=2665645.4@1>Fatty acid metabolism cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-ketoacyl-CoA thiolase (EC 2.3.1.16)&subsystem_name=Fatty_acid_metabolism_cluster@1>3-ketoacyl-CoA thiolase (EC 2.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2315@1>fig|2665645.4.peg.2315</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Myxoxanthophyll_biosynthesis_in_Cyanobacteria&organism=2665645.4@1>Myxoxanthophyll biosynthesis in Cyanobacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-carotene hydroxylase&subsystem_name=Myxoxanthophyll_biosynthesis_in_Cyanobacteria@1>Beta-carotene hydroxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5302@1>fig|2665645.4.peg.5302</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=2665645.4@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diphosphomevalonate decarboxylase (EC 4.1.1.33)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Diphosphomevalonate decarboxylase (EC 4.1.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3918@1>fig|2665645.4.peg.3918</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=2665645.4@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomevalonate kinase (EC 2.7.4.2)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Phosphomevalonate kinase (EC 2.7.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3920@1>fig|2665645.4.peg.3920</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=2665645.4@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mevalonate kinase (EC 2.7.1.36)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Mevalonate kinase (EC 2.7.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3886@1>fig|2665645.4.peg.3886</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=2665645.4@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4688@1>fig|2665645.4.peg.4688</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=2665645.4@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3885@1>fig|2665645.4.peg.3885</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=2665645.4@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.604@1>fig|2665645.4.peg.604</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=2665645.4@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Undecaprenyl diphosphate synthase (EC 2.5.1.31)&subsystem_name=Isoprenoinds_for_Quinones@1>Undecaprenyl diphosphate synthase (EC 2.5.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3588@1>fig|2665645.4.peg.3588</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=2665645.4@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Geranylgeranyl diphosphate synthase (EC 2.5.1.29)&subsystem_name=Isoprenoinds_for_Quinones@1>Geranylgeranyl diphosphate synthase (EC 2.5.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4067@1>fig|2665645.4.peg.4067</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=2665645.4@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hexaprenyl diphosphate synthase (farnesyl-diphosphate specific) (EC 2.5.1.83)&subsystem_name=Isoprenoinds_for_Quinones@1>Hexaprenyl diphosphate synthase (farnesyl-diphosphate specific) (EC 2.5.1.83)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3493@1>fig|2665645.4.peg.3493</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=2665645.4@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mevalonate kinase (EC 2.7.1.36)&subsystem_name=Isoprenoid_Biosynthesis@1>Mevalonate kinase (EC 2.7.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3886@1>fig|2665645.4.peg.3886</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=2665645.4@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Isoprenoid_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.604@1>fig|2665645.4.peg.604</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=2665645.4@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Isoprenoid_Biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3765@1>fig|2665645.4.peg.3765</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=2665645.4@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diphosphomevalonate decarboxylase (EC 4.1.1.33)&subsystem_name=Isoprenoid_Biosynthesis@1>Diphosphomevalonate decarboxylase (EC 4.1.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3918@1>fig|2665645.4.peg.3918</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=2665645.4@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Undecaprenyl diphosphate synthase (EC 2.5.1.31)&subsystem_name=Isoprenoid_Biosynthesis@1>Undecaprenyl diphosphate synthase (EC 2.5.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3588@1>fig|2665645.4.peg.3588</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=2665645.4@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomevalonate kinase (EC 2.7.4.2)&subsystem_name=Isoprenoid_Biosynthesis@1>Phosphomevalonate kinase (EC 2.7.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3920@1>fig|2665645.4.peg.3920</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=2665645.4@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)&subsystem_name=Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4688@1>fig|2665645.4.peg.4688</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=2665645.4@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)&subsystem_name=Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3885@1>fig|2665645.4.peg.3885</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=2665645.4@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Ammonia_assimilation@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2920@1>fig|2665645.4.peg.2920</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=2665645.4@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] small chain (EC 1.4.1.13)&subsystem_name=Ammonia_assimilation@1>Glutamate synthase [NADPH] small chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3117@1>fig|2665645.4.peg.3117</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=2665645.4@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)&subsystem_name=Ammonia_assimilation@1>Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2833@1>fig|2665645.4.peg.2833</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=2665645.4@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ammonium transporter&subsystem_name=Ammonia_assimilation@1>Ammonium transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3152@1>fig|2665645.4.peg.3152</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3975@1>fig|2665645.4.peg.3975</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=2665645.4@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)&subsystem_name=Ammonia_assimilation@1>Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2923@1>fig|2665645.4.peg.2923</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=2665645.4@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] large chain (EC 1.4.1.13)&subsystem_name=Ammonia_assimilation@1>Glutamate synthase [NADPH] large chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3116@1>fig|2665645.4.peg.3116</a>@~Dormancy and Sporulation@^Dormancy and Sporulation - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Sporulation-associated_proteins_with_broader_functions&organism=2665645.4@1>Sporulation-associated proteins with broader functions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidyl-tRNA hydrolase (EC 3.1.1.29)&subsystem_name=Sporulation-associated_proteins_with_broader_functions@1>Peptidyl-tRNA hydrolase (EC 3.1.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4184@1>fig|2665645.4.peg.4184</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit c (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit c (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4277@1>fig|2665645.4.peg.4277</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase delta chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase delta chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4275@1>fig|2665645.4.peg.4275</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase protein I&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase protein I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4279@1>fig|2665645.4.peg.4279</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4280@1>fig|2665645.4.peg.4280</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase beta chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase beta chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4524@1>fig|2665645.4.peg.4524</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase gamma chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase gamma chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4273@1>fig|2665645.4.peg.4273</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit b (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit b (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4276@1>fig|2665645.4.peg.4276</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit a (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit a (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4278@1>fig|2665645.4.peg.4278</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase alpha chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase alpha chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4274@1>fig|2665645.4.peg.4274</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=2665645.4@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase epsilon chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase epsilon chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4525@1>fig|2665645.4.peg.4525</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_d_ubiquinol_oxidases&organism=2665645.4@1>Terminal cytochrome d ubiquinol oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_d_ubiquinol_oxidases@1>Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.217@1>fig|2665645.4.peg.217</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_d_ubiquinol_oxidases&organism=2665645.4@1>Terminal cytochrome d ubiquinol oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_d_ubiquinol_oxidases@1>Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.218@1>fig|2665645.4.peg.218</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=2665645.4@1>Anaerobic respiratory reductases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Flavodoxin reductases (ferredoxin-NADPH reductases) family 1&subsystem_name=Anaerobic_respiratory_reductases@1>Flavodoxin reductases (ferredoxin-NADPH reductases) family 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3684@1>fig|2665645.4.peg.3684</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4285@1>fig|2665645.4.peg.4285</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_oxidases&organism=2665645.4@1>Terminal cytochrome oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_oxidases@1>Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.217@1>fig|2665645.4.peg.217</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_oxidases&organism=2665645.4@1>Terminal cytochrome oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_oxidases@1>Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.218@1>fig|2665645.4.peg.218</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4098@1>fig|2665645.4.peg.4098</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4099@1>fig|2665645.4.peg.4099</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4097@1>fig|2665645.4.peg.4097</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4101@1>fig|2665645.4.peg.4101</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4095@1>fig|2665645.4.peg.4095</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4092@1>fig|2665645.4.peg.4092</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4088@1>fig|2665645.4.peg.4088</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4096@1>fig|2665645.4.peg.4096</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4100@1>fig|2665645.4.peg.4100</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4093@1>fig|2665645.4.peg.4093</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4094@1>fig|2665645.4.peg.4094</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4089@1>fig|2665645.4.peg.4089</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4087@1>fig|2665645.4.peg.4087</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=2665645.4@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4090@1>fig|2665645.4.peg.4090</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=2665645.4@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Proline dehydrogenase (EC 1.5.99.8) (Proline oxidase)&subsystem_name=Respiratory_dehydrogenases_1@1>Proline dehydrogenase (EC 1.5.99.8) (Proline oxidase)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2170@1>fig|2665645.4.peg.2170</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=2665645.4@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH dehydrogenase (EC 1.6.99.3)&subsystem_name=Respiratory_dehydrogenases_1@1>NADH dehydrogenase (EC 1.6.99.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1793@1>fig|2665645.4.peg.1793</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=2665645.4@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-methanol-choline (GMC) oxidoreductase:NAD binding site&subsystem_name=Respiratory_dehydrogenases_1@1>Glucose-methanol-choline (GMC) oxidoreductase:NAD binding site</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1146@1>fig|2665645.4.peg.1146</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=2665645.4@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)&subsystem_name=Respiratory_dehydrogenases_1@1>Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.209@1>fig|2665645.4.peg.209</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4098@1>fig|2665645.4.peg.4098</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4099@1>fig|2665645.4.peg.4099</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4097@1>fig|2665645.4.peg.4097</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4101@1>fig|2665645.4.peg.4101</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4095@1>fig|2665645.4.peg.4095</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4092@1>fig|2665645.4.peg.4092</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4088@1>fig|2665645.4.peg.4088</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4096@1>fig|2665645.4.peg.4096</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4100@1>fig|2665645.4.peg.4100</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4093@1>fig|2665645.4.peg.4093</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4094@1>fig|2665645.4.peg.4094</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4089@1>fig|2665645.4.peg.4089</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4087@1>fig|2665645.4.peg.4087</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=2665645.4@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4090@1>fig|2665645.4.peg.4090</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Quinone_oxidoreductase_family&organism=2665645.4@1>Quinone oxidoreductase family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinone oxidoreductase (EC 1.6.5.5)&subsystem_name=Quinone_oxidoreductase_family@1>Quinone oxidoreductase (EC 1.6.5.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1783@1>fig|2665645.4.peg.1783</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5083@1>fig|2665645.4.peg.5083</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cytochrome_c_oxidases&organism=2665645.4@1>Biogenesis of cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Heme A synthase, cytochrome oxidase biogenesis protein Cox15-CtaA&subsystem_name=Biogenesis_of_cytochrome_c_oxidases@1>Heme A synthase, cytochrome oxidase biogenesis protein Cox15-CtaA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.181@1>fig|2665645.4.peg.181</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=2665645.4@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoI&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoI</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2698@1>fig|2665645.4.peg.2698</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=2665645.4@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoS, involved in heme b insertion&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoS, involved in heme b insertion</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2697@1>fig|2665645.4.peg.2697</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=2665645.4@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoG, involved in Cu oxidation&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoG, involved in Cu oxidation</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2693@1>fig|2665645.4.peg.2693</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=2665645.4@1>Biogenesis of c-type cytochromes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative cytochrome C-type biogenesis protein&subsystem_name=Biogenesis_of_c-type_cytochromes@1>Putative cytochrome C-type biogenesis protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1668@1>fig|2665645.4.peg.1668</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=2665645.4@1>Biogenesis of c-type cytochromes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)&subsystem_name=Biogenesis_of_c-type_cytochromes@1>Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1386@1>fig|2665645.4.peg.1386</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3153@1>fig|2665645.4.peg.3153</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=2665645.4@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3988@1>fig|2665645.4.peg.3988</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=2665645.4@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c551/c552&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Cytochrome c551/c552</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.623@1>fig|2665645.4.peg.623</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2586@1>fig|2665645.4.peg.2586</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=2665645.4@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Ferredoxin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2770@1>fig|2665645.4.peg.2770</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=2665645.4@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c552&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Cytochrome c552</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4383@1>fig|2665645.4.peg.4383</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=2665645.4@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane protein A precursor&subsystem_name=Osmoregulation@1>Outer membrane protein A precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4457@1>fig|2665645.4.peg.4457</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=2665645.4@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aquaporin Z&subsystem_name=Osmoregulation@1>Aquaporin Z</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2901@1>fig|2665645.4.peg.2901</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=2665645.4@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol uptake facilitator protein&subsystem_name=Osmoregulation@1>Glycerol uptake facilitator protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.207@1>fig|2665645.4.peg.207</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Organic hydroperoxide resistance transcriptional regulator&subsystem_name=Oxidative_stress@1>Organic hydroperoxide resistance transcriptional regulator</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1176@1>fig|2665645.4.peg.1176</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional regulator, Crp/Fnr family&subsystem_name=Oxidative_stress@1>Transcriptional regulator, Crp/Fnr family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.223@1>fig|2665645.4.peg.223</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.578@1>fig|2665645.4.peg.578</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1132@1>fig|2665645.4.peg.1132</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1191@1>fig|2665645.4.peg.1191</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2699@1>fig|2665645.4.peg.2699</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3698@1>fig|2665645.4.peg.3698</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Oxidative_stress@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.50@1>fig|2665645.4.peg.50</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3176@1>fig|2665645.4.peg.3176</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=bacteriophytochrome heme oxygenase BphO&subsystem_name=Oxidative_stress@1>bacteriophytochrome heme oxygenase BphO</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1341@1>fig|2665645.4.peg.1341</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)&subsystem_name=Oxidative_stress@1>Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.442@1>fig|2665645.4.peg.442</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric uptake regulation protein FUR&subsystem_name=Oxidative_stress@1>Ferric uptake regulation protein FUR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3050@1>fig|2665645.4.peg.3050</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Superoxide dismutase [Mn] (EC 1.15.1.1)&subsystem_name=Oxidative_stress@1>Superoxide dismutase [Mn] (EC 1.15.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2619@1>fig|2665645.4.peg.2619</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4183@1>fig|2665645.4.peg.4183</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Organic hydroperoxide resistance protein&subsystem_name=Oxidative_stress@1>Organic hydroperoxide resistance protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1177@1>fig|2665645.4.peg.1177</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1439@1>fig|2665645.4.peg.1439</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=2665645.4@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phytochrome, two-component sensor histidine kinase (EC 2.7.3.-)&subsystem_name=Oxidative_stress@1>Phytochrome, two-component sensor histidine kinase (EC 2.7.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1342@1>fig|2665645.4.peg.1342</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rubrerythrin&organism=2665645.4@1>Rubrerythrin</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rubredoxin&subsystem_name=Rubrerythrin@1>Rubredoxin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.576@1>fig|2665645.4.peg.576</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4030@1>fig|2665645.4.peg.4030</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rubrerythrin&organism=2665645.4@1>Rubrerythrin</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Rubrerythrin@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.50@1>fig|2665645.4.peg.50</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3176@1>fig|2665645.4.peg.3176</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Redox_cycle&organism=2665645.4@1>Glutathione: Redox cycle</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutathione peroxidase (EC 1.11.1.9)&subsystem_name=Glutathione:_Redox_cycle@1>Glutathione peroxidase (EC 1.11.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1680@1>fig|2665645.4.peg.1680</a>@~Stress Response@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Uptake_of_selenate_and_selenite&organism=2665645.4@1>Uptake of selenate and selenite</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DedA protein&subsystem_name=Uptake_of_selenate_and_selenite@1>DedA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3848@1>fig|2665645.4.peg.3848</a>@~Stress Response@^Stress Response - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dimethylarginine_metabolism&organism=2665645.4@1>Dimethylarginine metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ornithine aminotransferase (EC 2.6.1.13)&subsystem_name=Dimethylarginine_metabolism@1>Ornithine aminotransferase (EC 2.6.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.550@1>fig|2665645.4.peg.550</a>@~Stress Response@^Stress Response - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dimethylarginine_metabolism&organism=2665645.4@1>Dimethylarginine metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NG,NG-dimethylarginine dimethylaminohydrolase 1 (EC 3.5.3.18)&subsystem_name=Dimethylarginine_metabolism@1>NG,NG-dimethylarginine dimethylaminohydrolase 1 (EC 3.5.3.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5004@1>fig|2665645.4.peg.5004</a>@~Stress Response@^Periplasmic Stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=2665645.4@1>Periplasmic Stress Response</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane protein H precursor&subsystem_name=Periplasmic_Stress_Response@1>Outer membrane protein H precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3590@1>fig|2665645.4.peg.3590</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3591@1>fig|2665645.4.peg.3591</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4339@1>fig|2665645.4.peg.4339</a>@~Stress Response@^Periplasmic Stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=2665645.4@1>Periplasmic Stress Response</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Intramembrane protease RasP/YluC, implicated in cell division based on FtsL cleavage&subsystem_name=Periplasmic_Stress_Response@1>Intramembrane protease RasP/YluC, implicated in cell division based on FtsL cleavage</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4160@1>fig|2665645.4.peg.4160</a>@~Metabolism of Aromatic Compounds@^Peripheral pathways for catabolism of aromatic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Quinate_degradation&organism=2665645.4@1>Quinate degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate dehydratase II (EC 4.2.1.10)&subsystem_name=Quinate_degradation@1>3-dehydroquinate dehydratase II (EC 4.2.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5219@1>fig|2665645.4.peg.5219</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=2665645.4@1>Catechol branch of beta-ketoadipate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)&subsystem_name=Catechol_branch_of_beta-ketoadipate_pathway@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4428@1>fig|2665645.4.peg.4428</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=2665645.4@1>Catechol branch of beta-ketoadipate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)&subsystem_name=Catechol_branch_of_beta-ketoadipate_pathway@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4430@1>fig|2665645.4.peg.4430</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=2665645.4@1>Salicylate and gentisate catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetase (EC 3.7.1.2)&subsystem_name=Salicylate_and_gentisate_catabolism@1>Fumarylacetoacetase (EC 3.7.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2444@1>fig|2665645.4.peg.2444</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=2665645.4@1>Salicylate and gentisate catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetate hydrolase family protein&subsystem_name=Salicylate_and_gentisate_catabolism@1>Fumarylacetoacetate hydrolase family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.91@1>fig|2665645.4.peg.91</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4220@1>fig|2665645.4.peg.4220</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=2665645.4@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homogentisate 1,2-dioxygenase (EC 1.13.11.5)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>Homogentisate 1,2-dioxygenase (EC 1.13.11.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4627@1>fig|2665645.4.peg.4627</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=2665645.4@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetase (EC 3.7.1.2)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>Fumarylacetoacetase (EC 3.7.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2444@1>fig|2665645.4.peg.2444</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=2665645.4@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4626@1>fig|2665645.4.peg.4626</a>@~Metabolism of Aromatic Compounds@^Metabolism of Aromatic Compounds - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Gentisate_degradation&organism=2665645.4@1>Gentisate degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetate hydrolase family protein&subsystem_name=Gentisate_degradation@1>Fumarylacetoacetate hydrolase family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.91@1>fig|2665645.4.peg.91</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4220@1>fig|2665645.4.peg.4220</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-asparaginase I, cytoplasmic (EC 3.5.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>L-asparaginase I, cytoplasmic (EC 3.5.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4051@1>fig|2665645.4.peg.4051</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2833@1>fig|2665645.4.peg.2833</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-asparaginase (EC 3.5.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>L-asparaginase (EC 3.5.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1014@1>fig|2665645.4.peg.1014</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate aminotransferase (EC 2.6.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate aminotransferase (EC 2.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3450@1>fig|2665645.4.peg.3450</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3531@1>fig|2665645.4.peg.3531</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4838@1>fig|2665645.4.peg.4838</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5364@1>fig|2665645.4.peg.5364</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-specific glutamate dehydrogenase (EC 1.4.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>NAD-specific glutamate dehydrogenase (EC 1.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.906@1>fig|2665645.4.peg.906</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate racemase (EC 5.1.1.3)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate racemase (EC 5.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3592@1>fig|2665645.4.peg.3592</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3018@1>fig|2665645.4.peg.3018</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3020@1>fig|2665645.4.peg.3020</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2923@1>fig|2665645.4.peg.2923</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] large chain (EC 1.4.1.13)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate synthase [NADPH] large chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3116@1>fig|2665645.4.peg.3116</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2920@1>fig|2665645.4.peg.2920</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] small chain (EC 1.4.1.13)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate synthase [NADPH] small chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3117@1>fig|2665645.4.peg.3117</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4649@1>fig|2665645.4.peg.4649</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate ammonia-lyase (EC 4.3.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate ammonia-lyase (EC 4.3.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1101@1>fig|2665645.4.peg.1101</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2437@1>fig|2665645.4.peg.2437</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=2665645.4@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate racemase (EC 5.1.1.13)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate racemase (EC 5.1.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.511@1>fig|2665645.4.peg.511</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2740@1>fig|2665645.4.peg.2740</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3186@1>fig|2665645.4.peg.3186</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=2665645.4@1>Glutamate dehydrogenases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Glutamate_dehydrogenases@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4649@1>fig|2665645.4.peg.4649</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=2665645.4@1>Glutamate dehydrogenases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-specific glutamate dehydrogenase (EC 1.4.1.2)&subsystem_name=Glutamate_dehydrogenases@1>NAD-specific glutamate dehydrogenase (EC 1.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.906@1>fig|2665645.4.peg.906</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=2665645.4@1>Glutamine synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Glutamine_synthetases@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2920@1>fig|2665645.4.peg.2920</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=2665645.4@1>Glutamine synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)&subsystem_name=Glutamine_synthetases@1>Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2923@1>fig|2665645.4.peg.2923</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=2665645.4@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formiminoglutamase (EC 3.5.3.8)&subsystem_name=Histidine_Degradation@1>Formiminoglutamase (EC 3.5.3.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.673@1>fig|2665645.4.peg.673</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=2665645.4@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Imidazolonepropionase (EC 3.5.2.7)&subsystem_name=Histidine_Degradation@1>Imidazolonepropionase (EC 3.5.2.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.675@1>fig|2665645.4.peg.675</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=2665645.4@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urocanate hydratase (EC 4.2.1.49)&subsystem_name=Histidine_Degradation@1>Urocanate hydratase (EC 4.2.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3320@1>fig|2665645.4.peg.3320</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=2665645.4@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidine ammonia-lyase (EC 4.3.1.3)&subsystem_name=Histidine_Degradation@1>Histidine ammonia-lyase (EC 4.3.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2639@1>fig|2665645.4.peg.2639</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=2665645.4@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)&subsystem_name=Histidine_Biosynthesis@1>Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1817@1>fig|2665645.4.peg.1817</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=2665645.4@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidinol-phosphate aminotransferase (EC 2.6.1.9)&subsystem_name=Histidine_Biosynthesis@1>Histidinol-phosphate aminotransferase (EC 2.6.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1821@1>fig|2665645.4.peg.1821</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=2665645.4@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosyl-ATP pyrophosphatase (EC 3.6.1.31)&subsystem_name=Histidine_Biosynthesis@1>Phosphoribosyl-ATP pyrophosphatase (EC 3.6.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1814@1>fig|2665645.4.peg.1814</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=2665645.4@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Imidazoleglycerol-phosphate dehydratase (EC 4.2.1.19)&subsystem_name=Histidine_Biosynthesis@1>Imidazoleglycerol-phosphate dehydratase (EC 4.2.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1819@1>fig|2665645.4.peg.1819</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=2665645.4@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosyl-AMP cyclohydrolase (EC 3.5.4.19)&subsystem_name=Histidine_Biosynthesis@1>Phosphoribosyl-AMP cyclohydrolase (EC 3.5.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1814@1>fig|2665645.4.peg.1814</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=2665645.4@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidinol-phosphatase (EC 3.1.3.15)&subsystem_name=Histidine_Biosynthesis@1>Histidinol-phosphatase (EC 3.1.3.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1819@1>fig|2665645.4.peg.1819</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=2665645.4@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidinol dehydrogenase (EC 1.1.1.23)&subsystem_name=Histidine_Biosynthesis@1>Histidinol dehydrogenase (EC 1.1.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1822@1>fig|2665645.4.peg.1822</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=2665645.4@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Spermidine synthase (EC 2.5.1.16)&subsystem_name=Polyamine_Metabolism@1>Spermidine synthase (EC 2.5.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4037@1>fig|2665645.4.peg.4037</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=2665645.4@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Agmatine deiminase (EC 3.5.3.12)&subsystem_name=Polyamine_Metabolism@1>Agmatine deiminase (EC 3.5.3.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2445@1>fig|2665645.4.peg.2445</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=2665645.4@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biosynthetic arginine decarboxylase (EC 4.1.1.19)&subsystem_name=Polyamine_Metabolism@1>Biosynthetic arginine decarboxylase (EC 4.1.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1722@1>fig|2665645.4.peg.1722</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=2665645.4@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Spermidine N1-acetyltransferase (EC 2.3.1.57)&subsystem_name=Polyamine_Metabolism@1>Spermidine N1-acetyltransferase (EC 2.3.1.57)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5356@1>fig|2665645.4.peg.5356</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=2665645.4@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-methylthioadenosine nucleosidase (EC 3.2.2.16)&subsystem_name=Polyamine_Metabolism@1>5@1-methylthioadenosine nucleosidase (EC 3.2.2.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2141@1>fig|2665645.4.peg.2141</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter ATP-binding protein&subsystem_name=Methionine_Biosynthesis@1>Methionine ABC transporter ATP-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1403@1>fig|2665645.4.peg.1403</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylmethionine synthetase (EC 2.5.1.6)&subsystem_name=Methionine_Biosynthesis@1>S-adenosylmethionine synthetase (EC 2.5.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3812@1>fig|2665645.4.peg.3812</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylhomoserine sulfhydrylase (EC 2.5.1.48)&subsystem_name=Methionine_Biosynthesis@1>O-succinylhomoserine sulfhydrylase (EC 2.5.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3162@1>fig|2665645.4.peg.3162</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3800@1>fig|2665645.4.peg.3800</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3808@1>fig|2665645.4.peg.3808</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Methionine_Biosynthesis@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4647@1>fig|2665645.4.peg.4647</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine acetyltransferase (EC 2.3.1.30)&subsystem_name=Methionine_Biosynthesis@1>Serine acetyltransferase (EC 2.3.1.30)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2732@1>fig|2665645.4.peg.2732</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine kinase (EC 2.7.1.39)&subsystem_name=Methionine_Biosynthesis@1>Homoserine kinase (EC 2.7.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2643@1>fig|2665645.4.peg.2643</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3405@1>fig|2665645.4.peg.3405</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase (EC 2.1.1.14)&subsystem_name=Methionine_Biosynthesis@1>5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase (EC 2.1.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1909@1>fig|2665645.4.peg.1909</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3771@1>fig|2665645.4.peg.3771</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine synthase (EC 2.5.1.47)&subsystem_name=Methionine_Biosynthesis@1>Cysteine synthase (EC 2.5.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2733@1>fig|2665645.4.peg.2733</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-acetylhomoserine sulfhydrylase (EC 2.5.1.49)&subsystem_name=Methionine_Biosynthesis@1>O-acetylhomoserine sulfhydrylase (EC 2.5.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3162@1>fig|2665645.4.peg.3162</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3800@1>fig|2665645.4.peg.3800</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3808@1>fig|2665645.4.peg.3808</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=Methionine_Biosynthesis@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3783@1>fig|2665645.4.peg.3783</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Methionine_Biosynthesis@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2670@1>fig|2665645.4.peg.2670</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3296@1>fig|2665645.4.peg.3296</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine dehydrogenase (EC 1.1.1.3)&subsystem_name=Methionine_Biosynthesis@1>Homoserine dehydrogenase (EC 1.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2642@1>fig|2665645.4.peg.2642</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3803@1>fig|2665645.4.peg.3803</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3807@1>fig|2665645.4.peg.3807</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter substrate-binding protein&subsystem_name=Methionine_Biosynthesis@1>Methionine ABC transporter substrate-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1405@1>fig|2665645.4.peg.1405</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine O-acetyltransferase (EC 2.3.1.31)&subsystem_name=Methionine_Biosynthesis@1>Homoserine O-acetyltransferase (EC 2.3.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3804@1>fig|2665645.4.peg.3804</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)&subsystem_name=Methionine_Biosynthesis@1>S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2141@1>fig|2665645.4.peg.2141</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter permease protein&subsystem_name=Methionine_Biosynthesis@1>Methionine ABC transporter permease protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1404@1>fig|2665645.4.peg.1404</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator MetR&subsystem_name=Methionine_Biosynthesis@1>Transcriptional activator MetR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4446@1>fig|2665645.4.peg.4446</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=2665645.4@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylhomocysteinase (EC 3.3.1.1)&subsystem_name=Methionine_Biosynthesis@1>Adenosylhomocysteinase (EC 3.3.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3702@1>fig|2665645.4.peg.3702</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=2665645.4@1>Threonine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-allo-threonine aldolase (EC 2.1.2.1)&subsystem_name=Threonine_degradation@1>L-allo-threonine aldolase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2674@1>fig|2665645.4.peg.2674</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=2665645.4@1>Threonine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=low-specificity D-threonine aldolase&subsystem_name=Threonine_degradation@1>low-specificity D-threonine aldolase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4695@1>fig|2665645.4.peg.4695</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate decarboxylase (EC 4.1.1.20)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Diaminopimelate decarboxylase (EC 4.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3118@1>fig|2665645.4.peg.3118</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2708@1>fig|2665645.4.peg.2708</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate epimerase (EC 5.1.1.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Diaminopimelate epimerase (EC 5.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5355@1>fig|2665645.4.peg.5355</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3991@1>fig|2665645.4.peg.3991</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5152@1>fig|2665645.4.peg.5152</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2642@1>fig|2665645.4.peg.2642</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3803@1>fig|2665645.4.peg.3803</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4811@1>fig|2665645.4.peg.4811</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2500@1>fig|2665645.4.peg.2500</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=2665645.4@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)&subsystem_name=Methionine_Degradation@1>S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2141@1>fig|2665645.4.peg.2141</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=2665645.4@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter substrate-binding protein&subsystem_name=Methionine_Degradation@1>Methionine ABC transporter substrate-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1405@1>fig|2665645.4.peg.1405</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=2665645.4@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter ATP-binding protein&subsystem_name=Methionine_Degradation@1>Methionine ABC transporter ATP-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1403@1>fig|2665645.4.peg.1403</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=2665645.4@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter permease protein&subsystem_name=Methionine_Degradation@1>Methionine ABC transporter permease protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1404@1>fig|2665645.4.peg.1404</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=2665645.4@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylmethionine synthetase (EC 2.5.1.6)&subsystem_name=Methionine_Degradation@1>S-adenosylmethionine synthetase (EC 2.5.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3812@1>fig|2665645.4.peg.3812</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=2665645.4@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Methionine_Degradation@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4647@1>fig|2665645.4.peg.4647</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=2665645.4@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylhomocysteinase (EC 3.3.1.1)&subsystem_name=Methionine_Degradation@1>Adenosylhomocysteinase (EC 3.3.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3702@1>fig|2665645.4.peg.3702</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=2665645.4@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Methionine_Degradation@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2670@1>fig|2665645.4.peg.2670</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3296@1>fig|2665645.4.peg.3296</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=2665645.4@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine dehydrogenase (EC 1.1.1.3)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Homoserine dehydrogenase (EC 1.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2642@1>fig|2665645.4.peg.2642</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3803@1>fig|2665645.4.peg.3803</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3807@1>fig|2665645.4.peg.3807</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=2665645.4@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3991@1>fig|2665645.4.peg.3991</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=2665645.4@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate aminotransferase (EC 2.6.1.1)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartate aminotransferase (EC 2.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3450@1>fig|2665645.4.peg.3450</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3531@1>fig|2665645.4.peg.3531</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4838@1>fig|2665645.4.peg.4838</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5364@1>fig|2665645.4.peg.5364</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=2665645.4@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonine synthase (EC 4.2.3.1)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Threonine synthase (EC 4.2.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2644@1>fig|2665645.4.peg.2644</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=2665645.4@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine kinase (EC 2.7.1.39)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Homoserine kinase (EC 2.7.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2643@1>fig|2665645.4.peg.2643</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3405@1>fig|2665645.4.peg.3405</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=2665645.4@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2642@1>fig|2665645.4.peg.2642</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3803@1>fig|2665645.4.peg.3803</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4811@1>fig|2665645.4.peg.4811</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=2665645.4@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfate adenylyltransferase subunit 2 (EC 2.7.7.4)&subsystem_name=Cysteine_Biosynthesis@1>Sulfate adenylyltransferase subunit 2 (EC 2.7.7.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3794@1>fig|2665645.4.peg.3794</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=2665645.4@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfate adenylyltransferase subunit 1 (EC 2.7.7.4)&subsystem_name=Cysteine_Biosynthesis@1>Sulfate adenylyltransferase subunit 1 (EC 2.7.7.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3793@1>fig|2665645.4.peg.3793</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=2665645.4@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine synthase (EC 2.5.1.47)&subsystem_name=Cysteine_Biosynthesis@1>Cysteine synthase (EC 2.5.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2733@1>fig|2665645.4.peg.2733</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=2665645.4@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfate permease&subsystem_name=Cysteine_Biosynthesis@1>Sulfate permease</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3434@1>fig|2665645.4.peg.3434</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=2665645.4@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Cysteine_Biosynthesis@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4647@1>fig|2665645.4.peg.4647</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=2665645.4@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylylsulfate kinase (EC 2.7.1.25)&subsystem_name=Cysteine_Biosynthesis@1>Adenylylsulfate kinase (EC 2.7.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.55@1>fig|2665645.4.peg.55</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=2665645.4@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine acetyltransferase (EC 2.3.1.30)&subsystem_name=Cysteine_Biosynthesis@1>Serine acetyltransferase (EC 2.3.1.30)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2732@1>fig|2665645.4.peg.2732</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=2665645.4@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Cysteine_Biosynthesis@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2670@1>fig|2665645.4.peg.2670</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3296@1>fig|2665645.4.peg.3296</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate decarboxylase (EC 4.1.1.20)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Diaminopimelate decarboxylase (EC 4.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3118@1>fig|2665645.4.peg.3118</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2708@1>fig|2665645.4.peg.2708</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate epimerase (EC 5.1.1.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Diaminopimelate epimerase (EC 5.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5355@1>fig|2665645.4.peg.5355</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3991@1>fig|2665645.4.peg.3991</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5152@1>fig|2665645.4.peg.5152</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2642@1>fig|2665645.4.peg.2642</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3803@1>fig|2665645.4.peg.3803</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4811@1>fig|2665645.4.peg.4811</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=2665645.4@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2500@1>fig|2665645.4.peg.2500</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=2665645.4@1>Lysine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lysine ketoglutarate reductase (EC 1.5.1.8) (LOR) (LKR)&subsystem_name=Lysine_degradation@1>Lysine ketoglutarate reductase (EC 1.5.1.8) (LOR) (LKR)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.415@1>fig|2665645.4.peg.415</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=2665645.4@1>Lysine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Saccharopine dehydrogenase (EC 1.5.1.9)&subsystem_name=Lysine_degradation@1>Saccharopine dehydrogenase (EC 1.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.415@1>fig|2665645.4.peg.415</a>@~Amino Acids and Derivatives@^Amino Acids and Derivatives - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Creatine_and_Creatinine_Degradation&organism=2665645.4@1>Creatine and Creatinine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytosine deaminase (EC 3.5.4.1)&subsystem_name=Creatine_and_Creatinine_Degradation@1>Cytosine deaminase (EC 3.5.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2400@1>fig|2665645.4.peg.2400</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=2665645.4@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Isoleucine_degradation@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4401@1>fig|2665645.4.peg.4401</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=2665645.4@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Isoleucine_degradation@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.433@1>fig|2665645.4.peg.433</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=2665645.4@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Isoleucine_degradation@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.89@1>fig|2665645.4.peg.89</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=2665645.4@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Isoleucine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.753@1>fig|2665645.4.peg.753</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=2665645.4@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Isoleucine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.753@1>fig|2665645.4.peg.753</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=2665645.4@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-ketoacyl-CoA thiolase (EC 2.3.1.16)&subsystem_name=Isoleucine_degradation@1>3-ketoacyl-CoA thiolase (EC 2.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2315@1>fig|2665645.4.peg.2315</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=2665645.4@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)&subsystem_name=Isoleucine_degradation@1>Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.535@1>fig|2665645.4.peg.535</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA lyase (EC 4.1.3.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Hydroxymethylglutaryl-CoA lyase (EC 4.1.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3064@1>fig|2665645.4.peg.3064</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.433@1>fig|2665645.4.peg.433</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylglutaconyl-CoA hydratase (EC 4.2.1.18)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Methylglutaconyl-CoA hydratase (EC 4.2.1.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4742@1>fig|2665645.4.peg.4742</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylcrotonyl-CoA carboxylase carboxyl transferase subunit (EC 6.4.1.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Methylcrotonyl-CoA carboxylase carboxyl transferase subunit (EC 6.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2686@1>fig|2665645.4.peg.2686</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.753@1>fig|2665645.4.peg.753</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4430@1>fig|2665645.4.peg.4430</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.89@1>fig|2665645.4.peg.89</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4428@1>fig|2665645.4.peg.4428</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.753@1>fig|2665645.4.peg.753</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4688@1>fig|2665645.4.peg.4688</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.535@1>fig|2665645.4.peg.535</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.535@1>fig|2665645.4.peg.535</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2413@1>fig|2665645.4.peg.2413</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3159@1>fig|2665645.4.peg.3159</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=2665645.4@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3885@1>fig|2665645.4.peg.3885</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4008@1>fig|2665645.4.peg.4008</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-isopropylmalate synthase (EC 2.3.3.13)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>2-isopropylmalate synthase (EC 2.3.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1809@1>fig|2665645.4.peg.1809</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.433@1>fig|2665645.4.peg.433</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydrogenase (EC 1.1.1.85)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>3-isopropylmalate dehydrogenase (EC 1.1.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1808@1>fig|2665645.4.peg.1808</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4011@1>fig|2665645.4.peg.4011</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=(R)-citramalate synthase (EC 2.3.1.182)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>(R)-citramalate synthase (EC 2.3.1.182)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4010@1>fig|2665645.4.peg.4010</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4007@1>fig|2665645.4.peg.4007</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonine dehydratase biosynthetic (EC 4.3.1.19)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Threonine dehydratase biosynthetic (EC 4.3.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3322@1>fig|2665645.4.peg.3322</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase large subunit (EC 2.2.1.6)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Acetolactate synthase large subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1805@1>fig|2665645.4.peg.1805</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroxy-acid dehydratase (EC 4.2.1.9)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Dihydroxy-acid dehydratase (EC 4.2.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1807@1>fig|2665645.4.peg.1807</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=2665645.4@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase small subunit (EC 2.2.1.6)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Acetolactate synthase small subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1804@1>fig|2665645.4.peg.1804</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2104@1>fig|2665645.4.peg.2104</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=2665645.4@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Valine_degradation@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4401@1>fig|2665645.4.peg.4401</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=2665645.4@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Valine_degradation@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.433@1>fig|2665645.4.peg.433</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=2665645.4@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Valine_degradation@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.89@1>fig|2665645.4.peg.89</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=2665645.4@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Valine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.753@1>fig|2665645.4.peg.753</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=2665645.4@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Valine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.753@1>fig|2665645.4.peg.753</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=2665645.4@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)&subsystem_name=Valine_degradation@1>Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.535@1>fig|2665645.4.peg.535</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=2665645.4@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)&subsystem_name=Leucine_Biosynthesis@1>3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4008@1>fig|2665645.4.peg.4008</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=2665645.4@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)&subsystem_name=Leucine_Biosynthesis@1>3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4007@1>fig|2665645.4.peg.4007</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=2665645.4@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-isopropylmalate synthase (EC 2.3.3.13)&subsystem_name=Leucine_Biosynthesis@1>2-isopropylmalate synthase (EC 2.3.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1809@1>fig|2665645.4.peg.1809</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=2665645.4@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Leucine_Biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.433@1>fig|2665645.4.peg.433</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=2665645.4@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydrogenase (EC 1.1.1.85)&subsystem_name=Leucine_Biosynthesis@1>3-isopropylmalate dehydrogenase (EC 1.1.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1808@1>fig|2665645.4.peg.1808</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4011@1>fig|2665645.4.peg.4011</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=2665645.4@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Shikimate kinase I (EC 2.7.1.71)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Shikimate kinase I (EC 2.7.1.71)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5317@1>fig|2665645.4.peg.5317</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=2665645.4@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2624@1>fig|2665645.4.peg.2624</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=2665645.4@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate dehydratase II (EC 4.2.1.10)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>3-dehydroquinate dehydratase II (EC 4.2.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5219@1>fig|2665645.4.peg.5219</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=2665645.4@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I beta (EC 2.5.1.54)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I beta (EC 2.5.1.54)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4840@1>fig|2665645.4.peg.4840</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=2665645.4@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate synthase (EC 4.2.3.4)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>3-dehydroquinate synthase (EC 4.2.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1724@1>fig|2665645.4.peg.1724</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=2665645.4@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chorismate synthase (EC 4.2.3.5)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Chorismate synthase (EC 4.2.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.669@1>fig|2665645.4.peg.669</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1817@1>fig|2665645.4.peg.1817</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, amidotransferase component (EC 4.1.3.27)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate synthase, amidotransferase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.199@1>fig|2665645.4.peg.199</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5206@1>fig|2665645.4.peg.5206</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.190@1>fig|2665645.4.peg.190</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.197@1>fig|2665645.4.peg.197</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.194@1>fig|2665645.4.peg.194</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Indole-3-glycerol phosphate synthase (EC 4.1.1.48)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Indole-3-glycerol phosphate synthase (EC 4.1.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.196@1>fig|2665645.4.peg.196</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, aminase component (EC 4.1.3.27)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate synthase, aminase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.201@1>fig|2665645.4.peg.201</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.195@1>fig|2665645.4.peg.195</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isochorismate synthase (EC 5.4.4.2)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Isochorismate synthase (EC 5.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.477@1>fig|2665645.4.peg.477</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3155@1>fig|2665645.4.peg.3155</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=2665645.4@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.199@1>fig|2665645.4.peg.199</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=2665645.4@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chorismate mutase I (EC 5.4.99.5)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Chorismate mutase I (EC 5.4.99.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4840@1>fig|2665645.4.peg.4840</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=2665645.4@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prephenate dehydratase (EC 4.2.1.51)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Prephenate dehydratase (EC 4.2.1.51)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4837@1>fig|2665645.4.peg.4837</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=2665645.4@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prephenate and/or arogenate dehydrogenase (unknown specificity) (EC 1.3.1.12)(EC 1.3.1.43)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Prephenate and/or arogenate dehydrogenase (unknown specificity) (EC 1.3.1.12)(EC 1.3.1.43)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4839@1>fig|2665645.4.peg.4839</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=2665645.4@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biosynthetic Aromatic amino acid aminotransferase alpha (EC 2.6.1.57)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Biosynthetic Aromatic amino acid aminotransferase alpha (EC 2.6.1.57)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4838@1>fig|2665645.4.peg.4838</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, amidotransferase component (EC 4.1.3.27)&subsystem_name=Tryptophan_synthesis@1>Anthranilate synthase, amidotransferase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.199@1>fig|2665645.4.peg.199</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Tryptophan_synthesis@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5206@1>fig|2665645.4.peg.5206</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Tryptophan_synthesis@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.190@1>fig|2665645.4.peg.190</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Tryptophan_synthesis@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3155@1>fig|2665645.4.peg.3155</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Tryptophan_synthesis@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.197@1>fig|2665645.4.peg.197</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Tryptophan_synthesis@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.194@1>fig|2665645.4.peg.194</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Indole-3-glycerol phosphate synthase (EC 4.1.1.48)&subsystem_name=Tryptophan_synthesis@1>Indole-3-glycerol phosphate synthase (EC 4.1.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.196@1>fig|2665645.4.peg.196</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Tryptophan_synthesis@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.199@1>fig|2665645.4.peg.199</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, aminase component (EC 4.1.3.27)&subsystem_name=Tryptophan_synthesis@1>Anthranilate synthase, aminase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.201@1>fig|2665645.4.peg.201</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=2665645.4@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Tryptophan_synthesis@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.195@1>fig|2665645.4.peg.195</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=2665645.4@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan 2,3-dioxygenase (EC 1.13.11.11)&subsystem_name=Aromatic_amino_acid_degradation@1>Tryptophan 2,3-dioxygenase (EC 1.13.11.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4624@1>fig|2665645.4.peg.4624</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=2665645.4@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)&subsystem_name=Aromatic_amino_acid_degradation@1>Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3780@1>fig|2665645.4.peg.3780</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=2665645.4@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)&subsystem_name=Aromatic_amino_acid_degradation@1>4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4626@1>fig|2665645.4.peg.4626</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=2665645.4@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aromatic amino acid transport protein AroP&subsystem_name=Aromatic_amino_acid_degradation@1>Aromatic amino acid transport protein AroP</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1971@1>fig|2665645.4.peg.1971</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=2665645.4@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanine-4-hydroxylase (EC 1.14.16.1)&subsystem_name=Aromatic_amino_acid_degradation@1>Phenylalanine-4-hydroxylase (EC 1.14.16.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3326@1>fig|2665645.4.peg.3326</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=2665645.4@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyrroline-5-carboxylate reductase (EC 1.5.1.2)&subsystem_name=Proline_Synthesis@1>Pyrroline-5-carboxylate reductase (EC 1.5.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4492@1>fig|2665645.4.peg.4492</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=2665645.4@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Gamma-glutamyl phosphate reductase (EC 1.2.1.41)&subsystem_name=Proline_Synthesis@1>Gamma-glutamyl phosphate reductase (EC 1.2.1.41)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2463@1>fig|2665645.4.peg.2463</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=2665645.4@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Proline_Synthesis@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4649@1>fig|2665645.4.peg.4649</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=2665645.4@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate 5-kinase (EC 2.7.2.11)&subsystem_name=Proline_Synthesis@1>Glutamate 5-kinase (EC 2.7.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2462@1>fig|2665645.4.peg.2462</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=2665645.4@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Proline iminopeptidase (EC 3.4.11.5)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>Proline iminopeptidase (EC 3.4.11.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.776@1>fig|2665645.4.peg.776</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1681@1>fig|2665645.4.peg.1681</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=2665645.4@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyproline epimerase (EC 5.1.1.8)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>4-hydroxyproline epimerase (EC 5.1.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1489@1>fig|2665645.4.peg.1489</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=2665645.4@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-pyrroline-4-hydroxy-2-carboxylate deaminase (EC 3.5.4.22)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>1-pyrroline-4-hydroxy-2-carboxylate deaminase (EC 3.5.4.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1487@1>fig|2665645.4.peg.1487</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=2665645.4@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Proline dehydrogenase (EC 1.5.99.8) (Proline oxidase)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>Proline dehydrogenase (EC 1.5.99.8) (Proline oxidase)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2170@1>fig|2665645.4.peg.2170</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=2665645.4@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3522@1>fig|2665645.4.peg.3522</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=2665645.4@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dipeptidyl peptidase IV in 4-hydroxyproline catabolic gene cluster&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>Dipeptidyl peptidase IV in 4-hydroxyproline catabolic gene cluster</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1492@1>fig|2665645.4.peg.1492</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=2665645.4@1>Glycine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)&subsystem_name=Glycine_Biosynthesis@1>2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2825@1>fig|2665645.4.peg.2825</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4662@1>fig|2665645.4.peg.4662</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=2665645.4@1>Glycine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Glycine_Biosynthesis@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2443@1>fig|2665645.4.peg.2443</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=2665645.4@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Alanine_biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.433@1>fig|2665645.4.peg.433</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=2665645.4@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Alanine_biosynthesis@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3988@1>fig|2665645.4.peg.3988</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=2665645.4@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=Alanine_biosynthesis@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1292@1>fig|2665645.4.peg.1292</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1387@1>fig|2665645.4.peg.1387</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1706@1>fig|2665645.4.peg.1706</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2185@1>fig|2665645.4.peg.2185</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2727@1>fig|2665645.4.peg.2727</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=2665645.4@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine racemase (EC 5.1.1.1)&subsystem_name=Alanine_biosynthesis@1>Alanine racemase (EC 5.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3993@1>fig|2665645.4.peg.3993</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=2665645.4@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Serine_Biosynthesis@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2773@1>fig|2665645.4.peg.2773</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4497@1>fig|2665645.4.peg.4497</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=2665645.4@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine phosphatase (EC 3.1.3.3)&subsystem_name=Serine_Biosynthesis@1>Phosphoserine phosphatase (EC 3.1.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3570@1>fig|2665645.4.peg.3570</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3570@1>fig|2665645.4.peg.3570</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=2665645.4@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Serine_Biosynthesis@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2443@1>fig|2665645.4.peg.2443</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=2665645.4@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Serine_Biosynthesis@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2772@1>fig|2665645.4.peg.2772</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=2665645.4@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine cleavage system H protein&subsystem_name=Glycine_cleavage_system@1>Glycine cleavage system H protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3626@1>fig|2665645.4.peg.3626</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=2665645.4@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)&subsystem_name=Glycine_cleavage_system@1>Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4919@1>fig|2665645.4.peg.4919</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=2665645.4@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=Glycine_cleavage_system@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.535@1>fig|2665645.4.peg.535</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2413@1>fig|2665645.4.peg.2413</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3159@1>fig|2665645.4.peg.3159</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=2665645.4@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)&subsystem_name=Glycine_cleavage_system@1>Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2662@1>fig|2665645.4.peg.2662</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Glycine_and_Serine_Utilization@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2773@1>fig|2665645.4.peg.2773</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4497@1>fig|2665645.4.peg.4497</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, alpha subunit (EC 4.3.1.17)&subsystem_name=Glycine_and_Serine_Utilization@1>L-serine dehydratase, alpha subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.489@1>fig|2665645.4.peg.489</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine phosphatase (EC 3.1.3.3)&subsystem_name=Glycine_and_Serine_Utilization@1>Phosphoserine phosphatase (EC 3.1.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3570@1>fig|2665645.4.peg.3570</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, beta subunit (EC 4.3.1.17)&subsystem_name=Glycine_and_Serine_Utilization@1>L-serine dehydratase, beta subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.489@1>fig|2665645.4.peg.489</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Glycine_and_Serine_Utilization@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4647@1>fig|2665645.4.peg.4647</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Glycine_and_Serine_Utilization@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2443@1>fig|2665645.4.peg.2443</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Seryl-tRNA synthetase (EC 6.1.1.11)&subsystem_name=Glycine_and_Serine_Utilization@1>Seryl-tRNA synthetase (EC 6.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4485@1>fig|2665645.4.peg.4485</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Glycine_and_Serine_Utilization@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2772@1>fig|2665645.4.peg.2772</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine cleavage system H protein&subsystem_name=Glycine_and_Serine_Utilization@1>Glycine cleavage system H protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3626@1>fig|2665645.4.peg.3626</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)&subsystem_name=Glycine_and_Serine_Utilization@1>Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4919@1>fig|2665645.4.peg.4919</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)&subsystem_name=Glycine_and_Serine_Utilization@1>2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2825@1>fig|2665645.4.peg.2825</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4662@1>fig|2665645.4.peg.4662</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)&subsystem_name=Glycine_and_Serine_Utilization@1>Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2662@1>fig|2665645.4.peg.2662</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=2665645.4@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Glycine_and_Serine_Utilization@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2670@1>fig|2665645.4.peg.2670</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3296@1>fig|2665645.4.peg.3296</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=2665645.4@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiol peroxidase, Tpx-type (EC 1.11.1.15)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thiol peroxidase, Tpx-type (EC 1.11.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3179@1>fig|2665645.4.peg.3179</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=2665645.4@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiol peroxidase, Bcp-type (EC 1.11.1.15)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thiol peroxidase, Bcp-type (EC 1.11.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2801@1>fig|2665645.4.peg.2801</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5091@1>fig|2665645.4.peg.5091</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=2665645.4@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Thioredoxin-disulfide_reductase@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.50@1>fig|2665645.4.peg.50</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3176@1>fig|2665645.4.peg.3176</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=2665645.4@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thioredoxin reductase (EC 1.8.1.9)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thioredoxin reductase (EC 1.8.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1703@1>fig|2665645.4.peg.1703</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3789@1>fig|2665645.4.peg.3789</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4407@1>fig|2665645.4.peg.4407</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5161@1>fig|2665645.4.peg.5161</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=2665645.4@1>Galactosylceramide and Sulfatide metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=probable beta-D-galactosidase&subsystem_name=Galactosylceramide_and_Sulfatide_metabolism@1>probable beta-D-galactosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1311@1>fig|2665645.4.peg.1311</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=2665645.4@1>Galactosylceramide and Sulfatide metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Arylsulfatase (EC 3.1.6.1)&subsystem_name=Galactosylceramide_and_Sulfatide_metabolism@1>Arylsulfatase (EC 3.1.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.552@1>fig|2665645.4.peg.552</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.554@1>fig|2665645.4.peg.554</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.562@1>fig|2665645.4.peg.562</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.563@1>fig|2665645.4.peg.563</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=2665645.4@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkaline phosphatase (EC 3.1.3.1)&subsystem_name=Phosphate_metabolism@1>Alkaline phosphatase (EC 3.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.283@1>fig|2665645.4.peg.283</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2446@1>fig|2665645.4.peg.2446</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3528@1>fig|2665645.4.peg.3528</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5350@1>fig|2665645.4.peg.5350</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=2665645.4@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inorganic pyrophosphatase (EC 3.6.1.1)&subsystem_name=Phosphate_metabolism@1>Inorganic pyrophosphatase (EC 3.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3822@1>fig|2665645.4.peg.3822</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=2665645.4@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyrophosphate-energized proton pump (EC 3.6.1.1)&subsystem_name=Phosphate_metabolism@1>Pyrophosphate-energized proton pump (EC 3.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3821@1>fig|2665645.4.peg.3821</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=2665645.4@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Phosphate_metabolism@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.716@1>fig|2665645.4.peg.716</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=2665645.4@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exopolyphosphatase (EC 3.6.1.11)&subsystem_name=Phosphate_metabolism@1>Exopolyphosphatase (EC 3.6.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.717@1>fig|2665645.4.peg.717</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3402@1>fig|2665645.4.peg.3402</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=2665645.4@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Probable low-affinity inorganic phosphate transporter&subsystem_name=Phosphate_metabolism@1>Probable low-affinity inorganic phosphate transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2637@1>fig|2665645.4.peg.2637</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=2665645.4@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate transport regulator (distant homolog of PhoU)&subsystem_name=Phosphate_metabolism@1>Phosphate transport regulator (distant homolog of PhoU)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2638@1>fig|2665645.4.peg.2638</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=2665645.4@1>Polyphosphate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Polyphosphate@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.716@1>fig|2665645.4.peg.716</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=2665645.4@1>Polyphosphate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exopolyphosphatase (EC 3.6.1.11)&subsystem_name=Polyphosphate@1>Exopolyphosphatase (EC 3.6.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.717@1>fig|2665645.4.peg.717</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3402@1>fig|2665645.4.peg.3402</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=2665645.4@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylglyoxal synthase (EC 4.2.3.3)&subsystem_name=Methylglyoxal_Metabolism@1>Methylglyoxal synthase (EC 4.2.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.655@1>fig|2665645.4.peg.655</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.656@1>fig|2665645.4.peg.656</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=2665645.4@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase B (EC 1.2.1.22)&subsystem_name=Methylglyoxal_Metabolism@1>Aldehyde dehydrogenase B (EC 1.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4108@1>fig|2665645.4.peg.4108</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=2665645.4@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase (EC 1.2.1.3)&subsystem_name=Methylglyoxal_Metabolism@1>Aldehyde dehydrogenase (EC 1.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.485@1>fig|2665645.4.peg.485</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1893@1>fig|2665645.4.peg.1893</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3653@1>fig|2665645.4.peg.3653</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.485@1>fig|2665645.4.peg.485</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1893@1>fig|2665645.4.peg.1893</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3653@1>fig|2665645.4.peg.3653</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=2665645.4@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase (EC 1.2.1.3)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Aldehyde dehydrogenase (EC 1.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.485@1>fig|2665645.4.peg.485</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1893@1>fig|2665645.4.peg.1893</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3653@1>fig|2665645.4.peg.3653</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=2665645.4@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3825@1>fig|2665645.4.peg.3825</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=2665645.4@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3735@1>fig|2665645.4.peg.3735</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=2665645.4@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate acetyltransferase (EC 2.3.1.8)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Phosphate acetyltransferase (EC 2.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2568@1>fig|2665645.4.peg.2568</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=2665645.4@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetate kinase (EC 2.7.2.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Acetate kinase (EC 2.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2567@1>fig|2665645.4.peg.2567</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=2665645.4@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3734@1>fig|2665645.4.peg.3734</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=2665645.4@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-dependent protein deacetylase of SIR2 family&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>NAD-dependent protein deacetylase of SIR2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.54@1>fig|2665645.4.peg.54</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5198@1>fig|2665645.4.peg.5198</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=2665645.4@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, alpha subunit (EC 4.3.1.17)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>L-serine dehydratase, alpha subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.489@1>fig|2665645.4.peg.489</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=2665645.4@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.433@1>fig|2665645.4.peg.433</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=2665645.4@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, beta subunit (EC 4.3.1.17)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>L-serine dehydratase, beta subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.489@1>fig|2665645.4.peg.489</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=2665645.4@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine dehydrogenase (EC 1.4.1.1)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Alanine dehydrogenase (EC 1.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1856@1>fig|2665645.4.peg.1856</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5351@1>fig|2665645.4.peg.5351</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=2665645.4@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine racemase (EC 5.1.1.1)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Alanine racemase (EC 5.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3993@1>fig|2665645.4.peg.3993</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=2665645.4@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malate dehydrogenase (EC 1.1.1.37)&subsystem_name=Glyoxylate_bypass@1>Malate dehydrogenase (EC 1.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3014@1>fig|2665645.4.peg.3014</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=2665645.4@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Citrate synthase (si) (EC 2.3.3.1)&subsystem_name=Glyoxylate_bypass@1>Citrate synthase (si) (EC 2.3.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5003@1>fig|2665645.4.peg.5003</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=2665645.4@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aconitate hydratase 2 (EC 4.2.1.3)&subsystem_name=Glyoxylate_bypass@1>Aconitate hydratase 2 (EC 4.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3270@1>fig|2665645.4.peg.3270</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=2665645.4@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isocitrate lyase (EC 4.1.3.1)&subsystem_name=Glyoxylate_bypass@1>Isocitrate lyase (EC 4.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3541@1>fig|2665645.4.peg.3541</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=2665645.4@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malate synthase (EC 2.3.3.9)&subsystem_name=Glyoxylate_bypass@1>Malate synthase (EC 2.3.3.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3542@1>fig|2665645.4.peg.3542</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=2665645.4@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aconitate hydratase (EC 4.2.1.3)&subsystem_name=Glyoxylate_bypass@1>Aconitate hydratase (EC 4.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3269@1>fig|2665645.4.peg.3269</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4077@1>fig|2665645.4.peg.4077</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.89@1>fig|2665645.4.peg.89</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Dehydrogenase_complexes@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.753@1>fig|2665645.4.peg.753</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Dehydrogenase_complexes@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.753@1>fig|2665645.4.peg.753</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)&subsystem_name=Dehydrogenase_complexes@1>Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3825@1>fig|2665645.4.peg.3825</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.535@1>fig|2665645.4.peg.535</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3735@1>fig|2665645.4.peg.3735</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)&subsystem_name=Dehydrogenase_complexes@1>2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4076@1>fig|2665645.4.peg.4076</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=2665645.4@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)&subsystem_name=Dehydrogenase_complexes@1>Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3734@1>fig|2665645.4.peg.3734</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=2665645.4@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transketolase (EC 2.2.1.1)&subsystem_name=Pentose_phosphate_pathway@1>Transketolase (EC 2.2.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4209@1>fig|2665645.4.peg.4209</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=2665645.4@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transketolase, N-terminal section (EC 2.2.1.1)&subsystem_name=Pentose_phosphate_pathway@1>Transketolase, N-terminal section (EC 2.2.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5322@1>fig|2665645.4.peg.5322</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=2665645.4@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)&subsystem_name=Pentose_phosphate_pathway@1>Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4186@1>fig|2665645.4.peg.4186</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4596@1>fig|2665645.4.peg.4596</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=2665645.4@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transaldolase (EC 2.2.1.2)&subsystem_name=Pentose_phosphate_pathway@1>Transaldolase (EC 2.2.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.679@1>fig|2665645.4.peg.679</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1554@1>fig|2665645.4.peg.1554</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4208@1>fig|2665645.4.peg.4208</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=2665645.4@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transketolase, C-terminal section (EC 2.2.1.1)&subsystem_name=Pentose_phosphate_pathway@1>Transketolase, C-terminal section (EC 2.2.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5321@1>fig|2665645.4.peg.5321</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=2665645.4@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribulose-phosphate 3-epimerase (EC 5.1.3.1)&subsystem_name=Pentose_phosphate_pathway@1>Ribulose-phosphate 3-epimerase (EC 5.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3866@1>fig|2665645.4.peg.3866</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=2665645.4@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3942@1>fig|2665645.4.peg.3942</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=2665645.4@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoenolpyruvate carboxylase (EC 4.1.1.31)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Phosphoenolpyruvate carboxylase (EC 4.1.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1717@1>fig|2665645.4.peg.1717</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=2665645.4@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-dependent malic enzyme (EC 1.1.1.40)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>NADP-dependent malic enzyme (EC 1.1.1.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3629@1>fig|2665645.4.peg.3629</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=2665645.4@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate kinase (EC 2.7.1.40)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Pyruvate kinase (EC 2.7.1.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4168@1>fig|2665645.4.peg.4168</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycolate,_glyoxylate_interconversions&organism=2665645.4@1>Glycolate, glyoxylate interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoglycolate phosphatase (EC 3.1.3.18)&subsystem_name=Glycolate,_glyoxylate_interconversions@1>Phosphoglycolate phosphatase (EC 3.1.3.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3912@1>fig|2665645.4.peg.3912</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=2665645.4@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucosamine-6-phosphate deaminase (EC 3.5.99.6)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Glucosamine-6-phosphate deaminase (EC 3.5.99.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.307@1>fig|2665645.4.peg.307</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.689@1>fig|2665645.4.peg.689</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.876@1>fig|2665645.4.peg.876</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1994@1>fig|2665645.4.peg.1994</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2868@1>fig|2665645.4.peg.2868</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=2665645.4@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chitin binding protein&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Chitin binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1169@1>fig|2665645.4.peg.1169</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=2665645.4@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetyl glucosamine transporter, NagP&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>N-acetyl glucosamine transporter, NagP</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.859@1>fig|2665645.4.peg.859</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2869@1>fig|2665645.4.peg.2869</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=2665645.4@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted N-acetyl-glucosamine kinase 2, ROK family (EC 2.7.1.59)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Predicted N-acetyl-glucosamine kinase 2, ROK family (EC 2.7.1.59)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.880@1>fig|2665645.4.peg.880</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=2665645.4@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylglucosamine related transporter, NagX&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>N-acetylglucosamine related transporter, NagX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.83@1>fig|2665645.4.peg.83</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.105@1>fig|2665645.4.peg.105</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.297@1>fig|2665645.4.peg.297</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.838@1>fig|2665645.4.peg.838</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=2665645.4@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chitinase (EC 3.2.1.14)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Chitinase (EC 3.2.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.341@1>fig|2665645.4.peg.341</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.446@1>fig|2665645.4.peg.446</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.686@1>fig|2665645.4.peg.686</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.691@1>fig|2665645.4.peg.691</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=2665645.4@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-N-acetylglucosaminidase (EC 3.2.1.50)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Alpha-N-acetylglucosaminidase (EC 3.2.1.50)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.827@1>fig|2665645.4.peg.827</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=2665645.4@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylglucosamine-6-phosphate deacetylase (EC 3.5.1.25)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>N-acetylglucosamine-6-phosphate deacetylase (EC 3.5.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.278@1>fig|2665645.4.peg.278</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Sucrose_utilization&organism=2665645.4@1>Sucrose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sucrose-6-phosphate hydrolase (EC 3.2.1.26)&subsystem_name=Sucrose_utilization@1>Sucrose-6-phosphate hydrolase (EC 3.2.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2202@1>fig|2665645.4.peg.2202</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Sucrose_utilization&organism=2665645.4@1>Sucrose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fructokinase (EC 2.7.1.4)&subsystem_name=Sucrose_utilization@1>Fructokinase (EC 2.7.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1269@1>fig|2665645.4.peg.1269</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Trehalose_Uptake_and_Utilization&organism=2665645.4@1>Trehalose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Trehalase (EC 3.2.1.28)&subsystem_name=Trehalose_Uptake_and_Utilization@1>Trehalase (EC 3.2.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4068@1>fig|2665645.4.peg.4068</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Trehalose_Uptake_and_Utilization&organism=2665645.4@1>Trehalose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted trehalose permease, MFS family, FucP subfamily&subsystem_name=Trehalose_Uptake_and_Utilization@1>Predicted trehalose permease, MFS family, FucP subfamily</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4069@1>fig|2665645.4.peg.4069</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Trehalose_Uptake_and_Utilization&organism=2665645.4@1>Trehalose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-phosphoglucomutase (EC 5.4.2.6)&subsystem_name=Trehalose_Uptake_and_Utilization@1>Beta-phosphoglucomutase (EC 5.4.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3905@1>fig|2665645.4.peg.3905</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3951@1>fig|2665645.4.peg.3951</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=2665645.4@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3551@1>fig|2665645.4.peg.3551</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5012@1>fig|2665645.4.peg.5012</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=2665645.4@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Galactose-1-phosphate uridylyltransferase (EC 2.7.7.10)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Galactose-1-phosphate uridylyltransferase (EC 2.7.7.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4213@1>fig|2665645.4.peg.4213</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=2665645.4@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Galactokinase (EC 2.7.1.6)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Galactokinase (EC 2.7.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4212@1>fig|2665645.4.peg.4212</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=2665645.4@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldose 1-epimerase (EC 5.1.3.3)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Aldose 1-epimerase (EC 5.1.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2048@1>fig|2665645.4.peg.2048</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2336@1>fig|2665645.4.peg.2336</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4189@1>fig|2665645.4.peg.4189</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4211@1>fig|2665645.4.peg.4211</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=2665645.4@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formyltetrahydrofolate deformylase (EC 3.5.1.10)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Formyltetrahydrofolate deformylase (EC 3.5.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2959@1>fig|2665645.4.peg.2959</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=2665645.4@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4631@1>fig|2665645.4.peg.4631</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=2665645.4@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5113@1>fig|2665645.4.peg.5113</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=2665645.4@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3783@1>fig|2665645.4.peg.3783</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=2665645.4@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5113@1>fig|2665645.4.peg.5113</a>@~Carbohydrates@^Organic acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactate_utilization&organism=2665645.4@1>Lactate utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted L-lactate dehydrogenase, Iron-sulfur cluster-binding subunit YkgF&subsystem_name=Lactate_utilization@1>Predicted L-lactate dehydrogenase, Iron-sulfur cluster-binding subunit YkgF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1260@1>fig|2665645.4.peg.1260</a>@~Carbohydrates@^Organic acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactate_utilization&organism=2665645.4@1>Lactate utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted L-lactate dehydrogenase, hypothetical protein subunit SO1518&subsystem_name=Lactate_utilization@1>Predicted L-lactate dehydrogenase, hypothetical protein subunit SO1518</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1261@1>fig|2665645.4.peg.1261</a>@~Carbohydrates@^Organic acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactate_utilization&organism=2665645.4@1>Lactate utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted L-lactate dehydrogenase, Fe-S oxidoreductase subunit YkgE&subsystem_name=Lactate_utilization@1>Predicted L-lactate dehydrogenase, Fe-S oxidoreductase subunit YkgE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1259@1>fig|2665645.4.peg.1259</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=2665645.4@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alcohol dehydrogenase (EC 1.1.1.1)&subsystem_name=Butanol_Biosynthesis@1>Alcohol dehydrogenase (EC 1.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2402@1>fig|2665645.4.peg.2402</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=2665645.4@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-dependent butanol dehydrogenase A (EC 1.1.1.-)&subsystem_name=Butanol_Biosynthesis@1>NADH-dependent butanol dehydrogenase A (EC 1.1.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4592@1>fig|2665645.4.peg.4592</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=2665645.4@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Butanol_Biosynthesis@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4401@1>fig|2665645.4.peg.4401</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=2665645.4@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate formate-lyase (EC 2.3.1.54)&subsystem_name=Butanol_Biosynthesis@1>Pyruvate formate-lyase (EC 2.3.1.54)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3769@1>fig|2665645.4.peg.3769</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=2665645.4@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)&subsystem_name=Butanol_Biosynthesis@1>3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2479@1>fig|2665645.4.peg.2479</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=2665645.4@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Butanol_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.604@1>fig|2665645.4.peg.604</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Mixed_acid&organism=2665645.4@1>Fermentations: Mixed acid</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-lactate dehydrogenase (EC 1.1.1.28)&subsystem_name=Fermentations:_Mixed_acid@1>D-lactate dehydrogenase (EC 1.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3307@1>fig|2665645.4.peg.3307</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Mixed_acid&organism=2665645.4@1>Fermentations: Mixed acid</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alcohol dehydrogenase (EC 1.1.1.1)&subsystem_name=Fermentations:_Mixed_acid@1>Alcohol dehydrogenase (EC 1.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2402@1>fig|2665645.4.peg.2402</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Mixed_acid&organism=2665645.4@1>Fermentations: Mixed acid</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate formate-lyase (EC 2.3.1.54)&subsystem_name=Fermentations:_Mixed_acid@1>Pyruvate formate-lyase (EC 2.3.1.54)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3769@1>fig|2665645.4.peg.3769</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Mixed_acid&organism=2665645.4@1>Fermentations: Mixed acid</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoenolpyruvate carboxylase (EC 4.1.1.31)&subsystem_name=Fermentations:_Mixed_acid@1>Phosphoenolpyruvate carboxylase (EC 4.1.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1717@1>fig|2665645.4.peg.1717</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Mixed_acid&organism=2665645.4@1>Fermentations: Mixed acid</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate formate-lyase activating enzyme (EC 1.97.1.4)&subsystem_name=Fermentations:_Mixed_acid@1>Pyruvate formate-lyase activating enzyme (EC 1.97.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3768@1>fig|2665645.4.peg.3768</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Mixed_acid&organism=2665645.4@1>Fermentations: Mixed acid</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate acetyltransferase (EC 2.3.1.8)&subsystem_name=Fermentations:_Mixed_acid@1>Phosphate acetyltransferase (EC 2.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2568@1>fig|2665645.4.peg.2568</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Mixed_acid&organism=2665645.4@1>Fermentations: Mixed acid</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetate kinase (EC 2.7.2.1)&subsystem_name=Fermentations:_Mixed_acid@1>Acetate kinase (EC 2.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2567@1>fig|2665645.4.peg.2567</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetolactate_synthase_subunits&organism=2665645.4@1>Acetolactate synthase subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase large subunit (EC 2.2.1.6)&subsystem_name=Acetolactate_synthase_subunits@1>Acetolactate synthase large subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1805@1>fig|2665645.4.peg.1805</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetolactate_synthase_subunits&organism=2665645.4@1>Acetolactate synthase subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase small subunit (EC 2.2.1.6)&subsystem_name=Acetolactate_synthase_subunits@1>Acetolactate synthase small subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1804@1>fig|2665645.4.peg.1804</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2104@1>fig|2665645.4.peg.2104</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=2665645.4@1>Fermentations: Lactate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-lactate dehydrogenase (EC 1.1.1.28)&subsystem_name=Fermentations:_Lactate@1>D-lactate dehydrogenase (EC 1.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3307@1>fig|2665645.4.peg.3307</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=2665645.4@1>Fermentations: Lactate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate acetyltransferase (EC 2.3.1.8)&subsystem_name=Fermentations:_Lactate@1>Phosphate acetyltransferase (EC 2.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2568@1>fig|2665645.4.peg.2568</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=2665645.4@1>Fermentations: Lactate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetate kinase (EC 2.7.2.1)&subsystem_name=Fermentations:_Lactate@1>Acetate kinase (EC 2.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2567@1>fig|2665645.4.peg.2567</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=2665645.4@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4401@1>fig|2665645.4.peg.4401</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=2665645.4@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Electron transfer flavoprotein, beta subunit&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Electron transfer flavoprotein, beta subunit</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3826@1>fig|2665645.4.peg.3826</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=2665645.4@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Electron transfer flavoprotein, alpha subunit&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Electron transfer flavoprotein, alpha subunit</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3827@1>fig|2665645.4.peg.3827</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=2665645.4@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2479@1>fig|2665645.4.peg.2479</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=2665645.4@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.604@1>fig|2665645.4.peg.604</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=2665645.4@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2559@1>fig|2665645.4.peg.2559</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3910@1>fig|2665645.4.peg.3910</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=2665645.4@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.209@1>fig|2665645.4.peg.209</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=2665645.4@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate regulon repressor GlpR&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol-3-phosphate regulon repressor GlpR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.210@1>fig|2665645.4.peg.210</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=2665645.4@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase [NAD(P)+] (EC 1.1.1.94)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol-3-phosphate dehydrogenase [NAD(P)+] (EC 1.1.1.94)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4590@1>fig|2665645.4.peg.4590</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=2665645.4@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol kinase (EC 2.7.1.30)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol kinase (EC 2.7.1.30)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.208@1>fig|2665645.4.peg.208</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=2665645.4@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol uptake facilitator protein&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol uptake facilitator protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.207@1>fig|2665645.4.peg.207</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Inositol_catabolism&organism=2665645.4@1>Inositol catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inositol oxygenase (EC 1.13.99.1)&subsystem_name=Inositol_catabolism@1>Inositol oxygenase (EC 1.13.99.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.836@1>fig|2665645.4.peg.836</a>@~Carbohydrates@^Polysaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=2665645.4@1>Glycogen metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate adenylyltransferase (EC 2.7.7.27)&subsystem_name=Glycogen_metabolism@1>Glucose-1-phosphate adenylyltransferase (EC 2.7.7.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1608@1>fig|2665645.4.peg.1608</a>@~Carbohydrates@^Polysaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=2665645.4@1>Glycogen metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1,4-alpha-glucan (glycogen) branching enzyme, GH-13-type (EC 2.4.1.18)&subsystem_name=Glycogen_metabolism@1>1,4-alpha-glucan (glycogen) branching enzyme, GH-13-type (EC 2.4.1.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1609@1>fig|2665645.4.peg.1609</a>@~Carbohydrates@^Polysaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=2665645.4@1>Glycogen metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-glucosidase, family 31 of glycosyl hydrolases, COG1501&subsystem_name=Glycogen_metabolism@1>Alpha-glucosidase, family 31 of glycosyl hydrolases, COG1501</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1610@1>fig|2665645.4.peg.1610</a>@~Carbohydrates@^Polysaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=2665645.4@1>Glycogen metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycogen synthase, ADP-glucose transglucosylase (EC 2.4.1.21)&subsystem_name=Glycogen_metabolism@1>Glycogen synthase, ADP-glucose transglucosylase (EC 2.4.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1607@1>fig|2665645.4.peg.1607</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=2665645.4@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Endo-alpha-mannosidase&subsystem_name=Mannose_Metabolism@1>Endo-alpha-mannosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.914@1>fig|2665645.4.peg.914</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=2665645.4@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-1,2-mannosidase&subsystem_name=Mannose_Metabolism@1>Alpha-1,2-mannosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.831@1>fig|2665645.4.peg.831</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.863@1>fig|2665645.4.peg.863</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.873@1>fig|2665645.4.peg.873</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.882@1>fig|2665645.4.peg.882</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.883@1>fig|2665645.4.peg.883</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2340@1>fig|2665645.4.peg.2340</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2907@1>fig|2665645.4.peg.2907</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=2665645.4@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomannomutase (EC 5.4.2.8)&subsystem_name=Mannose_Metabolism@1>Phosphomannomutase (EC 5.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.770@1>fig|2665645.4.peg.770</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=2665645.4@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mannose-6-phosphate isomerase (EC 5.3.1.8)&subsystem_name=Mannose_Metabolism@1>Mannose-6-phosphate isomerase (EC 5.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.889@1>fig|2665645.4.peg.889</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5092@1>fig|2665645.4.peg.5092</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=2665645.4@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-mannosidase (EC 3.2.1.25)&subsystem_name=Mannose_Metabolism@1>Beta-mannosidase (EC 3.2.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.884@1>fig|2665645.4.peg.884</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.2342@1>fig|2665645.4.peg.2342</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=2665645.4@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Endo-1,4-beta-mannosidase&subsystem_name=Mannose_Metabolism@1>Endo-1,4-beta-mannosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.124@1>fig|2665645.4.peg.124</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=D-ribose_utilization&organism=2665645.4@1>D-ribose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribokinase (EC 2.7.1.15)&subsystem_name=D-ribose_utilization@1>Ribokinase (EC 2.7.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4180@1>fig|2665645.4.peg.4180</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Xylose_utilization&organism=2665645.4@1>Xylose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xylose isomerase (EC 5.3.1.5)&subsystem_name=Xylose_utilization@1>Xylose isomerase (EC 5.3.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1553@1>fig|2665645.4.peg.1553</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4568@1>fig|2665645.4.peg.4568</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Xylose_utilization&organism=2665645.4@1>Xylose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xylanase&subsystem_name=Xylose_utilization@1>Xylanase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.980@1>fig|2665645.4.peg.980</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Xylose_utilization&organism=2665645.4@1>Xylose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xylulose kinase (EC 2.7.1.17)&subsystem_name=Xylose_utilization@1>Xylulose kinase (EC 2.7.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1552@1>fig|2665645.4.peg.1552</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4567@1>fig|2665645.4.peg.4567</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Xylose_utilization&organism=2665645.4@1>Xylose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-xylose proton-symporter XylE&subsystem_name=Xylose_utilization@1>D-xylose proton-symporter XylE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1263@1>fig|2665645.4.peg.1263</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1513@1>fig|2665645.4.peg.1513</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=2665645.4@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1778@1>fig|2665645.4.peg.1778</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=2665645.4@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribokinase (EC 2.7.1.15)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Ribokinase (EC 2.7.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4180@1>fig|2665645.4.peg.4180</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=2665645.4@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative deoxyribonuclease YjjV&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Putative deoxyribonuclease YjjV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1642@1>fig|2665645.4.peg.1642</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=2665645.4@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribose-phosphate aldolase (EC 4.1.2.4)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Deoxyribose-phosphate aldolase (EC 4.1.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.3624@1>fig|2665645.4.peg.3624</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=2665645.4@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-ribulose-5-phosphate 4-epimerase (EC 5.1.3.4)&subsystem_name=L-Arabinose_utilization@1>L-ribulose-5-phosphate 4-epimerase (EC 5.1.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4191@1>fig|2665645.4.peg.4191</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=2665645.4@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypothetical glycoside hydrolase, family 43, similar to arabinosidase&subsystem_name=L-Arabinose_utilization@1>Hypothetical glycoside hydrolase, family 43, similar to arabinosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4199@1>fig|2665645.4.peg.4199</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=2665645.4@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribulokinase (EC 2.7.1.16)&subsystem_name=L-Arabinose_utilization@1>Ribulokinase (EC 2.7.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4192@1>fig|2665645.4.peg.4192</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=2665645.4@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Possible GPH family transporter (TC 2.A.2) for arabinosides&subsystem_name=L-Arabinose_utilization@1>Possible GPH family transporter (TC 2.A.2) for arabinosides</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.1548@1>fig|2665645.4.peg.1548</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=2665645.4@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-arabinose isomerase (EC 5.3.1.4)&subsystem_name=L-Arabinose_utilization@1>L-arabinose isomerase (EC 5.3.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4190@1>fig|2665645.4.peg.4190</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=2665645.4@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Arabinan endo-1,5-alpha-L-arabinosidase (EC 3.2.1.99)&subsystem_name=L-Arabinose_utilization@1>Arabinan endo-1,5-alpha-L-arabinosidase (EC 3.2.1.99)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4197@1>fig|2665645.4.peg.4197</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4200@1>fig|2665645.4.peg.4200</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.4202@1>fig|2665645.4.peg.4202</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=2665645.4@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein AraJ precursor&subsystem_name=L-Arabinose_utilization@1>Protein AraJ precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|2665645.4.peg.5176@1>fig|2665645.4.peg.5176</a>'>
<input type='hidden' id='table_onclicks_0' value='@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^'>
<input type='hidden' id='table_highlights_0' value='@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^'>
<input type='hidden' id='table_filtereddata_0' value=''>
<input type='hidden' id='table_rows_0' value='996'>
<input type='hidden' id='table_cols_0' value='5'>
<input type='hidden' id='table_start_0' value='0'>
<input type='hidden' id='table_sortdirection_0' value='up'>
<input type='hidden' id='table_column_types_0' value='@~@~@~@~'>
<input type='hidden' id='table_hoverid_0' value='1'>
<span id='table_input_space_0'></span><input type='hidden' id='ajax_url' value='https://rast.nmpdr.org/ajax.cgi'>
<input type='hidden' id='ajax_params' value='page=Organism&app=SeedViewer&url=seedviewer.cgi&cookies=0' />
<div style='display:none;' id='table_0_download'></div><table id='table_0' class='table_table' style=''><tr><td name='0_col_1' id='0_col_1' class='table_first_row' style=''><a href='javascript: table_sort("0", "1");' class='table_first_row' title='click to sort'><span id='colname_0_col_1'>Category</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_1_operator' value='equal' id='table_0_operator_1'><select name='0_col_1' class='filter_item' id='table_0_operand_1' onchange='return check_submit_filter2("0", this.options[this.selectedIndex].text);' style='width: 100%;' title='Select Filter'></select></td><td name='0_col_2' id='0_col_2' class='table_first_row' style=''><a href='javascript: table_sort("0", "2");' class='table_first_row' title='click to sort'><span id='colname_0_col_2'>Subcategory</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_2_operator' value='equal' id='table_0_operator_2'><select name='0_col_2' class='filter_item' id='table_0_operand_2' onchange='return check_submit_filter2("0", this.options[this.selectedIndex].text);' style='width: 100%;' title='Select Filter'></select></td><td name='0_col_3' id='0_col_3' class='table_first_row' style=''><a href='javascript: table_sort("0", "3");' class='table_first_row' title='click to sort'><span id='colname_0_col_3'>Subsystem</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_3_operator' value='like' id='table_0_operator_3'><input type='text' name='0_col_3' class='filter_item' value='' size=5 id='table_0_operand_3' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td><td name='0_col_4' id='0_col_4' class='table_first_row' style=''><a href='javascript: table_sort("0", "4");' class='table_first_row' title='click to sort'><span id='colname_0_col_4'>Role</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_4_operator' value='like' id='table_0_operator_4'><input type='text' name='0_col_4' class='filter_item' value='' size=5 id='table_0_operand_4' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td><td name='0_col_5' id='0_col_5' class='table_first_row' style=''><span id='colname_0_col_5'>Features</span><br><input type=hidden name='0_col_5_operator' value='like' id='table_0_operator_5'><input type='text' name='0_col_5' class='filter_item' value='' size=5 id='table_0_operand_5' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td></table><input type='hidden' id='table_visible_columns_0' value='1@~1@~1@~1@~1'>
<input type='hidden' id='table_combo_columns_0' value='1@~2'>
<input type='hidden' id='table_sc_0' value=''><img src='https://rast.nmpdr.org/Html/clear.gif' onload='initialize_table("0")'><input type='hidden' id='hover_redundancies_1' value=''></td></tr><tr><td style='width: 100%; text-align: center;'><table style='width: 100%;'><tr><td align='left' width='20%'><a href='javascript: table_first("0");' name='table_first_0'>&laquo;first</a>&nbsp;&nbsp;<a href='javascript: table_prev("0");' name='table_prev_0'>&laquo;prev</a></td><td align='center' width='60%'>displaying <span id='table_start_bottom_0'>1</span> - <span id='table_stop_bottom_0'>996</span> of <span id='table_total_bottom_0'>996</span></td><td align='right' width='20%'><a href='javascript: table_next("0");' name='table_next_0'>next&raquo;</a>&nbsp;&nbsp;<a href='javascript: table_last("0");' name='table_last_0'>last&raquo;</a></td></tr></table></td></tr></table></div></td></tr></table>
<img src='https://rast.nmpdr.org/Html/clear.gif'  onload='initialize_tab_view("1"); tab_view_select("1", "0");'>
<br><br><!-- tracking off -->
  </div>
    
</body>


</html>

