<!doctype HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">    
    <title>Seed Viewer - Organism</title>

    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/TabView.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/Hover.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/Table.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/web_app_default.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/commonviewer.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/seedviewer.css" >
       

    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/PopupTooltip.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/TabView.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Ajax.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Hover.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Tree.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Table.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/fade.js" ></script>
    
    <meta name="HTML_PATH" content="./Html" />
    
    <META HTTP-EQUIV="PRAGMA" CONTENT="NO-CACHE">
    
    <META HTTP-EQUIV="CACHE-CONTROL" CONTENT="NO-CACHE">
    
    <meta name="robots" content="noindex,nofollow" />
    
    

  </head>

  <body>
<div id="header">
  <span id="logo">
  <img height="95px" src="./Html/seed-logo-green.png" alt="The SEED" />
  </span>
  <p>
  <span id="title">
  <img src="./Html/seed-theseed-viewer.png" alt="The SEED Viewer" />
  </span>
  <span id="version">SEED Viewer version 2.0</span>
  </p>
  <p>Welcome to the SEED Viewer - a read-only browser of the curated SEED data. <br/>
  For more information about The SEED please visit <a href="http://www.theseed.org">theSEED.org.</a>

  </p>




</div>

<div id="topbar">
  <table style="width: 100%; border-spacing: 0px;">
    <tr>
      <td style='width: 100%; padding: 0px;'>
	
	  <div id='menu'>
	<ul id='nav'>
		<li><div><a href="?page=Home" >&raquo;Navigate</a></div>
		<ul>
			<li><a href="?page=Home" >Startpage</a></li>
			<li><a href="?page=OrganismSelect" >Organisms</a></li>
			<li><a href="?page=SubsystemSelect" >Browse Subsystems</a></li>
			<li><a href="https://rast.nmpdr.org/SubsysEditor.cgi" >Curate Subsystems</a></li>
			<li><a href="?page=Scenarios" >Scenarios</a></li>
			<li><a href="?page=FigFamViewer" >FigFams</a></li>
			<li><a href="?page=BlastRun" >BLAST Search</a></li>
			<li><a href="http://www.theseed.org/daily" >Visit the Daily SEED</a></li>
			<li><a href="?page=SubsystemPrimer" >Request a Subsystem</a></li>
		</ul>
		</li>
		<li><div><a  >&raquo;Organism</a></div>
		<ul>
			<li><a href="?page=Organism&organism=376686.40" >General Information</a></li>
			<li><a href="?page=BrowseGenome&tabular=1&organism=376686.40" >Feature Table</a></li>
			<li><a href="?page=BrowseGenome&organism=376686.40" >Genome Browser</a></li>
			<li><a href="?page=Scenarios&organism=376686.40" >Scenarios</a></li>
			<li><a href="?page=SubsystemSelect&organism=376686.40" >Subsystems</a></li>
			<li><a href="?page=Export&organism=376686.40" >Export</a></li>
		</ul>
		</li>
		<li><div><a  >&raquo;Comparative Tools</a></div>
		<ul>
			<li><a href="?page=CompareMetabolicReconstruction&organism=376686.40" >Function based Comparison</a></li>
			<li><a href="?page=MultiGenomeCompare&organism=376686.40" >Sequence based Comparison</a></li>
			<li><a href="?page=Kegg&organism=376686.40" >Kegg Metabolic Analysis</a></li>
			<li><a href="?page=BlastRun&organism=376686.40" >BLAST search</a></li>
		</ul>
		</li>
		<li><div><a href="http://www.theseed.org" target="help">&raquo;Help</a></div>
		<ul>
			<li><a href="http://www.theseed.org/wiki/index.php/Home_of_the_SEED" >What is the SEED</a></li>
			<li><a href="http://www.theseed.org/wiki/index.php/SEED_Viewer_Tutorial" >HowTo use the SEED Viewer</a></li>
			<li><a href="http://www.theseed.org/wiki/index.php/RAST_Tutorial" >Submitting Data to SEED</a></li>
			<li><a href="mailto:rast@mcs.anl.gov" >Contact</a></li>
			<li><a href="?page=Register" >Register</a></li>
			<li><a href="?page=RequestNewPassword" >I forgot my Password</a></li>
		</ul>
		</li>
	</ul>
</div>

	
      </td>
      <td style="border-right: 2px solid #5DA668; padding: 0px;">

	<div id='search'><form style='margin: 0px; padding: 0px;'><input type='submit' value='find' style='width:40px;cursor: pointer;'><input type='text' name='pattern'><input type='hidden' name='page' value='SearchResult'><input type='hidden' name='action' value='check_search'><input type='submit' style='display:none;'></form></div>

      </td>
      <td style="text-align: right; padding: 0px;">
	
	  <div id='user'><a href='seedviewer.cgi?page=AccountManagement'><img class='imglink' src='https://rast.nmpdr.org/Html/wac_people.png' title='Account Management' /></a>
<a href='seedviewer.cgi?page=Organism&organism=376686.40&logout=1'><img class='imglink' src='https://rast.nmpdr.org/Html/wac_logout.png' title='Logout' /></a>
Ekaterina Pushmina</div>
	
      </td>
    </tr>
  </table>
</div>


 

  
  
  <div id="content">
    <script>function initialize_all () {

}</script><style>.hideme { display: none; }</style><h2>Organism Overview for Flavobacterium johnsoniae UW101 (376686.40)</h2><table><tr><td width=550px><div><table><tr><th style='width: 170px;'>Genome</th><td>Flavobacterium johnsoniae UW101 (Taxonomy ID: <a href='http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=376686&lvl=3&lin=f&keep=1&srchmode=1&unlock' target=outbound>376686</a>)</td></tr><tr><th>Domain</th><td>Bacteria</td></tr><tr><th>Taxonomy</th><td>Bacteria; Bacteroidetes; Flavobacteriia; Flavobacteriales; Flavobacteriaceae; Flavobacterium; Flavobacterium johnsoniae UW101 ; Flavobacterium johnsoniae UW101</td></tr><tr><th>Neighbors</th><td><a href='seedviewer.cgi?page=ClosestNeighbors&organism=376686.40'>View closest neighbors</a></td></tr><tr><th>Size</th><td>6,096,872</td></tr><tr><th>GC Content</th><td>34.1</td></tr><tr><th>L50</th><td>1</td></tr><tr><th>Number of Contigs (with PEGs)</th><td>1</td></tr><tr><th>Number of Subsystems</th><td>274</td></tr><tr><th>Number of Coding Sequences</th><td>5362</td></tr><tr><th>Number of RNAs</th><td>79</td></tr></table></div></td><td>For each genome we offer a wide set of information to browse, compare and download.<br><br><input type='hidden' id='OrganismInfoContent' value='0' /><table style='height: 125px; width: 325px;  border-spacing: 0px;border-collapse: expression("separate", cellSpacing = "0px");' id='tab_view_table_0'><tr><td class='tab_view_title_selected' name='0_tabs' id='0_tab_0' onclick='tab_view_select("0", "0");'>Browse</td><td class='tab_view_title' name='0_tabs' id='0_tab_1' onclick='tab_view_select("0", "1");'>Compare</td><td class='tab_view_title' name='0_tabs' id='0_tab_2' onclick='tab_view_select("0", "2");'>Download</td><td class='tab_view_title' name='0_tabs' id='0_tab_3' onclick='tab_view_select("0", "3");'>Annotate</td><td class='tab_view_title_filler'>&nbsp;</td></tr><tr><td colspan='5' class='tab_view_content_td'><div style='height: 100%; width: 100%;' class='tab_view_content_selected' name='0_contents' id='0_content_0'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">Browse through the features of <a href="seedviewer.cgi?page=BrowseGenome&organism=376686.40">Flavobacterium johnsoniae UW101</a> both graphically and through a table. Both allow quick navigation and filtering for features of your interest. Each feature is linked to its own detail page.<br><br>Click <a href="seedviewer.cgi?page=BrowseGenome&organism=376686.40">here</a> to get to the Genome Browser</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_1'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">Compare the metabolic reconstruction of this organism to that of another organism.<br><br>Available comparisons are <a href="seedviewer.cgi?page=CompareMetabolicReconstruction&organism=376686.40">function based</a>, <a href="?page=MultiGenomeCompare&organism=376686.40">sequence based</a> or via <a href="seedviewer.cgi?page=Kegg&organism=376686.40">KEGG</a>. You can also <a href="seedviewer.cgi?page=BlastRun&organism=376686.40">BLAST</a> against this organism.</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_2'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">You can export all information about this organism (e.g. annotations, scenarios, subsystems) into a variety of formats (e.g. EMBL, Excel) for further analysis on your own system.<br><br>Click <a href="seedviewer.cgi?page=Export&organism=376686.40">here</a> to get to the Export page.</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_3'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">The SeedViewer has several options to edit the annotations of your private genome. To find out how to use these capabilities and enable them for your genomes, click <a href="?page=HowToAnnotate">here</a>.</div></div></td></tr></table>
<img src='https://rast.nmpdr.org/Html/clear.gif'  onload='initialize_tab_view("0"); tab_view_select("0", "0");'>
</td></tr></table><h2>Subsystem Information</h2><input type='hidden' id='SubsystemTabView' value='1' /><table style='height: 200px; width: 95%px;  border-spacing: 0px;border-collapse: expression("separate", cellSpacing = "0px");' id='tab_view_table_1'><tr><td class='tab_view_title_selected' name='1_tabs' id='1_tab_0' onclick='tab_view_select("1", "0");'>Subsystem Statistics</td><td class='tab_view_title' name='1_tabs' id='1_tab_1' onclick='tab_view_select("1", "1");'>Features in Subsystems</td><td class='tab_view_title_filler'>&nbsp;</td></tr><tr><td colspan='3' class='tab_view_content_td'><div style='height: 100%; width: 100%;' class='tab_view_content_selected' name='1_contents' id='1_content_0'><table style='padding-top: 25px;'><tr><td><b>Subsystem Coverage</b></td><td align=center><b>Subsystem Category Distribution</b></td><td align=center><b>Subsystem Feature Counts</b></td></tr><tr><td style='padding-right: 10px;'><img src="data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAAGQCAMAAAAOUBefAAAAolBMVEX///8AAADIyMjrBSg8PL4y/zLghQ8AAJv/3QAAm5vIZMgbhTSHQUEAWv9fZGRQ0pbh+qCqHpH/132MpdKgD/otm7mqzXgePL5zr6DChQ8AAH3h3QD/AAAA/wAAAP///wAA////AP//fwD/AH8A/39/AP9//wAAf///f39//39/f/8AAH9/AAAAfwB/fwB/AH8Af3//f////39///9QdJpdpmhNMpuCAAABEElEQVR4nO3dQQ6DIBAF0PFO9P5XK8WxVhc1Chvj+yYSJHn5iis3RhxlOpdDL6bXmQCBQCAQCATeGKzr9dQJRiTVrMj5dbBBTYlok1/vApiNEvwWHnXLe+86uDTceQPAfKajwLYnfeB8cX1tBjT8GyAQCAQCgQ8Hh38/LGcCBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIDBT1+upD4yG5BCfow9sTKxDbAp2g6VsC3bc8kJtvesNcz/2XgeYUpQYtynzLMa8NvNaidLd8CBAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAi8ETj6jwYiIiIiD84bB+8PjHpEBvgAAAAASUVORK5CYII=" usemap="#imap_barchart_0"><map name='imap_barchart_0'>
<area shape='rect' coords='33,64,81,370' onmouseover='hover(event, "bar_0", "2");'onclick='window.top.location="?page=BrowseGenome&organism=376686.40&tabular=1&noss=1"' style='cursor: pointer;'>
<area shape='rect' coords='33,0,81,64' onmouseover='hover(event, "bar_1", "2");'onclick='tab_view_select("1", 1);' style='cursor: pointer;'>
</map><input type='hidden' id='hover_redundancies_2' value='bar_1~#bar_0'><span style='display: none;' id='tooltip_2_bar_0'><table><tr><th>not in Subsystem</th></tr><tr><td>total (4421)</td></tr><tr><td>non-hypothetical (2148)</td></tr><tr><td>hypothetical (2273)</td></tr></table></span>
<span style='display: none;' id='tooltip_2_bar_1'><table><tr><th>in Subsystem</th></tr><tr><td>total (941)</td></tr><tr><td>non-hypothetical (896)</td></tr><tr><td>hypothetical (45)</td></tr></table></span>
</td><td style='padding-right: 10px;'><img style="border: none;" src="data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAZAAAAGQCAMAAAC3Ycb+AAABDlBMVEX///8AAADIyMjrBSg8PL4y/zL/AAAAgAAAAICAgACAAIDAwMCAgICZmf+ZM2b//8xmAGb/gIAAZszMzP//AP///wAA//+AAAAAgIAAAP8AzMzM///M/8z//5mZzP//mczMmf//zJkzZv8zzMyZzAD/zAD/mQD/ZgBmZpmWlpYAM2YzmWYAMwAzMwCZMwAzM5kzMzPghQ8AAJv/3QAAm5vIZMgbhTSHQUEAWv9fZGRQ0pbh+qCqHpH/132MpdKgD/otm7mqzXgePL5zr6DChQ8AAH3h3QAA/wD/fwD/AH8A/39/AP9//wAAf///f39//39/f/8AAH9/AAAAfwB/fwB/AH8Af3//f////39////pA/ySAAAQAUlEQVR4nO3dW3IbyQ4E0P66ssayx4qYscMmxdn/Li/fIrvrAaAAJNBkrkCdJ7LwJWmaUua/Rf43D/pHfIQsGRogTxrbtDA6IE8U1fQkaCBPF50QNcggT5OR0DU4IE8TflgSApCnCyMSDCHIE6UbqYYc5GlSz4DGEMjTpJAxjGGQJ8os4x7jIE+SaxQ4VECeIoeoaCiBPE20NPRAHtpEkUMT5FFJVDl0QR6QRFlDHeSxTPQ1LEAex8TEwwTkIUhsOKxA1i5ipWEHsm4TQw9DkNWSWHLYgqxTxNbDFmSFJMYc5iCrEjHH8ABZEYqLhwvIOkR8PHxA1iDi5OEEkp7Ei8MPJLeIn4cfSGISRw5XkKwirh6uIClJfDm8QfKJeHt4gyQjcecAgGQSAXgAQNKQIDgwIC/oqknBeCBAXvZBt90NiAMGEl0E5gEAeTkH3XkjOA4AyMtLeBGkhzvIy0t4EagHFCSmyG73SCAvs6DbX2R3yOOAzD3CieweC2TpEUxkt4OLwEEiiex2eBG4RyCR3S6ACN4jjMhuF0EkAkgMkd2jgdQ9IpDsFlk7SNMDLrL0QIkE8UCLlEAwIk+QqgdGJIoHVOTj49FACB44ko9DwogE8gCJfHyEEgkFAhH5iCUSygMB8vERSySUx8sXoEcQkVAgX/aBcdREVgfC83AV+ZgngEg4D0+RBUgAkXgefiBLjwAiYUC+fPEWKXngRQJ6OImUPcoiqwGRebiIbLcxRUJ6OIhst0FFQoAsPMxFttuGCPSMBPUwFtleEk8kqoetyFYg8jAgFQ9LkO02rkhYD0OR7TawCNijzmEnst1GFgnsYSQy9wgmAgXpeNiILEHoInlBdDwsQAoeNRKISGgPA5GyRyCR2B76IjWQMCIwEJqHNkjVI4xIcA9lkYYHWSQfiKqHrkgTJIYIBoThoQnS9oghEt5DUaTnUSHxFYnvoSdCAMGLJPDQAqF4EEVWBcL2UBKhecBFMnjoiFBBwCIpPDRAyB5gkRQeCiIMD6yIL4jUY1yEBVIkcRJJ4jEKwvQgiYQHsfT48urrgRNxBBnh2McZBCWSxmMEROKBEkng8XqOswdIJI+HWETqURJZDYgCh1hEDlIgMReJ7fH6CgbxF8nkIRMZ8vAXCeyx4ICAeIvYg6jN4/X1K8CjLxIPxGkeX/dBgPiKxPSocAhAFDwKJIYixiDKHmwRHQ9PkYAeDQ62iBaIn0g4jzYHE0TNoycSB8Te4+ssIBAvEUMQEw4WiKaHl0g6D46ILoiPSCAPGgcDRNljSWIhEsaDysEQ0QdpioQGGfeocpBBDDwcRGJ4cDjIIiYg5iIhPdocRBAbD3MRCxDbeVBFrECMReAeAg4wSEsECGLkQeEggdh5LEhURbAeMg6SiCmIoYg2yIgHmePrGxrETgToIZ7H2z5gDzsRmIeU4+0UOIiViCqI/Wv1dg3aoyHiDzLsMcwRAcRGBOEh43i7D95jTqIjksXjbZEIINsKyICIGogzRxAQfRGsh1ijDeLnURXxAxny0ONoiniCaIuoeCA4GiCuHtoiGiASj1GNlogziK6Imwd7Hj2OKoi3xz3JqAjEQ4WjKgIAKYtgQCw4SBqhQPREPDyMOCoiEA89EWcPTY1YIFoi5h6sefA4YoEoiQyBKM+Dy1EEgXmURQxBJB4MDr5GWQQIcksiFrH0MOcIB6Ig4uNhxFEAwXooiIhB1OYh1yiJoEGGRRw8DDkCghREDEC4Hl4cEUEGRWQgKhzjGksRNMYxQyKmHvYcIUFuSPgi6h6+HEFBBkSUPUivlZ5GWJCFiCHI6DxUOcKCiEU0PQAccUGkIhYefhqRQYQiah4YjpkI2uA+IhEeyMBrZcQRGeSThCGi7OGtERxkJjIOQvNActyBoOsvhD0RBkjPA8JxJ4JuvxSuCNlDxmGuER6EKzLo0X6tPDhuQNDVV8IT0fEActyIoJuvhXXYaSB8DjeNBCAXEpLIuAeaIwMIQ0TsUefw1cgBQhcRegTiyAFCFumDsDwAHG/fUoDcighAmh41Dn+KU1J4UEXEHliOb3fJAXIm6Yh0QAJyfFsmCwhFhOlR5sBRJAMhiCh4ICluRNBd03IVYYBUPRAcPYpsIF2RBgiBI4BFNpCeCM/DiYNBkQ+kI0L08OJgUyQEOZHURIQeYShSglxEGCCuHEMWKUEaIn0PS45xiqQgdRGSh4GGFkVWkKpICcSWQ5ciLUhNpOuhyWFhcQFB98vPSaQFMve4m0dQiqsIul5JSiJ1DyUOa4rMICWROUjxtYpMkRqkIFLxuOEITpEbZLs47DUPKYe/RW6Qhcitx3IeCSiyg2xnj9YNyJwjB8UpiUFmIgUPBgfa4ZrMIHuSG5GZx5kjk8UxuUFuRS4gDA50+aUkB7kRWXokozglO8he5Abk5HHkyEdxSnqQq8inR40D3TUp+UEuIgeQGge6ZUZWAHIWOXq8zjXQ/bKzBpDt9iBy8LjlQDcrzDpAtnuQl/08clscsxKQvciX7BTnrARk+x+6SK2sBGTz6/3Hj79P+f4dXepI1gHy5/df73uRq8mVJp3NKm7I5s/vXz//fT+RzEw+aXLgrAHkz5/fv//6+fP9IlIxuaWJi5MfZHPw+HUGuYi0Te5pYuGkBzl6/P7rAvIpQiGZywSwSQ6y5zh4HAby85/3OQnDZEkDwskNcvY4DuR41eciApMSjSNOZpDNxePXPci9iNykTGNskxjkwHH02A/kAPLzvSYybFKhscBJC7L59DgN5BakQKJkUqXRwkkKcuI4eVwGcr3qFRFlkwbNiE1OkDuPXxeQf997IjYmLRo2TkaQzZ1HDaQiYmrSpiHhJAQ5cyw85iB1EgeTHk0VJx3IZuZxA3J/RJoijiZdmXubbCBFjypIS8SbhETz/XsukCvH1aMD0hSBmbRoUv3WZ8djeUT6JGCTc+480oB8cnx6zEAKE+mKBDE5JxFI3aMD0hcJZJIGZFP0oIFQRKKYJAG55ah7FK86gySCSQ6QmscCpHjVGSJ4kwwgm47HLUh1ImQRsEkCkDuOOw8WCEMEaRIeZMPxaIKwRGAm0UHuOe49uCApSGL/ZetN22MJoiuCMAkN0va4DIQDwhbxNon8t9/nHAuPwkC6IAIRV5PAID2P4kD6IDISN5Ow/9Fl0/eQgshEfEyi/sujJcfC4zqQGYihiINJUBCSR3EgtInIRaxNQoIUOJYelYEQQYZILE0ighA9BkGGROxM4oFsaB6fA1mCuIgYkYQDKXEUPWoDoV11DRETkymWSHEeJY/6QBggCiTaJlMsEIaHDoiCiK5JKJAyR9GjBUI/IloiiiZTIBEtDx6IkoiSyRQHpMJR9uiAcN4sRRIFkzAgG7GHAoieyLDJFESE6aENoikyZhIDpMoh8ziAQEXkJlMIELZHD4R51U1IZCYRQDZjHmUQ9ptlICIhCQBS56h6GIHoi/BN4CCNeQx4SI6IkQjTBA3S4Kh72IHYiHBMJqhIax5UD10QMxKqCRRE6GELYiZCMpmAIE2OQQ/pVTcW6ZvgQDZiDzJISJGOyRzETWTIwxrEmqRusvBwAulwND1mA7EBsRapmYBABj0oAxkFsRcpmixBHER6HG0P4kD2IPFFFiQFD3uQYQ8ayPBEfEh+oEE2gx7zgViC+IjcmpRAbEW6HH0P4kA0QLxEribeIP159DzoAzmC5BE5mhQ97EA2Kh4MEIWJuJL8KINYiRA4uh6LgdiDOIpUPGxAKPOgeJAHonNEXEVqIBYiFI6+B2cgeiBeIlUPfRDSPEgeXBAdER+SOoi2iJYHG0RrIi4iDQ9dEBqHvocqiIOIF4ieBxTEXqQFoieyMfUggOiJGJM0PdRAiBwkDzaI5lW3F2mD6IhQ52HkofxmGYvYg5A5aB4RQAxFOh4KIg4e/iB2ItYgG20PMYiyiBFJ12MQhM5h6GEDYiPSBxkRYcyD6iECsXizbEQIHgMgXh4oEAMRSxAOB9ljBMRARJ2EAiIUieRhB6IsQvIQgbA46B7xQHRFaCACEUOPaCCaIkQPNsjGyqM8EAKIqYgeCRWEJ8LkYHqIBmIMoiVC9mCBGHrIB2IMoiRCB6GLbIw9goKoiDA8yCBcDg2PECAaJBwQmgh7HiyPkYEcQKKLsDxIIGwOFY8wIKMiPJC+CH8ePI8xEIc3a1CE6dEVgXkEAhkhYXu0QQQcTI/BgZxB4orwQVoiQA8GiMdEpCICjzrInMPCIw2IUEQCUhERcYg8coDISPRAfDyGB+J11YUiIo8SiIxD5qECElREBrIU8fKoD4QF4jYRrojQYy6y4DD1SAXCExF73IsIOQQeGgNxBuGQDHhM4/OQemiBBBQZAZkQHo2BMEBcrzpHZMjjLCLmEHvogLi+WUSRQY/J30NpIAAQEskoyDTAIffICtIXGfaYJmeP1kAkILFEFDwWIA4eiUE6Ihogk4xD6KE2kCOI/5vVJlHxuBNx8dABwRyRpoiSx+ToYQASSEQLZGJz2HikAamIqHmcRRw8NAeCBCmSKHpMPA4jj1QgBRFNkMnHQx8kkIiqxzQF8OCCgCfyburBAJF76A4EDnJPog1CFjHzSAjybuhBFRnwWCHIu6EHDcTQQwYSRMQChCIy4qE+kBAgJxITD4KIpYcAJMKbdRQx8uiKDHmsF+TdzKMjouBhBIIVMfRogox5WAwkyBGxBGmIaHisEsTUoy4y6NEdSFoQY4+aiIrHGkHMPcoiox42Awlw1R08SiI6HkYgyIm4eCxFhj36A8kJ4uQxF1HyWB+Im8ek7GE2kE8QiIgfyGTgsT4QR48bEgUPwkCkIMg3y9fjIqLmsTYQb46ziIaH5UBgbxbAY1L1eIJoiKh4rBIE4rGPl8cgiPsRQXmoiJgOBHLVcRwaIrYDQbxZUI8nSDSQURGSRyoQsMcoifVA3K86WuOQyAO5AfEQQVOc8wSJ5SEXoXkMgXgeETTDTcIOxBMEjXCXsAPxA0ETzBIbxEEEDbCM0GMlIOj2Swk5EB8QdPWVRByIyxFBF1+NxUAUQaxE0LW3oj8QHRDLiaA7byfcQKxB0IX3E2wgxiDotilRHYgqiLoIumpqFAcSGgTdMz2BBmL3ZqFL5iXMQMxA0A1zozOQsCDoeiXRGIgyiJoIultZYgzEAARdrDwRBnIGUXyz0K2OJAKI8hFBVzqaIQ91kHERdJ/jgQ9EEwRdpkrQA1EEQVeplBggCkcEXaRmRB6RQNAF6gc2EJU3C92eRVADUQBBV2eVrCDo3uzC81AGkR4RdGnGAQzkDoQpgq7LI+4DGQBBd+WTNCDoohzjCyK66uiK3OPocQdCEkGXg4k3CHki6GJwCQmCLgWcYCDoOkLE3IMKgi4iTqxBaFcd3UKk+IA0J4KuIF6AIOhPDxsrjyYI+qODxwHkn6cGM04g6M9MFzMR9IcljgEI+pPSRxUE/TEryiAI+sdfZ54YYfNkiJ7V9P9/3pGpmThWWcUAAAAASUVORK5CYII=" usemap="#imap_piechart_0"><map name='imap_piechart_0'><area shape='poly' coords='199,199,199,0,216,0,233,3,250,6,267,12,283,18,298,26,313,35,316,38' id='pie_0_slice_0'  onmouseover='hover(event, "pie_0_slice_0", "0");'>
<area shape='poly' coords='199,199,316,38,330,49,342,61' id='pie_0_slice_1'  onmouseover='hover(event, "pie_0_slice_1", "0");'>
<area shape='poly' coords='199,199,342,61,353,74,364,88,367,93' id='pie_0_slice_2'  onmouseover='hover(event, "pie_0_slice_2", "0");'>
<area shape='poly' coords='199,199,367,93,373,103' id='pie_0_slice_3'  onmouseover='hover(event, "pie_0_slice_3", "0");'>
<area shape='poly' coords='199,199,373,103,381,119,382,121' id='pie_0_slice_5'  onmouseover='hover(event, "pie_0_slice_5", "0");'>
<area shape='poly' coords='199,199,382,121,388,137,393,154,395,165' id='pie_0_slice_6'  onmouseover='hover(event, "pie_0_slice_6", "0");'>
<area shape='poly' coords='199,199,395,165,397,183,397,200,397,216' id='pie_0_slice_7'  onmouseover='hover(event, "pie_0_slice_7", "0");'>
<area shape='poly' coords='199,199,397,216,396,222' id='pie_0_slice_8'  onmouseover='hover(event, "pie_0_slice_8", "0");'>
<area shape='poly' coords='199,199,396,222,393,239,389,256,386,266' id='pie_0_slice_9'  onmouseover='hover(event, "pie_0_slice_9", "0");'>
<area shape='poly' coords='199,199,386,266,379,282,371,298,362,313,353,324' id='pie_0_slice_10'  onmouseover='hover(event, "pie_0_slice_10", "0");'>
<area shape='poly' coords='199,199,353,324,342,337,329,349,315,360,301,369,286,377,270,384,253,390,242,393' id='pie_0_slice_11'  onmouseover='hover(event, "pie_0_slice_11", "0");'>
<area shape='poly' coords='199,199,242,393,234,394' id='pie_0_slice_14'  onmouseover='hover(event, "pie_0_slice_14", "0");'>
<area shape='poly' coords='199,199,234,394,229,395' id='pie_0_slice_15'  onmouseover='hover(event, "pie_0_slice_15", "0");'>
<area shape='poly' coords='199,199,229,395,211,397,194,397,177,396,160,394,149,391' id='pie_0_slice_16'  onmouseover='hover(event, "pie_0_slice_16", "0");'>
<area shape='poly' coords='199,199,149,391,132,386,116,380,114,378' id='pie_0_slice_17'  onmouseover='hover(event, "pie_0_slice_17", "0");'>
<area shape='poly' coords='199,199,114,378,107,375' id='pie_0_slice_18'  onmouseover='hover(event, "pie_0_slice_18", "0");'>
<area shape='poly' coords='199,199,107,375,92,366,77,356,64,345,56,337' id='pie_0_slice_20'  onmouseover='hover(event, "pie_0_slice_20", "0");'>
<area shape='poly' coords='199,199,56,337,44,324,36,313' id='pie_0_slice_21'  onmouseover='hover(event, "pie_0_slice_21", "0");'>
<area shape='poly' coords='199,199,36,313,31,306' id='pie_0_slice_22'  onmouseover='hover(event, "pie_0_slice_22", "0");'>
<area shape='poly' coords='199,199,31,306,22,291,15,275,9,259,4,242,1,225,0,208,0,190,1,173,4,156,9,139,14,123,22,107,30,92,40,78,49,68' id='pie_0_slice_23'  onmouseover='hover(event, "pie_0_slice_23", "0");'>
<area shape='poly' coords='199,199,49,68,60,55' id='pie_0_slice_24'  onmouseover='hover(event, "pie_0_slice_24", "0");'>
<area shape='poly' coords='199,199,60,55,73,44,75,43' id='pie_0_slice_25'  onmouseover='hover(event, "pie_0_slice_25", "0");'>
<area shape='poly' coords='199,199,75,43,89,32,104,23,119,16,136,10,152,5,169,2,187,0,198,0' id='pie_0_slice_26'  onmouseover='hover(event, "pie_0_slice_26", "0");'></map><input type='hidden' id='hover_redundancies_0' value='pie_0_slice_26~#pie_0_slice_5~#pie_0_slice_9~#pie_0_slice_22~#pie_0_slice_16~#pie_0_slice_25~#pie_0_slice_17~#pie_0_slice_23~#pie_0_slice_15~#pie_0_slice_20~#pie_0_slice_6~#pie_0_slice_2~#pie_0_slice_24~#pie_0_slice_1~#pie_0_slice_8~#pie_0_slice_0~#pie_0_slice_3~#pie_0_slice_10~#pie_0_slice_7~#pie_0_slice_11~#pie_0_slice_14~#pie_0_slice_21~#pie_0_slice_18'><span style='display: none;' id='tooltip_0_pie_0_slice_0'><table><tr><th>Cofactors, Vitamins, Prosthetic Groups, Pigments</th></tr><tr><td>131 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_1'><table><tr><th>Cell Wall and Capsule</th></tr><tr><td>36 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_2'><table><tr><th>Virulence, Disease and Defense</th></tr><tr><td>43 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_3'><table><tr><th>Potassium metabolism</th></tr><tr><td>12 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_5'><table><tr><th>Miscellaneous</th></tr><tr><td>21 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_6'><table><tr><th>Phages, Prophages, Transposable elements, Plasmids</th></tr><tr><td>48 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_7'><table><tr><th>Membrane Transport</th></tr><tr><td>53 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_8'><table><tr><th>Iron acquisition and metabolism</th></tr><tr><td>6 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_9'><table><tr><th>RNA Metabolism</th></tr><tr><td>48 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_10'><table><tr><th>Nucleosides and Nucleotides</th></tr><tr><td>69 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_11'><table><tr><th>Protein Metabolism</th></tr><tr><td>139 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_14'><table><tr><th>Regulation and Cell signaling</th></tr><tr><td>8 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_15'><table><tr><th>Secondary Metabolism</th></tr><tr><td>6 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_16'><table><tr><th>DNA Metabolism</th></tr><tr><td>84 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_17'><table><tr><th>Fatty Acids, Lipids, and Isoprenoids</th></tr><tr><td>39 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_18'><table><tr><th>Nitrogen Metabolism</th></tr><tr><td>8 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_20'><table><tr><th>Respiration</th></tr><tr><td>66 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_21'><table><tr><th>Stress Response</th></tr><tr><td>33 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_22'><table><tr><th>Metabolism of Aromatic Compounds</th></tr><tr><td>9 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_23'><table><tr><th>Amino Acids and Derivatives</th></tr><tr><td>267 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_24'><table><tr><th>Sulfur Metabolism</th></tr><tr><td>18 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_25'><table><tr><th>Phosphorus Metabolism</th></tr><tr><td>20 / 1303</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_26'><table><tr><th>Carbohydrates</th></tr><tr><td>138 / 1303</td></tr></table></span>
</td><td>
<div id='tree_0'>
<div name='tree_level_0' id='tree_node_0_0'><img id='tree_img_0_0' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "0");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/AAAZ4gk3AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cofactors, Vitamins, Prosthetic Groups, Pigments (131)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_0'>
<img id='tree_img_0_1' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "1");'>&nbsp;Biotin (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_1'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=376686.40'>Biotin biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=376686.40'>Biotin biosynthesis Experimental</a> (4)<br></div>
<img id='tree_img_0_4' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "4");'>&nbsp;Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_4'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=376686.40'>Thiamin biosynthesis</a> (7)<br></div>
<span style='cursor: pointer;'>&nbsp;Quinone cofactors (0)<br>
<img id='tree_img_0_7' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "7");'>&nbsp;Tetrapyrroles (19)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_7'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=376686.40'>Cobalamin synthesis</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40'>Heme and Siroheme Biosynthesis</a> (14)<br></div>
<img id='tree_img_0_10' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "10");'>&nbsp;Riboflavin, FMN, FAD (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_10'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=376686.40'>riboflavin to FAD</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=376686.40'>Riboflavin, FMN and FAD metabolism</a> (9)<br></div>
<span style='cursor: pointer;'>&nbsp;Fe-S clusters (0)<br>
<span style='cursor: pointer;'>&nbsp;Mycofactocin (0)<br>
<img id='tree_img_0_15' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "15");'>&nbsp;Pyridoxine (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_15'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=376686.40'>Pyridoxin (Vitamin B6) Biosynthesis</a> (10)<br></div>
<img id='tree_img_0_17' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "17");'>&nbsp;NAD and NADP (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_17'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40'>NAD and NADP cofactor biosynthesis global</a> (12)<br></div>
<span style='cursor: pointer;'>&nbsp;Coenzyme B (0)<br>
<img id='tree_img_0_20' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "20");'>&nbsp;Folate and pterines (45)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_20'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40'>Folate biosynthesis cluster</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40'>Folate Biosynthesis</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40'>5-FCL-like protein</a> (16)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=376686.40'>Pterin synthesis related cluster</a> (8)<br></div>
<span style='cursor: pointer;'>&nbsp;Lipoic acid (0)<br>
<span style='cursor: pointer;'>&nbsp;Coenzyme F420 (0)<br>
<span style='cursor: pointer;'>&nbsp;Coenzyme M (0)<br>
<img id='tree_img_0_28' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "28");'>&nbsp;Coenzyme A (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_28'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=376686.40'>Coenzyme A Biosynthesis cluster</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40'>Coenzyme A Biosynthesis</a> (9)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_31'><img id='tree_img_0_31' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "31");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAgACc+aWRAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cell Wall and Capsule (36)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_31'>
<img id='tree_img_0_32' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "32");'>&nbsp;Capsular and extracellular polysacchrides (21)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_32'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=376686.40'>dTDP-rhamnose synthesis</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Streptococcal_Hyaluronic_Acid_Capsule&organism=376686.40'>Streptococcal Hyaluronic Acid Capsule</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=376686.40'>CMP-N-acetylneuraminate Biosynthesis</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polysaccharide_deacetylases&organism=376686.40'>Polysaccharide deacetylases</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Capsular_Polysaccharides_Biosynthesis_and_Assembly&organism=376686.40'>Capsular Polysaccharides Biosynthesis and Assembly</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=376686.40'>Rhamnose containing glycans</a> (7)<br></div>
<img id='tree_img_0_39' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "39");'>&nbsp;Gram-Negative cell wall components (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_39'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lipoprotein_sorting_system&organism=376686.40'>Lipoprotein sorting system</a> (1)<br></div>
<img id='tree_img_0_41' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "41");'>&nbsp;Cell Wall and Capsule - no subcategory (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_41'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=376686.40'>Murein Hydrolases</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=376686.40'>Recycling of Peptidoglycan Amino Acids</a> (4)<br></div>
<img id='tree_img_0_44' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "44");'>&nbsp;Gram-Positive cell wall components (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_44'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Teichoic_and_lipoteichoic_acids_biosynthesis&organism=376686.40'>Teichoic and lipoteichoic acids biosynthesis</a> (4)<br></div>
<span style='cursor: pointer;'>&nbsp;Cell wall of Mycobacteria (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_47'><img id='tree_img_0_47' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "47");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAAIBKwr76AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Virulence, Disease and Defense (43)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_47'>
<span style='cursor: pointer;'>&nbsp;Adhesion (0)<br>
<span style='cursor: pointer;'>&nbsp;Toxins and superantigens (0)<br>
<span style='cursor: pointer;'>&nbsp;Bacteriocins, ribosomally synthesized antibacterial peptides (0)<br>
<img id='tree_img_0_51' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "51");'>&nbsp;Resistance to antibiotics and toxic compounds (24)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_51'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=376686.40'>Copper homeostasis</a> (8)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=376686.40'>Cobalt-zinc-cadmium resistance</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=376686.40'>Resistance to fluoroquinolones</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis:_copper_tolerance&organism=376686.40'>Copper homeostasis: copper tolerance</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Beta-lactamase&organism=376686.40'>Beta-lactamase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_chromium_compounds&organism=376686.40'>Resistance to chromium compounds</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=376686.40'>Multidrug Resistance Efflux Pumps</a> (4)<br></div>
<span style='cursor: pointer;'>&nbsp;Virulence, Disease and Defense - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Detection (0)<br>
<img id='tree_img_0_61' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "61");'>&nbsp;Invasion and intracellular resistance (19)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_61'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=376686.40'>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=376686.40'>Mycobacterium virulence operon involved in DNA transcription</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=376686.40'>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Listeria_surface_proteins:_Internalin-like_proteins&organism=376686.40'>Listeria surface proteins: Internalin-like proteins</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=376686.40'>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a> (3)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_67'><img id='tree_img_0_67' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "67");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAgAB9zL4RAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Potassium metabolism (12)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_67'>
<img id='tree_img_0_68' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "68");'>&nbsp;Potassium metabolism - no subcategory (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_68'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40'>Potassium homeostasis</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutathione-regulated_potassium-efflux_system_and_associated_functions&organism=376686.40'>Glutathione-regulated potassium-efflux system and associated functions</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_71'><img id='tree_img_0_71' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "71");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAICr96V6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Photosynthesis (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_71'>
<span style='cursor: pointer;'>&nbsp;Light-harvesting complexes (0)<br>
<span style='cursor: pointer;'>&nbsp;Photosynthesis - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Electron transport and photophosphorylation (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_75'><img id='tree_img_0_75' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "75");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXAwMBmS75kAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Miscellaneous (21)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_75'>
<span style='cursor: pointer;'>&nbsp;Conversion of Succinyl-CoA to Propionyl-CoA (0)<br>
<img id='tree_img_0_77' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "77");'>&nbsp;Plant-Prokaryote DOE project (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_77'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=376686.40'>Iron-sulfur cluster assembly</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Single-Rhodanese-domain_proteins&organism=376686.40'>Single-Rhodanese-domain proteins</a> (1)<br></div>
<img id='tree_img_0_80' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "80");'>&nbsp;Miscellaneous - no subcategory (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_80'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phosphoglycerate_mutase_protein_family&organism=376686.40'>Phosphoglycerate mutase protein family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DedA_family_of_inner_membrane_proteins&organism=376686.40'>DedA family of inner membrane proteins</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Muconate_lactonizing_enzyme_family&organism=376686.40'>Muconate lactonizing enzyme family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Bacillus_subtilis_scratch_-_gjo&organism=376686.40'>Bacillus subtilis scratch - gjo</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=376686.40'>Broadly distributed proteins not in subsystems</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_86'><img id='tree_img_0_86' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "86");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAgICQdD0xAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Phages, Prophages, Transposable elements, Plasmids (48)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_86'>
<span style='cursor: pointer;'>&nbsp;Phage family-specific subsystems (0)<br>
<img id='tree_img_0_88' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "88");'>&nbsp;Transposable elements (48)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_88'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40'>Conjugative transposon, Bacteroidales</a> (48)<br></div>
<span style='cursor: pointer;'>&nbsp;Phages, Prophages (0)<br>
<span style='cursor: pointer;'>&nbsp;Phages, Prophages, Transposable elements, Plasmids - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Pathogenicity islands (0)<br>
<span style='cursor: pointer;'>&nbsp;Gene Transfer Agent (GTA) (0)<br>
<span style='cursor: pointer;'>&nbsp;Plasmid related functions (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_95'><img id='tree_img_0_95' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "95");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZmf/YOWB7AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Membrane Transport (53)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_95'>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type II (0)<br>
<span style='cursor: pointer;'>&nbsp;ABC transporters (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VII (Chaperone/Usher pathway, CU) (0)<br>
<img id='tree_img_0_99' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "99");'>&nbsp;Protein translocation across cytoplasmic membrane (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_99'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=376686.40'>Twin-arginine translocation system</a> (4)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type V (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type I (0)<br>
<img id='tree_img_0_103' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "103");'>&nbsp;Cation transporters (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_103'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Magnesium_transport&organism=376686.40'>Magnesium transport</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_Transport_System&organism=376686.40'>Copper Transport System</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_transport_and_blue_copper_proteins&organism=376686.40'>Copper transport and blue copper proteins</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type III (0)<br>
<img id='tree_img_0_108' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "108");'>&nbsp;Uni- Sym- and Antiporters (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_108'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters&organism=376686.40'>NhaA, NhaD and Sodium-dependent phosphate transporters</a> (2)<br></div>
<img id='tree_img_0_110' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "110");'>&nbsp;Membrane Transport - no subcategory (34)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_110'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=376686.40'>Ton and Tol transport systems</a> (34)<br></div>
<span style='cursor: pointer;'>&nbsp;TRAP transporters (0)<br>
<span style='cursor: pointer;'>&nbsp;Sugar Phosphotransferase Systems, PTS (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VI (0)<br>
<img id='tree_img_0_115' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "115");'>&nbsp;Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP) (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_115'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Curli_production&organism=376686.40'>Curli production</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein and nucleoprotein secretion system, Type IV (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_118'><img id='tree_img_0_118' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "118");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZM2YFAh/4AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Iron acquisition and metabolism (6)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_118'>
<img id='tree_img_0_119' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "119");'>&nbsp;Siderophores (6)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_119'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=376686.40'>Siderophore assembly kit</a> (6)<br></div>
<span style='cursor: pointer;'>&nbsp;Iron acquisition and metabolism - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Iron transpot (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_123'><img id='tree_img_0_123' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "123");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//8wYFHreAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;RNA Metabolism (48)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_123'>
<img id='tree_img_0_124' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "124");'>&nbsp;RNA processing and modification (23)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_124'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=376686.40'>mnm5U34 biosynthesis bacteria</a> (8)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=376686.40'>Ribonuclease H</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_3'-terminal_phosphate_cyclase&organism=376686.40'>RNA 3'-terminal phosphate cyclase</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=376686.40'>RNA processing and degradation, bacterial</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=376686.40'>tRNA processing</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=376686.40'>RNA methylation</a> (5)<br></div>
<img id='tree_img_0_131' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "131");'>&nbsp;Transcription (23)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_131'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=376686.40'>Transcription initiation, bacterial sigma factors</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=376686.40'>RNA polymerase bacterial</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40'>Transcription factors bacterial</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=376686.40'>Rrf2 family transcriptional regulators</a> (4)<br></div>
<img id='tree_img_0_136' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "136");'>&nbsp;RNA Metabolism - no subcategory (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_136'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Group_II_intron-associated_genes&organism=376686.40'>Group II intron-associated genes</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_138'><img id='tree_img_0_138' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "138");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEVmAGZP8U4lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Nucleosides and Nucleotides (69)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_138'>
<img id='tree_img_0_139' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "139");'>&nbsp;Pyrimidines (15)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_139'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40'>pyrimidine conversions</a> (15)<br></div>
<img id='tree_img_0_141' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "141");'>&nbsp;Purines (39)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_141'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40'>De Novo Purine Biosynthesis</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Purine_Utilization&organism=376686.40'>Purine Utilization</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=376686.40'>GMP synthase</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40'>Purine conversions</a> (23)<br></div>
<img id='tree_img_0_146' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "146");'>&nbsp;Nucleosides and Nucleotides - no subcategory (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_146'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=376686.40'>Ribonucleotide reduction</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=dNTP_triphosphohydrolase_protein_family&organism=376686.40'>dNTP triphosphohydrolase protein family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=376686.40'>Adenosyl nucleosidases</a> (3)<br></div>
<img id='tree_img_0_150' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "150");'>&nbsp;Detoxification (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_150'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nucleoside_triphosphate_pyrophosphohydrolase_MazG&organism=376686.40'>Nucleoside triphosphate pyrophosphohydrolase MazG</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=376686.40'>Nudix proteins (nucleoside triphosphate hydrolases)</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=376686.40'>Housecleaning nucleoside triphosphate pyrophosphatases</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_154'><img id='tree_img_0_154' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "154");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/gIDP2RJcAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Protein Metabolism (139)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_154'>
<img id='tree_img_0_155' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "155");'>&nbsp;Protein folding (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_155'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=376686.40'>Protein chaperones</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_disulfide_interchange&organism=376686.40'>Periplasmic disulfide interchange</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Peptidyl-prolyl_cis-trans_isomerase&organism=376686.40'>Peptidyl-prolyl cis-trans isomerase</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Selenoproteins (0)<br>
<img id='tree_img_0_160' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "160");'>&nbsp;Protein biosynthesis (87)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_160'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Val&organism=376686.40'>tRNA aminoacylation, Val</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Met&organism=376686.40'>tRNA aminoacylation, Met</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=rRNAs&organism=376686.40'>rRNAs</a> (18)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ile&organism=376686.40'>tRNA aminoacylation, Ile</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Arg&organism=376686.40'>tRNA aminoacylation, Arg</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=376686.40'>Translation initiation factors bacterial</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40'>Ribosome SSU bacterial</a> (23)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Gly&organism=376686.40'>tRNA aminoacylation, Gly</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ala&organism=376686.40'>tRNA aminoacylation, Ala</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Trp&organism=376686.40'>tRNA aminoacylation, Trp</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=376686.40'>Programmed frameshift</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Cys&organism=376686.40'>tRNA aminoacylation, Cys</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40'>Translation termination factors bacterial</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_His&organism=376686.40'>tRNA aminoacylation, His</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=376686.40'>tRNA aminoacylation, Asp and Asn</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=376686.40'>Translation elongation factors bacterial</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Lys&organism=376686.40'>tRNA aminoacylation, Lys</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Thr&organism=376686.40'>tRNA aminoacylation, Thr</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Pro&organism=376686.40'>tRNA aminoacylation, Pro</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factor_G_family&organism=376686.40'>Translation elongation factor G family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=376686.40'>tRNA aminoacylation, Glu and Gln</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ser&organism=376686.40'>tRNA aminoacylation, Ser</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Tyr&organism=376686.40'>tRNA aminoacylation, Tyr</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Leu&organism=376686.40'>tRNA aminoacylation, Leu</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=376686.40'>tRNA aminoacylation, Phe</a> (2)<br></div>
<img id='tree_img_0_186' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "186");'>&nbsp;Protein processing and modification (24)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_186'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein-L-isoaspartate_O-methyltransferase&organism=376686.40'>Protein-L-isoaspartate O-methyltransferase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=376686.40'>Signal peptidase</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=376686.40'>Modification of eukaryotic initiation factor 5A</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40'>G3E family of P-loop GTPases (metallocenter biosynthesis)</a> (14)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=376686.40'>N-linked Glycosylation in Bacteria</a> (5)<br></div>
<img id='tree_img_0_192' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "192");'>&nbsp;Protein degradation (18)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_192'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=376686.40'>Aminopeptidases (EC 3.4.11.-)</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=376686.40'>Protein degradation</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=376686.40'>Metallocarboxypeptidases (EC 3.4.17.-)</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dipeptidases_(EC_3.4.13.-)&organism=376686.40'>Dipeptidases (EC 3.4.13.-)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Serine_endopeptidase_(EC_3.4.21.-)&organism=376686.40'>Serine endopeptidase (EC 3.4.21.-)</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=376686.40'>Proteolysis in bacteria, ATP-dependent</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_199'><img id='tree_img_0_199' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "199");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAZswGD39gAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cell Division and Cell Cycle (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_199'>
<span style='cursor: pointer;'>&nbsp;Checkpoint control (0)<br>
<span style='cursor: pointer;'>&nbsp;Heterocyst formation (0)<br>
<span style='cursor: pointer;'>&nbsp;Cell Division and Cell Cycle - no subcategory (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_203'><img id='tree_img_0_203' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "203");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXMzP91giUxAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Motility and Chemotaxis (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_203'>
<span style='cursor: pointer;'>&nbsp;Magnetotaxis (0)<br>
<span style='cursor: pointer;'>&nbsp;Motility and Chemotaxis - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Flagellar motility in Prokaryota (0)<br>
<span style='cursor: pointer;'>&nbsp;Social motility and nonflagellar swimming in bacteria (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_208'><img id='tree_img_0_208' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "208");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/AP804Oa6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Regulation and Cell signaling (8)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_208'>
<img id='tree_img_0_209' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "209");'>&nbsp;Regulation and Cell signaling - no subcategory (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_209'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Salmonella_enterica_Typhimurium&organism=376686.40'>LysR-family proteins in Salmonella enterica Typhimurium</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Escherichia_coli&organism=376686.40'>LysR-family proteins in Escherichia coli</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=376686.40'>Stringent Response, (p)ppGpp metabolism</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Signal transduction in Eukaryotes (0)<br>
<span style='cursor: pointer;'>&nbsp;Quorum sensing and biofilm formation (0)<br>
<span style='cursor: pointer;'>&nbsp;Proteolytic pathway (0)<br>
<span style='cursor: pointer;'>&nbsp;Regulation of virulence (0)<br>
<img id='tree_img_0_217' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "217");'>&nbsp;Programmed Cell Death and Toxin-antitoxin Systems (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_217'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Toxin-antitoxin_replicon_stabilization_systems&organism=376686.40'>Toxin-antitoxin replicon stabilization systems</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phd-Doc,_YdcE-YdcD_toxin-antitoxin_(programmed_cell_death)_systems&organism=376686.40'>Phd-Doc, YdcE-YdcD toxin-antitoxin (programmed cell death) systems</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_220'><img id='tree_img_0_220' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "220");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//wCKxvRFAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Secondary Metabolism (6)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_220'>
<span style='cursor: pointer;'>&nbsp;Secondary Metabolism - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Lipid-derived mediators (0)<br>
<span style='cursor: pointer;'>&nbsp;Plant Octadecanoids (0)<br>
<span style='cursor: pointer;'>&nbsp;Bacterial cytostatics, differentiation factors and antibiotics (0)<br>
<span style='cursor: pointer;'>&nbsp;Biosynthesis of phenylpropanoids (0)<br>
<span style='cursor: pointer;'>&nbsp;Hydrocarbons (0)<br>
<span style='cursor: pointer;'>&nbsp;Aromatic amino acids and derivatives (0)<br>
<span style='cursor: pointer;'>&nbsp;UV-absorbing secondary metabolites (0)<br>
<img id='tree_img_0_229' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "229");'>&nbsp;Plant Alkaloids (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_229'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Alkaloid_biosynthesis_from_L-lysine&organism=376686.40'>Alkaloid biosynthesis from L-lysine</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Biologically active compounds in metazoan cell defence and differentiation (0)<br>
<img id='tree_img_0_232' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "232");'>&nbsp;Plant Hormones (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_232'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=376686.40'>Auxin biosynthesis</a> (5)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_234'><img id='tree_img_0_234' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "234");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUA//8ZXC8lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;DNA Metabolism (84)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_234'>
<img id='tree_img_0_235' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "235");'>&nbsp;DNA repair (54)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_235'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=376686.40'>DNA repair, bacterial MutL-MutS system</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=376686.40'>DNA repair, UvrABC system</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_photolyase&organism=376686.40'>DNA repair, bacterial photolyase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=2-phosphoglycolate_salvage&organism=376686.40'>2-phosphoglycolate salvage</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RecA_and_RecX&organism=376686.40'>RecA and RecX</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=376686.40'>DNA repair system including RecA, MutS and a hypothetical protein</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40'>DNA repair, bacterial</a> (24)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=376686.40'>DNA repair, bacterial RecFOR pathway</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nonhomologous_End-Joining_in_Bacteria&organism=376686.40'>Nonhomologous End-Joining in Bacteria</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;CRISPs (0)<br>
<img id='tree_img_0_246' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "246");'>&nbsp;DNA Metabolism - no subcategory (16)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_246'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=376686.40'>Type I Restriction-Modification</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=376686.40'>Restriction-Modification System</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_ligases&organism=376686.40'>DNA ligases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=YcfH&organism=376686.40'>YcfH</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=376686.40'>DNA structural proteins, bacterial</a> (2)<br></div>
<img id='tree_img_0_252' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "252");'>&nbsp;DNA replication (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_252'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=376686.40'>DNA topoisomerases, Type I, ATP-independent</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_replication_strays&organism=376686.40'>DNA replication strays</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=376686.40'>DNA topoisomerases, Type II, ATP-dependent</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;DNA recombination (0)<br>
<img id='tree_img_0_257' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "257");'>&nbsp;DNA uptake, competence (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_257'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=376686.40'>DNA processing cluster</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_259'><img id='tree_img_0_259' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "259");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAICr96V6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Fatty Acids, Lipids, and Isoprenoids (39)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_259'>
<span style='cursor: pointer;'>&nbsp;Phospholipids (0)<br>
<span style='cursor: pointer;'>&nbsp;Triacylglycerols (0)<br>
<img id='tree_img_0_262' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "262");'>&nbsp;Fatty acids (21)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_262'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40'>Fatty Acid Biosynthesis FASII</a> (17)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=376686.40'>Fatty acid metabolism cluster</a> (4)<br></div>
<span style='cursor: pointer;'>&nbsp;Fatty Acids, Lipids, and Isoprenoids - no subcategory (0)<br>
<img id='tree_img_0_266' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "266");'>&nbsp;Isoprenoids (18)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_266'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Myxoxanthophyll_biosynthesis_in_Cyanobacteria&organism=376686.40'>Myxoxanthophyll biosynthesis in Cyanobacteria</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=376686.40'>Mevalonate Branch of Isoprenoid Biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=376686.40'>Isoprenoinds for Quinones</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=376686.40'>Isoprenoid Biosynthesis</a> (8)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_271'><img id='tree_img_0_271' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "271");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAABGTyZaAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Nitrogen Metabolism (8)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_271'>
<img id='tree_img_0_272' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "272");'>&nbsp;Nitrogen Metabolism - no subcategory (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_272'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nitrosative_stress&organism=376686.40'>Nitrosative stress</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=376686.40'>Ammonia assimilation</a> (7)<br></div>
<span style='cursor: pointer;'>&nbsp;Denitrification (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_276'><img id='tree_img_0_276' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "276");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAgIBxQSaxAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Dormancy and Sporulation (1)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_276'>
<span style='cursor: pointer;'>&nbsp;Spore DNA protection (0)<br>
<img id='tree_img_0_278' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "278");'>&nbsp;Dormancy and Sporulation - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_278'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Sporulation-associated_proteins_with_broader_functions&organism=376686.40'>Sporulation-associated proteins with broader functions</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_280'><img id='tree_img_0_280' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "280");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAAP+KeNJXAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Respiration (66)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_280'>
<span style='cursor: pointer;'>&nbsp;Biotin (10)<br>
<img id='tree_img_0_282' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "282");'>&nbsp;ATP synthases (10)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_282'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40'>F0F1-type ATP synthase</a> (10)<br></div>
<img id='tree_img_0_284' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "284");'>&nbsp;Electron accepting reactions (6)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_284'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_d_ubiquinol_oxidases&organism=376686.40'>Terminal cytochrome d ubiquinol oxidases</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=376686.40'>Anaerobic respiratory reductases</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_oxidases&organism=376686.40'>Terminal cytochrome oxidases</a> (2)<br></div>
<img id='tree_img_0_288' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "288");'>&nbsp;Electron donating reactions (36)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_288'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40'>Respiratory Complex I</a> (14)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=376686.40'>Respiratory dehydrogenases 1</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=376686.40'>NiFe hydrogenase maturation</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40'>NADH ubiquinone oxidoreductase</a> (14)<br></div>
<span style='cursor: pointer;'>&nbsp;Reverse electron transport (0)<br>
<img id='tree_img_0_294' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "294");'>&nbsp;Respiration - no subcategory (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_294'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Quinone_oxidoreductase_family&organism=376686.40'>Quinone oxidoreductase family</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cytochrome_c_oxidases&organism=376686.40'>Biogenesis of cytochrome c oxidases</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=376686.40'>Biogenesis of cbb3-type cytochrome c oxidases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=376686.40'>Biogenesis of c-type cytochromes</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=376686.40'>Soluble cytochromes and functionally related electron carriers</a> (5)<br></div>
<span style='cursor: pointer;'>&nbsp;Sodium Ion-Coupled Energetics (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_301'><img id='tree_img_0_301' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "301");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAzMxS5ysDAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Stress Response (33)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_301'>
<img id='tree_img_0_302' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "302");'>&nbsp;Osmotic stress (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_302'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=376686.40'>Osmoregulation</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Dessication stress (0)<br>
<span style='cursor: pointer;'>&nbsp;Acid stress (0)<br>
<img id='tree_img_0_306' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "306");'>&nbsp;Oxidative stress (23)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_306'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=376686.40'>Oxidative stress</a> (18)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rubrerythrin&organism=376686.40'>Rubrerythrin</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Redox_cycle&organism=376686.40'>Glutathione: Redox cycle</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Cold shock (0)<br>
<span style='cursor: pointer;'>&nbsp;Heat shock (0)<br>
<img id='tree_img_0_312' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "312");'>&nbsp;Detoxification (6)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_312'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Uptake_of_selenate_and_selenite&organism=376686.40'>Uptake of selenate and selenite</a> (1)<br></div>
<img id='tree_img_0_314' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "314");'>&nbsp;Stress Response - no subcategory (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_314'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dimethylarginine_metabolism&organism=376686.40'>Dimethylarginine metabolism</a> (2)<br></div>
<img id='tree_img_0_316' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "316");'>&nbsp;Periplasmic Stress (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_316'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=376686.40'>Periplasmic Stress Response</a> (5)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_318'><img id='tree_img_0_318' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "318");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXM//+B6UABAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Metabolism of Aromatic Compounds (9)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_318'>
<img id='tree_img_0_319' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "319");'>&nbsp;Peripheral pathways for catabolism of aromatic compounds (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_319'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Quinate_degradation&organism=376686.40'>Quinate degradation</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Anaerobic degradation of aromatic compounds (0)<br>
<img id='tree_img_0_322' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "322");'>&nbsp;Metabolism of central aromatic intermediates (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_322'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=376686.40'>Catechol branch of beta-ketoadipate pathway</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=376686.40'>Salicylate and gentisate catabolism</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=376686.40'>Homogentisate pathway of aromatic compound degradation</a> (3)<br></div>
<img id='tree_img_0_326' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "326");'>&nbsp;Metabolism of Aromatic Compounds - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_326'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Gentisate_degradation&organism=376686.40'>Gentisate degradation</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_328'><img id='tree_img_0_328' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "328");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXM/8w+OSEXAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Amino Acids and Derivatives (267)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_328'>
<img id='tree_img_0_329' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "329");'>&nbsp;Glutamine, glutamate, aspartate, asparagine; ammonia assimilation (22)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_329'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40'>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a> (18)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=376686.40'>Glutamate dehydrogenases</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=376686.40'>Glutamine synthetases</a> (2)<br></div>
<img id='tree_img_0_333' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "333");'>&nbsp;Histidine Metabolism (11)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_333'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=376686.40'>Histidine Degradation</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=376686.40'>Histidine Biosynthesis</a> (7)<br></div>
<img id='tree_img_0_336' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "336");'>&nbsp;Arginine; urea cycle, polyamines (16)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_336'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=376686.40'>Urease subunits</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40'>Urea decomposition</a> (9)<br></div>
<img id='tree_img_0_339' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "339");'>&nbsp;Lysine, threonine, methionine, and cysteine (81)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_339'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40'>Methionine Biosynthesis</a> (27)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=376686.40'>Threonine degradation</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=376686.40'>Lysine Biosynthesis DAP Pathway, GJO scratch</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=376686.40'>Methionine Degradation</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=376686.40'>Threonine and Homoserine Biosynthesis</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=376686.40'>Cysteine Biosynthesis</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=376686.40'>Lysine Biosynthesis DAP Pathway</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=376686.40'>Lysine degradation</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Amino Acids and Derivatives - no subcategory (0)<br>
<img id='tree_img_0_349' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "349");'>&nbsp;Branched-chain amino acids (46)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_349'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=376686.40'>Isoleucine degradation</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40'>Leucine Degradation and HMG-CoA Metabolism</a> (14)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40'>Branched-Chain Amino Acid Biosynthesis</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=376686.40'>Valine degradation</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=376686.40'>Leucine Biosynthesis</a> (6)<br></div>
<span style='cursor: pointer;'>&nbsp;Polyamines (0)<br>
<img id='tree_img_0_356' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "356");'>&nbsp;Aromatic amino acids and derivatives (38)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_356'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=376686.40'>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40'>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=376686.40'>Phenylalanine and Tyrosine Branches from Chorismate</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40'>Tryptophan synthesis</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=376686.40'>Aromatic amino acid degradation</a> (6)<br></div>
<img id='tree_img_0_362' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "362");'>&nbsp;Proline and 4-hydroxyproline (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_362'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=376686.40'>Proline Synthesis</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=376686.40'>Proline, 4-hydroxyproline uptake and utilization</a> (3)<br></div>
<img id='tree_img_0_365' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "365");'>&nbsp;Alanine, serine, and glycine (46)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_365'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=376686.40'>Glycine Biosynthesis</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=376686.40'>Alanine biosynthesis</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=376686.40'>Serine Biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=376686.40'>Glycine cleavage system</a> (8)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40'>Glycine and Serine Utilization</a> (19)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_371'><img id='tree_img_0_371' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "371");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//5kDFd+lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Sulfur Metabolism (18)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_371'>
<span style='cursor: pointer;'>&nbsp;Inorganic sulfur assimilation (0)<br>
<img id='tree_img_0_373' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "373");'>&nbsp;Sulfur Metabolism - no subcategory (18)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_373'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=376686.40'>Thioredoxin-disulfide reductase</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=376686.40'>Galactosylceramide and Sulfatide metabolism</a> (9)<br></div>
<span style='cursor: pointer;'>&nbsp;DsrE (0)<br>
<span style='cursor: pointer;'>&nbsp;Organic sulfur assimilation (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_378'><img id='tree_img_0_378' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "378");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZzP8f9clqAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Phosphorus Metabolism (20)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_378'>
<img id='tree_img_0_379' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "379");'>&nbsp;Phosphorus Metabolism - no subcategory (20)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_379'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40'>Phosphate metabolism</a> (16)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=376686.40'>Polyphosphate</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_382'><img id='tree_img_0_382' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "382");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/mcwrs7b/AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Carbohydrates (138)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_382'>
<img id='tree_img_0_383' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "383");'>&nbsp;Central carbohydrate metabolism (55)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_383'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=376686.40'>Methylglyoxal Metabolism</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=376686.40'>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=376686.40'>Pyruvate Alanine Serine Interconversions</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=376686.40'>Glyoxylate bypass</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40'>Dehydrogenase complexes</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=376686.40'>Pentose phosphate pathway</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=376686.40'>Pyruvate metabolism I: anaplerotic reactions, PEP</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycolate,_glyoxylate_interconversions&organism=376686.40'>Glycolate, glyoxylate interconversions</a> (4)<br></div>
<img id='tree_img_0_392' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "392");'>&nbsp;Aminosugars (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_392'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=376686.40'>Chitin and N-acetylglucosamine utilization</a> (14)<br></div>
<img id='tree_img_0_394' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "394");'>&nbsp;Di- and oligosaccharides (13)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_394'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Sucrose_utilization&organism=376686.40'>Sucrose utilization</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Trehalose_Uptake_and_Utilization&organism=376686.40'>Trehalose Uptake and Utilization</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=376686.40'>Lactose and Galactose Uptake and Utilization</a> (7)<br></div>
<span style='cursor: pointer;'>&nbsp;Glycoside hydrolases (0)<br>
<img id='tree_img_0_399' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "399");'>&nbsp;One-carbon Metabolism (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_399'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=376686.40'>One-carbon metabolism by tetrahydropterines</a> (5)<br></div>
<img id='tree_img_0_401' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "401");'>&nbsp;Organic acids (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_401'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lactate_utilization&organism=376686.40'>Lactate utilization</a> (3)<br></div>
<img id='tree_img_0_403' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "403");'>&nbsp;Fermentation (17)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_403'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=376686.40'>Butanol Biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Acetolactate_synthase_subunits&organism=376686.40'>Acetolactate synthase subunits</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=376686.40'>Fermentations: Lactate</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=376686.40'>Acetyl-CoA fermentation to Butyrate</a> (5)<br></div>
<span style='cursor: pointer;'>&nbsp;CO2 fixation (0)<br>
<img id='tree_img_0_409' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "409");'>&nbsp;Sugar alcohols (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_409'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Inositol_catabolism&organism=376686.40'>Inositol catabolism</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Carbohydrates - no subcategory (0)<br>
<img id='tree_img_0_412' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "412");'>&nbsp;Polysaccharides (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_412'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=376686.40'>Glycogen metabolism</a> (5)<br></div>
<img id='tree_img_0_414' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "414");'>&nbsp;Monosaccharides (24)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_414'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=376686.40'>Mannose Metabolism</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=D-ribose_utilization&organism=376686.40'>D-ribose utilization</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Xylose_utilization&organism=376686.40'>Xylose utilization</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=376686.40'>Deoxyribose and Deoxynucleoside Catabolism</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=376686.40'>L-Arabinose utilization</a> (5)<br></div></div></div></div>
</td></tr></table></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='1_contents' id='1_content_1'><table class='table_table'><tr><td><input type='button' class='button' value='export to file' onclick='export_table("0", 0, 1, 0);'>&nbsp;&nbsp;<input type='button' class='button' value='clear all filters' onclick='table_reset_filters("0");'></td></tr><tr><td style='width: 100%; text-align: center; vertical-align: middle;'><table style='width: 100%;'>
<tr><td align=center><span class='table_perpage'>display&nbsp;<input type='text' id='table_perpage_0' name='table_perpage_0' size='3' value='15' onkeypress='return check_submit_filter(event, "0");'>&nbsp;items per page</span></td></tr></table></td></tr>
<tr><td style='width: 100%; text-align: center;'><table style='width: 100%;'><tr><td align='left' width='20%'><a href='javascript: table_first("0");' name='table_first_0'>&laquo;first</a>&nbsp;&nbsp;<a href='javascript: table_prev("0");' name='table_prev_0'>&laquo;prev</a></td><td align='center' width='60%'>displaying <span id='table_start_top_0'>1</span> - <span id='table_stop_top_0'>995</span> of <span id='table_total_top_0'>995</span></td><td align='right' width='20%'><a href='javascript: table_next("0");' name='table_next_0'>next&raquo;</a>&nbsp;&nbsp;<a href='javascript: table_last("0");' name='table_last_0'>last&raquo;</a></td></tr></table></td></tr><tr><td><input type='hidden' id='SubsystemTable' value='0' />
<input type='hidden' id='table_data_0' value='Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=376686.40@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)&subsystem_name=Biotin_biosynthesis@1>Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3353@1>fig|376686.40.peg.3353</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5009@1>fig|376686.40.peg.5009</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=376686.40@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin synthase (EC 2.8.1.6)&subsystem_name=Biotin_biosynthesis@1>Biotin synthase (EC 2.8.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.836@1>fig|376686.40.peg.836</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=376686.40@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-ketoacyl-CoA thiolase (EC 2.3.1.16)&subsystem_name=Biotin_biosynthesis@1>3-ketoacyl-CoA thiolase (EC 2.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3502@1>fig|376686.40.peg.3502</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=376686.40@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)&subsystem_name=Biotin_biosynthesis@1>Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.845@1>fig|376686.40.peg.845</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=376686.40@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=8-amino-7-oxononanoate synthase (EC 2.3.1.47)&subsystem_name=Biotin_biosynthesis@1>8-amino-7-oxononanoate synthase (EC 2.3.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.849@1>fig|376686.40.peg.849</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=376686.40@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin synthase (EC 2.8.1.6)&subsystem_name=Biotin_biosynthesis_Experimental@1>Biotin synthase (EC 2.8.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.836@1>fig|376686.40.peg.836</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=376686.40@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Competence protein F homolog, phosphoribosyltransferase domain&subsystem_name=Biotin_biosynthesis_Experimental@1>Competence protein F homolog, phosphoribosyltransferase domain</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.832@1>fig|376686.40.peg.832</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=376686.40@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)&subsystem_name=Biotin_biosynthesis_Experimental@1>Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.845@1>fig|376686.40.peg.845</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=376686.40@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=8-amino-7-oxononanoate synthase (EC 2.3.1.47)&subsystem_name=Biotin_biosynthesis_Experimental@1>8-amino-7-oxononanoate synthase (EC 2.3.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.849@1>fig|376686.40.peg.849</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=376686.40@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-regulated outer membrane receptor Omr1&subsystem_name=Thiamin_biosynthesis@1>Thiamin-regulated outer membrane receptor Omr1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1232@1>fig|376686.40.peg.1232</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1669@1>fig|376686.40.peg.1669</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=376686.40@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin pyrophosphokinase (EC 2.7.6.2)&subsystem_name=Thiamin_biosynthesis@1>Thiamin pyrophosphokinase (EC 2.7.6.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2290@1>fig|376686.40.peg.2290</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=376686.40@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamine-monophosphate kinase (EC 2.7.4.16)&subsystem_name=Thiamin_biosynthesis@1>Thiamine-monophosphate kinase (EC 2.7.4.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.826@1>fig|376686.40.peg.826</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=376686.40@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)&subsystem_name=Thiamin_biosynthesis@1>Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1681@1>fig|376686.40.peg.1681</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=376686.40@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfur carrier protein ThiS&subsystem_name=Thiamin_biosynthesis@1>Sulfur carrier protein ThiS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1685@1>fig|376686.40.peg.1685</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=376686.40@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Thiamin_biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1604@1>fig|376686.40.peg.1604</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=376686.40@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)&subsystem_name=Cobalamin_synthesis@1>Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.97@1>fig|376686.40.peg.97</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=376686.40@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62)&subsystem_name=Cobalamin_synthesis@1>Adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.93@1>fig|376686.40.peg.93</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=376686.40@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobalamin synthase (EC 2.7.8.26)&subsystem_name=Cobalamin_synthesis@1>Cobalamin synthase (EC 2.7.8.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.96@1>fig|376686.40.peg.96</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=376686.40@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylcobinamide kinase (EC 2.7.1.156)&subsystem_name=Cobalamin_synthesis@1>Adenosylcobinamide kinase (EC 2.7.1.156)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.93@1>fig|376686.40.peg.93</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=376686.40@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)&subsystem_name=Cobalamin_synthesis@1>Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.94@1>fig|376686.40.peg.94</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA synthetase (EC 6.1.1.17)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Glutamyl-tRNA synthetase (EC 6.1.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1062@1>fig|376686.40.peg.1062</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.998@1>fig|376686.40.peg.998</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen-III synthase (EC 4.2.1.75)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen-III synthase (EC 4.2.1.75)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.580@1>fig|376686.40.peg.580</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Precorrin-2 oxidase (EC 1.3.1.76)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Precorrin-2 oxidase (EC 1.3.1.76)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1587@1>fig|376686.40.peg.1587</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Porphobilinogen synthase (EC 4.2.1.24)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Porphobilinogen synthase (EC 4.2.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1005@1>fig|376686.40.peg.1005</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2669@1>fig|376686.40.peg.2669</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA reductase (EC 1.2.1.70)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Glutamyl-tRNA reductase (EC 1.2.1.70)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.990@1>fig|376686.40.peg.990</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3968@1>fig|376686.40.peg.3968</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Porphobilinogen deaminase (EC 2.5.1.61)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Porphobilinogen deaminase (EC 2.5.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.992@1>fig|376686.40.peg.992</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen-III methyltransferase (EC 2.1.1.107)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen-III methyltransferase (EC 2.1.1.107)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1586@1>fig|376686.40.peg.1586</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4917@1>fig|376686.40.peg.4917</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protoporphyrinogen IX oxidase, novel form, HemJ (EC 1.3.-.-)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Protoporphyrinogen IX oxidase, novel form, HemJ (EC 1.3.-.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.986@1>fig|376686.40.peg.986</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.987@1>fig|376686.40.peg.987</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=376686.40@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen III decarboxylase (EC 4.1.1.37)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen III decarboxylase (EC 4.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.994@1>fig|376686.40.peg.994</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=376686.40@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FMN adenylyltransferase (EC 2.7.7.2)&subsystem_name=riboflavin_to_FAD@1>FMN adenylyltransferase (EC 2.7.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.834@1>fig|376686.40.peg.834</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=376686.40@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)&subsystem_name=riboflavin_to_FAD@1>Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.677@1>fig|376686.40.peg.677</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=376686.40@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin kinase (EC 2.7.1.26)&subsystem_name=riboflavin_to_FAD@1>Riboflavin kinase (EC 2.7.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.834@1>fig|376686.40.peg.834</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=376686.40@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)&subsystem_name=riboflavin_to_FAD@1>3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2220@1>fig|376686.40.peg.2220</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4300@1>fig|376686.40.peg.4300</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=376686.40@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FMN adenylyltransferase (EC 2.7.7.2)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>FMN adenylyltransferase (EC 2.7.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.834@1>fig|376686.40.peg.834</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=376686.40@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.677@1>fig|376686.40.peg.677</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=376686.40@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminohydroxyphosphoribosylaminopyrimidine deaminase (EC 3.5.4.26)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Diaminohydroxyphosphoribosylaminopyrimidine deaminase (EC 3.5.4.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1@1>fig|376686.40.peg.1</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=376686.40@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.225@1>fig|376686.40.peg.225</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=376686.40@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-amino-6-(5-phosphoribosylamino)uracil reductase (EC 1.1.1.193)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>5-amino-6-(5-phosphoribosylamino)uracil reductase (EC 1.1.1.193)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1@1>fig|376686.40.peg.1</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=376686.40@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin kinase (EC 2.7.1.26)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Riboflavin kinase (EC 2.7.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.834@1>fig|376686.40.peg.834</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=376686.40@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase II (EC 3.5.4.25)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>GTP cyclohydrolase II (EC 3.5.4.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2220@1>fig|376686.40.peg.2220</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=376686.40@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2220@1>fig|376686.40.peg.2220</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4300@1>fig|376686.40.peg.4300</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=376686.40@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.876@1>fig|376686.40.peg.876</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2584@1>fig|376686.40.peg.2584</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=376686.40@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.676@1>fig|376686.40.peg.676</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=376686.40@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyridoxamine 5@1-phosphate oxidase (EC 1.4.3.5)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Pyridoxamine 5@1-phosphate oxidase (EC 1.4.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4812@1>fig|376686.40.peg.4812</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=376686.40@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyridoxine 5@1-phosphate synthase (EC 2.6.99.2)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Pyridoxine 5@1-phosphate synthase (EC 2.6.99.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1779@1>fig|376686.40.peg.1779</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=376686.40@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2585@1>fig|376686.40.peg.2585</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=376686.40@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.21@1>fig|376686.40.peg.21</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.242@1>fig|376686.40.peg.242</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4864@1>fig|376686.40.peg.4864</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=376686.40@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1604@1>fig|376686.40.peg.1604</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate synthetase (EC 2.5.1.72)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Quinolinate synthetase (EC 2.5.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4290@1>fig|376686.40.peg.4290</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3678@1>fig|376686.40.peg.3678</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD kinase (EC 2.7.1.23)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>NAD kinase (EC 2.7.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1777@1>fig|376686.40.peg.1777</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinamide phosphoribosyltransferase (EC 2.4.2.12)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Nicotinamide phosphoribosyltransferase (EC 2.4.2.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.777@1>fig|376686.40.peg.777</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxyanthranilate 3,4-dioxygenase (EC 1.13.11.6)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>3-hydroxyanthranilate 3,4-dioxygenase (EC 1.13.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1290@1>fig|376686.40.peg.1290</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD synthetase (EC 6.3.1.5)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>NAD synthetase (EC 6.3.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1879@1>fig|376686.40.peg.1879</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan 2,3-dioxygenase (EC 1.13.11.11)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Tryptophan 2,3-dioxygenase (EC 1.13.11.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.762@1>fig|376686.40.peg.762</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-aspartate oxidase (EC 1.4.3.16)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>L-aspartate oxidase (EC 1.4.3.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4291@1>fig|376686.40.peg.4291</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosyl nicotinamide transporter, PnuC-like&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Ribosyl nicotinamide transporter, PnuC-like</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1666@1>fig|376686.40.peg.1666</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kynurenine 3-monooxygenase (EC 1.14.13.9)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Kynurenine 3-monooxygenase (EC 1.14.13.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.518@1>fig|376686.40.peg.518</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.775@1>fig|376686.40.peg.775</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=376686.40@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kynureninase (EC 3.7.1.3)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Kynureninase (EC 3.7.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.529@1>fig|376686.40.peg.529</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Folate_biosynthesis_cluster@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3111@1>fig|376686.40.peg.3111</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydropteroate synthase (EC 2.5.1.15)&subsystem_name=Folate_biosynthesis_cluster@1>Dihydropteroate synthase (EC 2.5.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3687@1>fig|376686.40.peg.3687</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)&subsystem_name=Folate_biosynthesis_cluster@1>tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2250@1>fig|376686.40.peg.2250</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)&subsystem_name=Folate_biosynthesis_cluster@1>Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.438@1>fig|376686.40.peg.438</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Folate_biosynthesis_cluster@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.862@1>fig|376686.40.peg.862</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Folate_biosynthesis_cluster@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2386@1>fig|376686.40.peg.2386</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Folate_biosynthesis_cluster@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.861@1>fig|376686.40.peg.861</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)&subsystem_name=Folate_biosynthesis_cluster@1>2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1803@1>fig|376686.40.peg.1803</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=376686.40@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroneopterin aldolase (EC 4.1.2.25)&subsystem_name=Folate_biosynthesis_cluster@1>Dihydroneopterin aldolase (EC 4.1.2.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1071@1>fig|376686.40.peg.1071</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Folate_Biosynthesis@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.148@1>fig|376686.40.peg.148</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Folylpolyglutamate synthase (EC 6.3.2.17)&subsystem_name=Folate_Biosynthesis@1>Folylpolyglutamate synthase (EC 6.3.2.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.478@1>fig|376686.40.peg.478</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydropteroate synthase (EC 2.5.1.15)&subsystem_name=Folate_Biosynthesis@1>Dihydropteroate synthase (EC 2.5.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3687@1>fig|376686.40.peg.3687</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=Folate_Biosynthesis@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.755@1>fig|376686.40.peg.755</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate reductase (EC 1.5.1.3)&subsystem_name=Folate_Biosynthesis@1>Dihydrofolate reductase (EC 1.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1546@1>fig|376686.40.peg.1546</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate synthase (EC 6.3.2.12)&subsystem_name=Folate_Biosynthesis@1>Dihydrofolate synthase (EC 6.3.2.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.478@1>fig|376686.40.peg.478</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Folate_Biosynthesis@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2386@1>fig|376686.40.peg.2386</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Folate_Biosynthesis@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2249@1>fig|376686.40.peg.2249</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidylate synthase (EC 2.1.1.45)&subsystem_name=Folate_Biosynthesis@1>Thymidylate synthase (EC 2.1.1.45)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1549@1>fig|376686.40.peg.1549</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)&subsystem_name=Folate_Biosynthesis@1>2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1803@1>fig|376686.40.peg.1803</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroneopterin aldolase (EC 4.1.2.25)&subsystem_name=Folate_Biosynthesis@1>Dihydroneopterin aldolase (EC 4.1.2.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1071@1>fig|376686.40.peg.1071</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=376686.40@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Folate_Biosynthesis@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5190@1>fig|376686.40.peg.5190</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alcohol dehydrogenase (EC 1.1.1.1)&subsystem_name=5-FCL-like_protein@1>Alcohol dehydrogenase (EC 1.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3474@1>fig|376686.40.peg.3474</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4407@1>fig|376686.40.peg.4407</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)&subsystem_name=5-FCL-like_protein@1>Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2296@1>fig|376686.40.peg.2296</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formyltetrahydrofolate deformylase (EC 3.5.1.10)&subsystem_name=5-FCL-like_protein@1>Formyltetrahydrofolate deformylase (EC 3.5.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2445@1>fig|376686.40.peg.2445</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=5-FCL-like_protein@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.755@1>fig|376686.40.peg.755</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate reductase (EC 1.5.1.3)&subsystem_name=5-FCL-like_protein@1>Dihydrofolate reductase (EC 1.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1546@1>fig|376686.40.peg.1546</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamine-monophosphate kinase (EC 2.7.4.16)&subsystem_name=5-FCL-like_protein@1>Thiamine-monophosphate kinase (EC 2.7.4.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.826@1>fig|376686.40.peg.826</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=5-FCL-like_protein@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3596@1>fig|376686.40.peg.3596</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)&subsystem_name=5-FCL-like_protein@1>Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.257@1>fig|376686.40.peg.257</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)&subsystem_name=5-FCL-like_protein@1>Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1197@1>fig|376686.40.peg.1197</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)&subsystem_name=5-FCL-like_protein@1>Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1536@1>fig|376686.40.peg.1536</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)&subsystem_name=5-FCL-like_protein@1>Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1681@1>fig|376686.40.peg.1681</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=5-FCL-like_protein@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1592@1>fig|376686.40.peg.1592</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=5-FCL-like_protein@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1633@1>fig|376686.40.peg.1633</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=376686.40@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=5-FCL-like_protein@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2245@1>fig|376686.40.peg.2245</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4955@1>fig|376686.40.peg.4955</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=376686.40@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkaline phosphatase (EC 3.1.3.1)&subsystem_name=Pterin_synthesis_related_cluster@1>Alkaline phosphatase (EC 3.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.24@1>fig|376686.40.peg.24</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3363@1>fig|376686.40.peg.3363</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3428@1>fig|376686.40.peg.3428</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3600@1>fig|376686.40.peg.3600</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=376686.40@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Pterin_synthesis_related_cluster@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2386@1>fig|376686.40.peg.2386</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=376686.40@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)&subsystem_name=Pterin_synthesis_related_cluster@1>Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.46@1>fig|376686.40.peg.46</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=376686.40@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Pterin_synthesis_related_cluster@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5193@1>fig|376686.40.peg.5193</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=376686.40@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6-pyruvoyl tetrahydrobiopterin synthase (EC 4.2.3.12)&subsystem_name=Pterin_synthesis_related_cluster@1>6-pyruvoyl tetrahydrobiopterin synthase (EC 4.2.3.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.285@1>fig|376686.40.peg.285</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=376686.40@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3111@1>fig|376686.40.peg.3111</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=376686.40@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4990@1>fig|376686.40.peg.4990</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=376686.40@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.861@1>fig|376686.40.peg.861</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=376686.40@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1485@1>fig|376686.40.peg.1485</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=376686.40@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.862@1>fig|376686.40.peg.862</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3111@1>fig|376686.40.peg.3111</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)&subsystem_name=Coenzyme_A_Biosynthesis@1>3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4990@1>fig|376686.40.peg.4990</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dephospho-CoA kinase (EC 2.7.1.24)&subsystem_name=Coenzyme_A_Biosynthesis@1>Dephospho-CoA kinase (EC 2.7.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3651@1>fig|376686.40.peg.3651</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)&subsystem_name=Coenzyme_A_Biosynthesis@1>Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1485@1>fig|376686.40.peg.1485</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Coenzyme_A_Biosynthesis@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.862@1>fig|376686.40.peg.862</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantetheine adenylyltransferase (EC 2.7.7.3)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantetheine adenylyltransferase (EC 2.7.7.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2750@1>fig|376686.40.peg.2750</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantothenoylcysteine synthetase (EC 6.3.2.5)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantothenoylcysteine synthetase (EC 6.3.2.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3656@1>fig|376686.40.peg.3656</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Coenzyme_A_Biosynthesis@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.861@1>fig|376686.40.peg.861</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=376686.40@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantothenoylcysteine decarboxylase (EC 4.1.1.36)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantothenoylcysteine decarboxylase (EC 4.1.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3656@1>fig|376686.40.peg.3656</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=376686.40@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.350@1>fig|376686.40.peg.350</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=376686.40@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.372@1>fig|376686.40.peg.372</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=376686.40@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.351@1>fig|376686.40.peg.351</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=376686.40@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)&subsystem_name=dTDP-rhamnose_synthesis@1>Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.349@1>fig|376686.40.peg.349</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1122@1>fig|376686.40.peg.1122</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Streptococcal_Hyaluronic_Acid_Capsule&organism=376686.40@1>Streptococcal Hyaluronic Acid Capsule</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hyaluronan synthase (EC 2.4.1.212)&subsystem_name=Streptococcal_Hyaluronic_Acid_Capsule@1>Hyaluronan synthase (EC 2.4.1.212)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5134@1>fig|376686.40.peg.5134</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=376686.40@1>CMP-N-acetylneuraminate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylneuraminate synthase (EC 2.5.1.56)&subsystem_name=CMP-N-acetylneuraminate_Biosynthesis@1>N-acetylneuraminate synthase (EC 2.5.1.56)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.323@1>fig|376686.40.peg.323</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=376686.40@1>CMP-N-acetylneuraminate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)&subsystem_name=CMP-N-acetylneuraminate_Biosynthesis@1>UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1097@1>fig|376686.40.peg.1097</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polysaccharide_deacetylases&organism=376686.40@1>Polysaccharide deacetylases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidoglycan N-acetylglucosamine deacetylase (EC 3.5.1.-)&subsystem_name=Polysaccharide_deacetylases@1>Peptidoglycan N-acetylglucosamine deacetylase (EC 3.5.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1146@1>fig|376686.40.peg.1146</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polysaccharide_deacetylases&organism=376686.40@1>Polysaccharide deacetylases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polysaccharide deacetylase&subsystem_name=Polysaccharide_deacetylases@1>Polysaccharide deacetylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1706@1>fig|376686.40.peg.1706</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2647@1>fig|376686.40.peg.2647</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4757@1>fig|376686.40.peg.4757</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Capsular_Polysaccharides_Biosynthesis_and_Assembly&organism=376686.40@1>Capsular Polysaccharides Biosynthesis and Assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-antigen flippase Wzx&subsystem_name=Capsular_Polysaccharides_Biosynthesis_and_Assembly@1>O-antigen flippase Wzx</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.363@1>fig|376686.40.peg.363</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Capsular_Polysaccharides_Biosynthesis_and_Assembly&organism=376686.40@1>Capsular Polysaccharides Biosynthesis and Assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Oligosaccharide repeat unit polymerase Wzy&subsystem_name=Capsular_Polysaccharides_Biosynthesis_and_Assembly@1>Oligosaccharide repeat unit polymerase Wzy</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.309@1>fig|376686.40.peg.309</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=376686.40@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=Rhamnose_containing_glycans@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.355@1>fig|376686.40.peg.355</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1812@1>fig|376686.40.peg.1812</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=376686.40@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.350@1>fig|376686.40.peg.350</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=376686.40@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.372@1>fig|376686.40.peg.372</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=376686.40@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.351@1>fig|376686.40.peg.351</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=376686.40@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)&subsystem_name=Rhamnose_containing_glycans@1>Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.349@1>fig|376686.40.peg.349</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1122@1>fig|376686.40.peg.1122</a>@~Cell Wall and Capsule@^Gram-Negative cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lipoprotein_sorting_system&organism=376686.40@1>Lipoprotein sorting system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane lipoprotein carrier protein LolA&subsystem_name=Lipoprotein_sorting_system@1>Outer membrane lipoprotein carrier protein LolA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2222@1>fig|376686.40.peg.2222</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=376686.40@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)&subsystem_name=Murein_Hydrolases@1>N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4858@1>fig|376686.40.peg.4858</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5244@1>fig|376686.40.peg.5244</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=376686.40@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)&subsystem_name=Murein_Hydrolases@1>D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.522@1>fig|376686.40.peg.522</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2213@1>fig|376686.40.peg.2213</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=376686.40@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Murein_Hydrolases@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1287@1>fig|376686.40.peg.1287</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4723@1>fig|376686.40.peg.4723</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=376686.40@1>Recycling of Peptidoglycan Amino Acids</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)&subsystem_name=Recycling_of_Peptidoglycan_Amino_Acids@1>N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4858@1>fig|376686.40.peg.4858</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5244@1>fig|376686.40.peg.5244</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=376686.40@1>Recycling of Peptidoglycan Amino Acids</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Recycling_of_Peptidoglycan_Amino_Acids@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1287@1>fig|376686.40.peg.1287</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4723@1>fig|376686.40.peg.4723</a>@~Cell Wall and Capsule@^Gram-Positive cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Teichoic_and_lipoteichoic_acids_biosynthesis&organism=376686.40@1>Teichoic and lipoteichoic acids biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative CDP-glycerol:glycerophosphate glycerophosphotransferase (EC 2.7.8.-)&subsystem_name=Teichoic_and_lipoteichoic_acids_biosynthesis@1>Putative CDP-glycerol:glycerophosphate glycerophosphotransferase (EC 2.7.8.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.336@1>fig|376686.40.peg.336</a>@~Cell Wall and Capsule@^Gram-Positive cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Teichoic_and_lipoteichoic_acids_biosynthesis&organism=376686.40@1>Teichoic and lipoteichoic acids biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Poly(glycerol-phosphate) alpha-glucosyltransferase (EC 2.4.1.52)&subsystem_name=Teichoic_and_lipoteichoic_acids_biosynthesis@1>Poly(glycerol-phosphate) alpha-glucosyltransferase (EC 2.4.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2359@1>fig|376686.40.peg.2359</a>@~Cell Wall and Capsule@^Gram-Positive cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Teichoic_and_lipoteichoic_acids_biosynthesis&organism=376686.40@1>Teichoic and lipoteichoic acids biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=CDP-glycerol:poly(glycerophosphate) glycerophosphotransferase (EC 2.7.8.12)&subsystem_name=Teichoic_and_lipoteichoic_acids_biosynthesis@1>CDP-glycerol:poly(glycerophosphate) glycerophosphotransferase (EC 2.7.8.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2354@1>fig|376686.40.peg.2354</a>@~Cell Wall and Capsule@^Gram-Positive cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Teichoic_and_lipoteichoic_acids_biosynthesis&organism=376686.40@1>Teichoic and lipoteichoic acids biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=CDP-glycerol: N-acetyl-beta-D-mannosaminyl-1,4-N-acetyl-D-glucosaminyldiphosphoundecaprenyl glycerophosphotransferase&subsystem_name=Teichoic_and_lipoteichoic_acids_biosynthesis@1>CDP-glycerol: N-acetyl-beta-D-mannosaminyl-1,4-N-acetyl-D-glucosaminyldiphosphoundecaprenyl glycerophosphotransferase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2644@1>fig|376686.40.peg.2644</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=376686.40@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multicopper oxidase&subsystem_name=Copper_homeostasis@1>Multicopper oxidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.203@1>fig|376686.40.peg.203</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3248@1>fig|376686.40.peg.3248</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=376686.40@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-translocating P-type ATPase (EC 3.6.3.4)&subsystem_name=Copper_homeostasis@1>Copper-translocating P-type ATPase (EC 3.6.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.202@1>fig|376686.40.peg.202</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2661@1>fig|376686.40.peg.2661</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3244@1>fig|376686.40.peg.3244</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3526@1>fig|376686.40.peg.3526</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=376686.40@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multidrug resistance transporter, Bcr/CflA family&subsystem_name=Copper_homeostasis@1>Multidrug resistance transporter, Bcr/CflA family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3004@1>fig|376686.40.peg.3004</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=376686.40@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cu(I)-responsive transcriptional regulator&subsystem_name=Copper_homeostasis@1>Cu(I)-responsive transcriptional regulator</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.483@1>fig|376686.40.peg.483</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=376686.40@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-binding heavy metal response regulator&subsystem_name=Cobalt-zinc-cadmium_resistance@1>DNA-binding heavy metal response regulator</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3034@1>fig|376686.40.peg.3034</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=376686.40@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobalt-zinc-cadmium resistance protein&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Cobalt-zinc-cadmium resistance protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.137@1>fig|376686.40.peg.137</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=376686.40@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional regulator, MerR family&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Transcriptional regulator, MerR family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.61@1>fig|376686.40.peg.61</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1431@1>fig|376686.40.peg.1431</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4540@1>fig|376686.40.peg.4540</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=376686.40@1>Resistance to fluoroquinolones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit B (EC 5.99.1.3)&subsystem_name=Resistance_to_fluoroquinolones@1>DNA gyrase subunit B (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2370@1>fig|376686.40.peg.2370</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=376686.40@1>Resistance to fluoroquinolones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit A (EC 5.99.1.3)&subsystem_name=Resistance_to_fluoroquinolones@1>DNA gyrase subunit A (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2714@1>fig|376686.40.peg.2714</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis:_copper_tolerance&organism=376686.40@1>Copper homeostasis: copper tolerance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytoplasmic copper homeostasis protein CutC&subsystem_name=Copper_homeostasis:_copper_tolerance@1>Cytoplasmic copper homeostasis protein CutC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.270@1>fig|376686.40.peg.270</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis:_copper_tolerance&organism=376686.40@1>Copper homeostasis: copper tolerance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper homeostasis protein CutF precursor&subsystem_name=Copper_homeostasis:_copper_tolerance@1>Copper homeostasis protein CutF precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4447@1>fig|376686.40.peg.4447</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Beta-lactamase&organism=376686.40@1>Beta-lactamase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Metal-dependent hydrolases of the beta-lactamase superfamily I&subsystem_name=Beta-lactamase@1>Metal-dependent hydrolases of the beta-lactamase superfamily I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2565@1>fig|376686.40.peg.2565</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_chromium_compounds&organism=376686.40@1>Resistance to chromium compounds</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chromate transport protein ChrA&subsystem_name=Resistance_to_chromium_compounds@1>Chromate transport protein ChrA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3158@1>fig|376686.40.peg.3158</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4676@1>fig|376686.40.peg.4676</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=376686.40@1>Multidrug Resistance Efflux Pumps</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription regulator of multidrug efflux pump operon, TetR (AcrR) family&subsystem_name=Multidrug_Resistance_Efflux_Pumps@1>Transcription regulator of multidrug efflux pump operon, TetR (AcrR) family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4195@1>fig|376686.40.peg.4195</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=376686.40@1>Multidrug Resistance Efflux Pumps</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multi antimicrobial extrusion protein (Na(+)/drug antiporter), MATE family of MDR efflux pumps&subsystem_name=Multidrug_Resistance_Efflux_Pumps@1>Multi antimicrobial extrusion protein (Na(+)/drug antiporter), MATE family of MDR efflux pumps</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3351@1>fig|376686.40.peg.3351</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=376686.40@1>Multidrug Resistance Efflux Pumps</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acriflavin resistance protein&subsystem_name=Multidrug_Resistance_Efflux_Pumps@1>Acriflavin resistance protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4528@1>fig|376686.40.peg.4528</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=376686.40@1>Multidrug Resistance Efflux Pumps</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multidrug and toxin extrusion (MATE) family efflux pump YdhE/NorM, homolog&subsystem_name=Multidrug_Resistance_Efflux_Pumps@1>Multidrug and toxin extrusion (MATE) family efflux pump YdhE/NorM, homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5329@1>fig|376686.40.peg.5329</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=376686.40@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S7p (S5e)&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>SSU ribosomal protein S7p (S5e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.417@1>fig|376686.40.peg.417</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=376686.40@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.416@1>fig|376686.40.peg.416</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=376686.40@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Tu&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>Translation elongation factor Tu</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2032@1>fig|376686.40.peg.2032</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=376686.40@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S12p (S23e)&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>SSU ribosomal protein S12p (S23e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.418@1>fig|376686.40.peg.418</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=376686.40@1>Mycobacterium virulence operon involved in DNA transcription</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)&subsystem_name=Mycobacterium_virulence_operon_involved_in_DNA_transcription@1>DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2040@1>fig|376686.40.peg.2040</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=376686.40@1>Mycobacterium virulence operon involved in DNA transcription</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)&subsystem_name=Mycobacterium_virulence_operon_involved_in_DNA_transcription@1>DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2039@1>fig|376686.40.peg.2039</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=376686.40@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate synthetase (EC 2.5.1.72)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>Quinolinate synthetase (EC 2.5.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4290@1>fig|376686.40.peg.4290</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=376686.40@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3678@1>fig|376686.40.peg.3678</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=376686.40@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-aspartate oxidase (EC 1.4.3.16)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>L-aspartate oxidase (EC 1.4.3.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4291@1>fig|376686.40.peg.4291</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Listeria_surface_proteins:_Internalin-like_proteins&organism=376686.40@1>Listeria surface proteins: Internalin-like proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=internalin, putative&subsystem_name=Listeria_surface_proteins:_Internalin-like_proteins@1>internalin, putative</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.273@1>fig|376686.40.peg.273</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.840@1>fig|376686.40.peg.840</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1033@1>fig|376686.40.peg.1033</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1132@1>fig|376686.40.peg.1132</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1338@1>fig|376686.40.peg.1338</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Listeria_surface_proteins:_Internalin-like_proteins&organism=376686.40@1>Listeria surface proteins: Internalin-like proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Internalin-like protein (LPXTG motif) Lmo0331 homolog&subsystem_name=Listeria_surface_proteins:_Internalin-like_proteins@1>Internalin-like protein (LPXTG motif) Lmo0331 homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.657@1>fig|376686.40.peg.657</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Listeria_surface_proteins:_Internalin-like_proteins&organism=376686.40@1>Listeria surface proteins: Internalin-like proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Internalin A (LPXTG motif)&subsystem_name=Listeria_surface_proteins:_Internalin-like_proteins@1>Internalin A (LPXTG motif)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3525@1>fig|376686.40.peg.3525</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=376686.40@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L35p&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>LSU ribosomal protein L35p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.29@1>fig|376686.40.peg.29</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=376686.40@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 3&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>Translation initiation factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.28@1>fig|376686.40.peg.28</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=376686.40@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L20p&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>LSU ribosomal protein L20p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.30@1>fig|376686.40.peg.30</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium-transporting ATPase A chain (EC 3.6.3.12) (TC 3.A.3.7.1)&subsystem_name=Potassium_homeostasis@1>Potassium-transporting ATPase A chain (EC 3.6.3.12) (TC 3.A.3.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2073@1>fig|376686.40.peg.2073</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium-transporting ATPase C chain (EC 3.6.3.12) (TC 3.A.3.7.1)&subsystem_name=Potassium_homeostasis@1>Potassium-transporting ATPase C chain (EC 3.6.3.12) (TC 3.A.3.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2071@1>fig|376686.40.peg.2071</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium voltage-gated channel subfamily KQT&subsystem_name=Potassium_homeostasis@1>Potassium voltage-gated channel subfamily KQT</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3640@1>fig|376686.40.peg.3640</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=POTASSIUM/PROTON ANTIPORTER ROSB&subsystem_name=Potassium_homeostasis@1>POTASSIUM/PROTON ANTIPORTER ROSB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5320@1>fig|376686.40.peg.5320</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium efflux system KefA protein&subsystem_name=Potassium_homeostasis@1>Potassium efflux system KefA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2152@1>fig|376686.40.peg.2152</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2239@1>fig|376686.40.peg.2239</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3378@1>fig|376686.40.peg.3378</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium-transporting ATPase B chain (EC 3.6.3.12) (TC 3.A.3.7.1)&subsystem_name=Potassium_homeostasis@1>Potassium-transporting ATPase B chain (EC 3.6.3.12) (TC 3.A.3.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2072@1>fig|376686.40.peg.2072</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Large-conductance mechanosensitive channel&subsystem_name=Potassium_homeostasis@1>Large-conductance mechanosensitive channel</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1393@1>fig|376686.40.peg.1393</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)&subsystem_name=Potassium_homeostasis@1>FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.55@1>fig|376686.40.peg.55</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=376686.40@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kup system potassium uptake protein&subsystem_name=Potassium_homeostasis@1>Kup system potassium uptake protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5068@1>fig|376686.40.peg.5068</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutathione-regulated_potassium-efflux_system_and_associated_functions&organism=376686.40@1>Glutathione-regulated potassium-efflux system and associated functions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutathione-regulated potassium-efflux system protein KefB&subsystem_name=Glutathione-regulated_potassium-efflux_system_and_associated_functions@1>Glutathione-regulated potassium-efflux system protein KefB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4672@1>fig|376686.40.peg.4672</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=376686.40@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=PaaD-like protein (DUF59) involved in Fe-S cluster assembly&subsystem_name=Iron-sulfur_cluster_assembly@1>PaaD-like protein (DUF59) involved in Fe-S cluster assembly</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1241@1>fig|376686.40.peg.1241</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=376686.40@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=probable iron binding protein from the HesB_IscA_SufA family&subsystem_name=Iron-sulfur_cluster_assembly@1>probable iron binding protein from the HesB_IscA_SufA family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1251@1>fig|376686.40.peg.1251</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=376686.40@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Iron-sulfur_cluster_assembly@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1398@1>fig|376686.40.peg.1398</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=376686.40@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=Iron-sulfur_cluster_assembly@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2113@1>fig|376686.40.peg.2113</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2636@1>fig|376686.40.peg.2636</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2945@1>fig|376686.40.peg.2945</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3169@1>fig|376686.40.peg.3169</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4222@1>fig|376686.40.peg.4222</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4444@1>fig|376686.40.peg.4444</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4933@1>fig|376686.40.peg.4933</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=376686.40@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly protein SufB&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly protein SufB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1250@1>fig|376686.40.peg.1250</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=376686.40@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly protein SufD&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly protein SufD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1246@1>fig|376686.40.peg.1246</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=376686.40@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly ATPase protein SufC&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly ATPase protein SufC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1248@1>fig|376686.40.peg.1248</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Single-Rhodanese-domain_proteins&organism=376686.40@1>Single-Rhodanese-domain proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rhodanese domain protein, Enterobacterial subgroup, YceA homolog&subsystem_name=Single-Rhodanese-domain_proteins@1>Rhodanese domain protein, Enterobacterial subgroup, YceA homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.947@1>fig|376686.40.peg.947</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphoglycerate_mutase_protein_family&organism=376686.40@1>Phosphoglycerate mutase protein family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)&subsystem_name=Phosphoglycerate_mutase_protein_family@1>Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.97@1>fig|376686.40.peg.97</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DedA_family_of_inner_membrane_proteins&organism=376686.40@1>DedA family of inner membrane proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DedA protein&subsystem_name=DedA_family_of_inner_membrane_proteins@1>DedA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1530@1>fig|376686.40.peg.1530</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Muconate_lactonizing_enzyme_family&organism=376686.40@1>Muconate lactonizing enzyme family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylbenzoate synthase (EC 4.2.1.113)&subsystem_name=Muconate_lactonizing_enzyme_family@1>O-succinylbenzoate synthase (EC 4.2.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2938@1>fig|376686.40.peg.2938</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Bacillus_subtilis_scratch_-_gjo&organism=376686.40@1>Bacillus subtilis scratch - gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ku domain protein&subsystem_name=Bacillus_subtilis_scratch_-_gjo@1>Ku domain protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3485@1>fig|376686.40.peg.3485</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Bacillus_subtilis_scratch_-_gjo&organism=376686.40@1>Bacillus subtilis scratch - gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD&subsystem_name=Bacillus_subtilis_scratch_-_gjo@1>ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3484@1>fig|376686.40.peg.3484</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=376686.40@1>Broadly distributed proteins not in subsystems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=YpfJ protein, zinc metalloprotease superfamily&subsystem_name=Broadly_distributed_proteins_not_in_subsystems@1>YpfJ protein, zinc metalloprotease superfamily</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4995@1>fig|376686.40.peg.4995</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=376686.40@1>Broadly distributed proteins not in subsystems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UPF0225 protein YchJ&subsystem_name=Broadly_distributed_proteins_not_in_subsystems@1>UPF0225 protein YchJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4585@1>fig|376686.40.peg.4585</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraB&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3177@1>fig|376686.40.peg.3177</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraN&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3161@1>fig|376686.40.peg.3161</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3729@1>fig|376686.40.peg.3729</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4600@1>fig|376686.40.peg.4600</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraG&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3167@1>fig|376686.40.peg.3167</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3736@1>fig|376686.40.peg.3736</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4608@1>fig|376686.40.peg.4608</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraA&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3178@1>fig|376686.40.peg.3178</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraM&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraM</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3162@1>fig|376686.40.peg.3162</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3730@1>fig|376686.40.peg.3730</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4601@1>fig|376686.40.peg.4601</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraJ&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3165@1>fig|376686.40.peg.3165</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3732@1>fig|376686.40.peg.3732</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4604@1>fig|376686.40.peg.4604</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraK&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraK</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3164@1>fig|376686.40.peg.3164</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3731@1>fig|376686.40.peg.3731</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4603@1>fig|376686.40.peg.4603</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraI&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraI</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3166@1>fig|376686.40.peg.3166</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3733@1>fig|376686.40.peg.3733</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3734@1>fig|376686.40.peg.3734</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3735@1>fig|376686.40.peg.3735</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4605@1>fig|376686.40.peg.4605</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4606@1>fig|376686.40.peg.4606</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4607@1>fig|376686.40.peg.4607</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraE&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3168@1>fig|376686.40.peg.3168</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3739@1>fig|376686.40.peg.3739</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4610@1>fig|376686.40.peg.4610</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative conjugative transposon mobilization protein BF0132&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Putative conjugative transposon mobilization protein BF0132</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3180@1>fig|376686.40.peg.3180</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3754@1>fig|376686.40.peg.3754</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3859@1>fig|376686.40.peg.3859</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3893@1>fig|376686.40.peg.3893</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3927@1>fig|376686.40.peg.3927</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3978@1>fig|376686.40.peg.3978</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4632@1>fig|376686.40.peg.4632</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=hypothetical protein clusted with conjugative transposons, BF0131&subsystem_name=Conjugative_transposon,_Bacteroidales@1>hypothetical protein clusted with conjugative transposons, BF0131</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3179@1>fig|376686.40.peg.3179</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3753@1>fig|376686.40.peg.3753</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3822@1>fig|376686.40.peg.3822</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3858@1>fig|376686.40.peg.3858</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3892@1>fig|376686.40.peg.3892</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3928@1>fig|376686.40.peg.3928</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative mobilization protein BF0133&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Putative mobilization protein BF0133</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3181@1>fig|376686.40.peg.3181</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3755@1>fig|376686.40.peg.3755</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3977@1>fig|376686.40.peg.3977</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4633@1>fig|376686.40.peg.4633</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4725@1>fig|376686.40.peg.4725</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraD&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3176@1>fig|376686.40.peg.3176</a>@~Phages, Prophages, Transposable elements, Plasmids@^Transposable elements@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Conjugative_transposon,_Bacteroidales&organism=376686.40@1>Conjugative transposon, Bacteroidales</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Conjugative transposon protein TraF&subsystem_name=Conjugative_transposon,_Bacteroidales@1>Conjugative transposon protein TraF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3738@1>fig|376686.40.peg.3738</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4609@1>fig|376686.40.peg.4609</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=376686.40@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Twin-arginine translocation protein TatC&subsystem_name=Twin-arginine_translocation_system@1>Twin-arginine translocation protein TatC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.300@1>fig|376686.40.peg.300</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=376686.40@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Twin-arginine translocation protein TatA&subsystem_name=Twin-arginine_translocation_system@1>Twin-arginine translocation protein TatA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1081@1>fig|376686.40.peg.1081</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1860@1>fig|376686.40.peg.1860</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=376686.40@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribonuclease TatD&subsystem_name=Twin-arginine_translocation_system@1>Deoxyribonuclease TatD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5027@1>fig|376686.40.peg.5027</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Magnesium_transport&organism=376686.40@1>Magnesium transport</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Magnesium and cobalt transport protein CorA&subsystem_name=Magnesium_transport@1>Magnesium and cobalt transport protein CorA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.256@1>fig|376686.40.peg.256</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2787@1>fig|376686.40.peg.2787</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2788@1>fig|376686.40.peg.2788</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3721@1>fig|376686.40.peg.3721</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4004@1>fig|376686.40.peg.4004</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_Transport_System&organism=376686.40@1>Copper Transport System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-translocating P-type ATPase (EC 3.6.3.4)&subsystem_name=Copper_Transport_System@1>Copper-translocating P-type ATPase (EC 3.6.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.202@1>fig|376686.40.peg.202</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2661@1>fig|376686.40.peg.2661</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3244@1>fig|376686.40.peg.3244</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3526@1>fig|376686.40.peg.3526</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_transport_and_blue_copper_proteins&organism=376686.40@1>Copper transport and blue copper proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Azurin&subsystem_name=Copper_transport_and_blue_copper_proteins@1>Azurin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2534@1>fig|376686.40.peg.2534</a>@~Membrane Transport@^Uni- Sym- and Antiporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters&organism=376686.40@1>NhaA, NhaD and Sodium-dependent phosphate transporters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Na+/H+ antiporter NhaA type&subsystem_name=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters@1>Na+/H+ antiporter NhaA type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.878@1>fig|376686.40.peg.878</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4662@1>fig|376686.40.peg.4662</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=376686.40@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-chelator utilization protein&subsystem_name=Ton_and_Tol_transport_systems@1>Iron-chelator utilization protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4262@1>fig|376686.40.peg.4262</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=376686.40@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric siderophore transport system, biopolymer transport protein ExbB&subsystem_name=Ton_and_Tol_transport_systems@1>Ferric siderophore transport system, biopolymer transport protein ExbB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.477@1>fig|376686.40.peg.477</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=376686.40@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=TonB-dependent receptor&subsystem_name=Ton_and_Tol_transport_systems@1>TonB-dependent receptor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.169@1>fig|376686.40.peg.169</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.265@1>fig|376686.40.peg.265</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.856@1>fig|376686.40.peg.856</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.976@1>fig|376686.40.peg.976</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1329@1>fig|376686.40.peg.1329</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1437@1>fig|376686.40.peg.1437</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1442@1>fig|376686.40.peg.1442</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1569@1>fig|376686.40.peg.1569</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1968@1>fig|376686.40.peg.1968</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1982@1>fig|376686.40.peg.1982</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2589@1>fig|376686.40.peg.2589</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2918@1>fig|376686.40.peg.2918</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2920@1>fig|376686.40.peg.2920</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3263@1>fig|376686.40.peg.3263</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3355@1>fig|376686.40.peg.3355</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3388@1>fig|376686.40.peg.3388</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3469@1>fig|376686.40.peg.3469</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3496@1>fig|376686.40.peg.3496</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4079@1>fig|376686.40.peg.4079</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4308@1>fig|376686.40.peg.4308</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4512@1>fig|376686.40.peg.4512</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=376686.40@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane lipoprotein omp16 precursor&subsystem_name=Ton_and_Tol_transport_systems@1>Outer membrane lipoprotein omp16 precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2391@1>fig|376686.40.peg.2391</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3665@1>fig|376686.40.peg.3665</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4205@1>fig|376686.40.peg.4205</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=376686.40@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biopolymer transport protein ExbD/TolR&subsystem_name=Ton_and_Tol_transport_systems@1>Biopolymer transport protein ExbD/TolR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.476@1>fig|376686.40.peg.476</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.514@1>fig|376686.40.peg.514</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.515@1>fig|376686.40.peg.515</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2911@1>fig|376686.40.peg.2911</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2912@1>fig|376686.40.peg.2912</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=376686.40@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxybenzoyl-CoA thioesterase family active site&subsystem_name=Ton_and_Tol_transport_systems@1>4-hydroxybenzoyl-CoA thioesterase family active site</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4@1>fig|376686.40.peg.4</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=376686.40@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Ton_and_Tol_transport_systems@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3634@1>fig|376686.40.peg.3634</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=376686.40@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=MotA/TolQ/ExbB proton channel family protein&subsystem_name=Ton_and_Tol_transport_systems@1>MotA/TolQ/ExbB proton channel family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.516@1>fig|376686.40.peg.516</a>@~Membrane Transport@^Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP)@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Curli_production&organism=376686.40@1>Curli production</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Minor curlin subunit CsgB, nucleation component of curlin monomers&subsystem_name=Curli_production@1>Minor curlin subunit CsgB, nucleation component of curlin monomers</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2467@1>fig|376686.40.peg.2467</a>@~Membrane Transport@^Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP)@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Curli_production&organism=376686.40@1>Curli production</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Curli production assembly/transport component CsgF&subsystem_name=Curli_production@1>Curli production assembly/transport component CsgF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2471@1>fig|376686.40.peg.2471</a>@~Membrane Transport@^Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP)@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Curli_production&organism=376686.40@1>Curli production</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Curli production assembly/transport component CsgG&subsystem_name=Curli_production@1>Curli production assembly/transport component CsgG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2472@1>fig|376686.40.peg.2472</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=376686.40@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Siderophore synthetase large component, acetyltransferase&subsystem_name=Siderophore_assembly_kit@1>Siderophore synthetase large component, acetyltransferase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3348@1>fig|376686.40.peg.3348</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=376686.40@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Siderophore biosynthesis L-2,4-diaminobutyrate decarboxylase&subsystem_name=Siderophore_assembly_kit@1>Siderophore biosynthesis L-2,4-diaminobutyrate decarboxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3347@1>fig|376686.40.peg.3347</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=376686.40@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Siderophore synthetase small component, acetyltransferase&subsystem_name=Siderophore_assembly_kit@1>Siderophore synthetase small component, acetyltransferase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3348@1>fig|376686.40.peg.3348</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=376686.40@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Siderophore synthetase component, ligase&subsystem_name=Siderophore_assembly_kit@1>Siderophore synthetase component, ligase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3350@1>fig|376686.40.peg.3350</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=376686.40@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Siderophore biosynthesis protein, monooxygenase&subsystem_name=Siderophore_assembly_kit@1>Siderophore biosynthesis protein, monooxygenase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3349@1>fig|376686.40.peg.3349</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=376686.40@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ABC-type siderophore export system, fused ATPase and permease components&subsystem_name=Siderophore_assembly_kit@1>ABC-type siderophore export system, fused ATPase and permease components</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2209@1>fig|376686.40.peg.2209</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=376686.40@1>mnm5U34 biosynthesis bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)&subsystem_name=mnm5U34_biosynthesis_bacteria@1>tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5055@1>fig|376686.40.peg.5055</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=376686.40@1>mnm5U34 biosynthesis bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=mnm5U34_biosynthesis_bacteria@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2113@1>fig|376686.40.peg.2113</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2636@1>fig|376686.40.peg.2636</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2945@1>fig|376686.40.peg.2945</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3169@1>fig|376686.40.peg.3169</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4222@1>fig|376686.40.peg.4222</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4444@1>fig|376686.40.peg.4444</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4933@1>fig|376686.40.peg.4933</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=376686.40@1>Ribonuclease H</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease HI (EC 3.1.26.4)&subsystem_name=Ribonuclease_H@1>Ribonuclease HI (EC 3.1.26.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1196@1>fig|376686.40.peg.1196</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=376686.40@1>Ribonuclease H</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease HII (EC 3.1.26.4)&subsystem_name=Ribonuclease_H@1>Ribonuclease HII (EC 3.1.26.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2019@1>fig|376686.40.peg.2019</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_3@1-terminal_phosphate_cyclase&organism=376686.40@1>RNA 3@1-terminal phosphate cyclase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA-2@1,3@1-PO4:RNA-5@1-OH ligase&subsystem_name=RNA_3@1-terminal_phosphate_cyclase@1>RNA-2@1,3@1-PO4:RNA-5@1-OH ligase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3646@1>fig|376686.40.peg.3646</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5031@1>fig|376686.40.peg.5031</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=376686.40@1>RNA processing and degradation, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3@1-to-5@1 exoribonuclease RNase R&subsystem_name=RNA_processing_and_degradation,_bacterial@1>3@1-to-5@1 exoribonuclease RNase R</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1074@1>fig|376686.40.peg.1074</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=376686.40@1>RNA processing and degradation, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease III (EC 3.1.26.3)&subsystem_name=RNA_processing_and_degradation,_bacterial@1>Ribonuclease III (EC 3.1.26.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1200@1>fig|376686.40.peg.1200</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=376686.40@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease Z (EC 3.1.26.11)&subsystem_name=tRNA_processing@1>Ribonuclease Z (EC 3.1.26.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4813@1>fig|376686.40.peg.4813</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=376686.40@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease P protein component (EC 3.1.26.5)&subsystem_name=tRNA_processing@1>Ribonuclease P protein component (EC 3.1.26.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1024@1>fig|376686.40.peg.1024</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=376686.40@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA dimethylallyltransferase (EC 2.5.1.75)&subsystem_name=tRNA_processing@1>tRNA dimethylallyltransferase (EC 2.5.1.75)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3641@1>fig|376686.40.peg.3641</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=376686.40@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)&subsystem_name=tRNA_processing@1>tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2250@1>fig|376686.40.peg.2250</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=376686.40@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (guanosine(18)-2@1-O)-methyltransferase (EC 2.1.1.34)&subsystem_name=RNA_methylation@1>tRNA (guanosine(18)-2@1-O)-methyltransferase (EC 2.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5313@1>fig|376686.40.peg.5313</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=376686.40@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)&subsystem_name=RNA_methylation@1>tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5055@1>fig|376686.40.peg.5055</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=376686.40@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (cytidine(34)-2@1-O)-methyltransferase (EC 2.1.1.207)&subsystem_name=RNA_methylation@1>tRNA (cytidine(34)-2@1-O)-methyltransferase (EC 2.1.1.207)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.85@1>fig|376686.40.peg.85</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=376686.40@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)&subsystem_name=RNA_methylation@1>SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.882@1>fig|376686.40.peg.882</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=376686.40@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)&subsystem_name=RNA_methylation@1>16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4787@1>fig|376686.40.peg.4787</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=376686.40@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma-54 factor RpoN&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma-54 factor RpoN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.608@1>fig|376686.40.peg.608</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=376686.40@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma factor RpoD&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma factor RpoD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1509@1>fig|376686.40.peg.1509</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=376686.40@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma-70 factor&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma-70 factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3104@1>fig|376686.40.peg.3104</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=376686.40@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma factor SigZ&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma factor SigZ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3258@1>fig|376686.40.peg.3258</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4687@1>fig|376686.40.peg.4687</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=376686.40@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.389@1>fig|376686.40.peg.389</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=376686.40@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2040@1>fig|376686.40.peg.2040</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=376686.40@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3657@1>fig|376686.40.peg.3657</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=376686.40@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2039@1>fig|376686.40.peg.2039</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination protein NusB&subsystem_name=Transcription_factors_bacterial@1>Transcription termination protein NusB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2318@1>fig|376686.40.peg.2318</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription accessory protein (S1 RNA-binding domain)&subsystem_name=Transcription_factors_bacterial@1>Transcription accessory protein (S1 RNA-binding domain)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1079@1>fig|376686.40.peg.1079</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription elongation factor GreA&subsystem_name=Transcription_factors_bacterial@1>Transcription elongation factor GreA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1671@1>fig|376686.40.peg.1671</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination factor Rho&subsystem_name=Transcription_factors_bacterial@1>Transcription termination factor Rho</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4941@1>fig|376686.40.peg.4941</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription antitermination protein NusG&subsystem_name=Transcription_factors_bacterial@1>Transcription antitermination protein NusG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2034@1>fig|376686.40.peg.2034</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription-repair coupling factor&subsystem_name=Transcription_factors_bacterial@1>Transcription-repair coupling factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1270@1>fig|376686.40.peg.1270</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Regulator of nucleoside diphosphate kinase&subsystem_name=Transcription_factors_bacterial@1>Regulator of nucleoside diphosphate kinase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2308@1>fig|376686.40.peg.2308</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2415@1>fig|376686.40.peg.2415</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator RfaH&subsystem_name=Transcription_factors_bacterial@1>Transcriptional activator RfaH</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.375@1>fig|376686.40.peg.375</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=376686.40@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination protein NusA&subsystem_name=Transcription_factors_bacterial@1>Transcription termination protein NusA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1711@1>fig|376686.40.peg.1711</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=376686.40@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted transcriptional regulator of sulfate adenylyltransferase, Rrf2 family&subsystem_name=Rrf2_family_transcriptional_regulators@1>Predicted transcriptional regulator of sulfate adenylyltransferase, Rrf2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1983@1>fig|376686.40.peg.1983</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=376686.40@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted transcriptional regulator of sulfate transport, Rrf2 family&subsystem_name=Rrf2_family_transcriptional_regulators@1>Predicted transcriptional regulator of sulfate transport, Rrf2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1579@1>fig|376686.40.peg.1579</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=376686.40@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rrf2 family transcriptional regulator&subsystem_name=Rrf2_family_transcriptional_regulators@1>Rrf2 family transcriptional regulator</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2533@1>fig|376686.40.peg.2533</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=376686.40@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rrf2 family transcriptional regulator, group III&subsystem_name=Rrf2_family_transcriptional_regulators@1>Rrf2 family transcriptional regulator, group III</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5323@1>fig|376686.40.peg.5323</a>@~RNA Metabolism@^RNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Group_II_intron-associated_genes&organism=376686.40@1>Group II intron-associated genes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Retron-type RNA-directed DNA polymerase (EC 2.7.7.49)&subsystem_name=Group_II_intron-associated_genes@1>Retron-type RNA-directed DNA polymerase (EC 2.7.7.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2452@1>fig|376686.40.peg.2452</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3943@1>fig|376686.40.peg.3943</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside diphosphate kinase (EC 2.7.4.6)&subsystem_name=pyrimidine_conversions@1>Nucleoside diphosphate kinase (EC 2.7.4.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5279@1>fig|376686.40.peg.5279</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thioredoxin reductase (EC 1.8.1.9)&subsystem_name=pyrimidine_conversions@1>Thioredoxin reductase (EC 1.8.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.211@1>fig|376686.40.peg.211</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.977@1>fig|376686.40.peg.977</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1588@1>fig|376686.40.peg.1588</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2942@1>fig|376686.40.peg.2942</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytidylate kinase (EC 2.7.4.25)&subsystem_name=pyrimidine_conversions@1>Cytidylate kinase (EC 2.7.4.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2903@1>fig|376686.40.peg.2903</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidine kinase (EC 2.7.1.21)&subsystem_name=pyrimidine_conversions@1>Thymidine kinase (EC 2.7.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.610@1>fig|376686.40.peg.610</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=pyrimidine_conversions@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2989@1>fig|376686.40.peg.2989</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-nucleotidase (EC 3.1.3.5)&subsystem_name=pyrimidine_conversions@1>5@1-nucleotidase (EC 3.1.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3661@1>fig|376686.40.peg.3661</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3662@1>fig|376686.40.peg.3662</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytidine deaminase (EC 3.5.4.5)&subsystem_name=pyrimidine_conversions@1>Cytidine deaminase (EC 3.5.4.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1635@1>fig|376686.40.peg.1635</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidylate synthase (EC 2.1.1.45)&subsystem_name=pyrimidine_conversions@1>Thymidylate synthase (EC 2.1.1.45)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1549@1>fig|376686.40.peg.1549</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uridine kinase (EC 2.7.1.48)&subsystem_name=pyrimidine_conversions@1>Uridine kinase (EC 2.7.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.11@1>fig|376686.40.peg.11</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uracil phosphoribosyltransferase (EC 2.4.2.9)&subsystem_name=pyrimidine_conversions@1>Uracil phosphoribosyltransferase (EC 2.4.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1102@1>fig|376686.40.peg.1102</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=376686.40@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=CTP synthase (EC 6.3.4.2)&subsystem_name=pyrimidine_conversions@1>CTP synthase (EC 6.3.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2504@1>fig|376686.40.peg.2504</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylamine--glycine ligase (EC 6.3.4.13)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylamine--glycine ligase (EC 6.3.4.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1099@1>fig|376686.40.peg.1099</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=IMP cyclohydrolase (EC 3.5.4.10)&subsystem_name=De_Novo_Purine_Biosynthesis@1>IMP cyclohydrolase (EC 3.5.4.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1536@1>fig|376686.40.peg.1536</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Amidophosphoribosyltransferase (EC 2.4.2.14)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Amidophosphoribosyltransferase (EC 2.4.2.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1191@1>fig|376686.40.peg.1191</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.787@1>fig|376686.40.peg.787</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1187@1>fig|376686.40.peg.1187</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazole-succinocarboxamide synthase (EC 6.3.2.6)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylaminoimidazole-succinocarboxamide synthase (EC 6.3.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2875@1>fig|376686.40.peg.2875</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1197@1>fig|376686.40.peg.1197</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate lyase (EC 4.3.2.2)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Adenylosuccinate lyase (EC 4.3.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5319@1>fig|376686.40.peg.5319</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformylglycinamidine cyclo-ligase (EC 6.3.3.1)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylformylglycinamidine cyclo-ligase (EC 6.3.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2474@1>fig|376686.40.peg.2474</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1536@1>fig|376686.40.peg.1536</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5004@1>fig|376686.40.peg.5004</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=376686.40@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformylglycinamidine synthase, glutamine amidotransferase subunit (EC 6.3.5.3)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylformylglycinamidine synthase, glutamine amidotransferase subunit (EC 6.3.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5004@1>fig|376686.40.peg.5004</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_Utilization&organism=376686.40@1>Purine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Periplasmic aromatic aldehyde oxidoreductase, FAD binding subunit YagS&subsystem_name=Purine_Utilization@1>Periplasmic aromatic aldehyde oxidoreductase, FAD binding subunit YagS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4442@1>fig|376686.40.peg.4442</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_Utilization&organism=376686.40@1>Purine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xanthine and CO dehydrogenases maturation factor, XdhC/CoxF family&subsystem_name=Purine_Utilization@1>Xanthine and CO dehydrogenases maturation factor, XdhC/CoxF family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4446@1>fig|376686.40.peg.4446</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=376686.40@1>GMP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)&subsystem_name=GMP_synthase@1>GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2498@1>fig|376686.40.peg.2498</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=376686.40@1>GMP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)&subsystem_name=GMP_synthase@1>GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2498@1>fig|376686.40.peg.2498</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate synthetase (EC 6.3.4.4)&subsystem_name=Purine_conversions@1>Adenylosuccinate synthetase (EC 6.3.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2337@1>fig|376686.40.peg.2337</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dNTP triphosphohydrolase, broad substrate specificity, subgroup 3&subsystem_name=Purine_conversions@1>dNTP triphosphohydrolase, broad substrate specificity, subgroup 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1602@1>fig|376686.40.peg.1602</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)&subsystem_name=Purine_conversions@1>GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2498@1>fig|376686.40.peg.2498</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inosine-5@1-monophosphate dehydrogenase (EC 1.1.1.205)&subsystem_name=Purine_conversions@1>Inosine-5@1-monophosphate dehydrogenase (EC 1.1.1.205)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2333@1>fig|376686.40.peg.2333</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP reductase (EC 1.7.1.7)&subsystem_name=Purine_conversions@1>GMP reductase (EC 1.7.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3594@1>fig|376686.40.peg.3594</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)&subsystem_name=Purine_conversions@1>Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.438@1>fig|376686.40.peg.438</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate lyase (EC 4.3.2.2)&subsystem_name=Purine_conversions@1>Adenylosuccinate lyase (EC 4.3.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5319@1>fig|376686.40.peg.5319</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Purine_conversions@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4073@1>fig|376686.40.peg.4073</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4805@1>fig|376686.40.peg.4805</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenine phosphoribosyltransferase (EC 2.4.2.7)&subsystem_name=Purine_conversions@1>Adenine phosphoribosyltransferase (EC 2.4.2.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3134@1>fig|376686.40.peg.3134</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyadenosine kinase (EC 2.7.1.76)&subsystem_name=Purine_conversions@1>Deoxyadenosine kinase (EC 2.7.1.76)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1561@1>fig|376686.40.peg.1561</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1803@1>fig|376686.40.peg.1803</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylate kinase (EC 2.7.4.3)&subsystem_name=Purine_conversions@1>Adenylate kinase (EC 2.7.4.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.437@1>fig|376686.40.peg.437</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside diphosphate kinase (EC 2.7.4.6)&subsystem_name=Purine_conversions@1>Nucleoside diphosphate kinase (EC 2.7.4.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5279@1>fig|376686.40.peg.5279</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyguanosine kinase (EC 2.7.1.113)&subsystem_name=Purine_conversions@1>Deoxyguanosine kinase (EC 2.7.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1561@1>fig|376686.40.peg.1561</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1803@1>fig|376686.40.peg.1803</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleotide pyrophosphatase (EC 3.6.1.9)&subsystem_name=Purine_conversions@1>Nucleotide pyrophosphatase (EC 3.6.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1934@1>fig|376686.40.peg.1934</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Purine_conversions@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2989@1>fig|376686.40.peg.2989</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-nucleotidase (EC 3.1.3.5)&subsystem_name=Purine_conversions@1>5@1-nucleotidase (EC 3.1.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3661@1>fig|376686.40.peg.3661</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3662@1>fig|376686.40.peg.3662</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)&subsystem_name=Purine_conversions@1>GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2498@1>fig|376686.40.peg.2498</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosine deaminase (EC 3.5.4.4)&subsystem_name=Purine_conversions@1>Adenosine deaminase (EC 3.5.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3683@1>fig|376686.40.peg.3683</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=376686.40@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Guanylate kinase (EC 2.7.4.8)&subsystem_name=Purine_conversions@1>Guanylate kinase (EC 2.7.4.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.773@1>fig|376686.40.peg.773</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=376686.40@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class III (anaerobic), activating protein (EC 1.97.1.4)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class III (anaerobic), activating protein (EC 1.97.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4295@1>fig|376686.40.peg.4295</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=376686.40@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class III (anaerobic), large subunit (EC 1.17.4.2)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class III (anaerobic), large subunit (EC 1.17.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4293@1>fig|376686.40.peg.4293</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=376686.40@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.546@1>fig|376686.40.peg.546</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4303@1>fig|376686.40.peg.4303</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=376686.40@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.545@1>fig|376686.40.peg.545</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4302@1>fig|376686.40.peg.4302</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dNTP_triphosphohydrolase_protein_family&organism=376686.40@1>dNTP triphosphohydrolase protein family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dNTP triphosphohydrolase, broad substrate specificity, subgroup 3&subsystem_name=dNTP_triphosphohydrolase_protein_family@1>dNTP triphosphohydrolase, broad substrate specificity, subgroup 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1602@1>fig|376686.40.peg.1602</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=376686.40@1>Adenosyl nucleosidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)&subsystem_name=Adenosyl_nucleosidases@1>S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3420@1>fig|376686.40.peg.3420</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=376686.40@1>Adenosyl nucleosidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Adenosyl_nucleosidases@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2989@1>fig|376686.40.peg.2989</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=376686.40@1>Adenosyl nucleosidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-methylthioadenosine nucleosidase (EC 3.2.2.16)&subsystem_name=Adenosyl_nucleosidases@1>5@1-methylthioadenosine nucleosidase (EC 3.2.2.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3420@1>fig|376686.40.peg.3420</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nucleoside_triphosphate_pyrophosphohydrolase_MazG&organism=376686.40@1>Nucleoside triphosphate pyrophosphohydrolase MazG</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)&subsystem_name=Nucleoside_triphosphate_pyrophosphohydrolase_MazG@1>Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2659@1>fig|376686.40.peg.2659</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=376686.40@1>Nudix proteins (nucleoside triphosphate hydrolases)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nudix hydrolase family protein YffH&subsystem_name=Nudix_proteins_(nucleoside_triphosphate_hydrolases)@1>Nudix hydrolase family protein YffH</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2336@1>fig|376686.40.peg.2336</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=376686.40@1>Nudix proteins (nucleoside triphosphate hydrolases)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)&subsystem_name=Nudix_proteins_(nucleoside_triphosphate_hydrolases)@1>Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1121@1>fig|376686.40.peg.1121</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=376686.40@1>Housecleaning nucleoside triphosphate pyrophosphatases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)&subsystem_name=Housecleaning_nucleoside_triphosphate_pyrophosphatases@1>Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1121@1>fig|376686.40.peg.1121</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=376686.40@1>Housecleaning nucleoside triphosphate pyrophosphatases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cof protein, HD superfamily hydrolase&subsystem_name=Housecleaning_nucleoside_triphosphate_pyrophosphatases@1>Cof protein, HD superfamily hydrolase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2874@1>fig|376686.40.peg.2874</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=376686.40@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Heat shock protein GrpE&subsystem_name=Protein_chaperones@1>Heat shock protein GrpE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5357@1>fig|376686.40.peg.5357</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=376686.40@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein DnaK&subsystem_name=Protein_chaperones@1>Chaperone protein DnaK</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2766@1>fig|376686.40.peg.2766</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=376686.40@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein DnaJ&subsystem_name=Protein_chaperones@1>Chaperone protein DnaJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5356@1>fig|376686.40.peg.5356</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=376686.40@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DnaJ-class molecular chaperone CbpA&subsystem_name=Protein_chaperones@1>DnaJ-class molecular chaperone CbpA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2495@1>fig|376686.40.peg.2495</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=376686.40@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein HtpG&subsystem_name=Protein_chaperones@1>Chaperone protein HtpG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.277@1>fig|376686.40.peg.277</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2701@1>fig|376686.40.peg.2701</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_disulfide_interchange&organism=376686.40@1>Periplasmic disulfide interchange</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)&subsystem_name=Periplasmic_disulfide_interchange@1>Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2112@1>fig|376686.40.peg.2112</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2251@1>fig|376686.40.peg.2251</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Peptidyl-prolyl_cis-trans_isomerase&organism=376686.40@1>Peptidyl-prolyl cis-trans isomerase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)&subsystem_name=Peptidyl-prolyl_cis-trans_isomerase@1>FKBP-type peptidyl-prolyl cis-trans isomerase FkpA precursor (EC 5.2.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.55@1>fig|376686.40.peg.55</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Peptidyl-prolyl_cis-trans_isomerase&organism=376686.40@1>Peptidyl-prolyl cis-trans isomerase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FKBP-type peptidyl-prolyl cis-trans isomerase FklB (EC 5.2.1.8)&subsystem_name=Peptidyl-prolyl_cis-trans_isomerase@1>FKBP-type peptidyl-prolyl cis-trans isomerase FklB (EC 5.2.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4139@1>fig|376686.40.peg.4139</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Val&organism=376686.40@1>tRNA aminoacylation, Val</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Valyl-tRNA synthetase (EC 6.1.1.9)&subsystem_name=tRNA_aminoacylation,_Val@1>Valyl-tRNA synthetase (EC 6.1.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2394@1>fig|376686.40.peg.2394</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Met&organism=376686.40@1>tRNA aminoacylation, Met</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionyl-tRNA synthetase (EC 6.1.1.10)&subsystem_name=tRNA_aminoacylation,_Met@1>Methionyl-tRNA synthetase (EC 6.1.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.895@1>fig|376686.40.peg.895</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=rRNAs&organism=376686.40@1>rRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5S rRNA&subsystem_name=rRNAs@1>5S rRNA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.1@1>fig|376686.40.rna.1</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.15@1>fig|376686.40.rna.15</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.49@1>fig|376686.40.rna.49</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.55@1>fig|376686.40.rna.55</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.61@1>fig|376686.40.rna.61</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.66@1>fig|376686.40.rna.66</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=rRNAs&organism=376686.40@1>rRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU rRNA&subsystem_name=rRNAs@1>SSU rRNA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.5@1>fig|376686.40.rna.5</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.19@1>fig|376686.40.rna.19</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.45@1>fig|376686.40.rna.45</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.51@1>fig|376686.40.rna.51</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.57@1>fig|376686.40.rna.57</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.70@1>fig|376686.40.rna.70</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=rRNAs&organism=376686.40@1>rRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU rRNA&subsystem_name=rRNAs@1>LSU rRNA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.2@1>fig|376686.40.rna.2</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.16@1>fig|376686.40.rna.16</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.48@1>fig|376686.40.rna.48</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.54@1>fig|376686.40.rna.54</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.60@1>fig|376686.40.rna.60</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.rna.67@1>fig|376686.40.rna.67</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ile&organism=376686.40@1>tRNA aminoacylation, Ile</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isoleucyl-tRNA synthetase (EC 6.1.1.5)&subsystem_name=tRNA_aminoacylation,_Ile@1>Isoleucyl-tRNA synthetase (EC 6.1.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.735@1>fig|376686.40.peg.735</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Arg&organism=376686.40@1>tRNA aminoacylation, Arg</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Arginyl-tRNA synthetase (EC 6.1.1.19)&subsystem_name=tRNA_aminoacylation,_Arg@1>Arginyl-tRNA synthetase (EC 6.1.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.268@1>fig|376686.40.peg.268</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=376686.40@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor SUI1-related protein&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor SUI1-related protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2990@1>fig|376686.40.peg.2990</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=376686.40@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 1&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.393@1>fig|376686.40.peg.393</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=376686.40@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionyl-tRNA formyltransferase (EC 2.1.2.9)&subsystem_name=Translation_initiation_factors_bacterial@1>Methionyl-tRNA formyltransferase (EC 2.1.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.145@1>fig|376686.40.peg.145</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=376686.40@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 3&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.28@1>fig|376686.40.peg.28</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=376686.40@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 2&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1712@1>fig|376686.40.peg.1712</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=376686.40@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosome-binding factor A&subsystem_name=Translation_initiation_factors_bacterial@1>Ribosome-binding factor A</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1127@1>fig|376686.40.peg.1127</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S11p (S14e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S11p (S14e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.391@1>fig|376686.40.peg.391</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S1p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S1p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2910@1>fig|376686.40.peg.2910</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S6p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S6p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3672@1>fig|376686.40.peg.3672</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S15p (S13e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S15p (S13e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1522@1>fig|376686.40.peg.1522</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S14p (S29e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S14p (S29e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.401@1>fig|376686.40.peg.401</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S5p (S2e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S5p (S2e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.397@1>fig|376686.40.peg.397</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S18p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S18p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3671@1>fig|376686.40.peg.3671</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S17p (S11e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S17p (S11e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.405@1>fig|376686.40.peg.405</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S20p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S20p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5238@1>fig|376686.40.peg.5238</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S12p (S23e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S12p (S23e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.418@1>fig|376686.40.peg.418</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S9p (S16e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S9p (S16e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1692@1>fig|376686.40.peg.1692</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S8p (S15Ae)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S8p (S15Ae)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.400@1>fig|376686.40.peg.400</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S4p (S9e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S4p (S9e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.390@1>fig|376686.40.peg.390</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S14p (S29e), zinc-independent&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S14p (S29e), zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.401@1>fig|376686.40.peg.401</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S19p (S15e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S19p (S15e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.410@1>fig|376686.40.peg.410</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S10p (S20e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S10p (S20e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.415@1>fig|376686.40.peg.415</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S16p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S16p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1372@1>fig|376686.40.peg.1372</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S3p (S3e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S3p (S3e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.408@1>fig|376686.40.peg.408</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S18p, zinc-independent&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S18p, zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3671@1>fig|376686.40.peg.3671</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S7p (S5e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S7p (S5e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.417@1>fig|376686.40.peg.417</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S2p (SAe)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S2p (SAe)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1691@1>fig|376686.40.peg.1691</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S21p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S21p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2029@1>fig|376686.40.peg.2029</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=376686.40@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S13p (S18e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S13p (S18e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.392@1>fig|376686.40.peg.392</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Gly&organism=376686.40@1>tRNA aminoacylation, Gly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycyl-tRNA synthetase (EC 6.1.1.14)&subsystem_name=tRNA_aminoacylation,_Gly@1>Glycyl-tRNA synthetase (EC 6.1.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.831@1>fig|376686.40.peg.831</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ala&organism=376686.40@1>tRNA aminoacylation, Ala</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanyl-tRNA synthetase (EC 6.1.1.7)&subsystem_name=tRNA_aminoacylation,_Ala@1>Alanyl-tRNA synthetase (EC 6.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1989@1>fig|376686.40.peg.1989</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Trp&organism=376686.40@1>tRNA aminoacylation, Trp</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophanyl-tRNA synthetase (EC 6.1.1.2)&subsystem_name=tRNA_aminoacylation,_Trp@1>Tryptophanyl-tRNA synthetase (EC 6.1.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1044@1>fig|376686.40.peg.1044</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=376686.40@1>Programmed frameshift</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=programmed frameshift-containing&subsystem_name=Programmed_frameshift@1>programmed frameshift-containing</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2680@1>fig|376686.40.peg.2680</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=376686.40@1>Programmed frameshift</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 2&subsystem_name=Programmed_frameshift@1>Peptide chain release factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2680@1>fig|376686.40.peg.2680</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Cys&organism=376686.40@1>tRNA aminoacylation, Cys</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteinyl-tRNA synthetase (EC 6.1.1.16)&subsystem_name=tRNA_aminoacylation,_Cys@1>Cysteinyl-tRNA synthetase (EC 6.1.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2384@1>fig|376686.40.peg.2384</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosome recycling factor&subsystem_name=Translation_termination_factors_bacterial@1>Ribosome recycling factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.599@1>fig|376686.40.peg.599</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor homolog&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3644@1>fig|376686.40.peg.3644</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidyl-tRNA hydrolase (EC 3.1.1.29)&subsystem_name=Translation_termination_factors_bacterial@1>Peptidyl-tRNA hydrolase (EC 3.1.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1189@1>fig|376686.40.peg.1189</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide deformylase (EC 3.5.1.88)&subsystem_name=Translation_termination_factors_bacterial@1>Peptide deformylase (EC 3.5.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2657@1>fig|376686.40.peg.2657</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 3&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2043@1>fig|376686.40.peg.2043</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tmRNA-binding protein SmpB&subsystem_name=Translation_termination_factors_bacterial@1>tmRNA-binding protein SmpB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3631@1>fig|376686.40.peg.3631</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 1&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2427@1>fig|376686.40.peg.2427</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine aminopeptidase (EC 3.4.11.18)&subsystem_name=Translation_termination_factors_bacterial@1>Methionine aminopeptidase (EC 3.4.11.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.165@1>fig|376686.40.peg.165</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1846@1>fig|376686.40.peg.1846</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=376686.40@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 2&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2680@1>fig|376686.40.peg.2680</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_His&organism=376686.40@1>tRNA aminoacylation, His</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidyl-tRNA synthetase (EC 6.1.1.21)&subsystem_name=tRNA_aminoacylation,_His@1>Histidyl-tRNA synthetase (EC 6.1.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5358@1>fig|376686.40.peg.5358</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=376686.40@1>tRNA aminoacylation, Asp and Asn</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartyl-tRNA synthetase (EC 6.1.1.12)&subsystem_name=tRNA_aminoacylation,_Asp_and_Asn@1>Aspartyl-tRNA synthetase (EC 6.1.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1317@1>fig|376686.40.peg.1317</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=376686.40@1>tRNA aminoacylation, Asp and Asn</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Asparaginyl-tRNA synthetase (EC 6.1.1.22)&subsystem_name=tRNA_aminoacylation,_Asp_and_Asn@1>Asparaginyl-tRNA synthetase (EC 6.1.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.607@1>fig|376686.40.peg.607</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=376686.40@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.416@1>fig|376686.40.peg.416</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=376686.40@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor LepA&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor LepA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.821@1>fig|376686.40.peg.821</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=376686.40@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Tu&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor Tu</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2032@1>fig|376686.40.peg.2032</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=376686.40@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Ts&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor Ts</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1690@1>fig|376686.40.peg.1690</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=376686.40@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor P&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor P</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3056@1>fig|376686.40.peg.3056</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Lys&organism=376686.40@1>tRNA aminoacylation, Lys</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)&subsystem_name=tRNA_aminoacylation,_Lys@1>Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2015@1>fig|376686.40.peg.2015</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Thr&organism=376686.40@1>tRNA aminoacylation, Thr</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonyl-tRNA synthetase (EC 6.1.1.3)&subsystem_name=tRNA_aminoacylation,_Thr@1>Threonyl-tRNA synthetase (EC 6.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.27@1>fig|376686.40.peg.27</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Pro&organism=376686.40@1>tRNA aminoacylation, Pro</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prolyl-tRNA synthetase (EC 6.1.1.15), archaeal/eukaryal type&subsystem_name=tRNA_aminoacylation,_Pro@1>Prolyl-tRNA synthetase (EC 6.1.1.15), archaeal/eukaryal type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5239@1>fig|376686.40.peg.5239</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factor_G_family&organism=376686.40@1>Translation elongation factor G family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Translation_elongation_factor_G_family@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.416@1>fig|376686.40.peg.416</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=376686.40@1>tRNA aminoacylation, Glu and Gln</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA synthetase (EC 6.1.1.17)&subsystem_name=tRNA_aminoacylation,_Glu_and_Gln@1>Glutamyl-tRNA synthetase (EC 6.1.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1062@1>fig|376686.40.peg.1062</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=376686.40@1>tRNA aminoacylation, Glu and Gln</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutaminyl-tRNA synthetase (EC 6.1.1.18)&subsystem_name=tRNA_aminoacylation,_Glu_and_Gln@1>Glutaminyl-tRNA synthetase (EC 6.1.1.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1068@1>fig|376686.40.peg.1068</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ser&organism=376686.40@1>tRNA aminoacylation, Ser</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Seryl-tRNA synthetase (EC 6.1.1.11)&subsystem_name=tRNA_aminoacylation,_Ser@1>Seryl-tRNA synthetase (EC 6.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.885@1>fig|376686.40.peg.885</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Tyr&organism=376686.40@1>tRNA aminoacylation, Tyr</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tyrosyl-tRNA synthetase (EC 6.1.1.1)&subsystem_name=tRNA_aminoacylation,_Tyr@1>Tyrosyl-tRNA synthetase (EC 6.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.574@1>fig|376686.40.peg.574</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3571@1>fig|376686.40.peg.3571</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Leu&organism=376686.40@1>tRNA aminoacylation, Leu</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Leucyl-tRNA synthetase (EC 6.1.1.4)&subsystem_name=tRNA_aminoacylation,_Leu@1>Leucyl-tRNA synthetase (EC 6.1.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.589@1>fig|376686.40.peg.589</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=376686.40@1>tRNA aminoacylation, Phe</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)&subsystem_name=tRNA_aminoacylation,_Phe@1>Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.801@1>fig|376686.40.peg.801</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=376686.40@1>tRNA aminoacylation, Phe</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)&subsystem_name=tRNA_aminoacylation,_Phe@1>Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3128@1>fig|376686.40.peg.3128</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein-L-isoaspartate_O-methyltransferase&organism=376686.40@1>Protein-L-isoaspartate O-methyltransferase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Protein-L-isoaspartate_O-methyltransferase@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3634@1>fig|376686.40.peg.3634</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=376686.40@1>Signal peptidase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Signal peptidase I (EC 3.4.21.89)&subsystem_name=Signal_peptidase@1>Signal peptidase I (EC 3.4.21.89)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.219@1>fig|376686.40.peg.219</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=376686.40@1>Signal peptidase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lipoprotein signal peptidase (EC 3.4.23.36)&subsystem_name=Signal_peptidase@1>Lipoprotein signal peptidase (EC 3.4.23.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.730@1>fig|376686.40.peg.730</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=376686.40@1>Modification of eukaryotic initiation factor 5A</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyhypusine synthase (EC 2.5.1.46)&subsystem_name=Modification_of_eukaryotic_initiation_factor_5A@1>Deoxyhypusine synthase (EC 2.5.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2961@1>fig|376686.40.peg.2961</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=376686.40@1>Modification of eukaryotic initiation factor 5A</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor P&subsystem_name=Modification_of_eukaryotic_initiation_factor_5A@1>Translation elongation factor P</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3056@1>fig|376686.40.peg.3056</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase nickel incorporation-associated protein HypB&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase nickel incorporation-associated protein HypB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4132@1>fig|376686.40.peg.4132</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreF&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease accessory protein UreF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5127@1>fig|376686.40.peg.5127</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypD&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase metallocenter assembly protein HypD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4140@1>fig|376686.40.peg.4140</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypE&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase metallocenter assembly protein HypE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4130@1>fig|376686.40.peg.4130</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease gamma subunit (EC 3.5.1.5)&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease gamma subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5131@1>fig|376686.40.peg.5131</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreE&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease accessory protein UreE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5128@1>fig|376686.40.peg.5128</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease alpha subunit (EC 3.5.1.5)&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease alpha subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5129@1>fig|376686.40.peg.5129</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypF&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase metallocenter assembly protein HypF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4138@1>fig|376686.40.peg.4138</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreD&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease accessory protein UreD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5125@1>fig|376686.40.peg.5125</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase nickel incorporation protein HypA&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase nickel incorporation protein HypA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4133@1>fig|376686.40.peg.4133</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypC&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>[NiFe] hydrogenase metallocenter assembly protein HypC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4131@1>fig|376686.40.peg.4131</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease beta subunit (EC 3.5.1.5)&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease beta subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5130@1>fig|376686.40.peg.5130</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylmalonyl-CoA mutase (EC 5.4.99.2)&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Methylmalonyl-CoA mutase (EC 5.4.99.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2451@1>fig|376686.40.peg.2451</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)&organism=376686.40@1>G3E family of P-loop GTPases (metallocenter biosynthesis)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreG&subsystem_name=G3E_family_of_P-loop_GTPases_(metallocenter_biosynthesis)@1>Urease accessory protein UreG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5126@1>fig|376686.40.peg.5126</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=376686.40@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.355@1>fig|376686.40.peg.355</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1812@1>fig|376686.40.peg.1812</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=376686.40@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-1,4-N-acetylgalactosamine transferase PglH (EC 2.4.1.-)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>Alpha-1,4-N-acetylgalactosamine transferase PglH (EC 2.4.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.310@1>fig|376686.40.peg.310</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=376686.40@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycosyltransferase PglI (EC 2.4.1.-)&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>Glycosyltransferase PglI (EC 2.4.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2357@1>fig|376686.40.peg.2357</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=N-linked_Glycosylation_in_Bacteria&organism=376686.40@1>N-linked Glycosylation in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-keto-6-deoxy-N-Acetyl-D-hexosaminyl-(Lipid carrier) aminotransferase&subsystem_name=N-linked_Glycosylation_in_Bacteria@1>4-keto-6-deoxy-N-Acetyl-D-hexosaminyl-(Lipid carrier) aminotransferase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.352@1>fig|376686.40.peg.352</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=376686.40@1>Aminopeptidases (EC 3.4.11.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xaa-Pro aminopeptidase (EC 3.4.11.9)&subsystem_name=Aminopeptidases_(EC_3.4.11.-)@1>Xaa-Pro aminopeptidase (EC 3.4.11.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.293@1>fig|376686.40.peg.293</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.717@1>fig|376686.40.peg.717</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=376686.40@1>Aminopeptidases (EC 3.4.11.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Membrane alanine aminopeptidase N (EC 3.4.11.2)&subsystem_name=Aminopeptidases_(EC_3.4.11.-)@1>Membrane alanine aminopeptidase N (EC 3.4.11.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1422@1>fig|376686.40.peg.1422</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=376686.40@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)&subsystem_name=Protein_degradation@1>Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1676@1>fig|376686.40.peg.1676</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=376686.40@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)&subsystem_name=Protein_degradation@1>Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.446@1>fig|376686.40.peg.446</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1689@1>fig|376686.40.peg.1689</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=376686.40@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminopeptidase C (EC 3.4.22.40)&subsystem_name=Protein_degradation@1>Aminopeptidase C (EC 3.4.22.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1213@1>fig|376686.40.peg.1213</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=376686.40@1>Metallocarboxypeptidases (EC 3.4.17.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)&subsystem_name=Metallocarboxypeptidases_(EC_3.4.17.-)@1>D-alanyl-D-alanine carboxypeptidase (EC 3.4.16.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.522@1>fig|376686.40.peg.522</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2213@1>fig|376686.40.peg.2213</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=376686.40@1>Metallocarboxypeptidases (EC 3.4.17.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Metallocarboxypeptidases_(EC_3.4.17.-)@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1287@1>fig|376686.40.peg.1287</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4723@1>fig|376686.40.peg.4723</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dipeptidases_(EC_3.4.13.-)&organism=376686.40@1>Dipeptidases (EC 3.4.13.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)&subsystem_name=Dipeptidases_(EC_3.4.13.-)@1>Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1854@1>fig|376686.40.peg.1854</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_endopeptidase_(EC_3.4.21.-)&organism=376686.40@1>Serine endopeptidase (EC 3.4.21.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prolyl endopeptidase (EC 3.4.21.26)&subsystem_name=Serine_endopeptidase_(EC_3.4.21.-)@1>Prolyl endopeptidase (EC 3.4.21.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1387@1>fig|376686.40.peg.1387</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2988@1>fig|376686.40.peg.2988</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=376686.40@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RadA&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>DNA repair protein RadA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.864@1>fig|376686.40.peg.864</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=376686.40@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent Clp protease ATP-binding subunit ClpX&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent Clp protease ATP-binding subunit ClpX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1787@1>fig|376686.40.peg.1787</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=376686.40@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent Clp protease, ATP-binding subunit ClpC&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent Clp protease, ATP-binding subunit ClpC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2713@1>fig|376686.40.peg.2713</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=376686.40@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent protease La (EC 3.4.21.53) Type I&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent protease La (EC 3.4.21.53) Type I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2901@1>fig|376686.40.peg.2901</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Salmonella_enterica_Typhimurium&organism=376686.40@1>LysR-family proteins in Salmonella enterica Typhimurium</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator MetR&subsystem_name=LysR-family_proteins_in_Salmonella_enterica_Typhimurium@1>Transcriptional activator MetR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.912@1>fig|376686.40.peg.912</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=LysR-family_proteins_in_Escherichia_coli&organism=376686.40@1>LysR-family proteins in Escherichia coli</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator MetR&subsystem_name=LysR-family_proteins_in_Escherichia_coli@1>Transcriptional activator MetR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.912@1>fig|376686.40.peg.912</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=376686.40@1>Stringent Response, (p)ppGpp metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II&subsystem_name=Stringent_Response,_(p)ppGpp_metabolism@1>GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2339@1>fig|376686.40.peg.2339</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=376686.40@1>Stringent Response, (p)ppGpp metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Guanosine-3@1,5@1-bis(diphosphate) 3@1-pyrophosphohydrolase (EC 3.1.7.2)&subsystem_name=Stringent_Response,_(p)ppGpp_metabolism@1>Guanosine-3@1,5@1-bis(diphosphate) 3@1-pyrophosphohydrolase (EC 3.1.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2339@1>fig|376686.40.peg.2339</a>@~Regulation and Cell signaling@^Programmed Cell Death and Toxin-antitoxin Systems@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Toxin-antitoxin_replicon_stabilization_systems&organism=376686.40@1>Toxin-antitoxin replicon stabilization systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ParD protein (antitoxin to ParE)&subsystem_name=Toxin-antitoxin_replicon_stabilization_systems@1>ParD protein (antitoxin to ParE)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4796@1>fig|376686.40.peg.4796</a>@~Regulation and Cell signaling@^Programmed Cell Death and Toxin-antitoxin Systems@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Toxin-antitoxin_replicon_stabilization_systems&organism=376686.40@1>Toxin-antitoxin replicon stabilization systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ParE toxin protein&subsystem_name=Toxin-antitoxin_replicon_stabilization_systems@1>ParE toxin protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4795@1>fig|376686.40.peg.4795</a>@~Regulation and Cell signaling@^Programmed Cell Death and Toxin-antitoxin Systems@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phd-Doc,_YdcE-YdcD_toxin-antitoxin_(programmed_cell_death)_systems&organism=376686.40@1>Phd-Doc, YdcE-YdcD toxin-antitoxin (programmed cell death) systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Death on curing protein, Doc toxin&subsystem_name=Phd-Doc,_YdcE-YdcD_toxin-antitoxin_(programmed_cell_death)_systems@1>Death on curing protein, Doc toxin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2287@1>fig|376686.40.peg.2287</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2612@1>fig|376686.40.peg.2612</a>@~Secondary Metabolism@^Plant Alkaloids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alkaloid_biosynthesis_from_L-lysine&organism=376686.40@1>Alkaloid biosynthesis from L-lysine</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyhypusine synthase (EC 2.5.1.46)&subsystem_name=Alkaloid_biosynthesis_from_L-lysine@1>Deoxyhypusine synthase (EC 2.5.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2961@1>fig|376686.40.peg.2961</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=376686.40@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Auxin_biosynthesis@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5197@1>fig|376686.40.peg.5197</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=376686.40@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)&subsystem_name=Auxin_biosynthesis@1>Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1595@1>fig|376686.40.peg.1595</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=376686.40@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Auxin_biosynthesis@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5191@1>fig|376686.40.peg.5191</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=376686.40@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Auxin_biosynthesis@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5194@1>fig|376686.40.peg.5194</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=376686.40@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Auxin_biosynthesis@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5193@1>fig|376686.40.peg.5193</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=376686.40@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination inhibitory protein MutS2&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>Recombination inhibitory protein MutS2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3068@1>fig|376686.40.peg.3068</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=376686.40@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=MutS-related protein, family 1&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>MutS-related protein, family 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2854@1>fig|376686.40.peg.2854</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=376686.40@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutS&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>DNA mismatch repair protein MutS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1807@1>fig|376686.40.peg.1807</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=376686.40@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutL&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>DNA mismatch repair protein MutL</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.224@1>fig|376686.40.peg.224</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=376686.40@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit A paralog in greater Bacteroides group&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit A paralog in greater Bacteroides group</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2551@1>fig|376686.40.peg.2551</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=376686.40@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit C&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit C</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.757@1>fig|376686.40.peg.757</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=376686.40@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit A&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit A</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4778@1>fig|376686.40.peg.4778</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=376686.40@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit B&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit B</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2284@1>fig|376686.40.peg.2284</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_photolyase&organism=376686.40@1>DNA repair, bacterial photolyase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribodipyrimidine photolyase (EC 4.1.99.3)&subsystem_name=DNA_repair,_bacterial_photolyase@1>Deoxyribodipyrimidine photolyase (EC 4.1.99.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.67@1>fig|376686.40.peg.67</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=2-phosphoglycolate_salvage&organism=376686.40@1>2-phosphoglycolate salvage</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoglycolate phosphatase (EC 3.1.3.18)&subsystem_name=2-phosphoglycolate_salvage@1>Phosphoglycolate phosphatase (EC 3.1.3.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1468@1>fig|376686.40.peg.1468</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RecA_and_RecX&organism=376686.40@1>RecA and RecX</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=RecA_and_RecX@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1042@1>fig|376686.40.peg.1042</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RecA_and_RecX&organism=376686.40@1>RecA and RecX</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Regulatory protein RecX&subsystem_name=RecA_and_RecX@1>Regulatory protein RecX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.842@1>fig|376686.40.peg.842</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=376686.40@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutS&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>DNA mismatch repair protein MutS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1807@1>fig|376686.40.peg.1807</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=376686.40@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1042@1>fig|376686.40.peg.1042</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=376686.40@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Regulatory protein RecX&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>Regulatory protein RecX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.842@1>fig|376686.40.peg.842</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease VII large subunit (EC 3.1.11.6)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease VII large subunit (EC 3.1.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3993@1>fig|376686.40.peg.3993</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RadA&subsystem_name=DNA_repair,_bacterial@1>DNA repair protein RadA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.864@1>fig|376686.40.peg.864</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ADA regulatory protein&subsystem_name=DNA_repair,_bacterial@1>ADA regulatory protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1656@1>fig|376686.40.peg.1656</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RecN&subsystem_name=DNA_repair,_bacterial@1>DNA repair protein RecN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3654@1>fig|376686.40.peg.3654</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination protein RmuC&subsystem_name=DNA_repair,_bacterial@1>DNA recombination protein RmuC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.74@1>fig|376686.40.peg.74</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=A/G-specific adenine glycosylase (EC 3.2.2.-)&subsystem_name=DNA_repair,_bacterial@1>A/G-specific adenine glycosylase (EC 3.2.2.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1615@1>fig|376686.40.peg.1615</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-cytosine methyltransferase (EC 2.1.1.37)&subsystem_name=DNA_repair,_bacterial@1>DNA-cytosine methyltransferase (EC 2.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.680@1>fig|376686.40.peg.680</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA polymerase IV (EC 2.7.7.7)&subsystem_name=DNA_repair,_bacterial@1>DNA polymerase IV (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.924@1>fig|376686.40.peg.924</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2790@1>fig|376686.40.peg.2790</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3759@1>fig|376686.40.peg.3759</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3986@1>fig|376686.40.peg.3986</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Very-short-patch mismatch repair endonuclease (G-T specific)&subsystem_name=DNA_repair,_bacterial@1>Very-short-patch mismatch repair endonuclease (G-T specific)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.682@1>fig|376686.40.peg.682</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Single-stranded DNA-binding protein&subsystem_name=DNA_repair,_bacterial@1>Single-stranded DNA-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.907@1>fig|376686.40.peg.907</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1619@1>fig|376686.40.peg.1619</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3142@1>fig|376686.40.peg.3142</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3228@1>fig|376686.40.peg.3228</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4635@1>fig|376686.40.peg.4635</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)&subsystem_name=DNA_repair,_bacterial@1>Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.915@1>fig|376686.40.peg.915</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1656@1>fig|376686.40.peg.1656</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair,_bacterial@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1042@1>fig|376686.40.peg.1042</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkylated DNA repair protein AlkB&subsystem_name=DNA_repair,_bacterial@1>Alkylated DNA repair protein AlkB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3988@1>fig|376686.40.peg.3988</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease III (EC 3.1.11.2)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease III (EC 3.1.11.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.869@1>fig|376686.40.peg.869</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2062@1>fig|376686.40.peg.2062</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=376686.40@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease VII small subunit (EC 3.1.11.6)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease VII small subunit (EC 3.1.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3992@1>fig|376686.40.peg.3992</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=376686.40@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA helicase RecQ&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>ATP-dependent DNA helicase RecQ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.298@1>fig|376686.40.peg.298</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2959@1>fig|376686.40.peg.2959</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=376686.40@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Single-stranded DNA-binding protein&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>Single-stranded DNA-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.907@1>fig|376686.40.peg.907</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1619@1>fig|376686.40.peg.1619</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3142@1>fig|376686.40.peg.3142</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3228@1>fig|376686.40.peg.3228</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4635@1>fig|376686.40.peg.4635</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=376686.40@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination protein RecR&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>Recombination protein RecR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.380@1>fig|376686.40.peg.380</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=376686.40@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1042@1>fig|376686.40.peg.1042</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=376686.40@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination and repair protein RecO&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>DNA recombination and repair protein RecO</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.737@1>fig|376686.40.peg.737</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=376686.40@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination and repair protein RecF&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>DNA recombination and repair protein RecF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.228@1>fig|376686.40.peg.228</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=376686.40@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA helicase RecS (RecQ family)&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>ATP-dependent DNA helicase RecS (RecQ family)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.144@1>fig|376686.40.peg.144</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nonhomologous_End-Joining_in_Bacteria&organism=376686.40@1>Nonhomologous End-Joining in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) LigC&subsystem_name=Nonhomologous_End-Joining_in_Bacteria@1>ATP-dependent DNA ligase (EC 6.5.1.1) LigC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1839@1>fig|376686.40.peg.1839</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nonhomologous_End-Joining_in_Bacteria&organism=376686.40@1>Nonhomologous End-Joining in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ku domain protein&subsystem_name=Nonhomologous_End-Joining_in_Bacteria@1>Ku domain protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3485@1>fig|376686.40.peg.3485</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nonhomologous_End-Joining_in_Bacteria&organism=376686.40@1>Nonhomologous End-Joining in Bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD&subsystem_name=Nonhomologous_End-Joining_in_Bacteria@1>ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3484@1>fig|376686.40.peg.3484</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=376686.40@1>Type I Restriction-Modification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, specificity subunit S (EC 3.1.21.3)&subsystem_name=Type_I_Restriction-Modification@1>Type I restriction-modification system, specificity subunit S (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2844@1>fig|376686.40.peg.2844</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=376686.40@1>Type I Restriction-Modification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)&subsystem_name=Type_I_Restriction-Modification@1>Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2841@1>fig|376686.40.peg.2841</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4650@1>fig|376686.40.peg.4650</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=376686.40@1>Type I Restriction-Modification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)&subsystem_name=Type_I_Restriction-Modification@1>Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4647@1>fig|376686.40.peg.4647</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=376686.40@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, specificity subunit S (EC 3.1.21.3)&subsystem_name=Restriction-Modification_System@1>Type I restriction-modification system, specificity subunit S (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2844@1>fig|376686.40.peg.2844</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=376686.40@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)&subsystem_name=Restriction-Modification_System@1>Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2841@1>fig|376686.40.peg.2841</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4650@1>fig|376686.40.peg.4650</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=376686.40@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)&subsystem_name=Restriction-Modification_System@1>Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4647@1>fig|376686.40.peg.4647</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=376686.40@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative DNA-binding protein in cluster with Type I restriction-modification system&subsystem_name=Restriction-Modification_System@1>Putative DNA-binding protein in cluster with Type I restriction-modification system</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1151@1>fig|376686.40.peg.1151</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_ligases&organism=376686.40@1>DNA ligases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) LigC&subsystem_name=DNA_ligases@1>ATP-dependent DNA ligase (EC 6.5.1.1) LigC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1839@1>fig|376686.40.peg.1839</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_ligases&organism=376686.40@1>DNA ligases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ku domain protein&subsystem_name=DNA_ligases@1>Ku domain protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3485@1>fig|376686.40.peg.3485</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_ligases&organism=376686.40@1>DNA ligases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD&subsystem_name=DNA_ligases@1>ATP-dependent DNA ligase (EC 6.5.1.1) clustered with Ku protein, LigD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3484@1>fig|376686.40.peg.3484</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=YcfH&organism=376686.40@1>YcfH</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribonuclease TatD&subsystem_name=YcfH@1>Deoxyribonuclease TatD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5027@1>fig|376686.40.peg.5027</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=YcfH&organism=376686.40@1>YcfH</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative deoxyribonuclease YjjV&subsystem_name=YcfH@1>Putative deoxyribonuclease YjjV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2924@1>fig|376686.40.peg.2924</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=376686.40@1>DNA structural proteins, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-binding protein HU-beta&subsystem_name=DNA_structural_proteins,_bacterial@1>DNA-binding protein HU-beta</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.146@1>fig|376686.40.peg.146</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=376686.40@1>DNA structural proteins, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Integration host factor alpha/beta&subsystem_name=DNA_structural_proteins,_bacterial@1>Integration host factor alpha/beta</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1614@1>fig|376686.40.peg.1614</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=376686.40@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypothetical protein PA2244 (similar to DNA topoisomerase IB, but possibly involved in glycosyl-transfer)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>Hypothetical protein PA2244 (similar to DNA topoisomerase IB, but possibly involved in glycosyl-transfer)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3551@1>fig|376686.40.peg.3551</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=376686.40@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase III (EC 5.99.1.2)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>DNA topoisomerase III (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2405@1>fig|376686.40.peg.2405</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=376686.40@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase I (EC 5.99.1.2)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>DNA topoisomerase I (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1944@1>fig|376686.40.peg.1944</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=376686.40@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase III, Bacteroidales-type (EC 5.99.1.2)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>DNA topoisomerase III, Bacteroidales-type (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3183@1>fig|376686.40.peg.3183</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_replication_strays&organism=376686.40@1>DNA replication strays</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DEDDh 3@1-5@1 exonuclease domain of the epsilon subunit of DNA polymerase III&subsystem_name=DNA_replication_strays@1>DEDDh 3@1-5@1 exonuclease domain of the epsilon subunit of DNA polymerase III</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2604@1>fig|376686.40.peg.2604</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_replication_strays&organism=376686.40@1>DNA replication strays</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Error-prone repair homolog of DNA polymerase III alpha subunit (EC 2.7.7.7)&subsystem_name=DNA_replication_strays@1>Error-prone repair homolog of DNA polymerase III alpha subunit (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3222@1>fig|376686.40.peg.3222</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3760@1>fig|376686.40.peg.3760</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3987@1>fig|376686.40.peg.3987</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=376686.40@1>DNA topoisomerases, Type II, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit B (EC 5.99.1.3)&subsystem_name=DNA_topoisomerases,_Type_II,_ATP-dependent@1>DNA gyrase subunit B (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2370@1>fig|376686.40.peg.2370</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=376686.40@1>DNA topoisomerases, Type II, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit A (EC 5.99.1.3)&subsystem_name=DNA_topoisomerases,_Type_II,_ATP-dependent@1>DNA gyrase subunit A (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2714@1>fig|376686.40.peg.2714</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=376686.40@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination protein RecR&subsystem_name=DNA_processing_cluster@1>Recombination protein RecR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.380@1>fig|376686.40.peg.380</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=376686.40@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase III (EC 5.99.1.2)&subsystem_name=DNA_processing_cluster@1>DNA topoisomerase III (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2405@1>fig|376686.40.peg.2405</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=376686.40@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA polymerase III subunits gamma and tau (EC 2.7.7.7)&subsystem_name=DNA_processing_cluster@1>DNA polymerase III subunits gamma and tau (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4800@1>fig|376686.40.peg.4800</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4801@1>fig|376686.40.peg.4801</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin carboxyl carrier protein of acetyl-CoA carboxylase&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Biotin carboxyl carrier protein of acetyl-CoA carboxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.672@1>fig|376686.40.peg.672</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.671@1>fig|376686.40.peg.671</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1539@1>fig|376686.40.peg.1539</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4@1-phosphopantetheinyl transferase (EC 2.7.8.-)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>4@1-phosphopantetheinyl transferase (EC 2.7.8.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2194@1>fig|376686.40.peg.2194</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acyl carrier protein&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acyl carrier protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1149@1>fig|376686.40.peg.1149</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1165@1>fig|376686.40.peg.1165</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1198@1>fig|376686.40.peg.1198</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4031@1>fig|376686.40.peg.4031</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.46@1>fig|376686.40.peg.46</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1166@1>fig|376686.40.peg.1166</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4030@1>fig|376686.40.peg.4030</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acyl carrier protein phosphodiesterase (EC 3.1.4.14)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acyl carrier protein phosphodiesterase (EC 3.1.4.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4774@1>fig|376686.40.peg.4774</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1814@1>fig|376686.40.peg.1814</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=376686.40@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1141@1>fig|376686.40.peg.1141</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3058@1>fig|376686.40.peg.3058</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4029@1>fig|376686.40.peg.4029</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4968@1>fig|376686.40.peg.4968</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=376686.40@1>Fatty acid metabolism cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)&subsystem_name=Fatty_acid_metabolism_cluster@1>Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3353@1>fig|376686.40.peg.3353</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5009@1>fig|376686.40.peg.5009</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=376686.40@1>Fatty acid metabolism cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Fatty_acid_metabolism_cluster@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.984@1>fig|376686.40.peg.984</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_acid_metabolism_cluster&organism=376686.40@1>Fatty acid metabolism cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-ketoacyl-CoA thiolase (EC 2.3.1.16)&subsystem_name=Fatty_acid_metabolism_cluster@1>3-ketoacyl-CoA thiolase (EC 2.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3502@1>fig|376686.40.peg.3502</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Myxoxanthophyll_biosynthesis_in_Cyanobacteria&organism=376686.40@1>Myxoxanthophyll biosynthesis in Cyanobacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-carotene hydroxylase&subsystem_name=Myxoxanthophyll_biosynthesis_in_Cyanobacteria@1>Beta-carotene hydroxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.65@1>fig|376686.40.peg.65</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=376686.40@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diphosphomevalonate decarboxylase (EC 4.1.1.33)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Diphosphomevalonate decarboxylase (EC 4.1.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1463@1>fig|376686.40.peg.1463</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=376686.40@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomevalonate kinase (EC 2.7.4.2)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Phosphomevalonate kinase (EC 2.7.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1461@1>fig|376686.40.peg.1461</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=376686.40@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mevalonate kinase (EC 2.7.1.36)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Mevalonate kinase (EC 2.7.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1492@1>fig|376686.40.peg.1492</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=376686.40@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.706@1>fig|376686.40.peg.706</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=376686.40@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1493@1>fig|376686.40.peg.1493</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=376686.40@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4887@1>fig|376686.40.peg.4887</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=376686.40@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Undecaprenyl diphosphate synthase (EC 2.5.1.31)&subsystem_name=Isoprenoinds_for_Quinones@1>Undecaprenyl diphosphate synthase (EC 2.5.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1775@1>fig|376686.40.peg.1775</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=376686.40@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Geranylgeranyl diphosphate synthase (EC 2.5.1.29)&subsystem_name=Isoprenoinds_for_Quinones@1>Geranylgeranyl diphosphate synthase (EC 2.5.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1332@1>fig|376686.40.peg.1332</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=376686.40@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hexaprenyl diphosphate synthase (farnesyl-diphosphate specific) (EC 2.5.1.83)&subsystem_name=Isoprenoinds_for_Quinones@1>Hexaprenyl diphosphate synthase (farnesyl-diphosphate specific) (EC 2.5.1.83)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1871@1>fig|376686.40.peg.1871</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=376686.40@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mevalonate kinase (EC 2.7.1.36)&subsystem_name=Isoprenoid_Biosynthesis@1>Mevalonate kinase (EC 2.7.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1492@1>fig|376686.40.peg.1492</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=376686.40@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Isoprenoid_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4887@1>fig|376686.40.peg.4887</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=376686.40@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Isoprenoid_Biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1604@1>fig|376686.40.peg.1604</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=376686.40@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diphosphomevalonate decarboxylase (EC 4.1.1.33)&subsystem_name=Isoprenoid_Biosynthesis@1>Diphosphomevalonate decarboxylase (EC 4.1.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1463@1>fig|376686.40.peg.1463</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=376686.40@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Undecaprenyl diphosphate synthase (EC 2.5.1.31)&subsystem_name=Isoprenoid_Biosynthesis@1>Undecaprenyl diphosphate synthase (EC 2.5.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1775@1>fig|376686.40.peg.1775</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=376686.40@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomevalonate kinase (EC 2.7.4.2)&subsystem_name=Isoprenoid_Biosynthesis@1>Phosphomevalonate kinase (EC 2.7.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1461@1>fig|376686.40.peg.1461</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=376686.40@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)&subsystem_name=Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.706@1>fig|376686.40.peg.706</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoid_Biosynthesis&organism=376686.40@1>Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)&subsystem_name=Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1493@1>fig|376686.40.peg.1493</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nitrosative_stress&organism=376686.40@1>Nitrosative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric-oxide reductase (EC 1.7.99.7), quinol-dependent&subsystem_name=Nitrosative_stress@1>Nitric-oxide reductase (EC 1.7.99.7), quinol-dependent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2535@1>fig|376686.40.peg.2535</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=376686.40@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Ammonia_assimilation@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2465@1>fig|376686.40.peg.2465</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=376686.40@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] small chain (EC 1.4.1.13)&subsystem_name=Ammonia_assimilation@1>Glutamate synthase [NADPH] small chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2274@1>fig|376686.40.peg.2274</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=376686.40@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)&subsystem_name=Ammonia_assimilation@1>Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2531@1>fig|376686.40.peg.2531</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=376686.40@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ammonium transporter&subsystem_name=Ammonia_assimilation@1>Ammonium transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1411@1>fig|376686.40.peg.1411</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2252@1>fig|376686.40.peg.2252</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=376686.40@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)&subsystem_name=Ammonia_assimilation@1>Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2463@1>fig|376686.40.peg.2463</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ammonia_assimilation&organism=376686.40@1>Ammonia assimilation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] large chain (EC 1.4.1.13)&subsystem_name=Ammonia_assimilation@1>Glutamate synthase [NADPH] large chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2275@1>fig|376686.40.peg.2275</a>@~Dormancy and Sporulation@^Dormancy and Sporulation - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Sporulation-associated_proteins_with_broader_functions&organism=376686.40@1>Sporulation-associated proteins with broader functions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidyl-tRNA hydrolase (EC 3.1.1.29)&subsystem_name=Sporulation-associated_proteins_with_broader_functions@1>Peptidyl-tRNA hydrolase (EC 3.1.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1189@1>fig|376686.40.peg.1189</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit c (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit c (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1114@1>fig|376686.40.peg.1114</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase delta chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase delta chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1116@1>fig|376686.40.peg.1116</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase protein I&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase protein I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1111@1>fig|376686.40.peg.1111</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1112@1>fig|376686.40.peg.1112</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase beta chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase beta chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.854@1>fig|376686.40.peg.854</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase gamma chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase gamma chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1118@1>fig|376686.40.peg.1118</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit b (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit b (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1115@1>fig|376686.40.peg.1115</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit a (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit a (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1113@1>fig|376686.40.peg.1113</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase alpha chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase alpha chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1117@1>fig|376686.40.peg.1117</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=376686.40@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase epsilon chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase epsilon chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.853@1>fig|376686.40.peg.853</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_d_ubiquinol_oxidases&organism=376686.40@1>Terminal cytochrome d ubiquinol oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_d_ubiquinol_oxidases@1>Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5177@1>fig|376686.40.peg.5177</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_d_ubiquinol_oxidases&organism=376686.40@1>Terminal cytochrome d ubiquinol oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_d_ubiquinol_oxidases@1>Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5176@1>fig|376686.40.peg.5176</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=376686.40@1>Anaerobic respiratory reductases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Flavodoxin reductases (ferredoxin-NADPH reductases) family 1&subsystem_name=Anaerobic_respiratory_reductases@1>Flavodoxin reductases (ferredoxin-NADPH reductases) family 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1106@1>fig|376686.40.peg.1106</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1677@1>fig|376686.40.peg.1677</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_oxidases&organism=376686.40@1>Terminal cytochrome oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_oxidases@1>Cytochrome d ubiquinol oxidase subunit I (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5177@1>fig|376686.40.peg.5177</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Terminal_cytochrome_oxidases&organism=376686.40@1>Terminal cytochrome oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)&subsystem_name=Terminal_cytochrome_oxidases@1>Cytochrome d ubiquinol oxidase subunit II (EC 1.10.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5176@1>fig|376686.40.peg.5176</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1305@1>fig|376686.40.peg.1305</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1304@1>fig|376686.40.peg.1304</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1306@1>fig|376686.40.peg.1306</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1302@1>fig|376686.40.peg.1302</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1308@1>fig|376686.40.peg.1308</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1311@1>fig|376686.40.peg.1311</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1314@1>fig|376686.40.peg.1314</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1307@1>fig|376686.40.peg.1307</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1303@1>fig|376686.40.peg.1303</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1310@1>fig|376686.40.peg.1310</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1309@1>fig|376686.40.peg.1309</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1313@1>fig|376686.40.peg.1313</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1315@1>fig|376686.40.peg.1315</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=376686.40@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1312@1>fig|376686.40.peg.1312</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=376686.40@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH dehydrogenase (EC 1.6.99.3)&subsystem_name=Respiratory_dehydrogenases_1@1>NADH dehydrogenase (EC 1.6.99.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3003@1>fig|376686.40.peg.3003</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=376686.40@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)&subsystem_name=Respiratory_dehydrogenases_1@1>Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5183@1>fig|376686.40.peg.5183</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=376686.40@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypF&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase metallocenter assembly protein HypF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4138@1>fig|376686.40.peg.4138</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=376686.40@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase nickel incorporation protein HypA&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase nickel incorporation protein HypA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4133@1>fig|376686.40.peg.4133</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=376686.40@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypC&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase metallocenter assembly protein HypC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4131@1>fig|376686.40.peg.4131</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=376686.40@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase nickel incorporation-associated protein HypB&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase nickel incorporation-associated protein HypB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4132@1>fig|376686.40.peg.4132</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=376686.40@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypD&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase metallocenter assembly protein HypD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4140@1>fig|376686.40.peg.4140</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NiFe_hydrogenase_maturation&organism=376686.40@1>NiFe hydrogenase maturation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=[NiFe] hydrogenase metallocenter assembly protein HypE&subsystem_name=NiFe_hydrogenase_maturation@1>[NiFe] hydrogenase metallocenter assembly protein HypE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4130@1>fig|376686.40.peg.4130</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1305@1>fig|376686.40.peg.1305</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1304@1>fig|376686.40.peg.1304</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1306@1>fig|376686.40.peg.1306</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1302@1>fig|376686.40.peg.1302</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1308@1>fig|376686.40.peg.1308</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1311@1>fig|376686.40.peg.1311</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1314@1>fig|376686.40.peg.1314</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1307@1>fig|376686.40.peg.1307</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1303@1>fig|376686.40.peg.1303</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1310@1>fig|376686.40.peg.1310</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1309@1>fig|376686.40.peg.1309</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1313@1>fig|376686.40.peg.1313</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1315@1>fig|376686.40.peg.1315</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=376686.40@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1312@1>fig|376686.40.peg.1312</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Quinone_oxidoreductase_family&organism=376686.40@1>Quinone oxidoreductase family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinone oxidoreductase (EC 1.6.5.5)&subsystem_name=Quinone_oxidoreductase_family@1>Quinone oxidoreductase (EC 1.6.5.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.294@1>fig|376686.40.peg.294</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2993@1>fig|376686.40.peg.2993</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cytochrome_c_oxidases&organism=376686.40@1>Biogenesis of cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Heme A synthase, cytochrome oxidase biogenesis protein Cox15-CtaA&subsystem_name=Biogenesis_of_cytochrome_c_oxidases@1>Heme A synthase, cytochrome oxidase biogenesis protein Cox15-CtaA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5205@1>fig|376686.40.peg.5205</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=376686.40@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoI&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoI</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2661@1>fig|376686.40.peg.2661</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=376686.40@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoS, involved in heme b insertion&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoS, involved in heme b insertion</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2662@1>fig|376686.40.peg.2662</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=376686.40@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoG, involved in Cu oxidation&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoG, involved in Cu oxidation</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2666@1>fig|376686.40.peg.2666</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=376686.40@1>Biogenesis of c-type cytochromes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative cytochrome C-type biogenesis protein&subsystem_name=Biogenesis_of_c-type_cytochromes@1>Putative cytochrome C-type biogenesis protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3110@1>fig|376686.40.peg.3110</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=376686.40@1>Biogenesis of c-type cytochromes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)&subsystem_name=Biogenesis_of_c-type_cytochromes@1>Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2112@1>fig|376686.40.peg.2112</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2251@1>fig|376686.40.peg.2251</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=376686.40@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1398@1>fig|376686.40.peg.1398</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=376686.40@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c551/c552&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Cytochrome c551/c552</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2758@1>fig|376686.40.peg.2758</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4883@1>fig|376686.40.peg.4883</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=376686.40@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Ferredoxin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2587@1>fig|376686.40.peg.2587</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=376686.40@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c552&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Cytochrome c552</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1006@1>fig|376686.40.peg.1006</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=376686.40@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane protein A precursor&subsystem_name=Osmoregulation@1>Outer membrane protein A precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.902@1>fig|376686.40.peg.902</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=376686.40@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aquaporin Z&subsystem_name=Osmoregulation@1>Aquaporin Z</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2485@1>fig|376686.40.peg.2485</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=376686.40@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric uptake regulation protein FUR&subsystem_name=Oxidative_stress@1>Ferric uptake regulation protein FUR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2338@1>fig|376686.40.peg.2338</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=376686.40@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Superoxide dismutase [Mn] (EC 1.15.1.1)&subsystem_name=Oxidative_stress@1>Superoxide dismutase [Mn] (EC 1.15.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1190@1>fig|376686.40.peg.1190</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2727@1>fig|376686.40.peg.2727</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=376686.40@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional regulator, Crp/Fnr family&subsystem_name=Oxidative_stress@1>Transcriptional regulator, Crp/Fnr family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1664@1>fig|376686.40.peg.1664</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2660@1>fig|376686.40.peg.2660</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4288@1>fig|376686.40.peg.4288</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4710@1>fig|376686.40.peg.4710</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4908@1>fig|376686.40.peg.4908</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5169@1>fig|376686.40.peg.5169</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=376686.40@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Oxidative_stress@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.284@1>fig|376686.40.peg.284</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2228@1>fig|376686.40.peg.2228</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5321@1>fig|376686.40.peg.5321</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=376686.40@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=bacteriophytochrome heme oxygenase BphO&subsystem_name=Oxidative_stress@1>bacteriophytochrome heme oxygenase BphO</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4167@1>fig|376686.40.peg.4167</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=376686.40@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)&subsystem_name=Oxidative_stress@1>Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2802@1>fig|376686.40.peg.2802</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5047@1>fig|376686.40.peg.5047</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=376686.40@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phytochrome, two-component sensor histidine kinase (EC 2.7.3.-)&subsystem_name=Oxidative_stress@1>Phytochrome, two-component sensor histidine kinase (EC 2.7.3.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4041@1>fig|376686.40.peg.4041</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4166@1>fig|376686.40.peg.4166</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=376686.40@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Organic hydroperoxide resistance protein&subsystem_name=Oxidative_stress@1>Organic hydroperoxide resistance protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4236@1>fig|376686.40.peg.4236</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rubrerythrin&organism=376686.40@1>Rubrerythrin</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rubredoxin&subsystem_name=Rubrerythrin@1>Rubredoxin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4910@1>fig|376686.40.peg.4910</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rubrerythrin&organism=376686.40@1>Rubrerythrin</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Rubrerythrin@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.284@1>fig|376686.40.peg.284</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2228@1>fig|376686.40.peg.2228</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5321@1>fig|376686.40.peg.5321</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Redox_cycle&organism=376686.40@1>Glutathione: Redox cycle</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutathione peroxidase (EC 1.11.1.9)&subsystem_name=Glutathione:_Redox_cycle@1>Glutathione peroxidase (EC 1.11.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3102@1>fig|376686.40.peg.3102</a>@~Stress Response@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Uptake_of_selenate_and_selenite&organism=376686.40@1>Uptake of selenate and selenite</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DedA protein&subsystem_name=Uptake_of_selenate_and_selenite@1>DedA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1530@1>fig|376686.40.peg.1530</a>@~Stress Response@^Stress Response - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dimethylarginine_metabolism&organism=376686.40@1>Dimethylarginine metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ornithine aminotransferase (EC 2.6.1.13)&subsystem_name=Dimethylarginine_metabolism@1>Ornithine aminotransferase (EC 2.6.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4936@1>fig|376686.40.peg.4936</a>@~Stress Response@^Stress Response - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dimethylarginine_metabolism&organism=376686.40@1>Dimethylarginine metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NG,NG-dimethylarginine dimethylaminohydrolase 1 (EC 3.5.3.18)&subsystem_name=Dimethylarginine_metabolism@1>NG,NG-dimethylarginine dimethylaminohydrolase 1 (EC 3.5.3.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.384@1>fig|376686.40.peg.384</a>@~Stress Response@^Periplasmic Stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=376686.40@1>Periplasmic Stress Response</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane protein H precursor&subsystem_name=Periplasmic_Stress_Response@1>Outer membrane protein H precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.623@1>fig|376686.40.peg.623</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1055@1>fig|376686.40.peg.1055</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1772@1>fig|376686.40.peg.1772</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1773@1>fig|376686.40.peg.1773</a>@~Stress Response@^Periplasmic Stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=376686.40@1>Periplasmic Stress Response</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Intramembrane protease RasP/YluC, implicated in cell division based on FtsL cleavage&subsystem_name=Periplasmic_Stress_Response@1>Intramembrane protease RasP/YluC, implicated in cell division based on FtsL cleavage</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1221@1>fig|376686.40.peg.1221</a>@~Metabolism of Aromatic Compounds@^Peripheral pathways for catabolism of aromatic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Quinate_degradation&organism=376686.40@1>Quinate degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate dehydratase II (EC 4.2.1.10)&subsystem_name=Quinate_degradation@1>3-dehydroquinate dehydratase II (EC 4.2.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.135@1>fig|376686.40.peg.135</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=376686.40@1>Catechol branch of beta-ketoadipate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)&subsystem_name=Catechol_branch_of_beta-ketoadipate_pathway@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.930@1>fig|376686.40.peg.930</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=376686.40@1>Catechol branch of beta-ketoadipate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)&subsystem_name=Catechol_branch_of_beta-ketoadipate_pathway@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.929@1>fig|376686.40.peg.929</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=376686.40@1>Salicylate and gentisate catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetase (EC 3.7.1.2)&subsystem_name=Salicylate_and_gentisate_catabolism@1>Fumarylacetoacetase (EC 3.7.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3597@1>fig|376686.40.peg.3597</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=376686.40@1>Salicylate and gentisate catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetate hydrolase family protein&subsystem_name=Salicylate_and_gentisate_catabolism@1>Fumarylacetoacetate hydrolase family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5290@1>fig|376686.40.peg.5290</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=376686.40@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homogentisate 1,2-dioxygenase (EC 1.13.11.5)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>Homogentisate 1,2-dioxygenase (EC 1.13.11.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.759@1>fig|376686.40.peg.759</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=376686.40@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetase (EC 3.7.1.2)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>Fumarylacetoacetase (EC 3.7.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3597@1>fig|376686.40.peg.3597</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=376686.40@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.760@1>fig|376686.40.peg.760</a>@~Metabolism of Aromatic Compounds@^Metabolism of Aromatic Compounds - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Gentisate_degradation&organism=376686.40@1>Gentisate degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetate hydrolase family protein&subsystem_name=Gentisate_degradation@1>Fumarylacetoacetate hydrolase family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5290@1>fig|376686.40.peg.5290</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-asparaginase I, cytoplasmic (EC 3.5.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>L-asparaginase I, cytoplasmic (EC 3.5.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1349@1>fig|376686.40.peg.1349</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Ferredoxin-dependent glutamate synthase (EC 1.4.7.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2531@1>fig|376686.40.peg.2531</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate aminotransferase (EC 2.6.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate aminotransferase (EC 2.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.9@1>fig|376686.40.peg.9</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.540@1>fig|376686.40.peg.540</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1831@1>fig|376686.40.peg.1831</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4377@1>fig|376686.40.peg.4377</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-specific glutamate dehydrogenase (EC 1.4.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>NAD-specific glutamate dehydrogenase (EC 1.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4581@1>fig|376686.40.peg.4581</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate racemase (EC 5.1.1.3)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate racemase (EC 5.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1770@1>fig|376686.40.peg.1770</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2367@1>fig|376686.40.peg.2367</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2368@1>fig|376686.40.peg.2368</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2463@1>fig|376686.40.peg.2463</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] large chain (EC 1.4.1.13)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate synthase [NADPH] large chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2275@1>fig|376686.40.peg.2275</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2465@1>fig|376686.40.peg.2465</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate synthase [NADPH] small chain (EC 1.4.1.13)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate synthase [NADPH] small chain (EC 1.4.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2274@1>fig|376686.40.peg.2274</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.739@1>fig|376686.40.peg.739</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate ammonia-lyase (EC 4.3.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate ammonia-lyase (EC 4.3.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3589@1>fig|376686.40.peg.3589</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=376686.40@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate racemase (EC 5.1.1.13)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate racemase (EC 5.1.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2200@1>fig|376686.40.peg.2200</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2622@1>fig|376686.40.peg.2622</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=376686.40@1>Glutamate dehydrogenases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Glutamate_dehydrogenases@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.739@1>fig|376686.40.peg.739</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=376686.40@1>Glutamate dehydrogenases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-specific glutamate dehydrogenase (EC 1.4.1.2)&subsystem_name=Glutamate_dehydrogenases@1>NAD-specific glutamate dehydrogenase (EC 1.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4581@1>fig|376686.40.peg.4581</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=376686.40@1>Glutamine synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Glutamine_synthetases@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2465@1>fig|376686.40.peg.2465</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=376686.40@1>Glutamine synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)&subsystem_name=Glutamine_synthetases@1>Glutamine synthetase type II, eukaryotic (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2463@1>fig|376686.40.peg.2463</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=376686.40@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formiminoglutamase (EC 3.5.3.8)&subsystem_name=Histidine_Degradation@1>Formiminoglutamase (EC 3.5.3.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4844@1>fig|376686.40.peg.4844</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=376686.40@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Imidazolonepropionase (EC 3.5.2.7)&subsystem_name=Histidine_Degradation@1>Imidazolonepropionase (EC 3.5.2.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4842@1>fig|376686.40.peg.4842</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=376686.40@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urocanate hydratase (EC 4.2.1.49)&subsystem_name=Histidine_Degradation@1>Urocanate hydratase (EC 4.2.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2003@1>fig|376686.40.peg.2003</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=376686.40@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidine ammonia-lyase (EC 4.3.1.3)&subsystem_name=Histidine_Degradation@1>Histidine ammonia-lyase (EC 4.3.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2706@1>fig|376686.40.peg.2706</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=376686.40@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)&subsystem_name=Histidine_Biosynthesis@1>Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3027@1>fig|376686.40.peg.3027</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=376686.40@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidinol-phosphate aminotransferase (EC 2.6.1.9)&subsystem_name=Histidine_Biosynthesis@1>Histidinol-phosphate aminotransferase (EC 2.6.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3030@1>fig|376686.40.peg.3030</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=376686.40@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosyl-ATP pyrophosphatase (EC 3.6.1.31)&subsystem_name=Histidine_Biosynthesis@1>Phosphoribosyl-ATP pyrophosphatase (EC 3.6.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3025@1>fig|376686.40.peg.3025</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=376686.40@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Imidazoleglycerol-phosphate dehydratase (EC 4.2.1.19)&subsystem_name=Histidine_Biosynthesis@1>Imidazoleglycerol-phosphate dehydratase (EC 4.2.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3029@1>fig|376686.40.peg.3029</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=376686.40@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosyl-AMP cyclohydrolase (EC 3.5.4.19)&subsystem_name=Histidine_Biosynthesis@1>Phosphoribosyl-AMP cyclohydrolase (EC 3.5.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3025@1>fig|376686.40.peg.3025</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=376686.40@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidinol-phosphatase (EC 3.1.3.15)&subsystem_name=Histidine_Biosynthesis@1>Histidinol-phosphatase (EC 3.1.3.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3029@1>fig|376686.40.peg.3029</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Biosynthesis&organism=376686.40@1>Histidine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidinol dehydrogenase (EC 1.1.1.23)&subsystem_name=Histidine_Biosynthesis@1>Histidinol dehydrogenase (EC 1.1.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3031@1>fig|376686.40.peg.3031</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=376686.40@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreD&subsystem_name=Urease_subunits@1>Urease accessory protein UreD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5125@1>fig|376686.40.peg.5125</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=376686.40@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreF&subsystem_name=Urease_subunits@1>Urease accessory protein UreF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5127@1>fig|376686.40.peg.5127</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=376686.40@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease beta subunit (EC 3.5.1.5)&subsystem_name=Urease_subunits@1>Urease beta subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5130@1>fig|376686.40.peg.5130</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=376686.40@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreG&subsystem_name=Urease_subunits@1>Urease accessory protein UreG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5126@1>fig|376686.40.peg.5126</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=376686.40@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease gamma subunit (EC 3.5.1.5)&subsystem_name=Urease_subunits@1>Urease gamma subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5131@1>fig|376686.40.peg.5131</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=376686.40@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreE&subsystem_name=Urease_subunits@1>Urease accessory protein UreE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5128@1>fig|376686.40.peg.5128</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urease_subunits&organism=376686.40@1>Urease subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease alpha subunit (EC 3.5.1.5)&subsystem_name=Urease_subunits@1>Urease alpha subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5129@1>fig|376686.40.peg.5129</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nickel transporter UreH&subsystem_name=Urea_decomposition@1>Nickel transporter UreH</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4141@1>fig|376686.40.peg.4141</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreF&subsystem_name=Urea_decomposition@1>Urease accessory protein UreF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5127@1>fig|376686.40.peg.5127</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease gamma subunit (EC 3.5.1.5)&subsystem_name=Urea_decomposition@1>Urease gamma subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5131@1>fig|376686.40.peg.5131</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreE&subsystem_name=Urea_decomposition@1>Urease accessory protein UreE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5128@1>fig|376686.40.peg.5128</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Eukaryotic-type low-affinity urea transporter&subsystem_name=Urea_decomposition@1>Eukaryotic-type low-affinity urea transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5123@1>fig|376686.40.peg.5123</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease alpha subunit (EC 3.5.1.5)&subsystem_name=Urea_decomposition@1>Urease alpha subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5129@1>fig|376686.40.peg.5129</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreD&subsystem_name=Urea_decomposition@1>Urease accessory protein UreD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5125@1>fig|376686.40.peg.5125</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease beta subunit (EC 3.5.1.5)&subsystem_name=Urea_decomposition@1>Urease beta subunit (EC 3.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5130@1>fig|376686.40.peg.5130</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Urea_decomposition&organism=376686.40@1>Urea decomposition</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urease accessory protein UreG&subsystem_name=Urea_decomposition@1>Urease accessory protein UreG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5126@1>fig|376686.40.peg.5126</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter ATP-binding protein&subsystem_name=Methionine_Biosynthesis@1>Methionine ABC transporter ATP-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4218@1>fig|376686.40.peg.4218</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylmethionine synthetase (EC 2.5.1.6)&subsystem_name=Methionine_Biosynthesis@1>S-adenosylmethionine synthetase (EC 2.5.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1568@1>fig|376686.40.peg.1568</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylhomoserine sulfhydrylase (EC 2.5.1.48)&subsystem_name=Methionine_Biosynthesis@1>O-succinylhomoserine sulfhydrylase (EC 2.5.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1571@1>fig|376686.40.peg.1571</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1577@1>fig|376686.40.peg.1577</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2243@1>fig|376686.40.peg.2243</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Methionine_Biosynthesis@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.741@1>fig|376686.40.peg.741</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine acetyltransferase (EC 2.3.1.30)&subsystem_name=Methionine_Biosynthesis@1>Serine acetyltransferase (EC 2.3.1.30)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2630@1>fig|376686.40.peg.2630</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine kinase (EC 2.7.1.39)&subsystem_name=Methionine_Biosynthesis@1>Homoserine kinase (EC 2.7.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2703@1>fig|376686.40.peg.2703</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase (EC 2.1.1.14)&subsystem_name=Methionine_Biosynthesis@1>5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase (EC 2.1.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1600@1>fig|376686.40.peg.1600</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4061@1>fig|376686.40.peg.4061</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine synthase (EC 2.5.1.47)&subsystem_name=Methionine_Biosynthesis@1>Cysteine synthase (EC 2.5.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2629@1>fig|376686.40.peg.2629</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4931@1>fig|376686.40.peg.4931</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-acetylhomoserine sulfhydrylase (EC 2.5.1.49)&subsystem_name=Methionine_Biosynthesis@1>O-acetylhomoserine sulfhydrylase (EC 2.5.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1571@1>fig|376686.40.peg.1571</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1577@1>fig|376686.40.peg.1577</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2243@1>fig|376686.40.peg.2243</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=Methionine_Biosynthesis@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1592@1>fig|376686.40.peg.1592</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Methionine_Biosynthesis@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2023@1>fig|376686.40.peg.2023</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2684@1>fig|376686.40.peg.2684</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine dehydrogenase (EC 1.1.1.3)&subsystem_name=Methionine_Biosynthesis@1>Homoserine dehydrogenase (EC 1.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1572@1>fig|376686.40.peg.1572</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1575@1>fig|376686.40.peg.1575</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2704@1>fig|376686.40.peg.2704</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter substrate-binding protein&subsystem_name=Methionine_Biosynthesis@1>Methionine ABC transporter substrate-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4216@1>fig|376686.40.peg.4216</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine O-acetyltransferase (EC 2.3.1.31)&subsystem_name=Methionine_Biosynthesis@1>Homoserine O-acetyltransferase (EC 2.3.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1574@1>fig|376686.40.peg.1574</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)&subsystem_name=Methionine_Biosynthesis@1>S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3420@1>fig|376686.40.peg.3420</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter permease protein&subsystem_name=Methionine_Biosynthesis@1>Methionine ABC transporter permease protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4217@1>fig|376686.40.peg.4217</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional activator MetR&subsystem_name=Methionine_Biosynthesis@1>Transcriptional activator MetR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.912@1>fig|376686.40.peg.912</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Biosynthesis&organism=376686.40@1>Methionine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylhomocysteinase (EC 3.3.1.1)&subsystem_name=Methionine_Biosynthesis@1>Adenosylhomocysteinase (EC 3.3.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1660@1>fig|376686.40.peg.1660</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=376686.40@1>Threonine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-allo-threonine aldolase (EC 2.1.2.1)&subsystem_name=Threonine_degradation@1>L-allo-threonine aldolase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2681@1>fig|376686.40.peg.2681</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=376686.40@1>Threonine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=low-specificity D-threonine aldolase&subsystem_name=Threonine_degradation@1>low-specificity D-threonine aldolase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.699@1>fig|376686.40.peg.699</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=376686.40@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate decarboxylase (EC 4.1.1.20)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Diaminopimelate decarboxylase (EC 4.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2273@1>fig|376686.40.peg.2273</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=376686.40@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2649@1>fig|376686.40.peg.2649</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=376686.40@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate epimerase (EC 5.1.1.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Diaminopimelate epimerase (EC 5.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.19@1>fig|376686.40.peg.19</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=376686.40@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1394@1>fig|376686.40.peg.1394</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=376686.40@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.218@1>fig|376686.40.peg.218</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=376686.40@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.559@1>fig|376686.40.peg.559</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1575@1>fig|376686.40.peg.1575</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2704@1>fig|376686.40.peg.2704</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=376686.40@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3659@1>fig|376686.40.peg.3659</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=376686.40@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)&subsystem_name=Methionine_Degradation@1>S-adenosylhomocysteine nucleosidase (EC 3.2.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3420@1>fig|376686.40.peg.3420</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=376686.40@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter substrate-binding protein&subsystem_name=Methionine_Degradation@1>Methionine ABC transporter substrate-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4216@1>fig|376686.40.peg.4216</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=376686.40@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter ATP-binding protein&subsystem_name=Methionine_Degradation@1>Methionine ABC transporter ATP-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4218@1>fig|376686.40.peg.4218</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=376686.40@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine ABC transporter permease protein&subsystem_name=Methionine_Degradation@1>Methionine ABC transporter permease protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4217@1>fig|376686.40.peg.4217</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=376686.40@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=S-adenosylmethionine synthetase (EC 2.5.1.6)&subsystem_name=Methionine_Degradation@1>S-adenosylmethionine synthetase (EC 2.5.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1568@1>fig|376686.40.peg.1568</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=376686.40@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Methionine_Degradation@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.741@1>fig|376686.40.peg.741</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=376686.40@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylhomocysteinase (EC 3.3.1.1)&subsystem_name=Methionine_Degradation@1>Adenosylhomocysteinase (EC 3.3.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1660@1>fig|376686.40.peg.1660</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methionine_Degradation&organism=376686.40@1>Methionine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Methionine_Degradation@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2023@1>fig|376686.40.peg.2023</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2684@1>fig|376686.40.peg.2684</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=376686.40@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine dehydrogenase (EC 1.1.1.3)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Homoserine dehydrogenase (EC 1.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1572@1>fig|376686.40.peg.1572</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1575@1>fig|376686.40.peg.1575</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2704@1>fig|376686.40.peg.2704</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=376686.40@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1394@1>fig|376686.40.peg.1394</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=376686.40@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate aminotransferase (EC 2.6.1.1)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartate aminotransferase (EC 2.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.9@1>fig|376686.40.peg.9</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.540@1>fig|376686.40.peg.540</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1831@1>fig|376686.40.peg.1831</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4377@1>fig|376686.40.peg.4377</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=376686.40@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonine synthase (EC 4.2.3.1)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Threonine synthase (EC 4.2.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2702@1>fig|376686.40.peg.2702</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=376686.40@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine kinase (EC 2.7.1.39)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Homoserine kinase (EC 2.7.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2703@1>fig|376686.40.peg.2703</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=376686.40@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.559@1>fig|376686.40.peg.559</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1575@1>fig|376686.40.peg.1575</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2704@1>fig|376686.40.peg.2704</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=376686.40@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfate adenylyltransferase subunit 2 (EC 2.7.7.4)&subsystem_name=Cysteine_Biosynthesis@1>Sulfate adenylyltransferase subunit 2 (EC 2.7.7.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1583@1>fig|376686.40.peg.1583</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=376686.40@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfate adenylyltransferase subunit 1 (EC 2.7.7.4)&subsystem_name=Cysteine_Biosynthesis@1>Sulfate adenylyltransferase subunit 1 (EC 2.7.7.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1584@1>fig|376686.40.peg.1584</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=376686.40@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine synthase (EC 2.5.1.47)&subsystem_name=Cysteine_Biosynthesis@1>Cysteine synthase (EC 2.5.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2629@1>fig|376686.40.peg.2629</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4931@1>fig|376686.40.peg.4931</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=376686.40@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfate permease&subsystem_name=Cysteine_Biosynthesis@1>Sulfate permease</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1930@1>fig|376686.40.peg.1930</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=376686.40@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Cysteine_Biosynthesis@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.741@1>fig|376686.40.peg.741</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=376686.40@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylylsulfate kinase (EC 2.7.1.25)&subsystem_name=Cysteine_Biosynthesis@1>Adenylylsulfate kinase (EC 2.7.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5316@1>fig|376686.40.peg.5316</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=376686.40@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine acetyltransferase (EC 2.3.1.30)&subsystem_name=Cysteine_Biosynthesis@1>Serine acetyltransferase (EC 2.3.1.30)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2630@1>fig|376686.40.peg.2630</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cysteine_Biosynthesis&organism=376686.40@1>Cysteine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Cysteine_Biosynthesis@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2023@1>fig|376686.40.peg.2023</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2684@1>fig|376686.40.peg.2684</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=376686.40@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate decarboxylase (EC 4.1.1.20)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Diaminopimelate decarboxylase (EC 4.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2273@1>fig|376686.40.peg.2273</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=376686.40@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2649@1>fig|376686.40.peg.2649</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=376686.40@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate epimerase (EC 5.1.1.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Diaminopimelate epimerase (EC 5.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.19@1>fig|376686.40.peg.19</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=376686.40@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1394@1>fig|376686.40.peg.1394</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=376686.40@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.218@1>fig|376686.40.peg.218</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=376686.40@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.559@1>fig|376686.40.peg.559</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1575@1>fig|376686.40.peg.1575</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2704@1>fig|376686.40.peg.2704</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=376686.40@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3659@1>fig|376686.40.peg.3659</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=376686.40@1>Lysine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lysine ketoglutarate reductase (EC 1.5.1.8) (LOR) (LKR)&subsystem_name=Lysine_degradation@1>Lysine ketoglutarate reductase (EC 1.5.1.8) (LOR) (LKR)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5076@1>fig|376686.40.peg.5076</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=376686.40@1>Lysine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Saccharopine dehydrogenase (EC 1.5.1.9)&subsystem_name=Lysine_degradation@1>Saccharopine dehydrogenase (EC 1.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5076@1>fig|376686.40.peg.5076</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=376686.40@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Isoleucine_degradation@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.984@1>fig|376686.40.peg.984</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=376686.40@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Isoleucine_degradation@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5057@1>fig|376686.40.peg.5057</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=376686.40@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Isoleucine_degradation@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5292@1>fig|376686.40.peg.5292</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=376686.40@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Isoleucine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4782@1>fig|376686.40.peg.4782</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=376686.40@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Isoleucine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4782@1>fig|376686.40.peg.4782</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=376686.40@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-ketoacyl-CoA thiolase (EC 2.3.1.16)&subsystem_name=Isoleucine_degradation@1>3-ketoacyl-CoA thiolase (EC 2.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3502@1>fig|376686.40.peg.3502</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=376686.40@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)&subsystem_name=Isoleucine_degradation@1>Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4955@1>fig|376686.40.peg.4955</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA lyase (EC 4.1.3.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Hydroxymethylglutaryl-CoA lyase (EC 4.1.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2329@1>fig|376686.40.peg.2329</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5057@1>fig|376686.40.peg.5057</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylglutaconyl-CoA hydratase (EC 4.2.1.18)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Methylglutaconyl-CoA hydratase (EC 4.2.1.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.622@1>fig|376686.40.peg.622</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylcrotonyl-CoA carboxylase carboxyl transferase subunit (EC 6.4.1.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Methylcrotonyl-CoA carboxylase carboxyl transferase subunit (EC 6.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2673@1>fig|376686.40.peg.2673</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4782@1>fig|376686.40.peg.4782</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.929@1>fig|376686.40.peg.929</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5292@1>fig|376686.40.peg.5292</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.930@1>fig|376686.40.peg.930</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4782@1>fig|376686.40.peg.4782</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Hydroxymethylglutaryl-CoA synthase (EC 2.3.3.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.706@1>fig|376686.40.peg.706</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4955@1>fig|376686.40.peg.4955</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2245@1>fig|376686.40.peg.2245</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4955@1>fig|376686.40.peg.4955</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Degradation_and_HMG-CoA_Metabolism&organism=376686.40@1>Leucine Degradation and HMG-CoA Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)&subsystem_name=Leucine_Degradation_and_HMG-CoA_Metabolism@1>Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1493@1>fig|376686.40.peg.1493</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1378@1>fig|376686.40.peg.1378</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-isopropylmalate synthase (EC 2.3.3.13)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>2-isopropylmalate synthase (EC 2.3.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3019@1>fig|376686.40.peg.3019</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5057@1>fig|376686.40.peg.5057</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydrogenase (EC 1.1.1.85)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>3-isopropylmalate dehydrogenase (EC 1.1.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1375@1>fig|376686.40.peg.1375</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3018@1>fig|376686.40.peg.3018</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=(R)-citramalate synthase (EC 2.3.1.182)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>(R)-citramalate synthase (EC 2.3.1.182)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1376@1>fig|376686.40.peg.1376</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1379@1>fig|376686.40.peg.1379</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonine dehydratase biosynthetic (EC 4.3.1.19)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Threonine dehydratase biosynthetic (EC 4.3.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2001@1>fig|376686.40.peg.2001</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase large subunit (EC 2.2.1.6)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Acetolactate synthase large subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3016@1>fig|376686.40.peg.3016</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroxy-acid dehydratase (EC 4.2.1.9)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Dihydroxy-acid dehydratase (EC 4.2.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3017@1>fig|376686.40.peg.3017</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3396@1>fig|376686.40.peg.3396</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Branched-Chain_Amino_Acid_Biosynthesis&organism=376686.40@1>Branched-Chain Amino Acid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase small subunit (EC 2.2.1.6)&subsystem_name=Branched-Chain_Amino_Acid_Biosynthesis@1>Acetolactate synthase small subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3015@1>fig|376686.40.peg.3015</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4067@1>fig|376686.40.peg.4067</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=376686.40@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Valine_degradation@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.984@1>fig|376686.40.peg.984</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=376686.40@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Valine_degradation@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5057@1>fig|376686.40.peg.5057</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=376686.40@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Valine_degradation@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5292@1>fig|376686.40.peg.5292</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=376686.40@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Valine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4782@1>fig|376686.40.peg.4782</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=376686.40@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Valine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4782@1>fig|376686.40.peg.4782</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=376686.40@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)&subsystem_name=Valine_degradation@1>Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4955@1>fig|376686.40.peg.4955</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=376686.40@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)&subsystem_name=Leucine_Biosynthesis@1>3-isopropylmalate dehydratase small subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1378@1>fig|376686.40.peg.1378</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=376686.40@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)&subsystem_name=Leucine_Biosynthesis@1>3-isopropylmalate dehydratase large subunit (EC 4.2.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1379@1>fig|376686.40.peg.1379</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=376686.40@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-isopropylmalate synthase (EC 2.3.3.13)&subsystem_name=Leucine_Biosynthesis@1>2-isopropylmalate synthase (EC 2.3.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3019@1>fig|376686.40.peg.3019</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=376686.40@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Leucine_Biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5057@1>fig|376686.40.peg.5057</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Leucine_Biosynthesis&organism=376686.40@1>Leucine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-isopropylmalate dehydrogenase (EC 1.1.1.85)&subsystem_name=Leucine_Biosynthesis@1>3-isopropylmalate dehydrogenase (EC 1.1.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1375@1>fig|376686.40.peg.1375</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3018@1>fig|376686.40.peg.3018</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=376686.40@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Shikimate kinase I (EC 2.7.1.71)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Shikimate kinase I (EC 2.7.1.71)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.58@1>fig|376686.40.peg.58</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=376686.40@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2720@1>fig|376686.40.peg.2720</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=376686.40@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate dehydratase II (EC 4.2.1.10)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>3-dehydroquinate dehydratase II (EC 4.2.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.135@1>fig|376686.40.peg.135</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=376686.40@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I beta (EC 2.5.1.54)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I beta (EC 2.5.1.54)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.538@1>fig|376686.40.peg.538</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=376686.40@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate synthase (EC 4.2.3.4)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>3-dehydroquinate synthase (EC 4.2.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2964@1>fig|376686.40.peg.2964</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=376686.40@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chorismate synthase (EC 4.2.3.5)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Chorismate synthase (EC 4.2.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4849@1>fig|376686.40.peg.4849</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Phosphoribosylformimino-5-aminoimidazole carboxamide ribotide isomerase (EC 5.3.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3027@1>fig|376686.40.peg.3027</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, amidotransferase component (EC 4.1.3.27)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate synthase, amidotransferase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5190@1>fig|376686.40.peg.5190</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.148@1>fig|376686.40.peg.148</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5197@1>fig|376686.40.peg.5197</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5191@1>fig|376686.40.peg.5191</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5194@1>fig|376686.40.peg.5194</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Indole-3-glycerol phosphate synthase (EC 4.1.1.48)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Indole-3-glycerol phosphate synthase (EC 4.1.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5192@1>fig|376686.40.peg.5192</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, aminase component (EC 4.1.3.27)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate synthase, aminase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5188@1>fig|376686.40.peg.5188</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5193@1>fig|376686.40.peg.5193</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isochorismate synthase (EC 5.4.4.2)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Isochorismate synthase (EC 5.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4997@1>fig|376686.40.peg.4997</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2249@1>fig|376686.40.peg.2249</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=376686.40@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5190@1>fig|376686.40.peg.5190</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=376686.40@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chorismate mutase I (EC 5.4.99.5)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Chorismate mutase I (EC 5.4.99.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.538@1>fig|376686.40.peg.538</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=376686.40@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prephenate dehydratase (EC 4.2.1.51)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Prephenate dehydratase (EC 4.2.1.51)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.541@1>fig|376686.40.peg.541</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=376686.40@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prephenate and/or arogenate dehydrogenase (unknown specificity) (EC 1.3.1.12)(EC 1.3.1.43)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Prephenate and/or arogenate dehydrogenase (unknown specificity) (EC 1.3.1.12)(EC 1.3.1.43)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.539@1>fig|376686.40.peg.539</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=376686.40@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biosynthetic Aromatic amino acid aminotransferase alpha (EC 2.6.1.57)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Biosynthetic Aromatic amino acid aminotransferase alpha (EC 2.6.1.57)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.540@1>fig|376686.40.peg.540</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, amidotransferase component (EC 4.1.3.27)&subsystem_name=Tryptophan_synthesis@1>Anthranilate synthase, amidotransferase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5190@1>fig|376686.40.peg.5190</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Tryptophan_synthesis@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.148@1>fig|376686.40.peg.148</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Tryptophan_synthesis@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5197@1>fig|376686.40.peg.5197</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Tryptophan_synthesis@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2249@1>fig|376686.40.peg.2249</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Tryptophan_synthesis@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5191@1>fig|376686.40.peg.5191</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Tryptophan_synthesis@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5194@1>fig|376686.40.peg.5194</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Indole-3-glycerol phosphate synthase (EC 4.1.1.48)&subsystem_name=Tryptophan_synthesis@1>Indole-3-glycerol phosphate synthase (EC 4.1.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5192@1>fig|376686.40.peg.5192</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Tryptophan_synthesis@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5190@1>fig|376686.40.peg.5190</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, aminase component (EC 4.1.3.27)&subsystem_name=Tryptophan_synthesis@1>Anthranilate synthase, aminase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5188@1>fig|376686.40.peg.5188</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=376686.40@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Tryptophan_synthesis@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5193@1>fig|376686.40.peg.5193</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=376686.40@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan 2,3-dioxygenase (EC 1.13.11.11)&subsystem_name=Aromatic_amino_acid_degradation@1>Tryptophan 2,3-dioxygenase (EC 1.13.11.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.762@1>fig|376686.40.peg.762</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=376686.40@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)&subsystem_name=Aromatic_amino_acid_degradation@1>Aromatic-L-amino-acid decarboxylase (EC 4.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1595@1>fig|376686.40.peg.1595</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=376686.40@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)&subsystem_name=Aromatic_amino_acid_degradation@1>4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.760@1>fig|376686.40.peg.760</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=376686.40@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aromatic amino acid transport protein AroP&subsystem_name=Aromatic_amino_acid_degradation@1>Aromatic amino acid transport protein AroP</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3322@1>fig|376686.40.peg.3322</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=376686.40@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-carboxymethyl-2-oxo-hex-3- ene-1,7-dioate decarboxylase (EC 4.1.1.68)&subsystem_name=Aromatic_amino_acid_degradation@1>5-carboxymethyl-2-oxo-hex-3- ene-1,7-dioate decarboxylase (EC 4.1.1.68)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4359@1>fig|376686.40.peg.4359</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=376686.40@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanine-4-hydroxylase (EC 1.14.16.1)&subsystem_name=Aromatic_amino_acid_degradation@1>Phenylalanine-4-hydroxylase (EC 1.14.16.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1997@1>fig|376686.40.peg.1997</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=376686.40@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyrroline-5-carboxylate reductase (EC 1.5.1.2)&subsystem_name=Proline_Synthesis@1>Pyrroline-5-carboxylate reductase (EC 1.5.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.880@1>fig|376686.40.peg.880</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=376686.40@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Gamma-glutamyl phosphate reductase (EC 1.2.1.41)&subsystem_name=Proline_Synthesis@1>Gamma-glutamyl phosphate reductase (EC 1.2.1.41)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3619@1>fig|376686.40.peg.3619</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=376686.40@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Proline_Synthesis@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.739@1>fig|376686.40.peg.739</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=376686.40@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate 5-kinase (EC 2.7.2.11)&subsystem_name=Proline_Synthesis@1>Glutamate 5-kinase (EC 2.7.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3618@1>fig|376686.40.peg.3618</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=376686.40@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Proline iminopeptidase (EC 3.4.11.5)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>Proline iminopeptidase (EC 3.4.11.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.718@1>fig|376686.40.peg.718</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2083@1>fig|376686.40.peg.2083</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=376686.40@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1837@1>fig|376686.40.peg.1837</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=376686.40@1>Glycine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)&subsystem_name=Glycine_Biosynthesis@1>2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.728@1>fig|376686.40.peg.728</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2541@1>fig|376686.40.peg.2541</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=376686.40@1>Glycine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Glycine_Biosynthesis@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3596@1>fig|376686.40.peg.3596</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=376686.40@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Alanine_biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5057@1>fig|376686.40.peg.5057</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=376686.40@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Alanine_biosynthesis@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1398@1>fig|376686.40.peg.1398</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=376686.40@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=Alanine_biosynthesis@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2113@1>fig|376686.40.peg.2113</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2636@1>fig|376686.40.peg.2636</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2945@1>fig|376686.40.peg.2945</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3169@1>fig|376686.40.peg.3169</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4222@1>fig|376686.40.peg.4222</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4444@1>fig|376686.40.peg.4444</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4933@1>fig|376686.40.peg.4933</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=376686.40@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine racemase (EC 5.1.1.1)&subsystem_name=Alanine_biosynthesis@1>Alanine racemase (EC 5.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1392@1>fig|376686.40.peg.1392</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=376686.40@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Serine_Biosynthesis@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.876@1>fig|376686.40.peg.876</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2584@1>fig|376686.40.peg.2584</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=376686.40@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine phosphatase (EC 3.1.3.3)&subsystem_name=Serine_Biosynthesis@1>Phosphoserine phosphatase (EC 3.1.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1796@1>fig|376686.40.peg.1796</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1796@1>fig|376686.40.peg.1796</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=376686.40@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Serine_Biosynthesis@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3596@1>fig|376686.40.peg.3596</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=376686.40@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Serine_Biosynthesis@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2585@1>fig|376686.40.peg.2585</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=376686.40@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine cleavage system H protein&subsystem_name=Glycine_cleavage_system@1>Glycine cleavage system H protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1737@1>fig|376686.40.peg.1737</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4712@1>fig|376686.40.peg.4712</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4714@1>fig|376686.40.peg.4714</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=376686.40@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)&subsystem_name=Glycine_cleavage_system@1>Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.462@1>fig|376686.40.peg.462</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=376686.40@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=Glycine_cleavage_system@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2245@1>fig|376686.40.peg.2245</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4955@1>fig|376686.40.peg.4955</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=376686.40@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)&subsystem_name=Glycine_cleavage_system@1>Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2689@1>fig|376686.40.peg.2689</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4713@1>fig|376686.40.peg.4713</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Glycine_and_Serine_Utilization@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.876@1>fig|376686.40.peg.876</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2584@1>fig|376686.40.peg.2584</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, alpha subunit (EC 4.3.1.17)&subsystem_name=Glycine_and_Serine_Utilization@1>L-serine dehydratase, alpha subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4988@1>fig|376686.40.peg.4988</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine phosphatase (EC 3.1.3.3)&subsystem_name=Glycine_and_Serine_Utilization@1>Phosphoserine phosphatase (EC 3.1.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1796@1>fig|376686.40.peg.1796</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, beta subunit (EC 4.3.1.17)&subsystem_name=Glycine_and_Serine_Utilization@1>L-serine dehydratase, beta subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4988@1>fig|376686.40.peg.4988</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Glycine_and_Serine_Utilization@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.741@1>fig|376686.40.peg.741</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Glycine_and_Serine_Utilization@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3596@1>fig|376686.40.peg.3596</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Seryl-tRNA synthetase (EC 6.1.1.11)&subsystem_name=Glycine_and_Serine_Utilization@1>Seryl-tRNA synthetase (EC 6.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.885@1>fig|376686.40.peg.885</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Glycine_and_Serine_Utilization@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2585@1>fig|376686.40.peg.2585</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine cleavage system H protein&subsystem_name=Glycine_and_Serine_Utilization@1>Glycine cleavage system H protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1737@1>fig|376686.40.peg.1737</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4712@1>fig|376686.40.peg.4712</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4714@1>fig|376686.40.peg.4714</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)&subsystem_name=Glycine_and_Serine_Utilization@1>Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.462@1>fig|376686.40.peg.462</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)&subsystem_name=Glycine_and_Serine_Utilization@1>2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.728@1>fig|376686.40.peg.728</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2541@1>fig|376686.40.peg.2541</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)&subsystem_name=Glycine_and_Serine_Utilization@1>Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2689@1>fig|376686.40.peg.2689</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4713@1>fig|376686.40.peg.4713</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=376686.40@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Glycine_and_Serine_Utilization@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2023@1>fig|376686.40.peg.2023</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2684@1>fig|376686.40.peg.2684</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=376686.40@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiol peroxidase, Tpx-type (EC 1.11.1.15)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thiol peroxidase, Tpx-type (EC 1.11.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2225@1>fig|376686.40.peg.2225</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=376686.40@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiol peroxidase, Bcp-type (EC 1.11.1.15)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thiol peroxidase, Bcp-type (EC 1.11.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2563@1>fig|376686.40.peg.2563</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=376686.40@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Thioredoxin-disulfide_reductase@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.284@1>fig|376686.40.peg.284</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2228@1>fig|376686.40.peg.2228</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5321@1>fig|376686.40.peg.5321</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=376686.40@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thioredoxin reductase (EC 1.8.1.9)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thioredoxin reductase (EC 1.8.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.211@1>fig|376686.40.peg.211</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.977@1>fig|376686.40.peg.977</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1588@1>fig|376686.40.peg.1588</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2942@1>fig|376686.40.peg.2942</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=376686.40@1>Galactosylceramide and Sulfatide metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=probable beta-D-galactosidase&subsystem_name=Galactosylceramide_and_Sulfatide_metabolism@1>probable beta-D-galactosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4430@1>fig|376686.40.peg.4430</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Galactosylceramide_and_Sulfatide_metabolism&organism=376686.40@1>Galactosylceramide and Sulfatide metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Arylsulfatase (EC 3.1.6.1)&subsystem_name=Galactosylceramide_and_Sulfatide_metabolism@1>Arylsulfatase (EC 3.1.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1203@1>fig|376686.40.peg.1203</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3317@1>fig|376686.40.peg.3317</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3318@1>fig|376686.40.peg.3318</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3464@1>fig|376686.40.peg.3464</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3779@1>fig|376686.40.peg.3779</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4923@1>fig|376686.40.peg.4923</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4927@1>fig|376686.40.peg.4927</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4930@1>fig|376686.40.peg.4930</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inorganic pyrophosphatase (EC 3.6.1.1)&subsystem_name=Phosphate_metabolism@1>Inorganic pyrophosphatase (EC 3.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1557@1>fig|376686.40.peg.1557</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyrophosphate-energized proton pump (EC 3.6.1.1)&subsystem_name=Phosphate_metabolism@1>Pyrophosphate-energized proton pump (EC 3.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1558@1>fig|376686.40.peg.1558</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exopolyphosphatase (EC 3.6.1.11)&subsystem_name=Phosphate_metabolism@1>Exopolyphosphatase (EC 3.6.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1961@1>fig|376686.40.peg.1961</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4804@1>fig|376686.40.peg.4804</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyrophosphate-specific outer membrane porin OprO&subsystem_name=Phosphate_metabolism@1>Pyrophosphate-specific outer membrane porin OprO</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2324@1>fig|376686.40.peg.2324</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Probable low-affinity inorganic phosphate transporter&subsystem_name=Phosphate_metabolism@1>Probable low-affinity inorganic phosphate transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2708@1>fig|376686.40.peg.2708</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkaline phosphatase (EC 3.1.3.1)&subsystem_name=Phosphate_metabolism@1>Alkaline phosphatase (EC 3.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.24@1>fig|376686.40.peg.24</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3363@1>fig|376686.40.peg.3363</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3428@1>fig|376686.40.peg.3428</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3600@1>fig|376686.40.peg.3600</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate-specific outer membrane porin OprP&subsystem_name=Phosphate_metabolism@1>Phosphate-specific outer membrane porin OprP</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2324@1>fig|376686.40.peg.2324</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Phosphate_metabolism@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4073@1>fig|376686.40.peg.4073</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4805@1>fig|376686.40.peg.4805</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate starvation-inducible protein PhoH, predicted ATPase&subsystem_name=Phosphate_metabolism@1>Phosphate starvation-inducible protein PhoH, predicted ATPase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2872@1>fig|376686.40.peg.2872</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2872@1>fig|376686.40.peg.2872</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=376686.40@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate transport regulator (distant homolog of PhoU)&subsystem_name=Phosphate_metabolism@1>Phosphate transport regulator (distant homolog of PhoU)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2707@1>fig|376686.40.peg.2707</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=376686.40@1>Polyphosphate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Polyphosphate@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4073@1>fig|376686.40.peg.4073</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4805@1>fig|376686.40.peg.4805</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=376686.40@1>Polyphosphate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exopolyphosphatase (EC 3.6.1.11)&subsystem_name=Polyphosphate@1>Exopolyphosphatase (EC 3.6.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1961@1>fig|376686.40.peg.1961</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4804@1>fig|376686.40.peg.4804</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=376686.40@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylglyoxal synthase (EC 4.2.3.3)&subsystem_name=Methylglyoxal_Metabolism@1>Methylglyoxal synthase (EC 4.2.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4861@1>fig|376686.40.peg.4861</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=376686.40@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase B (EC 1.2.1.22)&subsystem_name=Methylglyoxal_Metabolism@1>Aldehyde dehydrogenase B (EC 1.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1294@1>fig|376686.40.peg.1294</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=376686.40@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase (EC 1.2.1.3)&subsystem_name=Methylglyoxal_Metabolism@1>Aldehyde dehydrogenase (EC 1.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3127@1>fig|376686.40.peg.3127</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4992@1>fig|376686.40.peg.4992</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3127@1>fig|376686.40.peg.3127</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4992@1>fig|376686.40.peg.4992</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=376686.40@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase (EC 1.2.1.3)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Aldehyde dehydrogenase (EC 1.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3127@1>fig|376686.40.peg.3127</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4992@1>fig|376686.40.peg.4992</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=376686.40@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1554@1>fig|376686.40.peg.1554</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=376686.40@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1633@1>fig|376686.40.peg.1633</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=376686.40@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate acetyltransferase (EC 2.3.1.8)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Phosphate acetyltransferase (EC 2.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3720@1>fig|376686.40.peg.3720</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=376686.40@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetate kinase (EC 2.7.2.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Acetate kinase (EC 2.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3719@1>fig|376686.40.peg.3719</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=376686.40@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1634@1>fig|376686.40.peg.1634</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=376686.40@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate decarboxylase (EC 4.1.1.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate decarboxylase (EC 4.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4058@1>fig|376686.40.peg.4058</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=376686.40@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-dependent protein deacetylase of SIR2 family&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>NAD-dependent protein deacetylase of SIR2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5317@1>fig|376686.40.peg.5317</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=376686.40@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, alpha subunit (EC 4.3.1.17)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>L-serine dehydratase, alpha subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4988@1>fig|376686.40.peg.4988</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=376686.40@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5057@1>fig|376686.40.peg.5057</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=376686.40@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, beta subunit (EC 4.3.1.17)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>L-serine dehydratase, beta subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4988@1>fig|376686.40.peg.4988</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=376686.40@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine dehydrogenase (EC 1.4.1.1)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Alanine dehydrogenase (EC 1.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.23@1>fig|376686.40.peg.23</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3064@1>fig|376686.40.peg.3064</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=376686.40@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine racemase (EC 5.1.1.1)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Alanine racemase (EC 5.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1392@1>fig|376686.40.peg.1392</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=376686.40@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malate dehydrogenase (EC 1.1.1.37)&subsystem_name=Glyoxylate_bypass@1>Malate dehydrogenase (EC 1.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2371@1>fig|376686.40.peg.2371</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=376686.40@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Citrate synthase (si) (EC 2.3.3.1)&subsystem_name=Glyoxylate_bypass@1>Citrate synthase (si) (EC 2.3.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.385@1>fig|376686.40.peg.385</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=376686.40@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aconitate hydratase 2 (EC 4.2.1.3)&subsystem_name=Glyoxylate_bypass@1>Aconitate hydratase 2 (EC 4.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2047@1>fig|376686.40.peg.2047</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=376686.40@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isocitrate lyase (EC 4.1.3.1)&subsystem_name=Glyoxylate_bypass@1>Isocitrate lyase (EC 4.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1823@1>fig|376686.40.peg.1823</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=376686.40@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malate synthase (EC 2.3.3.9)&subsystem_name=Glyoxylate_bypass@1>Malate synthase (EC 2.3.3.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1822@1>fig|376686.40.peg.1822</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glyoxylate_bypass&organism=376686.40@1>Glyoxylate bypass</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aconitate hydratase (EC 4.2.1.3)&subsystem_name=Glyoxylate_bypass@1>Aconitate hydratase (EC 4.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2048@1>fig|376686.40.peg.2048</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1324@1>fig|376686.40.peg.1324</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5292@1>fig|376686.40.peg.5292</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Dehydrogenase_complexes@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4782@1>fig|376686.40.peg.4782</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Dehydrogenase_complexes@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4782@1>fig|376686.40.peg.4782</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)&subsystem_name=Dehydrogenase_complexes@1>Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1554@1>fig|376686.40.peg.1554</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide dehydrogenase of branched-chain alpha-keto acid dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4955@1>fig|376686.40.peg.4955</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1633@1>fig|376686.40.peg.1633</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)&subsystem_name=Dehydrogenase_complexes@1>2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1325@1>fig|376686.40.peg.1325</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=376686.40@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)&subsystem_name=Dehydrogenase_complexes@1>Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1634@1>fig|376686.40.peg.1634</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=376686.40@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transketolase, N-terminal section (EC 2.2.1.1)&subsystem_name=Pentose_phosphate_pathway@1>Transketolase, N-terminal section (EC 2.2.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.53@1>fig|376686.40.peg.53</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=376686.40@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)&subsystem_name=Pentose_phosphate_pathway@1>Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.787@1>fig|376686.40.peg.787</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1187@1>fig|376686.40.peg.1187</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=376686.40@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transketolase, C-terminal section (EC 2.2.1.1)&subsystem_name=Pentose_phosphate_pathway@1>Transketolase, C-terminal section (EC 2.2.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.54@1>fig|376686.40.peg.54</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=376686.40@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribulose-phosphate 3-epimerase (EC 5.1.3.1)&subsystem_name=Pentose_phosphate_pathway@1>Ribulose-phosphate 3-epimerase (EC 5.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1508@1>fig|376686.40.peg.1508</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=376686.40@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-6-phosphate 1-dehydrogenase (EC 1.1.1.49)&subsystem_name=Pentose_phosphate_pathway@1>Glucose-6-phosphate 1-dehydrogenase (EC 1.1.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5093@1>fig|376686.40.peg.5093</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=376686.40@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6-phosphogluconate dehydrogenase, decarboxylating (EC 1.1.1.44)&subsystem_name=Pentose_phosphate_pathway@1>6-phosphogluconate dehydrogenase, decarboxylating (EC 1.1.1.44)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5092@1>fig|376686.40.peg.5092</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=376686.40@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6-phosphogluconolactonase (EC 3.1.1.31), eukaryotic type&subsystem_name=Pentose_phosphate_pathway@1>6-phosphogluconolactonase (EC 3.1.1.31), eukaryotic type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5094@1>fig|376686.40.peg.5094</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=376686.40@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transaldolase (EC 2.2.1.2)&subsystem_name=Pentose_phosphate_pathway@1>Transaldolase (EC 2.2.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2101@1>fig|376686.40.peg.2101</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4838@1>fig|376686.40.peg.4838</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=376686.40@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1440@1>fig|376686.40.peg.1440</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4711@1>fig|376686.40.peg.4711</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=376686.40@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoenolpyruvate carboxylase (EC 4.1.1.31)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Phosphoenolpyruvate carboxylase (EC 4.1.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2957@1>fig|376686.40.peg.2957</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=376686.40@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-dependent malic enzyme (EC 1.1.1.40)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>NADP-dependent malic enzyme (EC 1.1.1.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1734@1>fig|376686.40.peg.1734</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=376686.40@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate kinase (EC 2.7.1.40)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Pyruvate kinase (EC 2.7.1.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1202@1>fig|376686.40.peg.1202</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycolate,_glyoxylate_interconversions&organism=376686.40@1>Glycolate, glyoxylate interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glyoxylate reductase (EC 1.1.1.26)&subsystem_name=Glycolate,_glyoxylate_interconversions@1>Glyoxylate reductase (EC 1.1.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2005@1>fig|376686.40.peg.2005</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycolate,_glyoxylate_interconversions&organism=376686.40@1>Glycolate, glyoxylate interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoglycolate phosphatase (EC 3.1.3.18)&subsystem_name=Glycolate,_glyoxylate_interconversions@1>Phosphoglycolate phosphatase (EC 3.1.3.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1468@1>fig|376686.40.peg.1468</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycolate,_glyoxylate_interconversions&organism=376686.40@1>Glycolate, glyoxylate interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxypyruvate reductase (EC 1.1.1.81)&subsystem_name=Glycolate,_glyoxylate_interconversions@1>Hydroxypyruvate reductase (EC 1.1.1.81)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2005@1>fig|376686.40.peg.2005</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycolate,_glyoxylate_interconversions&organism=376686.40@1>Glycolate, glyoxylate interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glyoxylate reductase (EC 1.1.1.79)&subsystem_name=Glycolate,_glyoxylate_interconversions@1>Glyoxylate reductase (EC 1.1.1.79)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2005@1>fig|376686.40.peg.2005</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=376686.40@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylglucosamine related transporter, NagX&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>N-acetylglucosamine related transporter, NagX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3291@1>fig|376686.40.peg.3291</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5114@1>fig|376686.40.peg.5114</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5278@1>fig|376686.40.peg.5278</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5296@1>fig|376686.40.peg.5296</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=376686.40@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucosamine-6-phosphate deaminase (EC 3.5.99.6)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Glucosamine-6-phosphate deaminase (EC 3.5.99.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2130@1>fig|376686.40.peg.2130</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2502@1>fig|376686.40.peg.2502</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4828@1>fig|376686.40.peg.4828</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5106@1>fig|376686.40.peg.5106</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=376686.40@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chitinase (EC 3.2.1.14)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Chitinase (EC 3.2.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4826@1>fig|376686.40.peg.4826</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4831@1>fig|376686.40.peg.4831</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5043@1>fig|376686.40.peg.5043</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=376686.40@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted N-acetyl-glucosamine kinase 2, ROK family (EC 2.7.1.59)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Predicted N-acetyl-glucosamine kinase 2, ROK family (EC 2.7.1.59)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2863@1>fig|376686.40.peg.2863</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=376686.40@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylglucosamine-6-phosphate deacetylase (EC 3.5.1.25)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>N-acetylglucosamine-6-phosphate deacetylase (EC 3.5.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4206@1>fig|376686.40.peg.4206</a>@~Carbohydrates@^Aminosugars@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chitin_and_N-acetylglucosamine_utilization&organism=376686.40@1>Chitin and N-acetylglucosamine utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-N-acetylglucosaminidase (EC 3.2.1.50)&subsystem_name=Chitin_and_N-acetylglucosamine_utilization@1>Alpha-N-acetylglucosaminidase (EC 3.2.1.50)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3302@1>fig|376686.40.peg.3302</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Sucrose_utilization&organism=376686.40@1>Sucrose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sucrose-6-phosphate hydrolase (EC 3.2.1.26)&subsystem_name=Sucrose_utilization@1>Sucrose-6-phosphate hydrolase (EC 3.2.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4198@1>fig|376686.40.peg.4198</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Sucrose_utilization&organism=376686.40@1>Sucrose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fructokinase (EC 2.7.1.4)&subsystem_name=Sucrose_utilization@1>Fructokinase (EC 2.7.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4277@1>fig|376686.40.peg.4277</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Trehalose_Uptake_and_Utilization&organism=376686.40@1>Trehalose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Trehalase (EC 3.2.1.28)&subsystem_name=Trehalose_Uptake_and_Utilization@1>Trehalase (EC 3.2.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1331@1>fig|376686.40.peg.1331</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Trehalose_Uptake_and_Utilization&organism=376686.40@1>Trehalose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted trehalose permease, MFS family, FucP subfamily&subsystem_name=Trehalose_Uptake_and_Utilization@1>Predicted trehalose permease, MFS family, FucP subfamily</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1330@1>fig|376686.40.peg.1330</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Trehalose_Uptake_and_Utilization&organism=376686.40@1>Trehalose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-phosphoglucomutase (EC 5.4.2.6)&subsystem_name=Trehalose_Uptake_and_Utilization@1>Beta-phosphoglucomutase (EC 5.4.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1430@1>fig|376686.40.peg.1430</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1475@1>fig|376686.40.peg.1475</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=376686.40@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.355@1>fig|376686.40.peg.355</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1812@1>fig|376686.40.peg.1812</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=376686.40@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Galactose-1-phosphate uridylyltransferase (EC 2.7.7.10)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Galactose-1-phosphate uridylyltransferase (EC 2.7.7.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1173@1>fig|376686.40.peg.1173</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=376686.40@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Galactokinase (EC 2.7.1.6)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Galactokinase (EC 2.7.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1174@1>fig|376686.40.peg.1174</a>@~Carbohydrates@^Di- and oligosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactose_and_Galactose_Uptake_and_Utilization&organism=376686.40@1>Lactose and Galactose Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldose 1-epimerase (EC 5.1.3.3)&subsystem_name=Lactose_and_Galactose_Uptake_and_Utilization@1>Aldose 1-epimerase (EC 5.1.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1175@1>fig|376686.40.peg.1175</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1183@1>fig|376686.40.peg.1183</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2127@1>fig|376686.40.peg.2127</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=376686.40@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formyltetrahydrofolate deformylase (EC 3.5.1.10)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Formyltetrahydrofolate deformylase (EC 3.5.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2445@1>fig|376686.40.peg.2445</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=376686.40@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.755@1>fig|376686.40.peg.755</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=376686.40@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.257@1>fig|376686.40.peg.257</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=376686.40@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1592@1>fig|376686.40.peg.1592</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=376686.40@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.257@1>fig|376686.40.peg.257</a>@~Carbohydrates@^Organic acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactate_utilization&organism=376686.40@1>Lactate utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted L-lactate dehydrogenase, Iron-sulfur cluster-binding subunit YkgF&subsystem_name=Lactate_utilization@1>Predicted L-lactate dehydrogenase, Iron-sulfur cluster-binding subunit YkgF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4452@1>fig|376686.40.peg.4452</a>@~Carbohydrates@^Organic acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactate_utilization&organism=376686.40@1>Lactate utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted L-lactate dehydrogenase, hypothetical protein subunit SO1518&subsystem_name=Lactate_utilization@1>Predicted L-lactate dehydrogenase, hypothetical protein subunit SO1518</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4451@1>fig|376686.40.peg.4451</a>@~Carbohydrates@^Organic acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lactate_utilization&organism=376686.40@1>Lactate utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Predicted L-lactate dehydrogenase, Fe-S oxidoreductase subunit YkgE&subsystem_name=Lactate_utilization@1>Predicted L-lactate dehydrogenase, Fe-S oxidoreductase subunit YkgE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4453@1>fig|376686.40.peg.4453</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=376686.40@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alcohol dehydrogenase (EC 1.1.1.1)&subsystem_name=Butanol_Biosynthesis@1>Alcohol dehydrogenase (EC 1.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3474@1>fig|376686.40.peg.3474</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4407@1>fig|376686.40.peg.4407</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=376686.40@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-dependent butanol dehydrogenase A (EC 1.1.1.-)&subsystem_name=Butanol_Biosynthesis@1>NADH-dependent butanol dehydrogenase A (EC 1.1.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.793@1>fig|376686.40.peg.793</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=376686.40@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Butanol_Biosynthesis@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.984@1>fig|376686.40.peg.984</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=376686.40@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)&subsystem_name=Butanol_Biosynthesis@1>3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3636@1>fig|376686.40.peg.3636</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=376686.40@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Butanol_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4887@1>fig|376686.40.peg.4887</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetolactate_synthase_subunits&organism=376686.40@1>Acetolactate synthase subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase large subunit (EC 2.2.1.6)&subsystem_name=Acetolactate_synthase_subunits@1>Acetolactate synthase large subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3016@1>fig|376686.40.peg.3016</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetolactate_synthase_subunits&organism=376686.40@1>Acetolactate synthase subunits</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetolactate synthase small subunit (EC 2.2.1.6)&subsystem_name=Acetolactate_synthase_subunits@1>Acetolactate synthase small subunit (EC 2.2.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3015@1>fig|376686.40.peg.3015</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4067@1>fig|376686.40.peg.4067</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=376686.40@1>Fermentations: Lactate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-lactate dehydrogenase (EC 1.1.1.28)&subsystem_name=Fermentations:_Lactate@1>D-lactate dehydrogenase (EC 1.1.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2013@1>fig|376686.40.peg.2013</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=376686.40@1>Fermentations: Lactate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate acetyltransferase (EC 2.3.1.8)&subsystem_name=Fermentations:_Lactate@1>Phosphate acetyltransferase (EC 2.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3720@1>fig|376686.40.peg.3720</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fermentations:_Lactate&organism=376686.40@1>Fermentations: Lactate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetate kinase (EC 2.7.2.1)&subsystem_name=Fermentations:_Lactate@1>Acetate kinase (EC 2.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3719@1>fig|376686.40.peg.3719</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=376686.40@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.984@1>fig|376686.40.peg.984</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=376686.40@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Electron transfer flavoprotein, beta subunit&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Electron transfer flavoprotein, beta subunit</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1552@1>fig|376686.40.peg.1552</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=376686.40@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Electron transfer flavoprotein, alpha subunit&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Electron transfer flavoprotein, alpha subunit</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1551@1>fig|376686.40.peg.1551</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=376686.40@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3636@1>fig|376686.40.peg.3636</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=376686.40@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4887@1>fig|376686.40.peg.4887</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Inositol_catabolism&organism=376686.40@1>Inositol catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=uncharacterized domain&subsystem_name=Inositol_catabolism@1>uncharacterized domain</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4698@1>fig|376686.40.peg.4698</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Inositol_catabolism&organism=376686.40@1>Inositol catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inositol oxygenase (EC 1.13.99.1)&subsystem_name=Inositol_catabolism@1>Inositol oxygenase (EC 1.13.99.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3293@1>fig|376686.40.peg.3293</a>@~Carbohydrates@^Polysaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=376686.40@1>Glycogen metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate adenylyltransferase (EC 2.7.7.27)&subsystem_name=Glycogen_metabolism@1>Glucose-1-phosphate adenylyltransferase (EC 2.7.7.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2890@1>fig|376686.40.peg.2890</a>@~Carbohydrates@^Polysaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=376686.40@1>Glycogen metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1,4-alpha-glucan (glycogen) branching enzyme, GH-13-type (EC 2.4.1.18)&subsystem_name=Glycogen_metabolism@1>1,4-alpha-glucan (glycogen) branching enzyme, GH-13-type (EC 2.4.1.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2892@1>fig|376686.40.peg.2892</a>@~Carbohydrates@^Polysaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=376686.40@1>Glycogen metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-glucosidase, family 31 of glycosyl hydrolases, COG1501&subsystem_name=Glycogen_metabolism@1>Alpha-glucosidase, family 31 of glycosyl hydrolases, COG1501</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2893@1>fig|376686.40.peg.2893</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4692@1>fig|376686.40.peg.4692</a>@~Carbohydrates@^Polysaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycogen_metabolism&organism=376686.40@1>Glycogen metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycogen synthase, ADP-glucose transglucosylase (EC 2.4.1.21)&subsystem_name=Glycogen_metabolism@1>Glycogen synthase, ADP-glucose transglucosylase (EC 2.4.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2889@1>fig|376686.40.peg.2889</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=376686.40@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Endo-alpha-mannosidase&subsystem_name=Mannose_Metabolism@1>Endo-alpha-mannosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4572@1>fig|376686.40.peg.4572</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=376686.40@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-1,2-mannosidase&subsystem_name=Mannose_Metabolism@1>Alpha-1,2-mannosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2133@1>fig|376686.40.peg.2133</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2143@1>fig|376686.40.peg.2143</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2479@1>fig|376686.40.peg.2479</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2860@1>fig|376686.40.peg.2860</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2861@1>fig|376686.40.peg.2861</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.3299@1>fig|376686.40.peg.3299</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=376686.40@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomannomutase (EC 5.4.2.8)&subsystem_name=Mannose_Metabolism@1>Phosphomannomutase (EC 5.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4773@1>fig|376686.40.peg.4773</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=376686.40@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mannose-6-phosphate isomerase (EC 5.3.1.8)&subsystem_name=Mannose_Metabolism@1>Mannose-6-phosphate isomerase (EC 5.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.283@1>fig|376686.40.peg.283</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=376686.40@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-mannosidase (EC 3.2.1.25)&subsystem_name=Mannose_Metabolism@1>Beta-mannosidase (EC 3.2.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2859@1>fig|376686.40.peg.2859</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=376686.40@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Endo-1,4-beta-mannosidase&subsystem_name=Mannose_Metabolism@1>Endo-1,4-beta-mannosidase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.5259@1>fig|376686.40.peg.5259</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=D-ribose_utilization&organism=376686.40@1>D-ribose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribokinase (EC 2.7.1.15)&subsystem_name=D-ribose_utilization@1>Ribokinase (EC 2.7.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1192@1>fig|376686.40.peg.1192</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Xylose_utilization&organism=376686.40@1>Xylose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xylose isomerase (EC 5.3.1.5)&subsystem_name=Xylose_utilization@1>Xylose isomerase (EC 5.3.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2100@1>fig|376686.40.peg.2100</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Xylose_utilization&organism=376686.40@1>Xylose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xylanase&subsystem_name=Xylose_utilization@1>Xylanase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.4509@1>fig|376686.40.peg.4509</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Xylose_utilization&organism=376686.40@1>Xylose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xylulose kinase (EC 2.7.1.17)&subsystem_name=Xylose_utilization@1>Xylulose kinase (EC 2.7.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2099@1>fig|376686.40.peg.2099</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=376686.40@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2989@1>fig|376686.40.peg.2989</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=376686.40@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribokinase (EC 2.7.1.15)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Ribokinase (EC 2.7.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1192@1>fig|376686.40.peg.1192</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=376686.40@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative deoxyribonuclease YjjV&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Putative deoxyribonuclease YjjV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2924@1>fig|376686.40.peg.2924</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=376686.40@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribose-phosphate aldolase (EC 4.1.2.4)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Deoxyribose-phosphate aldolase (EC 4.1.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1739@1>fig|376686.40.peg.1739</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=376686.40@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-ribulose-5-phosphate 4-epimerase (EC 5.1.3.4)&subsystem_name=L-Arabinose_utilization@1>L-ribulose-5-phosphate 4-epimerase (EC 5.1.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1181@1>fig|376686.40.peg.1181</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=376686.40@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribulokinase (EC 2.7.1.16)&subsystem_name=L-Arabinose_utilization@1>Ribulokinase (EC 2.7.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1180@1>fig|376686.40.peg.1180</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=376686.40@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Possible GPH family transporter (TC 2.A.2) for arabinosides&subsystem_name=L-Arabinose_utilization@1>Possible GPH family transporter (TC 2.A.2) for arabinosides</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.2096@1>fig|376686.40.peg.2096</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=376686.40@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-arabinose isomerase (EC 5.3.1.4)&subsystem_name=L-Arabinose_utilization@1>L-arabinose isomerase (EC 5.3.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.1182@1>fig|376686.40.peg.1182</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=L-Arabinose_utilization&organism=376686.40@1>L-Arabinose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein AraJ precursor&subsystem_name=L-Arabinose_utilization@1>Protein AraJ precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|376686.40.peg.197@1>fig|376686.40.peg.197</a>'>
<input type='hidden' id='table_onclicks_0' value='@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^'>
<input type='hidden' id='table_highlights_0' value='@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^'>
<input type='hidden' id='table_filtereddata_0' value=''>
<input type='hidden' id='table_rows_0' value='995'>
<input type='hidden' id='table_cols_0' value='5'>
<input type='hidden' id='table_start_0' value='0'>
<input type='hidden' id='table_sortdirection_0' value='up'>
<input type='hidden' id='table_column_types_0' value='@~@~@~@~'>
<input type='hidden' id='table_hoverid_0' value='1'>
<span id='table_input_space_0'></span><input type='hidden' id='ajax_url' value='https://rast.nmpdr.org/ajax.cgi'>
<input type='hidden' id='ajax_params' value='page=Organism&app=SeedViewer&url=seedviewer.cgi&cookies=0' />
<div style='display:none;' id='table_0_download'></div><table id='table_0' class='table_table' style=''><tr><td name='0_col_1' id='0_col_1' class='table_first_row' style=''><a href='javascript: table_sort("0", "1");' class='table_first_row' title='click to sort'><span id='colname_0_col_1'>Category</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_1_operator' value='equal' id='table_0_operator_1'><select name='0_col_1' class='filter_item' id='table_0_operand_1' onchange='return check_submit_filter2("0", this.options[this.selectedIndex].text);' style='width: 100%;' title='Select Filter'></select></td><td name='0_col_2' id='0_col_2' class='table_first_row' style=''><a href='javascript: table_sort("0", "2");' class='table_first_row' title='click to sort'><span id='colname_0_col_2'>Subcategory</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_2_operator' value='equal' id='table_0_operator_2'><select name='0_col_2' class='filter_item' id='table_0_operand_2' onchange='return check_submit_filter2("0", this.options[this.selectedIndex].text);' style='width: 100%;' title='Select Filter'></select></td><td name='0_col_3' id='0_col_3' class='table_first_row' style=''><a href='javascript: table_sort("0", "3");' class='table_first_row' title='click to sort'><span id='colname_0_col_3'>Subsystem</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_3_operator' value='like' id='table_0_operator_3'><input type='text' name='0_col_3' class='filter_item' value='' size=5 id='table_0_operand_3' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td><td name='0_col_4' id='0_col_4' class='table_first_row' style=''><a href='javascript: table_sort("0", "4");' class='table_first_row' title='click to sort'><span id='colname_0_col_4'>Role</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_4_operator' value='like' id='table_0_operator_4'><input type='text' name='0_col_4' class='filter_item' value='' size=5 id='table_0_operand_4' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td><td name='0_col_5' id='0_col_5' class='table_first_row' style=''><span id='colname_0_col_5'>Features</span><br><input type=hidden name='0_col_5_operator' value='like' id='table_0_operator_5'><input type='text' name='0_col_5' class='filter_item' value='' size=5 id='table_0_operand_5' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td></table><input type='hidden' id='table_visible_columns_0' value='1@~1@~1@~1@~1'>
<input type='hidden' id='table_combo_columns_0' value='1@~2'>
<input type='hidden' id='table_sc_0' value=''><img src='https://rast.nmpdr.org/Html/clear.gif' onload='initialize_table("0")'><input type='hidden' id='hover_redundancies_1' value=''></td></tr><tr><td style='width: 100%; text-align: center;'><table style='width: 100%;'><tr><td align='left' width='20%'><a href='javascript: table_first("0");' name='table_first_0'>&laquo;first</a>&nbsp;&nbsp;<a href='javascript: table_prev("0");' name='table_prev_0'>&laquo;prev</a></td><td align='center' width='60%'>displaying <span id='table_start_bottom_0'>1</span> - <span id='table_stop_bottom_0'>995</span> of <span id='table_total_bottom_0'>995</span></td><td align='right' width='20%'><a href='javascript: table_next("0");' name='table_next_0'>next&raquo;</a>&nbsp;&nbsp;<a href='javascript: table_last("0");' name='table_last_0'>last&raquo;</a></td></tr></table></td></tr></table></div></td></tr></table>
<img src='https://rast.nmpdr.org/Html/clear.gif'  onload='initialize_tab_view("1"); tab_view_select("1", "0");'>
<br><br><!-- tracking off -->
  </div>
    
</body>


</html>

