<!doctype HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">    
    <title>Seed Viewer - Organism</title>

    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/TabView.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/Hover.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/Table.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/web_app_default.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/commonviewer.css" >
    
    <link rel="stylesheet" type="text/css" href="https://rast.nmpdr.org/Html/seedviewer.css" >
       

    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/PopupTooltip.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/TabView.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Ajax.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Hover.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Tree.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/Table.js" ></script>
    
    <script type="text/javascript" src="https://rast.nmpdr.org/Html/fade.js" ></script>
    
    <meta name="HTML_PATH" content="./Html" />
    
    <META HTTP-EQUIV="PRAGMA" CONTENT="NO-CACHE">
    
    <META HTTP-EQUIV="CACHE-CONTROL" CONTENT="NO-CACHE">
    
    <meta name="robots" content="noindex,nofollow" />
    
    

  </head>

  <body>
<div id="header">
  <span id="logo">
  <img height="95px" src="./Html/seed-logo-green.png" alt="The SEED" />
  </span>
  <p>
  <span id="title">
  <img src="./Html/seed-theseed-viewer.png" alt="The SEED Viewer" />
  </span>
  <span id="version">SEED Viewer version 2.0</span>
  </p>
  <p>Welcome to the SEED Viewer - a read-only browser of the curated SEED data. <br/>
  For more information about The SEED please visit <a href="http://www.theseed.org">theSEED.org.</a>

  </p>




</div>

<div id="topbar">
  <table style="width: 100%; border-spacing: 0px;">
    <tr>
      <td style='width: 100%; padding: 0px;'>
	
	  <div id='menu'>
	<ul id='nav'>
		<li><div><a href="?page=Home" >&raquo;Navigate</a></div>
		<ul>
			<li><a href="?page=Home" >Startpage</a></li>
			<li><a href="?page=OrganismSelect" >Organisms</a></li>
			<li><a href="?page=SubsystemSelect" >Browse Subsystems</a></li>
			<li><a href="https://rast.nmpdr.org/SubsysEditor.cgi" >Curate Subsystems</a></li>
			<li><a href="?page=Scenarios" >Scenarios</a></li>
			<li><a href="?page=FigFamViewer" >FigFams</a></li>
			<li><a href="?page=BlastRun" >BLAST Search</a></li>
			<li><a href="http://www.theseed.org/daily" >Visit the Daily SEED</a></li>
			<li><a href="?page=SubsystemPrimer" >Request a Subsystem</a></li>
		</ul>
		</li>
		<li><div><a  >&raquo;Organism</a></div>
		<ul>
			<li><a href="?page=Organism&organism=1041826.106" >General Information</a></li>
			<li><a href="?page=BrowseGenome&tabular=1&organism=1041826.106" >Feature Table</a></li>
			<li><a href="?page=BrowseGenome&organism=1041826.106" >Genome Browser</a></li>
			<li><a href="?page=Scenarios&organism=1041826.106" >Scenarios</a></li>
			<li><a href="?page=SubsystemSelect&organism=1041826.106" >Subsystems</a></li>
			<li><a href="?page=Export&organism=1041826.106" >Export</a></li>
		</ul>
		</li>
		<li><div><a  >&raquo;Comparative Tools</a></div>
		<ul>
			<li><a href="?page=CompareMetabolicReconstruction&organism=1041826.106" >Function based Comparison</a></li>
			<li><a href="?page=MultiGenomeCompare&organism=1041826.106" >Sequence based Comparison</a></li>
			<li><a href="?page=Kegg&organism=1041826.106" >Kegg Metabolic Analysis</a></li>
			<li><a href="?page=BlastRun&organism=1041826.106" >BLAST search</a></li>
		</ul>
		</li>
		<li><div><a href="http://www.theseed.org" target="help">&raquo;Help</a></div>
		<ul>
			<li><a href="http://www.theseed.org/wiki/index.php/Home_of_the_SEED" >What is the SEED</a></li>
			<li><a href="http://www.theseed.org/wiki/index.php/SEED_Viewer_Tutorial" >HowTo use the SEED Viewer</a></li>
			<li><a href="http://www.theseed.org/wiki/index.php/RAST_Tutorial" >Submitting Data to SEED</a></li>
			<li><a href="mailto:rast@mcs.anl.gov" >Contact</a></li>
			<li><a href="?page=Register" >Register</a></li>
			<li><a href="?page=RequestNewPassword" >I forgot my Password</a></li>
		</ul>
		</li>
	</ul>
</div>

	
      </td>
      <td style="border-right: 2px solid #5DA668; padding: 0px;">

	<div id='search'><form style='margin: 0px; padding: 0px;'><input type='submit' value='find' style='width:40px;cursor: pointer;'><input type='text' name='pattern'><input type='hidden' name='page' value='SearchResult'><input type='hidden' name='action' value='check_search'><input type='submit' style='display:none;'></form></div>

      </td>
      <td style="text-align: right; padding: 0px;">
	
	  <div id='user'><a href='seedviewer.cgi?page=AccountManagement'><img class='imglink' src='https://rast.nmpdr.org/Html/wac_people.png' title='Account Management' /></a>
<a href='seedviewer.cgi?page=Organism&organism=1041826.106&logout=1'><img class='imglink' src='https://rast.nmpdr.org/Html/wac_logout.png' title='Logout' /></a>
Ekaterina Pushmina</div>
	
      </td>
    </tr>
  </table>
</div>


 

  
  
  <div id="content">
    <script>function initialize_all () {

}</script><style>.hideme { display: none; }</style><h2>Organism Overview for Flavobacterium columnare ATCC 49512 (1041826.106)</h2><table><tr><td width=550px><div><table><tr><th style='width: 170px;'>Genome</th><td>Flavobacterium columnare ATCC 49512 (Taxonomy ID: <a href='http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=1041826&lvl=3&lin=f&keep=1&srchmode=1&unlock' target=outbound>1041826</a>)</td></tr><tr><th>Domain</th><td>Bacteria</td></tr><tr><th>Taxonomy</th><td>Bacteria; Bacteroidetes; Flavobacteriia; Flavobacteriales; Flavobacteriaceae; Flavobacterium; Flavobacterium columnare ATCC 49512 ; Flavobacterium columnare ATCC 49512</td></tr><tr><th>Neighbors</th><td><a href='seedviewer.cgi?page=ClosestNeighbors&organism=1041826.106'>View closest neighbors</a></td></tr><tr><th>Size</th><td>3,162,432</td></tr><tr><th>GC Content</th><td>31.5</td></tr><tr><th>L50</th><td>1</td></tr><tr><th>Number of Contigs (with PEGs)</th><td>1</td></tr><tr><th>Number of Subsystems</th><td>225</td></tr><tr><th>Number of Coding Sequences</th><td>2833</td></tr><tr><th>Number of RNAs</th><td>84</td></tr></table></div></td><td>For each genome we offer a wide set of information to browse, compare and download.<br><br><input type='hidden' id='OrganismInfoContent' value='0' /><table style='height: 125px; width: 325px;  border-spacing: 0px;border-collapse: expression("separate", cellSpacing = "0px");' id='tab_view_table_0'><tr><td class='tab_view_title_selected' name='0_tabs' id='0_tab_0' onclick='tab_view_select("0", "0");'>Browse</td><td class='tab_view_title' name='0_tabs' id='0_tab_1' onclick='tab_view_select("0", "1");'>Compare</td><td class='tab_view_title' name='0_tabs' id='0_tab_2' onclick='tab_view_select("0", "2");'>Download</td><td class='tab_view_title' name='0_tabs' id='0_tab_3' onclick='tab_view_select("0", "3");'>Annotate</td><td class='tab_view_title_filler'>&nbsp;</td></tr><tr><td colspan='5' class='tab_view_content_td'><div style='height: 100%; width: 100%;' class='tab_view_content_selected' name='0_contents' id='0_content_0'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">Browse through the features of <a href="seedviewer.cgi?page=BrowseGenome&organism=1041826.106">Flavobacterium columnare ATCC 49512</a> both graphically and through a table. Both allow quick navigation and filtering for features of your interest. Each feature is linked to its own detail page.<br><br>Click <a href="seedviewer.cgi?page=BrowseGenome&organism=1041826.106">here</a> to get to the Genome Browser</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_1'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">Compare the metabolic reconstruction of this organism to that of another organism.<br><br>Available comparisons are <a href="seedviewer.cgi?page=CompareMetabolicReconstruction&organism=1041826.106">function based</a>, <a href="?page=MultiGenomeCompare&organism=1041826.106">sequence based</a> or via <a href="seedviewer.cgi?page=Kegg&organism=1041826.106">KEGG</a>. You can also <a href="seedviewer.cgi?page=BlastRun&organism=1041826.106">BLAST</a> against this organism.</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_2'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">You can export all information about this organism (e.g. annotations, scenarios, subsystems) into a variety of formats (e.g. EMBL, Excel) for further analysis on your own system.<br><br>Click <a href="seedviewer.cgi?page=Export&organism=1041826.106">here</a> to get to the Export page.</div></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='0_contents' id='0_content_3'><div style="padding-left: 5px; padding-right: 5px; text-align: justify;">The SeedViewer has several options to edit the annotations of your private genome. To find out how to use these capabilities and enable them for your genomes, click <a href="?page=HowToAnnotate">here</a>.</div></div></td></tr></table>
<img src='https://rast.nmpdr.org/Html/clear.gif'  onload='initialize_tab_view("0"); tab_view_select("0", "0");'>
</td></tr></table><h2>Subsystem Information</h2><input type='hidden' id='SubsystemTabView' value='1' /><table style='height: 200px; width: 95%px;  border-spacing: 0px;border-collapse: expression("separate", cellSpacing = "0px");' id='tab_view_table_1'><tr><td class='tab_view_title_selected' name='1_tabs' id='1_tab_0' onclick='tab_view_select("1", "0");'>Subsystem Statistics</td><td class='tab_view_title' name='1_tabs' id='1_tab_1' onclick='tab_view_select("1", "1");'>Features in Subsystems</td><td class='tab_view_title_filler'>&nbsp;</td></tr><tr><td colspan='3' class='tab_view_content_td'><div style='height: 100%; width: 100%;' class='tab_view_content_selected' name='1_contents' id='1_content_0'><table style='padding-top: 25px;'><tr><td><b>Subsystem Coverage</b></td><td align=center><b>Subsystem Category Distribution</b></td><td align=center><b>Subsystem Feature Counts</b></td></tr><tr><td style='padding-right: 10px;'><img src="data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAAGQCAMAAAAOUBefAAAAolBMVEX///8AAADIyMjrBSg8PL4y/zLghQ8AAJv/3QAAm5vIZMgbhTSHQUEAWv9fZGRQ0pbh+qCqHpH/132MpdKgD/otm7mqzXgePL5zr6DChQ8AAH3h3QD/AAAA/wAAAP///wAA////AP//fwD/AH8A/39/AP9//wAAf///f39//39/f/8AAH9/AAAAfwB/fwB/AH8Af3//f////39///9QdJpdpmhNMpuCAAABG0lEQVR4nO3cUQqDMBQEwOed9P5XazRatZRa8/wodlZIQGHYJP4JRhylO5dDL7rhTIBAIBAIBAKBwJ8Gy/My5MCYkHEeh/HKgVOlmLtN49ZrB9dikQTX1cYbrwmMzR6+em2Hsp1j8VN7uFt3JF+bevM5DemGBwECgUAgEAgE3h68/FtAfyZAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAj8CizPy5AC6806j1a5cmBF+2pODbdeGzhjc7W4GNx7TeBSLNaWV4BL03rQGXDduXo2kXxtloLzivt8w88BAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgbcBr/4LuoiIiMgf5wHPlBRRr6t4iwAAAABJRU5ErkJggg==" usemap="#imap_barchart_0"><map name='imap_barchart_0'>
<area shape='rect' coords='33,76,81,370' onmouseover='hover(event, "bar_0", "2");'onclick='window.top.location="?page=BrowseGenome&organism=1041826.106&tabular=1&noss=1"' style='cursor: pointer;'>
<area shape='rect' coords='33,0,81,76' onmouseover='hover(event, "bar_1", "2");'onclick='tab_view_select("1", 1);' style='cursor: pointer;'>
</map><input type='hidden' id='hover_redundancies_2' value='bar_1~#bar_0'><span style='display: none;' id='tooltip_2_bar_0'><table><tr><th>not in Subsystem</th></tr><tr><td>total (2246)</td></tr><tr><td>non-hypothetical (970)</td></tr><tr><td>hypothetical (1276)</td></tr></table></span>
<span style='display: none;' id='tooltip_2_bar_1'><table><tr><th>in Subsystem</th></tr><tr><td>total (587)</td></tr><tr><td>non-hypothetical (574)</td></tr><tr><td>hypothetical (13)</td></tr></table></span>
</td><td style='padding-right: 10px;'><img style="border: none;" src="data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAZAAAAGQCAMAAAC3Ycb+AAABDlBMVEX///8AAADIyMjrBSg8PL4y/zL/AAAAgAAAAICAgACAAIDAwMCAgICZmf+ZM2b//8xmAGb/gIAAZszMzP//AP///wAA//+AAAAAgIAAAP8AzMzM///M/8z//5mZzP//mczMmf//zJkzZv8zzMyZzAD/zAD/mQD/ZgBmZpmWlpYAM2YzmWYAMwAzMwCZMwAzM5kzMzPghQ8AAJv/3QAAm5vIZMgbhTSHQUEAWv9fZGRQ0pbh+qCqHpH/132MpdKgD/otm7mqzXgePL5zr6DChQ8AAH3h3QAA/wD/fwD/AH8A/39/AP9//wAAf///f39//39/f/8AAH9/AAAAfwB/fwB/AH8Af3//f////39////pA/ySAAAPlklEQVR4nO3cSXZbSQ6F4Tcy7awsWxyk69h+VO5/l0Wx02uiASIAXCDIfwXi/Q6CHGmaQvbvri/b0H/iM7RnKIC8aHQrYVRAXiii1SRoIC8XmYgaZJCXSU90DQ7Iy4QfS6IB5OXCqAWjEeSFUq1Vox3kZZKvQ6ML5GWSqA+jG+SFsqnfox/kRfJIgEME5CXykYiGEMjLREpDDuSpTQQ5JEGelUSUQxbkCUmENcRBnstEXkMD5HlMVDxUQJ6CRIdDC2R0ES0NPZCxTRQ9FEGGJdHk0AUZU+T9PS7IgCTv77FBhhJ5vxcZZCCU92FAxhB5fzcRMQEZQeR9KJDwJO+b4oPEFtl6jAASmGTHoSliCBJVJOUxBkhIkiSHoogtSDyRnMcoIMFIshx6IuYgkUROp2cACUNyOhVBlEQQIDFETqeKyEAgAUhO96xFQCDeRU4UEBURFIhrktOypwHxKzLPQBEgiFeReSaLDAbiU2RmgCiIQEEcksy3YCcCBvEmMs9oETSIL5F5Tos8E4gnkTkHYiiC5vjiSGSeG0QGBPEiMm+DnAga4xKa4tLOgygyIogDkgQHRgQtcc+lB+1rZEwQsEgGBHAiaIdHLj1oImOCIEXyHvYiaIZlDjnWIs8GghGpeFifCNpgnUcQYxE0wTqPHsaPFppgk0cPWxG0wC5/HKRHa1wQSxGih6kIev5EDkEIj9YLxNLDUAS9fiqHHgSRgUFsRHgeZieC3j6dPw+zE0FPn8mfh5UIevlc/jyWIk8IoivS5GEjgt49n2uQtMgLxNbDRAQ9eyF/HtVHa2wQLZEODwMR9OjF/HnoP1rozcv5A1E/EfTk5fx5qIugJ6/kz6MmMjiIuIgAyKx6IujBazn00D0R9ODV/HnoiqD3rucQpCzyAjH3qHyNDA4iJyLnoXgi6LUpOQTRE0GPTcmhR/nRGh1ERkTWQ00EvTUtjyAlkRcIwEPpRNBTE3PooXQi6KWpeQRREUEPTc2jh4oIemhqB48enyLPBnI45xpETgQ9NalDJ4iaR0lkXJDDLY8e8o8Weu16h0OniKpHQWRMkMMy1yBCIujByx0O/kGETwQ9ebHDNo8ewiLozQvtOLyCZEXGAkl4tIAYeIiKoGfPleJoEnmBSJThaAAx8ZAUQU+fLOvBFjHyyIoMAVLgYIuYgcxSJ4Jef1eZgwli5yF2Iuj9t9U8eCKGIFIiaIB1dQ4WiKWH1KOFJlhF8eCI2IJkROKC0DgYIMYeMiJohUdUDoYICuQ0AgjDgwpi7iFyImiIaxwOsggAJC0SD4TpQQNBeAiIoC2+8DmIIhiQOTxIA0cAkHaRkB4UEJBHWiQMSBsHSQQG0vloxfRwDZISCQHSzkEQAXr0PVoxOXyDpETcg3R61ESgHl2PVlAO5yAJEc8gAhwVELRHh0hUj7IImiMSiBDH4atvkGaRmB5fz/n2aBaJylEUQVtc8w8iqVECQUvcaxIJ5vF1WQiQk1cQaY08CNrhsxaRMB5bjbwImmGRWxAVjgAgDSIRONIaORG0warto+UCREsjAgj/RCJzRADZisBBFDXSIGiAXUyRyBxJEfT++/yAaGsEAqGLePSgaqRA0OunWougQCw4EiLo8ZNxRJx58DSigMxwECONUCBEkdgcOxH08rnoIk48GjXCgNAfreAcYUDIIniOHo1AICsRMxBrjaggBZHgHIFAiCeC85DQCAVCEwnOsRFBT15pIaINgtKIBTITTsTeQ1Tj69dvwUBqJxKa49tHgTwoIqYe0hbfNiLouQnpgwA0vq36TyiQqkgwjo3FpVAei0dLA8RUI2GxFkFvTatyIkE4shjxQObyiWh7KFtEBCmLqHKIYiQtQoIURdxy0CxeIFQPK4yQICURFQ4hDILFEgS9Mqe7iAiIGkeDxVIEPTKr7IlIc/Rj8CzigmROxAFH82FEBsmKCHqYH8ZaBL0wNwEQWQ0hi7AgGREMh6BFXJCbSCuImIY0RmSQhEgvB9wiMEjyRPo8GjCkLSKDpEQ6OHxYhAa5ijBB+jj0MUKDzLsTafNgYehaRAfZnkgDhy+Lj/6OC7ITYXtQMQwYPovrcX20iCB8DYPD+DtZYJCNiByHqkWaYRCQlQjZo45hzzAGyEWkBkLXULCgO4wBsnq0KB42GA0Mo4DMpzIIRUPKoothGJCFSJkjj+GC4SlAKhw9FuIM44B8imQ5xA5DkeHRf8ODnEUSIHkOtoXi+Knig8y3E0l5tFuYjD8oyLwFSWnQLIzHHxXk+mhtPBiHARt/VJD5tABZa5QsHKw/Ksjl0XpwPDC6GCzGHxdkvoHcOXYWPsdPNQjI6QPkqvGtzIAevNYgIOcT+XK4WcQZP9kgIPO/6CGlCg3y59Lv379//fp5fPvxAz2mRPFA/jz6fe3Xub9+Ho9vZ5IfP76jF+0ryHfIn02/H31o/PrfXz8vIBeRs0lgFMcgW4QtxV3j7HEG+ed4EbmRfKDEVHEGkkbYS3xi3DzOB3ITeZDERPEAUlBIUqw0bh5XkKvIgiQeCgqkhpCl2GhcPLYga5IPkzgqliA0hILEXuPMcfW4gTxE1iSBUNRBGAoVir3GzWMJkhW5o3hXUQHhItQlkhoXj9uB/HPciSRIAqCIgTQhECnSGkuPJEia5I7iVKUPpEOBTJHTuHtsQVYiGZK7iUMUPkgvAkeioHHheHj8PB4zIjkSryg0EBEENkUeY+GRAKGKLFD8qORBBBEaKMoaO48SSJnEG8oGRFyhRaKqcfP4fLCWXyEpkQrJAgWucgZRQWinqGssPDIge5EqycIEijJNrigoGjeOpccOJCFSJ3GBMkmLtEvQNFYeLBCKyBoFoDIJgvRREDXSHnuQpAiRBIkiA9JNQdbIePzceWREyCQrE0OVThAJCQbGpwcBJCNCJ4GgNIMIUfA0sh77FysPwiIxR+GDyEmwNR4cV48aSF6ER7JFUVXhgMhSsDUKHhkQORE7FBqIOEWDRskjB1IQaSDZmqioVEA0JNo0dh69IE0k+ihZEC2KRo2FR+JAsiBlkTYSXZRpL6JI0ayx4OAcSFWklSSBIqSyBNGV6NCoeZRA1ER0UK7/+92AogMj6SEG0keSMOlTuYL41qh7FEHqIn0ksiiThUifBsGjDEIQ6SVJozSoTPogvRpLjkYQikg/iQiKNki3BtFDAERCJGPCQFEFEdDIeHBBaCIyJDkUmooeiIgG0aMOQhSRImlHUQIR0qB61L5CGCJyJFmUoooGiJjGmuPTQxVElISPMgmLyGHsPAoHQgFhiMiSZE1SKJMoiKhG3qMRBCdSQvmuBiKswfMggXBE5EloKJOQiLgGy0MDRIWkhPJ949EBoqCx5Vh4dIAwRXRIiigCIBoaXA8yCFdEiyRrMnWK6GjsPaogRA8/IhmULhAtDb6HIogqyQ5lagbRw2jyoIM0iOiSrFAaQVQ19hxLj36QFhFtkh89ILoaTR48kCYRdZIrChtEW6PmgQMxEDnHA9HXaPRggjSKWJBMdBELjZQH4cFig7SK6JNQQWw0UhykA2H86u0UUSaZSCBGGj0ebJB2EVUSAoiZBsHDB4gmyRZkgmF0ebSAuBTZeUwwjS6PJpAuESWSPIi1RpJDGaRPRIVkDzJBNHo9ICAKJAmPCaFB8ygeSBNIt4g0SQokokfjgfgTSYEgRHo92kH6RURJnICQPNyCCJIkPcxB0hysA+kAERGRIkmDGIsQPdRAZERESDIetiBCHh0gfkRyIJYiIh4uQPpJsh6GIFQPVRAxkU6SPIiVSI6D69ELIifSQ1LwMALJejSAdHlIirSTwEHkPFyBNJOUQCxEBD36QRyIFD0MQBgehAPpBpEVaSEpg2iL5DkaPERAhEX4JFAQlkdMEC5JxUNXpODRciAyIPIiLBIkiLSHEAhUpOqhCMLzoIFIeGiIkEnqIFoiJY5mD78gRBKChxII18MUREeEQgID0fAQBFESqZNQQDREih6ND5YoCEiE5KEAwvYYBqRCQgORFilztHuIguiJFEiIHsIgDR7kAxEEURTJklBBREXUPGQPRFckTUL2kARp8RgQJElCB5ETqXj0HIg4iLUIw0MKpMbR5SEPoiyyJeGAyIi0eQBBtEXemj1EQJQ9IoIsSXggAiKNHpwDEQcxEHlr8+gX0fbQATETYXt0glQ5uh8sjRfLSOStCaRLpNnjOUDe3ho8ekAsPLRAbERaQNpF2j08gJiImIKYeSiBGIg0eTSCEDgkPJ4RpEmE4uEdRF2k0aNFxMxDFURZpNmDL9Ll4QhEVaTDgwti6KH3I8s1CEuExCHooQmiKNLlwRHp9HAGoibS6UEHsfXQB9ES6QWhitA8IoHoiHR7EEWsPaKCCHiQQMw9LEA0RCRA6iJEjoKHTxB5ERGPqsiwHuIiQh4VEAGPF4igCNVD9ECsQGRFxDxKIhgPMxBJEUGPLAiZQ/bBMgQRFJEEyYjAPCKCiHqkRWQ8Wg/ECERKRNgjBYLzMAUREpEG2YvQPcQfLMMXS0pE3GMrwuCQP5B4IAoekyMPYxABEQ2QScMjBki3iIrHQsSDhylIp4iSx0ME7RENRM3jKsLhGAakR0TRY/LhgQDpENEEmUQ9ukCMPdpFVD2myYNHKBBlD5aIlgcGpE1E3YMhUvHoOxAASIuIgQddRPVAECB8ERMPqojagcQBMfKgieh54ECYImYeFBBFDyAIT8QOpC5S84gKwhEx9KiSDOvBALH1qIiMC0IVseYoi6h6gEFoIgCPAoiuBxqEJIIAyYpUPcYHgXjkSLQ94CB1EZRHUqTuIXEgUJCKCI4jKWJzIFiQogjUYw+ifiD4F8s1yFZE38MFSEEE7LEhMfDwAZIVQWt8xPIYBSQtgqa4ZerhGQQN8cj8QByAJETQDIusD8QDyE4EjbDKzMMRyNGxx1mE4iEGgqa45tnjnI2HJ5Cjaw+KyGggR88cU11EzMMNyNG1R01EwsMlCHr2Us8GcnTuURIR8XAHckQPXu+5QNBrU9L08PUjKwTHR3oevkDQO9PTPRAnIOiReWkeiA8Q9MLcBgdBz9uSgocbEPS2bcl7OAFBD9vemCDoVXtS8cCCoCftbbQDQe/Zn6AHHgQ9pkgDgaCnFGoYEPSQkoUHQQ8on5gHBAS9nkaBQdDTaRUVBL2bXiIgLw7RgoGg57IoEgh6K5vCgKCHMqzDwwoEPZF5rkHQ42ByC4IeBpdLEPQo4NggLw313ICgh/CTjxcLvYKnHICgJ/AXEAT90d0GAUF/aOcZg6A/boyMQNAfM1xqHugPFjgFEPRHCp8oCPrDDFQnCPrPH7NGEPSf/Qy9GLw3zP7/B8JLelFFXzLFAAAAAElFTkSuQmCC" usemap="#imap_piechart_0"><map name='imap_piechart_0'><area shape='poly' coords='199,199,199,0,216,0,233,3,250,6,267,12,283,18,298,26,313,35,326,46,339,58,351,71,362,84,368,94' id='pie_0_slice_0'  onmouseover='hover(event, "pie_0_slice_0", "0");'>
<area shape='poly' coords='199,199,368,94,376,110,384,125,385,129' id='pie_0_slice_1'  onmouseover='hover(event, "pie_0_slice_1", "0");'>
<area shape='poly' coords='199,199,385,129,390,145,393,156' id='pie_0_slice_2'  onmouseover='hover(event, "pie_0_slice_2", "0");'>
<area shape='poly' coords='199,199,393,156,394,160' id='pie_0_slice_3'  onmouseover='hover(event, "pie_0_slice_3", "0");'>
<area shape='poly' coords='199,199,394,160,396,177,397,181' id='pie_0_slice_5'  onmouseover='hover(event, "pie_0_slice_5", "0");'>
<area shape='poly' coords='199,199,397,181,397,190' id='pie_0_slice_6'  onmouseover='hover(event, "pie_0_slice_6", "0");'>
<area shape='poly' coords='199,199,397,190,397,207,396,224,393,241,392,244' id='pie_0_slice_7'  onmouseover='hover(event, "pie_0_slice_7", "0");'>
<area shape='poly' coords='199,199,392,244,390,253' id='pie_0_slice_8'  onmouseover='hover(event, "pie_0_slice_8", "0");'>
<area shape='poly' coords='199,199,390,253,385,269,378,285,372,296' id='pie_0_slice_9'  onmouseover='hover(event, "pie_0_slice_9", "0");'>
<area shape='poly' coords='199,199,372,296,363,311,352,325,341,338,328,350,316,359' id='pie_0_slice_10'  onmouseover='hover(event, "pie_0_slice_10", "0");'>
<area shape='poly' coords='199,199,316,359,302,369,287,377,271,384,254,390,237,394,220,396,203,397,186,397,172,396' id='pie_0_slice_11'  onmouseover='hover(event, "pie_0_slice_11", "0");'>
<area shape='poly' coords='199,199,172,396,165,395' id='pie_0_slice_15'  onmouseover='hover(event, "pie_0_slice_15", "0");'>
<area shape='poly' coords='199,199,165,395,148,391,131,386,115,379,100,371,94,368' id='pie_0_slice_16'  onmouseover='hover(event, "pie_0_slice_16", "0");'>
<area shape='poly' coords='199,199,94,368,80,358,68,349' id='pie_0_slice_17'  onmouseover='hover(event, "pie_0_slice_17", "0");'>
<area shape='poly' coords='199,199,68,349,55,337,44,324' id='pie_0_slice_18'  onmouseover='hover(event, "pie_0_slice_18", "0");'>
<area shape='poly' coords='199,199,44,324,34,310,25,296,17,280,11,264,9,258' id='pie_0_slice_20'  onmouseover='hover(event, "pie_0_slice_20", "0");'>
<area shape='poly' coords='199,199,9,258,4,242,2,228' id='pie_0_slice_21'  onmouseover='hover(event, "pie_0_slice_21", "0");'>
<area shape='poly' coords='199,199,2,228,0,213' id='pie_0_slice_22'  onmouseover='hover(event, "pie_0_slice_22", "0");'>
<area shape='poly' coords='199,199,0,213,0,196,0,179,3,161,7,145,12,128,19,112,27,97,37,82,48,69,60,56,73,44,78,40' id='pie_0_slice_23'  onmouseover='hover(event, "pie_0_slice_23", "0");'>
<area shape='poly' coords='199,199,78,40,87,34' id='pie_0_slice_24'  onmouseover='hover(event, "pie_0_slice_24", "0");'>
<area shape='poly' coords='199,199,87,34,102,25,106,22' id='pie_0_slice_25'  onmouseover='hover(event, "pie_0_slice_25", "0");'>
<area shape='poly' coords='199,199,106,22,122,15,138,9,155,4,172,1,189,0,194,0' id='pie_0_slice_26'  onmouseover='hover(event, "pie_0_slice_26", "0");'></map><input type='hidden' id='hover_redundancies_0' value='pie_0_slice_21~#pie_0_slice_18~#pie_0_slice_2~#pie_0_slice_24~#pie_0_slice_8~#pie_0_slice_0~#pie_0_slice_11~#pie_0_slice_22~#pie_0_slice_17~#pie_0_slice_20~#pie_0_slice_10~#pie_0_slice_23~#pie_0_slice_3~#pie_0_slice_1~#pie_0_slice_9~#pie_0_slice_5~#pie_0_slice_26~#pie_0_slice_6~#pie_0_slice_15~#pie_0_slice_25~#pie_0_slice_16~#pie_0_slice_7'><span style='display: none;' id='tooltip_0_pie_0_slice_0'><table><tr><th>Cofactors, Vitamins, Prosthetic Groups, Pigments</th></tr><tr><td>137 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_1'><table><tr><th>Cell Wall and Capsule</th></tr><tr><td>26 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_2'><table><tr><th>Virulence, Disease and Defense</th></tr><tr><td>19 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_3'><table><tr><th>Potassium metabolism</th></tr><tr><td>3 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_5'><table><tr><th>Miscellaneous</th></tr><tr><td>14 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_6'><table><tr><th>Phages, Prophages, Transposable elements, Plasmids</th></tr><tr><td>6 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_7'><table><tr><th>Membrane Transport</th></tr><tr><td>37 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_8'><table><tr><th>Iron acquisition and metabolism</th></tr><tr><td>6 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_9'><table><tr><th>RNA Metabolism</th></tr><tr><td>32 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_10'><table><tr><th>Nucleosides and Nucleotides</th></tr><tr><td>57 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_11'><table><tr><th>Protein Metabolism</th></tr><tr><td>103 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_15'><table><tr><th>Secondary Metabolism</th></tr><tr><td>5 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_16'><table><tr><th>DNA Metabolism</th></tr><tr><td>51 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_17'><table><tr><th>Fatty Acids, Lipids, and Isoprenoids</th></tr><tr><td>22 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_18'><table><tr><th>Nitrogen Metabolism</th></tr><tr><td>23 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_20'><table><tr><th>Respiration</th></tr><tr><td>51 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_21'><table><tr><th>Stress Response</th></tr><tr><td>21 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_22'><table><tr><th>Metabolism of Aromatic Compounds</th></tr><tr><td>10 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_23'><table><tr><th>Amino Acids and Derivatives</th></tr><tr><td>134 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_24'><table><tr><th>Sulfur Metabolism</th></tr><tr><td>7 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_25'><table><tr><th>Phosphorus Metabolism</th></tr><tr><td>15 / 844</td></tr></table></span>
<span style='display: none;' id='tooltip_0_pie_0_slice_26'><table><tr><th>Carbohydrates</th></tr><tr><td>62 / 844</td></tr></table></span>
</td><td>
<div id='tree_0'>
<div name='tree_level_0' id='tree_node_0_0'><img id='tree_img_0_0' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "0");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/AAAZ4gk3AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cofactors, Vitamins, Prosthetic Groups, Pigments (137)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_0'>
<img id='tree_img_0_1' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "1");'>&nbsp;Biotin (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_1'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=1041826.106'>Biotin biosynthesis</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=1041826.106'>Biotin biosynthesis Experimental</a> (4)<br></div>
<img id='tree_img_0_4' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "4");'>&nbsp;Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_4'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=1041826.106'>Thiamin biosynthesis</a> (7)<br></div>
<img id='tree_img_0_6' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "6");'>&nbsp;Quinone cofactors (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_6'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis&organism=1041826.106'>Menaquinone and Phylloquinone Biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo&organism=1041826.106'>Menaquinone and Phylloquinone Biosynthesis -- gjo</a> (6)<br></div>
<img id='tree_img_0_9' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "9");'>&nbsp;Tetrapyrroles (20)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_9'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106'>Cobalamin synthesis</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106'>Heme and Siroheme Biosynthesis</a> (10)<br></div>
<img id='tree_img_0_12' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "12");'>&nbsp;Riboflavin, FMN, FAD (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_12'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=1041826.106'>riboflavin to FAD</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=1041826.106'>Riboflavin, FMN and FAD metabolism</a> (8)<br></div>
<span style='cursor: pointer;'>&nbsp;Fe-S clusters (0)<br>
<span style='cursor: pointer;'>&nbsp;Mycofactocin (0)<br>
<img id='tree_img_0_17' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "17");'>&nbsp;Pyridoxine (11)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_17'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=1041826.106'>Pyridoxin (Vitamin B6) Biosynthesis</a> (11)<br></div>
<img id='tree_img_0_19' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "19");'>&nbsp;NAD and NADP (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_19'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106'>NAD and NADP cofactor biosynthesis global</a> (12)<br></div>
<span style='cursor: pointer;'>&nbsp;Coenzyme B (0)<br>
<img id='tree_img_0_22' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "22");'>&nbsp;Folate and pterines (41)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_22'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106'>Folate biosynthesis cluster</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106'>Folate Biosynthesis</a> (12)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106'>5-FCL-like protein</a> (14)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=1041826.106'>Pterin synthesis related cluster</a> (6)<br></div>
<span style='cursor: pointer;'>&nbsp;Lipoic acid (0)<br>
<span style='cursor: pointer;'>&nbsp;Coenzyme F420 (0)<br>
<span style='cursor: pointer;'>&nbsp;Coenzyme M (0)<br>
<img id='tree_img_0_30' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "30");'>&nbsp;Coenzyme A (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_30'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=1041826.106'>Coenzyme A Biosynthesis cluster</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106'>Coenzyme A Biosynthesis</a> (9)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_33'><img id='tree_img_0_33' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "33");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAgACc+aWRAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cell Wall and Capsule (26)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_33'>
<img id='tree_img_0_34' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "34");'>&nbsp;Capsular and extracellular polysacchrides (18)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_34'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=1041826.106'>dTDP-rhamnose synthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=1041826.106'>CMP-N-acetylneuraminate Biosynthesis</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=1041826.106'>Rhamnose containing glycans</a> (10)<br></div>
<img id='tree_img_0_38' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "38");'>&nbsp;Gram-Negative cell wall components (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_38'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lipoprotein_sorting_system&organism=1041826.106'>Lipoprotein sorting system</a> (1)<br></div>
<img id='tree_img_0_40' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "40");'>&nbsp;Cell Wall and Capsule - no subcategory (6)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_40'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=1041826.106'>Murein Hydrolases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=1041826.106'>Recycling of Peptidoglycan Amino Acids</a> (3)<br></div>
<img id='tree_img_0_43' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "43");'>&nbsp;Gram-Positive cell wall components (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_43'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Teichuronic_acid_biosynthesis&organism=1041826.106'>Teichuronic acid biosynthesis</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Cell wall of Mycobacteria (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_46'><img id='tree_img_0_46' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "46");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAAIBKwr76AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Virulence, Disease and Defense (19)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_46'>
<span style='cursor: pointer;'>&nbsp;Adhesion (0)<br>
<span style='cursor: pointer;'>&nbsp;Toxins and superantigens (0)<br>
<span style='cursor: pointer;'>&nbsp;Bacteriocins, ribosomally synthesized antibacterial peptides (0)<br>
<img id='tree_img_0_50' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "50");'>&nbsp;Resistance to antibiotics and toxic compounds (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_50'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=1041826.106'>Copper homeostasis</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=1041826.106'>Cobalt-zinc-cadmium resistance</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=1041826.106'>Resistance to fluoroquinolones</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Beta-lactamase&organism=1041826.106'>Beta-lactamase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=1041826.106'>Multidrug Resistance Efflux Pumps</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Virulence, Disease and Defense - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Detection (0)<br>
<img id='tree_img_0_58' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "58");'>&nbsp;Invasion and intracellular resistance (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_58'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=1041826.106'>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=1041826.106'>Mycobacterium virulence operon involved in DNA transcription</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=1041826.106'>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=1041826.106'>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a> (3)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_63'><img id='tree_img_0_63' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "63");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAgAB9zL4RAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Potassium metabolism (3)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_63'>
<img id='tree_img_0_64' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "64");'>&nbsp;Potassium metabolism - no subcategory (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_64'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=1041826.106'>Potassium homeostasis</a> (3)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_66'><img id='tree_img_0_66' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "66");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAICr96V6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Photosynthesis (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_66'>
<span style='cursor: pointer;'>&nbsp;Light-harvesting complexes (0)<br>
<span style='cursor: pointer;'>&nbsp;Photosynthesis - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Electron transport and photophosphorylation (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_70'><img id='tree_img_0_70' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "70");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXAwMBmS75kAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Miscellaneous (14)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_70'>
<span style='cursor: pointer;'>&nbsp;Conversion of Succinyl-CoA to Propionyl-CoA (0)<br>
<img id='tree_img_0_72' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "72");'>&nbsp;Plant-Prokaryote DOE project (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_72'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=1041826.106'>Iron-sulfur cluster assembly</a> (8)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Single-Rhodanese-domain_proteins&organism=1041826.106'>Single-Rhodanese-domain proteins</a> (1)<br></div>
<img id='tree_img_0_75' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "75");'>&nbsp;Miscellaneous - no subcategory (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_75'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phosphoglycerate_mutase_protein_family&organism=1041826.106'>Phosphoglycerate mutase protein family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DedA_family_of_inner_membrane_proteins&organism=1041826.106'>DedA family of inner membrane proteins</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Muconate_lactonizing_enzyme_family&organism=1041826.106'>Muconate lactonizing enzyme family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=1041826.106'>Broadly distributed proteins not in subsystems</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_80'><img id='tree_img_0_80' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "80");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAgICQdD0xAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Phages, Prophages, Transposable elements, Plasmids (6)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_80'>
<span style='cursor: pointer;'>&nbsp;Phage family-specific subsystems (0)<br>
<span style='cursor: pointer;'>&nbsp;Transposable elements (0)<br>
<img id='tree_img_0_83' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "83");'>&nbsp;Phages, Prophages (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_83'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=IbrA_and_IbrB:_co-activators_of_prophage_gene_expression&organism=1041826.106'>IbrA and IbrB: co-activators of prophage gene expression</a> (2)<br></div>
<img id='tree_img_0_85' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "85");'>&nbsp;Phages, Prophages, Transposable elements, Plasmids - no subcategory (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_85'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Integrons&organism=1041826.106'>Integrons</a> (3)<br></div>
<img id='tree_img_0_87' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "87");'>&nbsp;Pathogenicity islands (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_87'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Listeria_Pathogenicity_Island_LIPI-1_extended&organism=1041826.106'>Listeria Pathogenicity Island LIPI-1 extended</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Gene Transfer Agent (GTA) (0)<br>
<span style='cursor: pointer;'>&nbsp;Plasmid related functions (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_91'><img id='tree_img_0_91' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "91");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZmf/YOWB7AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Membrane Transport (37)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_91'>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type II (0)<br>
<span style='cursor: pointer;'>&nbsp;ABC transporters (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VII (Chaperone/Usher pathway, CU) (0)<br>
<img id='tree_img_0_95' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "95");'>&nbsp;Protein translocation across cytoplasmic membrane (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_95'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=1041826.106'>Twin-arginine translocation system</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type V (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type I (0)<br>
<img id='tree_img_0_99' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "99");'>&nbsp;Cation transporters (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_99'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Magnesium_transport&organism=1041826.106'>Magnesium transport</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Transport_of_Nickel_and_Cobalt&organism=1041826.106'>Transport of Nickel and Cobalt</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Copper_Transport_System&organism=1041826.106'>Copper Transport System</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type III (0)<br>
<img id='tree_img_0_104' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "104");'>&nbsp;Uni- Sym- and Antiporters (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_104'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters&organism=1041826.106'>NhaA, NhaD and Sodium-dependent phosphate transporters</a> (1)<br></div>
<img id='tree_img_0_106' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "106");'>&nbsp;Membrane Transport - no subcategory (28)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_106'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=1041826.106'>Ton and Tol transport systems</a> (28)<br></div>
<span style='cursor: pointer;'>&nbsp;TRAP transporters (0)<br>
<span style='cursor: pointer;'>&nbsp;Sugar Phosphotransferase Systems, PTS (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VI (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein secretion system, Type VIII (Extracellular nucleation/precipitation pathway, ENP) (0)<br>
<span style='cursor: pointer;'>&nbsp;Protein and nucleoprotein secretion system, Type IV (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_113'><img id='tree_img_0_113' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "113");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZM2YFAh/4AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Iron acquisition and metabolism (6)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_113'>
<img id='tree_img_0_114' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "114");'>&nbsp;Siderophores (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_114'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=1041826.106'>Siderophore assembly kit</a> (4)<br></div>
<img id='tree_img_0_116' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "116");'>&nbsp;Iron acquisition and metabolism - no subcategory (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_116'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Hemin_transport_system&organism=1041826.106'>Hemin transport system</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Iron transpot (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_119'><img id='tree_img_0_119' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "119");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//8wYFHreAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;RNA Metabolism (32)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_119'>
<img id='tree_img_0_120' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "120");'>&nbsp;RNA processing and modification (17)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_120'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=1041826.106'>mnm5U34 biosynthesis bacteria</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=1041826.106'>Ribonuclease H</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_3'-terminal_phosphate_cyclase&organism=1041826.106'>RNA 3'-terminal phosphate cyclase</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=1041826.106'>RNA processing and degradation, bacterial</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=1041826.106'>tRNA processing</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=1041826.106'>RNA methylation</a> (4)<br></div>
<img id='tree_img_0_127' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "127");'>&nbsp;Transcription (15)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_127'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=1041826.106'>Transcription initiation, bacterial sigma factors</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=1041826.106'>RNA polymerase bacterial</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=1041826.106'>Transcription factors bacterial</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=1041826.106'>Rrf2 family transcriptional regulators</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;RNA Metabolism - no subcategory (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_133'><img id='tree_img_0_133' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "133");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEVmAGZP8U4lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Nucleosides and Nucleotides (57)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_133'>
<img id='tree_img_0_134' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "134");'>&nbsp;Pyrimidines (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_134'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106'>pyrimidine conversions</a> (14)<br></div>
<img id='tree_img_0_136' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "136");'>&nbsp;Purines (35)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_136'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106'>De Novo Purine Biosynthesis</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=1041826.106'>GMP synthase</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106'>Purine conversions</a> (20)<br></div>
<img id='tree_img_0_140' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "140");'>&nbsp;Nucleosides and Nucleotides - no subcategory (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_140'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=1041826.106'>Ribonucleotide reduction</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=dNTP_triphosphohydrolase_protein_family&organism=1041826.106'>dNTP triphosphohydrolase protein family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=1041826.106'>Adenosyl nucleosidases</a> (2)<br></div>
<img id='tree_img_0_144' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "144");'>&nbsp;Detoxification (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_144'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nucleoside_triphosphate_pyrophosphohydrolase_MazG&organism=1041826.106'>Nucleoside triphosphate pyrophosphohydrolase MazG</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=1041826.106'>Nudix proteins (nucleoside triphosphate hydrolases)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=1041826.106'>Housecleaning nucleoside triphosphate pyrophosphatases</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_148'><img id='tree_img_0_148' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "148");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/gIDP2RJcAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Protein Metabolism (103)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_148'>
<img id='tree_img_0_149' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "149");'>&nbsp;Protein folding (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_149'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=1041826.106'>Protein chaperones</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_disulfide_interchange&organism=1041826.106'>Periplasmic disulfide interchange</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Selenoproteins (0)<br>
<img id='tree_img_0_153' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "153");'>&nbsp;Protein biosynthesis (81)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_153'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Val&organism=1041826.106'>tRNA aminoacylation, Val</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Met&organism=1041826.106'>tRNA aminoacylation, Met</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ile&organism=1041826.106'>tRNA aminoacylation, Ile</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Arg&organism=1041826.106'>tRNA aminoacylation, Arg</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=1041826.106'>Translation initiation factors bacterial</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106'>Ribosome SSU bacterial</a> (23)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Gly&organism=1041826.106'>tRNA aminoacylation, Gly</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ala&organism=1041826.106'>tRNA aminoacylation, Ala</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Trp&organism=1041826.106'>tRNA aminoacylation, Trp</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=1041826.106'>Programmed frameshift</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Cys&organism=1041826.106'>tRNA aminoacylation, Cys</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106'>Translation termination factors bacterial</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_His&organism=1041826.106'>tRNA aminoacylation, His</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=1041826.106'>tRNA aminoacylation, Asp and Asn</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=1041826.106'>Translation elongation factors bacterial</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106'>tRNAs</a> (14)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Lys&organism=1041826.106'>tRNA aminoacylation, Lys</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Thr&organism=1041826.106'>tRNA aminoacylation, Thr</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Pro&organism=1041826.106'>tRNA aminoacylation, Pro</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factor_G_family&organism=1041826.106'>Translation elongation factor G family</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=1041826.106'>tRNA aminoacylation, Glu and Gln</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ser&organism=1041826.106'>tRNA aminoacylation, Ser</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Tyr&organism=1041826.106'>tRNA aminoacylation, Tyr</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Leu&organism=1041826.106'>tRNA aminoacylation, Leu</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=1041826.106'>tRNA aminoacylation, Phe</a> (2)<br></div>
<img id='tree_img_0_179' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "179");'>&nbsp;Protein processing and modification (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_179'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein-L-isoaspartate_O-methyltransferase&organism=1041826.106'>Protein-L-isoaspartate O-methyltransferase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=1041826.106'>Signal peptidase</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=1041826.106'>Modification of eukaryotic initiation factor 5A</a> (2)<br></div>
<img id='tree_img_0_183' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "183");'>&nbsp;Protein degradation (12)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_183'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=1041826.106'>Aminopeptidases (EC 3.4.11.-)</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=1041826.106'>Protein degradation</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=1041826.106'>Metallocarboxypeptidases (EC 3.4.17.-)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dipeptidases_(EC_3.4.13.-)&organism=1041826.106'>Dipeptidases (EC 3.4.13.-)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Serine_endopeptidase_(EC_3.4.21.-)&organism=1041826.106'>Serine endopeptidase (EC 3.4.21.-)</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=1041826.106'>Proteolysis in bacteria, ATP-dependent</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_190'><img id='tree_img_0_190' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "190");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAZswGD39gAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Cell Division and Cell Cycle (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_190'>
<span style='cursor: pointer;'>&nbsp;Checkpoint control (0)<br>
<span style='cursor: pointer;'>&nbsp;Heterocyst formation (0)<br>
<span style='cursor: pointer;'>&nbsp;Cell Division and Cell Cycle - no subcategory (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_194'><img id='tree_img_0_194' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "194");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXMzP91giUxAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Motility and Chemotaxis (0)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_194'>
<span style='cursor: pointer;'>&nbsp;Magnetotaxis (0)<br>
<span style='cursor: pointer;'>&nbsp;Motility and Chemotaxis - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Flagellar motility in Prokaryota (0)<br>
<span style='cursor: pointer;'>&nbsp;Social motility and nonflagellar swimming in bacteria (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_199'><img id='tree_img_0_199' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "199");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/AP804Oa6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Regulation and Cell signaling (2)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_199'>
<img id='tree_img_0_200' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "200");'>&nbsp;Regulation and Cell signaling - no subcategory (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_200'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=1041826.106'>Stringent Response, (p)ppGpp metabolism</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Signal transduction in Eukaryotes (0)<br>
<span style='cursor: pointer;'>&nbsp;Quorum sensing and biofilm formation (0)<br>
<span style='cursor: pointer;'>&nbsp;Proteolytic pathway (0)<br>
<span style='cursor: pointer;'>&nbsp;Regulation of virulence (0)<br>
<span style='cursor: pointer;'>&nbsp;Programmed Cell Death and Toxin-antitoxin Systems (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_207'><img id='tree_img_0_207' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "207");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//wCKxvRFAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Secondary Metabolism (5)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_207'>
<span style='cursor: pointer;'>&nbsp;Secondary Metabolism - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Lipid-derived mediators (0)<br>
<span style='cursor: pointer;'>&nbsp;Plant Octadecanoids (0)<br>
<span style='cursor: pointer;'>&nbsp;Bacterial cytostatics, differentiation factors and antibiotics (0)<br>
<span style='cursor: pointer;'>&nbsp;Biosynthesis of phenylpropanoids (0)<br>
<span style='cursor: pointer;'>&nbsp;Hydrocarbons (0)<br>
<span style='cursor: pointer;'>&nbsp;Aromatic amino acids and derivatives (0)<br>
<span style='cursor: pointer;'>&nbsp;UV-absorbing secondary metabolites (0)<br>
<img id='tree_img_0_216' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "216");'>&nbsp;Plant Alkaloids (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_216'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Alkaloid_biosynthesis_from_L-lysine&organism=1041826.106'>Alkaloid biosynthesis from L-lysine</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Biologically active compounds in metazoan cell defence and differentiation (0)<br>
<img id='tree_img_0_219' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "219");'>&nbsp;Plant Hormones (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_219'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=1041826.106'>Auxin biosynthesis</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_221'><img id='tree_img_0_221' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "221");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUA//8ZXC8lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;DNA Metabolism (51)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_221'>
<img id='tree_img_0_222' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "222");'>&nbsp;DNA repair (36)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_222'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=1041826.106'>DNA repair, bacterial MutL-MutS system</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=1041826.106'>DNA repair, UvrABC system</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_photolyase&organism=1041826.106'>DNA repair, bacterial photolyase</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=2-phosphoglycolate_salvage&organism=1041826.106'>2-phosphoglycolate salvage</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=RecA_and_RecX&organism=1041826.106'>RecA and RecX</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=1041826.106'>DNA repair system including RecA, MutS and a hypothetical protein</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106'>DNA repair, bacterial</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=1041826.106'>DNA repair, bacterial RecFOR pathway</a> (8)<br></div>
<img id='tree_img_0_231' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "231");'>&nbsp;CRISPs (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_231'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=CRISPRs&organism=1041826.106'>CRISPRs</a> (1)<br></div>
<img id='tree_img_0_233' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "233");'>&nbsp;DNA Metabolism - no subcategory (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_233'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=1041826.106'>Type I Restriction-Modification</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=1041826.106'>Restriction-Modification System</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=1041826.106'>DNA structural proteins, bacterial</a> (2)<br></div>
<img id='tree_img_0_237' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "237");'>&nbsp;DNA replication (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_237'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=1041826.106'>DNA topoisomerases, Type I, ATP-independent</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=1041826.106'>DNA topoisomerases, Type II, ATP-dependent</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;DNA recombination (0)<br>
<img id='tree_img_0_241' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "241");'>&nbsp;DNA uptake, competence (3)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_241'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=1041826.106'>DNA processing cluster</a> (3)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_243'><img id='tree_img_0_243' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "243");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAICr96V6AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Fatty Acids, Lipids, and Isoprenoids (22)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_243'>
<span style='cursor: pointer;'>&nbsp;Phospholipids (0)<br>
<span style='cursor: pointer;'>&nbsp;Triacylglycerols (0)<br>
<img id='tree_img_0_246' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "246");'>&nbsp;Fatty acids (13)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_246'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106'>Fatty Acid Biosynthesis FASII</a> (13)<br></div>
<span style='cursor: pointer;'>&nbsp;Fatty Acids, Lipids, and Isoprenoids - no subcategory (0)<br>
<img id='tree_img_0_249' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "249");'>&nbsp;Isoprenoids (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_249'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Myxoxanthophyll_biosynthesis_in_Cyanobacteria&organism=1041826.106'>Myxoxanthophyll biosynthesis in Cyanobacteria</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=1041826.106'>Mevalonate Branch of Isoprenoid Biosynthesis</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=1041826.106'>Isoprenoinds for Quinones</a> (3)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_253'><img id='tree_img_0_253' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "253");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWAAABGTyZaAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Nitrogen Metabolism (23)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_253'>
<img id='tree_img_0_254' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "254");'>&nbsp;Nitrogen Metabolism - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_254'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Nitrosative_stress&organism=1041826.106'>Nitrosative stress</a> (1)<br></div>
<img id='tree_img_0_256' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "256");'>&nbsp;Denitrification (22)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_256'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106'>Denitrifying reductase gene clusters</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106'>Denitrification</a> (11)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_259'><img id='tree_img_0_259' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "259");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAgIBxQSaxAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Dormancy and Sporulation (1)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_259'>
<span style='cursor: pointer;'>&nbsp;Spore DNA protection (0)<br>
<img id='tree_img_0_261' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "261");'>&nbsp;Dormancy and Sporulation - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_261'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Sporulation-associated_proteins_with_broader_functions&organism=1041826.106'>Sporulation-associated proteins with broader functions</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_263'><img id='tree_img_0_263' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "263");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAAP+KeNJXAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Respiration (51)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_263'>
<span style='cursor: pointer;'>&nbsp;Biotin (8)<br>
<img id='tree_img_0_265' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "265");'>&nbsp;ATP synthases (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_265'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106'>F0F1-type ATP synthase</a> (9)<br></div>
<img id='tree_img_0_267' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "267");'>&nbsp;Electron accepting reactions (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_267'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=1041826.106'>Anaerobic respiratory reductases</a> (2)<br></div>
<img id='tree_img_0_269' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "269");'>&nbsp;Electron donating reactions (31)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_269'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106'>Respiratory Complex I</a> (15)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=1041826.106'>Respiratory dehydrogenases 1</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106'>NADH ubiquinone oxidoreductase</a> (14)<br></div>
<span style='cursor: pointer;'>&nbsp;Reverse electron transport (0)<br>
<img id='tree_img_0_274' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "274");'>&nbsp;Respiration - no subcategory (9)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_274'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cytochrome_c_oxidases&organism=1041826.106'>Biogenesis of cytochrome c oxidases</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=1041826.106'>Biogenesis of cbb3-type cytochrome c oxidases</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=1041826.106'>Biogenesis of c-type cytochromes</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=1041826.106'>Soluble cytochromes and functionally related electron carriers</a> (3)<br></div>
<span style='cursor: pointer;'>&nbsp;Sodium Ion-Coupled Energetics (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_280'><img id='tree_img_0_280' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "280");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEUAzMxS5ysDAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Stress Response (21)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_280'>
<img id='tree_img_0_281' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "281");'>&nbsp;Osmotic stress (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_281'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=1041826.106'>Osmoregulation</a> (2)<br></div>
<span style='cursor: pointer;'>&nbsp;Dessication stress (0)<br>
<span style='cursor: pointer;'>&nbsp;Acid stress (0)<br>
<img id='tree_img_0_285' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "285");'>&nbsp;Oxidative stress (16)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_285'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=1041826.106'>Oxidative stress</a> (13)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Redox_cycle&organism=1041826.106'>Glutathione: Redox cycle</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutathionylspermidine_and_Trypanothione&organism=1041826.106'>Glutathionylspermidine and Trypanothione</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Cold shock (0)<br>
<span style='cursor: pointer;'>&nbsp;Heat shock (0)<br>
<img id='tree_img_0_291' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "291");'>&nbsp;Detoxification (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_291'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Uptake_of_selenate_and_selenite&organism=1041826.106'>Uptake of selenate and selenite</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Stress Response - no subcategory (0)<br>
<img id='tree_img_0_294' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "294");'>&nbsp;Periplasmic Stress (2)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_294'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=1041826.106'>Periplasmic Stress Response</a> (2)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_296'><img id='tree_img_0_296' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "296");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXM//+B6UABAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Metabolism of Aromatic Compounds (10)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_296'>
<img id='tree_img_0_297' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "297");'>&nbsp;Peripheral pathways for catabolism of aromatic compounds (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_297'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Quinate_degradation&organism=1041826.106'>Quinate degradation</a> (1)<br></div>
<span style='cursor: pointer;'>&nbsp;Anaerobic degradation of aromatic compounds (0)<br>
<img id='tree_img_0_300' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "300");'>&nbsp;Metabolism of central aromatic intermediates (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_300'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=1041826.106'>Catechol branch of beta-ketoadipate pathway</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=1041826.106'>Salicylate and gentisate catabolism</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=1041826.106'>Homogentisate pathway of aromatic compound degradation</a> (3)<br></div>
<img id='tree_img_0_304' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "304");'>&nbsp;Metabolism of Aromatic Compounds - no subcategory (1)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_304'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Gentisate_degradation&organism=1041826.106'>Gentisate degradation</a> (1)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_306'><img id='tree_img_0_306' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "306");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEXM/8w+OSEXAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Amino Acids and Derivatives (134)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_306'>
<img id='tree_img_0_307' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "307");'>&nbsp;Glutamine, glutamate, aspartate, asparagine; ammonia assimilation (11)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_307'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=1041826.106'>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a> (9)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=1041826.106'>Glutamate dehydrogenases</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=1041826.106'>Glutamine synthetases</a> (1)<br></div>
<img id='tree_img_0_311' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "311");'>&nbsp;Histidine Metabolism (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_311'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=1041826.106'>Histidine Degradation</a> (5)<br></div>
<img id='tree_img_0_313' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "313");'>&nbsp;Arginine; urea cycle, polyamines (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_313'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=1041826.106'>Polyamine Metabolism</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=1041826.106'>Arginine and Ornithine Degradation</a> (6)<br></div>
<img id='tree_img_0_316' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "316");'>&nbsp;Lysine, threonine, methionine, and cysteine (24)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_316'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=1041826.106'>Threonine and Homoserine Biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=1041826.106'>Threonine degradation</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=1041826.106'>Lysine Biosynthesis DAP Pathway, GJO scratch</a> (8)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=1041826.106'>Lysine degradation</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=1041826.106'>Lysine Biosynthesis DAP Pathway</a> (8)<br></div>
<span style='cursor: pointer;'>&nbsp;Amino Acids and Derivatives - no subcategory (0)<br>
<img id='tree_img_0_323' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "323");'>&nbsp;Branched-chain amino acids (14)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_323'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=1041826.106'>Isoleucine degradation</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=1041826.106'>Valine degradation</a> (7)<br></div>
<span style='cursor: pointer;'>&nbsp;Polyamines (0)<br>
<img id='tree_img_0_327' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "327");'>&nbsp;Aromatic amino acids and derivatives (32)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_327'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=1041826.106'>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106'>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=1041826.106'>Phenylalanine and Tyrosine Branches from Chorismate</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106'>Tryptophan synthesis</a> (10)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=1041826.106'>Aromatic amino acid degradation</a> (3)<br></div>
<img id='tree_img_0_333' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "333");'>&nbsp;Proline and 4-hydroxyproline (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_333'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=1041826.106'>Proline Synthesis</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=1041826.106'>Proline, 4-hydroxyproline uptake and utilization</a> (3)<br></div>
<img id='tree_img_0_336' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "336");'>&nbsp;Alanine, serine, and glycine (35)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_336'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=1041826.106'>Glycine Biosynthesis</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=1041826.106'>Alanine biosynthesis</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=1041826.106'>Serine Biosynthesis</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=1041826.106'>Glycine cleavage system</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106'>Glycine and Serine Utilization</a> (16)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_342'><img id='tree_img_0_342' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "342");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX//5kDFd+lAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Sulfur Metabolism (7)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_342'>
<span style='cursor: pointer;'>&nbsp;Inorganic sulfur assimilation (0)<br>
<img id='tree_img_0_344' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "344");'>&nbsp;Sulfur Metabolism - no subcategory (7)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_344'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=1041826.106'>Thioredoxin-disulfide reductase</a> (7)<br></div>
<span style='cursor: pointer;'>&nbsp;DsrE (0)<br>
<span style='cursor: pointer;'>&nbsp;Organic sulfur assimilation (0)<br></div></div>
<div name='tree_level_0' id='tree_node_0_348'><img id='tree_img_0_348' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "348");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEWZzP8f9clqAAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Phosphorus Metabolism (15)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_348'>
<img id='tree_img_0_349' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "349");'>&nbsp;Phosphorus Metabolism - no subcategory (15)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_349'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=1041826.106'>Phosphate metabolism</a> (11)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=1041826.106'>Polyphosphate</a> (4)<br></div></div></div>
<div name='tree_level_0' id='tree_node_0_352'><img id='tree_img_0_352' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "352");'>&nbsp;<img src='data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKAQMAAAC3/F3+AAAAA1BMVEX/mcwrs7b/AAAAC0lEQVQImWNgwAcAAB4AAe72cCEAAAAASUVORK5CYII='>&nbsp;&nbsp;Carbohydrates (62)<div style='margin-left: 15px; display: none;' name='tree_level_1'  id='tree_div_0_352'>
<img id='tree_img_0_353' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "353");'>&nbsp;Central carbohydrate metabolism (34)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_353'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=1041826.106'>Methylglyoxal Metabolism</a> (4)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=1041826.106'>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=1041826.106'>Pyruvate metabolism I: anaplerotic reactions, PEP</a> (3)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=1041826.106'>Pyruvate Alanine Serine Interconversions</a> (6)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=1041826.106'>Pentose phosphate pathway</a> (7)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycolate,_glyoxylate_interconversions&organism=1041826.106'>Glycolate, glyoxylate interconversions</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=1041826.106'>Dehydrogenase complexes</a> (8)<br></div>
<span style='cursor: pointer;'>&nbsp;Aminosugars (0)<br>
<span style='cursor: pointer;'>&nbsp;Di- and oligosaccharides (0)<br>
<span style='cursor: pointer;'>&nbsp;Glycoside hydrolases (0)<br>
<img id='tree_img_0_364' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "364");'>&nbsp;One-carbon Metabolism (4)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_364'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=1041826.106'>One-carbon metabolism by tetrahydropterines</a> (4)<br></div>
<span style='cursor: pointer;'>&nbsp;Organic acids (0)<br>
<img id='tree_img_0_367' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "367");'>&nbsp;Fermentation (11)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_367'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=1041826.106'>Butanol Biosynthesis</a> (5)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=1041826.106'>Acetyl-CoA fermentation to Butyrate</a> (6)<br></div>
<span style='cursor: pointer;'>&nbsp;CO2 fixation (0)<br>
<img id='tree_img_0_371' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "371");'>&nbsp;Sugar alcohols (5)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_371'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=1041826.106'>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a> (5)<br></div>
<span style='cursor: pointer;'>&nbsp;Carbohydrates - no subcategory (0)<br>
<span style='cursor: pointer;'>&nbsp;Polysaccharides (0)<br>
<img id='tree_img_0_375' src="https://rast.nmpdr.org/Html/plus.gif" style='cursor: pointer;' onclick='expand("0", "375");'>&nbsp;Monosaccharides (8)<br><div style='margin-left: 15px; display: none;' name='tree_level_2'  id='tree_div_0_375'><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=1041826.106'>Mannose Metabolism</a> (2)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=D-ribose_utilization&organism=1041826.106'>D-ribose utilization</a> (1)<br><span style='cursor: pointer;'>&nbsp;<a href='seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=1041826.106'>Deoxyribose and Deoxynucleoside Catabolism</a> (5)<br></div></div></div></div>
</td></tr></table></div><div style='height: 100%; width: 100%;' class='tab_view_content' name='1_contents' id='1_content_1'><table class='table_table'><tr><td><input type='button' class='button' value='export to file' onclick='export_table("0", 0, 1, 0);'>&nbsp;&nbsp;<input type='button' class='button' value='clear all filters' onclick='table_reset_filters("0");'></td></tr><tr><td style='width: 100%; text-align: center; vertical-align: middle;'><table style='width: 100%;'>
<tr><td align=center><span class='table_perpage'>display&nbsp;<input type='text' id='table_perpage_0' name='table_perpage_0' size='3' value='15' onkeypress='return check_submit_filter(event, "0");'>&nbsp;items per page</span></td></tr></table></td></tr>
<tr><td style='width: 100%; text-align: center;'><table style='width: 100%;'><tr><td align='left' width='20%'><a href='javascript: table_first("0");' name='table_first_0'>&laquo;first</a>&nbsp;&nbsp;<a href='javascript: table_prev("0");' name='table_prev_0'>&laquo;prev</a></td><td align='center' width='60%'>displaying <span id='table_start_top_0'>1</span> - <span id='table_stop_top_0'>738</span> of <span id='table_total_top_0'>738</span></td><td align='right' width='20%'><a href='javascript: table_next("0");' name='table_next_0'>next&raquo;</a>&nbsp;&nbsp;<a href='javascript: table_last("0");' name='table_last_0'>last&raquo;</a></td></tr></table></td></tr><tr><td><input type='hidden' id='SubsystemTable' value='0' />
<input type='hidden' id='table_data_0' value='Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=1041826.106@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)&subsystem_name=Biotin_biosynthesis@1>Long-chain-fatty-acid--CoA ligase (EC 6.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.59@1>fig|1041826.106.peg.59</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=1041826.106@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin synthase (EC 2.8.1.6)&subsystem_name=Biotin_biosynthesis@1>Biotin synthase (EC 2.8.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2124@1>fig|1041826.106.peg.2124</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=1041826.106@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)&subsystem_name=Biotin_biosynthesis@1>Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2789@1>fig|1041826.106.peg.2789</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis&organism=1041826.106@1>Biotin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=8-amino-7-oxononanoate synthase (EC 2.3.1.47)&subsystem_name=Biotin_biosynthesis@1>8-amino-7-oxononanoate synthase (EC 2.3.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2702@1>fig|1041826.106.peg.2702</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=1041826.106@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin synthase (EC 2.8.1.6)&subsystem_name=Biotin_biosynthesis_Experimental@1>Biotin synthase (EC 2.8.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2124@1>fig|1041826.106.peg.2124</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=1041826.106@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Competence protein F homolog, phosphoribosyltransferase domain&subsystem_name=Biotin_biosynthesis_Experimental@1>Competence protein F homolog, phosphoribosyltransferase domain</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1949@1>fig|1041826.106.peg.1949</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=1041826.106@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)&subsystem_name=Biotin_biosynthesis_Experimental@1>Adenosylmethionine-8-amino-7-oxononanoate aminotransferase (EC 2.6.1.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2789@1>fig|1041826.106.peg.2789</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Biotin@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biotin_biosynthesis_Experimental&organism=1041826.106@1>Biotin biosynthesis Experimental</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=8-amino-7-oxononanoate synthase (EC 2.3.1.47)&subsystem_name=Biotin_biosynthesis_Experimental@1>8-amino-7-oxononanoate synthase (EC 2.3.1.47)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2702@1>fig|1041826.106.peg.2702</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=1041826.106@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-regulated outer membrane receptor Omr1&subsystem_name=Thiamin_biosynthesis@1>Thiamin-regulated outer membrane receptor Omr1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2013@1>fig|1041826.106.peg.2013</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2749@1>fig|1041826.106.peg.2749</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=1041826.106@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin pyrophosphokinase (EC 2.7.6.2)&subsystem_name=Thiamin_biosynthesis@1>Thiamin pyrophosphokinase (EC 2.7.6.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1186@1>fig|1041826.106.peg.1186</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=1041826.106@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamine-monophosphate kinase (EC 2.7.4.16)&subsystem_name=Thiamin_biosynthesis@1>Thiamine-monophosphate kinase (EC 2.7.4.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1952@1>fig|1041826.106.peg.1952</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=1041826.106@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)&subsystem_name=Thiamin_biosynthesis@1>Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1890@1>fig|1041826.106.peg.1890</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=1041826.106@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Sulfur carrier protein ThiS&subsystem_name=Thiamin_biosynthesis@1>Sulfur carrier protein ThiS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1886@1>fig|1041826.106.peg.1886</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Cofactors, Vitamins, Prosthetic Groups, Pigments - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thiamin_biosynthesis&organism=1041826.106@1>Thiamin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Thiamin_biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2146@1>fig|1041826.106.peg.2146</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Naphthoate synthase (EC 4.1.3.36)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis@1>Naphthoate synthase (EC 4.1.3.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.482@1>fig|1041826.106.peg.482</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Menaquinone-specific isochorismate synthase (EC 5.4.4.2)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis@1>Menaquinone-specific isochorismate synthase (EC 5.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.240@1>fig|1041826.106.peg.240</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic-acid synthase (EC 2.2.1.9)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis@1>2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic-acid synthase (EC 2.2.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1530@1>fig|1041826.106.peg.1530</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylbenzoic acid--CoA ligase (EC 6.2.1.26)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis@1>O-succinylbenzoic acid--CoA ligase (EC 6.2.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1416@1>fig|1041826.106.peg.1416</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylbenzoate synthase (EC 4.2.1.113)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis@1>O-succinylbenzoate synthase (EC 4.2.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.477@1>fig|1041826.106.peg.477</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1,4-dihydroxy-2-naphthoate polyprenyltransferase (EC 2.5.1.74)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis@1>1,4-dihydroxy-2-naphthoate polyprenyltransferase (EC 2.5.1.74)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.481@1>fig|1041826.106.peg.481</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis -- gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Naphthoate synthase (EC 4.1.3.36)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo@1>Naphthoate synthase (EC 4.1.3.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.482@1>fig|1041826.106.peg.482</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis -- gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Menaquinone-specific isochorismate synthase (EC 5.4.4.2)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo@1>Menaquinone-specific isochorismate synthase (EC 5.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.240@1>fig|1041826.106.peg.240</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis -- gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic-acid synthase (EC 2.2.1.9)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo@1>2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylic-acid synthase (EC 2.2.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1530@1>fig|1041826.106.peg.1530</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis -- gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylbenzoic acid--CoA ligase (EC 6.2.1.26)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo@1>O-succinylbenzoic acid--CoA ligase (EC 6.2.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1416@1>fig|1041826.106.peg.1416</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis -- gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylbenzoate synthase (EC 4.2.1.113)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo@1>O-succinylbenzoate synthase (EC 4.2.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.477@1>fig|1041826.106.peg.477</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Quinone cofactors@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo&organism=1041826.106@1>Menaquinone and Phylloquinone Biosynthesis -- gjo</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1,4-dihydroxy-2-naphthoate polyprenyltransferase (EC 2.5.1.74)&subsystem_name=Menaquinone_and_Phylloquinone_Biosynthesis_--_gjo@1>1,4-dihydroxy-2-naphthoate polyprenyltransferase (EC 2.5.1.74)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.481@1>fig|1041826.106.peg.481</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobalamin biosynthesis protein BluB&subsystem_name=Cobalamin_synthesis@1>Cobalamin biosynthesis protein BluB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.513@1>fig|1041826.106.peg.513</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-threonine 3-O-phosphate decarboxylase (EC 4.1.1.81)&subsystem_name=Cobalamin_synthesis@1>L-threonine 3-O-phosphate decarboxylase (EC 4.1.1.81)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.522@1>fig|1041826.106.peg.522</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62)&subsystem_name=Cobalamin_synthesis@1>Adenosylcobinamide-phosphate guanylyltransferase (EC 2.7.7.62)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1071@1>fig|1041826.106.peg.1071</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobyric acid synthase (EC 6.3.5.10)&subsystem_name=Cobalamin_synthesis@1>Cobyric acid synthase (EC 6.3.5.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.523@1>fig|1041826.106.peg.523</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)&subsystem_name=Cobalamin_synthesis@1>Nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase (EC 2.4.2.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1069@1>fig|1041826.106.peg.1069</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylcobinamide-phosphate synthase (EC 6.3.1.10)&subsystem_name=Cobalamin_synthesis@1>Adenosylcobinamide-phosphate synthase (EC 6.3.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.521@1>fig|1041826.106.peg.521</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)&subsystem_name=Cobalamin_synthesis@1>Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1067@1>fig|1041826.106.peg.1067</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonine kinase in B12 biosynthesis&subsystem_name=Cobalamin_synthesis@1>Threonine kinase in B12 biosynthesis</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.524@1>fig|1041826.106.peg.524</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cobalamin synthase (EC 2.7.8.26)&subsystem_name=Cobalamin_synthesis@1>Cobalamin synthase (EC 2.7.8.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1068@1>fig|1041826.106.peg.1068</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalamin_synthesis&organism=1041826.106@1>Cobalamin synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenosylcobinamide kinase (EC 2.7.1.156)&subsystem_name=Cobalamin_synthesis@1>Adenosylcobinamide kinase (EC 2.7.1.156)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1071@1>fig|1041826.106.peg.1071</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA synthetase (EC 6.1.1.17)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Glutamyl-tRNA synthetase (EC 6.1.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1851@1>fig|1041826.106.peg.1851</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Coproporphyrinogen III oxidase, aerobic (EC 1.3.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1979@1>fig|1041826.106.peg.1979</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen-III synthase (EC 4.2.1.75)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen-III synthase (EC 4.2.1.75)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2820@1>fig|1041826.106.peg.2820</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Porphobilinogen synthase (EC 4.2.1.24)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Porphobilinogen synthase (EC 4.2.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1977@1>fig|1041826.106.peg.1977</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Coproporphyrinogen III oxidase, oxygen-independent (EC 1.3.99.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.251@1>fig|1041826.106.peg.251</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA reductase (EC 1.2.1.70)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Glutamyl-tRNA reductase (EC 1.2.1.70)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1984@1>fig|1041826.106.peg.1984</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Porphobilinogen deaminase (EC 2.5.1.61)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Porphobilinogen deaminase (EC 2.5.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1983@1>fig|1041826.106.peg.1983</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Ferrochelatase, protoheme ferro-lyase (EC 4.99.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1986@1>fig|1041826.106.peg.1986</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protoporphyrinogen IX oxidase, novel form, HemJ (EC 1.3.-.-)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Protoporphyrinogen IX oxidase, novel form, HemJ (EC 1.3.-.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1987@1>fig|1041826.106.peg.1987</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Tetrapyrroles@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Heme_and_Siroheme_Biosynthesis&organism=1041826.106@1>Heme and Siroheme Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uroporphyrinogen III decarboxylase (EC 4.1.1.37)&subsystem_name=Heme_and_Siroheme_Biosynthesis@1>Uroporphyrinogen III decarboxylase (EC 4.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1981@1>fig|1041826.106.peg.1981</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=1041826.106@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FMN adenylyltransferase (EC 2.7.7.2)&subsystem_name=riboflavin_to_FAD@1>FMN adenylyltransferase (EC 2.7.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2121@1>fig|1041826.106.peg.2121</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=1041826.106@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)&subsystem_name=riboflavin_to_FAD@1>Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1715@1>fig|1041826.106.peg.1715</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=1041826.106@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin kinase (EC 2.7.1.26)&subsystem_name=riboflavin_to_FAD@1>Riboflavin kinase (EC 2.7.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2121@1>fig|1041826.106.peg.2121</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=riboflavin_to_FAD&organism=1041826.106@1>riboflavin to FAD</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)&subsystem_name=riboflavin_to_FAD@1>3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.560@1>fig|1041826.106.peg.560</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=1041826.106@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=FMN adenylyltransferase (EC 2.7.7.2)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>FMN adenylyltransferase (EC 2.7.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2121@1>fig|1041826.106.peg.2121</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=1041826.106@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Riboflavin synthase eubacterial/eukaryotic (EC 2.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1715@1>fig|1041826.106.peg.1715</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=1041826.106@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminohydroxyphosphoribosylaminopyrimidine deaminase (EC 3.5.4.26)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Diaminohydroxyphosphoribosylaminopyrimidine deaminase (EC 3.5.4.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1009@1>fig|1041826.106.peg.1009</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=1041826.106@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>6,7-dimethyl-8-ribityllumazine synthase (EC 2.5.1.78)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2569@1>fig|1041826.106.peg.2569</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=1041826.106@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-amino-6-(5-phosphoribosylamino)uracil reductase (EC 1.1.1.193)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>5-amino-6-(5-phosphoribosylamino)uracil reductase (EC 1.1.1.193)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1009@1>fig|1041826.106.peg.1009</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=1041826.106@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Riboflavin kinase (EC 2.7.1.26)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>Riboflavin kinase (EC 2.7.1.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2121@1>fig|1041826.106.peg.2121</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=1041826.106@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase II (EC 3.5.4.25)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>GTP cyclohydrolase II (EC 3.5.4.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.560@1>fig|1041826.106.peg.560</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Riboflavin, FMN, FAD@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Riboflavin,_FMN_and_FAD_metabolism&organism=1041826.106@1>Riboflavin, FMN and FAD metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)&subsystem_name=Riboflavin,_FMN_and_FAD_metabolism@1>3,4-dihydroxy-2-butanone 4-phosphate synthase (EC 4.1.99.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.560@1>fig|1041826.106.peg.560</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=1041826.106@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.313@1>fig|1041826.106.peg.313</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2012@1>fig|1041826.106.peg.2012</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2421@1>fig|1041826.106.peg.2421</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=1041826.106@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>4-hydroxythreonine-4-phosphate dehydrogenase (EC 1.1.1.262)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1714@1>fig|1041826.106.peg.1714</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=1041826.106@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyridoxamine 5@1-phosphate oxidase (EC 1.4.3.5)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Pyridoxamine 5@1-phosphate oxidase (EC 1.4.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.79@1>fig|1041826.106.peg.79</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=1041826.106@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyridoxine 5@1-phosphate synthase (EC 2.6.99.2)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Pyridoxine 5@1-phosphate synthase (EC 2.6.99.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.947@1>fig|1041826.106.peg.947</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=1041826.106@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.314@1>fig|1041826.106.peg.314</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=1041826.106@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>NAD-dependent glyceraldehyde-3-phosphate dehydrogenase (EC 1.2.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.230@1>fig|1041826.106.peg.230</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.996@1>fig|1041826.106.peg.996</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1400@1>fig|1041826.106.peg.1400</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Pyridoxine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyridoxin_(Vitamin_B6)_Biosynthesis&organism=1041826.106@1>Pyridoxin (Vitamin B6) Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)&subsystem_name=Pyridoxin_(Vitamin_B6)_Biosynthesis@1>1-deoxy-D-xylulose 5-phosphate synthase (EC 2.2.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2146@1>fig|1041826.106.peg.2146</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate synthetase (EC 2.5.1.72)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Quinolinate synthetase (EC 2.5.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1604@1>fig|1041826.106.peg.1604</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.275@1>fig|1041826.106.peg.275</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD kinase (EC 2.7.1.23)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>NAD kinase (EC 2.7.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.949@1>fig|1041826.106.peg.949</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinamide phosphoribosyltransferase (EC 2.4.2.12)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Nicotinamide phosphoribosyltransferase (EC 2.4.2.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2714@1>fig|1041826.106.peg.2714</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxyanthranilate 3,4-dioxygenase (EC 1.13.11.6)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>3-hydroxyanthranilate 3,4-dioxygenase (EC 1.13.11.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2751@1>fig|1041826.106.peg.2751</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD synthetase (EC 6.3.1.5)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>NAD synthetase (EC 6.3.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.833@1>fig|1041826.106.peg.833</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan 2,3-dioxygenase (EC 1.13.11.11)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Tryptophan 2,3-dioxygenase (EC 1.13.11.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2096@1>fig|1041826.106.peg.2096</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-aspartate oxidase (EC 1.4.3.16)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>L-aspartate oxidase (EC 1.4.3.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1605@1>fig|1041826.106.peg.1605</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosyl nicotinamide transporter, PnuC-like&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Ribosyl nicotinamide transporter, PnuC-like</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2823@1>fig|1041826.106.peg.2823</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kynurenine 3-monooxygenase (EC 1.14.13.9)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Kynurenine 3-monooxygenase (EC 1.14.13.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.732@1>fig|1041826.106.peg.732</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Nicotinate-nucleotide adenylyltransferase (EC 2.7.7.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2711@1>fig|1041826.106.peg.2711</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^NAD and NADP@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NAD_and_NADP_cofactor_biosynthesis_global&organism=1041826.106@1>NAD and NADP cofactor biosynthesis global</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kynureninase (EC 3.7.1.3)&subsystem_name=NAD_and_NADP_cofactor_biosynthesis_global@1>Kynureninase (EC 3.7.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.728@1>fig|1041826.106.peg.728</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Folate_biosynthesis_cluster@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1570@1>fig|1041826.106.peg.1570</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydropteroate synthase (EC 2.5.1.15)&subsystem_name=Folate_biosynthesis_cluster@1>Dihydropteroate synthase (EC 2.5.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1579@1>fig|1041826.106.peg.1579</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)&subsystem_name=Folate_biosynthesis_cluster@1>tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1287@1>fig|1041826.106.peg.1287</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)&subsystem_name=Folate_biosynthesis_cluster@1>Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2370@1>fig|1041826.106.peg.2370</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Folate_biosynthesis_cluster@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2807@1>fig|1041826.106.peg.2807</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Folate_biosynthesis_cluster@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.320@1>fig|1041826.106.peg.320</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Folate_biosynthesis_cluster@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2808@1>fig|1041826.106.peg.2808</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)&subsystem_name=Folate_biosynthesis_cluster@1>2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.937@1>fig|1041826.106.peg.937</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_biosynthesis_cluster&organism=1041826.106@1>Folate biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroneopterin aldolase (EC 4.1.2.25)&subsystem_name=Folate_biosynthesis_cluster@1>Dihydroneopterin aldolase (EC 4.1.2.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1846@1>fig|1041826.106.peg.1846</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Folate_Biosynthesis@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1028@1>fig|1041826.106.peg.1028</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Folylpolyglutamate synthase (EC 6.3.2.17)&subsystem_name=Folate_Biosynthesis@1>Folylpolyglutamate synthase (EC 6.3.2.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.498@1>fig|1041826.106.peg.498</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydropteroate synthase (EC 2.5.1.15)&subsystem_name=Folate_Biosynthesis@1>Dihydropteroate synthase (EC 2.5.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1579@1>fig|1041826.106.peg.1579</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=Folate_Biosynthesis@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1915@1>fig|1041826.106.peg.1915</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate reductase (EC 1.5.1.3)&subsystem_name=Folate_Biosynthesis@1>Dihydrofolate reductase (EC 1.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.864@1>fig|1041826.106.peg.864</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate synthase (EC 6.3.2.12)&subsystem_name=Folate_Biosynthesis@1>Dihydrofolate synthase (EC 6.3.2.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.498@1>fig|1041826.106.peg.498</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Folate_Biosynthesis@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.320@1>fig|1041826.106.peg.320</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Folate_Biosynthesis@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1286@1>fig|1041826.106.peg.1286</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidylate synthase (EC 2.1.1.45)&subsystem_name=Folate_Biosynthesis@1>Thymidylate synthase (EC 2.1.1.45)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.866@1>fig|1041826.106.peg.866</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)&subsystem_name=Folate_Biosynthesis@1>2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase (EC 2.7.6.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.937@1>fig|1041826.106.peg.937</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydroneopterin aldolase (EC 4.1.2.25)&subsystem_name=Folate_Biosynthesis@1>Dihydroneopterin aldolase (EC 4.1.2.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1846@1>fig|1041826.106.peg.1846</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Folate_Biosynthesis&organism=1041826.106@1>Folate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Folate_Biosynthesis@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.223@1>fig|1041826.106.peg.223</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)&subsystem_name=5-FCL-like_protein@1>Isocitrate dehydrogenase [NADP] (EC 1.1.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2612@1>fig|1041826.106.peg.2612</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=5-FCL-like_protein@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1915@1>fig|1041826.106.peg.1915</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrofolate reductase (EC 1.5.1.3)&subsystem_name=5-FCL-like_protein@1>Dihydrofolate reductase (EC 1.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.864@1>fig|1041826.106.peg.864</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamine-monophosphate kinase (EC 2.7.4.16)&subsystem_name=5-FCL-like_protein@1>Thiamine-monophosphate kinase (EC 2.7.4.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1952@1>fig|1041826.106.peg.1952</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=5-FCL-like_protein@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1203@1>fig|1041826.106.peg.1203</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)&subsystem_name=5-FCL-like_protein@1>Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1264@1>fig|1041826.106.peg.1264</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)&subsystem_name=5-FCL-like_protein@1>Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2765@1>fig|1041826.106.peg.2765</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Butyryl-CoA dehydrogenase (EC 1.3.8.1)&subsystem_name=5-FCL-like_protein@1>Butyryl-CoA dehydrogenase (EC 1.3.8.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1365@1>fig|1041826.106.peg.1365</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)&subsystem_name=5-FCL-like_protein@1>Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.880@1>fig|1041826.106.peg.880</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)&subsystem_name=5-FCL-like_protein@1>Thiamin-phosphate pyrophosphorylase (EC 2.5.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1890@1>fig|1041826.106.peg.1890</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=5-FCL-like_protein@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.843@1>fig|1041826.106.peg.843</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=5-FCL-like_protein@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2284@1>fig|1041826.106.peg.2284</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=5-FCL-like_protein&organism=1041826.106@1>5-FCL-like protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=5-FCL-like_protein@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.195@1>fig|1041826.106.peg.195</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.243@1>fig|1041826.106.peg.243</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=1041826.106@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkaline phosphatase (EC 3.1.3.1)&subsystem_name=Pterin_synthesis_related_cluster@1>Alkaline phosphatase (EC 3.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2023@1>fig|1041826.106.peg.2023</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2557@1>fig|1041826.106.peg.2557</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=1041826.106@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP cyclohydrolase I (EC 3.5.4.16) type 1&subsystem_name=Pterin_synthesis_related_cluster@1>GTP cyclohydrolase I (EC 3.5.4.16) type 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.320@1>fig|1041826.106.peg.320</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=1041826.106@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)&subsystem_name=Pterin_synthesis_related_cluster@1>Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.137@1>fig|1041826.106.peg.137</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=1041826.106@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Pterin_synthesis_related_cluster@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.220@1>fig|1041826.106.peg.220</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Folate and pterines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pterin_synthesis_related_cluster&organism=1041826.106@1>Pterin synthesis related cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=6-pyruvoyl tetrahydrobiopterin synthase (EC 4.2.3.12)&subsystem_name=Pterin_synthesis_related_cluster@1>6-pyruvoyl tetrahydrobiopterin synthase (EC 4.2.3.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1051@1>fig|1041826.106.peg.1051</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=1041826.106@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1570@1>fig|1041826.106.peg.1570</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=1041826.106@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.294@1>fig|1041826.106.peg.294</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=1041826.106@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2808@1>fig|1041826.106.peg.2808</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=1041826.106@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.929@1>fig|1041826.106.peg.929</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis_cluster&organism=1041826.106@1>Coenzyme A Biosynthesis cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Coenzyme_A_Biosynthesis_cluster@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2807@1>fig|1041826.106.peg.2807</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ketopantoate reductase PanG (EC 1.1.1.169)&subsystem_name=Coenzyme_A_Biosynthesis@1>Ketopantoate reductase PanG (EC 1.1.1.169)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1570@1>fig|1041826.106.peg.1570</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)&subsystem_name=Coenzyme_A_Biosynthesis@1>3-methyl-2-oxobutanoate hydroxymethyltransferase (EC 2.1.2.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.294@1>fig|1041826.106.peg.294</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dephospho-CoA kinase (EC 2.7.1.24)&subsystem_name=Coenzyme_A_Biosynthesis@1>Dephospho-CoA kinase (EC 2.7.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1484@1>fig|1041826.106.peg.1484</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)&subsystem_name=Coenzyme_A_Biosynthesis@1>Pantothenate kinase type III, CoaX-like (EC 2.7.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.929@1>fig|1041826.106.peg.929</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate 1-decarboxylase (EC 4.1.1.11)&subsystem_name=Coenzyme_A_Biosynthesis@1>Aspartate 1-decarboxylase (EC 4.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2807@1>fig|1041826.106.peg.2807</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantetheine adenylyltransferase (EC 2.7.7.3)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantetheine adenylyltransferase (EC 2.7.7.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1451@1>fig|1041826.106.peg.1451</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantothenoylcysteine synthetase (EC 6.3.2.5)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantothenoylcysteine synthetase (EC 6.3.2.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1490@1>fig|1041826.106.peg.1490</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pantoate--beta-alanine ligase (EC 6.3.2.1)&subsystem_name=Coenzyme_A_Biosynthesis@1>Pantoate--beta-alanine ligase (EC 6.3.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2808@1>fig|1041826.106.peg.2808</a>@~Cofactors, Vitamins, Prosthetic Groups, Pigments@^Coenzyme A@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Coenzyme_A_Biosynthesis&organism=1041826.106@1>Coenzyme A Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphopantothenoylcysteine decarboxylase (EC 4.1.1.36)&subsystem_name=Coenzyme_A_Biosynthesis@1>Phosphopantothenoylcysteine decarboxylase (EC 4.1.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1490@1>fig|1041826.106.peg.1490</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=1041826.106@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1739@1>fig|1041826.106.peg.1739</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=1041826.106@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.776@1>fig|1041826.106.peg.776</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=1041826.106@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1740@1>fig|1041826.106.peg.1740</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=1041826.106@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)&subsystem_name=dTDP-rhamnose_synthesis@1>Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.777@1>fig|1041826.106.peg.777</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1998@1>fig|1041826.106.peg.1998</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dTDP-rhamnose_synthesis&organism=1041826.106@1>dTDP-rhamnose synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-Rha:A-D-GlcNAc-diphosphoryl polyprenol, A-3-L-rhamnosyl transferase WbbL&subsystem_name=dTDP-rhamnose_synthesis@1>dTDP-Rha:A-D-GlcNAc-diphosphoryl polyprenol, A-3-L-rhamnosyl transferase WbbL</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2537@1>fig|1041826.106.peg.2537</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=1041826.106@1>CMP-N-acetylneuraminate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylneuraminate synthase (EC 2.5.1.56)&subsystem_name=CMP-N-acetylneuraminate_Biosynthesis@1>N-acetylneuraminate synthase (EC 2.5.1.56)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2646@1>fig|1041826.106.peg.2646</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=CMP-N-acetylneuraminate_Biosynthesis&organism=1041826.106@1>CMP-N-acetylneuraminate Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)&subsystem_name=CMP-N-acetylneuraminate_Biosynthesis@1>UDP-N-acetylglucosamine 2-epimerase (EC 5.1.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2635@1>fig|1041826.106.peg.2635</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=1041826.106@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UDP-glucose 4-epimerase (EC 5.1.3.2)&subsystem_name=Rhamnose_containing_glycans@1>UDP-glucose 4-epimerase (EC 5.1.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.701@1>fig|1041826.106.peg.701</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.773@1>fig|1041826.106.peg.773</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=1041826.106@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-4-dehydrorhamnose reductase (EC 1.1.1.133)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1739@1>fig|1041826.106.peg.1739</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=1041826.106@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=capsular polysaccharide biosynthesis protein&subsystem_name=Rhamnose_containing_glycans@1>capsular polysaccharide biosynthesis protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2631@1>fig|1041826.106.peg.2631</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=1041826.106@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-L-Rha alpha-1,3-L-rhamnosyltransferase (EC 2.4.1.-)&subsystem_name=Rhamnose_containing_glycans@1>Alpha-L-Rha alpha-1,3-L-rhamnosyltransferase (EC 2.4.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.261@1>fig|1041826.106.peg.261</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=1041826.106@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=putative glycosyltransferase - possibly involved in cell wall localization and side chain formation of rhamnose-glucose polysaccharide&subsystem_name=Rhamnose_containing_glycans@1>putative glycosyltransferase - possibly involved in cell wall localization and side chain formation of rhamnose-glucose polysaccharide</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.260@1>fig|1041826.106.peg.260</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=1041826.106@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-glucose 4,6-dehydratase (EC 4.2.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.776@1>fig|1041826.106.peg.776</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=1041826.106@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)&subsystem_name=Rhamnose_containing_glycans@1>dTDP-4-dehydrorhamnose 3,5-epimerase (EC 5.1.3.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1740@1>fig|1041826.106.peg.1740</a>@~Cell Wall and Capsule@^Capsular and extracellular polysacchrides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rhamnose_containing_glycans&organism=1041826.106@1>Rhamnose containing glycans</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)&subsystem_name=Rhamnose_containing_glycans@1>Glucose-1-phosphate thymidylyltransferase (EC 2.7.7.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.777@1>fig|1041826.106.peg.777</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1998@1>fig|1041826.106.peg.1998</a>@~Cell Wall and Capsule@^Gram-Negative cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lipoprotein_sorting_system&organism=1041826.106@1>Lipoprotein sorting system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane lipoprotein carrier protein LolA&subsystem_name=Lipoprotein_sorting_system@1>Outer membrane lipoprotein carrier protein LolA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.562@1>fig|1041826.106.peg.562</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=1041826.106@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)&subsystem_name=Murein_Hydrolases@1>N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1503@1>fig|1041826.106.peg.1503</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2712@1>fig|1041826.106.peg.2712</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Murein_Hydrolases&organism=1041826.106@1>Murein Hydrolases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Murein_Hydrolases@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1756@1>fig|1041826.106.peg.1756</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=1041826.106@1>Recycling of Peptidoglycan Amino Acids</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)&subsystem_name=Recycling_of_Peptidoglycan_Amino_Acids@1>N-acetylmuramoyl-L-alanine amidase (EC 3.5.1.28)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1503@1>fig|1041826.106.peg.1503</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2712@1>fig|1041826.106.peg.2712</a>@~Cell Wall and Capsule@^Cell Wall and Capsule - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Recycling_of_Peptidoglycan_Amino_Acids&organism=1041826.106@1>Recycling of Peptidoglycan Amino Acids</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Recycling_of_Peptidoglycan_Amino_Acids@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1756@1>fig|1041826.106.peg.1756</a>@~Cell Wall and Capsule@^Gram-Positive cell wall components@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Teichuronic_acid_biosynthesis&organism=1041826.106@1>Teichuronic acid biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative N-acetylgalactosaminyl-diphosphoundecaprenol glucuronosyltransferase&subsystem_name=Teichuronic_acid_biosynthesis@1>Putative N-acetylgalactosaminyl-diphosphoundecaprenol glucuronosyltransferase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1736@1>fig|1041826.106.peg.1736</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_homeostasis&organism=1041826.106@1>Copper homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-translocating P-type ATPase (EC 3.6.3.4)&subsystem_name=Copper_homeostasis@1>Copper-translocating P-type ATPase (EC 3.6.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.630@1>fig|1041826.106.peg.630</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1375@1>fig|1041826.106.peg.1375</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Cobalt-zinc-cadmium_resistance&organism=1041826.106@1>Cobalt-zinc-cadmium resistance</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional regulator, MerR family&subsystem_name=Cobalt-zinc-cadmium_resistance@1>Transcriptional regulator, MerR family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2269@1>fig|1041826.106.peg.2269</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=1041826.106@1>Resistance to fluoroquinolones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit B (EC 5.99.1.3)&subsystem_name=Resistance_to_fluoroquinolones@1>DNA gyrase subunit B (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1436@1>fig|1041826.106.peg.1436</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Resistance_to_fluoroquinolones&organism=1041826.106@1>Resistance to fluoroquinolones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit A (EC 5.99.1.3)&subsystem_name=Resistance_to_fluoroquinolones@1>DNA gyrase subunit A (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.112@1>fig|1041826.106.peg.112</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Beta-lactamase&organism=1041826.106@1>Beta-lactamase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Metal-dependent hydrolases of the beta-lactamase superfamily I&subsystem_name=Beta-lactamase@1>Metal-dependent hydrolases of the beta-lactamase superfamily I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1274@1>fig|1041826.106.peg.1274</a>@~Virulence, Disease and Defense@^Resistance to antibiotics and toxic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Multidrug_Resistance_Efflux_Pumps&organism=1041826.106@1>Multidrug Resistance Efflux Pumps</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Multi antimicrobial extrusion protein (Na(+)/drug antiporter), MATE family of MDR efflux pumps&subsystem_name=Multidrug_Resistance_Efflux_Pumps@1>Multi antimicrobial extrusion protein (Na(+)/drug antiporter), MATE family of MDR efflux pumps</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2355@1>fig|1041826.106.peg.2355</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=1041826.106@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S7p (S5e)&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>SSU ribosomal protein S7p (S5e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2329@1>fig|1041826.106.peg.2329</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=1041826.106@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2328@1>fig|1041826.106.peg.2328</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=1041826.106@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Tu&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>Translation elongation factor Tu</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2193@1>fig|1041826.106.peg.2193</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)&organism=1041826.106@1>Mycobacterium virulence operon involved in protein synthesis (SSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S12p (S23e)&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(SSU_ribosomal_proteins)@1>SSU ribosomal protein S12p (S23e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2330@1>fig|1041826.106.peg.2330</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=1041826.106@1>Mycobacterium virulence operon involved in DNA transcription</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)&subsystem_name=Mycobacterium_virulence_operon_involved_in_DNA_transcription@1>DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2185@1>fig|1041826.106.peg.2185</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_DNA_transcription&organism=1041826.106@1>Mycobacterium virulence operon involved in DNA transcription</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)&subsystem_name=Mycobacterium_virulence_operon_involved_in_DNA_transcription@1>DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2186@1>fig|1041826.106.peg.2186</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=1041826.106@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate synthetase (EC 2.5.1.72)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>Quinolinate synthetase (EC 2.5.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1604@1>fig|1041826.106.peg.1604</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=1041826.106@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>Quinolinate phosphoribosyltransferase [decarboxylating] (EC 2.4.2.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.275@1>fig|1041826.106.peg.275</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis&organism=1041826.106@1>Mycobacterium virulence operon possibly involved in quinolinate biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-aspartate oxidase (EC 1.4.3.16)&subsystem_name=Mycobacterium_virulence_operon_possibly_involved_in_quinolinate_biosynthesis@1>L-aspartate oxidase (EC 1.4.3.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1605@1>fig|1041826.106.peg.1605</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=1041826.106@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L35p&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>LSU ribosomal protein L35p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.142@1>fig|1041826.106.peg.142</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=1041826.106@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 3&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>Translation initiation factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.143@1>fig|1041826.106.peg.143</a>@~Virulence, Disease and Defense@^Invasion and intracellular resistance@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)&organism=1041826.106@1>Mycobacterium virulence operon involved in protein synthesis (LSU ribosomal proteins)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=LSU ribosomal protein L20p&subsystem_name=Mycobacterium_virulence_operon_involved_in_protein_synthesis_(LSU_ribosomal_proteins)@1>LSU ribosomal protein L20p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.141@1>fig|1041826.106.peg.141</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=1041826.106@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Potassium efflux system KefA protein&subsystem_name=Potassium_homeostasis@1>Potassium efflux system KefA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.960@1>fig|1041826.106.peg.960</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=1041826.106@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Large-conductance mechanosensitive channel&subsystem_name=Potassium_homeostasis@1>Large-conductance mechanosensitive channel</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.293@1>fig|1041826.106.peg.293</a>@~Potassium metabolism@^Potassium metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Potassium_homeostasis&organism=1041826.106@1>Potassium homeostasis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Kup system potassium uptake protein&subsystem_name=Potassium_homeostasis@1>Kup system potassium uptake protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.461@1>fig|1041826.106.peg.461</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=1041826.106@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=PaaD-like protein (DUF59) involved in Fe-S cluster assembly&subsystem_name=Iron-sulfur_cluster_assembly@1>PaaD-like protein (DUF59) involved in Fe-S cluster assembly</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2785@1>fig|1041826.106.peg.2785</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=1041826.106@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=probable iron binding protein from the HesB_IscA_SufA family&subsystem_name=Iron-sulfur_cluster_assembly@1>probable iron binding protein from the HesB_IscA_SufA family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2738@1>fig|1041826.106.peg.2738</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=1041826.106@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Iron-sulfur_cluster_assembly@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2113@1>fig|1041826.106.peg.2113</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=1041826.106@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=Iron-sulfur_cluster_assembly@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.167@1>fig|1041826.106.peg.167</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.377@1>fig|1041826.106.peg.377</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=1041826.106@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly protein SufB&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly protein SufB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2737@1>fig|1041826.106.peg.2737</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=1041826.106@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly protein SufD&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly protein SufD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2733@1>fig|1041826.106.peg.2733</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Iron-sulfur_cluster_assembly&organism=1041826.106@1>Iron-sulfur cluster assembly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Iron-sulfur cluster assembly ATPase protein SufC&subsystem_name=Iron-sulfur_cluster_assembly@1>Iron-sulfur cluster assembly ATPase protein SufC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2735@1>fig|1041826.106.peg.2735</a>@~Miscellaneous@^Plant-Prokaryote DOE project@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Single-Rhodanese-domain_proteins&organism=1041826.106@1>Single-Rhodanese-domain proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rhodanese domain protein, Enterobacterial subgroup, YceA homolog&subsystem_name=Single-Rhodanese-domain_proteins@1>Rhodanese domain protein, Enterobacterial subgroup, YceA homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1789@1>fig|1041826.106.peg.1789</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphoglycerate_mutase_protein_family&organism=1041826.106@1>Phosphoglycerate mutase protein family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)&subsystem_name=Phosphoglycerate_mutase_protein_family@1>Alpha-ribazole-5@1-phosphate phosphatase (EC 3.1.3.73)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1067@1>fig|1041826.106.peg.1067</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DedA_family_of_inner_membrane_proteins&organism=1041826.106@1>DedA family of inner membrane proteins</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DedA protein&subsystem_name=DedA_family_of_inner_membrane_proteins@1>DedA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2503@1>fig|1041826.106.peg.2503</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Muconate_lactonizing_enzyme_family&organism=1041826.106@1>Muconate lactonizing enzyme family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=O-succinylbenzoate synthase (EC 4.2.1.113)&subsystem_name=Muconate_lactonizing_enzyme_family@1>O-succinylbenzoate synthase (EC 4.2.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.477@1>fig|1041826.106.peg.477</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=1041826.106@1>Broadly distributed proteins not in subsystems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=YpfJ protein, zinc metalloprotease superfamily&subsystem_name=Broadly_distributed_proteins_not_in_subsystems@1>YpfJ protein, zinc metalloprotease superfamily</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1408@1>fig|1041826.106.peg.1408</a>@~Miscellaneous@^Miscellaneous - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Broadly_distributed_proteins_not_in_subsystems&organism=1041826.106@1>Broadly distributed proteins not in subsystems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=UPF0225 protein YchJ&subsystem_name=Broadly_distributed_proteins_not_in_subsystems@1>UPF0225 protein YchJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.397@1>fig|1041826.106.peg.397</a>@~Phages, Prophages, Transposable elements, Plasmids@^Phages, Prophages@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=IbrA_and_IbrB:_co-activators_of_prophage_gene_expression&organism=1041826.106@1>IbrA and IbrB: co-activators of prophage gene expression</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Co-activator of prophage gene expression IbrB&subsystem_name=IbrA_and_IbrB:_co-activators_of_prophage_gene_expression@1>Co-activator of prophage gene expression IbrB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1116@1>fig|1041826.106.peg.1116</a>@~Phages, Prophages, Transposable elements, Plasmids@^Phages, Prophages@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=IbrA_and_IbrB:_co-activators_of_prophage_gene_expression&organism=1041826.106@1>IbrA and IbrB: co-activators of prophage gene expression</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Co-activator of prophage gene expression IbrA&subsystem_name=IbrA_and_IbrB:_co-activators_of_prophage_gene_expression@1>Co-activator of prophage gene expression IbrA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1117@1>fig|1041826.106.peg.1117</a>@~Phages, Prophages, Transposable elements, Plasmids@^Phages, Prophages, Transposable elements, Plasmids - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Integrons&organism=1041826.106@1>Integrons</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Integron integrase&subsystem_name=Integrons@1>Integron integrase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1178@1>fig|1041826.106.peg.1178</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1470@1>fig|1041826.106.peg.1470</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2613@1>fig|1041826.106.peg.2613</a>@~Phages, Prophages, Transposable elements, Plasmids@^Pathogenicity islands@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Listeria_Pathogenicity_Island_LIPI-1_extended&organism=1041826.106@1>Listeria Pathogenicity Island LIPI-1 extended</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphatidylinositol-specific phospholipase C (EC 4.6.1.13)&subsystem_name=Listeria_Pathogenicity_Island_LIPI-1_extended@1>Phosphatidylinositol-specific phospholipase C (EC 4.6.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2014@1>fig|1041826.106.peg.2014</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=1041826.106@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Twin-arginine translocation protein TatC&subsystem_name=Twin-arginine_translocation_system@1>Twin-arginine translocation protein TatC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2670@1>fig|1041826.106.peg.2670</a>@~Membrane Transport@^Protein translocation across cytoplasmic membrane@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Twin-arginine_translocation_system&organism=1041826.106@1>Twin-arginine translocation system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Twin-arginine translocation protein TatA&subsystem_name=Twin-arginine_translocation_system@1>Twin-arginine translocation protein TatA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.714@1>fig|1041826.106.peg.714</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1817@1>fig|1041826.106.peg.1817</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Magnesium_transport&organism=1041826.106@1>Magnesium transport</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Magnesium and cobalt transport protein CorA&subsystem_name=Magnesium_transport@1>Magnesium and cobalt transport protein CorA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1421@1>fig|1041826.106.peg.1421</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transport_of_Nickel_and_Cobalt&organism=1041826.106@1>Transport of Nickel and Cobalt</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nickel transporter UreH&subsystem_name=Transport_of_Nickel_and_Cobalt@1>Nickel transporter UreH</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.508@1>fig|1041826.106.peg.508</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transport_of_Nickel_and_Cobalt&organism=1041826.106@1>Transport of Nickel and Cobalt</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=HoxN/HupN/NixA family nickel/cobalt transporter&subsystem_name=Transport_of_Nickel_and_Cobalt@1>HoxN/HupN/NixA family nickel/cobalt transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.519@1>fig|1041826.106.peg.519</a>@~Membrane Transport@^Cation transporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Copper_Transport_System&organism=1041826.106@1>Copper Transport System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-translocating P-type ATPase (EC 3.6.3.4)&subsystem_name=Copper_Transport_System@1>Copper-translocating P-type ATPase (EC 3.6.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.630@1>fig|1041826.106.peg.630</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1375@1>fig|1041826.106.peg.1375</a>@~Membrane Transport@^Uni- Sym- and Antiporters@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters&organism=1041826.106@1>NhaA, NhaD and Sodium-dependent phosphate transporters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Na+/H+ antiporter NhaA type&subsystem_name=NhaA,_NhaD_and_Sodium-dependent_phosphate_transporters@1>Na+/H+ antiporter NhaA type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1088@1>fig|1041826.106.peg.1088</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=1041826.106@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric siderophore transport system, periplasmic binding protein TonB&subsystem_name=Ton_and_Tol_transport_systems@1>Ferric siderophore transport system, periplasmic binding protein TonB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.499@1>fig|1041826.106.peg.499</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=1041826.106@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=TonB-dependent receptor&subsystem_name=Ton_and_Tol_transport_systems@1>TonB-dependent receptor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.672@1>fig|1041826.106.peg.672</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1065@1>fig|1041826.106.peg.1065</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1188@1>fig|1041826.106.peg.1188</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1267@1>fig|1041826.106.peg.1267</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1432@1>fig|1041826.106.peg.1432</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1625@1>fig|1041826.106.peg.1625</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1703@1>fig|1041826.106.peg.1703</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1831@1>fig|1041826.106.peg.1831</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1966@1>fig|1041826.106.peg.1966</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2098@1>fig|1041826.106.peg.2098</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2215@1>fig|1041826.106.peg.2215</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2220@1>fig|1041826.106.peg.2220</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2524@1>fig|1041826.106.peg.2524</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2697@1>fig|1041826.106.peg.2697</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=1041826.106@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane lipoprotein omp16 precursor&subsystem_name=Ton_and_Tol_transport_systems@1>Outer membrane lipoprotein omp16 precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.448@1>fig|1041826.106.peg.448</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.976@1>fig|1041826.106.peg.976</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2174@1>fig|1041826.106.peg.2174</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=1041826.106@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biopolymer transport protein ExbD/TolR&subsystem_name=Ton_and_Tol_transport_systems@1>Biopolymer transport protein ExbD/TolR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.500@1>fig|1041826.106.peg.500</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.644@1>fig|1041826.106.peg.644</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.736@1>fig|1041826.106.peg.736</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.737@1>fig|1041826.106.peg.737</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=1041826.106@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxybenzoyl-CoA thioesterase family active site&subsystem_name=Ton_and_Tol_transport_systems@1>4-hydroxybenzoyl-CoA thioesterase family active site</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1006@1>fig|1041826.106.peg.1006</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1577@1>fig|1041826.106.peg.1577</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=1041826.106@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Ton_and_Tol_transport_systems@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1603@1>fig|1041826.106.peg.1603</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=1041826.106@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=MotA/TolQ/ExbB proton channel family protein&subsystem_name=Ton_and_Tol_transport_systems@1>MotA/TolQ/ExbB proton channel family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.501@1>fig|1041826.106.peg.501</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.735@1>fig|1041826.106.peg.735</a>@~Membrane Transport@^Membrane Transport - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ton_and_Tol_transport_systems&organism=1041826.106@1>Ton and Tol transport systems</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=TonB-dependent siderophore receptor&subsystem_name=Ton_and_Tol_transport_systems@1>TonB-dependent siderophore receptor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2345@1>fig|1041826.106.peg.2345</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=1041826.106@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Siderophore biosynthesis L-2,4-diaminobutyrate decarboxylase&subsystem_name=Siderophore_assembly_kit@1>Siderophore biosynthesis L-2,4-diaminobutyrate decarboxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2349@1>fig|1041826.106.peg.2349</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=1041826.106@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Siderophore synthetase component, ligase&subsystem_name=Siderophore_assembly_kit@1>Siderophore synthetase component, ligase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2352@1>fig|1041826.106.peg.2352</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=1041826.106@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Siderophore biosynthesis protein, monooxygenase&subsystem_name=Siderophore_assembly_kit@1>Siderophore biosynthesis protein, monooxygenase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2350@1>fig|1041826.106.peg.2350</a>@~Iron acquisition and metabolism@^Siderophores@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Siderophore_assembly_kit&organism=1041826.106@1>Siderophore assembly kit</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=TonB-dependent siderophore receptor&subsystem_name=Siderophore_assembly_kit@1>TonB-dependent siderophore receptor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2345@1>fig|1041826.106.peg.2345</a>@~Iron acquisition and metabolism@^Iron acquisition and metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Hemin_transport_system&organism=1041826.106@1>Hemin transport system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane receptor proteins, mostly Fe transport&subsystem_name=Hemin_transport_system@1>Outer membrane receptor proteins, mostly Fe transport</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.520@1>fig|1041826.106.peg.520</a>@~Iron acquisition and metabolism@^Iron acquisition and metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Hemin_transport_system&organism=1041826.106@1>Hemin transport system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric siderophore transport system, periplasmic binding protein TonB&subsystem_name=Hemin_transport_system@1>Ferric siderophore transport system, periplasmic binding protein TonB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.499@1>fig|1041826.106.peg.499</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=1041826.106@1>mnm5U34 biosynthesis bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)&subsystem_name=mnm5U34_biosynthesis_bacteria@1>tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.101@1>fig|1041826.106.peg.101</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=mnm5U34_biosynthesis_bacteria&organism=1041826.106@1>mnm5U34 biosynthesis bacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=mnm5U34_biosynthesis_bacteria@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.167@1>fig|1041826.106.peg.167</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.377@1>fig|1041826.106.peg.377</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=1041826.106@1>Ribonuclease H</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease HI (EC 3.1.26.4)&subsystem_name=Ribonuclease_H@1>Ribonuclease HI (EC 3.1.26.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2764@1>fig|1041826.106.peg.2764</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonuclease_H&organism=1041826.106@1>Ribonuclease H</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease HII (EC 3.1.26.4)&subsystem_name=Ribonuclease_H@1>Ribonuclease HII (EC 3.1.26.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2208@1>fig|1041826.106.peg.2208</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_3@1-terminal_phosphate_cyclase&organism=1041826.106@1>RNA 3@1-terminal phosphate cyclase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA-2@1,3@1-PO4:RNA-5@1-OH ligase&subsystem_name=RNA_3@1-terminal_phosphate_cyclase@1>RNA-2@1,3@1-PO4:RNA-5@1-OH ligase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.649@1>fig|1041826.106.peg.649</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1597@1>fig|1041826.106.peg.1597</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=1041826.106@1>RNA processing and degradation, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3@1-to-5@1 exoribonuclease RNase R&subsystem_name=RNA_processing_and_degradation,_bacterial@1>3@1-to-5@1 exoribonuclease RNase R</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1841@1>fig|1041826.106.peg.1841</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_processing_and_degradation,_bacterial&organism=1041826.106@1>RNA processing and degradation, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease III (EC 3.1.26.3)&subsystem_name=RNA_processing_and_degradation,_bacterial@1>Ribonuclease III (EC 3.1.26.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2768@1>fig|1041826.106.peg.2768</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=1041826.106@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease Z (EC 3.1.26.11)&subsystem_name=tRNA_processing@1>Ribonuclease Z (EC 3.1.26.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.78@1>fig|1041826.106.peg.78</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=1041826.106@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonuclease P protein component (EC 3.1.26.5)&subsystem_name=tRNA_processing@1>Ribonuclease P protein component (EC 3.1.26.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1687@1>fig|1041826.106.peg.1687</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=1041826.106@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA dimethylallyltransferase (EC 2.5.1.75)&subsystem_name=tRNA_processing@1>tRNA dimethylallyltransferase (EC 2.5.1.75)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1618@1>fig|1041826.106.peg.1618</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_processing&organism=1041826.106@1>tRNA processing</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)&subsystem_name=tRNA_processing@1>tRNA(Ile)-lysidine synthetase (EC 6.3.4.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1287@1>fig|1041826.106.peg.1287</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=1041826.106@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (guanosine(18)-2@1-O)-methyltransferase (EC 2.1.1.34)&subsystem_name=RNA_methylation@1>tRNA (guanosine(18)-2@1-O)-methyltransferase (EC 2.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1242@1>fig|1041826.106.peg.1242</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=1041826.106@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)&subsystem_name=RNA_methylation@1>tRNA (5-methylaminomethyl-2-thiouridylate)-methyltransferase (EC 2.1.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.101@1>fig|1041826.106.peg.101</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=1041826.106@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)&subsystem_name=RNA_methylation@1>SSU rRNA (adenine(1518)-N(6)/adenine(1519)-N(6))-dimethyltransferase (EC 2.1.1.182)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2007@1>fig|1041826.106.peg.2007</a>@~RNA Metabolism@^RNA processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_methylation&organism=1041826.106@1>RNA methylation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)&subsystem_name=RNA_methylation@1>16S rRNA (guanine(966)-N(2))-methyltransferase (EC 2.1.1.171)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.151@1>fig|1041826.106.peg.151</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=1041826.106@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma-54 factor RpoN&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma-54 factor RpoN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.628@1>fig|1041826.106.peg.628</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_initiation,_bacterial_sigma_factors&organism=1041826.106@1>Transcription initiation, bacterial sigma factors</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RNA polymerase sigma factor RpoD&subsystem_name=Transcription_initiation,_bacterial_sigma_factors@1>RNA polymerase sigma factor RpoD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.794@1>fig|1041826.106.peg.794</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=1041826.106@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase alpha subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2301@1>fig|1041826.106.peg.2301</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=1041826.106@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase beta@1 subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2185@1>fig|1041826.106.peg.2185</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=1041826.106@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase omega subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1491@1>fig|1041826.106.peg.1491</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RNA_polymerase_bacterial&organism=1041826.106@1>RNA polymerase bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)&subsystem_name=RNA_polymerase_bacterial@1>DNA-directed RNA polymerase beta subunit (EC 2.7.7.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2186@1>fig|1041826.106.peg.2186</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=1041826.106@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination protein NusB&subsystem_name=Transcription_factors_bacterial@1>Transcription termination protein NusB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.126@1>fig|1041826.106.peg.126</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=1041826.106@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription accessory protein (S1 RNA-binding domain)&subsystem_name=Transcription_factors_bacterial@1>Transcription accessory protein (S1 RNA-binding domain)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1830@1>fig|1041826.106.peg.1830</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=1041826.106@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription elongation factor GreA&subsystem_name=Transcription_factors_bacterial@1>Transcription elongation factor GreA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2128@1>fig|1041826.106.peg.2128</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=1041826.106@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination factor Rho&subsystem_name=Transcription_factors_bacterial@1>Transcription termination factor Rho</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2245@1>fig|1041826.106.peg.2245</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=1041826.106@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription antitermination protein NusG&subsystem_name=Transcription_factors_bacterial@1>Transcription antitermination protein NusG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2191@1>fig|1041826.106.peg.2191</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=1041826.106@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription-repair coupling factor&subsystem_name=Transcription_factors_bacterial@1>Transcription-repair coupling factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1753@1>fig|1041826.106.peg.1753</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Transcription_factors_bacterial&organism=1041826.106@1>Transcription factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcription termination protein NusA&subsystem_name=Transcription_factors_bacterial@1>Transcription termination protein NusA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2162@1>fig|1041826.106.peg.2162</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=1041826.106@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrite-sensitive transcriptional repressor NsrR&subsystem_name=Rrf2_family_transcriptional_regulators@1>Nitrite-sensitive transcriptional repressor NsrR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1641@1>fig|1041826.106.peg.1641</a>@~RNA Metabolism@^Transcription@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Rrf2_family_transcriptional_regulators&organism=1041826.106@1>Rrf2 family transcriptional regulators</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Rrf2 family transcriptional regulator&subsystem_name=Rrf2_family_transcriptional_regulators@1>Rrf2 family transcriptional regulator</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.449@1>fig|1041826.106.peg.449</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside diphosphate kinase (EC 2.7.4.6)&subsystem_name=pyrimidine_conversions@1>Nucleoside diphosphate kinase (EC 2.7.4.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1538@1>fig|1041826.106.peg.1538</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thioredoxin reductase (EC 1.8.1.9)&subsystem_name=pyrimidine_conversions@1>Thioredoxin reductase (EC 1.8.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1683@1>fig|1041826.106.peg.1683</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2526@1>fig|1041826.106.peg.2526</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytidylate kinase (EC 2.7.4.25)&subsystem_name=pyrimidine_conversions@1>Cytidylate kinase (EC 2.7.4.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.65@1>fig|1041826.106.peg.65</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidine kinase (EC 2.7.1.21)&subsystem_name=pyrimidine_conversions@1>Thymidine kinase (EC 2.7.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.291@1>fig|1041826.106.peg.291</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=pyrimidine_conversions@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.232@1>fig|1041826.106.peg.232</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.368@1>fig|1041826.106.peg.368</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-nucleotidase (EC 3.1.3.5)&subsystem_name=pyrimidine_conversions@1>5@1-nucleotidase (EC 3.1.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1495@1>fig|1041826.106.peg.1495</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1496@1>fig|1041826.106.peg.1496</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytidine deaminase (EC 3.5.4.5)&subsystem_name=pyrimidine_conversions@1>Cytidine deaminase (EC 3.5.4.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2286@1>fig|1041826.106.peg.2286</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thymidylate synthase (EC 2.1.1.45)&subsystem_name=pyrimidine_conversions@1>Thymidylate synthase (EC 2.1.1.45)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.866@1>fig|1041826.106.peg.866</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uridine kinase (EC 2.7.1.48)&subsystem_name=pyrimidine_conversions@1>Uridine kinase (EC 2.7.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1584@1>fig|1041826.106.peg.1584</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Uracil phosphoribosyltransferase (EC 2.4.2.9)&subsystem_name=pyrimidine_conversions@1>Uracil phosphoribosyltransferase (EC 2.4.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1731@1>fig|1041826.106.peg.1731</a>@~Nucleosides and Nucleotides@^Pyrimidines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=pyrimidine_conversions&organism=1041826.106@1>pyrimidine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=CTP synthase (EC 6.3.4.2)&subsystem_name=pyrimidine_conversions@1>CTP synthase (EC 6.3.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.617@1>fig|1041826.106.peg.617</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylamine--glycine ligase (EC 6.3.4.13)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylamine--glycine ligase (EC 6.3.4.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1734@1>fig|1041826.106.peg.1734</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=IMP cyclohydrolase (EC 3.5.4.10)&subsystem_name=De_Novo_Purine_Biosynthesis@1>IMP cyclohydrolase (EC 3.5.4.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.880@1>fig|1041826.106.peg.880</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Amidophosphoribosyltransferase (EC 2.4.2.14)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Amidophosphoribosyltransferase (EC 2.4.2.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2744@1>fig|1041826.106.peg.2744</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2110@1>fig|1041826.106.peg.2110</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2723@1>fig|1041826.106.peg.2723</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazole-succinocarboxamide synthase (EC 6.3.2.6)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylaminoimidazole-succinocarboxamide synthase (EC 6.3.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1639@1>fig|1041826.106.peg.1639</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2393@1>fig|1041826.106.peg.2393</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylglycinamide formyltransferase (EC 2.1.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2765@1>fig|1041826.106.peg.2765</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate lyase (EC 4.3.2.2)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Adenylosuccinate lyase (EC 4.3.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.179@1>fig|1041826.106.peg.179</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformylglycinamidine cyclo-ligase (EC 6.3.3.1)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylformylglycinamidine cyclo-ligase (EC 6.3.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.104@1>fig|1041826.106.peg.104</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylaminoimidazolecarboxamide formyltransferase (EC 2.1.2.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.880@1>fig|1041826.106.peg.880</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylformylglycinamidine synthase, synthetase subunit (EC 6.3.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1378@1>fig|1041826.106.peg.1378</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=De_Novo_Purine_Biosynthesis&organism=1041826.106@1>De Novo Purine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylformylglycinamidine synthase, glutamine amidotransferase subunit (EC 6.3.5.3)&subsystem_name=De_Novo_Purine_Biosynthesis@1>Phosphoribosylformylglycinamidine synthase, glutamine amidotransferase subunit (EC 6.3.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1378@1>fig|1041826.106.peg.1378</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=1041826.106@1>GMP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)&subsystem_name=GMP_synthase@1>GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.622@1>fig|1041826.106.peg.622</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=GMP_synthase&organism=1041826.106@1>GMP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)&subsystem_name=GMP_synthase@1>GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.622@1>fig|1041826.106.peg.622</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside diphosphate kinase (EC 2.7.4.6)&subsystem_name=Purine_conversions@1>Nucleoside diphosphate kinase (EC 2.7.4.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1538@1>fig|1041826.106.peg.1538</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylate kinase (EC 2.7.4.3)&subsystem_name=Purine_conversions@1>Adenylate kinase (EC 2.7.4.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2369@1>fig|1041826.106.peg.2369</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyguanosine kinase (EC 2.7.1.113)&subsystem_name=Purine_conversions@1>Deoxyguanosine kinase (EC 2.7.1.113)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.863@1>fig|1041826.106.peg.863</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.937@1>fig|1041826.106.peg.937</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate synthetase (EC 6.3.4.4)&subsystem_name=Purine_conversions@1>Adenylosuccinate synthetase (EC 6.3.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.826@1>fig|1041826.106.peg.826</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dNTP triphosphohydrolase, broad substrate specificity, subgroup 3&subsystem_name=Purine_conversions@1>dNTP triphosphohydrolase, broad substrate specificity, subgroup 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2148@1>fig|1041826.106.peg.2148</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)&subsystem_name=Purine_conversions@1>GMP synthase [glutamine-hydrolyzing], amidotransferase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.622@1>fig|1041826.106.peg.622</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inosine-5@1-monophosphate dehydrogenase (EC 1.1.1.205)&subsystem_name=Purine_conversions@1>Inosine-5@1-monophosphate dehydrogenase (EC 1.1.1.205)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.404@1>fig|1041826.106.peg.404</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)&subsystem_name=Purine_conversions@1>Hypoxanthine-guanine phosphoribosyltransferase (EC 2.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2370@1>fig|1041826.106.peg.2370</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Purine_conversions@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.232@1>fig|1041826.106.peg.232</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.368@1>fig|1041826.106.peg.368</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenylosuccinate lyase (EC 4.3.2.2)&subsystem_name=Purine_conversions@1>Adenylosuccinate lyase (EC 4.3.2.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.179@1>fig|1041826.106.peg.179</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5@1-nucleotidase (EC 3.1.3.5)&subsystem_name=Purine_conversions@1>5@1-nucleotidase (EC 3.1.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1495@1>fig|1041826.106.peg.1495</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1496@1>fig|1041826.106.peg.1496</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)&subsystem_name=Purine_conversions@1>GMP synthase [glutamine-hydrolyzing], ATP pyrophosphatase subunit (EC 6.3.5.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.622@1>fig|1041826.106.peg.622</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Purine_conversions@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.493@1>fig|1041826.106.peg.493</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Adenine phosphoribosyltransferase (EC 2.4.2.7)&subsystem_name=Purine_conversions@1>Adenine phosphoribosyltransferase (EC 2.4.2.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.200@1>fig|1041826.106.peg.200</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyadenosine kinase (EC 2.7.1.76)&subsystem_name=Purine_conversions@1>Deoxyadenosine kinase (EC 2.7.1.76)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.863@1>fig|1041826.106.peg.863</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.937@1>fig|1041826.106.peg.937</a>@~Nucleosides and Nucleotides@^Purines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Purine_conversions&organism=1041826.106@1>Purine conversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Guanylate kinase (EC 2.7.4.8)&subsystem_name=Purine_conversions@1>Guanylate kinase (EC 2.7.4.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2709@1>fig|1041826.106.peg.2709</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=1041826.106@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class Ia (aerobic), beta subunit (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2152@1>fig|1041826.106.peg.2152</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribonucleotide_reduction&organism=1041826.106@1>Ribonucleotide reduction</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)&subsystem_name=Ribonucleotide_reduction@1>Ribonucleotide reductase of class Ia (aerobic), alpha subunit (EC 1.17.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2151@1>fig|1041826.106.peg.2151</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=dNTP_triphosphohydrolase_protein_family&organism=1041826.106@1>dNTP triphosphohydrolase protein family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=dNTP triphosphohydrolase, broad substrate specificity, subgroup 3&subsystem_name=dNTP_triphosphohydrolase_protein_family@1>dNTP triphosphohydrolase, broad substrate specificity, subgroup 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2148@1>fig|1041826.106.peg.2148</a>@~Nucleosides and Nucleotides@^Nucleosides and Nucleotides - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Adenosyl_nucleosidases&organism=1041826.106@1>Adenosyl nucleosidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Adenosyl_nucleosidases@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.232@1>fig|1041826.106.peg.232</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.368@1>fig|1041826.106.peg.368</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nucleoside_triphosphate_pyrophosphohydrolase_MazG&organism=1041826.106@1>Nucleoside triphosphate pyrophosphohydrolase MazG</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)&subsystem_name=Nucleoside_triphosphate_pyrophosphohydrolase_MazG@1>Nucleoside triphosphate pyrophosphohydrolase MazG (EC 3.6.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1292@1>fig|1041826.106.peg.1292</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nudix_proteins_(nucleoside_triphosphate_hydrolases)&organism=1041826.106@1>Nudix proteins (nucleoside triphosphate hydrolases)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)&subsystem_name=Nudix_proteins_(nucleoside_triphosphate_hydrolases)@1>Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1999@1>fig|1041826.106.peg.1999</a>@~Nucleosides and Nucleotides@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Housecleaning_nucleoside_triphosphate_pyrophosphatases&organism=1041826.106@1>Housecleaning nucleoside triphosphate pyrophosphatases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)&subsystem_name=Housecleaning_nucleoside_triphosphate_pyrophosphatases@1>Deoxyuridine 5@1-triphosphate nucleotidohydrolase (EC 3.6.1.23)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1999@1>fig|1041826.106.peg.1999</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=1041826.106@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Heat shock protein GrpE&subsystem_name=Protein_chaperones@1>Heat shock protein GrpE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2512@1>fig|1041826.106.peg.2512</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=1041826.106@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein DnaK&subsystem_name=Protein_chaperones@1>Chaperone protein DnaK</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1506@1>fig|1041826.106.peg.1506</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=1041826.106@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein DnaJ&subsystem_name=Protein_chaperones@1>Chaperone protein DnaJ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2511@1>fig|1041826.106.peg.2511</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_chaperones&organism=1041826.106@1>Protein chaperones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chaperone protein HtpG&subsystem_name=Protein_chaperones@1>Chaperone protein HtpG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1313@1>fig|1041826.106.peg.1313</a>@~Protein Metabolism@^Protein folding@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_disulfide_interchange&organism=1041826.106@1>Periplasmic disulfide interchange</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)&subsystem_name=Periplasmic_disulfide_interchange@1>Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1288@1>fig|1041826.106.peg.1288</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Val&organism=1041826.106@1>tRNA aminoacylation, Val</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Valyl-tRNA synthetase (EC 6.1.1.9)&subsystem_name=tRNA_aminoacylation,_Val@1>Valyl-tRNA synthetase (EC 6.1.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.157@1>fig|1041826.106.peg.157</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Met&organism=1041826.106@1>tRNA aminoacylation, Met</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionyl-tRNA synthetase (EC 6.1.1.10)&subsystem_name=tRNA_aminoacylation,_Met@1>Methionyl-tRNA synthetase (EC 6.1.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1758@1>fig|1041826.106.peg.1758</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ile&organism=1041826.106@1>tRNA aminoacylation, Ile</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isoleucyl-tRNA synthetase (EC 6.1.1.5)&subsystem_name=tRNA_aminoacylation,_Ile@1>Isoleucyl-tRNA synthetase (EC 6.1.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2761@1>fig|1041826.106.peg.2761</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Arg&organism=1041826.106@1>tRNA aminoacylation, Arg</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Arginyl-tRNA synthetase (EC 6.1.1.19)&subsystem_name=tRNA_aminoacylation,_Arg@1>Arginyl-tRNA synthetase (EC 6.1.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1183@1>fig|1041826.106.peg.1183</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=1041826.106@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor SUI1-related protein&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor SUI1-related protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.369@1>fig|1041826.106.peg.369</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=1041826.106@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 1&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2305@1>fig|1041826.106.peg.2305</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=1041826.106@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionyl-tRNA formyltransferase (EC 2.1.2.9)&subsystem_name=Translation_initiation_factors_bacterial@1>Methionyl-tRNA formyltransferase (EC 2.1.2.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1025@1>fig|1041826.106.peg.1025</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=1041826.106@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 3&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.143@1>fig|1041826.106.peg.143</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=1041826.106@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation initiation factor 2&subsystem_name=Translation_initiation_factors_bacterial@1>Translation initiation factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2163@1>fig|1041826.106.peg.2163</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_initiation_factors_bacterial&organism=1041826.106@1>Translation initiation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosome-binding factor A&subsystem_name=Translation_initiation_factors_bacterial@1>Ribosome-binding factor A</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2117@1>fig|1041826.106.peg.2117</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S11p (S14e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S11p (S14e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2303@1>fig|1041826.106.peg.2303</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S1p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S1p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.67@1>fig|1041826.106.peg.67</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S6p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S6p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.280@1>fig|1041826.106.peg.280</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S15p (S13e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S15p (S13e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.792@1>fig|1041826.106.peg.792</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S14p (S29e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S14p (S29e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2313@1>fig|1041826.106.peg.2313</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S5p (S2e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S5p (S2e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2309@1>fig|1041826.106.peg.2309</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S18p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S18p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.281@1>fig|1041826.106.peg.281</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S17p (S11e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S17p (S11e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2317@1>fig|1041826.106.peg.2317</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S20p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S20p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1544@1>fig|1041826.106.peg.1544</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S12p (S23e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S12p (S23e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2330@1>fig|1041826.106.peg.2330</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S9p (S16e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S9p (S16e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.684@1>fig|1041826.106.peg.684</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S8p (S15Ae)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S8p (S15Ae)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2312@1>fig|1041826.106.peg.2312</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S4p (S9e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S4p (S9e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2302@1>fig|1041826.106.peg.2302</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S14p (S29e), zinc-independent&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S14p (S29e), zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2313@1>fig|1041826.106.peg.2313</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S19p (S15e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S19p (S15e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2322@1>fig|1041826.106.peg.2322</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S10p (S20e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S10p (S20e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2327@1>fig|1041826.106.peg.2327</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S16p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S16p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2134@1>fig|1041826.106.peg.2134</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S3p (S3e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S3p (S3e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2320@1>fig|1041826.106.peg.2320</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S18p, zinc-independent&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S18p, zinc-independent</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.281@1>fig|1041826.106.peg.281</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S7p (S5e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S7p (S5e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2329@1>fig|1041826.106.peg.2329</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S2p (SAe)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S2p (SAe)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.683@1>fig|1041826.106.peg.683</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S21p&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S21p</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2196@1>fig|1041826.106.peg.2196</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Ribosome_SSU_bacterial&organism=1041826.106@1>Ribosome SSU bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=SSU ribosomal protein S13p (S18e)&subsystem_name=Ribosome_SSU_bacterial@1>SSU ribosomal protein S13p (S18e)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2304@1>fig|1041826.106.peg.2304</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Gly&organism=1041826.106@1>tRNA aminoacylation, Gly</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycyl-tRNA synthetase (EC 6.1.1.14)&subsystem_name=tRNA_aminoacylation,_Gly@1>Glycyl-tRNA synthetase (EC 6.1.1.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1951@1>fig|1041826.106.peg.1951</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ala&organism=1041826.106@1>tRNA aminoacylation, Ala</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanyl-tRNA synthetase (EC 6.1.1.7)&subsystem_name=tRNA_aminoacylation,_Ala@1>Alanyl-tRNA synthetase (EC 6.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2273@1>fig|1041826.106.peg.2273</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Trp&organism=1041826.106@1>tRNA aminoacylation, Trp</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophanyl-tRNA synthetase (EC 6.1.1.2)&subsystem_name=tRNA_aminoacylation,_Trp@1>Tryptophanyl-tRNA synthetase (EC 6.1.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2094@1>fig|1041826.106.peg.2094</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=1041826.106@1>Programmed frameshift</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=programmed frameshift-containing&subsystem_name=Programmed_frameshift@1>programmed frameshift-containing</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.407@1>fig|1041826.106.peg.407</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Programmed_frameshift&organism=1041826.106@1>Programmed frameshift</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 2&subsystem_name=Programmed_frameshift@1>Peptide chain release factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.407@1>fig|1041826.106.peg.407</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Cys&organism=1041826.106@1>tRNA aminoacylation, Cys</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteinyl-tRNA synthetase (EC 6.1.1.16)&subsystem_name=tRNA_aminoacylation,_Cys@1>Cysteinyl-tRNA synthetase (EC 6.1.1.16)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.319@1>fig|1041826.106.peg.319</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribosome recycling factor&subsystem_name=Translation_termination_factors_bacterial@1>Ribosome recycling factor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.669@1>fig|1041826.106.peg.669</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor homolog&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1595@1>fig|1041826.106.peg.1595</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidyl-tRNA hydrolase (EC 3.1.1.29)&subsystem_name=Translation_termination_factors_bacterial@1>Peptidyl-tRNA hydrolase (EC 3.1.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2108@1>fig|1041826.106.peg.2108</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide deformylase (EC 3.5.1.88)&subsystem_name=Translation_termination_factors_bacterial@1>Peptide deformylase (EC 3.5.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1294@1>fig|1041826.106.peg.1294</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 3&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 3</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2181@1>fig|1041826.106.peg.2181</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tmRNA-binding protein SmpB&subsystem_name=Translation_termination_factors_bacterial@1>tmRNA-binding protein SmpB</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1602@1>fig|1041826.106.peg.1602</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 1&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1358@1>fig|1041826.106.peg.1358</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methionine aminopeptidase (EC 3.4.11.18)&subsystem_name=Translation_termination_factors_bacterial@1>Methionine aminopeptidase (EC 3.4.11.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.634@1>fig|1041826.106.peg.634</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_termination_factors_bacterial&organism=1041826.106@1>Translation termination factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptide chain release factor 2&subsystem_name=Translation_termination_factors_bacterial@1>Peptide chain release factor 2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.407@1>fig|1041826.106.peg.407</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_His&organism=1041826.106@1>tRNA aminoacylation, His</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidyl-tRNA synthetase (EC 6.1.1.21)&subsystem_name=tRNA_aminoacylation,_His@1>Histidyl-tRNA synthetase (EC 6.1.1.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1014@1>fig|1041826.106.peg.1014</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=1041826.106@1>tRNA aminoacylation, Asp and Asn</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartyl-tRNA synthetase (EC 6.1.1.12)&subsystem_name=tRNA_aminoacylation,_Asp_and_Asn@1>Aspartyl-tRNA synthetase (EC 6.1.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2139@1>fig|1041826.106.peg.2139</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Asp_and_Asn&organism=1041826.106@1>tRNA aminoacylation, Asp and Asn</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Asparaginyl-tRNA synthetase (EC 6.1.1.22)&subsystem_name=tRNA_aminoacylation,_Asp_and_Asn@1>Asparaginyl-tRNA synthetase (EC 6.1.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.627@1>fig|1041826.106.peg.627</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=1041826.106@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2328@1>fig|1041826.106.peg.2328</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=1041826.106@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor LepA&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor LepA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1911@1>fig|1041826.106.peg.1911</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=1041826.106@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Tu&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor Tu</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2193@1>fig|1041826.106.peg.2193</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=1041826.106@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor Ts&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor Ts</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.682@1>fig|1041826.106.peg.682</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factors_bacterial&organism=1041826.106@1>Translation elongation factors bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor P&subsystem_name=Translation_elongation_factors_bacterial@1>Translation elongation factor P</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.56@1>fig|1041826.106.peg.56</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Phe-GAA&subsystem_name=tRNAs@1>tRNA-Phe-GAA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.31@1>fig|1041826.106.rna.31</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.32@1>fig|1041826.106.rna.32</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Arg-ACG&subsystem_name=tRNAs@1>tRNA-Arg-ACG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.9@1>fig|1041826.106.rna.9</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.10@1>fig|1041826.106.rna.10</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Trp-CCA&subsystem_name=tRNAs@1>tRNA-Trp-CCA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.79@1>fig|1041826.106.rna.79</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Leu-CAA&subsystem_name=tRNAs@1>tRNA-Leu-CAA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.74@1>fig|1041826.106.rna.74</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Gly-GCC&subsystem_name=tRNAs@1>tRNA-Gly-GCC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.34@1>fig|1041826.106.rna.34</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.37@1>fig|1041826.106.rna.37</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Leu-GAG&subsystem_name=tRNAs@1>tRNA-Leu-GAG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.65@1>fig|1041826.106.rna.65</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Arg-CCG&subsystem_name=tRNAs@1>tRNA-Arg-CCG</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.30@1>fig|1041826.106.rna.30</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Ala-GGC&subsystem_name=tRNAs@1>tRNA-Ala-GGC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.76@1>fig|1041826.106.rna.76</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Ser-GGA&subsystem_name=tRNAs@1>tRNA-Ser-GGA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.41@1>fig|1041826.106.rna.41</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.84@1>fig|1041826.106.rna.84</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNAs&organism=1041826.106@1>tRNAs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=tRNA-Cys-GCA&subsystem_name=tRNAs@1>tRNA-Cys-GCA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.rna.72@1>fig|1041826.106.rna.72</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Lys&organism=1041826.106@1>tRNA aminoacylation, Lys</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)&subsystem_name=tRNA_aminoacylation,_Lys@1>Lysyl-tRNA synthetase (class II) (EC 6.1.1.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2212@1>fig|1041826.106.peg.2212</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Thr&organism=1041826.106@1>tRNA aminoacylation, Thr</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Threonyl-tRNA synthetase (EC 6.1.1.3)&subsystem_name=tRNA_aminoacylation,_Thr@1>Threonyl-tRNA synthetase (EC 6.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.144@1>fig|1041826.106.peg.144</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Pro&organism=1041826.106@1>tRNA aminoacylation, Pro</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prolyl-tRNA synthetase (EC 6.1.1.15), archaeal/eukaryal type&subsystem_name=tRNA_aminoacylation,_Pro@1>Prolyl-tRNA synthetase (EC 6.1.1.15), archaeal/eukaryal type</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1545@1>fig|1041826.106.peg.1545</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Translation_elongation_factor_G_family&organism=1041826.106@1>Translation elongation factor G family</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor G&subsystem_name=Translation_elongation_factor_G_family@1>Translation elongation factor G</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2328@1>fig|1041826.106.peg.2328</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=1041826.106@1>tRNA aminoacylation, Glu and Gln</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamyl-tRNA synthetase (EC 6.1.1.17)&subsystem_name=tRNA_aminoacylation,_Glu_and_Gln@1>Glutamyl-tRNA synthetase (EC 6.1.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1851@1>fig|1041826.106.peg.1851</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Glu_and_Gln&organism=1041826.106@1>tRNA aminoacylation, Glu and Gln</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutaminyl-tRNA synthetase (EC 6.1.1.18)&subsystem_name=tRNA_aminoacylation,_Glu_and_Gln@1>Glutaminyl-tRNA synthetase (EC 6.1.1.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1847@1>fig|1041826.106.peg.1847</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Ser&organism=1041826.106@1>tRNA aminoacylation, Ser</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Seryl-tRNA synthetase (EC 6.1.1.11)&subsystem_name=tRNA_aminoacylation,_Ser@1>Seryl-tRNA synthetase (EC 6.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2002@1>fig|1041826.106.peg.2002</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Tyr&organism=1041826.106@1>tRNA aminoacylation, Tyr</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tyrosyl-tRNA synthetase (EC 6.1.1.1)&subsystem_name=tRNA_aminoacylation,_Tyr@1>Tyrosyl-tRNA synthetase (EC 6.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2814@1>fig|1041826.106.peg.2814</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Leu&organism=1041826.106@1>tRNA aminoacylation, Leu</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Leucyl-tRNA synthetase (EC 6.1.1.4)&subsystem_name=tRNA_aminoacylation,_Leu@1>Leucyl-tRNA synthetase (EC 6.1.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1877@1>fig|1041826.106.peg.1877</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=1041826.106@1>tRNA aminoacylation, Phe</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)&subsystem_name=tRNA_aminoacylation,_Phe@1>Phenylalanyl-tRNA synthetase alpha chain (EC 6.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2729@1>fig|1041826.106.peg.2729</a>@~Protein Metabolism@^Protein biosynthesis@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=tRNA_aminoacylation,_Phe&organism=1041826.106@1>tRNA aminoacylation, Phe</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)&subsystem_name=tRNA_aminoacylation,_Phe@1>Phenylalanyl-tRNA synthetase beta chain (EC 6.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1200@1>fig|1041826.106.peg.1200</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein-L-isoaspartate_O-methyltransferase&organism=1041826.106@1>Protein-L-isoaspartate O-methyltransferase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)&subsystem_name=Protein-L-isoaspartate_O-methyltransferase@1>Protein-L-isoaspartate O-methyltransferase (EC 2.1.1.77)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1603@1>fig|1041826.106.peg.1603</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=1041826.106@1>Signal peptidase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Signal peptidase I (EC 3.4.21.89)&subsystem_name=Signal_peptidase@1>Signal peptidase I (EC 3.4.21.89)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2563@1>fig|1041826.106.peg.2563</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Signal_peptidase&organism=1041826.106@1>Signal peptidase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Lipoprotein signal peptidase (EC 3.4.23.36)&subsystem_name=Signal_peptidase@1>Lipoprotein signal peptidase (EC 3.4.23.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2763@1>fig|1041826.106.peg.2763</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=1041826.106@1>Modification of eukaryotic initiation factor 5A</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyhypusine synthase (EC 2.5.1.46)&subsystem_name=Modification_of_eukaryotic_initiation_factor_5A@1>Deoxyhypusine synthase (EC 2.5.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.73@1>fig|1041826.106.peg.73</a>@~Protein Metabolism@^Protein processing and modification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Modification_of_eukaryotic_initiation_factor_5A&organism=1041826.106@1>Modification of eukaryotic initiation factor 5A</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Translation elongation factor P&subsystem_name=Modification_of_eukaryotic_initiation_factor_5A@1>Translation elongation factor P</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.56@1>fig|1041826.106.peg.56</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=1041826.106@1>Aminopeptidases (EC 3.4.11.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Xaa-Pro aminopeptidase (EC 3.4.11.9)&subsystem_name=Aminopeptidases_(EC_3.4.11.-)@1>Xaa-Pro aminopeptidase (EC 3.4.11.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1923@1>fig|1041826.106.peg.1923</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aminopeptidases_(EC_3.4.11.-)&organism=1041826.106@1>Aminopeptidases (EC 3.4.11.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminopeptidase Y (Arg, Lys, Leu preference) (EC 3.4.11.15)&subsystem_name=Aminopeptidases_(EC_3.4.11.-)@1>Aminopeptidase Y (Arg, Lys, Leu preference) (EC 3.4.11.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1220@1>fig|1041826.106.peg.1220</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=1041826.106@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)&subsystem_name=Protein_degradation@1>Leucyl/phenylalanyl-tRNA--protein transferase (EC 2.3.2.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1898@1>fig|1041826.106.peg.1898</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=1041826.106@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)&subsystem_name=Protein_degradation@1>Dipeptidyl carboxypeptidase Dcp (EC 3.4.15.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2375@1>fig|1041826.106.peg.2375</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Protein_degradation&organism=1041826.106@1>Protein degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminopeptidase C (EC 3.4.22.40)&subsystem_name=Protein_degradation@1>Aminopeptidase C (EC 3.4.22.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1708@1>fig|1041826.106.peg.1708</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Metallocarboxypeptidases_(EC_3.4.17.-)&organism=1041826.106@1>Metallocarboxypeptidases (EC 3.4.17.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)&subsystem_name=Metallocarboxypeptidases_(EC_3.4.17.-)@1>Muramoyltetrapeptide carboxypeptidase (EC 3.4.17.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1756@1>fig|1041826.106.peg.1756</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dipeptidases_(EC_3.4.13.-)&organism=1041826.106@1>Dipeptidases (EC 3.4.13.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)&subsystem_name=Dipeptidases_(EC_3.4.13.-)@1>Alpha-aspartyl dipeptidase Peptidase E (EC 3.4.13.21)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.708@1>fig|1041826.106.peg.708</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_endopeptidase_(EC_3.4.21.-)&organism=1041826.106@1>Serine endopeptidase (EC 3.4.21.-)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prolyl endopeptidase (EC 3.4.21.26)&subsystem_name=Serine_endopeptidase_(EC_3.4.21.-)@1>Prolyl endopeptidase (EC 3.4.21.26)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1266@1>fig|1041826.106.peg.1266</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=1041826.106@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RadA&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>DNA repair protein RadA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2804@1>fig|1041826.106.peg.2804</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=1041826.106@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent Clp protease ATP-binding subunit ClpX&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent Clp protease ATP-binding subunit ClpX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.942@1>fig|1041826.106.peg.942</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=1041826.106@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent Clp protease, ATP-binding subunit ClpC&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent Clp protease, ATP-binding subunit ClpC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.111@1>fig|1041826.106.peg.111</a>@~Protein Metabolism@^Protein degradation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proteolysis_in_bacteria,_ATP-dependent&organism=1041826.106@1>Proteolysis in bacteria, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent protease La (EC 3.4.21.53) Type I&subsystem_name=Proteolysis_in_bacteria,_ATP-dependent@1>ATP-dependent protease La (EC 3.4.21.53) Type I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.63@1>fig|1041826.106.peg.63</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=1041826.106@1>Stringent Response, (p)ppGpp metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II&subsystem_name=Stringent_Response,_(p)ppGpp_metabolism@1>GTP pyrophosphokinase (EC 2.7.6.5), (p)ppGpp synthetase II</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.311@1>fig|1041826.106.peg.311</a>@~Regulation and Cell signaling@^Regulation and Cell signaling - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Stringent_Response,_(p)ppGpp_metabolism&organism=1041826.106@1>Stringent Response, (p)ppGpp metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Guanosine-3@1,5@1-bis(diphosphate) 3@1-pyrophosphohydrolase (EC 3.1.7.2)&subsystem_name=Stringent_Response,_(p)ppGpp_metabolism@1>Guanosine-3@1,5@1-bis(diphosphate) 3@1-pyrophosphohydrolase (EC 3.1.7.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.311@1>fig|1041826.106.peg.311</a>@~Secondary Metabolism@^Plant Alkaloids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alkaloid_biosynthesis_from_L-lysine&organism=1041826.106@1>Alkaloid biosynthesis from L-lysine</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyhypusine synthase (EC 2.5.1.46)&subsystem_name=Alkaloid_biosynthesis_from_L-lysine@1>Deoxyhypusine synthase (EC 2.5.1.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.73@1>fig|1041826.106.peg.73</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=1041826.106@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Auxin_biosynthesis@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.218@1>fig|1041826.106.peg.218</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=1041826.106@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Auxin_biosynthesis@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.222@1>fig|1041826.106.peg.222</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=1041826.106@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Auxin_biosynthesis@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.219@1>fig|1041826.106.peg.219</a>@~Secondary Metabolism@^Plant Hormones@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Auxin_biosynthesis&organism=1041826.106@1>Auxin biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Auxin_biosynthesis@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.220@1>fig|1041826.106.peg.220</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=1041826.106@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination inhibitory protein MutS2&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>Recombination inhibitory protein MutS2</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.357@1>fig|1041826.106.peg.357</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=1041826.106@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=MutS-related protein, family 1&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>MutS-related protein, family 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.624@1>fig|1041826.106.peg.624</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=1041826.106@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutS&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>DNA mismatch repair protein MutS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.934@1>fig|1041826.106.peg.934</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_MutL-MutS_system&organism=1041826.106@1>DNA repair, bacterial MutL-MutS system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutL&subsystem_name=DNA_repair,_bacterial_MutL-MutS_system@1>DNA mismatch repair protein MutL</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2568@1>fig|1041826.106.peg.2568</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=1041826.106@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit A paralog in greater Bacteroides group&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit A paralog in greater Bacteroides group</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1221@1>fig|1041826.106.peg.1221</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=1041826.106@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit C&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit C</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1914@1>fig|1041826.106.peg.1914</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=1041826.106@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit A&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit A</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.199@1>fig|1041826.106.peg.199</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_UvrABC_system&organism=1041826.106@1>DNA repair, UvrABC system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Excinuclease ABC subunit B&subsystem_name=DNA_repair,_UvrABC_system@1>Excinuclease ABC subunit B</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1185@1>fig|1041826.106.peg.1185</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_photolyase&organism=1041826.106@1>DNA repair, bacterial photolyase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribodipyrimidine photolyase (EC 4.1.99.3)&subsystem_name=DNA_repair,_bacterial_photolyase@1>Deoxyribodipyrimidine photolyase (EC 4.1.99.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1556@1>fig|1041826.106.peg.1556</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=2-phosphoglycolate_salvage&organism=1041826.106@1>2-phosphoglycolate salvage</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoglycolate phosphatase (EC 3.1.3.18)&subsystem_name=2-phosphoglycolate_salvage@1>Phosphoglycolate phosphatase (EC 3.1.3.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.381@1>fig|1041826.106.peg.381</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RecA_and_RecX&organism=1041826.106@1>RecA and RecX</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=RecA_and_RecX@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2126@1>fig|1041826.106.peg.2126</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=RecA_and_RecX&organism=1041826.106@1>RecA and RecX</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Regulatory protein RecX&subsystem_name=RecA_and_RecX@1>Regulatory protein RecX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2696@1>fig|1041826.106.peg.2696</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=1041826.106@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA mismatch repair protein MutS&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>DNA mismatch repair protein MutS</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.934@1>fig|1041826.106.peg.934</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=1041826.106@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2126@1>fig|1041826.106.peg.2126</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein&organism=1041826.106@1>DNA repair system including RecA, MutS and a hypothetical protein</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Regulatory protein RecX&subsystem_name=DNA_repair_system_including_RecA,_MutS_and_a_hypothetical_protein@1>Regulatory protein RecX</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2696@1>fig|1041826.106.peg.2696</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RadA&subsystem_name=DNA_repair,_bacterial@1>DNA repair protein RadA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2804@1>fig|1041826.106.peg.2804</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA repair protein RecN&subsystem_name=DNA_repair,_bacterial@1>DNA repair protein RecN</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1488@1>fig|1041826.106.peg.1488</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination protein RmuC&subsystem_name=DNA_repair,_bacterial@1>DNA recombination protein RmuC</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1080@1>fig|1041826.106.peg.1080</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=A/G-specific adenine glycosylase (EC 3.2.2.-)&subsystem_name=DNA_repair,_bacterial@1>A/G-specific adenine glycosylase (EC 3.2.2.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.13@1>fig|1041826.106.peg.13</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-cytosine methyltransferase (EC 2.1.1.37)&subsystem_name=DNA_repair,_bacterial@1>DNA-cytosine methyltransferase (EC 2.1.1.37)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1589@1>fig|1041826.106.peg.1589</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1607@1>fig|1041826.106.peg.1607</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA polymerase IV (EC 2.7.7.7)&subsystem_name=DNA_repair,_bacterial@1>DNA polymerase IV (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2793@1>fig|1041826.106.peg.2793</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Endonuclease V (EC 3.1.21.7)&subsystem_name=DNA_repair,_bacterial@1>Endonuclease V (EC 3.1.21.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.989@1>fig|1041826.106.peg.989</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Single-stranded DNA-binding protein&subsystem_name=DNA_repair,_bacterial@1>Single-stranded DNA-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.12@1>fig|1041826.106.peg.12</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1796@1>fig|1041826.106.peg.1796</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)&subsystem_name=DNA_repair,_bacterial@1>Methylated-DNA--protein-cysteine methyltransferase (EC 2.1.1.63)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1975@1>fig|1041826.106.peg.1975</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair,_bacterial@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2126@1>fig|1041826.106.peg.2126</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial&organism=1041826.106@1>DNA repair, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exodeoxyribonuclease III (EC 3.1.11.2)&subsystem_name=DNA_repair,_bacterial@1>Exodeoxyribonuclease III (EC 3.1.11.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2798@1>fig|1041826.106.peg.2798</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=1041826.106@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP-dependent DNA helicase RecQ&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>ATP-dependent DNA helicase RecQ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1024@1>fig|1041826.106.peg.1024</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2672@1>fig|1041826.106.peg.2672</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=1041826.106@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Single-stranded DNA-binding protein&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>Single-stranded DNA-binding protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.12@1>fig|1041826.106.peg.12</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1796@1>fig|1041826.106.peg.1796</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=1041826.106@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination protein RecR&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>Recombination protein RecR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.770@1>fig|1041826.106.peg.770</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=1041826.106@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=RecA protein&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>RecA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2126@1>fig|1041826.106.peg.2126</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=1041826.106@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination and repair protein RecO&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>DNA recombination and repair protein RecO</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2758@1>fig|1041826.106.peg.2758</a>@~DNA Metabolism@^DNA repair@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_repair,_bacterial_RecFOR_pathway&organism=1041826.106@1>DNA repair, bacterial RecFOR pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA recombination and repair protein RecF&subsystem_name=DNA_repair,_bacterial_RecFOR_pathway@1>DNA recombination and repair protein RecF</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2571@1>fig|1041826.106.peg.2571</a>@~DNA Metabolism@^CRISPs@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=CRISPRs&organism=1041826.106@1>CRISPRs</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=CRISPR-associated protein Cas1&subsystem_name=CRISPRs@1>CRISPR-associated protein Cas1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.350@1>fig|1041826.106.peg.350</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=1041826.106@1>Type I Restriction-Modification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)&subsystem_name=Type_I_Restriction-Modification@1>Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.556@1>fig|1041826.106.peg.556</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Type_I_Restriction-Modification&organism=1041826.106@1>Type I Restriction-Modification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)&subsystem_name=Type_I_Restriction-Modification@1>Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.554@1>fig|1041826.106.peg.554</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2017@1>fig|1041826.106.peg.2017</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=1041826.106@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)&subsystem_name=Restriction-Modification_System@1>Type I restriction-modification system, restriction subunit R (EC 3.1.21.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.556@1>fig|1041826.106.peg.556</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Restriction-Modification_System&organism=1041826.106@1>Restriction-Modification System</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)&subsystem_name=Restriction-Modification_System@1>Type I restriction-modification system, DNA-methyltransferase subunit M (EC 2.1.1.72)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.554@1>fig|1041826.106.peg.554</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2017@1>fig|1041826.106.peg.2017</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=1041826.106@1>DNA structural proteins, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA-binding protein HU-beta&subsystem_name=DNA_structural_proteins,_bacterial@1>DNA-binding protein HU-beta</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1026@1>fig|1041826.106.peg.1026</a>@~DNA Metabolism@^DNA Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_structural_proteins,_bacterial&organism=1041826.106@1>DNA structural proteins, bacterial</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Integration host factor alpha/beta&subsystem_name=DNA_structural_proteins,_bacterial@1>Integration host factor alpha/beta</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.14@1>fig|1041826.106.peg.14</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_I,_ATP-independent&organism=1041826.106@1>DNA topoisomerases, Type I, ATP-independent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA topoisomerase I (EC 5.99.1.2)&subsystem_name=DNA_topoisomerases,_Type_I,_ATP-independent@1>DNA topoisomerase I (EC 5.99.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2441@1>fig|1041826.106.peg.2441</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=1041826.106@1>DNA topoisomerases, Type II, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit B (EC 5.99.1.3)&subsystem_name=DNA_topoisomerases,_Type_II,_ATP-dependent@1>DNA gyrase subunit B (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1436@1>fig|1041826.106.peg.1436</a>@~DNA Metabolism@^DNA replication@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_topoisomerases,_Type_II,_ATP-dependent&organism=1041826.106@1>DNA topoisomerases, Type II, ATP-dependent</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA gyrase subunit A (EC 5.99.1.3)&subsystem_name=DNA_topoisomerases,_Type_II,_ATP-dependent@1>DNA gyrase subunit A (EC 5.99.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.112@1>fig|1041826.106.peg.112</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=1041826.106@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Recombination protein RecR&subsystem_name=DNA_processing_cluster@1>Recombination protein RecR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.770@1>fig|1041826.106.peg.770</a>@~DNA Metabolism@^DNA uptake, competence@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=DNA_processing_cluster&organism=1041826.106@1>DNA processing cluster</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DNA polymerase III subunits gamma and tau (EC 2.7.7.7)&subsystem_name=DNA_processing_cluster@1>DNA polymerase III subunits gamma and tau (EC 2.7.7.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.81@1>fig|1041826.106.peg.81</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.82@1>fig|1041826.106.peg.82</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin carboxyl carrier protein of acetyl-CoA carboxylase&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Biotin carboxyl carrier protein of acetyl-CoA carboxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1710@1>fig|1041826.106.peg.1710</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Biotin carboxylase of acetyl-CoA carboxylase (EC 6.3.4.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1709@1>fig|1041826.106.peg.1709</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acetyl-coenzyme A carboxyl transferase beta chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.878@1>fig|1041826.106.peg.878</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acyl carrier protein&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acyl carrier protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2473@1>fig|1041826.106.peg.2473</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2486@1>fig|1041826.106.peg.2486</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2766@1>fig|1041826.106.peg.2766</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acetyl-coenzyme A carboxyl transferase alpha chain (EC 6.4.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.137@1>fig|1041826.106.peg.137</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>3-oxoacyl-[acyl-carrier-protein] synthase, KASI (EC 2.3.1.41)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2487@1>fig|1041826.106.peg.2487</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acyl carrier protein phosphodiesterase (EC 3.1.4.14)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Acyl carrier protein phosphodiesterase (EC 3.1.4.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.91@1>fig|1041826.106.peg.91</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>Malonyl CoA-acyl carrier protein transacylase (EC 2.3.1.39)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.641@1>fig|1041826.106.peg.641</a>@~Fatty Acids, Lipids, and Isoprenoids@^Fatty acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Fatty_Acid_Biosynthesis_FASII&organism=1041826.106@1>Fatty Acid Biosynthesis FASII</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)&subsystem_name=Fatty_Acid_Biosynthesis_FASII@1>3-hydroxyacyl-[acyl-carrier-protein] dehydratase, FabZ form (EC 4.2.1.59)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.54@1>fig|1041826.106.peg.54</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2466@1>fig|1041826.106.peg.2466</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2482@1>fig|1041826.106.peg.2482</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Myxoxanthophyll_biosynthesis_in_Cyanobacteria&organism=1041826.106@1>Myxoxanthophyll biosynthesis in Cyanobacteria</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-carotene hydroxylase&subsystem_name=Myxoxanthophyll_biosynthesis_in_Cyanobacteria@1>Beta-carotene hydroxylase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1559@1>fig|1041826.106.peg.1559</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=1041826.106@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diphosphomevalonate decarboxylase (EC 4.1.1.33)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Diphosphomevalonate decarboxylase (EC 4.1.1.33)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.912@1>fig|1041826.106.peg.912</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=1041826.106@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomevalonate kinase (EC 2.7.4.2)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Phosphomevalonate kinase (EC 2.7.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.911@1>fig|1041826.106.peg.911</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=1041826.106@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mevalonate kinase (EC 2.7.1.36)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Mevalonate kinase (EC 2.7.1.36)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.890@1>fig|1041826.106.peg.890</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=1041826.106@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Hydroxymethylglutaryl-CoA reductase (EC 1.1.1.34)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.891@1>fig|1041826.106.peg.891</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mevalonate_Branch_of_Isoprenoid_Biosynthesis&organism=1041826.106@1>Mevalonate Branch of Isoprenoid Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Mevalonate_Branch_of_Isoprenoid_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1347@1>fig|1041826.106.peg.1347</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=1041826.106@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Undecaprenyl diphosphate synthase (EC 2.5.1.31)&subsystem_name=Isoprenoinds_for_Quinones@1>Undecaprenyl diphosphate synthase (EC 2.5.1.31)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.951@1>fig|1041826.106.peg.951</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=1041826.106@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Geranylgeranyl diphosphate synthase (EC 2.5.1.29)&subsystem_name=Isoprenoinds_for_Quinones@1>Geranylgeranyl diphosphate synthase (EC 2.5.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2025@1>fig|1041826.106.peg.2025</a>@~Fatty Acids, Lipids, and Isoprenoids@^Isoprenoids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoprenoinds_for_Quinones&organism=1041826.106@1>Isoprenoinds for Quinones</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hexaprenyl diphosphate synthase (farnesyl-diphosphate specific) (EC 2.5.1.83)&subsystem_name=Isoprenoinds_for_Quinones@1>Hexaprenyl diphosphate synthase (farnesyl-diphosphate specific) (EC 2.5.1.83)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.841@1>fig|1041826.106.peg.841</a>@~Nitrogen Metabolism@^Nitrogen Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Nitrosative_stress&organism=1041826.106@1>Nitrosative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrite-sensitive transcriptional repressor NsrR&subsystem_name=Nitrosative_stress@1>Nitrite-sensitive transcriptional repressor NsrR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1641@1>fig|1041826.106.peg.1641</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric oxide reductase activation protein NorE&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitric oxide reductase activation protein NorE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.324@1>fig|1041826.106.peg.324</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric-oxide reductase subunit C (EC 1.7.99.7)&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitric-oxide reductase subunit C (EC 1.7.99.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.332@1>fig|1041826.106.peg.332</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric oxide reductase activation protein NorQ&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitric oxide reductase activation protein NorQ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.326@1>fig|1041826.106.peg.326</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric oxide reductase activation protein NorD&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitric oxide reductase activation protein NorD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.325@1>fig|1041826.106.peg.325</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrous oxide reductase maturation protein NosD&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitrous oxide reductase maturation protein NosD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2227@1>fig|1041826.106.peg.2227</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrous oxide reductase maturation protein NosF (ATPase)&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitrous oxide reductase maturation protein NosF (ATPase)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2226@1>fig|1041826.106.peg.2226</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrous oxide reductase maturation transmembrane protein NosY&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitrous oxide reductase maturation transmembrane protein NosY</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2225@1>fig|1041826.106.peg.2225</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrous-oxide reductase (EC 1.7.99.6)&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitrous-oxide reductase (EC 1.7.99.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2230@1>fig|1041826.106.peg.2230</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-containing nitrite reductase (EC 1.7.2.1)&subsystem_name=Denitrifying_reductase_gene_clusters@1>Copper-containing nitrite reductase (EC 1.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.329@1>fig|1041826.106.peg.329</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrite reductase accessory protein NirV&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitrite reductase accessory protein NirV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.328@1>fig|1041826.106.peg.328</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrifying_reductase_gene_clusters&organism=1041826.106@1>Denitrifying reductase gene clusters</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric-oxide reductase subunit B (EC 1.7.99.7)&subsystem_name=Denitrifying_reductase_gene_clusters@1>Nitric-oxide reductase subunit B (EC 1.7.99.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.331@1>fig|1041826.106.peg.331</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric oxide reductase activation protein NorE&subsystem_name=Denitrification@1>Nitric oxide reductase activation protein NorE</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.324@1>fig|1041826.106.peg.324</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric-oxide reductase subunit C (EC 1.7.99.7)&subsystem_name=Denitrification@1>Nitric-oxide reductase subunit C (EC 1.7.99.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.332@1>fig|1041826.106.peg.332</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric oxide reductase activation protein NorQ&subsystem_name=Denitrification@1>Nitric oxide reductase activation protein NorQ</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.326@1>fig|1041826.106.peg.326</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric oxide reductase activation protein NorD&subsystem_name=Denitrification@1>Nitric oxide reductase activation protein NorD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.325@1>fig|1041826.106.peg.325</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrous oxide reductase maturation protein NosD&subsystem_name=Denitrification@1>Nitrous oxide reductase maturation protein NosD</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2227@1>fig|1041826.106.peg.2227</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrous oxide reductase maturation protein NosF (ATPase)&subsystem_name=Denitrification@1>Nitrous oxide reductase maturation protein NosF (ATPase)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2226@1>fig|1041826.106.peg.2226</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrous oxide reductase maturation transmembrane protein NosY&subsystem_name=Denitrification@1>Nitrous oxide reductase maturation transmembrane protein NosY</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2225@1>fig|1041826.106.peg.2225</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrous-oxide reductase (EC 1.7.99.6)&subsystem_name=Denitrification@1>Nitrous-oxide reductase (EC 1.7.99.6)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2230@1>fig|1041826.106.peg.2230</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Copper-containing nitrite reductase (EC 1.7.2.1)&subsystem_name=Denitrification@1>Copper-containing nitrite reductase (EC 1.7.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.329@1>fig|1041826.106.peg.329</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrite reductase accessory protein NirV&subsystem_name=Denitrification@1>Nitrite reductase accessory protein NirV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.328@1>fig|1041826.106.peg.328</a>@~Nitrogen Metabolism@^Denitrification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Denitrification&organism=1041826.106@1>Denitrification</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitric-oxide reductase subunit B (EC 1.7.99.7)&subsystem_name=Denitrification@1>Nitric-oxide reductase subunit B (EC 1.7.99.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.331@1>fig|1041826.106.peg.331</a>@~Dormancy and Sporulation@^Dormancy and Sporulation - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Sporulation-associated_proteins_with_broader_functions&organism=1041826.106@1>Sporulation-associated proteins with broader functions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Peptidyl-tRNA hydrolase (EC 3.1.1.29)&subsystem_name=Sporulation-associated_proteins_with_broader_functions@1>Peptidyl-tRNA hydrolase (EC 3.1.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2108@1>fig|1041826.106.peg.2108</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit c (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit c (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1720@1>fig|1041826.106.peg.1720</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase delta chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase delta chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1718@1>fig|1041826.106.peg.1718</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase protein I&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase protein I</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1723@1>fig|1041826.106.peg.1723</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase beta chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase beta chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2699@1>fig|1041826.106.peg.2699</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase gamma chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase gamma chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1716@1>fig|1041826.106.peg.1716</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit b (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit b (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1719@1>fig|1041826.106.peg.1719</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase F0 sector subunit a (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase F0 sector subunit a (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1721@1>fig|1041826.106.peg.1721</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase alpha chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase alpha chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1717@1>fig|1041826.106.peg.1717</a>@~Respiration@^ATP synthases@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=F0F1-type_ATP_synthase&organism=1041826.106@1>F0F1-type ATP synthase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=ATP synthase epsilon chain (EC 3.6.3.14)&subsystem_name=F0F1-type_ATP_synthase@1>ATP synthase epsilon chain (EC 3.6.3.14)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2700@1>fig|1041826.106.peg.2700</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=1041826.106@1>Anaerobic respiratory reductases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Butyryl-CoA dehydrogenase (EC 1.3.8.1)&subsystem_name=Anaerobic_respiratory_reductases@1>Butyryl-CoA dehydrogenase (EC 1.3.8.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1365@1>fig|1041826.106.peg.1365</a>@~Respiration@^Electron accepting reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Anaerobic_respiratory_reductases&organism=1041826.106@1>Anaerobic respiratory reductases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Flavodoxin reductases (ferredoxin-NADPH reductases) family 1&subsystem_name=Anaerobic_respiratory_reductases@1>Flavodoxin reductases (ferredoxin-NADPH reductases) family 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1727@1>fig|1041826.106.peg.1727</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.27@1>fig|1041826.106.peg.27</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.28@1>fig|1041826.106.peg.28</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.26@1>fig|1041826.106.peg.26</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.30@1>fig|1041826.106.peg.30</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.23@1>fig|1041826.106.peg.23</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.20@1>fig|1041826.106.peg.20</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Hypothetical transmembrane protein coupled to NADH-ubiquinone oxidoreductase chain 5 homolog&subsystem_name=Respiratory_Complex_I@1>Hypothetical transmembrane protein coupled to NADH-ubiquinone oxidoreductase chain 5 homolog</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2501@1>fig|1041826.106.peg.2501</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.17@1>fig|1041826.106.peg.17</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.25@1>fig|1041826.106.peg.25</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.29@1>fig|1041826.106.peg.29</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.21@1>fig|1041826.106.peg.21</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.22@1>fig|1041826.106.peg.22</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.18@1>fig|1041826.106.peg.18</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.16@1>fig|1041826.106.peg.16</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_Complex_I&organism=1041826.106@1>Respiratory Complex I</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)&subsystem_name=Respiratory_Complex_I@1>NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.19@1>fig|1041826.106.peg.19</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=1041826.106@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH dehydrogenase (EC 1.6.99.3)&subsystem_name=Respiratory_dehydrogenases_1@1>NADH dehydrogenase (EC 1.6.99.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1285@1>fig|1041826.106.peg.1285</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Respiratory_dehydrogenases_1&organism=1041826.106@1>Respiratory dehydrogenases 1</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)&subsystem_name=Respiratory_dehydrogenases_1@1>Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.85@1>fig|1041826.106.peg.85</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain K (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.27@1>fig|1041826.106.peg.27</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain L (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.28@1>fig|1041826.106.peg.28</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain J (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.26@1>fig|1041826.106.peg.26</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain N (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.30@1>fig|1041826.106.peg.30</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain H (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.23@1>fig|1041826.106.peg.23</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain E (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.20@1>fig|1041826.106.peg.20</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain B (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.17@1>fig|1041826.106.peg.17</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain I (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.25@1>fig|1041826.106.peg.25</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain M (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.29@1>fig|1041826.106.peg.29</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain F (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.21@1>fig|1041826.106.peg.21</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain G (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.22@1>fig|1041826.106.peg.22</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain C (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.18@1>fig|1041826.106.peg.18</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH ubiquinone oxidoreductase chain A (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.16@1>fig|1041826.106.peg.16</a>@~Respiration@^Electron donating reactions@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=NADH_ubiquinone_oxidoreductase&organism=1041826.106@1>NADH ubiquinone oxidoreductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)&subsystem_name=NADH_ubiquinone_oxidoreductase@1>NADH-ubiquinone oxidoreductase chain D (EC 1.6.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.19@1>fig|1041826.106.peg.19</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cytochrome_c_oxidases&organism=1041826.106@1>Biogenesis of cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Heme A synthase, cytochrome oxidase biogenesis protein Cox15-CtaA&subsystem_name=Biogenesis_of_cytochrome_c_oxidases@1>Heme A synthase, cytochrome oxidase biogenesis protein Cox15-CtaA</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.254@1>fig|1041826.106.peg.254</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=1041826.106@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoI&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoI</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1375@1>fig|1041826.106.peg.1375</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=1041826.106@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoS, involved in heme b insertion&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoS, involved in heme b insertion</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1374@1>fig|1041826.106.peg.1374</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_cbb3-type_cytochrome_c_oxidases&organism=1041826.106@1>Biogenesis of cbb3-type cytochrome c oxidases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Type cbb3 cytochrome oxidase biogenesis protein CcoG, involved in Cu oxidation&subsystem_name=Biogenesis_of_cbb3-type_cytochrome_c_oxidases@1>Type cbb3 cytochrome oxidase biogenesis protein CcoG, involved in Cu oxidation</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1369@1>fig|1041826.106.peg.1369</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=1041826.106@1>Biogenesis of c-type cytochromes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative cytochrome C-type biogenesis protein&subsystem_name=Biogenesis_of_c-type_cytochromes@1>Putative cytochrome C-type biogenesis protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1568@1>fig|1041826.106.peg.1568</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Biogenesis_of_c-type_cytochromes&organism=1041826.106@1>Biogenesis of c-type cytochromes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)&subsystem_name=Biogenesis_of_c-type_cytochromes@1>Cytochrome c-type biogenesis protein DsbD, protein-disulfide reductase (EC 1.8.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1288@1>fig|1041826.106.peg.1288</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=1041826.106@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2113@1>fig|1041826.106.peg.2113</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=1041826.106@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Ferredoxin</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.316@1>fig|1041826.106.peg.316</a>@~Respiration@^Respiration - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Soluble_cytochromes_and_functionally_related_electron_carriers&organism=1041826.106@1>Soluble cytochromes and functionally related electron carriers</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cytochrome c552&subsystem_name=Soluble_cytochromes_and_functionally_related_electron_carriers@1>Cytochrome c552</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1976@1>fig|1041826.106.peg.1976</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=1041826.106@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aquaporin Z&subsystem_name=Osmoregulation@1>Aquaporin Z</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2122@1>fig|1041826.106.peg.2122</a>@~Stress Response@^Osmotic stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Osmoregulation&organism=1041826.106@1>Osmoregulation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol uptake facilitator protein&subsystem_name=Osmoregulation@1>Glycerol uptake facilitator protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.86@1>fig|1041826.106.peg.86</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=1041826.106@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferric uptake regulation protein FUR&subsystem_name=Oxidative_stress@1>Ferric uptake regulation protein FUR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.825@1>fig|1041826.106.peg.825</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=1041826.106@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Superoxide dismutase [Mn] (EC 1.15.1.1)&subsystem_name=Oxidative_stress@1>Superoxide dismutase [Mn] (EC 1.15.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.422@1>fig|1041826.106.peg.422</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1916@1>fig|1041826.106.peg.1916</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=1041826.106@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Nitrite-sensitive transcriptional repressor NsrR&subsystem_name=Oxidative_stress@1>Nitrite-sensitive transcriptional repressor NsrR</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1641@1>fig|1041826.106.peg.1641</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=1041826.106@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transcriptional regulator, Crp/Fnr family&subsystem_name=Oxidative_stress@1>Transcriptional regulator, Crp/Fnr family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.399@1>fig|1041826.106.peg.399</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.450@1>fig|1041826.106.peg.450</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1229@1>fig|1041826.106.peg.1229</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1376@1>fig|1041826.106.peg.1376</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=1041826.106@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Oxidative_stress@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.572@1>fig|1041826.106.peg.572</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.882@1>fig|1041826.106.peg.882</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1050@1>fig|1041826.106.peg.1050</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=1041826.106@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)&subsystem_name=Oxidative_stress@1>Superoxide dismutase [Cu-Zn] precursor (EC 1.15.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1230@1>fig|1041826.106.peg.1230</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Oxidative_stress&organism=1041826.106@1>Oxidative stress</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Organic hydroperoxide resistance protein&subsystem_name=Oxidative_stress@1>Organic hydroperoxide resistance protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.577@1>fig|1041826.106.peg.577</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutathione:_Redox_cycle&organism=1041826.106@1>Glutathione: Redox cycle</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutathione peroxidase (EC 1.11.1.9)&subsystem_name=Glutathione:_Redox_cycle@1>Glutathione peroxidase (EC 1.11.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1566@1>fig|1041826.106.peg.1566</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1567@1>fig|1041826.106.peg.1567</a>@~Stress Response@^Oxidative stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutathionylspermidine_and_Trypanothione&organism=1041826.106@1>Glutathionylspermidine and Trypanothione</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Similarity with glutathionylspermidine synthase (EC 6.3.1.8), group 1&subsystem_name=Glutathionylspermidine_and_Trypanothione@1>Similarity with glutathionylspermidine synthase (EC 6.3.1.8), group 1</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.613@1>fig|1041826.106.peg.613</a>@~Stress Response@^Detoxification@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Uptake_of_selenate_and_selenite&organism=1041826.106@1>Uptake of selenate and selenite</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=DedA protein&subsystem_name=Uptake_of_selenate_and_selenite@1>DedA protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2503@1>fig|1041826.106.peg.2503</a>@~Stress Response@^Periplasmic Stress@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Periplasmic_Stress_Response&organism=1041826.106@1>Periplasmic Stress Response</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Outer membrane protein H precursor&subsystem_name=Periplasmic_Stress_Response@1>Outer membrane protein H precursor</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.954@1>fig|1041826.106.peg.954</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1857@1>fig|1041826.106.peg.1857</a>@~Metabolism of Aromatic Compounds@^Peripheral pathways for catabolism of aromatic compounds@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Quinate_degradation&organism=1041826.106@1>Quinate degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate dehydratase II (EC 4.2.1.10)&subsystem_name=Quinate_degradation@1>3-dehydroquinate dehydratase II (EC 4.2.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1018@1>fig|1041826.106.peg.1018</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=1041826.106@1>Catechol branch of beta-ketoadipate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)&subsystem_name=Catechol_branch_of_beta-ketoadipate_pathway@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit A (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1783@1>fig|1041826.106.peg.1783</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=1041826.106@1>Catechol branch of beta-ketoadipate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Beta-ketoadipate enol-lactone hydrolase (EC 3.1.1.24)&subsystem_name=Catechol_branch_of_beta-ketoadipate_pathway@1>Beta-ketoadipate enol-lactone hydrolase (EC 3.1.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1921@1>fig|1041826.106.peg.1921</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Catechol_branch_of_beta-ketoadipate_pathway&organism=1041826.106@1>Catechol branch of beta-ketoadipate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)&subsystem_name=Catechol_branch_of_beta-ketoadipate_pathway@1>Succinyl-CoA:3-ketoacid-coenzyme A transferase subunit B (EC 2.8.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1781@1>fig|1041826.106.peg.1781</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=1041826.106@1>Salicylate and gentisate catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetase (EC 3.7.1.2)&subsystem_name=Salicylate_and_gentisate_catabolism@1>Fumarylacetoacetase (EC 3.7.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1202@1>fig|1041826.106.peg.1202</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Salicylate_and_gentisate_catabolism&organism=1041826.106@1>Salicylate and gentisate catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetate hydrolase family protein&subsystem_name=Salicylate_and_gentisate_catabolism@1>Fumarylacetoacetate hydrolase family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2534@1>fig|1041826.106.peg.2534</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=1041826.106@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homogentisate 1,2-dioxygenase (EC 1.13.11.5)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>Homogentisate 1,2-dioxygenase (EC 1.13.11.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2130@1>fig|1041826.106.peg.2130</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=1041826.106@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetase (EC 3.7.1.2)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>Fumarylacetoacetase (EC 3.7.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1202@1>fig|1041826.106.peg.1202</a>@~Metabolism of Aromatic Compounds@^Metabolism of central aromatic intermediates@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Homogentisate_pathway_of_aromatic_compound_degradation&organism=1041826.106@1>Homogentisate pathway of aromatic compound degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)&subsystem_name=Homogentisate_pathway_of_aromatic_compound_degradation@1>4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2131@1>fig|1041826.106.peg.2131</a>@~Metabolism of Aromatic Compounds@^Metabolism of Aromatic Compounds - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Gentisate_degradation&organism=1041826.106@1>Gentisate degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Fumarylacetoacetate hydrolase family protein&subsystem_name=Gentisate_degradation@1>Fumarylacetoacetate hydrolase family protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2534@1>fig|1041826.106.peg.2534</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=1041826.106@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-asparaginase I, cytoplasmic (EC 3.5.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>L-asparaginase I, cytoplasmic (EC 3.5.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1105@1>fig|1041826.106.peg.1105</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2830@1>fig|1041826.106.peg.2830</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=1041826.106@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.299@1>fig|1041826.106.peg.299</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=1041826.106@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate aminotransferase (EC 2.6.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate aminotransferase (EC 2.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.652@1>fig|1041826.106.peg.652</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2558@1>fig|1041826.106.peg.2558</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=1041826.106@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2755@1>fig|1041826.106.peg.2755</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=1041826.106@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamate racemase (EC 5.1.1.3)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Glutamate racemase (EC 5.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.955@1>fig|1041826.106.peg.955</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=1041826.106@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate ammonia-lyase (EC 4.3.1.1)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Aspartate ammonia-lyase (EC 4.3.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1046@1>fig|1041826.106.peg.1046</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis&organism=1041826.106@1>Glutamine, Glutamate, Aspartate and Asparagine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)&subsystem_name=Glutamine,_Glutamate,_Aspartate_and_Asparagine_Biosynthesis@1>Asparagine synthetase [glutamine-hydrolyzing] (EC 6.3.5.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1438@1>fig|1041826.106.peg.1438</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamate_dehydrogenases&organism=1041826.106@1>Glutamate dehydrogenases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Glutamate_dehydrogenases@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2755@1>fig|1041826.106.peg.2755</a>@~Amino Acids and Derivatives@^Glutamine, glutamate, aspartate, asparagine; ammonia assimilation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glutamine_synthetases&organism=1041826.106@1>Glutamine synthetases</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glutamine synthetase type III, GlnN (EC 6.3.1.2)&subsystem_name=Glutamine_synthetases@1>Glutamine synthetase type III, GlnN (EC 6.3.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.299@1>fig|1041826.106.peg.299</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=1041826.106@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Formiminoglutamase (EC 3.5.3.8)&subsystem_name=Histidine_Degradation@1>Formiminoglutamase (EC 3.5.3.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1414@1>fig|1041826.106.peg.1414</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=1041826.106@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Imidazolonepropionase (EC 3.5.2.7)&subsystem_name=Histidine_Degradation@1>Imidazolonepropionase (EC 3.5.2.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1422@1>fig|1041826.106.peg.1422</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=1041826.106@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Urocanate hydratase (EC 4.2.1.49)&subsystem_name=Histidine_Degradation@1>Urocanate hydratase (EC 4.2.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2238@1>fig|1041826.106.peg.2238</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2239@1>fig|1041826.106.peg.2239</a>@~Amino Acids and Derivatives@^Histidine Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Histidine_Degradation&organism=1041826.106@1>Histidine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Histidine ammonia-lyase (EC 4.3.1.3)&subsystem_name=Histidine_Degradation@1>Histidine ammonia-lyase (EC 4.3.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.170@1>fig|1041826.106.peg.170</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=1041826.106@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biosynthetic arginine decarboxylase (EC 4.1.1.19)&subsystem_name=Polyamine_Metabolism@1>Biosynthetic arginine decarboxylase (EC 4.1.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.74@1>fig|1041826.106.peg.74</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyamine_Metabolism&organism=1041826.106@1>Polyamine Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Agmatinase (EC 3.5.3.11)&subsystem_name=Polyamine_Metabolism@1>Agmatinase (EC 3.5.3.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.730@1>fig|1041826.106.peg.730</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=1041826.106@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.639@1>fig|1041826.106.peg.639</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=1041826.106@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Arginase (EC 3.5.3.1)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Arginase (EC 3.5.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.762@1>fig|1041826.106.peg.762</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=1041826.106@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ornithine aminotransferase (EC 2.6.1.13)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Ornithine aminotransferase (EC 2.6.1.13)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1515@1>fig|1041826.106.peg.1515</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=1041826.106@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Arginine_and_Ornithine_Degradation@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2755@1>fig|1041826.106.peg.2755</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=1041826.106@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Biosynthetic arginine decarboxylase (EC 4.1.1.19)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Biosynthetic arginine decarboxylase (EC 4.1.1.19)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.74@1>fig|1041826.106.peg.74</a>@~Amino Acids and Derivatives@^Arginine; urea cycle, polyamines@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Arginine_and_Ornithine_Degradation&organism=1041826.106@1>Arginine and Ornithine Degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Agmatinase (EC 3.5.3.11)&subsystem_name=Arginine_and_Ornithine_Degradation@1>Agmatinase (EC 3.5.3.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.730@1>fig|1041826.106.peg.730</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=1041826.106@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Homoserine dehydrogenase (EC 1.1.1.3)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Homoserine dehydrogenase (EC 1.1.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.846@1>fig|1041826.106.peg.846</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=1041826.106@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.270@1>fig|1041826.106.peg.270</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=1041826.106@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate aminotransferase (EC 2.6.1.1)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartate aminotransferase (EC 2.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.652@1>fig|1041826.106.peg.652</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2558@1>fig|1041826.106.peg.2558</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_and_Homoserine_Biosynthesis&organism=1041826.106@1>Threonine and Homoserine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Threonine_and_Homoserine_Biosynthesis@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.846@1>fig|1041826.106.peg.846</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.918@1>fig|1041826.106.peg.918</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Threonine_degradation&organism=1041826.106@1>Threonine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-allo-threonine aldolase (EC 2.1.2.1)&subsystem_name=Threonine_degradation@1>L-allo-threonine aldolase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.406@1>fig|1041826.106.peg.406</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate decarboxylase (EC 4.1.1.20)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Diaminopimelate decarboxylase (EC 4.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.202@1>fig|1041826.106.peg.202</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1297@1>fig|1041826.106.peg.1297</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate epimerase (EC 5.1.1.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Diaminopimelate epimerase (EC 5.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.998@1>fig|1041826.106.peg.998</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.270@1>fig|1041826.106.peg.270</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2562@1>fig|1041826.106.peg.2562</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.846@1>fig|1041826.106.peg.846</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.918@1>fig|1041826.106.peg.918</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway, GJO scratch</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway,_GJO_scratch@1>4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1493@1>fig|1041826.106.peg.1493</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_degradation&organism=1041826.106@1>Lysine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Saccharopine dehydrogenase (EC 1.5.1.9)&subsystem_name=Lysine_degradation@1>Saccharopine dehydrogenase (EC 1.5.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2400@1>fig|1041826.106.peg.2400</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate decarboxylase (EC 4.1.1.20)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Diaminopimelate decarboxylase (EC 4.1.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.202@1>fig|1041826.106.peg.202</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase (EC 2.3.1.117)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1297@1>fig|1041826.106.peg.1297</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Diaminopimelate epimerase (EC 5.1.1.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Diaminopimelate epimerase (EC 5.1.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.998@1>fig|1041826.106.peg.998</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Aspartate-semialdehyde dehydrogenase (EC 1.2.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.270@1>fig|1041826.106.peg.270</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>4-hydroxy-tetrahydrodipicolinate reductase (EC 1.17.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2562@1>fig|1041826.106.peg.2562</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aspartokinase (EC 2.7.2.4)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>Aspartokinase (EC 2.7.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.846@1>fig|1041826.106.peg.846</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.918@1>fig|1041826.106.peg.918</a>@~Amino Acids and Derivatives@^Lysine, threonine, methionine, and cysteine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Lysine_Biosynthesis_DAP_Pathway&organism=1041826.106@1>Lysine Biosynthesis DAP Pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)&subsystem_name=Lysine_Biosynthesis_DAP_Pathway@1>4-hydroxy-tetrahydrodipicolinate synthase (EC 4.3.3.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1493@1>fig|1041826.106.peg.1493</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=1041826.106@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Isoleucine_degradation@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1989@1>fig|1041826.106.peg.1989</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=1041826.106@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Isoleucine_degradation@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.100@1>fig|1041826.106.peg.100</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2618@1>fig|1041826.106.peg.2618</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=1041826.106@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Isoleucine_degradation@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2536@1>fig|1041826.106.peg.2536</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=1041826.106@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Butyryl-CoA dehydrogenase (EC 1.3.8.1)&subsystem_name=Isoleucine_degradation@1>Butyryl-CoA dehydrogenase (EC 1.3.8.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1365@1>fig|1041826.106.peg.1365</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=1041826.106@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Isoleucine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1217@1>fig|1041826.106.peg.1217</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Isoleucine_degradation&organism=1041826.106@1>Isoleucine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Isoleucine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1217@1>fig|1041826.106.peg.1217</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=1041826.106@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Valine_degradation@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1989@1>fig|1041826.106.peg.1989</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=1041826.106@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Valine_degradation@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.100@1>fig|1041826.106.peg.100</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2618@1>fig|1041826.106.peg.2618</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=1041826.106@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Valine_degradation@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2536@1>fig|1041826.106.peg.2536</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=1041826.106@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Butyryl-CoA dehydrogenase (EC 1.3.8.1)&subsystem_name=Valine_degradation@1>Butyryl-CoA dehydrogenase (EC 1.3.8.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1365@1>fig|1041826.106.peg.1365</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=1041826.106@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Valine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1217@1>fig|1041826.106.peg.1217</a>@~Amino Acids and Derivatives@^Branched-chain amino acids@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Valine_degradation&organism=1041826.106@1>Valine degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Valine_degradation@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1217@1>fig|1041826.106.peg.1217</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=1041826.106@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Shikimate kinase I (EC 2.7.1.71)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Shikimate kinase I (EC 2.7.1.71)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1661@1>fig|1041826.106.peg.1661</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=1041826.106@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Shikimate 5-dehydrogenase I alpha (EC 1.1.1.25)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.214@1>fig|1041826.106.peg.214</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=1041826.106@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate dehydratase II (EC 4.2.1.10)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>3-dehydroquinate dehydratase II (EC 4.2.1.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1018@1>fig|1041826.106.peg.1018</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=1041826.106@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I beta (EC 2.5.1.54)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>2-keto-3-deoxy-D-arabino-heptulosonate-7-phosphate synthase I beta (EC 2.5.1.54)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.721@1>fig|1041826.106.peg.721</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=1041826.106@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-dehydroquinate synthase (EC 4.2.3.4)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>3-dehydroquinate synthase (EC 4.2.3.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.76@1>fig|1041826.106.peg.76</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)&organism=1041826.106@1>Common Pathway For Synthesis of Aromatic Compounds (DAHP synthase to chorismate)</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chorismate synthase (EC 4.2.3.5)&subsystem_name=Common_Pathway_For_Synthesis_of_Aromatic_Compounds_(DAHP_synthase_to_chorismate)@1>Chorismate synthase (EC 4.2.3.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1204@1>fig|1041826.106.peg.1204</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, amidotransferase component (EC 4.1.3.27)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate synthase, amidotransferase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.223@1>fig|1041826.106.peg.223</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1028@1>fig|1041826.106.peg.1028</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.218@1>fig|1041826.106.peg.218</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.222@1>fig|1041826.106.peg.222</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.219@1>fig|1041826.106.peg.219</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Indole-3-glycerol phosphate synthase (EC 4.1.1.48)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Indole-3-glycerol phosphate synthase (EC 4.1.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.221@1>fig|1041826.106.peg.221</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, aminase component (EC 4.1.3.27)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Anthranilate synthase, aminase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.224@1>fig|1041826.106.peg.224</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.220@1>fig|1041826.106.peg.220</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Isochorismate synthase (EC 5.4.4.2)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Isochorismate synthase (EC 5.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.240@1>fig|1041826.106.peg.240</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1286@1>fig|1041826.106.peg.1286</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.&organism=1041826.106@1>Chorismate: Intermediate for synthesis of Tryptophan, PAPA antibiotics, PABA, 3-hydroxyanthranilate and more.</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Chorismate:_Intermediate_for_synthesis_of_Tryptophan,_PAPA_antibiotics,_PABA,_3-hydroxyanthranilate_and_more.@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.223@1>fig|1041826.106.peg.223</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=1041826.106@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Chorismate mutase I (EC 5.4.99.5)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Chorismate mutase I (EC 5.4.99.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.721@1>fig|1041826.106.peg.721</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phenylalanine_and_Tyrosine_Branches_from_Chorismate&organism=1041826.106@1>Phenylalanine and Tyrosine Branches from Chorismate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Prephenate dehydratase (EC 4.2.1.51)&subsystem_name=Phenylalanine_and_Tyrosine_Branches_from_Chorismate@1>Prephenate dehydratase (EC 4.2.1.51)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.720@1>fig|1041826.106.peg.720</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, amidotransferase component (EC 4.1.3.27)&subsystem_name=Tryptophan_synthesis@1>Anthranilate synthase, amidotransferase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.223@1>fig|1041826.106.peg.223</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminodeoxychorismate lyase (EC 4.1.3.38)&subsystem_name=Tryptophan_synthesis@1>Aminodeoxychorismate lyase (EC 4.1.3.38)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1028@1>fig|1041826.106.peg.1028</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase alpha chain (EC 4.2.1.20)&subsystem_name=Tryptophan_synthesis@1>Tryptophan synthase alpha chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.218@1>fig|1041826.106.peg.218</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)&subsystem_name=Tryptophan_synthesis@1>Para-aminobenzoate synthase, aminase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1286@1>fig|1041826.106.peg.1286</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate phosphoribosyltransferase (EC 2.4.2.18)&subsystem_name=Tryptophan_synthesis@1>Anthranilate phosphoribosyltransferase (EC 2.4.2.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.222@1>fig|1041826.106.peg.222</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan synthase beta chain (EC 4.2.1.20)&subsystem_name=Tryptophan_synthesis@1>Tryptophan synthase beta chain (EC 4.2.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.219@1>fig|1041826.106.peg.219</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Indole-3-glycerol phosphate synthase (EC 4.1.1.48)&subsystem_name=Tryptophan_synthesis@1>Indole-3-glycerol phosphate synthase (EC 4.1.1.48)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.221@1>fig|1041826.106.peg.221</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)&subsystem_name=Tryptophan_synthesis@1>Para-aminobenzoate synthase, amidotransferase component (EC 2.6.1.85)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.223@1>fig|1041826.106.peg.223</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Anthranilate synthase, aminase component (EC 4.1.3.27)&subsystem_name=Tryptophan_synthesis@1>Anthranilate synthase, aminase component (EC 4.1.3.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.224@1>fig|1041826.106.peg.224</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Tryptophan_synthesis&organism=1041826.106@1>Tryptophan synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoribosylanthranilate isomerase (EC 5.3.1.24)&subsystem_name=Tryptophan_synthesis@1>Phosphoribosylanthranilate isomerase (EC 5.3.1.24)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.220@1>fig|1041826.106.peg.220</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=1041826.106@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Tryptophan 2,3-dioxygenase (EC 1.13.11.11)&subsystem_name=Aromatic_amino_acid_degradation@1>Tryptophan 2,3-dioxygenase (EC 1.13.11.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2096@1>fig|1041826.106.peg.2096</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=1041826.106@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)&subsystem_name=Aromatic_amino_acid_degradation@1>4-hydroxyphenylpyruvate dioxygenase (EC 1.13.11.27)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2131@1>fig|1041826.106.peg.2131</a>@~Amino Acids and Derivatives@^Aromatic amino acids and derivatives@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Aromatic_amino_acid_degradation&organism=1041826.106@1>Aromatic amino acid degradation</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phenylalanine-4-hydroxylase (EC 1.14.16.1)&subsystem_name=Aromatic_amino_acid_degradation@1>Phenylalanine-4-hydroxylase (EC 1.14.16.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2242@1>fig|1041826.106.peg.2242</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=1041826.106@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyrroline-5-carboxylate reductase (EC 1.5.1.2)&subsystem_name=Proline_Synthesis@1>Pyrroline-5-carboxylate reductase (EC 1.5.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.409@1>fig|1041826.106.peg.409</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline_Synthesis&organism=1041826.106@1>Proline Synthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-specific glutamate dehydrogenase (EC 1.4.1.4)&subsystem_name=Proline_Synthesis@1>NADP-specific glutamate dehydrogenase (EC 1.4.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2755@1>fig|1041826.106.peg.2755</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=1041826.106@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=4-hydroxyproline epimerase (EC 5.1.1.8)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>4-hydroxyproline epimerase (EC 5.1.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2260@1>fig|1041826.106.peg.2260</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=1041826.106@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=1-pyrroline-4-hydroxy-2-carboxylate deaminase (EC 3.5.4.22)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>1-pyrroline-4-hydroxy-2-carboxylate deaminase (EC 3.5.4.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2262@1>fig|1041826.106.peg.2262</a>@~Amino Acids and Derivatives@^Proline and 4-hydroxyproline@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Proline,_4-hydroxyproline_uptake_and_utilization&organism=1041826.106@1>Proline, 4-hydroxyproline uptake and utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)&subsystem_name=Proline,_4-hydroxyproline_uptake_and_utilization@1>Delta-1-pyrroline-5-carboxylate dehydrogenase (EC 1.2.1.88)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.639@1>fig|1041826.106.peg.639</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=1041826.106@1>Glycine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)&subsystem_name=Glycine_Biosynthesis@1>2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.131@1>fig|1041826.106.peg.131</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1918@1>fig|1041826.106.peg.1918</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_Biosynthesis&organism=1041826.106@1>Glycine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Glycine_Biosynthesis@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1203@1>fig|1041826.106.peg.1203</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=1041826.106@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Alanine_biosynthesis@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.100@1>fig|1041826.106.peg.100</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2618@1>fig|1041826.106.peg.2618</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=1041826.106@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ferredoxin, 2Fe-2S&subsystem_name=Alanine_biosynthesis@1>Ferredoxin, 2Fe-2S</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2113@1>fig|1041826.106.peg.2113</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=1041826.106@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cysteine desulfurase (EC 2.8.1.7)&subsystem_name=Alanine_biosynthesis@1>Cysteine desulfurase (EC 2.8.1.7)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.167@1>fig|1041826.106.peg.167</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.377@1>fig|1041826.106.peg.377</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Alanine_biosynthesis&organism=1041826.106@1>Alanine biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine racemase (EC 5.1.1.1)&subsystem_name=Alanine_biosynthesis@1>Alanine racemase (EC 5.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.292@1>fig|1041826.106.peg.292</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=1041826.106@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Serine_Biosynthesis@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.313@1>fig|1041826.106.peg.313</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2012@1>fig|1041826.106.peg.2012</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2421@1>fig|1041826.106.peg.2421</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=1041826.106@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Serine_Biosynthesis@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1203@1>fig|1041826.106.peg.1203</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Serine_Biosynthesis&organism=1041826.106@1>Serine Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Serine_Biosynthesis@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.314@1>fig|1041826.106.peg.314</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=1041826.106@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine cleavage system H protein&subsystem_name=Glycine_cleavage_system@1>Glycine cleavage system H protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2170@1>fig|1041826.106.peg.2170</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=1041826.106@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)&subsystem_name=Glycine_cleavage_system@1>Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2389@1>fig|1041826.106.peg.2389</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=1041826.106@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide dehydrogenase (EC 1.8.1.4)&subsystem_name=Glycine_cleavage_system@1>Dihydrolipoamide dehydrogenase (EC 1.8.1.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.195@1>fig|1041826.106.peg.195</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.243@1>fig|1041826.106.peg.243</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_cleavage_system&organism=1041826.106@1>Glycine cleavage system</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)&subsystem_name=Glycine_cleavage_system@1>Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.169@1>fig|1041826.106.peg.169</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)&subsystem_name=Glycine_and_Serine_Utilization@1>D-3-phosphoglycerate dehydrogenase (EC 1.1.1.95)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.313@1>fig|1041826.106.peg.313</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2012@1>fig|1041826.106.peg.2012</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2421@1>fig|1041826.106.peg.2421</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, alpha subunit (EC 4.3.1.17)&subsystem_name=Glycine_and_Serine_Utilization@1>L-serine dehydratase, alpha subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.206@1>fig|1041826.106.peg.206</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, beta subunit (EC 4.3.1.17)&subsystem_name=Glycine_and_Serine_Utilization@1>L-serine dehydratase, beta subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.206@1>fig|1041826.106.peg.206</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine gamma-lyase (EC 4.4.1.1)&subsystem_name=Glycine_and_Serine_Utilization@1>Cystathionine gamma-lyase (EC 4.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2753@1>fig|1041826.106.peg.2753</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Serine hydroxymethyltransferase (EC 2.1.2.1)&subsystem_name=Glycine_and_Serine_Utilization@1>Serine hydroxymethyltransferase (EC 2.1.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1203@1>fig|1041826.106.peg.1203</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Seryl-tRNA synthetase (EC 6.1.1.11)&subsystem_name=Glycine_and_Serine_Utilization@1>Seryl-tRNA synthetase (EC 6.1.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2002@1>fig|1041826.106.peg.2002</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoserine aminotransferase (EC 2.6.1.52)&subsystem_name=Glycine_and_Serine_Utilization@1>Phosphoserine aminotransferase (EC 2.6.1.52)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.314@1>fig|1041826.106.peg.314</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine cleavage system H protein&subsystem_name=Glycine_and_Serine_Utilization@1>Glycine cleavage system H protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2170@1>fig|1041826.106.peg.2170</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)&subsystem_name=Glycine_and_Serine_Utilization@1>Glycine dehydrogenase [decarboxylating] (glycine cleavage system P protein) (EC 1.4.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2389@1>fig|1041826.106.peg.2389</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)&subsystem_name=Glycine_and_Serine_Utilization@1>2-amino-3-ketobutyrate coenzyme A ligase (EC 2.3.1.29)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.131@1>fig|1041826.106.peg.131</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1918@1>fig|1041826.106.peg.1918</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)&subsystem_name=Glycine_and_Serine_Utilization@1>Aminomethyltransferase (glycine cleavage system T protein) (EC 2.1.2.10)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.169@1>fig|1041826.106.peg.169</a>@~Amino Acids and Derivatives@^Alanine, serine, and glycine@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycine_and_Serine_Utilization&organism=1041826.106@1>Glycine and Serine Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Cystathionine beta-synthase (EC 4.2.1.22)&subsystem_name=Glycine_and_Serine_Utilization@1>Cystathionine beta-synthase (EC 4.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.212@1>fig|1041826.106.peg.212</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2203@1>fig|1041826.106.peg.2203</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=1041826.106@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiol peroxidase, Tpx-type (EC 1.11.1.15)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thiol peroxidase, Tpx-type (EC 1.11.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.565@1>fig|1041826.106.peg.565</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=1041826.106@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thiol peroxidase, Bcp-type (EC 1.11.1.15)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thiol peroxidase, Bcp-type (EC 1.11.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.90@1>fig|1041826.106.peg.90</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=1041826.106@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkyl hydroperoxide reductase subunit C-like protein&subsystem_name=Thioredoxin-disulfide_reductase@1>Alkyl hydroperoxide reductase subunit C-like protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.572@1>fig|1041826.106.peg.572</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.882@1>fig|1041826.106.peg.882</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1050@1>fig|1041826.106.peg.1050</a>@~Sulfur Metabolism@^Sulfur Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Thioredoxin-disulfide_reductase&organism=1041826.106@1>Thioredoxin-disulfide reductase</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Thioredoxin reductase (EC 1.8.1.9)&subsystem_name=Thioredoxin-disulfide_reductase@1>Thioredoxin reductase (EC 1.8.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1683@1>fig|1041826.106.peg.1683</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2526@1>fig|1041826.106.peg.2526</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=1041826.106@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alkaline phosphatase (EC 3.1.3.1)&subsystem_name=Phosphate_metabolism@1>Alkaline phosphatase (EC 3.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2023@1>fig|1041826.106.peg.2023</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2557@1>fig|1041826.106.peg.2557</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=1041826.106@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Inorganic pyrophosphatase (EC 3.6.1.1)&subsystem_name=Phosphate_metabolism@1>Inorganic pyrophosphatase (EC 3.6.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.872@1>fig|1041826.106.peg.872</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=1041826.106@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Phosphate_metabolism@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.493@1>fig|1041826.106.peg.493</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=1041826.106@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exopolyphosphatase (EC 3.6.1.11)&subsystem_name=Phosphate_metabolism@1>Exopolyphosphatase (EC 3.6.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.494@1>fig|1041826.106.peg.494</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.663@1>fig|1041826.106.peg.663</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.670@1>fig|1041826.106.peg.670</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=1041826.106@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate starvation-inducible protein PhoH, predicted ATPase&subsystem_name=Phosphate_metabolism@1>Phosphate starvation-inducible protein PhoH, predicted ATPase</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1644@1>fig|1041826.106.peg.1644</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1644@1>fig|1041826.106.peg.1644</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=1041826.106@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Probable low-affinity inorganic phosphate transporter&subsystem_name=Phosphate_metabolism@1>Probable low-affinity inorganic phosphate transporter</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1280@1>fig|1041826.106.peg.1280</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Phosphate_metabolism&organism=1041826.106@1>Phosphate metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphate transport regulator (distant homolog of PhoU)&subsystem_name=Phosphate_metabolism@1>Phosphate transport regulator (distant homolog of PhoU)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1281@1>fig|1041826.106.peg.1281</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=1041826.106@1>Polyphosphate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Polyphosphate kinase (EC 2.7.4.1)&subsystem_name=Polyphosphate@1>Polyphosphate kinase (EC 2.7.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.493@1>fig|1041826.106.peg.493</a>@~Phosphorus Metabolism@^Phosphorus Metabolism - no subcategory@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Polyphosphate&organism=1041826.106@1>Polyphosphate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Exopolyphosphatase (EC 3.6.1.11)&subsystem_name=Polyphosphate@1>Exopolyphosphatase (EC 3.6.1.11)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.494@1>fig|1041826.106.peg.494</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.663@1>fig|1041826.106.peg.663</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.670@1>fig|1041826.106.peg.670</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=1041826.106@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylglyoxal synthase (EC 4.2.3.3)&subsystem_name=Methylglyoxal_Metabolism@1>Methylglyoxal synthase (EC 4.2.3.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1398@1>fig|1041826.106.peg.1398</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=1041826.106@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase B (EC 1.2.1.22)&subsystem_name=Methylglyoxal_Metabolism@1>Aldehyde dehydrogenase B (EC 1.2.1.22)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2752@1>fig|1041826.106.peg.2752</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Methylglyoxal_Metabolism&organism=1041826.106@1>Methylglyoxal Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase (EC 1.2.1.3)&subsystem_name=Methylglyoxal_Metabolism@1>Aldehyde dehydrogenase (EC 1.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.296@1>fig|1041826.106.peg.296</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.296@1>fig|1041826.106.peg.296</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=1041826.106@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Aldehyde dehydrogenase (EC 1.2.1.3)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Aldehyde dehydrogenase (EC 1.2.1.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.296@1>fig|1041826.106.peg.296</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=1041826.106@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.870@1>fig|1041826.106.peg.870</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=1041826.106@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2284@1>fig|1041826.106.peg.2284</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=1041826.106@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2285@1>fig|1041826.106.peg.2285</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate&organism=1041826.106@1>Pyruvate metabolism II: acetyl-CoA, acetogenesis from pyruvate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NAD-dependent protein deacetylase of SIR2 family&subsystem_name=Pyruvate_metabolism_II:_acetyl-CoA,_acetogenesis_from_pyruvate@1>NAD-dependent protein deacetylase of SIR2 family</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1241@1>fig|1041826.106.peg.1241</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=1041826.106@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Phosphoenolpyruvate carboxykinase [ATP] (EC 4.1.1.49)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1909@1>fig|1041826.106.peg.1909</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=1041826.106@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADP-dependent malic enzyme (EC 1.1.1.40)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>NADP-dependent malic enzyme (EC 1.1.1.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2167@1>fig|1041826.106.peg.2167</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP&organism=1041826.106@1>Pyruvate metabolism I: anaplerotic reactions, PEP</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate kinase (EC 2.7.1.40)&subsystem_name=Pyruvate_metabolism_I:_anaplerotic_reactions,_PEP@1>Pyruvate kinase (EC 2.7.1.40)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2792@1>fig|1041826.106.peg.2792</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=1041826.106@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, alpha subunit (EC 4.3.1.17)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>L-serine dehydratase, alpha subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.206@1>fig|1041826.106.peg.206</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=1041826.106@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain amino acid aminotransferase (EC 2.6.1.42)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Branched-chain amino acid aminotransferase (EC 2.6.1.42)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.100@1>fig|1041826.106.peg.100</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2618@1>fig|1041826.106.peg.2618</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=1041826.106@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=L-serine dehydratase, beta subunit (EC 4.3.1.17)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>L-serine dehydratase, beta subunit (EC 4.3.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.206@1>fig|1041826.106.peg.206</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=1041826.106@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine dehydrogenase (EC 1.4.1.1)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Alanine dehydrogenase (EC 1.4.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1261@1>fig|1041826.106.peg.1261</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pyruvate_Alanine_Serine_Interconversions&organism=1041826.106@1>Pyruvate Alanine Serine Interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Alanine racemase (EC 5.1.1.1)&subsystem_name=Pyruvate_Alanine_Serine_Interconversions@1>Alanine racemase (EC 5.1.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.292@1>fig|1041826.106.peg.292</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=1041826.106@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transketolase, N-terminal section (EC 2.2.1.1)&subsystem_name=Pentose_phosphate_pathway@1>Transketolase, N-terminal section (EC 2.2.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.133@1>fig|1041826.106.peg.133</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=1041826.106@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)&subsystem_name=Pentose_phosphate_pathway@1>Ribose-phosphate pyrophosphokinase (EC 2.7.6.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2110@1>fig|1041826.106.peg.2110</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2723@1>fig|1041826.106.peg.2723</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=1041826.106@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transaldolase (EC 2.2.1.2)&subsystem_name=Pentose_phosphate_pathway@1>Transaldolase (EC 2.2.1.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.185@1>fig|1041826.106.peg.185</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.186@1>fig|1041826.106.peg.186</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=1041826.106@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Transketolase, C-terminal section (EC 2.2.1.1)&subsystem_name=Pentose_phosphate_pathway@1>Transketolase, C-terminal section (EC 2.2.1.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.132@1>fig|1041826.106.peg.132</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Pentose_phosphate_pathway&organism=1041826.106@1>Pentose phosphate pathway</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribulose-phosphate 3-epimerase (EC 5.1.3.1)&subsystem_name=Pentose_phosphate_pathway@1>Ribulose-phosphate 3-epimerase (EC 5.1.3.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.354@1>fig|1041826.106.peg.354</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycolate,_glyoxylate_interconversions&organism=1041826.106@1>Glycolate, glyoxylate interconversions</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphoglycolate phosphatase (EC 3.1.3.18)&subsystem_name=Glycolate,_glyoxylate_interconversions@1>Phosphoglycolate phosphatase (EC 3.1.3.18)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.381@1>fig|1041826.106.peg.381</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=1041826.106@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide succinyltransferase component (E2) of 2-oxoglutarate dehydrogenase complex (EC 2.3.1.61)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2018@1>fig|1041826.106.peg.2018</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=1041826.106@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide acyltransferase component of branched-chain alpha-keto acid dehydrogenase complex (EC 2.3.1.168)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2536@1>fig|1041826.106.peg.2536</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=1041826.106@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)&subsystem_name=Dehydrogenase_complexes@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, beta subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1217@1>fig|1041826.106.peg.1217</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=1041826.106@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)&subsystem_name=Dehydrogenase_complexes@1>Branched-chain alpha-keto acid dehydrogenase, E1 component, alpha subunit (EC 1.2.4.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1217@1>fig|1041826.106.peg.1217</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=1041826.106@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)&subsystem_name=Dehydrogenase_complexes@1>Pyruvate dehydrogenase E1 component beta subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.870@1>fig|1041826.106.peg.870</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=1041826.106@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)&subsystem_name=Dehydrogenase_complexes@1>Dihydrolipoamide acetyltransferase component of pyruvate dehydrogenase complex (EC 2.3.1.12)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2284@1>fig|1041826.106.peg.2284</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=1041826.106@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)&subsystem_name=Dehydrogenase_complexes@1>2-oxoglutarate dehydrogenase E1 component (EC 1.2.4.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2019@1>fig|1041826.106.peg.2019</a>@~Carbohydrates@^Central carbohydrate metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Dehydrogenase_complexes&organism=1041826.106@1>Dehydrogenase complexes</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)&subsystem_name=Dehydrogenase_complexes@1>Pyruvate dehydrogenase E1 component alpha subunit (EC 1.2.4.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2285@1>fig|1041826.106.peg.2285</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=1041826.106@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>5-formyltetrahydrofolate cyclo-ligase (EC 6.3.3.2)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1915@1>fig|1041826.106.peg.1915</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=1041826.106@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Methylenetetrahydrofolate dehydrogenase (NADP+) (EC 1.5.1.5)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1264@1>fig|1041826.106.peg.1264</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=1041826.106@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>5,10-methylenetetrahydrofolate reductase (EC 1.5.1.20)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.843@1>fig|1041826.106.peg.843</a>@~Carbohydrates@^One-carbon Metabolism@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=One-carbon_metabolism_by_tetrahydropterines&organism=1041826.106@1>One-carbon metabolism by tetrahydropterines</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9)&subsystem_name=One-carbon_metabolism_by_tetrahydropterines@1>Methenyltetrahydrofolate cyclohydrolase (EC 3.5.4.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1264@1>fig|1041826.106.peg.1264</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=1041826.106@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=NADH-dependent butanol dehydrogenase A (EC 1.1.1.-)&subsystem_name=Butanol_Biosynthesis@1>NADH-dependent butanol dehydrogenase A (EC 1.1.1.-)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2725@1>fig|1041826.106.peg.2725</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=1041826.106@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Butanol_Biosynthesis@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1989@1>fig|1041826.106.peg.1989</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=1041826.106@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Butyryl-CoA dehydrogenase (EC 1.3.8.1)&subsystem_name=Butanol_Biosynthesis@1>Butyryl-CoA dehydrogenase (EC 1.3.8.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1365@1>fig|1041826.106.peg.1365</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=1041826.106@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)&subsystem_name=Butanol_Biosynthesis@1>3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1610@1>fig|1041826.106.peg.1610</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Butanol_Biosynthesis&organism=1041826.106@1>Butanol Biosynthesis</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Butanol_Biosynthesis@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1347@1>fig|1041826.106.peg.1347</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=1041826.106@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Enoyl-CoA hydratase (EC 4.2.1.17)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Enoyl-CoA hydratase (EC 4.2.1.17)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1989@1>fig|1041826.106.peg.1989</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=1041826.106@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Electron transfer flavoprotein, beta subunit&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Electron transfer flavoprotein, beta subunit</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.869@1>fig|1041826.106.peg.869</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=1041826.106@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Butyryl-CoA dehydrogenase (EC 1.3.8.1)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Butyryl-CoA dehydrogenase (EC 1.3.8.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1365@1>fig|1041826.106.peg.1365</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=1041826.106@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Electron transfer flavoprotein, alpha subunit&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Electron transfer flavoprotein, alpha subunit</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.868@1>fig|1041826.106.peg.868</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=1041826.106@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>3-hydroxybutyryl-CoA dehydrogenase (EC 1.1.1.157)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1610@1>fig|1041826.106.peg.1610</a>@~Carbohydrates@^Fermentation@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Acetyl-CoA_fermentation_to_Butyrate&organism=1041826.106@1>Acetyl-CoA fermentation to Butyrate</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Acetyl-CoA acetyltransferase (EC 2.3.1.9)&subsystem_name=Acetyl-CoA_fermentation_to_Butyrate@1>Acetyl-CoA acetyltransferase (EC 2.3.1.9)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1347@1>fig|1041826.106.peg.1347</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=1041826.106@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerophosphoryl diester phosphodiesterase (EC 3.1.4.46)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.168@1>fig|1041826.106.peg.168</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=1041826.106@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol-3-phosphate dehydrogenase (EC 1.1.5.3)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.85@1>fig|1041826.106.peg.85</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=1041826.106@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol-3-phosphate dehydrogenase [NAD(P)+] (EC 1.1.1.94)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol-3-phosphate dehydrogenase [NAD(P)+] (EC 1.1.1.94)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2727@1>fig|1041826.106.peg.2727</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=1041826.106@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol kinase (EC 2.7.1.30)&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol kinase (EC 2.7.1.30)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.84@1>fig|1041826.106.peg.84</a>@~Carbohydrates@^Sugar alcohols@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization&organism=1041826.106@1>Glycerol and Glycerol-3-phosphate Uptake and Utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Glycerol uptake facilitator protein&subsystem_name=Glycerol_and_Glycerol-3-phosphate_Uptake_and_Utilization@1>Glycerol uptake facilitator protein</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.86@1>fig|1041826.106.peg.86</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=1041826.106@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Phosphomannomutase (EC 5.4.2.8)&subsystem_name=Mannose_Metabolism@1>Phosphomannomutase (EC 5.4.2.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.92@1>fig|1041826.106.peg.92</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Mannose_Metabolism&organism=1041826.106@1>Mannose Metabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Mannose-6-phosphate isomerase (EC 5.3.1.8)&subsystem_name=Mannose_Metabolism@1>Mannose-6-phosphate isomerase (EC 5.3.1.8)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1049@1>fig|1041826.106.peg.1049</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=D-ribose_utilization&organism=1041826.106@1>D-ribose utilization</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribokinase (EC 2.7.1.15)&subsystem_name=D-ribose_utilization@1>Ribokinase (EC 2.7.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2745@1>fig|1041826.106.peg.2745</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=1041826.106@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Purine nucleoside phosphorylase (EC 2.4.2.1)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Purine nucleoside phosphorylase (EC 2.4.2.1)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.232@1>fig|1041826.106.peg.232</a>, <br><a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.368@1>fig|1041826.106.peg.368</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=1041826.106@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Ribokinase (EC 2.7.1.15)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Ribokinase (EC 2.7.1.15)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2745@1>fig|1041826.106.peg.2745</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=1041826.106@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Putative deoxyribonuclease YjjV&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Putative deoxyribonuclease YjjV</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.1427@1>fig|1041826.106.peg.1427</a>@~Carbohydrates@^Monosaccharides@^<a href=@1seedviewer.cgi?page=Subsystems&subsystem=Deoxyribose_and_Deoxynucleoside_Catabolism&organism=1041826.106@1>Deoxyribose and Deoxynucleoside Catabolism</a>@^<a href=@1seedviewer.cgi?page=FunctionalRole&role=Deoxyribose-phosphate aldolase (EC 4.1.2.4)&subsystem_name=Deoxyribose_and_Deoxynucleoside_Catabolism@1>Deoxyribose-phosphate aldolase (EC 4.1.2.4)</a>@^<a href=@1seedviewer.cgi?page=Annotation&feature=fig|1041826.106.peg.2172@1>fig|1041826.106.peg.2172</a>'>
<input type='hidden' id='table_onclicks_0' value='@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^'>
<input type='hidden' id='table_highlights_0' value='@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^@~@^@^@^@^'>
<input type='hidden' id='table_filtereddata_0' value=''>
<input type='hidden' id='table_rows_0' value='738'>
<input type='hidden' id='table_cols_0' value='5'>
<input type='hidden' id='table_start_0' value='0'>
<input type='hidden' id='table_sortdirection_0' value='up'>
<input type='hidden' id='table_column_types_0' value='@~@~@~@~'>
<input type='hidden' id='table_hoverid_0' value='1'>
<span id='table_input_space_0'></span><input type='hidden' id='ajax_url' value='https://rast.nmpdr.org/ajax.cgi'>
<input type='hidden' id='ajax_params' value='page=Organism&app=SeedViewer&url=seedviewer.cgi&cookies=0' />
<div style='display:none;' id='table_0_download'></div><table id='table_0' class='table_table' style=''><tr><td name='0_col_1' id='0_col_1' class='table_first_row' style=''><a href='javascript: table_sort("0", "1");' class='table_first_row' title='click to sort'><span id='colname_0_col_1'>Category</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_1_operator' value='equal' id='table_0_operator_1'><select name='0_col_1' class='filter_item' id='table_0_operand_1' onchange='return check_submit_filter2("0", this.options[this.selectedIndex].text);' style='width: 100%;' title='Select Filter'></select></td><td name='0_col_2' id='0_col_2' class='table_first_row' style=''><a href='javascript: table_sort("0", "2");' class='table_first_row' title='click to sort'><span id='colname_0_col_2'>Subcategory</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_2_operator' value='equal' id='table_0_operator_2'><select name='0_col_2' class='filter_item' id='table_0_operand_2' onchange='return check_submit_filter2("0", this.options[this.selectedIndex].text);' style='width: 100%;' title='Select Filter'></select></td><td name='0_col_3' id='0_col_3' class='table_first_row' style=''><a href='javascript: table_sort("0", "3");' class='table_first_row' title='click to sort'><span id='colname_0_col_3'>Subsystem</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_3_operator' value='like' id='table_0_operator_3'><input type='text' name='0_col_3' class='filter_item' value='' size=5 id='table_0_operand_3' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td><td name='0_col_4' id='0_col_4' class='table_first_row' style=''><a href='javascript: table_sort("0", "4");' class='table_first_row' title='click to sort'><span id='colname_0_col_4'>Role</span>&nbsp;<img src="./Html/up-arrow.gif"><img src="./Html/down-arrow.gif"></a><br><input type=hidden name='0_col_4_operator' value='like' id='table_0_operator_4'><input type='text' name='0_col_4' class='filter_item' value='' size=5 id='table_0_operand_4' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td><td name='0_col_5' id='0_col_5' class='table_first_row' style=''><span id='colname_0_col_5'>Features</span><br><input type=hidden name='0_col_5_operator' value='like' id='table_0_operator_5'><input type='text' name='0_col_5' class='filter_item' value='' size=5 id='table_0_operand_5' onkeypress='return check_submit_filter(event, "0");' style='width: 100%;' title='Enter Search Text'></td></table><input type='hidden' id='table_visible_columns_0' value='1@~1@~1@~1@~1'>
<input type='hidden' id='table_combo_columns_0' value='1@~2'>
<input type='hidden' id='table_sc_0' value=''><img src='https://rast.nmpdr.org/Html/clear.gif' onload='initialize_table("0")'><input type='hidden' id='hover_redundancies_1' value=''></td></tr><tr><td style='width: 100%; text-align: center;'><table style='width: 100%;'><tr><td align='left' width='20%'><a href='javascript: table_first("0");' name='table_first_0'>&laquo;first</a>&nbsp;&nbsp;<a href='javascript: table_prev("0");' name='table_prev_0'>&laquo;prev</a></td><td align='center' width='60%'>displaying <span id='table_start_bottom_0'>1</span> - <span id='table_stop_bottom_0'>738</span> of <span id='table_total_bottom_0'>738</span></td><td align='right' width='20%'><a href='javascript: table_next("0");' name='table_next_0'>next&raquo;</a>&nbsp;&nbsp;<a href='javascript: table_last("0");' name='table_last_0'>last&raquo;</a></td></tr></table></td></tr></table></div></td></tr></table>
<img src='https://rast.nmpdr.org/Html/clear.gif'  onload='initialize_tab_view("1"); tab_view_select("1", "0");'>
<br><br><!-- tracking off -->
  </div>
    
</body>


</html>

